<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Data analysis in the current era | Museum of Spatial Transcriptomics</title>
  <meta name="description" content="Chapter 7 Data analysis in the current era | Museum of Spatial Transcriptomics" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Data analysis in the current era | Museum of Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Chapter 7 Data analysis in the current era | Museum of Spatial Transcriptomics" />
  <meta name="github-repo" content="pachterlab/LP_2021" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Data analysis in the current era | Museum of Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="Chapter 7 Data analysis in the current era | Museum of Spatial Transcriptomics" />
  

<meta name="author" content="Lambda Moses, dlu2@caltech.edu" />
<meta name="author" content="Lior Pachter, lpachter@caltech.edu" />


<meta name="date" content="2021-05-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="text-mining.html"/>
<link rel="next" href="future.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Museum of Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="part"><span><b>I Prequel era</b></span></li>
<li class="chapter" data-level="2" data-path="prequel.html"><a href="prequel.html"><i class="fa fa-check"></i><b>2</b> Prequel era</a><ul>
<li class="chapter" data-level="2.1" data-path="prequel.html"><a href="prequel.html#traps"><i class="fa fa-check"></i><b>2.1</b> Enhancer and gene traps</a></li>
<li class="chapter" data-level="2.2" data-path="prequel.html"><a href="prequel.html#in-situ-reporter"><i class="fa fa-check"></i><b>2.2</b> In situ reporter</a></li>
<li class="chapter" data-level="2.3" data-path="prequel.html"><a href="prequel.html#wmish"><i class="fa fa-check"></i><b>2.3</b> ISH and WMISH atlases</a></li>
<li class="chapter" data-level="2.4" data-path="prequel.html"><a href="prequel.html#database_prequel"><i class="fa fa-check"></i><b>2.4</b> Databases of the prequel era</a></li>
<li class="chapter" data-level="2.5" data-path="prequel.html"><a href="prequel.html#geo_prequel"><i class="fa fa-check"></i><b>2.5</b> Geography of the prequel era</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prequel-analysis.html"><a href="prequel-analysis.html"><i class="fa fa-check"></i><b>3</b> Data analysis in the prequel era</a><ul>
<li class="chapter" data-level="3.1" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-patterns"><i class="fa fa-check"></i><b>3.1</b> Gene patterns</a></li>
<li class="chapter" data-level="3.2" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-spatial-regions"><i class="fa fa-check"></i><b>3.2</b> Spatial regions</a></li>
<li class="chapter" data-level="3.3" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-interactions"><i class="fa fa-check"></i><b>3.3</b> Gene interactions</a></li>
<li class="chapter" data-level="3.4" data-path="prequel-analysis.html"><a href="prequel-analysis.html#decline"><i class="fa fa-check"></i><b>3.4</b> Decline</a></li>
<li class="chapter" data-level="3.5" data-path="prequel-analysis.html"><a href="prequel-analysis.html#geo-prequel-analysis"><i class="fa fa-check"></i><b>3.5</b> Geography of prequel data analysis</a></li>
</ul></li>
<li class="part"><span><b>II Current era</b></span></li>
<li class="chapter" data-level="4" data-path="current.html"><a href="current.html"><i class="fa fa-check"></i><b>4</b> From the past to the present</a></li>
<li class="chapter" data-level="5" data-path="current-techs.html"><a href="current-techs.html"><i class="fa fa-check"></i><b>5</b> Current era technologies</a><ul>
<li class="chapter" data-level="5.1" data-path="current-techs.html"><a href="current-techs.html#microdissection"><i class="fa fa-check"></i><b>5.1</b> Microdissection</a><ul>
<li class="chapter" data-level="5.1.1" data-path="current-techs.html"><a href="current-techs.html#history-of-lcm"><i class="fa fa-check"></i><b>5.1.1</b> History of LCM</a></li>
<li class="chapter" data-level="5.1.2" data-path="current-techs.html"><a href="current-techs.html#lcm-usage"><i class="fa fa-check"></i><b>5.1.2</b> Usage of LCM</a></li>
<li class="chapter" data-level="5.1.3" data-path="current-techs.html"><a href="current-techs.html#physical-microdissection"><i class="fa fa-check"></i><b>5.1.3</b> Physical microdissection</a></li>
<li class="chapter" data-level="5.1.4" data-path="current-techs.html"><a href="current-techs.html#de-facto-microdissection"><i class="fa fa-check"></i><b>5.1.4</b> <em>De facto</em> microdissection</a></li>
<li class="chapter" data-level="5.1.5" data-path="current-techs.html"><a href="current-techs.html#summary"><i class="fa fa-check"></i><b>5.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="current-techs.html"><a href="current-techs.html#smfish"><i class="fa fa-check"></i><b>5.2</b> Single molecular FISH</a><ul>
<li class="chapter" data-level="5.2.1" data-path="current-techs.html"><a href="current-techs.html#barcoding"><i class="fa fa-check"></i><b>5.2.1</b> Barcoding strategies</a></li>
<li class="chapter" data-level="5.2.2" data-path="current-techs.html"><a href="current-techs.html#amplification"><i class="fa fa-check"></i><b>5.2.2</b> Signal amplification</a></li>
<li class="chapter" data-level="5.2.3" data-path="current-techs.html"><a href="current-techs.html#crowding"><i class="fa fa-check"></i><b>5.2.3</b> Optical crowding</a></li>
<li class="chapter" data-level="5.2.4" data-path="current-techs.html"><a href="current-techs.html#smfish-usage"><i class="fa fa-check"></i><b>5.2.4</b> Usage of smFISH based techniques</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="current-techs.html"><a href="current-techs.html#iss"><i class="fa fa-check"></i><b>5.3</b> In situ sequencing</a><ul>
<li class="chapter" data-level="5.3.1" data-path="current-techs.html"><a href="current-techs.html#solid"><i class="fa fa-check"></i><b>5.3.1</b> SOLiD and FISSEQ</a></li>
<li class="chapter" data-level="5.3.2" data-path="current-techs.html"><a href="current-techs.html#cpal"><i class="fa fa-check"></i><b>5.3.2</b> cPAL and ISS</a></li>
<li class="chapter" data-level="5.3.3" data-path="current-techs.html"><a href="current-techs.html#sedal"><i class="fa fa-check"></i><b>5.3.3</b> SEDAL and STARmap</a></li>
<li class="chapter" data-level="5.3.4" data-path="current-techs.html"><a href="current-techs.html#sbs"><i class="fa fa-check"></i><b>5.3.4</b> Sequencing by synthesis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="current-techs.html"><a href="current-techs.html#array"><i class="fa fa-check"></i><b>5.4</b> In situ array capture</a></li>
<li class="chapter" data-level="5.5" data-path="current-techs.html"><a href="current-techs.html#no-imaging"><i class="fa fa-check"></i><b>5.5</b> No imaging</a></li>
<li class="chapter" data-level="5.6" data-path="current-techs.html"><a href="current-techs.html#multiomics"><i class="fa fa-check"></i><b>5.6</b> Spatial multi-omics</a></li>
<li class="chapter" data-level="5.7" data-path="current-techs.html"><a href="current-techs.html#database-current"><i class="fa fa-check"></i><b>5.7</b> Databases of the current era</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="text-mining.html"><a href="text-mining.html"><i class="fa fa-check"></i><b>6</b> Text mining LCM transcriptomics abstracts</a><ul>
<li class="chapter" data-level="6.1" data-path="text-mining.html"><a href="text-mining.html#topic-model"><i class="fa fa-check"></i><b>6.1</b> Topic modeling</a></li>
<li class="chapter" data-level="6.2" data-path="text-mining.html"><a href="text-mining.html#word-time"><i class="fa fa-check"></i><b>6.2</b> Changes of word usage through time</a></li>
<li class="chapter" data-level="6.3" data-path="text-mining.html"><a href="text-mining.html#topic-time"><i class="fa fa-check"></i><b>6.3</b> Changes of topic prevalence through time</a></li>
<li class="chapter" data-level="6.4" data-path="text-mining.html"><a href="text-mining.html#association-of-topics-with-city"><i class="fa fa-check"></i><b>6.4</b> Association of topics with city</a></li>
<li class="chapter" data-level="6.5" data-path="text-mining.html"><a href="text-mining.html#glove"><i class="fa fa-check"></i><b>6.5</b> GloVe word embedding</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="current-analysis.html"><a href="current-analysis.html"><i class="fa fa-check"></i><b>7</b> Data analysis in the current era</a><ul>
<li class="chapter" data-level="7.1" data-path="current-analysis.html"><a href="current-analysis.html#preprocessing"><i class="fa fa-check"></i><b>7.1</b> Preprocessing</a><ul>
<li class="chapter" data-level="7.1.1" data-path="current-analysis.html"><a href="current-analysis.html#registration"><i class="fa fa-check"></i><b>7.1.1</b> Image registration</a></li>
<li class="chapter" data-level="7.1.2" data-path="current-analysis.html"><a href="current-analysis.html#calling"><i class="fa fa-check"></i><b>7.1.2</b> Spot and barcode calling</a></li>
<li class="chapter" data-level="7.1.3" data-path="current-analysis.html"><a href="current-analysis.html#segmentation"><i class="fa fa-check"></i><b>7.1.3</b> Cell segmentation</a></li>
<li class="chapter" data-level="7.1.4" data-path="current-analysis.html"><a href="current-analysis.html#seg-alternatives"><i class="fa fa-check"></i><b>7.1.4</b> Alternatives to cell segmentation</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="current-analysis.html"><a href="current-analysis.html#eda"><i class="fa fa-check"></i><b>7.2</b> Exploratory data analysis</a></li>
<li class="chapter" data-level="7.3" data-path="current-analysis.html"><a href="current-analysis.html#reconstruction"><i class="fa fa-check"></i><b>7.3</b> Spatial reconstruction of scRNA-seq data</a><ul>
<li class="chapter" data-level="7.3.1" data-path="current-analysis.html"><a href="current-analysis.html#direct-dr"><i class="fa fa-check"></i><b>7.3.1</b> Direct dimension reduction</a></li>
<li class="chapter" data-level="7.3.2" data-path="current-analysis.html"><a href="current-analysis.html#ad-hoc"><i class="fa fa-check"></i><b>7.3.2</b> <em>Ad hoc</em> scoring</a></li>
<li class="chapter" data-level="7.3.3" data-path="current-analysis.html"><a href="current-analysis.html#generative"><i class="fa fa-check"></i><b>7.3.3</b> Generative models</a></li>
<li class="chapter" data-level="7.3.4" data-path="current-analysis.html"><a href="current-analysis.html#latent"><i class="fa fa-check"></i><b>7.3.4</b> Shared latent space</a></li>
<li class="chapter" data-level="7.3.5" data-path="current-analysis.html"><a href="current-analysis.html#reconst-other"><i class="fa fa-check"></i><b>7.3.5</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="current-analysis.html"><a href="current-analysis.html#deconvolution"><i class="fa fa-check"></i><b>7.4</b> Cell type deconvolution</a></li>
<li class="chapter" data-level="7.5" data-path="current-analysis.html"><a href="current-analysis.html#variable"><i class="fa fa-check"></i><b>7.5</b> Spatially variable genes</a><ul>
<li class="chapter" data-level="7.5.1" data-path="current-analysis.html"><a href="current-analysis.html#gpr"><i class="fa fa-check"></i><b>7.5.1</b> Gaussian process regression</a></li>
<li class="chapter" data-level="7.5.2" data-path="current-analysis.html"><a href="current-analysis.html#laplacian"><i class="fa fa-check"></i><b>7.5.2</b> Laplacian score</a></li>
<li class="chapter" data-level="7.5.3" data-path="current-analysis.html"><a href="current-analysis.html#variable-other"><i class="fa fa-check"></i><b>7.5.3</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="current-analysis.html"><a href="current-analysis.html#pattern"><i class="fa fa-check"></i><b>7.6</b> Gene patterns</a></li>
<li class="chapter" data-level="7.7" data-path="current-analysis.html"><a href="current-analysis.html#region"><i class="fa fa-check"></i><b>7.7</b> Spatial regions</a></li>
<li class="chapter" data-level="7.8" data-path="current-analysis.html"><a href="current-analysis.html#cell-interaction"><i class="fa fa-check"></i><b>7.8</b> Cell-cell interaction</a><ul>
<li class="chapter" data-level="7.8.1" data-path="current-analysis.html"><a href="current-analysis.html#l-r"><i class="fa fa-check"></i><b>7.8.1</b> Ligand-receptor pairs</a></li>
<li class="chapter" data-level="7.8.2" data-path="current-analysis.html"><a href="current-analysis.html#gene-cell-interact"><i class="fa fa-check"></i><b>7.8.2</b> Genes associated with cell-cell interaction</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="current-analysis.html"><a href="current-analysis.html#gene-interaction"><i class="fa fa-check"></i><b>7.9</b> Gene-gene interaction</a></li>
<li class="chapter" data-level="7.10" data-path="current-analysis.html"><a href="current-analysis.html#subcellular"><i class="fa fa-check"></i><b>7.10</b> Subcellular transcript localization</a></li>
<li class="chapter" data-level="7.11" data-path="current-analysis.html"><a href="current-analysis.html#HnE"><i class="fa fa-check"></i><b>7.11</b> Gene expression imputation from H&amp;E</a></li>
</ul></li>
<li class="part"><span><b>III Future perspectives</b></span></li>
<li class="chapter" data-level="8" data-path="future.html"><a href="future.html"><i class="fa fa-check"></i><b>8</b> From the past to the present to the future</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://bookdown.org" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Museum of Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="current-analysis" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Data analysis in the current era</h1>

<div class="rmdtip">
Many machine learning and statistics methods are mentioned in this chapter. The names of these methods are linked to articles explaining them for those who are unfamiliar. Some of them are math heavy.
</div>

<p>So far we have reviewed numerous techniques to collect spatial transcriptomics data. In this chapter, we review computational methods to analyze data generated by current era techniques and methods that, while only having WMISH, FISH, or ISH as spatial data, involve scRNA-seq data as well. For a publication to be included in the “Analysis” sheet of this database, it must either focus on a data analysis method, or present alongside new data, sophisticated data analysis going beyond using existing packages. While some data analysis methods originally not designed for spatial data can be used for spatial data, this chapter is about methods designed specifically with spatial data in mind. This means that the methods should be demonstrated on a spatial transcriptomic dataset in the publication, even if not explicitly using spatial coordinates.</p>
<div class="figure" style="text-align: center"><span id="fig:analysis-fp"></span>
<img src="07-current_analysis_files/figure-html/analysis-fp-1.png" alt="Number of publications over time for current era and prequel data analysis. Bin width is 365 days. Preprints are included for this figure." width="672" />
<p class="caption">
Figure 7.1: Number of publications over time for current era and prequel data analysis. Bin width is 365 days. Preprints are included for this figure.
</p>
</div>
<p>Since 2019, there has been a sharp increase in interest in current era data analysis (Figure <a href="current-analysis.html#fig:analysis-current">7.2</a>). If our collection of prequel data analysis literature is somewhat representative and complete, then interest in current era data analysis dwarfs the golden age of prequel data analysis from 2008 to 2014 (Figure <a href="current-analysis.html#fig:analysis-fp">7.1</a>). As already shown, interests in current era data collection increased sharply since 2018 (Figure <a href="current.html#fig:current-vs-prequel">4.2</a>, Figure <a href="current-analysis.html#fig:analysis-current">7.2</a>); interest in data analysis lagged behind interest in data collection, until around 2020 (Figure <a href="current-analysis.html#fig:analysis-current">7.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:analysis-current"></span>
<img src="07-current_analysis_files/figure-html/analysis-current-1.png" alt="Number of publication over time for current era data collection and data analysis. Bin width is 120 days. Note that the count drops in 2021 because this plot was made in April 2021." width="672" />
<p class="caption">
Figure 7.2: Number of publication over time for current era data collection and data analysis. Bin width is 120 days. Note that the count drops in 2021 because this plot was made in April 2021.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:analysis-prequel"></span>
<img src="07-current_analysis_files/figure-html/analysis-prequel-1.png" alt="Number of publications over time for prequel data collection and data analysis. Bin width is 365 days." width="672" />
<p class="caption">
Figure 7.3: Number of publications over time for prequel data collection and data analysis. Bin width is 365 days.
</p>
</div>
<p>In contrast, in the prequel era, interest in data analysis peaked after the peak for data collection, and eventually interest both eventually diminished but continues (Figure <a href="current-analysis.html#fig:analysis-prequel">7.3</a>). There are many different types of data analysis, the ones with the most interest are mapping dissociated cells in scRNA-seq to location in a spatial reference (cells to locations) and imputing expression of genes not profiled in the spatial reference according to transcriptome wide scRNA-seq data (gene expression imputation at locations) (Figure <a href="current-analysis.html#fig:analysis-cats">7.4</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:analysis-cats"></span>
<img src="07-current_analysis_files/figure-html/analysis-cats-1.png" alt="Number of publications for each category of data analysis; note that the same publication can fall into multiple categories." width="576" />
<p class="caption">
Figure 7.4: Number of publications for each category of data analysis; note that the same publication can fall into multiple categories.
</p>
</div>
<p>In 2020, several methods for cell type deconvolution in array based techniques that don’t have single cell resolution were developed (cell type inference), but the drastic growth in data analysis seems to be driven by multiple categories of analyses (Figure <a href="current-analysis.html#fig:analysis-cat-hist">7.5</a>). Top contributors to data analysis methods in the current and prequel eras are different as well. Again, the current era seems to be more of an elite club than the prequel era although some not as elite institutions have contributed as well; among the top contributors in the prequel era are less famous institutions such as Arizona State University (ASU), Old Dominion University (ODU), and Lawrence Berkeley National Laboratory (LBL), which developed the BDTNP and the Fly Enhancer atlases (Figure <a href="current-analysis.html#fig:analysis-map">7.6</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:analysis-cat-hist"></span>
<img src="07-current_analysis_files/figure-html/analysis-cat-hist-1.png" alt="Number of publications over time broken down by type of data analysis. The 3 categories most popular in the past year are shown, and the others are lumped into 'Other'. Bin width is 365 days." width="576" />
<p class="caption">
Figure 7.5: Number of publications over time broken down by type of data analysis. The 3 categories most popular in the past year are shown, and the others are lumped into ‘Other’. Bin width is 365 days.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:analysis-map"></span>
<img src="07-current_analysis_files/figure-html/analysis-map-1.png" alt="Map of where first authors of current era and prequel data analysis papers were located as of publication. Top 5 institutions in each era are labeled." width="960" />
<p class="caption">
Figure 7.6: Map of where first authors of current era and prequel data analysis papers were located as of publication. Top 5 institutions in each era are labeled.
</p>
</div>
<p>In our database, we have recorded programming languages used in data analysis or package development. All programming languages that played a major role in the project were recorded. For downstream analysis, this includes languages of the user interface of existing packages used and languages of new functions written for the project. For package development, this includes any language used to write the package essential to the functioning of the package. In publications that focus on data collection, R is by far the most popular programming language used in downstream data analysis (Figure <a href="current-analysis.html#fig:user-langs">7.7</a>). The second most popular is MATLAB, which is more common in smFISH (Figure <a href="current-techs.html#fig:smfish-langs">5.26</a>) and ISS for its image processing functionality. Python follows closely, and is used for both image processing and other types of analyses. C and C++ are not as common in downstream analysis.</p>
<div class="figure" style="text-align: center"><span id="fig:user-langs"></span>
<img src="07-current_analysis_files/figure-html/user-langs-1.png" alt="Number of publications for data collection using each of the 5 most popular programming languages for downstream data analysis." width="576" />
<p class="caption">
Figure 7.7: Number of publications for data collection using each of the 5 most popular programming languages for downstream data analysis.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:dev-langs"></span>
<img src="07-current_analysis_files/figure-html/dev-langs-1.png" alt="Number of publication for data analysis using each of the 5 most popular programming languages for package development. In this and the previous figure, each icon stands for 10 publications, and the x axes of both figures are aligned. Note that multiple programming languages can be used in one publication." width="576" />
<p class="caption">
Figure 7.8: Number of publication for data analysis using each of the 5 most popular programming languages for package development. In this and the previous figure, each icon stands for 10 publications, and the x axes of both figures are aligned. Note that multiple programming languages can be used in one publication.
</p>
</div>
<p>The same top 5 programming languages are the most common for developing data analysis packages (Figure <a href="current-analysis.html#fig:dev-langs">7.8</a>). Python is the most popular, especially for packages involving deep learning, image processing, using Torch for optimization, or are command line tools. R follows, and is more popular for exploratory data analysis (EDA) and data visualization, but often both R and Python are used in the same package. Other languages aren’t nearly as commonly used for packages reported on in our database. The above observations about usage of R and Python seem to reflect the broader cultural differences between the R and Python communities; the former caters more to the users and statisticians who do not specialize in computer science, while the latter caters more to developers and computer science specialists. MATLAB is not as commonly used for package development. While popularity of Python and R have grown (and some others such as Julia), the popularity of MATLAB seems more level (Figure <a href="current-analysis.html#fig:documented">7.9</a>). C and C++ are more common in package development than in downstream analysis, but are often used in conjunction with either R or Python or both as C and C++ are used for performance while R and Python are for user interface. With packages such as <a href="https://rstudio.github.io/reticulate/"><code>reticulate</code></a>, <a href="https://rpy2.github.io/doc/latest/html/introduction.html"><code>rpy2</code></a>, <a href="https://bioconductor.org/packages/release/bioc/html/basilisk.html"><code>basilisk</code></a>, <a href="http://rcpp.org"><code>Rcpp</code></a>, and <a href="https://cython.org"><code>Cython</code></a>, the most popular open source languages can be made interoperable to each other to some extent, making use of the best resources from each language.</p>
<div class="figure" style="text-align: center"><span id="fig:documented"></span>
<img src="07-current_analysis_files/figure-html/documented-1.png" alt="Among data analysis publications, the number of packages that are or are not well documented over time." width="864" />
<p class="caption">
Figure 7.9: Among data analysis publications, the number of packages that are or are not well documented over time.
</p>
</div>
<p>We have also recorded whether the package is well documented and whether it’s hosted on a public repository as a loose proxy of user friendliness and quality. Here “well documented” means at least all arguments of all functions exposed to the user are documented, though we consider it better when examples are included. Public repositories can to some extent indicate user friendliness and quality because the packages need to pass some sort of checking in order to be hosted on the repositories, though some repositories, such as Bioconductor, have stricter standards than others. Moreover, installation of the package is easier when the package is on a public repository. A modest majority of Python packages and the vast majority of R and C++ packages are well documented, while most MATLAB packages are not (Figure <a href="current-analysis.html#fig:documented">7.9</a>). Most packages are not on a public repository such as CRAN, Bioconductor, pip, and conda (Figure <a href="current-analysis.html#fig:repo">7.10</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:repo"></span>
<img src="07-current_analysis_files/figure-html/repo-1.png" alt="The number of packages that are or are not on a public repository such as CRAN, Bioconductor, pip, or conda over time. In both C and D, the bin width is 365 days. NA means the source code repository is not available." width="864" />
<p class="caption">
Figure 7.10: The number of packages that are or are not on a public repository such as CRAN, Bioconductor, pip, or conda over time. In both C and D, the bin width is 365 days. NA means the source code repository is not available.
</p>
</div>
<p>Some of the most popular categories of analyses (Figure <a href="current-analysis.html#fig:analysis-cats">7.4</a>) are reviewed in the rest of this section, arranged roughly in the order each task is performed in a data analysis workflow. Each category will first be defined, and the common core principles will be summarized.</p>
<div id="preprocessing" class="section level2">
<h2><span class="header-section-number">7.1</span> Preprocessing</h2>
<p>By “preprocessing” we mean extracting information from raw data so common analysis methods can be applied. “Raw data” can mean any form of data, even if processed in some ways, that still needs to have information extracted for common analysis tasks to apply, such as PCA, clustering, and DE. Preprocessing for array based techniques that use NGS is similar to preprocessing for scRNA-seq. The same aligners can be used to align reads to the genome or pseudoalign to the transcriptome, and the spot barcodes can be demultiplexed just like in scRNA-seq; indeed, ST and Visium, the preprocessing pipelines ST Pipeline and Space Ranger wrap the STAR aligner. As microdissection based techniques also use NGS, preprocessing would not be very different from that of scRNA-seq or bulk RNA-seq data. However preprocessing of smFISH and ISS data is very different from that of NGS based data, and this would be the focus of this section.</p>
<p>The raw data is images. As mentioned earlier, preprocessing of images was typically performed with poorly documented MATLAB code difficult to decipher by users. While some switched to Python recently, such as in MERlin for MERFISH, the preprocessing tool is still specific to the technique of interest. Also, the proprietary language MATLAB is still quite commonly used, such as for preprocessing HybISS and ISS data <span class="citation">(Gyllborg et al. <a href="#ref-Gyllborg2020" role="doc-biblioref">2020</a>; Qian et al. <a href="#ref-Qian2020" role="doc-biblioref">2020</a>)</span>. Some groups used GUI based tools such as Fiji, ImageJ, and CellProfiler <span class="citation">(Shah et al. <a href="#ref-Shah2018" role="doc-biblioref">2018</a>; Chen et al. <a href="#ref-Chen2020" role="doc-biblioref">2020</a>; Sountoulidis et al. <a href="#ref-Sountoulidis2020" role="doc-biblioref">2020</a>)</span>. However, as the GUI based analyses are not recorded and shared or are manual, it is difficult to reproduce such analyses.</p>
<p>To provide a free, open source, and well-documented preprocessing tool applicable to data from multiple techniques, the Chan Zuckerberg Initiative developed the Python package <code>starfish</code> implementing image registration, spot calling, barcode calling, cell segmentation, and etc. with classical image processing methods such as thresholding, image registration by translation, top hat filtering, Laplacian of Gaussian, watershed segmentation, and etc. While a good start, it’s not clear how to apply <code>starfish</code> to multiple FOVs based on its tutorials. To improve <code>starfish</code>, another Python pipeline, SMART-Q was developed, with more modularity and improvements upon <code>starfish</code> such as additional parameter to mitigate over-segmentation (individual cell or nucleus broken into too many pieces) by watershed and integration with immunofluorescence images of marker genes <span class="citation">(Yang et al. <a href="#ref-Yang2020" role="doc-biblioref">2020</a>)</span>. However, SMART-Q was only demonstrated in RNAscope data without combinatorial barcoding, with one FOV at a time. Another such smFISH pipeline based on classical image processing is <code>dotdotdot</code>, which is written in MATLAB but the functions are well documented <span class="citation">(Maynard et al. <a href="#ref-Maynard2020" role="doc-biblioref">2020</a>)</span>. Again, <code>dotdotdot</code> was only demonstrated on RNAscope without combinatorial barcoding. There are other open source tools for one or more of the preprocessing steps, but are not meant to be a comprehensive pipeline. Below we review each step in preprocessing of smFISH and ISS raw data, how this was done in the original papers of datasets with classical image processing, and alternative and improved approaches such as ones based on deep learning or Bayesian statistics.</p>
<p>The packages mentioned in this section are summarized in the Table <a href="current-analysis.html#tab:seg-packages">7.1</a>. The package names link to the code repo if available, and the titles link to the paper associated with the package. Each section in this chapter has a table like this. There are relevant packages not mentioned in this book; they can be found in the <a href="https://docs.google.com/spreadsheets/d/1sJDb9B7AtYmfKv4-m8XR7uc3XXw_k4kGSout8cqZ8bY/edit#gid=1424019374">database</a>.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:seg-packages">Table 7.1: </span>Packages mentioned for smFISH and ISS image processing
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/singlecelllab/correlationFISH"> corrFISH </a>
</td>
<td style="text-align:left;width: 5em; ">
MATLAB
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/nmeth.3895"> Dense transcript profiling in single cells by image correlation decoding </a>
</td>
<td style="text-align:left;width: 5em; ">
2016-06-06
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/wahlby-lab/graph-iss"> graph-ISS </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/765842"> Identification of spatial compartments in tissue from in situ sequencing data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-09-18
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/eilslabs/ssam"> SSAM </a>
</td>
<td style="text-align:left;width: 5em; ">
Python; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/800748"> Segmentation-free inference of cell types from in situ transcriptomics data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-10-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/kdharris101/iss"> pciSeq </a>
</td>
<td style="text-align:left;width: 5em; ">
MATLAB
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41592-019-0631-4"> Probabilistic cell typing enables fine mapping of closely related cell types in situ </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-11-18
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/shenlab-ucsf/SMART-Q"> SMART-Q </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1371/journal.pone.0228760"> SMART-Q: An Integrative Pipeline Quantifying Cell Type-Specific RNA Transcription </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-04-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/wollmanlab/JSTA"> JSTA </a>
</td>
<td style="text-align:left;width: 5em; ">
Python; C
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.09.18.304147"> JSTA: joint cell segmentation and cell type annotation for spatial transcriptomics </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-09-20
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/wahlby-lab/spage2vec"> spage2vec </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1111/febs.15572"> Spage2vec: Unsupervised detection of spatial gene expression constellations </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-09-25
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/kharchenkolab/Baysor"> Baysor </a>
</td>
<td style="text-align:left;width: 5em; ">
Julia
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.10.05.326777"> Bayesian segmentation of spatially resolved transcriptomics data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-10-06
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/BBQuercus/deepBlink"> deepBlink </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.12.14.422631"> deepBlink: Threshold-independent detection and localization of diffraction-limited spots </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-12-15
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/axanderssonuu/istdeco"> ISTDECO </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2021.03.01.433040"> ISTDECO: In Situ Transcriptomics Decoding by Deconvolution </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-02
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/jacksonloper/bardensr"> BarDensr </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1371/journal.pcbi.1008256"> BARcode DEmixing through Non-negative Spatial Regression (BarDensr) </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-08
</td>
</tr>
</tbody>
</table>
<div id="registration" class="section level3">
<h3><span class="header-section-number">7.1.1</span> Image registration</h3>
<p>First, images of each FOV from different rounds of hybridization must be aligned; this is image registration. The images can be aligned to a reference of fiducial beads or DAPI staining, which is especially useful when “no fluorescence” is part of the barcode <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>; Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span>. If “no fluorescence” is not involved, then the reference can be a particular round of hybridization <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>; Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>. Image registration is usually affine, i.e. images are translated, scaled, or rotated to match the reference, and often only translation is used. However, non-linear registration has been used in case the sample does not lie flat and chromatic aberration shifts spots in different channels <span class="citation">(Qian et al. <a href="#ref-Qian2020" role="doc-biblioref">2020</a>)</span>.</p>
</div>
<div id="calling" class="section level3">
<h3><span class="header-section-number">7.1.2</span> Spot and barcode calling</h3>
<p>Then the spots representing individual transcripts are identified (spot calling). The background of autofluorescence and non-specific hybridization is often removed by thresholding or top hat filtering, only preserving brighter pixels. Spots can be identified with <a href="http://www.inf.ed.ac.uk/teaching/courses/inf2b/learnSlides/inf2b12-learnlec09.pdf">multi-Gaussian fitting</a> with fixed width, which can distinguish between partially overlapping spots <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>)</span>, or tightened by <a href="https://en.wikipedia.org/wiki/Richardson–Lucy_deconvolution">Lucy-Richardson deconvolution</a> <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2018" role="doc-biblioref">2018</a>)</span>, or by identifying local maxima in intensity after identifying potential spots with <a href="http://fourier.eng.hmc.edu/e161/lectures/gradient/node8.html">Laplacian of Gaussian</a> <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>; Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>.
The spots can also be identified with deep learning. In Python package graph-ISS <span class="citation">(Partel et al. <a href="#ref-Partel2019" role="doc-biblioref">2019</a>)</span>, a <a href="https://towardsdatascience.com/a-comprehensive-guide-to-convolutional-neural-networks-the-eli5-way-3bd2b1164a53">convolutional neural network (CNN)</a> is pretrained on manually annotated candidate signal spots from another dataset, and probability that a new candidate obtained after <a href="https://micro.magnet.fsu.edu/primer/java/digitalimaging/russ/tophatfilter/index.html">top hat filtering</a> and <a href="https://en.wikipedia.org/wiki/H-maxima_transform">h-maxima transform</a> is a signal is returned by the last softmax layer of the CNN. Another CNN based spot calling tool is deepBlink <span class="citation">(Eichenberger et al. <a href="#ref-Eichenberger2020" role="doc-biblioref">2020</a>)</span>, which builds on the popular U-net architecture.</p>
<p>Once spots are called in each round of hybridization, spots that most likely to correspond to the same transcript are read as barcode and decoded to identify the gene encoded by the barcode (barcode calling). As image registration is imperfect, the spot coming from the same transcript may still be slightly shifted between rounds of hybridization. To identify the barcode from the rounds of hybridization, the spot in one round of hybridization is typically identified with a spot in another round if the spatial distance between the two is sufficiently small, such as less than between 1 and 3 pixels, or smaller than the distance to a barcode that contains error <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>; Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>; Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>; Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span>.</p>
<p>In graph-ISS <span class="citation">(Partel et al. <a href="#ref-Partel2019" role="doc-biblioref">2019</a>)</span>, spots identified from CNN from different rounds of hybridization are connected in a graph, with each spot in each round of hybridization a node and the edge weight decreases with increasing distance between spots across rounds up to a maximum distance. Edges connecting spots not from consecutive rounds are removed. The barcode is called by maximum flow of minimum costs between the sink and the source of the graph. Then a quality score is calculated for the barcode according to the CNN probability of spots and distance between spots from different rounds. Although graph-ISS was originally designed for ISS data, it might be adapted to seqFISH, HybISS, STARmap, and SCRINSHOT as well. However, for MERFISH and seqFISH+, in which a transcript may not have signal in some rounds of hybridization, graph-ISS would need to be altered. Alteration would also be required to decode STARmap’s 2 base encoding.</p>
<p>For MERFISH specifically, transcript counts have been statistically modeled in the Rust package MERFISHtools, which takes errors in barcode calling into account <span class="citation">(Köster, Brown, and Liu <a href="#ref-Koster2019" role="doc-biblioref">2019</a>)</span>. While MERFISH’s inbuilt error correction (HD4) accounts for 1 to 0 error, which is more common, 0 to 1 errors can still occur, and there are still barcodes with so many errors that they can’t be matched to genes (dropout). The errors are modeled as a <a href="https://online.stat.psu.edu/stat504/node/40/">multinomial distribution</a> with event probabilities as probabilities of identifying transcripts of a gene correctly with and without the inbuilt correction, misidentifying transcripts of a gene as those from each other gene with and without the inbuilt correction, and dropouts, with actual transcript counts, number of correct and incorrect identifications, and dropouts as latent variables to be estimated by Bayesian inference. The flat prior is used for now.</p>
<p>Computational methods to overcome optical crowding and to deconvolute spots were summarized in Section <a href="current-techs.html#crowding">5.2.3</a>: corrFISH, BarDensr, and ISTDECO. The above mentioned spot calling methods all treat spot detection and decoding as separate tasks. In contrast, in both BarDensr and ISTDECO, the two related tasks are performed jointly.</p>
</div>
<div id="segmentation" class="section level3">
<h3><span class="header-section-number">7.1.3</span> Cell segmentation</h3>
<p>To assign transcript spots to cells, the cells need to be segmented and spots within the segmented boundary of a cell must be assigned to that cell. For neurons, Nissl staining, which stains the cell body and dendrites but not axons, has been used for cell segmentation <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>; Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>. Without Nissl staining, total poly-A staining can be used instead, and segmented with watershed transform, although poly-a staining concentrates in the cell body and misses cellular processes such as dendrites <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2018" role="doc-biblioref">2018</a>)</span>. This misses some interesting biological information; dendrites can have different transcriptomes from the cell body of the same neuron, both in vitro and in vivo <span class="citation">(Middleton, Eberwine, and Kim <a href="#ref-Middleton2019" role="doc-biblioref">2019</a>; Ciolli Mattioli et al. <a href="#ref-CiolliMattioli2019" role="doc-biblioref">2019</a>; Farris et al. <a href="#ref-Farris2019" role="doc-biblioref">2019</a>)</span>. Cell segmentation can be done manually as automated methods may not be sufficiently reliable and would still require manual inspection and correction, or automated with machine learning models trained by manual segmentation of smaller number of cells such as the random forest model in Ilastik <span class="citation">(Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>; Lohoff et al. <a href="#ref-Lohoff2020" role="doc-biblioref">2020</a>)</span> and CNN models such as DeepCell <span class="citation">(Van Valen et al. <a href="#ref-VanValen2016" role="doc-biblioref">2016</a>)</span> and U-net <span class="citation">(Ronneberger, Fischer, and Brox <a href="#ref-Ronneberger2015" role="doc-biblioref">2015</a>)</span>. <a href="https://www.mathworks.com/company/newsletters/articles/the-watershed-transform-strategies-for-image-segmentation.html">Watershed segmentation</a> is more commonly used.</p>
<p>Without seeing the actual extent of the cell, the quality of manual segmentation is questionable, especially in regions with high cell density, thus limiting the performance of machine learning models. Sometimes problematic methods were used to segment cells, such as 3D <a href="https://en.wikipedia.org/wiki/Voronoi_diagram">Voroni tessellation</a> <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>)</span> and <a href="https://en.wikipedia.org/wiki/Convex_hull">convex hull</a> of Nissl staining based segmentation <span class="citation">(Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>; these are problematic because cells need not to take a convex shape so such segmentation may mis-assign transcripts from other cells, or to be conservative about mis-assigning transcripts from other cells, miss transcripts that in fact belong to the cell of interest. However, one study did specifically stain for membrane bound proteins for the actual extent of the plasma membrane and accurate cell segmentation <span class="citation">(Lohoff et al. <a href="#ref-Lohoff2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>To address the challenges of cell segmentation, segmentation methods utilizing scRNA-seq data with annotated cell types have been developed recently. One such method is Python package JSTA <span class="citation">(Littman et al. <a href="#ref-Littman2020" role="doc-biblioref">2020</a>)</span>, in which a deep neural network (DNN) learns a segmentation and cell type annotation using the information from a scRNA-seq reference with cell type annotations. First, watershed is used for an initial cell segmentation, both MERFISH and scRNA-seq data are scaled and centered. Then a DNN is trained on the scRNA-seq data to predict cell type from gene expression. Then a separate DNN is trained to refine the cell boundaries iteratively with <a href="http://ai.stanford.edu/~chuongdo/papers/em_tutorial.pdf">expectation maximization (EM)</a>: The cell type classifier is applied on the watershed segmented MERFISH data to classify putative cells (E). Then a random subset of the pixels are used to train the pixel classifier, maximizing a loss function comparing the new pixel cell type probabilities to the initial/previous assignment (M). The new cell type probabilities are then scaled per pixel according to distance to nuclei. Only probabilities of cell types of neighboring cells are kept and the other cell types are assigned probability 0. The new cell type probabilities of each pixel is then used as event probabilities of a multinomial distribution and randomly assign a new cell type label to the pixel. Then the new cell type assignment to pixels is used to train the pixel classifier again, until the cell type assignments converge. This may refine boundaries between neighboring cells of different types, and the initial watershed boundaries are kept for neighboring cells of the same type. A problem with this package is that inhomogeneous transcript localization is not taken into account.</p>
</div>
<div id="seg-alternatives" class="section level3">
<h3><span class="header-section-number">7.1.4</span> Alternatives to cell segmentation</h3>
<p>Due to the challenges in accurate cell segmentation, some analysis methods did away with cell segmentation altogether, directly using the transcript locations. In the Julia package Baysor <span class="citation">(Petukhov et al. <a href="#ref-Petukhov2020" role="doc-biblioref">2020</a>)</span>, based on <a href="https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-867-machine-learning-fall-2006/lecture-notes/lec23.pdf">Markov random field (MRF)</a>, which encourages nearby transcripts to take the same label. A spatial neighborhood graph is constructed with <a href="https://towardsdatascience.com/delaunay-triangulation-228a86d1ddad">Delaunay triangulation</a> with each transcript as a node. The probability of each transcript taking each label is modeled with a MRF and initial edge weights decrease with distance. This package first distinguishes between intracellular transcripts and extracellular background. Then it can also assign transcripts to cell types without cell segmentation, with a scRNA-seq reference with cell type annotations; as locations of the transcripts are known, this amounts to annotating tissue regions with cell types. It can also segment cells, with existing segmentation and staining (e.g. Nissl, DAPI, and poly-A) as priors. Cell segmentation can also be informed by cell type labels, so transcripts from different cell types are not assigned to the same cells. Each of the three functionalities, identifying intracellular transcripts, cell type annotation of transcripts, and cell segmentation, is based on a different MRF model. The parameters of the model, such as edge weights, labels of other transcripts, and etc. are estimated with EM. The drawbacks of this package are that its current implementation is limited to 2D and it does not take inhomogeneous subcellular transcript localization into account.</p>
<p>Besides cell type annotation of transcripts based on MRF, another segmentation-free method is also described in the Baysor paper <span class="citation">(Petukhov et al. <a href="#ref-Petukhov2020" role="doc-biblioref">2020</a>)</span>, in which the <span class="math inline">\(k\)</span> nearest neighbors of each transcript are taken to be a pseudo-cell and analyzed by standard scRNA-seq data analysis methods such as clustering, PCA, and UMAP. For ISS, transcripts can be probabilistically assigned to cells and cells to cell types, with pciSeq <span class="citation">(Qian et al. <a href="#ref-Qian2020" role="doc-biblioref">2020</a>)</span>. Briefly, spatial locations of transcripts are modeled by a <a href="https://hpaulkeeler.com/poisson-point-process/">Poisson point process</a> whose intensity is scaled by a term following <a href="https://en.wikipedia.org/wiki/Gamma_distribution">Gamma distribution</a> to give the <a href="https://en.wikipedia.org/wiki/Negative_binomial_distribution">negative binomial distribution</a> of transcript counts in cells. The intensity for each gene and each cell is also informed by distance between transcripts and nucleus centroids (from DAPI), scRNA-seq data of the cell type this cell belongs to, and the detection efficiency of ISS. The data consists of locations of transcripts and the genes they come from. The unknown parameters, such as probability of each transcript to come from each cell and each cell from each cell type, are estimated by <a href="https://omarelb.github.io/variational-bayes/">variational Bayesian inference</a>. Cell types and spatial domains can also be identified without scRNA-seq cell type annotations as well.</p>
<p>In the Python package SSAM <span class="citation">(Park et al. <a href="#ref-Park2019" role="doc-biblioref">2019</a>)</span>, transcript density is first estimated with <a href="https://wiki.analytica.com/index.php/Kernel_Density_Smoothing">Gaussian kernel</a> density, which is then projected into a square lattice. Local maxima of transcript density are taken as pseudo-cells and clustered to infer <em>de novo</em> cell types. Then tissue domains are identified by clustering sliding windows of spatial cell type maps. Tissue domains can also be identified without appealing to cell types.</p>
<p>In the Python package spage2vec <span class="citation">(Partel and Wählby <a href="#ref-Partel2020" role="doc-biblioref">2020</a>)</span>, graphs are constructed by connecting each transcript spot to its neighbors within a certain distance such that at 97% of all transcript spot are connected to at least one neighbor. Then the transcript spots with these graphs are projected by a <a href="https://blog.exxactcorp.com/a-friendly-introduction-to-graph-neural-networks/">graph neural network (GNN)</a> into a 50 dimensional space which is informed by the graphs and thus local neighborhoods of transcripts. The transcript spots in the 50 dimensional space can then be clustered or projected to 2 or 3 dimensions with <a href="https://umap-learn.readthedocs.io/en/latest/how_umap_works.html">UMAP</a> to show tissue domains.</p>
</div>
</div>
<div id="eda" class="section level2">
<h2><span class="header-section-number">7.2</span> Exploratory data analysis</h2>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:eda-packages">Table 7.2: </span>Packages mentioned for EDA
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/RachelQueen1/Spaniel"> Spaniel </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/619197"> Spaniel: analysis and interactive sharing of Spatial Transcriptomics data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-05-05
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/satijalab/seurat"> Seurat3 </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1016/j.cell.2019.05.031"> Comprehensive Integration of Single-Cell Data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-06-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/jbergenstrahle/SpatialCPie"> SpatialCPie </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s12859-020-3489-7"> SpatialCPie: an R/Bioconductor package for spatial transcriptomics cluster evaluation </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-04-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/jbergenstrahle/STUtility"> STUtility </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s12864-020-06832-3"> Seamless integration of image and molecular analysis for spatial transcriptomics workflows </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-07-16
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/theMILOlab/SPATA"> SPATA </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.10.20.346544"> Inferring spatially transient gene expression pattern from spatial transcriptomic studies </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-10-21
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://bioconductor.org/packages/SpatialExperiment"> SpatialExperiment </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2021.01.27.428431"> SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-01-27
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/theislab/squidpy"> Squidpy </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2021.02.19.431994"> Squidpy: a scalable framework for spatial single cell analysis </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-02-20
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/RubD/Giotto"> Giotto </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s13059-021-02286-2"> Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-08
</td>
</tr>
</tbody>
</table>
<p>After data preprocessing, as described above, for array or microdissection based data, we get a gene count matrix with locations of voxels, and for smFISH and ISS based data, we get locations of transcripts, and if cell segmentation is performed, a gene count matrix and cell boundaries as well. For scRNA-seq, Seurat <span class="citation">(Stuart et al. <a href="#ref-Stuart2019a" role="doc-biblioref">2019</a>)</span>, scanpy, and packages surrounding SingleCellExperiment on Bioconductor such as scran and scater implement further preprocessing of the gene count matrix, such as data normalization and scaling, as well as basic EDA methods to inspect and create an overview of the data, such as quality control (QC), data visualization, finding highly variable genes, dimension reduction, and clustering, and have user friendly tutorials, consistent user interface, and decent documentation. Such integrative EDA packages, as well as more specialized data visualization packages, have emerged for spatial transcriptomics as well, and are reviewed in this section.</p>
<p>In practice, spatial transcriptomics data is often analyzed with standard scRNA-seq analysis at the EDA stage, with one or more of PCA, <a href="https://en.wikipedia.org/wiki/T-distributed_stochastic_neighbor_embedding">tSNE</a>, UMAP, clustering cells or spots, and finding marker genes for clusters, and differential expression (DE) between case and control <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>; Moffitt et al. <a href="#ref-Moffitt2018" role="doc-biblioref">2018</a>; Zhang et al. <a href="#ref-Zhang2020" role="doc-biblioref">2020</a>; Moncada et al. <a href="#ref-Moncada2020" role="doc-biblioref">2020</a>; Berglund et al. <a href="#ref-Berglund2018" role="doc-biblioref">2018</a>)</span>. For ST and Visium, the data is also often normalized like in scRNA-seq with <a href="https://reneshbedre.github.io/blog/expression_units.html">CPM</a> or classical Seurat <a href="https://rdrr.io/cran/Seurat/man/NormalizeData.html">log normalization</a> and <a href="https://rdrr.io/cran/Seurat/man/ScaleData.html">scaling</a> <span class="citation">(Moncada et al. <a href="#ref-Moncada2020" role="doc-biblioref">2020</a>; Ji et al. <a href="#ref-Ji2020" role="doc-biblioref">2020</a>; Berglund et al. <a href="#ref-Berglund2018" role="doc-biblioref">2018</a>)</span>. Seurat also implements data integration, which has been used to transfer cell type labels from scRNA-seq to Visium for cell type deconvolution <span class="citation">(Mantri et al. <a href="#ref-Mantri2020" role="doc-biblioref">2020</a>)</span>, and can potentially be used to impute gene expression in non-transcriptome wide spatial data from scRNA-seq (discussed in Section <a href="current-analysis.html#reconstruction">7.3</a>). Then the clusters, marker genes, and genes of interest from scRNA-seq are often visualized within spatial context, and some studies proceed to other analyses that utilize the spatial information. Due to the relevance of scRNA-seq data normalization, EDA, and data integration to spatial data, the existing scRNA-seq ecosystems of Seurat, scanpy (spatial part in Squidpy <span class="citation">(Palla et al. <a href="#ref-Palla2021" role="doc-biblioref">2021</a>)</span>), and SingleCellExperiment (spatial part in SpatialExperiment <span class="citation">(Righelli et al. <a href="#ref-Righelli2021" role="doc-biblioref">2021</a>)</span>) are adapting to the rise of spatial transcriptomics, with new data structures, visualization of gene expression and cell metadata (e.g. total UMI counts, cluster, and cell type) on the spatial coordinantes, with H&amp;E as background for ST and Visium, and perhaps other spatial functionalities such as spatial neighborhood graphs and spatially variable genes.</p>
<p>There are other EDA packages not originating from an existing scRNA-seq EDA ecosystem as well. R packages Giotto <span class="citation">(Dries et al. <a href="#ref-Dries2019" role="doc-biblioref">2019</a>)</span>, STUtility <span class="citation">(Bergenstråhle et al. <a href="#ref-Bergenstrahle2020" role="doc-biblioref">2020</a>)</span>, and SPATA <span class="citation">(Kueckelhaus et al. <a href="#ref-Kueckelhaus2020" role="doc-biblioref">2020</a>)</span> not only support basic QC and EDA functionalities like those in Seurat, but also spatial analyses not supported by Seurat. These packages are well documented, but are not (yet?) on CRAN or Bioconductor.</p>
<p>Giotto has two main parts: Giotto Analyzer and Giotto Viewer. Besides basic Seurat functionalities and spatial data visualization, Giotto Analyzer implements several types of spatial analyses to be reviewed in more detail in the rest of this section: cell type enrichment in spatial data without single cell resolution, identifying spatially variable genes, gene co-expression patterns, cellular neighborhoods, interactions between cell types and ligand-receptor pairs in such interactions, and genes whose expression is associated with cell type interactions. However, the methods implemented in Giotto tend to have simpler principles than those of more specialized packages for each of the above tasks, such as hypergeometric test for cell type enrichment and spatially coherent genes, though Giotto wraps specialized packages such as SpatialDE <span class="citation">(Svensson, Teichmann, and Stegle <a href="#ref-Svensson2018" role="doc-biblioref">2018</a>)</span>, trendsceek <span class="citation">(Edsgärd, Johnsson, and Sandberg <a href="#ref-Edsgard2018" role="doc-biblioref">2018</a>)</span> for spatially variable genes, and smfishhmrf <span class="citation">(Zhu et al. <a href="#ref-Zhu2018" role="doc-biblioref">2018</a>)</span> to identify spatial cellular neighborhoods. Giotto Viewer provides interactive visualization of the data. As Giotto uses its own object class to store data, interoperability with other single cell and spatial software becomes more challenging given the popularity of Seurat and SingleCellExperiment.</p>
<p>In contrast, STUtility develops upon the Seurat class, so is interoperable with other Seurat functionalities. STUtility is specific to ST and Visium, while Giotto applies to all spatial technologies with cell or spot level data. Beyond Seurat, STUtility enables masking the array to remove spots outside the tissue, alignment of multiple sections, manual annotation and alignment with <a href="https://shiny.rstudio.com"><code>shiny</code></a>, visualization of the aligned sections in 3D, finding neighbors of spots of a given type, and using NMF to identify archetypal gene expression patterns. While Giotto and STUtility might not have the most sophisticated spatial analysis methods, their main advantage is akin to that of Seurat and SingleCellExperiment, namely that multiple analysis tasks, often with a variety of algorithms for each task, can be done with the same object class and user interface, saving the time and trouble on learning new syntax and converting objects to new classes.</p>
<p>SPAtial Transcriptomic Analysis (SPATA), while implementing its own class, uses Seurat for data normalization and dimension reduction. SPATA also implements functions to visualize spatial data and a <code>shiny</code> app for not only interactive data visualization but also manually setting spatial trajectories and annotation of spatial regions. It also wraps Monocle 3 <span class="citation">(Cao et al. <a href="#ref-Cao2019" role="doc-biblioref">2019</a>)</span> for pseudotime analysis and SPARK <span class="citation">(S. Sun, Zhu, and Zhou <a href="#ref-Sun2020" role="doc-biblioref">2020</a>)</span> for finding spatially variable genes. In addition, SPATA implements its own method of finding spatially variable genes, reviewed in Section <a href="current-analysis.html#variable">7.5</a>.</p>
<p>Some R packages have also been written for specific visualization tasks, but not the entire EDA process. Spaniel is a package that builds on Seurat and SingleCellExperiment for interoperability and implements QC plots that help the user to remove ST or Visium spots outside the tissue. However, Spaniel’s main difference from STUtility is that Spaniel can create a <code>shiny</code> app for interactive visualization and exploration of the data. While this may make Spaniel sound unremarkable, it was written about a year before Seurat supported spatial data. Another specialized package is SpatialCPie <span class="citation">(Bergenstråhle, Bergenstråhle, and Lundeberg <a href="#ref-Bergenstrahle2020a" role="doc-biblioref">2020</a>)</span>, which also uses <code>shiny</code> for interactive visualization. SpatialCPie cluster ST or Visium data at multiple resolutions and plots a graph showing how clusters from one resolution relates to those from other resolutions. It also plots a pie chart at each ST or Visium spot, on top of an H&amp;E background, showing similarity of each spot to each cluster, to give a more nuanced view than simply coloring the spots by cluster. Both packages are on Bioconductor.</p>
</div>
<div id="reconstruction" class="section level2">
<h2><span class="header-section-number">7.3</span> Spatial reconstruction of scRNA-seq data</h2>
<p>It may be fair to say that the holy grail of spatial transcriptomics is to profile the whole transcriptome at single cell resolution and without dropouts. We have already seen that, with seqFISH+ and ExM-MERFISH, this goal seems to possibly be within reach. However, the goal may be further than is seemingly the case, as the smFISH based techniques are still not generally applied to more than a few dozens to a few hundreds of genes, in the order of 10,000 cells (Figure <a href="current-techs.html#fig:smfish-lm-gene">5.19</a>, Figure <a href="current-techs.html#fig:smfish-lm-cell">5.21</a>), which only covers a small area of tissue. Meanwhile, techniques without single cell resolution and with lower detection efficiency but can cover large swaths of tissue have grown in popularity (Figure <a href="current-techs.html#fig:array-trend">5.34</a>). Hence spatial transcriptomics has not supplanted scRNA-seq – which has also grown tremendously in popularity in recent years <span class="citation">(Svensson, da Veiga Beltrame, and Pachter <a href="#ref-Svensson2020" role="doc-biblioref">2020</a>)</span> – but remains a complement. Spatial data that is not transcriptome wide can be complemented by scRNA-seq for information of other genes; this section reviews computational methods that map cells from scRNA-seq to spatial locations with a small panel of landmark genes and/or to impute gene expression not profiled by the spatial reference in space, or in short spatial reconstruction of scRNA-seq data. These are the most common types of data analysis(Figure <a href="current-analysis.html#fig:analysis-cats">7.4</a>). The two tasks are related but distinct, as when cells from scRNA-seq are mapped to spatial locations, spatial patterns of the genes expressed in the cells are also predicted. However, gene expression can also be predicted at spatial locations without mapping cells to the locations. Spatial data that does not have single cell resolution can be complemented by scRNA-seq for cell type deconvolution of the spots (Section <a href="current-analysis.html#deconvolution">7.4</a>). In turn, spatial data complements scRNA-seq with spatial information such as gene expression patterns and cell neighborhoods.</p>
<p>Attempts at spatial reconstruction of single cell data date back to 2014, when growth in the popularity of scRNA-seq started to pick up pace <span class="citation">(Svensson, da Veiga Beltrame, and Pachter <a href="#ref-Svensson2020" role="doc-biblioref">2020</a>)</span>. Early (2014-2017) methods tend to fall in three categories: direct dimension reduction with PCA, ad hoc scoring, and pseudotime projected into space. The first two have been by and large abandoned due to their limitations, and the third isn’t commonly used. Another category is generative modeling, which we consider intermediate due to its early origin and lasting legacy as some later methods involve more sophisticated generative modeling. Later (2018-present) methods commonly involve a lower dimensional latent space shared by the scRNA-seq and the spatial data, and many different approaches have been tried to obtain the shared latent space and project it back into the higher dimensional space of gene expression. However, other principles were used as well, such as optimal transport, nonlinear direct dimension reduction, black box machine learning, mixture of experts model, and etc.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:reconst-pkgs">Table 7.3: </span>Packages mentioned for spatial reconstruction of scRNA-seq data
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/satijalab/seurat/tree/37b8f45db23a6564c6dfcde238ef41177c8230e8"> Seurat1 </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/nbt.3192"> Spatial reconstruction of single-cell gene expression data </a>
</td>
<td style="text-align:left;width: 5em; ">
2015-04-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/rajewsky-lab/distmap"> DistMap </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1126/science.aan3235"> The Drosophila embryo at single-cell transcriptome resolution </a>
</td>
<td style="text-align:left;width: 5em; ">
2017-10-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/YosefLab/scVI"> gimVI </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://arxiv.org/abs/1905.02269"> A joint model of unpaired data from scRNA-seq and spatial transcriptomics for imputing missing gene expression measurements </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-05-06
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/satijalab/seurat"> Seurat3 </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1016/j.cell.2019.05.031"> Comprehensive Integration of Single-Cell Data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-06-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/MacoskoLab/liger"> LIGER </a>
</td>
<td style="text-align:left;width: 5em; ">
R; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1016/j.cell.2019.05.006"> Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-06-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/tmorikuicr/spresso"> SPRESSO </a>
</td>
<td style="text-align:left;width: 5em; ">
R; Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41598-019-49031-1"> Novel computational model of gastrula morphogenesis to identify spatial discriminator genes by self-organizing map (SOM) clustering </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-08-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/immunogenomics/harmony"> Harmony </a>
</td>
<td style="text-align:left;width: 5em; ">
R; C; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41592-019-0619-0"> Fast, sensitive and accurate integration of single-cell data with Harmony </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-11-18
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/rajewsky-lab/novosparc"> novoSpaRc </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41586-019-1773-3"> Gene expression cartography </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-11-20
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/architverma1/sc-manifold-alignment"> sstGPLVM </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.01.14.906313"> A Bayesian nonparametric semi-supervised model for integration of multiple single-cell experiments </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-01-21
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/zcang/SpaOTsc"> SpaOTsc </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41467-020-15968-5"> Inferring spatial and signaling relationships between cells from single cell transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-04-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/jfnavarro/st_analysis"> st_analysis </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1126/sciadv.abb3446"> Molecular atlas of the adult mouse brain </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-06-26
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> GLISS </a>
</td>
<td style="text-align:left;width: 5em; ">
NA
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.08.12.248971"> Integrative Spatial Single-cell Analysis with Graph-based Feature Learning </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> Tangram </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.08.29.272831"> Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-30
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/tabdelaal/SpaGE"> SpaGE </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1093/nar/gkaa740"> SpaGE: Spatial Gene Enhancement using scRNA-seq </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-09-21
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/kuanglab/FIST"> FIST </a>
</td>
<td style="text-align:left;width: 5em; ">
MATLAB
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1371/journal.pcbi.1008218"> Imputation of Spatially-resolved Transcriptomes by Graph-regularized Tensor Completion </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-04-07
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/welch-lab/liger"> LIGER </a>
</td>
<td style="text-align:left;width: 5em; ">
R; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41587-021-00867-x"> Iterative single-cell multi-omic integration using online learning </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-04-19
</td>
</tr>
</tbody>
</table>
<div id="direct-dr" class="section level3">
<h3><span class="header-section-number">7.3.1</span> Direct dimension reduction</h3>
<p>As already mentioned in our summary of Puzzle Imaging, spatial reconstruction of dissociated tissue can be considered a dimension reduction problem. Here with scRNA-seq, the high dimensional gene expression data is directly projected to 1 to 3 dimensions that correspond to the spatial dimensions.</p>
<p>One of the earliest reconstruction methods (2014) maps single cell qPCR data onto a sphere that mimics the developing mouse otocyst <span class="citation">(Durruthy-Durruthy et al. <a href="#ref-Durruthy-Durruthy2014" role="doc-biblioref">2014</a>)</span>. Ninety six genes were profiled with qPCR in single cells, and the gene expression profiles were projected to the first 3 principal components (PCs), which are then projected onto the surface of a sphere. The sphere is oriented on the dorsal-ventral (DV), anterior-posterior (AP), and left-right (LR) axes by expression of marker genes known to be expressed in one end of those axes. At least for the otocyst, this approach seemed to recapitulate expression patterns of many genes, at least qualitatively, at the resolution of octants. This approach was later adapted to reconstruct the human <span class="citation">(Durruthy-Durruthy et al. <a href="#ref-Durruthy-Durruthy2016" role="doc-biblioref">2016</a>)</span> and mouse <span class="citation">(Mori et al. <a href="#ref-Mori2017" role="doc-biblioref">2017</a>)</span> blastocysts. A one dimensional version of this approach was also adapted to spatially reconstruct cells from the organ of Corti along the apical and basal axis, though the PCA was performed only on DE genes between apical and basal cells and 2 PCs were projected to 1 dimension <span class="citation">(Waldhaus, Durruthy-Durruthy, and Heller <a href="#ref-Waldhaus2015" role="doc-biblioref">2015</a>)</span>.</p>
<p>Direct dimension reduction is still used after 2018, with dimension reductions other than PCA. Another form of dimension reduction for spatial reconstruction is the <a href="https://en.wikipedia.org/wiki/Self-organizing_map">self-organinzing map (SOM)</a> as in the package SPRESSO <span class="citation">(Mori et al. <a href="#ref-Mori2019" role="doc-biblioref">2019</a>)</span>. The Geo-seq mid-gastrula mouse embryo data <span class="citation">(Peng et al. <a href="#ref-Peng2016" role="doc-biblioref">2016</a>)</span> was reconstructed in 3D with genes selected from GO terms; 18 genes selected from a few GO terms could place all microdissected samples into the correct AP/LR quadrant with SOM. However, such genes were found by checking the SOM projections from thousands of GO combinations against the Geo-seq ground truth and may not apply to other biological systems. Also, the spatial reconstruction along the DV axis was not checked, though in Geo-seq, the samples were microdissected along the DV axis with a cryotome in addition to dissection into AP/LR quadrants with LCM.</p>
<p>A more recent, graph based dimension reduction is GLISS <span class="citation">(Zhu and Sabatti <a href="#ref-Zhu2020" role="doc-biblioref">2020</a>)</span>. After using a <a href="https://proceedings.neurips.cc/paper/2005/file/b5b03f06271f8917685d14cea7c6c50a-Paper.pdf">Laplacian score</a> based method to identify landmark genes from spatial data (to be reviewed in Section <a href="current-analysis.html#variable">7.5</a>), a graph is constructed for the scRNA-seq data based on similarity in expression profiles of the landmark genes among cells as a proxy to spatial locations. With this graph, a new set of genes whose expression depend on the structure of the graph, or spatially variable genes, are identified, and added to the landmark genes. A new similarity graph is then constructed with both the landmark genes and spatially variable genes, and the dimension reduction is the eigenvectors of the <a href="https://csustan.csustan.edu/~tom/Clustering/GraphLaplacian-tutorial.pdf">graph Laplacian</a> of this graph, starting from the second eigenvector. One dimensional projection would be the second eigenvector. Two dimensional projection would be the second and third, and so on.</p>
<p>Ligand-receptor (L-R) pairs have also been used for direct dimension reduction, in CSOmap <span class="citation">(Ren et al. <a href="#ref-Ren2020" role="doc-biblioref">2020</a>)</span>. Expression of L-R pairs in scRNA-seq cells is used to construct a cell-cell affinity matrix, with higher affinity meaning that two cells are more likely to be close to each other. Then an algorithm similar to tSNE is used to project the affinity matrix into 3 dimensions, corresponding to the physial dimensions. The <a href="https://en.wikipedia.org/wiki/Kullback–Leibler_divergence">Kullback–Leibler (KL) divergence</a> between the affinity and probability of the two cells to be neighbors is minimized, with constraints of the minimum physical size of the cell and the amount of space available.</p>
</div>
<div id="ad-hoc" class="section level3">
<h3><span class="header-section-number">7.3.2</span> <em>Ad hoc</em> scoring</h3>
<p>The methods above tend to only capture simple spatial patterns with simple gradients along axes, or have low resolution that is effectively restricted to octants or quadrants. More complex patterns with higher resolution can be reconstructed qualitatively with some score that measures similarity between each cell in scRNA-seq and each location in a spatial reference for the genes present in both datasets and favors genes more specific to a subset of cells. The spatial pattern of the score is the predicted gene expression pattern. As the score is qualitative and does not utilize statistical modeling of the data, this is called ad hoc scoring. The spatial reference is FISH (not smFISH) data of a panel of genes, with images for different genes registered onto a common coordinate system. As FISH is not very quantitative, both the spatial and the scRNA-seq data are binarized into “on” and “off” for each gene, and the predicted gene expression patterns based on the score is binarized as well since the score is only qualitative. Such approach is simple to implement, but the binarization misses quantitative nuances of gene expression patterns.</p>
<p>Ad hoc scoring has been used in <em>Platynereis dumerilii</em> brains; the FISH atlas was broken into voxels 3 <span class="math inline">\(\mu\)</span>m on each side, smaller than the average single cell, and 98 landmark genes in the atlas used to predict patterns of other genes in scRNA-seq with a score <span class="citation">(Achim et al. <a href="#ref-Achim2015" role="doc-biblioref">2015</a>)</span>. A different method, DistMap, uses a score based on <a href="https://en.wikipedia.org/wiki/Matthews_correlation_coefficient">Matthew correlation coefficient (MCC)</a> was used to soft assign cells from scRNA-seq to locations in the BDTNP atlas with 84 landmark genes and to predict expression patterns of the other genes <span class="citation">(Karaiskos et al. <a href="#ref-Karaiskos2017" role="doc-biblioref">2017</a>)</span>. The latter method inspired the DREAM Single-cell transcriptomics challenge in 2018 <span class="citation">(Tanevski et al. <a href="#ref-Tanevski2020" role="doc-biblioref">2020</a>)</span>, a competition in which participants select the most informative genes and predict cell locations with 60, 40, and 20 of the 84 BDTNP landmark genes. At least some participating teams adapted the scoring method used in the original DistMap after selecting genes with their own methods <span class="citation">(Alonso, Carrea, and Diambra <a href="#ref-Alonso2020" role="doc-biblioref">2020</a>; Pham et al. <a href="#ref-Pham2020a" role="doc-biblioref">2020</a>)</span>.</p>
</div>
<div id="generative" class="section level3">
<h3><span class="header-section-number">7.3.3</span> Generative models</h3>
<p>Many areas in spatial transcriptomics data analysis describe the data with a plausible statistical model and fit such a model to the data. Generative models have several advantages. First, uncertainties in parameter estimates and model predictions can be computed. Second, the model is more explainable, i.e. that humans may understand contributions of variables to the fitted model. Explainability plays an important role in models identifying spatially variable genes. As already mentioned, some of the segmentation-free smFISH or ISS analysis packages, such as pciSeq, rely on generative models. Generative models are used for spatial reconstruction of scRNA-seq data as well.</p>
<p>The popular scRNA-seq EDA package Seurat originated from spatial reconstruction of scRNA-seq data in 2015, to map cells from scRNA-seq to a WMISH reference with 47 landmark genes <span class="citation">(Satija et al. <a href="#ref-Satija2015" role="doc-biblioref">2015</a>)</span>. The WMISH images were mostly obtained from ZFIN, and divided into 128 bins, which was then collapsed into 64 due to LR symmetry. As WMISH is not very quantitative, the WMISH reference was binarized. Due to the sparsity of scRNA-seq data, the normalized scRNA-seq data was smoothed. Then a mixture of 2 Gaussian distributions was fitted to each gene, for the “on” and the “off” states. With such distributions, the posterior probability that each cell comes from each bin can be calculated with the probability that the cell is “on” of “off” like in the bin for the 47 genes, although cells can very well have intermediate and more nuanced gene expression. The spatial centroid of each cell is the center of mass of the spatial map of the posterior probabilities. So far, the landmark genes have been assumed to be independent, which is unrealistic. Centroids that are close to actual bins are then used to calculate a covariance matrix of a subset of the landmark genes for each bin, with which the Gaussian mixture models and posterior probabilities are updated. While this model seems reasonable, it is no longer used, likely because of the advances in highly multiplexed smFISH and ISS that produced quantitative spatial references that do no need binarization for some tissues, especially the mouse brain. Nevertheless, the scRNA-seq part of Seurat lived on. As already mentioned, WMISH or ISH atlases are the only spatial transcriptomics resources available for some biological systems and most of the atlases are not transcriptome wide, so this method can still be useful.</p>
<p>A different generative model was used to map scRNA-seq cells to a smFISH atlas in the mouse liver <span class="citation">(Halpern et al. <a href="#ref-Halpern2017" role="doc-biblioref">2017</a>)</span>. Six marker genes known to be patterned in the portal-central axis of the hepatic lobule were profiled with smFISH. Then the smFISH data was binned into 9 zone, normalized, and each gene in each zone was modeled with a gamma distribution, which was then multiplied by coefficients correcting for the fact that only part of the cell is in the tissue section for the <span class="math inline">\(\lambda\)</span> of a Poisson distribution to form a negative binomial distribution. The negative binomial distribution was sampled and normalized for the whole cells in scRNA-seq and proportion of UMIs from the gene of interest, which would approximate the distribution of a cell in each zone having expression levels of the gene of interest. The prior probability of a hepatocyte originating from each zone seems to be the relative area of the concentric ring that is each zone, centered on the central vein. With the prior and the sampled distribution of expression of marker genes, the posterior probability of each cell from each zone can be calculated with Bayes rule. To impute expression of genes other than the 6 markers in each zone, the gene count matrix is multiplied to the posterior probability matrix (after weighing the probabilities). Here the 6 markers are assumed to be independent, which might not be realistic. The same approach is still used by the same lab for more recent liver datasets <span class="citation">(Halpern et al. <a href="#ref-Halpern2018" role="doc-biblioref">2018</a>; Droin et al. <a href="#ref-Droin2020" role="doc-biblioref">2020</a>)</span>, although we are unaware of its use outside that lab.</p>
<p>Some of the shared latent space methods are based on generative models as well, with the latent space as part of the model. In gimVI <span class="citation">(Lopez et al. <a href="#ref-Lopez2019" role="doc-biblioref">2019</a>)</span>, which is adapted from scVI specifically to impute gene expression in space by integrating spatial and scRNA-seq data, gene expression in scRNA-seq is modeled with the negative binomial (NB) or zero inflated negative binomial (ZINB) distribution, and the spatial data is modeled with the Poisson or NB distribution (depending on the technology and dataset). The scRNA-seq and spatial data are modeled as coming from a shared latent lower dimensional space, which is decoded back to the higher dimensional gene expression space by a neural network to capture nonlinear structures as part of the mean parameters of the NB, ZINB, or Poisson distributions. The latent space is estimated when the model is fitted with variational Bayesian inference. To impute gene expression in space, the latent space is sampled and passed through the decoding neural network to get the mean parameters of the gene expression distributions for spatial data.</p>
<p>Another generative model with a shared latent space is semi-supervised t-distributed Gaussian process latent variable model (sstGPLVM) <span class="citation">(Verma and Engelhardt <a href="#ref-Verma2020" role="doc-biblioref">2020</a>)</span>. The scRNA-seq or spatial data is modeled as coming from a noisy sample in high dimension from a lower dimensional shared latent space. The latent space can be concatenated to fixed covariates such as batch, technology used to collect data, spatial coordinates, and etc. and is estimated with black box variational inference. Missing data in gene expression and covariates can be estimated from the latent space, thus enabling mapping scRNA-seq cells to spatial coordinates and imputing gene expression, and the latent space can be collapsed across a covariate to remove its effect. The latent space has a Gaussian prior with identity variance. The prior of the high dimensional noiseless space is a Gaussian process with covariance between cells defined by a kernel that is a weighted sum of <a href="https://en.wikipedia.org/wiki/Matérn_covariance_function">Matern 1/2</a> and Gaussian kernels to allow for a non-smooth manifold that better represents data. The input to the kernel is a weighted sum (length scales of kernel) of l1 distance between the cells in the latent space (including the covariates). The noise added to the noiseless high dimensional space to model actual data is a heavy tailed <a href="https://en.wikipedia.org/wiki/Student%27s_t-distribution">Student’s t distribution</a>, to account for overdispersion and non-Gaussian distribution of the data. This method is not specifically designed for spatial data, but can be used to integrate different scRNA-seq datasets as well.</p>
</div>
<div id="latent" class="section level3">
<h3><span class="header-section-number">7.3.4</span> Shared latent space</h3>
<p>There are some additional methods that project scRNA-seq and spatial data into a shared latent space to impute gene expression in space but without generative modeling. Some of them are designed for data integration in general, but included here the authors demonstrated integration of scRNA-seq and spatial data, seeming to intend their packages for such usage.</p>
<p>In version 3 of Seurat <span class="citation">(Stuart et al. <a href="#ref-Stuart2019a" role="doc-biblioref">2019</a>)</span>, the scRNA-seq and spatial datasets are projected into a shared latent space by <a href="https://en.wikipedia.org/wiki/Canonical_correlation">canonical correlation analysis (CCA)</a>, which finds a low dimensional space that maximizes correlation between the two dataset, or by projecting one dataset into a low dimensional PCA space of the other dataset. Then anchor cells are identified, as cells in the two datasets with sufficient shared neighborhood, and the weight of each anchor on each cell in the spatial dataset is calculated by ad hoc scoring favoring closeness in the latent space and more similar shared neighborhood to the anchor. Gene expression is then simply transferred from scRNA-seq to spatial data by multiplying the normalized gene count matrix of genes absent from the spatial data in scRNA-seq with the anchor weight matrix.</p>
<p>LIGER <span class="citation">(Welch et al. <a href="#ref-Welch2019" role="doc-biblioref">2019</a>)</span> is a different data integration method, of which a Seurat wrapper has been implemented. The latent space is inferred by integrative NMF, which finds a set of factors unique to the scRNA-seq or the spatial dataset, and a set of factors shared by both. Gene expression is imputed in spatial data by averaging the expression of genes of interest in the <span class="math inline">\(k\)</span> (50) nearest neighbors (kNN) from the scRNA-seq data in the space spanned by the shared factors.</p>
<p>In SpaGE <span class="citation">(Abdelaal et al. <a href="#ref-Abdelaal2020" role="doc-biblioref">2020</a>)</span>, a common latent space is inferred as such: gene shared by the spatial dataset and scRNA-seq are used to do PCA independently for the two datasets. Then the cosine similarity matrix of the PCs of the two dataset is passed to <a href="https://en.wikipedia.org/wiki/Singular_value_decomposition">singular value decomposition (SVD)</a>. Then the left and and right singular vectors are used to align the PCs to a common latent space of principal vectors. The original data is projected into the space spanned by the principal vectors of the scRNA-seq data. Then kNN is used to project gene expression from scRNA-seq to spatial data.</p>
<p>In Harmony <span class="citation">(Korsunsky et al. <a href="#ref-Korsunsky2019" role="doc-biblioref">2019</a>)</span>, the data, with different batches, is first PCA projected. Then the PCA projection is clustered with an altered <a href="https://medium.com/analytics-vidhya/k-means-clustering-explained-419ee66d095e">k-means clustering</a> algorithm that assigns cells probabilistically to clusters and maximizes diversity in batches in each cluster. Then the batch correction is found by <a href="https://people.cs.pitt.edu/~milos/courses/cs2750-Spring04/lectures/class22.pdf">mixture of expert model</a>. In each cluster, the PCA projection is modeled by a linear combination of variables in the design matrix (containing batch information), with an intercept term for batch free variation in each cluster. The batch correction term is a weighted sum of the linear model predictions excluding the intercept term, weighted by the probabilistic assignment of each cell to each cluster. Then the batch correction term is subtracted from the original PCA projection. The clustering and correction are repeated until convergence. This way, the cells from scRNA-seq and spatial data are aligned in a common latent space. Then gene expression is imputed in spatial data with kNN.</p>
</div>
<div id="reconst-other" class="section level3">
<h3><span class="header-section-number">7.3.5</span> Other principles</h3>
<p>Approaches that do not fall into the categories reviewed above are reviewed in this subsection, including projecting pseudotime into space, black box machine learning, and optimal transport.</p>
<p>In some biological systems, cell differentiation corresponds to physical locations of the cells, so pseudotime, which supposedly arranges cells along differentiation trajectories, have been mapped to space, thus placing dissociated cells in space. For instance, in the bone growth place, cells at different stages of differentiation are physically arranged along the length of the bone in a cylinder, so the pseudotime trajectory of the cells was simply warped into a straight line for spatial reconstruction <span class="citation">(Li et al. <a href="#ref-Li2016" role="doc-biblioref">2016</a>)</span>. Similarly, in Drosophila larva, cell differentiation corresponds to the proximal-distal axis in the antenna disk and the AP axis in the eye disk, so cells from both scRNA-seq and scATAC-seq were binned according to pseudotime and assigned to the corresponding bins in the eye-antenna disk <span class="citation">(Bravo González-Blas et al. <a href="#ref-BravoGonzalez-Blas2020" role="doc-biblioref">2020</a>)</span>. However, this would not work in tissues without such neat correspondence, such as the Drosophila embryo, in which some genes are expressed in periodic patterns to specify segments.</p>
<p>Deep learning libraries such as PyTorch also made it more effective to predict locations for scRNA-seq cells without a pre-conceived statistical model of the data. For instance, after data normalization and batch correction, a deep neural network can be trained on ST data with annotations of spatial regions to predict spatial regions for scRNA-seq data <span class="citation">(Ortiz et al. <a href="#ref-Ortiz2020" role="doc-biblioref">2020</a>)</span>. In addition, PyTorch’s gradient-based optimization has been used to probabilistically map scRNA-seq cells to spatial locations in Tangram <span class="citation">(Biancalani et al. <a href="#ref-Biancalani2020" role="doc-biblioref">2020</a>)</span>. The spatial reference is voxelated, and a mapping matrix of probability of each cell mapping to each voxel is inferred by minimizing KL divergence between mapped and actual cell density in each voxel and favoring stronger correlation between mapped data and the spatial reference in expression of each gene across voxels and gene expression profiles of each voxel.</p>
<p>Thus far, the reconstruction methods do not take spatial autocorrelation, i.e. that cells physically closer to each other are more likely to have more similar gene expression profiles, in the spatial data into account. <a href="https://www.stat.cmu.edu/~larry/=sml/Opt.pdf">Optimal transport</a>, i.e. finding a way to transport a pile of dirt from one place to others with minimum cost, has been used to exploit spatial autocorrelation to map scRNA-seq cells to spatial locations. In novosparc <span class="citation">(Nitzan et al. <a href="#ref-Nitzan2019" role="doc-biblioref">2019</a>)</span>, neighborhood graphs are constructed for scRNA-seq in gene expression space and for spatial reference data in physical space. Then assuming spatial autocorrelation, optimal transport is used to place cells in locations to make the two graphs match. This can be done without gene expression data in the spatial grid, but can be improved with a spatial gene expression reference. In SpaOTsc <span class="citation">(Cang and Nie <a href="#ref-Cang2020" role="doc-biblioref">2020</a>)</span>, first an optimal transport plan from scRNA-seq cells to spatial locations is inferred with gene expression dissimilarity matrices between scRNA-seq cells and between cells and locations and a spatial distance matrix between spatial locations. Then a spatial distance matrix for scRNA-seq cells is imputed based on that optimal transport plan. The plan can also be used to impute gene expression in space. SpaOTsc also uses optimal transport to infer cell-cell interaction, to be reviewed in the Cell-cell Interaction section. A drawback of this kind of method is that because different cell types can mix in the same spatial neighborhood, such as hepatocytes and Kupffer cells in the liver, spatial autocorrelation is not absolute.</p>
<p>Spatial autocorrelation can also be utilized without optimal transport, but with tensor completion in <a href="https://www.tensorlab.net/doc/cpd.html">Canonical Polyadic Decomposition (CPD)</a> form as in FIST <span class="citation">(Li et al. <a href="#ref-Li2020" role="doc-biblioref">2020</a>)</span>. The spatial data can be viewed as a 3 dimensional tensor, with the x and y coordinates and gene expression at each location ((or 4 with z coordinate). CPD is used to improve computational efficiency. In CPD, the tensor is approximated with a sum of rank 1 tensors, i.e. cross products of 3 vectors, one for each dimension. This decomposition, with extra dimensions for unknown gene expressions, is found by minimizing the difference between the reconstructed tensor with the existing tensor for known genes and by favoring spatial autocorrelation of gene expression on a neighborhood graph and favoring similarity of expression of genes with similar functions in a protein-protein interaction graph.</p>
</div>
</div>
<div id="deconvolution" class="section level2">
<h2><span class="header-section-number">7.4</span> Cell type deconvolution</h2>
<p>There is another aspect to how spatial and scRNA-seq data complement each other. In array based techniques that do not have single cell resolution, the cell type composition of each spot can be estimated with scRNA-seq data. Perhaps because of the increasing popularity of ST and Visium, several cell type deconvolution methods have been developed in the past year, falling into three categories: part of other packages, NMF, and statistical modeling. While any tool designed for cell type deconvolution of bulk RNA-seq data can be used, this section specifically focuses on cell type deconvolution tools designed with spatial data in mind.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:deconv-pkgs">Table 7.4: </span>Packages mentioned for cell type deconvolution
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> NMFreg </a>
</td>
<td style="text-align:left;width: 5em; ">
Python; MATLAB
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1126/science.aaw1219"> Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-03-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/satijalab/seurat"> Seurat3 </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1016/j.cell.2019.05.031"> Comprehensive Integration of Single-Cell Data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-06-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/dmcable/RCTD"> RCTD </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41587-021-00830-w"> Robust decomposition of cell type mixtures in spatial transcriptomics </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-05-08
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> Tangram </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.08.29.272831"> Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-30
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/almaan/stereoscope"> stereoscope </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s42003-020-01247-y"> Single-cell and spatial transcriptomics enables probabilistic inference of cell type topography </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-10-09
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> DSTG </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1093/bib/bbaa414"> DSTG: Deconvoluting Spatial Transcriptomics Data through Graph-based Artificial Intelligence </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-01-22
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/MarcElosua/SPOTlight"> SPOTlight </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1093/nar/gkab043"> SPOTlight:Seeded NMF regression to Deconvolute Spatial Transcriptomics Spots with Single-Cell Transcriptomes </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-02-05
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/RubD/Giotto"> Giotto </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s13059-021-02286-2"> Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-08
</td>
</tr>
</tbody>
</table>
<p>Some of the packages already mentioned in previous sections have cell type deconvolution functionalities as well. For instance, Seurat’s data transfer based on anchors between datasets can also be used to transfer cell type annotations, and the <em>ad hoc</em> score for the transferred cell types has been used as a qualitative measure of cell type composition in Visium spots <span class="citation">(Mantri et al. <a href="#ref-Mantri2020" role="doc-biblioref">2020</a>)</span>. Giotto implements 3 methods for qualitative cell type deconvolution: First, a score based on fold change in expression of marker genes in a spot compared to the mean across spots. Second, another score scoring genes for specificity in both scRNA-seq cell types and ST or Visium spots and the sum of the top 100 gene scores is the cell type enrichment score for each spot. For these two methods, p-values are calculated from <a href="https://www.r-bloggers.com/2019/04/what-is-a-permutation-test/">permutation testing</a>. Third, given a fixed set of cell type marker genes, a <a href="https://brilliant.org/wiki/hypergeometric-distribution/">hypergeometric test</a> is used to test for enrichment of marker genes among top 5% expressed genes of the spot. In Tangram, the cell mapping matrix from scRNA-seq to ST or Visium can be inferred as the ground truth cell density per spot can be measured from H&amp;E staining. When cells from scRNA-seq are mapped to spots in ST and Visium, the cell type annotations are also mapped.</p>
<p>In both the prequel and current era, <a href="http://www.cs.cmu.edu/~11755/lectures/Lee_Seung_NMF.pdf">NMF</a> is quite popular among data analysis methods as the factors (cell embeddings) and the gene loadings tend to exhibit block like structures and the values of the basis and the loadings are enforced to be non-negative, corresponding to the non-negative nature of gene expression data and making the results more interpretable. The blocks in the factors may reflect cell types or clusters, and the blocks in gene loadings may reflect cell type marker genes. NMF has been used for cell type deconvolution as well. To address slide-seq (version 1)’s lack of single cell resolution and poor efficiency, NMFreg was developed to reconstruct the expression profile of each spot as a weighted sum of cell type signatures from scRNA-seq <span class="citation">(Rodriques et al. <a href="#ref-Rodriques2019" role="doc-biblioref">2019</a>)</span>. First, scRNA-seq gene count matrix of cell types of interest and cell type annotations is decomposed with NMF, and each factor is assigned to a cell type and one cell type can have multiple factors. Then <a href="https://www.r-bloggers.com/2019/11/non-negative-least-squares/">non-negative least squares</a> is used to compute the weights of the weighted sum of the factors for each spot. As such weights tend not to cleanly assign spots to cell types, perhaps due to the sparsity of scRNA-seq and slide-seq data, the weights are then thresholded. The threshold is the maximum cell loading of cells not assigned to the cell type of interest among in the factors of this cell type. The weights for this cell type is only kept if the <a href="https://mathworld.wolfram.com/L2-Norm.html"><span class="math inline">\(l_2\)</span> norm</a> of the weight vector for these factors exceed the threshold. Another NMF based method, SPOTlight <span class="citation">(Elosua et al. <a href="#ref-Elosua2020" role="doc-biblioref">2020</a>)</span>, uses a very similar principle.</p>
<p>Cell type deconvolution can also be performed by explicitly modeling spot level gene expression in terms of individual cell types. In stereoscope <span class="citation">(Andersson et al. <a href="#ref-Andersson2019" role="doc-biblioref">2019</a>)</span>, a negative binomial distribution is fit to the expression of each gene in each cell type in scRNA-seq data. Then at each spot, gene expression is modeled as a weighted sum of the negative binomial distributions from each cell type, and the weights are estimated by <a href="https://brilliant.org/wiki/maximum-likelihood-estimation-mle/">maximum likelihood estimation (MLE)</a>. In Robust Cell Type Decomposition (RCTD) <span class="citation">(Cable et al. <a href="#ref-Cable2020" role="doc-biblioref">2020</a>)</span>, gene expression at each spot is modeled as a Poisson distribution, whose mean is an expected rate scaled by total transcript count at the spot. The log rate is the sum of the log of weighted sum of mean gene expression for each cell type from a scRNA-seq reference, a fixed spot specific effect term, a gene specific platform random effect, and another gene specific random effect term for <a href="http://biometry.github.io/APES/LectureNotes/2016-JAGS/Overdispersion/OverdispersionJAGS.html">overdispersion</a>. The parameters, including cell type weights, are then estimated with MLE.</p>
<p>More recently, the <a href="https://tkipf.github.io/graph-convolutional-networks/">graph convolutional neural network (GCN)</a> has been applied to cell type deconvolution, in DSTG <span class="citation">(Su and Song <a href="#ref-Su2020" role="doc-biblioref">2020</a>)</span>. First, scRNA-seq cells are randomly assigned to “spots” of 2 to 8 cells, forming a pseudo-ST dataset. Then the pseudo-ST and real ST data are projected to a CCA space, and a mutual <span class="math inline">\(k\)</span> nearest neighbor graph is built in this space. After that both the pseudo and real ST data and the graph are fed into a GCN, trained to minimize cross entropy between imputed cell composition and actual cell composition in the pseudo-ST spots. Finally, the trained model is used to predict cell composition in real ST data.</p>
</div>
<div id="variable" class="section level2">
<h2><span class="header-section-number">7.5</span> Spatially variable genes</h2>
<p>Some genes, such as house keeping genes, are ubiquitously expressed. Such genes, while highly variable at the single cell level, may be interspersed in space so they may not show a spatial trend. Expression of some genes depends on spatial location, which can be due to cell type localization or variation within or independent from cell types. One of the goals of early prequel studies was to identify spatially variable genes, which was done manually, which can be inconsistent and labor intensive. With more quantitative data and data analysis methods, the current era brought identification of spatially variable genes to the next level. Simple methods to identify such genes include dividing the extent of the tissue into a grid and use Fisher’s exact test to test for non-random distribution of transcript counts in the grid, or to run DE between one region — be it a grid cell or a manually annotated histological region — and another region. However, some more sophisticated methods have been developed that avoid the potential arbitrariness of grids and manual annotation, take advantages of increased resolution of spatial transcriptomics. This section reviews these computational methods that identifies genes with expression that depends on spatial locations. Two principles are the most common. One is <a href="https://bookdown.org/rbg/surrogates/chap5.html">Gaussian process regression</a> and generalization to discrete distributions with the log mean parameter modeled as Gaussian process. Another centers on Laplacian scores of graphs. There are also some additional methods using other principles.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:variable-pkgs">Table 7.5: </span>Packages mentioned for spatially variable genes
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/edsgard/trendsceek"> trendsceek </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/nmeth.4634"> Identification of spatial expression trends in single-cell gene expression data </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-03-19
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/Teichlab/SpatialDE"> SpatialDE </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/nmeth.4636"> SpatialDE: identification of spatially variable genes </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-03-19
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/WangPeng-Lab/scGCO"> scGCO </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/491472"> Identification of spatially variable genes with graph cuts </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-12-09
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/satijalab/seurat"> Seurat3 </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1016/j.cell.2019.05.031"> Comprehensive Integration of Single-Cell Data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-06-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/CamaraLab/RayleighSelection"> RayleighSelection </a>
</td>
<td style="text-align:left;width: 5em; ">
R; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1371/journal.pcbi.1007509"> Clustering-independent analysis of genomic data using spectral simplicial theory </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-11-22
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/xzhoulab/SPARK"> SPARK </a>
</td>
<td style="text-align:left;width: 5em; ">
R; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41592-019-0701-7"> Statistical analysis of spatial expression patterns for spatially resolved transcriptomic studies </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-01-27
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/ManchesterBioinference/GPcounts"> GPcounts </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.07.29.227207"> Non-parametric modelling of temporal and spatial counts data from RNA-seq experiments </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-07-30
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> GLISS </a>
</td>
<td style="text-align:left;width: 5em; ">
NA
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.08.12.248971"> Integrative Spatial Single-cell Analysis with Graph-based Feature Learning </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/alexisvdb/singleCellHaystack"> singleCellHaystack </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41467-020-17900-3"> A clustering-independent method for finding differentially expressed genes in single-cell transcriptome data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-28
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/theMILOlab/SPATA"> SPATA </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.10.20.346544"> Inferring spatially transient gene expression pattern from spatial transcriptomic studies </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-10-21
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/RubD/Giotto"> Giotto </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s13059-021-02286-2"> Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-08
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://pypi.org/project/somde/"> SOMDE </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.12.10.419549"> SOMDE: A scalable method for identifying spatially variable genes with self-organizing map </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-24
</td>
</tr>
</tbody>
</table>
<div id="gpr" class="section level3">
<h3><span class="header-section-number">7.5.1</span> Gaussian process regression</h3>
<p>Gene expression in space can be modeled as a 2D Gaussian process. Spatial dependence of gene expression from any finite collection of locations in space can be modeled with a joint multivariate Gaussian distribution, whose covariance matrix can be defined with a kernel, which is typically defined so spatially closer cells or spots have higher covariance.</p>
<p>SpatialDE <span class="citation">(Svensson, Teichmann, and Stegle <a href="#ref-Svensson2018" role="doc-biblioref">2018</a>)</span> is one of the more popular methods to identify spatially variable genes. Spatial gene expression is modeled as a Gaussian process, in which the mean is the mean expression level of the gene, and the covariance matrix has a spatial and non-spatial component. The spatial component uses the Gaussian kernel, in which the covariance decays exponentially with squared distance between cells or spots, with rate of decay controlled by a length scale parameter. In the null model, the gene expression follows a Gaussian distribution without covariance between cells or spots. Then the model likelihood of the fitted full model and the null model are compared with log <a href="https://www.probabilitycourse.com/chapter8/8_4_5_likelihood_ratio_tests.php">likelihood ratio test</a>. The log likelihood ratios under null model are asymptotically <span class="math inline">\(\chi^2\)</span> distributed, and this distribution is used to calculate the p-values of the test. If a gene is found to be significantly spatially variable, then the full model can be fitted with two other kernels, linear and periodic, and compared to the Gaussian kernel with <a href="https://en.wikipedia.org/wiki/Bayesian_information_criterion">Bayesian Information Criterion (BIC)</a> to discover linear and periodic patterns. As gene expression is discrete and not Gaussian, the data needs to be normalized before applying SpatialDE; even then, data normalization does not make the data Gaussian.</p>
<p>The discrete, non-Gaussian distribution of gene expression is directly modeled by SPARK <span class="citation">(S. Sun, Zhu, and Zhou <a href="#ref-Sun2020" role="doc-biblioref">2020</a>)</span>. Gene expression is modeled by a Poisson distribution, with a rate parameter scaled by total transcript count at the spot or cell of interest. The log rate parameter contains a linear model for non-spatial variation in gene expression and can include cell or spot level covariates such as cell type, with non-spatial residuals. The spatial dependence is modeled by a zero mean Gaussian process with either a Gaussian or cosine kernel for the covariance matrix and 5 different length scale parameters are tried for each kernel type, so 10 kernels are tried. The model is fitted with one kernel at a time, with a <a href="https://academic.oup.com/bioinformatics/article/35/3/487/5055584">penalized quasilikelihood algorithm</a>. The p-values are estimated by <a href="https://www.jstor.org/stable/3002019?casa_token=qXYt_raEK8IAAAAA%3A3bbUKlAh6ruVm7KRvoEpjPHfPM_qf4tnaRUadXujACmbEiXoUPINFhTMrE3m7GUTYjgPEYp1i8nIZ1ktuKs3Z5aX2alhiH0pMSnpKB5tiS8dWxA1Cg&amp;seq=1#metadata_info_tab_contents">Satterthwaite method</a>, and the p-values from the 10 kernels are combined with the <a href="https://www.jstor.org/stable/43974709?casa_token=GvHZB2UWAJIAAAAA%3A4h2Wa4vbQog_b_oIXg0uFh4HJeSp-2qHllLvx9K3m6of8C86U0Be_Y8CIGyZ7q0JMl2JPMeKvX1wMHSlxKVedW2-KMzR-wx74wEyqO1Rg74piqmtaw&amp;seq=1#metadata_info_tab_contents">Cauchy p-value combination rule</a>.</p>
<p>Gene expression data may better be modeled with NB or ZINB, which is done in GPcounts <span class="citation">(BinTayyash et al. <a href="#ref-BinTayyash2020" role="doc-biblioref">2020</a>)</span>. The log of the mean parameter of the NB or ZINB, scaled by total transcript count at the cell or spot, is modeled with a Gaussian process with Gaussian kernel for covariance. For ZINB, the dropout probability is related to the NB mean by a <a href="https://en.wikipedia.org/wiki/Michaelis–Menten_kinetics">Michaelis-Menten equation</a>. For one sample, the null hypothesis a constant model, a Gaussian with fixed mean and no covariance between cells or spots, i.e. gene expression does not vary in space. Spatially variable genes are identified with the log likelihood ratio test as in SpatialDE. For two samples, the null hypothesis is that two samples have the same gene expression pattern, and the alternative hypothesis is that two different Gaussian processes are required to model the two samples. Three models are fitted, one for each sample and another fit with both samples as replicates, and the SpatialDE likelihood ratio test is used to compare the separate models to the shared one. The models are fitted with a sparse approximation of variational Bayesian inference.</p>
<p>The size of the covariance matrix of the cells or spots grows quadratically with the number of cells or spots. To speed up computation, SMODE aggregates cells or spots into nodes with SOM, reducing the size of the covariance matrix, before proceeding to a SpatialDE-like test <span class="citation">(Hao, Hua, and Zhang <a href="#ref-Hao2021" role="doc-biblioref">2021</a>)</span>.</p>
</div>
<div id="laplacian" class="section level3">
<h3><span class="header-section-number">7.5.2</span> Laplacian score</h3>
<p>GLISS <span class="citation">(Zhu and Sabatti <a href="#ref-Zhu2020" role="doc-biblioref">2020</a>)</span> has already been mentioned as a method to reconstruct scRNA-seq data in space by projecting scRNA-seq data into a 1 to 3 dimensions that stand for spatial dimensions. The first step of GLISS is to identify spatially variable genes in the spatial reference as landmark genes. In 2005, the Laplacian score was proposed as a method of feature selection, which favors features that preserves the local structure of the data in the feature space and has large variance <span class="citation">(He et al. <a href="#ref-He2020" role="doc-biblioref">2020</a>)</span>. In GLISS, a spatial neighborhood graph is constructed on the spatial reference; two cells or spots have larger edge weight if they are physically close to each other. By default, the graph is a mutual nearest neighbor graph, in which cells or spots are nodes and an edge connects two nodes if they are mutual <span class="math inline">\(k\)</span> nearest neighbors. Then for each gene, a Laplacian score is computed using the gene of interest and the graph Laplacian of the spatial neighborhood graph. Genes with low Laplacian scores are chosen as landmark genes, as a low score favors similarity of gene expression in nearby cells or spots and large variance among the spots, which means spatially coherent regions with high and low expression of the gene. The p-value of the gene is computed by permuting expression of the gene of interest among cells and recomputing the score.</p>
<p>The <a href="https://www2.cs.duke.edu/courses/fall06/cps296.1/Lectures/sec-III-1.pdf">simplical complex</a> is a generalization of the graph that not only includes nodes and edges but also triangles, tetrahedrons, and their higher dimensional generalizations. RayleighSelection implements generalizations of the Laplacian score for simplical complexes for clustering-free DE <span class="citation">(Govek, Yamajala, and Camara <a href="#ref-Govek2019" role="doc-biblioref">2019</a>)</span>. The 1-dimensional Laplacian score, a generalization in which gene expression values are attributed to edges rather than nodes, has been used for DE in scRNA-seq data. The nodes here are clusters of cells and two nodes are connected by an edge when they intersect, as in <a href="https://medium.com/@varad.deshmukh/topological-data-analysis-a-very-short-introduction-611d3238a0bd">topological data analysis (TDA)</a> <span class="citation">(Rizvi et al. <a href="#ref-Rizvi2017" role="doc-biblioref">2017</a>)</span>. P-values of genes were computed by permutation test, permuting expression of a gene of interest among cells. For spatial data, the spatial neighborhood graph was created as the <a href="https://en.wikipedia.org/wiki/Vietoris–Rips_complex">Vietoris-Rips complex</a>. The 0-dimensional, which is the same as the original Laplacian score, was used to identify spatially variable genes. The graph was also created for cells from pairs of cell types and the Laplacian score, with feature as cell type label, was used to identify cell type colocalization.</p>
</div>
<div id="variable-other" class="section level3">
<h3><span class="header-section-number">7.5.3</span> Other principles</h3>
<p>A spatial point pattern is the observed spatial locations of things or events, and a point process is a stochastic mechanism that generated the point pattern. As already mentioned, in pciSeq, transcript spot locations are modeled by a Poisson point process whose intensity itself is modeled with a Gamma distribution. Cell locations can also be modeled as a point process, which is done in trendsceek <span class="citation">(Edsgärd, Johnsson, and Sandberg <a href="#ref-Edsgard2018" role="doc-biblioref">2018</a>)</span>. Each point in a spatial point process can have additional properties other than location, such as gene expression and cell type, which are called marks. If the marks are completely randomly distributed in space, then points with one mark would not be more or less likely to be near points with the same (for categorical marks) or similar (quantitative marks) marks than to points with dissimilar marks. To identify spatial distribution of gene expression that deviates from complete randomness, trendsceek uses 4 mark-segregation summary statistics, which are functions of distance between two points, taking the expected value of a summary statistics on the marks of every pair of points sepearated by the given distance: Stoyan’s mark-correlation function (squared geometric mean of marks of two points normalize by squared mean of marks), mean-mark function (average of marks in two points), variance-mark function (variance of the marks given distance between points), and <a href="https://rdrr.io/cran/spatstat/man/markvario.html">mark-variogram</a> (squared difference of marks of two points). Permutation testing is used to calculate p-values. Regions of interest in the tissue are the regions with <span class="math inline">\(p &lt; 0.05\)</span> from the permutation testing. Perhaps due to the permutation, trendsceek seems to be less scalable and less sensitive than SpatialDE and SPARK <span class="citation">(S. Sun, Zhu, and Zhou <a href="#ref-Sun2020" role="doc-biblioref">2020</a>; Zhang, Feng, and Wang <a href="#ref-Zhang2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>Seurat’s spatial functionalities include finding spatially variable genes, which currently provides two methods, one of this is mark-variogram, inspired by trendsceek. The other is <a href="https://pro.arcgis.com/en/pro-app/tool-reference/spatial-statistics/h-how-spatial-autocorrelation-moran-s-i-spatial-st.htm">Moran’s I</a>, which is a common summary statistics of spatial autocorrelation, as spatially patterned genes also exhibit autocorrelation. However, the problem with Moran’s I is that when almost all cells take very similar values in gene expression, such as when a gene is constitutively expressed or expressed by very few cells, Moran’s I will still indicate strong spatial autocorrelation although the gene is not actually spatially variable.</p>
<p>Giotto <span class="citation">(Dries et al. <a href="#ref-Dries2019" role="doc-biblioref">2019</a>)</span> implements 3 simple and fast methods to find spatially variable genes in addition to wrapping SpatialDE and trendsceek. First, a spatial neighborhood graph is constructed, which can be mutual <span class="math inline">\(k\)</span> nearest neighbors graph, a graph placing an edge when two cells are within a certain distance, or Delaunay triangulation. Then the gene expression is binarized. The first method uses the <a href="https://en.wikipedia.org/wiki/Silhouette_%28clustering%29">silhouette score</a>. In clustering, a measure of whether each point should be assigned to its current cluster or it should better be assigned to a neighboring cluster. The mean silhouette score indicates how tight and segregated the cluster are. Here the clusters are cells expressing the gene of interest and those not expressing. Then a high silhouette score means that cells expressing the gene and those not expressing are well-segregated in space, which means the gene is spatially variable. The second and third method only differ in the way gene expression is binarized. The second uses k-means with <span class="math inline">\(k=2\)</span>, and the third uses a threshold. Then a contingency table <span class="math inline">\(M\)</span> is constructed from neighboring cells in the graph expressing or not expressing the gene; each row is whether a cell expresses the gene, and each column is whether its neighbor also expresses it, so <span class="math inline">\(M_{1,1}\)</span> is the number of distinct pairs of cells both expressing the gene, <span class="math inline">\(M_{1,2}\)</span> is the number of pairs of cells in which source cell is expressing the gene and target cell is not, and so on. <a href="https://mathworld.wolfram.com/FishersExactTest.html">Fisher’s exact test</a> is used to test for dependency in gene expression on whether cells are neighbors.</p>
<p>The KL divergence is a measure of difference between two probability distributions. In singleCellHaystack <span class="citation">(Vandenbon and Diez <a href="#ref-Vandenbon2020" role="doc-biblioref">2020</a>)</span>, the cell density in the tissue (or a PCA, tSNE, or UMAP space) is estimated at grid points with Gaussian kernel density, and normalized to form a probability distribution of locations of cells. Then the probability distribution of whether a gene of interest is expressed at each grid point is compared to the cell density distribution with KL divergence. P-values are computed by permuting gene expression among cells. Again, this is a cluster-free DE method, not designed specifically for spatial data but can be applied to spatial data.</p>
<p>We have already mentioned Markov random field (MRF) models for partitioning a tissue section into cell types and cells. MRF has also been used to identify spatially variable genes, as in scGCO (single-cell graph cuts optimization) <span class="citation">(Zhang, Feng, and Wang <a href="#ref-Zhang2018" role="doc-biblioref">2018</a>)</span>. Expression values of a gene are binned into 2 to 10 categories with Gaussian mixture model clustering. Then a graph connecting cells in space is constructed over the tissue by Delaunay triangulation, and the graph, with the expression category of the gene, is modeled with a MRF. Then as the model favors neighbors in the graph with the same category, edges of the graph are cut to maximize the likelihood of the model, thus identifying not only regions of tissue with an expression category of the gene, but also genes forming such regions. As MRF enforces coherent regions of the tissue to take the same category, while when only gene expression, without spatial information, is considered, not all cells in the region warrant the category. Then the number of cells that truly deserve the category in each region is used to calculate statistical significance of the gene’s spatial variability. The null hypothesis is a homogeneous Poisson point process, in which cells (points) are completely randomly distributed in space and the location of one cell is independent from the location of any other cell. The smallest p-value of any category and any region is reported for the gene of interest.</p>
<p>So far the methods identifying spatially variable genes based on Gaussian process regression commonly use the Gaussian kernel for the covariance matrix, which assumes that the gene expression modeled is weakly stationary, i.e. covariance only depends on distance between cells or spots. This does not take into account <a href="https://arxiv.org/abs/1712.01634v2">anisoptropy</a>, i.e. spatial dependence of gene expression is different in different directions, observed in tissues such as the brain cortex and the hepatic lobule in which cell functions are primarily stratified along one direction or axis. SPATA <span class="citation">(Kueckelhaus et al. <a href="#ref-Kueckelhaus2020" role="doc-biblioref">2020</a>)</span> implements a method to find spatially variable genes for such primary axis. With the interactive <code>shiny</code> app, the user defines this axis, which may or may not be a straight line, and cells within a certain distance from the axis are included for further analysis. Then among the included cells, gene expression and cell type annotations along the axis can be visualized in the <code>shiny</code> app. Then SPATA fits a variety of functions with known forms, e.g. linear or nonlinear descent or ascend, peaks, periodic, and etc. to the gene expression along the axis. For each function, the sum of the residuals is calculated and compared to find functions that better represent the change in gene expression along the axis to identify patterns.</p>
</div>
</div>
<div id="pattern" class="section level2">
<h2><span class="header-section-number">7.6</span> Gene patterns</h2>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:pattern-pkgs">Table 7.6: </span>Packages mentioned for gene patterns
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/Teichlab/SpatialDE"> SpatialDE </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/nmeth.4636"> SpatialDE: identification of spatially variable genes </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-03-19
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/SpatialTranscriptomicsResearch/std-nb"> std-nb </a>
</td>
<td style="text-align:left;width: 5em; ">
C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/362624"> Charting Tissue Expression Anatomy by Spatial Transcriptome Decomposition </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-12-28
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> stLearn </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.05.31.125658"> stLearn: integrating spatial location, tissue morphology and gene expression to find cell types, cell-cell interactions and spatial trajectories within undissociated tissues </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-05-31
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> GLISS </a>
</td>
<td style="text-align:left;width: 5em; ">
NA
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.08.12.248971"> Integrative Spatial Single-cell Analysis with Graph-based Feature Learning </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-13
</td>
</tr>
</tbody>
</table>
<p>When spatially variable genes are identified, a question naturally arises: Are there archetypal patterns among these spatially variable genes? As already reviewed in Section <a href="prequel-analysis.html#prequel-analysis">3</a>, comparing and classifying gene expression patterns was a major topic in the prequel era. Such interest persists in the current era, although we find no evidence that current era gene pattern analysis is significantly influenced by the prequel antecedents, although factor analysis and NMF have been used in both eras.</p>
<p>The most straightforward way to identify archetypal gene patterns is to cluster the gene expression patterns and obtain the cluster centers to represent the cluster. This has been used to analyze mouse brain voxelation data in 2009 <span class="citation">(An et al. <a href="#ref-An2009" role="doc-biblioref">2009</a>)</span>. <a href="https://en.wikipedia.org/wiki/Wavelet_transform">Wavelet transform</a> was applied to the data and the Euclidean distance between the wavelet feature vectors was used to measure gene similarity. Gene similarity between pre-defined “typical” genes and other genes was one way to find groups of similar genes and k-means clustering is another. Some package already reviewed also have functionality to identify archetypal gene patterns. In DistMap and SPARK <span class="citation">(Karaiskos et al. <a href="#ref-Karaiskos2017" role="doc-biblioref">2017</a>; S. Sun, Zhu, and Zhou <a href="#ref-Sun2020" role="doc-biblioref">2020</a>)</span>, the gene patterns are clustered with <a href="https://towardsdatascience.com/understanding-the-concept-of-hierarchical-clustering-technique-c6e8243758ec">hierarchical clustering</a>, and the individual clusters are obtained by tree cut. In Giotto <span class="citation">(Dries et al. <a href="#ref-Dries2019" role="doc-biblioref">2019</a>)</span>, a gene-gene correlation matrix (by default Pearson) is computed, which is then hierarchically clustered. Then the mean or centroid of each cluster is taken to represent that cluster. SpatialDE <span class="citation">(Svensson, Teichmann, and Stegle <a href="#ref-Svensson2018" role="doc-biblioref">2018</a>)</span> also clusters gene expression patterns, in automatic expression histology (AEH), which implements a Gaussian process generalization of <a href="https://towardsdatascience.com/gaussian-mixture-models-d13a5e915c8e">Gaussian mixture model clustering</a>. The number of components is set by the user, and the model is fitted to infer the mean pattern of each component. In GLISS <span class="citation">(Zhu and Sabatti <a href="#ref-Zhu2020" role="doc-biblioref">2020</a>)</span>, the archetypal patterns are identified in the reconstructed latent space as gene expression in the latent space is <a href="https://en.wikipedia.org/wiki/Smoothing_spline">spline smoothed</a>, and the spline coefficients are clustered.</p>
<p>Beyond clustering, a common way to identify archetypal gene patterns is <a href="https://en.wikipedia.org/wiki/Factor_analysis">factor analysis</a>. This has already been done in the prequel era <span class="citation">(Pruteanu-Malinici, Mace, and Ohler <a href="#ref-Pruteanu-Malinici2011" role="doc-biblioref">2011</a>)</span>, but is further developed in the current era. Factor analysis tries to model higher dimensional data as a linear combination of a smaller number of variables called “factors”, and PCA is a type of factor analysis. A prostate cancer ST dataset has been modeled with Poisson factor analysis <span class="citation">(Berglund et al. <a href="#ref-Berglund2018" role="doc-biblioref">2018</a>)</span>. The observed UMI counts at each spot is modeled as a sum of factors, each of which is Poisson distributed, with its own rate parameter, which in turn depends on Gamma distributed factor, gene, and spot level parameters that may account for overdispersion though this model does not entirely capture the mean-variance relationship of NB. The parameters are estimated from <a href="https://en.wikipedia.org/wiki/Markov_chain_Monte_Carlo">MCMC</a> sampling of the posterior of this model. Once the parameters are estimated, the individual factors can be calculated from the parameters based on the model. The factors seem to indicate regions in the tumor, such as cancer, stroma, and regions with immune cell infiltration. As NB may describe gene expression better than the Poisson distribution, a NB adaptation of the above Poisson factor analysis model has been developed <span class="citation">(Maaskola et al. <a href="#ref-Maaskola2018" role="doc-biblioref">2018</a>)</span>. The observed UMI count at each spot is modeled as a sum of NB factors, whose rate parameter can incorporate gene, spot, and experiment level covariates. The package stLearn <span class="citation">(Pham et al. <a href="#ref-Pham2020a" role="doc-biblioref">2020</a>)</span>, which also implements methods to identify cell-cell interactions and spatial regions, uses PCA, <a href="http://wwwf.imperial.ac.uk/~nsjones/TalkSlides/HyvarinenSlides.pdf">ICA</a>, and factor analysis to detect microenvironments in the tissue as again, the factors can correspond to specific regions in the tissue.</p>
</div>
<div id="region" class="section level2">
<h2><span class="header-section-number">7.7</span> Spatial regions</h2>
<p>As already mentioned in trendsceek and scGCO, the problem of identifying spatially variable genes is closely related to identifying regions in tissue defined by gene expression. When archetypal gene patterns are identified, a related question arises: Do the patterns define novel anatomical regions in the tissue? As seen in the previous section, archetypal gene patterns, such as in factors, can reflect tissue regions. There are also methods that identify such regions without first identifying spatially variable genes and/or archetypal gene patterns. In the prequel era (Chapter <a href="prequel-analysis.html#prequel-analysis">3</a>), some studies clustered the voxels based on gene expression to identify spatial regions in the tissue, with either k-means clustering or co-clustering <span class="citation">(Zhang et al. <a href="#ref-Zhang2013" role="doc-biblioref">2013</a>; Bohland et al. <a href="#ref-Bohland2010a" role="doc-biblioref">2010</a>; Ko et al. <a href="#ref-Ko2013" role="doc-biblioref">2013</a>)</span>, or with Potts model <span class="citation">(Pettit et al. <a href="#ref-Pettit2014" role="doc-biblioref">2014</a>)</span>. More sophisticated clustering methods have been developed in the current era tot identify spatial regions. However, as different cell types can reside in the same spatial neighborhood, and conversely, cells from one cell type can reside in different regions of the tissue, MRF has been used to find spatially coherent regions that can contain multiple cell types.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:region-pkgs">Table 7.7: </span>Packages mentioned for spatial regions
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://bitbucket.org/qzhudfci/smfishhmrf-py/src/default/"> smfishHmrf </a>
</td>
<td style="text-align:left;width: 5em; ">
R; Python; C
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/nbt.4260"> Identification of spatially associated subpopulations by combining scRNAseq and sequential fluorescence in situ hybridization data </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-10-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/eilslabs/ssam"> SSAM </a>
</td>
<td style="text-align:left;width: 5em; ">
Python; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/800748"> Segmentation-free inference of cell types from in situ transcriptomics data </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-10-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> stLearn </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.05.31.125658"> stLearn: integrating spatial location, tissue morphology and gene expression to find cell types, cell-cell interactions and spatial trajectories within undissociated tissues </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-05-31
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/edward130603/BayesSpace"> BayesSpace </a>
</td>
<td style="text-align:left;width: 5em; ">
R; C++
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.09.04.283812"> BayesSpace enables the robust characterization of spatial gene expression architecture in tissue sections at increased resolution </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-09-05
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/kharchenkolab/Baysor"> Baysor </a>
</td>
<td style="text-align:left;width: 5em; ">
Julia
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.10.05.326777"> Bayesian segmentation of spatially resolved transcriptomics data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-10-06
</td>
</tr>
</tbody>
</table>
<p>SSAM <span class="citation">(Park et al. <a href="#ref-Park2019" role="doc-biblioref">2019</a>)</span>, already reviewed in Section <a href="current-analysis.html#preprocessing">7.1</a>, also uses clustering to identify tissue domains in smFISH or ISS data but without cell segmentation. StLearn <span class="citation">(Pham et al. <a href="#ref-Pham2020a" role="doc-biblioref">2020</a>)</span> develops further on top of clustering. First, a pretrained CNN is used to extract a 2048 dimensional feature vector from the H&amp;E image behind each ST or Visium spot. The cosine similarity between the feature vectors from neighboring spots is then calculated. To normalize data, the gene expression data is smoothed in space, and the smoothing is weighted by the cosine similarity of feature vectors between spots. Then the spots are clustered with <a href="https://en.wikipedia.org/wiki/Louvain_method">Louvain</a> or k-means. A spatial <span class="math inline">\(k\)</span> nearest neighbor graph is constructed, and used to refine the clustering. If a gene expression based cluster is broken into multiple pieces in space, then those pieces would become subclusters. Singleton spots are merged with a nearby cluster if the singletons have enough spatial neighbors in that cluster.</p>
<p>BayesSpace <span class="citation">(Zhao et al. <a href="#ref-Zhao2020" role="doc-biblioref">2020</a>)</span> incorporates both Gaussian mixture model clustering and MRF. The ST or Visium data is first projected to a low dimensional space, such as by PCA. Then for each spot, the low dimensional projection of that spot is modeled with a Gaussian mixture model, with a pre-defined number of components or clusters. The spatial neighborhood graph is simply the square grid of spots for ST and the hexagonal grid for Visium. The model has a MRF prior to encourage neighboring spots to be assigned to the same cluster. The cluster assignment is initiated with non-spatial clustering, and the parameters of the model are estimated by MCMC. In addition, BayesSpace can increase the resolution of ST and Visium. Each spot is subdivided and initiated with the dimension reduction values at the spot, and an additional parameter is added to the model that nudges the dimension reduction values at each sub-spot while preserving the sum at the spot level. The nudging parameters are estimated by MCMC along with with other parameters.</p>
<p>As already reviewed in Section <a href="current-analysis.html#preprocessing">7.1</a>, Baysor <span class="citation">(Petukhov et al. <a href="#ref-Petukhov2020" role="doc-biblioref">2020</a>)</span> uses MRF to delineate cell type regions in the tissue without cell segmentation. MRF is used to identify spatial regions for cell or spot level data as well. Like in the 2014 <em>Platynereis dumereilii</em> atlas <span class="citation">(Pettit et al. <a href="#ref-Pettit2014" role="doc-biblioref">2014</a>)</span>, smfishHmrf <span class="citation">(Zhu et al. <a href="#ref-Zhu2018" role="doc-biblioref">2018</a>)</span> also uses Potts model for dependence of label on neighborhood. As seqFISH data is quantitative, gene expression of each cell is modeled with a Gaussian mixture model, with as many components as there are there are region labels. The data needs to be normalized, although data normalization methods don’t typically turn the distribution of gene expression Gaussian. Again, the parameters, i.e. the label assignment, and mean and covariance matrices for each Gaussian component, are estimated by EM, initiated with k-means clustering of the cells.</p>
</div>
<div id="cell-interaction" class="section level2">
<h2><span class="header-section-number">7.8</span> Cell-cell interaction</h2>
<p>Related to spatial regions is cell-cell interaction: Suppose a distinct neighborhood of the tissue has been identified with one of the methods in the previous section, and the neighborhood contains different cell types. Then it’s natural to ask whether these cell types interact by their spatial proximity. Such information is lost in scRNA-seq. The composition of tissue neighborhoods can be characterized with existing tools. For instance, in smFISH or ISS data, we can count the number of cell types within a certain distance from each cell, as was done in the hypothalamus and the motor cortex MERFISH studies <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2018" role="doc-biblioref">2018</a>; Zhang et al. <a href="#ref-Zhang2020" role="doc-biblioref">2020</a>)</span>. We can model the data as a marked spatial point process, in which each point is a cell, with cell type annotations as marks, and use cross-type <a href="https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Kest">K or L function</a> to find cell types that colocalize; the <a href="https://www.rdocumentation.org/packages/spatstat/versions/1.64-1/topics/Lcross">cross-type L function</a> has been used in spicyR <span class="citation">(Canete and Patrick <a href="#ref-spicyR" role="doc-biblioref">2020</a>)</span> for this purpose. For ST and Visium, we can use one of the cell type deconvolution methods to find the number and proportion of cell types per unit area in each tissue region and cell type colocalization. When two cell types colocalize, they might interact with secreted ligands or ligands and receptors bound to the membrane. Expression of ligand-receptor (L-R) pairs in neighboring cells is often used to identify cell-cell interaction in spatial data, and the CellPhoneDB <span class="citation">(Efremova et al. <a href="#ref-Efremova2020" role="doc-biblioref">2020</a>)</span> database of ligands, receptors, and their interactions is often used to identify such L-R pairs. Another type of analysis going beyond colocalization tests for effects of cell-cell interaction or cell type colocalization on gene expression.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:interact-pkgs">Table 7.8: </span>Packages mentioned for cell-cell interactions
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/damienArnol/svca"> SVCA </a>
</td>
<td style="text-align:left;width: 5em; ">
R; Python; C; C++; Fortran
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1016/j.celrep.2019.08.077"> Modeling Cell-Cell Interactions from Spatial Molecular Data with Spatial Variance Component Analysis </a>
</td>
<td style="text-align:left;width: 5em; ">
2019-10-01
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/zcang/SpaOTsc"> SpaOTsc </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41467-020-15968-5"> Inferring spatial and signaling relationships between cells from single cell transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-04-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/saezlab/misty/"> MISTy </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.05.08.084145"> Explainable multi-view framework for dissecting inter-cellular signaling from highly multiplexed spatial data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-05-10
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="NA"> stLearn </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.05.31.125658"> stLearn: integrating spatial location, tissue morphology and gene expression to find cell types, cell-cell interactions and spatial trajectories within undissociated tissues </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-05-31
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/livnatje/DIALOGUE"> DIALOGUE </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.08.11.245472"> Mapping multicellular programs from single-cell profiles </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-08-11
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/xiaoyeye/GCNG"> GCNG </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s13059-020-02214-w"> GCNG: Graph convolutional networks for inferring cell-cell interactions </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-12-10
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/RubD/Giotto"> Giotto </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s13059-021-02286-2"> Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2021-03-08
</td>
</tr>
</tbody>
</table>
<div id="l-r" class="section level3">
<h3><span class="header-section-number">7.8.1</span> Ligand-receptor pairs</h3>
<p>In stLearn <span class="citation">(Pham et al. <a href="#ref-Pham2020a" role="doc-biblioref">2020</a>)</span>, CellPhoneDB is used to identify L-R coexpression in neighboring spots, and the p-value of the coexpression is computed by permutation testing. Then regions with diverse cell types (from Seurat label transfering or cell type deconvolution) and L-R coexpression in neighboring spots are identified as regions where cells are likely to be signaling to each other. A similar strategy is used in Giotto. Giotto identifies cell type colocalization by labeling edges of the spatial neighborhood graph as homo- or heterotypic and permutes cell type labels to find whether the cell types are more or less likely to colocalize than expected from completely random cell type localization. L-R coexpression in neighboring cells on the spatial neighborhood graph from two cell types is identified and the p-values of the coexpression scores are computed by permutation testing, permuting locations of cells within each cell type.</p>
<p>While MRF, stLearn, and Giotto only use the immediate neighbors on the spatial neighborhood graph, there is a method that can capture higher order structures of the graph. In GCNG <span class="citation">(Yuan and Bar-Joseph <a href="#ref-Yuan2019" role="doc-biblioref">2019</a>)</span>, the spatial neighborhood graph is constructed as an edge connects a cell to its 3 nearest neighbors. Then both the gene count matrix and the normalized Laplacian of the neighborhood graph are fed into a graph convolutional neural network (GCN), which is trained on known L-R pairs. The GCN can then predict novel pairs of genes involved in signaling, and if trained on the direction of interaction in the L-R pairs, it can also predict the direction of causality in the novel pairs.</p>
<p>SpaOTsc has already been mentioned in Section <a href="current-analysis.html#reconstruction">7.3</a>. To recapitulate, SpaOTsc uses optimal transport from scRNA-seq cells to spatial locations to impute a spatial cell-cell distance matrix for scRNA-seq cells, and the optimal transport plan can be used to impute gene expression in space. With the cell-cell distance matrix, another optimal transport plan from ligands to receptors can be inferred, interpreted as how likely one cell communicates with another. A disadvantage of spatial neighborhood graph is that common ways of construction are somewhat arbitrary. For instance, <span class="math inline">\(k\)</span> nearest neighbor is a common way to construct the graph, but this <span class="math inline">\(k\)</span> is somewhat arbitrary, although cell signaling can occur over a distance with secreted ligands. Here no such graph is used; the length scale of interaction is inferred by <a href="https://towardsdatascience.com/random-forest-3a55c3aca46d">random forest</a>. Random forest models are trained with expression of the ligand and genes correlated with a downstream target gene within a certain distance from the cells expressing the target gene are the input features. Receptor expression is the sample weights, and the target gene is to be predicted by the random forest model. Several different length scales are tried, and the one resulting into the most feature importance of the ligand is used. When L-R information is unavailable, interactions between genes can be inferred by partial information decomposition, i.e. how much unique information can a source gene provide on a target gene in a spatial neighborhood.</p>
<p>With a very different model, DIALOGUE <span class="citation">(Jerby-Arnon and Regev <a href="#ref-Jerby-Arnon2020" role="doc-biblioref">2020</a>)</span> identifies genes that may be involved in interactions between cell types. In a niche in a tissue, different cell types can respond to the same environmental cue in a concerted manner though each cell type changes gene expression in a different way. DIALOGUE aims to identify such concerted gene programs in each cell type. First, the gene expression data is projected into a lower dimensional space in which correlation between all pairs of cell types across niches is maximized, and the basis of this space is ordered in descending strength of correlation. This is similar to CCA, but with a penalty term to enforce sparsity in gene loading. Here the niche is a patch of cells in space with a predefined number of cells. Then each cell type has a rotation matrix that projects cells into this lower dimensional space, and different cell types from the same niche should be close to each other in this space. In this projection, for each dimension, a gene is added to the multicellular program (MCP) of each cell type if its expression among cells of this cell type correlates with the projection of this cell type in this dimension and is significantly associated with the projection of other cell types while accounting for cell type level and niche level covariates such as sample, age, and gender. Thus the MCPs could be cell type specific co-regulated gene programs. Putative signaling between cell types can be identified by finding known L-R pairs in the MCPs: Each cell type is added the L-R graph as a node, and is connected to a gene if the gene is present in the MCP for this cell type. Then a path connecting one cell type to a ligand to a receptor and then to another cell type suggests signaling between the two cell types.</p>
</div>
<div id="gene-cell-interact" class="section level3">
<h3><span class="header-section-number">7.8.2</span> Genes associated with cell-cell interaction</h3>
<p>Gene expression can be affected by several different factors, including cell type, local environment, interaction with other cells, and so on. Some packages have been developed to identify genes whose expression is associated with one or more of these factors, without using L-R databases. Within one cell type, Giotto uses classical DE (Student’s t-test, <a href="https://en.wikipedia.org/wiki/Mann–Whitney_U_test">Wilcoxon rank sum test</a>, <a href="https://bioconductor.org/packages/release/bioc/html/limma.html">limma</a>, and permutation of spatial locations) to find DE genes between neighbors of cells of another cell type and non-neighbors. Other packages implement more complex models that account for more of these factors associated with gene expression.</p>
<p>Spatial variance component analysis (SVCA) <span class="citation">(Arnol et al. <a href="#ref-Arnol2019a" role="doc-biblioref">2019</a>)</span> models the expression of each gene of interest among the cells as a 0 mean Gaussian process. The covariance has the following components: First, the intrinsic variability, which can be cell types or continuous cell states. In the latter case, the covariance matrix of this component is the covariance between cells with genes other than the gene of interest that is modeled. Second, the spatial neighborhood. A neighborhood graph is not constructed, and the covariance matrix of this component is computed with the Gaussian kernel in which covariance decreases with distance between cells. Third, cell-cell interaction. The covariance matrix of this term is the covariance between cells weighed by a Gaussian kernel for distance between cells, so gene expression in nearby cells contributes more to this component. Finally, the residual has an identity covariance matrix, so in the residual, cells are independent from each other. The parameter to be estimated are weights of each of these components and the length scale parameter of the Gaussian kernel, which are estimated with MLE. Significance of the cell-cell interaction component is calculated by likelihood ratio test between the full model and a reduced model without the cell-cell interaction component. Again, as gene expression is modeled as Gaussian, the data needs to be normalized before using this method.</p>
<p>Like SVCA, Multiview Intercellular SpaTial modeling framework (MISTy) <span class="citation">(Tanevski et al. <a href="#ref-Tanevski2020" role="doc-biblioref">2020</a>)</span> also models expression of each gene of interest among the cells, but with ensemble learning, in which any machine learning method that is explainable (i.e. feature importance can be extracted) and suitable for <a href="https://www.analyticsvidhya.com/blog/2015/08/introduction-ensemble-learning/">ensemble learning</a> can be used. In each view, which can be intrinsic cell state, spatial neighborhood (juxtaview), or wider tissue structure (paraview), features are extracted from gene expression that represent the view and used in machine learning methods such as random forest to predict expression of a gene of interest. For intrinsic cell state, the features are expression of other genes. For juxtaview, the features are sum of expression of other genes in neighboring cells in the spatial neighborhood graph. For paraview, the feature are sum of expression of other genes in all cells in the tissue weighed by distance to each cell with a Gaussian kernel. Other views, with other feature engineering, can also be used. The full model is a linear combination of predictions of each view. In other words, the contribution of each view is determined by linear regression with prediction of each view as a covariate to predict the expression of the gene of interest. For each view, the importance of each feature is assessed as the z-score of the feature importance (e.g. from random forest) multiplied by 1 minus the p-value of the coefficient of this view in the linear regression model, so views that contribute significantly to the ensemble model and features in each of these views that are more important than other features in the same views stand out. This way, interaction among genes at different spatial scales can be identified.</p>
</div>
</div>
<div id="gene-interaction" class="section level2">
<h2><span class="header-section-number">7.9</span> Gene-gene interaction</h2>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:gene-int-pkgs">Table 7.9: </span>Packages mentioned for gene-gene interactions
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/zcang/SpaOTsc"> SpaOTsc </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41467-020-15968-5"> Inferring spatial and signaling relationships between cells from single cell transcriptomic data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-04-29
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/saezlab/misty/"> MISTy </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.05.08.084145"> Explainable multi-view framework for dissecting inter-cellular signaling from highly multiplexed spatial data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-05-10
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://bioconductor.org/packages/release/bioc/html/scHOT.html"> scHOT </a>
</td>
<td style="text-align:left;width: 5em; ">
R
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41592-020-0885-x"> Investigating higher-order interactions in single-cell data with scHOT </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-07-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/doraadong/MESSI"> MESSI </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1093/bioinformatics/btaa769"> Identifying signaling genes in spatial single cell expression data </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-09-04
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/xiaoyeye/GCNG"> GCNG </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1186/s13059-020-02214-w"> GCNG: Graph convolutional networks for inferring cell-cell interactions </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-12-10
</td>
</tr>
</tbody>
</table>
<p>Some of the packages already reviewed can also infer interactions between genes, such as GCNG, SpaOTsc, and MISTy. GCNG and SpaOTsc predict potential L-R pairs, and MISTy identify genes whose expression at a given spatial scale is associated with another gene of interest. The package scHOT <span class="citation">(Ghazanfar et al. <a href="#ref-Ghazanfar2020" role="doc-biblioref">2020</a>)</span> tests for association of correlation between genes with pseudotime or spatial locations by permutation testing, permuting locations of cells along pseudotime or in space. The package Mixture of Experts for Spatial Signaling genes Identification (MESSI) <span class="citation">(Li et al. <a href="#ref-Li2020" role="doc-biblioref">2020</a>)</span> uses a mixture of experts model to predict expression of response genes with a set of features. A spatial neighborhood graph of the cells is constructed with Delaunay triangulation The features include all genes quantified in the dataset that are also found in a L-R database, expression of genes in the L-R database in neighboring cells, cell type of neighboring cells, and etc. The response genes are all genes quantified other than the L-R genes used as features. Each cell is assigned to exactly one “expert”, i.e. subtype. For each expert, expression of response genes in each cell is modeled with linear regression with the features as covariates. The parameters of the linear models and assignment of cells to experts are estimated with MLE, where the log likelihood is maximized with EM. This model can be trained in a control sample and used to predict gene expression in experimental samples. If expression of a gene is as well predicted as in the control, then signaling may not have changed in the experimental condition. If prediction becomes worse, then there may be a change in signaling involving this gene, and the experts whose coefficients significantly differ between the control and experimental models suggest cell populations involved in the signaling change.</p>
</div>
<div id="subcellular" class="section level2">
<h2><span class="header-section-number">7.10</span> Subcellular transcript localization</h2>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:subcellular-pkgs">Table 7.10: </span>Packages mentioned for subcellular transcript localization
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://bitbucket.org/muellerflorian/fish_quant/src/master/"> FISH_quant </a>
</td>
<td style="text-align:left;width: 5em; ">
MATLAB
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41467-018-06868-w"> A computational framework to study sub-cellular RNA localization </a>
</td>
<td style="text-align:left;width: 5em; ">
2018-11-02
</td>
</tr>
</tbody>
</table>
<p>So far, except for segmentation free data analysis methods of smFISH and ISS images, all analysis methods are at the cellular or spot level. However, transcripts do show inhomogeneous subcellular localization that can be biologically relevant, such as whether the transcripts are translated in the endoplasmic reticulum (ER) or the cytoplasm. Thirty four lncRNAs have been manually classified into 5 types of subcellular patterns: one or two large foci in nucleus, large foci and dispersed single molecules in nucleus, no foci in nucleus, nucleus and cytoplasm, and cytoplasmic <span class="citation">(Cabili et al. <a href="#ref-Cabili2015" role="doc-biblioref">2015</a>)</span>. The bDNA-smFISH study in 2013 that profiled 928 genes in cultured cells, though each gene was profiled in different cells, generated features that characterize subcellular transcript localization (mRNAs of protein coding genes) which were used to cluster cells <span class="citation">(Battich, Stoeger, and Pelkmans <a href="#ref-Battich2013" role="doc-biblioref">2013</a>)</span>. These features include closest distance of a transcript spot to cell outline, distance to cell centroid, distance to nuclear centroid, radius to include 5%, 10%, 15%, 25%, 50%, and 75% of all spots in the cell, mean distance of a spot to other spots (related to Ripley’s K or L function), and variance of distance to other spots. The package FISHquant <span class="citation">(Samacoits et al. <a href="#ref-Samacoits2018a" role="doc-biblioref">2018</a>)</span> uses additional features derived from Ripley’s L function of subcellular transcript localization. Then it uses these features to simulate smFISH data, cluster cells, and classify transcript localization patterns.</p>
<p>Whether transcripts are located in the nucleus or in the cytoplasm has also been used for RNA velocity in a MERFISH study <span class="citation">(C. Xia, Babcock, et al. <a href="#ref-Xia2019c" role="doc-biblioref">2019</a>)</span>. In traditional RNA velocity based on scRNA-seq <span class="citation">(La Manno et al. <a href="#ref-LaManno2018" role="doc-biblioref">2018</a>)</span>, when there are more transcripts with intronic reads — i.e. nascent transcripts not yet spliced — than expected from steady state in a cell, then the gene of interest is up regulated, and conversely, if there are fewer transcripts with intronic reads, then the gene may be down regulated. In other words, intronic reads not yet spliced out gives a glimpse into a near future transcriptome of the cell. In this MERFISH study, instead of introns that require separate probes from exons, transcripts inside the nucleus are taken to be nascent, i.e. not yet exported from the nucleus, and used in lieu of intronic reads as in scRNA-seq for RNA velocity. In this study, the ER was also stained for and segmented and genes with transcripts enriched in the ER were also identified.</p>
<p>These studies analyzing subcellular transcript localization were all performed on cultured cells rather than in tissues, so there is no highly multiplexed smFISH data on in vivo subcellular transcript localization yet. Furthermore, as the cells cultures used in these studies grow on a plate in single layer, while cells stack on top of each other through the thickness of the section, cell segmentation in tissue can be more challenging. Some of the features used to characterize subcellular transcript localization, such as distance to cell outline and Ripley’s L function (with edge correction), depend on accurate cell segmentation, which as already explained in Section <a href="current-analysis.html#preprocessing">7.1</a>, is challenging. Subcellular transcript location can be modeled as a spatial point pattern in 3D or collapsed into 2D, and analyses such as finding effects of covariates such as whether the spot is in the nucleus, distance from the nucleus, distance from the cell outline, and etc., and whether the pattern exhibits clustering (e.g. foci) or inhibition (i.e. more spaced out than expected from <a href="https://en.wikipedia.org/wiki/Complete_spatial_randomness">CSR</a>). However, the observational window of the point process, i.e. cell segmentation, can greatly affect results of spatial point pattern analysis. For instance, when the convex hull of some spots are taken to be the observational window, then the point pattern may not appear clustered. However, if the actual observational window is much larger than the convex hull, then the point process is in fact clustered. Hence accurate cell segmentation is important to analyses of subcellular transcript localization patterns. Furthermore, some smFISH or ISS datasets only provide 2D cell segmentations, and resolution in the z axis tends to be lower than that in the x and y axes. The implications of collapsing 3D into 2D, and when 3D segmentation is available, the lower resolution in the z axis are yet to be determined.</p>
</div>
<div id="HnE" class="section level2">
<h2><span class="header-section-number">7.11</span> Gene expression imputation from H&amp;E</h2>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:hne-pkgs">Table 7.11: </span>Packages mentioned for gene expression imputation from H&amp;E
</caption>
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Language
</th>
<th style="text-align:left;">
Title
</th>
<th style="text-align:left;">
Date published
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/ludvb/xfuse"> Xfuse </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1101/2020.02.28.963413"> Super-resolved spatial transcriptomics by deep data fusion </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-03-13
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/bryanhe/ST-Net"> ST-Net </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41551-020-0578-x"> Integrating Spatial Gene Expression and Breast Tumour Morphology via Deep Learning </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-06-22
</td>
</tr>
<tr>
<td style="text-align:left;width: 8em; ">
<a href="https://github.com/alonalj/PathoMCH"> PathoMCH </a>
</td>
<td style="text-align:left;width: 5em; ">
Python
</td>
<td style="text-align:left;width: 15em; ">
<a href="https://doi.org/10.1038/s41598-020-75708-z"> Spatial transcriptomics inferred from pathology whole-slide images links tumor heterogeneity to survival in breast and lung cancer </a>
</td>
<td style="text-align:left;width: 5em; ">
2020-11-02
</td>
</tr>
</tbody>
</table>
<p>Although ST and Visium do not have single cell resolution, the tissue sections can be H&amp;E stained prior to library preparation, thus the transcriptomes of the spots can be mapped to H&amp;E tissue morphology. H&amp;E is also commonly used in clinical pathology, while ST and Visium are not used for diagnostic purposes. The package ST-Net was developed to use a pretrained CNN to extract features from H&amp;E images behind the ST spots, and a dense neural net is trained on the extracted features and ST data to predict gene expression based on H&amp;E images from held out patients as log normalized UMI counts <span class="citation">(He et al. <a href="#ref-He2020" role="doc-biblioref">2020</a>)</span>. Another method to predict gene expression from H&amp;E is PathoMCH <span class="citation">(Levy-Jurgenson et al. <a href="#ref-Levy-Jurgenson2020" role="doc-biblioref">2020</a>)</span>. TGCA transcriptomics data is normalized and the corresponding H&amp;E slides are labeled with the percentile of expression of each gene of interest. Then the whole slide images are broken into small tiles, all of which take the percentile label of the slide. Then the Inception v3 classification neural network is trained with the tiles and labels with very high or very low expression, and when predicting on held out images, it gives a score of gene expression from low to high in each tile. Such gene expression prediction methods can give pathologists a more nuanced view of the tissue beyond morphology.</p>
<p>While cell segmentation is difficult in H&amp;E images, H&amp;E images do have enough resolution to exhibit subcellular details. The H&amp;E image is used in Xfuse to increase resolution of the spatial transcriptome from ST <span class="citation">(Bergenstråhle et al. <a href="#ref-Bergenstrahle2020" role="doc-biblioref">2020</a>)</span>. The H&amp;E image and the corresponding transcriptomes are modeled to come from a shared latent space. Intensity of each channel at each pixel is modeled as Gaussian, and gene expression at each pixel is modeled as NB so the observed value at each spot are the sums of values at the pixels in the spot. Parameters of these distributions are mapped from the latent space through a generator CNN. The parameters are estimated with variational Bayesian inference. With the parameters, gene expression at each pixel can be predicted, thus increasing the resolution of ST.</p>

</div>
</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Abdelaal2020">
<p>Abdelaal, Tamim, Soufiane Mourragui, Ahmed Mahfouz, and Marcel J T Reinders. 2020. “SpaGE: Spatial Gene Enhancement using scRNA-seq.” <em>Nucleic Acids Research</em> 48 (18): e107–e107. <a href="https://doi.org/10.1093/nar/gkaa740">https://doi.org/10.1093/nar/gkaa740</a>.</p>
</div>
<div id="ref-Achim2015">
<p>Achim, Kaia, Jean Baptiste Pettit, Luis R. Saraiva, Daria Gavriouchkina, Tomas Larsson, Detlev Arendt, and John C. Marioni. 2015. “High-throughput spatial mapping of single-cell RNA-seq data to tissue of origin.” <em>Nature Biotechnology</em> 33 (5): 503–9. <a href="https://doi.org/10.1038/nbt.3209">https://doi.org/10.1038/nbt.3209</a>.</p>
</div>
<div id="ref-Alonso2020">
<p>Alonso, A M, A Carrea, and L Diambra. 2020. “Prediction of cell position using single-cell transcriptomic data: an iterative procedure [version 2; peer review: 2 approved].” <em>F1000Research</em> 8 (1775). <a href="https://doi.org/10.12688/f1000research.20715.2">https://doi.org/10.12688/f1000research.20715.2</a>.</p>
</div>
<div id="ref-An2009">
<p>An, Li, Hongbo Xie, Mark H Chin, Zoran Obradovic, Desmond J Smith, and Vasileios Megalooikonomou. 2009. “Analysis of multiplex gene expression maps obtained by voxelation.” <em>BMC Bioinformatics</em> 10 (S4): S10. <a href="https://doi.org/10.1186/1471-2105-10-S4-S10">https://doi.org/10.1186/1471-2105-10-S4-S10</a>.</p>
</div>
<div id="ref-Andersson2019">
<p>Andersson, Alma, Joseph Bergenstråhle, Michaela Asp, Ludvig Bergenstråhle, Aleksandra Jurek, José Fernández Navarro, and Joakim Lundeberg. 2019. “Spatial mapping of cell types by integration of transcriptomics data.” <em>bioRxiv</em>, December, 2019.12.13.874495. <a href="https://doi.org/10.1101/2019.12.13.874495">https://doi.org/10.1101/2019.12.13.874495</a>.</p>
</div>
<div id="ref-Arnol2019a">
<p>Arnol, Damien, Denis Schapiro, Bernd Bodenmiller, Julio Saez-Rodriguez, and Oliver Stegle. 2019. “Modeling Cell-Cell Interactions from Spatial Molecular Data with Spatial Variance Component Analysis.” <em>Cell Reports</em> 29 (1): 202–211.e6. <a href="https://doi.org/10.1016/j.celrep.2019.08.077">https://doi.org/10.1016/j.celrep.2019.08.077</a>.</p>
</div>
<div id="ref-Battich2013">
<p>Battich, Nico, Thomas Stoeger, and Lucas Pelkmans. 2013. “Image-based transcriptomics in thousands of single human cells at single-molecule resolution.” <em>Nature Methods</em> 10 (11): 1127–36. <a href="https://doi.org/10.1038/nmeth.2657">https://doi.org/10.1038/nmeth.2657</a>.</p>
</div>
<div id="ref-Bergenstrahle2020a">
<p>Bergenstråhle, Joseph, Ludvig Bergenstråhle, and Joakim Lundeberg. 2020. “SpatialCPie: an R/Bioconductor package for spatial transcriptomics cluster evaluation.” <em>BMC Bioinformatics</em> 21 (1): 161. <a href="https://doi.org/10.1186/s12859-020-3489-7">https://doi.org/10.1186/s12859-020-3489-7</a>.</p>
</div>
<div id="ref-Bergenstrahle2020">
<p>Bergenstråhle, Ludvig, Bryan He, Joseph Bergenstråhle, Alma Andersson, Joakim Lundeberg, James Zou, and Jonas Maaskola. 2020. “Super-resolved spatial transcriptomics by deep data fusion.” <em>bioRxiv</em>, March, 2020.02.28.963413. <a href="https://doi.org/10.1101/2020.02.28.963413">https://doi.org/10.1101/2020.02.28.963413</a>.</p>
</div>
<div id="ref-Berglund2018">
<p>Berglund, Emelie, Jonas Maaskola, Niklas Schultz, Stefanie Friedrich, Maja Marklund, Joseph Bergenstråhle, Firas Tarish, et al. 2018. “Spatial maps of prostate cancer transcriptomes reveal an unexplored landscape of heterogeneity.” <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-018-04724-5">https://doi.org/10.1038/s41467-018-04724-5</a>.</p>
</div>
<div id="ref-Biancalani2020">
<p>Biancalani, Tommaso, Gabriele Scalia, Lorenzo Buffoni, Raghav Avasthi, Ziqing Lu, Aman Sanger, Neriman Tokcan, et al. 2020. “Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram.” <em>bioRxiv</em>, January, 2020.08.29.272831. <a href="https://doi.org/10.1101/2020.08.29.272831">https://doi.org/10.1101/2020.08.29.272831</a>.</p>
</div>
<div id="ref-BinTayyash2020">
<p>BinTayyash, Nuha, Sokratia Georgaka, S T John, Sumon Ahmed, Alexis Boukouvalas, James Hensman, and Magnus Rattray. 2020. “Non-parametric modelling of temporal and spatial counts data from RNA-seq experiments.” <em>bioRxiv</em>, January, 2020.07.29.227207. <a href="https://doi.org/10.1101/2020.07.29.227207">https://doi.org/10.1101/2020.07.29.227207</a>.</p>
</div>
<div id="ref-Bohland2010a">
<p>Bohland, Jason W., Hemant Bokil, Sayan D. Pathak, Chang-Kyu Lee, Lydia Ng, Christopher Lau, Chihchau Kuan, Michael Hawrylycz, and Partha P. Mitra. 2010. “Clustering of spatial gene expression patterns in the mouse brain and comparison with classical neuroanatomy.” <em>Methods</em> 50 (2): 105–12. <a href="https://doi.org/10.1016/j.ymeth.2009.09.001">https://doi.org/10.1016/j.ymeth.2009.09.001</a>.</p>
</div>
<div id="ref-BravoGonzalez-Blas2020">
<p>Bravo González-Blas, Carmen, Xiao-Jiang Quan, Ramon Duran-Romaña, Ibrahim Ihsan Taskiran, Duygu Koldere, Kristofer Davie, Valerie Christiaens, et al. 2020. “Identification of genomic enhancers through spatial integration of single-cell transcriptomics and epigenomics.” <em>Molecular Systems Biology</em> 16 (5): e9438. <a href="https://doi.org/10.15252/msb.20209438">https://doi.org/10.15252/msb.20209438</a>.</p>
</div>
<div id="ref-Cabili2015">
<p>Cabili, Moran N., Margaret C. Dunagin, Patrick D. McClanahan, Andrew Biaesch, Olivia Padovan-Merhar, Aviv Regev, John L. Rinn, and Arjun Raj. 2015. “Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution.” <em>Genome Biology</em> 16 (1): 20. <a href="https://doi.org/10.1186/s13059-015-0586-4">https://doi.org/10.1186/s13059-015-0586-4</a>.</p>
</div>
<div id="ref-Cable2020">
<p>Cable, Dylan M., Evan Murray, Luli S. Zou, Aleksandrina Goeva, Evan Z. Macosko, Fei Chen, and Rafael A. Irizarry. 2020. “Robust decomposition of cell type mixtures in spatial transcriptomics.” <em>bioRxiv</em>, May, 2020.05.07.082750. <a href="https://doi.org/10.1101/2020.05.07.082750">https://doi.org/10.1101/2020.05.07.082750</a>.</p>
</div>
<div id="ref-spicyR">
<p>Canete, Nicolas, and Ellis Patrick. 2020. <em>spicyR: Spatial analysis of in situ cytometry data</em>. <a href="https://doi.org/10.18129/B9.bioc.spicyR">https://doi.org/10.18129/B9.bioc.spicyR</a>.</p>
</div>
<div id="ref-Cang2020">
<p>Cang, Zixuan, and Qing Nie. 2020. “Inferring spatial and signaling relationships between cells from single cell transcriptomic data.” <em>Nature Communications</em> 11 (1): 2084. <a href="https://doi.org/10.1038/s41467-020-15968-5">https://doi.org/10.1038/s41467-020-15968-5</a>.</p>
</div>
<div id="ref-Cao2019">
<p>Cao, Junyue, Malte Spielmann, Xiaojie Qiu, Xingfan Huang, Daniel M Ibrahim, Andrew J Hill, Fan Zhang, et al. 2019. “The single-cell transcriptional landscape of mammalian organogenesis.” <em>Nature</em> 566 (7745): 496–502. <a href="https://doi.org/10.1038/s41586-019-0969-x">https://doi.org/10.1038/s41586-019-0969-x</a>.</p>
</div>
<div id="ref-Chen2015">
<p>Chen, Kok Hao, Alistair N. Boettiger, Jeffrey R. Moffitt, Siyuan Wang, and Xiaowei Zhuang. 2015. “Spatially resolved, highly multiplexed RNA profiling in single cells.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaa6090">https://doi.org/10.1126/science.aaa6090</a>.</p>
</div>
<div id="ref-Chen2020">
<p>Chen, Wei-Ting, Ashley Lu, Katleen Craessaerts, Benjamin Pavie, Carlo Sala Frigerio, Nikky Corthout, Xiaoyan Qian, et al. 2020. “Spatial Transcriptomics and In Situ Sequencing to Study Alzheimer’s Disease.” <em>Cell</em> 0 (0). <a href="https://doi.org/10.1016/j.cell.2020.06.038">https://doi.org/10.1016/j.cell.2020.06.038</a>.</p>
</div>
<div id="ref-CiolliMattioli2019">
<p>Ciolli Mattioli, Camilla, Aviv Rom, Vedran Franke, Koshi Imami, Gerard Arrey, Mandy Terne, Andrew Woehler, Altuna Akalin, Igor Ulitsky, and Marina Chekulaeva. 2019. “Alternative 3′ UTRs direct localization of functionally diverse protein isoforms in neuronal compartments.” <em>Nucleic Acids Research</em> 47 (5): 2560–73. <a href="https://doi.org/10.1093/nar/gky1270">https://doi.org/10.1093/nar/gky1270</a>.</p>
</div>
<div id="ref-Dries2019">
<p>Dries, Ruben, Qian Zhu, Chee-Huat Linus Eng, Arpan Sarkar, Feng Bao, Rani E George, Nico Pierson, Long Cai, and Guo-Cheng Yuan. 2019. “Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data.” <em>bioRxiv</em>, May, 701680. <a href="https://doi.org/10.1101/701680">https://doi.org/10.1101/701680</a>.</p>
</div>
<div id="ref-Droin2020">
<p>Droin, Colas, Jakob El Kholtei, Keren Bahar Halpern, Clémence Hurni, Milena Rozenberg, Sapir Muvkadi, Shalev Itzkovitz, and Felix Naef. 2020. “Space-time logic of liver gene expression at sublobular scale.” <em>bioRxiv</em>, January, 2020.03.05.976571. <a href="https://doi.org/10.1101/2020.03.05.976571">https://doi.org/10.1101/2020.03.05.976571</a>.</p>
</div>
<div id="ref-Durruthy-Durruthy2016">
<p>Durruthy-Durruthy, Jens, Mark Wossidlo, Sunil Pai, Yusuke Takahashi, Gugene Kang, Larsson Omberg, Bertha Chen, Hiromitsu Nakauchi, Renee Reijo Pera, and Vittorio Sebastiano. 2016. “Spatiotemporal Reconstruction of the Human Blastocyst by Single-Cell Gene-Expression Analysis Informs Induction of Naive Pluripotency.” <em>Developmental Cell</em> 38 (1): 100–115. <a href="https://doi.org/10.1016/j.devcel.2016.06.014">https://doi.org/10.1016/j.devcel.2016.06.014</a>.</p>
</div>
<div id="ref-Durruthy-Durruthy2014">
<p>Durruthy-Durruthy, Robert, Assaf Gottlieb, Byron H. Hartman, Jörg Waldhaus, Roman D. Laske, Russ Altman, and Stefan Heller. 2014. “Reconstruction of the Mouse Otocyst and Early Neuroblast Lineage at Single-Cell Resolution.” <em>Cell</em> 157 (4): 964–78. <a href="https://doi.org/10.1016/j.cell.2014.03.036">https://doi.org/10.1016/j.cell.2014.03.036</a>.</p>
</div>
<div id="ref-Edsgard2018">
<p>Edsgärd, Daniel, Per Johnsson, and Rickard Sandberg. 2018. “Identification of spatial expression trends in single-cell gene expression data.” <em>Nature Methods</em> 15 (5): 339–42. <a href="https://doi.org/10.1038/nmeth.4634">https://doi.org/10.1038/nmeth.4634</a>.</p>
</div>
<div id="ref-Efremova2020">
<p>Efremova, Mirjana, Miquel Vento-Tormo, Sarah A Teichmann, and Roser Vento-Tormo. 2020. “CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes.” <em>Nature Protocols</em> 15 (4): 1484–1506. <a href="https://doi.org/10.1038/s41596-020-0292-x">https://doi.org/10.1038/s41596-020-0292-x</a>.</p>
</div>
<div id="ref-Eichenberger2020">
<p>Eichenberger, Bastian Th., YinXiu Zhan, Markus Rempfler, Luca Giorgetti, and Jeffrey A Chao. 2020. “deepBlink: Threshold-independent detection and localization of diffraction-limited spots.” <em>bioRxiv</em>, January, 2020.12.14.422631. <a href="https://doi.org/10.1101/2020.12.14.422631">https://doi.org/10.1101/2020.12.14.422631</a>.</p>
</div>
<div id="ref-Elosua2020">
<p>Elosua, Marc, Paula Nieto, Elisabetta Mereu, Ivo Gut, and Holger Heyn. 2020. “SPOTlight: Seeded NMF regression to Deconvolute Spatial Transcriptomics Spots with Single-Cell Transcriptomes.” <em>bioRxiv</em>, June, 2020.06.03.131334. <a href="https://doi.org/10.1101/2020.06.03.131334">https://doi.org/10.1101/2020.06.03.131334</a>.</p>
</div>
<div id="ref-Eng2019a">
<p>Eng, Chee Huat Linus, Michael Lawson, Qian Zhu, Ruben Dries, Noushin Koulena, Yodai Takei, Jina Yun, et al. 2019. “Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+.” <em>Nature</em> 568 (7751): 235–39. <a href="https://doi.org/10.1038/s41586-019-1049-y">https://doi.org/10.1038/s41586-019-1049-y</a>.</p>
</div>
<div id="ref-Farris2019">
<p>Farris, Shannon, James M Ward, Kelly E Carstens, Mahsa Samadi, Yu Wang, and Serena M Dudek. 2019. “Hippocampal Subregions Express Distinct Dendritic Transcriptomes that Reveal Differences in Mitochondrial Function in CA2.” <em>Cell Reports</em> 29 (2): 522–539.e6. <a href="https://doi.org/10.1016/j.celrep.2019.08.093">https://doi.org/10.1016/j.celrep.2019.08.093</a>.</p>
</div>
<div id="ref-Ghazanfar2020">
<p>Ghazanfar, Shila, Yingxin Lin, Xianbin Su, David Ming Lin, Ellis Patrick, Ze-Guang Han, John C Marioni, and Jean Yee Hwa Yang. 2020. “Investigating higher-order interactions in single-cell data with scHOT.” <em>Nature Methods</em> 17 (8): 799–806. <a href="https://doi.org/10.1038/s41592-020-0885-x">https://doi.org/10.1038/s41592-020-0885-x</a>.</p>
</div>
<div id="ref-Govek2019">
<p>Govek, Kiya W, Venkata S Yamajala, and Pablo G Camara. 2019. “Clustering-independent analysis of genomic data using spectral simplicial theory.” <em>PLOS Computational Biology</em> 15 (11): e1007509. <a href="https://doi.org/10.1371/journal.pcbi.1007509">https://doi.org/10.1371/journal.pcbi.1007509</a>.</p>
</div>
<div id="ref-Gyllborg2020">
<p>Gyllborg, Daniel, Christoffer Mattsson Langseth, Xiaoyan Qian, Sergio Marco Salas, Markus Hilscher, Ed Lein, and Mats Nilsson. 2020. “Hybridization-based In Situ Sequencing (HybISS): spatial transcriptomic detection in human and mouse brain tissue.” <em>bioRxiv</em>, February, 2020.02.03.931618. <a href="https://doi.org/10.1101/2020.02.03.931618">https://doi.org/10.1101/2020.02.03.931618</a>.</p>
</div>
<div id="ref-Halpern2018">
<p>Halpern, Keren Bahar, Rom Shenhav, Hassan Massalha, Beata Toth, Adi Egozi, Efi E. Massasa, Chiara Medgalia, et al. 2018. “Paired-cell sequencing enables spatial gene expression mapping of liver endothelial cells.” <em>Nature Biotechnology</em> 36 (10): 962. <a href="https://doi.org/10.1038/nbt.4231">https://doi.org/10.1038/nbt.4231</a>.</p>
</div>
<div id="ref-Halpern2017">
<p>Halpern, Keren Bahar, Rom Shenhav, Orit Matcovitch-Natan, Beáta Tóth, Doron Lemze, Matan Golan, Efi E. Massasa, et al. 2017. “Single-cell spatial reconstruction reveals global division of labour in the mammalian liver.” <em>Nature</em> 542 (7641): 1–5. <a href="https://doi.org/10.1038/nature21065">https://doi.org/10.1038/nature21065</a>.</p>
</div>
<div id="ref-Hao2021">
<p>Hao, Minsheng, Kui Hua, and Xuegong Zhang. 2021. “SOMDE: A scalable method for identifying spatially variable genes with self-organizing map.” <em>bioRxiv</em>, January, 2020.12.10.419549. <a href="https://doi.org/10.1101/2020.12.10.419549">https://doi.org/10.1101/2020.12.10.419549</a>.</p>
</div>
<div id="ref-He2020">
<p>He, Bryan, Ludvig Bergenstråhle, Linnea Stenbeck, Abubakar Abid, Alma Andersson, Åke Borg, Jonas Maaskola, Joakim Lundeberg, and James Zou. 2020. “Integrating spatial gene expression and breast tumour morphology via deep learning.” <em>Nature Biomedical Engineering</em>, June, 1–8. <a href="https://doi.org/10.1038/s41551-020-0578-x">https://doi.org/10.1038/s41551-020-0578-x</a>.</p>
</div>
<div id="ref-Jerby-Arnon2020">
<p>Jerby-Arnon, Livnat, and Aviv Regev. 2020. “Mapping multicellular programs from single-cell profiles.” <em>bioRxiv</em>, January, 2020.08.11.245472. <a href="https://doi.org/10.1101/2020.08.11.245472">https://doi.org/10.1101/2020.08.11.245472</a>.</p>
</div>
<div id="ref-Ji2020">
<p>Ji, Andrew L, Adam J Rubin, Kim Thrane, Sizun Jiang, David L Reynolds, Robin M Meyers, Margaret G Guo, et al. 2020. “Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma.” <em>Cell</em> 182 (2): 497–514.e22. <a href="https://doi.org/https://doi.org/10.1016/j.cell.2020.05.039">https://doi.org/https://doi.org/10.1016/j.cell.2020.05.039</a>.</p>
</div>
<div id="ref-Karaiskos2017">
<p>Karaiskos, Nikos, Philipp Wahle, Jonathan Alles, Anastasiya Boltengagen, Salah Ayoub, Claudia Kipar, Christine Kocks, Nikolaus Rajewsky, and Robert P. Zinzen. 2017. “The Drosophila embryo at single-cell transcriptome resolution.” <em>Science</em> 358 (6360): 194–99. <a href="https://doi.org/10.1126/science.aan3235">https://doi.org/10.1126/science.aan3235</a>.</p>
</div>
<div id="ref-Ko2013">
<p>Ko, Younhee, Seth A. Ament, James A. Eddy, Juan Caballero, John C. Earls, Leroy Hood, and Nathan D. Price. 2013. “Cell type-specific genes show striking and distinct patterns of spatial expression in the mouse brain.” <em>Proceedings of the National Academy of Sciences</em> 110 (8): 3095–3100. <a href="https://doi.org/10.1073/pnas.1222897110">https://doi.org/10.1073/pnas.1222897110</a>.</p>
</div>
<div id="ref-Korsunsky2019">
<p>Korsunsky, Ilya, Nghia Millard, Jean Fan, Kamil Slowikowski, Fan Zhang, Kevin Wei, Yuriy Baglaenko, Michael Brenner, Po-ru Loh, and Soumya Raychaudhuri. 2019. “Fast, sensitive and accurate integration of single-cell data with Harmony.” <em>Nature Methods</em> 16 (12): 1289–96. <a href="https://doi.org/10.1038/s41592-019-0619-0">https://doi.org/10.1038/s41592-019-0619-0</a>.</p>
</div>
<div id="ref-Koster2019">
<p>Köster, Johannes, Myles Brown, and X. Shirley Liu. 2019. “A Bayesian model for single cell transcript expression analysis on MERFISH data.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/bty718">https://doi.org/10.1093/bioinformatics/bty718</a>.</p>
</div>
<div id="ref-Kueckelhaus2020">
<p>Kueckelhaus, Jan, Jasmin von Ehr, Vidhya M Ravi, Paulina Will, Kevin Joseph, Juergen Beck, Ulrich G Hofmann, Daniel Delev, Oliver Schnell, and Dieter Henrik Heiland. 2020. “Inferring spatially transient gene expression pattern from spatial transcriptomic studies.” <em>bioRxiv</em>, January, 2020.10.20.346544. <a href="https://doi.org/10.1101/2020.10.20.346544">https://doi.org/10.1101/2020.10.20.346544</a>.</p>
</div>
<div id="ref-LaManno2018">
<p>La Manno, Gioele, Ruslan Soldatov, Amit Zeisel, Emelie Braun, Hannah Hochgerner, Viktor Petukhov, Katja Lidschreiber, et al. 2018. “RNA velocity of single cells.” <em>Nature</em> 560 (7719): 494–98. <a href="https://doi.org/10.1038/s41586-018-0414-6">https://doi.org/10.1038/s41586-018-0414-6</a>.</p>
</div>
<div id="ref-Levy-Jurgenson2020">
<p>Levy-Jurgenson, Alona, Xavier Tekpli, Vessela N Kristensen, and Zohar Yakhini. 2020. “Spatial transcriptomics inferred from pathology whole-slide images links tumor heterogeneity to survival in breast and lung cancer.” <em>Scientific Reports</em> 10 (1): 18802. <a href="https://doi.org/10.1038/s41598-020-75708-z">https://doi.org/10.1038/s41598-020-75708-z</a>.</p>
</div>
<div id="ref-Li2016">
<p>Li, Junxiang, Haofei Luo, Rui Wang, Jidong Lang, Siyu Zhu, Zhenming Zhang, Jianhuo Fang, et al. 2016. “Systematic Reconstruction of Molecular Cascades Regulating GP Development Using Single-Cell RNA-Seq.” <em>Cell Reports</em> 15 (7): 1467–80. <a href="https://doi.org/10.1016/j.celrep.2016.04.043">https://doi.org/10.1016/j.celrep.2016.04.043</a>.</p>
</div>
<div id="ref-Li2020">
<p>Li, Zhuliu, Tianci Song, Jeongsik Yong, and Rui Kuang. 2020. “Imputation of Spatially-resolved Transcriptomes by Graph-regularized Tensor Completion.” <em>bioRxiv</em>, January, 2020.08.05.237560. <a href="https://doi.org/10.1101/2020.08.05.237560">https://doi.org/10.1101/2020.08.05.237560</a>.</p>
</div>
<div id="ref-Littman2020">
<p>Littman, Russell, Zachary Hemminger, Robert Foreman, Douglas Arneson, Guanglin Zhang, Fernando Gómez-Pinilla, Xia Yang, and Roy Wollman. 2020. “JSTA: joint cell segmentation and cell type annotation for spatial transcriptomics.” <em>bioRxiv</em>, January, 2020.09.18.304147. <a href="https://doi.org/10.1101/2020.09.18.304147">https://doi.org/10.1101/2020.09.18.304147</a>.</p>
</div>
<div id="ref-Lohoff2020">
<p>Lohoff, T, S Ghazanfar, A Missarova, N Koulena, N Pierson, J A Griffiths, E S Bardot, et al. 2020. “Highly multiplexed spatially resolved gene expression profiling of mouse organogenesis.” <em>bioRxiv</em>, January, 2020.11.20.391896. <a href="https://doi.org/10.1101/2020.11.20.391896">https://doi.org/10.1101/2020.11.20.391896</a>.</p>
</div>
<div id="ref-Lopez2019">
<p>Lopez, Romain, Achille Nazaret, Maxime Langevin, Jules Samaran, Jeffrey Regier, Michael I. Jordan, and Nir Yosef. 2019. “A joint model of unpaired data from scRNA-seq and spatial transcriptomics for imputing missing gene expression measurements,” May. <a href="http://arxiv.org/abs/1905.02269">http://arxiv.org/abs/1905.02269</a>.</p>
</div>
<div id="ref-Maaskola2018">
<p>Maaskola, Jonas, Ludvig Bergenstråhle, Aleksandra Jurek, José Fernández Navarro, Jens Lagergren, and Joakim Lundeberg. 2018. “Charting Tissue Expression Anatomy by Spatial Transcriptome Decomposition.” <em>bioRxiv</em>, December, 362624. <a href="https://doi.org/10.1101/362624">https://doi.org/10.1101/362624</a>.</p>
</div>
<div id="ref-Mantri2020">
<p>Mantri, Madhav, Gaetano J Scuderi, Roozbeh Abedini Nassab, Michael F Z Wang, David McKellar, Jonathan T Butcher, and Iwijn De Vlaminck. 2020. “Spatiotemporal single-cell RNA sequencing of developing hearts reveals interplay between cellular differentiation and morphogenesis.” <em>bioRxiv</em>, January, 2020.05.03.065102. <a href="https://doi.org/10.1101/2020.05.03.065102">https://doi.org/10.1101/2020.05.03.065102</a>.</p>
</div>
<div id="ref-Maynard2020">
<p>Maynard, Kristen R, Madhavi Tippani, Yoichiro Takahashi, BaDoi N Phan, Thomas M Hyde, Andrew E Jaffe, and Keri Martinowich. 2020. “dotdotdot: an automated approach to quantify multiplex single molecule fluorescent in situ hybridization (smFISH) images in complex tissues.” <em>Nucleic Acids Research</em> 48 (11): e66–e66. <a href="https://doi.org/10.1093/nar/gkaa312">https://doi.org/10.1093/nar/gkaa312</a>.</p>
</div>
<div id="ref-Middleton2019">
<p>Middleton, Sarah A, James Eberwine, and Junhyong Kim. 2019. “Comprehensive catalog of dendritically localized mRNA isoforms from sub-cellular sequencing of single mouse neurons.” <em>BMC Biology</em> 17 (1): 5. <a href="https://doi.org/10.1186/s12915-019-0630-z">https://doi.org/10.1186/s12915-019-0630-z</a>.</p>
</div>
<div id="ref-Moffitt2018">
<p>Moffitt, Jeffrey R., Dhananjay Bambah-Mukku, Stephen W. Eichhorn, Eric Vaughn, Karthik Shekhar, Julio D. Perez, Nimrod D. Rubinstein, et al. 2018. “Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aau5324">https://doi.org/10.1126/science.aau5324</a>.</p>
</div>
<div id="ref-Moffitt2016a">
<p>Moffitt, Jeffrey R., Junjie Hao, Guiping Wang, Kok Hao Chen, Hazen P. Babcock, and Xiaowei Zhuang. 2016. “High-throughput single-cell gene-expression profiling with multiplexed error-robust fluorescence in situ hybridization.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1612826113">https://doi.org/10.1073/pnas.1612826113</a>.</p>
</div>
<div id="ref-Moncada2020">
<p>Moncada, Reuben, Dalia Barkley, Florian Wagner, Marta Chiodin, Joseph C Devlin, Maayan Baron, Cristina H Hajdu, Diane M Simeone, and Itai Yanai. 2020. “Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas.” <em>Nature Biotechnology</em> 38 (3): 333–42. <a href="https://doi.org/10.1038/s41587-019-0392-8">https://doi.org/10.1038/s41587-019-0392-8</a>.</p>
</div>
<div id="ref-Mori2019">
<p>Mori, Tomoya, Haruka Takaoka, Junko Yamane, Cantas Alev, and Wataru Fujibuchi. 2019. “Novel computational model of gastrula morphogenesis to identify spatial discriminator genes by self-organizing map (SOM) clustering.” <em>Scientific Reports</em> 9 (1): 1–10. <a href="https://doi.org/10.1038/s41598-019-49031-1">https://doi.org/10.1038/s41598-019-49031-1</a>.</p>
</div>
<div id="ref-Mori2017">
<p>Mori, Tomoya, Junko Yamane, Kenta Kobayashi, Nobuko Taniyama, Takanori Tano, and Wataru Fujibuchi. 2017. “Development of 3D Tissue Reconstruction Method from Single-cell RNA-seq Data.” <em>Genomics and Computational Biology; Vol 3 No 1 (2017)</em>. <a href="https://doi.org/10.18547/gcb.2017.vol3.iss1.e53">https://doi.org/10.18547/gcb.2017.vol3.iss1.e53</a>.</p>
</div>
<div id="ref-Nitzan2019">
<p>Nitzan, Mor, Nikos Karaiskos, Nir Friedman, and Nikolaus Rajewsky. 2019. “Gene expression cartography.” <em>Nature</em> 576 (7785): 132–37. <a href="https://doi.org/10.1038/s41586-019-1773-3">https://doi.org/10.1038/s41586-019-1773-3</a>.</p>
</div>
<div id="ref-Ortiz2020">
<p>Ortiz, Cantin, Jose Fernandez Navarro, Aleksandra Jurek, Antje Märtin, Joakim Lundeberg, and Konstantinos Meletis. 2020. “Molecular atlas of the adult mouse brain.” <em>Science Advances</em> 6 (26): eabb3446. <a href="https://doi.org/10.1126/sciadv.abb3446">https://doi.org/10.1126/sciadv.abb3446</a>.</p>
</div>
<div id="ref-Palla2021">
<p>Palla, Giovanni, Hannah Spitzer, Michal Klein, David Fischer, Anna Christina Schaar, Louis Benedikt Kuemmerle, Sergei Rybakov, et al. 2021. “Squidpy: a scalable framework for spatial single cell analysis.” <em>bioRxiv</em>, January, 2021.02.19.431994. <a href="https://doi.org/10.1101/2021.02.19.431994">https://doi.org/10.1101/2021.02.19.431994</a>.</p>
</div>
<div id="ref-Park2019">
<p>Park, Jeongbin, Wonyl Choi, Sebastian Tiesmeyer, Brian Long, Lars E. Borm, Emma Garren, Thuc Nghi Nguyen, et al. 2019. “Segmentation-free inference of cell types from in situ transcriptomics data.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/800748">https://doi.org/10.1101/800748</a>.</p>
</div>
<div id="ref-Partel2019">
<p>Partel, Gabriele, Markus Hilscher, Giorgia Milli, Leslie Solorzano, Anna Klemm, Mats Nilsson, and Carolina Wählby. 2019. “Identification of spatial compartments in tissue from in situ sequencing data.” <em>bioRxiv</em>, September, 765842. <a href="https://doi.org/10.1101/765842">https://doi.org/10.1101/765842</a>.</p>
</div>
<div id="ref-Partel2020">
<p>Partel, Gabriele, and Carolina Wählby. 2020. “Spage2vec: Unsupervised detection of spatial gene expression constellations.” <em>bioRxiv</em>, February, 2020.02.12.945345. <a href="https://doi.org/10.1101/2020.02.12.945345">https://doi.org/10.1101/2020.02.12.945345</a>.</p>
</div>
<div id="ref-Peng2016">
<p>Peng, Guangdun, Shengbao Suo, Jun Chen, Weiyang Chen, Chang Liu, Fang Yu, Ran Wang, et al. 2016. “Spatial Transcriptome for the Molecular Annotation of Lineage Fates and Cell Identity in Mid-gastrula Mouse Embryo.” <em>Developmental Cell</em> 36 (6): 681–97. <a href="https://doi.org/10.1016/j.devcel.2016.02.020">https://doi.org/10.1016/j.devcel.2016.02.020</a>.</p>
</div>
<div id="ref-Pettit2014">
<p>Pettit, Jean-Baptiste, Raju Tomer, Kaia Achim, Sylvia Richardson, Lamiae Azizi, and John Marioni. 2014. “Identifying Cell Types from Spatially Referenced Single-Cell Expression Datasets.” Edited by Quaid Morris. <em>PLoS Computational Biology</em> 10 (9): e1003824. <a href="https://doi.org/10.1371/journal.pcbi.1003824">https://doi.org/10.1371/journal.pcbi.1003824</a>.</p>
</div>
<div id="ref-Petukhov2020">
<p>Petukhov, Viktor, Ruslan A Soldatov, Konstantin Khodosevich, and Peter V Kharchenko. 2020. “Bayesian segmentation of spatially resolved transcriptomics data.” <em>bioRxiv</em>, January, 2020.10.05.326777. <a href="https://doi.org/10.1101/2020.10.05.326777">https://doi.org/10.1101/2020.10.05.326777</a>.</p>
</div>
<div id="ref-Pham2020a">
<p>Pham, Duy, Xiao Tan, Jun Xu, Laura F Grice, Pui Yeng Lam, Arti Raghubar, Jana Vukovic, Marc J Ruitenberg, and Quan Nguyen. 2020. “stLearn: integrating spatial location, tissue morphology and gene expression to find cell types, cell-cell interactions and spatial trajectories within undissociated tissues.” <em>bioRxiv</em>, January, 2020.05.31.125658. <a href="https://doi.org/10.1101/2020.05.31.125658">https://doi.org/10.1101/2020.05.31.125658</a>.</p>
</div>
<div id="ref-Pruteanu-Malinici2011">
<p>Pruteanu-Malinici, Iulian, Daniel L. Mace, and Uwe Ohler. 2011. “Automatic Annotation of Spatial Expression Patterns via Sparse Bayesian Factor Models.” Edited by Joel S. Bader. <em>PLoS Computational Biology</em> 7 (7): e1002098. <a href="https://doi.org/10.1371/journal.pcbi.1002098">https://doi.org/10.1371/journal.pcbi.1002098</a>.</p>
</div>
<div id="ref-Qian2020">
<p>Qian, Xiaoyan, Kenneth D. Harris, Thomas Hauling, Dimitris Nicoloutsopoulos, Ana B. Muñoz-Manchado, Nathan Skene, Jens Hjerling-Leffler, and Mats Nilsson. 2020. “Probabilistic cell typing enables fine mapping of closely related cell types in situ.” <em>Nature Methods</em> 17 (1): 101–6. <a href="https://doi.org/10.1038/s41592-019-0631-4">https://doi.org/10.1038/s41592-019-0631-4</a>.</p>
</div>
<div id="ref-Ren2020">
<p>Ren, Xianwen, Guojie Zhong, Qiming Zhang, Lei Zhang, Yujie Sun, and Zemin Zhang. 2020. “Reconstruction of cell spatial organization based on ligand-receptor mediated self-assembly.” <em>bioRxiv</em>, January, 2020.02.13.948521. <a href="https://doi.org/10.1101/2020.02.13.948521">https://doi.org/10.1101/2020.02.13.948521</a>.</p>
</div>
<div id="ref-Righelli2021">
<p>Righelli, Dario, Lukas M Weber, Helena L Crowell, Brenda Pardo, Leonardo Collado-Torres, Shila Ghazanfar, Aaron T L Lun, Stephanie C Hicks, and Davide Risso. 2021. “SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor.” <em>bioRxiv</em>, January, 2021.01.27.428431. <a href="https://doi.org/10.1101/2021.01.27.428431">https://doi.org/10.1101/2021.01.27.428431</a>.</p>
</div>
<div id="ref-Rizvi2017">
<p>Rizvi, Abbas H, Pablo G Camara, Elena K Kandror, Thomas J Roberts, Ira Schieren, Tom Maniatis, and Raul Rabadan. 2017. “Single-cell topological RNA-seq analysis reveals insights into cellular differentiation and development.” <em>Nature Biotechnology</em> 35 (6): 551–60. <a href="https://doi.org/10.1038/nbt.3854">https://doi.org/10.1038/nbt.3854</a>.</p>
</div>
<div id="ref-Rodriques2019">
<p>Rodriques, Samuel G., Robert R. Stickels, Aleksandrina Goeva, Carly A. Martin, Evan Murray, Charles R. Vanderburg, Joshua Welch, Linlin M. Chen, Fei Chen, and Evan Z. Macosko. 2019. “Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaw1219">https://doi.org/10.1126/science.aaw1219</a>.</p>
</div>
<div id="ref-Ronneberger2015">
<p>Ronneberger, Olaf, Philipp Fischer, and Thomas Brox. 2015. “U-Net: Convolutional Networks for Biomedical Image Segmentation BT - Medical Image Computing and Computer-Assisted Intervention – MICCAI 2015.” In, edited by Nassir Navab, Joachim Hornegger, William M Wells, and Alejandro F Frangi, 234–41. Cham: Springer International Publishing.</p>
</div>
<div id="ref-Samacoits2018a">
<p>Samacoits, Aubin, Racha Chouaib, Adham Safieddine, Abdel-Meneem Traboulsi, Wei Ouyang, Christophe Zimmer, Marion Peter, Edouard Bertrand, Thomas Walter, and Florian Mueller. 2018. “A computational framework to study sub-cellular RNA localization.” <em>Nature Communications</em> 9 (1): 4584. <a href="https://doi.org/10.1038/s41467-018-06868-w">https://doi.org/10.1038/s41467-018-06868-w</a>.</p>
</div>
<div id="ref-Satija2015">
<p>Satija, Rahul, Jeffrey A. Farrell, David Gennert, Alexander F. Schier, and Aviv Regev. 2015. “Spatial reconstruction of single-cell gene expression data.” <em>Nature Biotechnology</em> 33 (5): 495–502. <a href="https://doi.org/10.1038/nbt.3192">https://doi.org/10.1038/nbt.3192</a>.</p>
</div>
<div id="ref-Shah2016">
<p>Shah, Sheel, Eric Lubeck, Wen Zhou, and Long Cai. 2016. “In Situ Transcription Profiling of Single Cells Reveals Spatial Organization of Cells in the Mouse Hippocampus.” <em>Neuron</em>. <a href="https://doi.org/10.1016/j.neuron.2016.10.001">https://doi.org/10.1016/j.neuron.2016.10.001</a>.</p>
</div>
<div id="ref-Shah2018">
<p>Shah, Sheel, Yodai Takei, Wen Zhou, Eric Lubeck, Jina Yun, Chee Huat Linus Eng, Noushin Koulena, et al. 2018. “Dynamics and Spatial Genomics of the Nascent Transcriptome by Intron seqFISH.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2018.05.035">https://doi.org/10.1016/j.cell.2018.05.035</a>.</p>
</div>
<div id="ref-Sountoulidis2020">
<p>Sountoulidis, Alexandros, Andreas Liontos, Hong Phuong Nguyen, Alexandra B Firsova, Athanasios Fysikopoulos, Xiaoyan Qian, Werner Seeger, Erik Sundström, Mats Nilsson, and Christos Samakovlis. 2020. “SCRINSHOT, a spatial method for single-cell resolution mapping of cell states in tissue sections.” <em>bioRxiv</em>, February, 2020.02.07.938571. <a href="https://doi.org/10.1101/2020.02.07.938571">https://doi.org/10.1101/2020.02.07.938571</a>.</p>
</div>
<div id="ref-Stuart2019a">
<p>Stuart, Tim, Andrew Butler, Paul Hoffman, Christoph Hafemeister, Efthymia Papalexi, William M. Mauck, Yuhan Hao, Marlon Stoeckius, Peter Smibert, and Rahul Satija. 2019. “Comprehensive Integration of Single-Cell Data.” <em>Cell</em> 177 (7): 1888–1902.e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>.</p>
</div>
<div id="ref-Su2020">
<p>Su, Jing, and Qianqian Song. 2020. “DSTG: Deconvoluting Spatial Transcriptomics Data through Graph-based Artificial Intelligence.” <em>bioRxiv</em>, January, 2020.10.20.347195. <a href="https://doi.org/10.1101/2020.10.20.347195">https://doi.org/10.1101/2020.10.20.347195</a>.</p>
</div>
<div id="ref-Sun2020">
<p>Sun, Shiquan, Jiaqiang Zhu, and Xiang Zhou. 2020. “Statistical analysis of spatial expression patterns for spatially resolved transcriptomic studies.” <em>Nature Methods</em> 17 (2): 193–200. <a href="https://doi.org/10.1038/s41592-019-0701-7">https://doi.org/10.1038/s41592-019-0701-7</a>.</p>
</div>
<div id="ref-Svensson2020">
<p>Svensson, Valentine, Eduardo da Veiga Beltrame, and Lior Pachter. 2020. “A curated database reveals trends in single-cell transcriptomics.” <em>Database</em> 2020 (January). <a href="https://doi.org/10.1093/database/baaa073">https://doi.org/10.1093/database/baaa073</a>.</p>
</div>
<div id="ref-Svensson2018">
<p>Svensson, Valentine, Sarah A. Teichmann, and Oliver Stegle. 2018. “SpatialDE: Identification of spatially variable genes.” <em>Nature Methods</em> 15 (5): 343–46. <a href="https://doi.org/10.1038/nmeth.4636">https://doi.org/10.1038/nmeth.4636</a>.</p>
</div>
<div id="ref-Tanevski2020">
<p>Tanevski, Jovan, Thin Nguyen, Buu Truong, Nikos Karaiskos, Mehmet Eren Ahsen, Xinyu Zhang, Chang Shu, et al. 2020. “Gene selection for optimal prediction of cell position in tissues from single-cell transcriptomics data.” <em>Life Science Alliance</em> 3 (11): e202000867. <a href="https://doi.org/10.26508/lsa.202000867">https://doi.org/10.26508/lsa.202000867</a>.</p>
</div>
<div id="ref-Vandenbon2020">
<p>Vandenbon, Alexis, and Diego Diez. 2020. “A clustering-independent method for finding differentially expressed genes in single-cell transcriptome data.” <em>Nature Communications</em> 11 (1): 4318. <a href="https://doi.org/10.1038/s41467-020-17900-3">https://doi.org/10.1038/s41467-020-17900-3</a>.</p>
</div>
<div id="ref-VanValen2016">
<p>Van Valen, David A, Takamasa Kudo, Keara M Lane, Derek N Macklin, Nicolas T Quach, Mialy M DeFelice, Inbal Maayan, Yu Tanouchi, Euan A Ashley, and Markus W Covert. 2016. “Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments.” <em>PLOS Computational Biology</em> 12 (11): e1005177. <a href="https://doi.org/10.1371/journal.pcbi.1005177">https://doi.org/10.1371/journal.pcbi.1005177</a>.</p>
</div>
<div id="ref-Verma2020">
<p>Verma, Archit, and Barbara Engelhardt. 2020. “A Bayesian nonparametric semi-supervised model for integration of multiple single-cell experiments.” <em>bioRxiv</em>, January, 2020.01.14.906313. <a href="https://doi.org/10.1101/2020.01.14.906313">https://doi.org/10.1101/2020.01.14.906313</a>.</p>
</div>
<div id="ref-Waldhaus2015">
<p>Waldhaus, Jörg, Robert Durruthy-Durruthy, and Stefan Heller. 2015. “Quantitative High-Resolution Cellular Map of the Organ of Corti.” <em>Cell Reports</em> 11 (9): 1385–99. <a href="https://doi.org/10.1016/j.celrep.2015.04.062">https://doi.org/10.1016/j.celrep.2015.04.062</a>.</p>
</div>
<div id="ref-Wang2018">
<p>Wang, Guiping, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2018. “Multiplexed imaging of high-density libraries of RNAs with MERFISH and expansion microscopy.” <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/s41598-018-22297-7">https://doi.org/10.1038/s41598-018-22297-7</a>.</p>
</div>
<div id="ref-Welch2019">
<p>Welch, Joshua D., Velina Kozareva, Ashley Ferreira, Charles Vanderburg, Carly Martin, and Evan Z. Macosko. 2019. “Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity.” <em>Cell</em> 177 (7): 1873–1887.e17. <a href="https://doi.org/10.1016/j.cell.2019.05.006">https://doi.org/10.1016/j.cell.2019.05.006</a>.</p>
</div>
<div id="ref-Xia2019c">
<p>Xia, Chenglong, Hazen P. Babcock, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2019. “Multiplexed detection of RNA using MERFISH and branched DNA amplification.” <em>Scientific Reports</em> 9 (1): 1–13. <a href="https://doi.org/10.1038/s41598-019-43943-8">https://doi.org/10.1038/s41598-019-43943-8</a>.</p>
</div>
<div id="ref-Yang2020">
<p>Yang, Xiaoyu, Seth Bergenholtz, Lenka Maliskova, Mark-Phillip Pebworth, Arnold R Kriegstein, Yun Li, and Yin Shen. 2020. “SMART-Q: An Integrative Pipeline Quantifying Cell Type-Specific RNA Transcription.” <em>PLOS ONE</em> 15 (4): e0228760. <a href="https://doi.org/10.1371/journal.pone.0228760">https://doi.org/10.1371/journal.pone.0228760</a>.</p>
</div>
<div id="ref-Yuan2019">
<p>Yuan, Ye, and Ziv Bar-Joseph. 2019. “GCNG: Graph convolutional networks for inferring cell-cell interactions.” <em>bioRxiv</em>, January, 2019.12.23.887133. <a href="https://doi.org/10.1101/2019.12.23.887133">https://doi.org/10.1101/2019.12.23.887133</a>.</p>
</div>
<div id="ref-Zhang2018">
<p>Zhang, Ke, Wanwan Feng, and Peng Wang. 2018. “Identification of spatially variable genes with graph cuts.” <em>bioRxiv</em>, January, 491472. <a href="https://doi.org/10.1101/491472">https://doi.org/10.1101/491472</a>.</p>
</div>
<div id="ref-Zhang2020">
<p>Zhang, Meng, Stephen W Eichhorn, Brian Zingg, Zizhen Yao, Hongkui Zeng, Hongwei Dong, and Xiaowei Zhuang. 2020. “Molecular, spatial and projection diversity of neurons in primary motor cortex revealed by in situ single-cell transcriptomics.” <em>bioRxiv</em>, January, 2020.06.04.105700. <a href="https://doi.org/10.1101/2020.06.04.105700">https://doi.org/10.1101/2020.06.04.105700</a>.</p>
</div>
<div id="ref-Zhang2013">
<p>Zhang, Wenlu, Daming Feng, Rongjian Li, Andrey Chernikov, Nikos Chrisochoides, Christopher Osgood, Charlotte Konikoff, Stuart Newfeld, Sudhir Kumar, and Shuiwang Ji. 2013. “A mesh generation and machine learning framework for Drosophila gene expression pattern image analysis.” <em>BMC Bioinformatics</em> 14 (1): 372. <a href="https://doi.org/10.1186/1471-2105-14-372">https://doi.org/10.1186/1471-2105-14-372</a>.</p>
</div>
<div id="ref-Zhao2020">
<p>Zhao, Edward, Matthew R Stone, Xing Ren, Thomas Pulliam, Paul Nghiem, Jason H Bielas, and Raphael Gottardo. 2020. “BayesSpace enables the robust characterization of spatial gene expression architecture in tissue sections at increased resolution.” <em>bioRxiv</em>, January, 2020.09.04.283812. <a href="https://doi.org/10.1101/2020.09.04.283812">https://doi.org/10.1101/2020.09.04.283812</a>.</p>
</div>
<div id="ref-Zhu2020">
<p>Zhu, Junjie, and Chiara Sabatti. 2020. “Integrative Spatial Single-cell Analysis with Graph-based Feature Learning.” <em>bioRxiv</em>, January, 2020.08.12.248971. <a href="https://doi.org/10.1101/2020.08.12.248971">https://doi.org/10.1101/2020.08.12.248971</a>.</p>
</div>
<div id="ref-Zhu2018">
<p>Zhu, Qian, Sheel Shah, Ruben Dries, Long Cai, and Guo Cheng Yuan. 2018. “Identification of spatially associated subpopulations by combining scRNAseq and sequential fluorescence in situ hybridization data.” <em>Nature Biotechnology</em> 36 (12): 1183–90. <a href="https://doi.org/10.1038/nbt.4260">https://doi.org/10.1038/nbt.4260</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="text-mining.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="future.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["museumst.pdf"],
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
