<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Current era technologies | Museum of Spatial Transcriptomics</title>
  <meta name="description" content="Chapter 5 Current era technologies | Museum of Spatial Transcriptomics" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Current era technologies | Museum of Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Chapter 5 Current era technologies | Museum of Spatial Transcriptomics" />
  <meta name="github-repo" content="pachterlab/LP_2021" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Current era technologies | Museum of Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="Chapter 5 Current era technologies | Museum of Spatial Transcriptomics" />
  

<meta name="author" content="Lambda Moses, dlu2@caltech.edu" />
<meta name="author" content="Lior Pachter, lpachter@caltech.edu" />


<meta name="date" content="2021-06-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="current.html"/>
<link rel="next" href="text-mining.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Museum of Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="part"><span><b>I Prequel era</b></span></li>
<li class="chapter" data-level="2" data-path="prequel.html"><a href="prequel.html"><i class="fa fa-check"></i><b>2</b> Prequel era</a><ul>
<li class="chapter" data-level="2.1" data-path="prequel.html"><a href="prequel.html#traps"><i class="fa fa-check"></i><b>2.1</b> Enhancer and gene traps</a></li>
<li class="chapter" data-level="2.2" data-path="prequel.html"><a href="prequel.html#in-situ-reporter"><i class="fa fa-check"></i><b>2.2</b> In situ reporter</a></li>
<li class="chapter" data-level="2.3" data-path="prequel.html"><a href="prequel.html#wmish"><i class="fa fa-check"></i><b>2.3</b> ISH and WMISH atlases</a></li>
<li class="chapter" data-level="2.4" data-path="prequel.html"><a href="prequel.html#database_prequel"><i class="fa fa-check"></i><b>2.4</b> Databases of the prequel era</a></li>
<li class="chapter" data-level="2.5" data-path="prequel.html"><a href="prequel.html#geo_prequel"><i class="fa fa-check"></i><b>2.5</b> Geography of the prequel era</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prequel-analysis.html"><a href="prequel-analysis.html"><i class="fa fa-check"></i><b>3</b> Data analysis in the prequel era</a><ul>
<li class="chapter" data-level="3.1" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-patterns"><i class="fa fa-check"></i><b>3.1</b> Gene patterns</a></li>
<li class="chapter" data-level="3.2" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-spatial-regions"><i class="fa fa-check"></i><b>3.2</b> Spatial regions</a></li>
<li class="chapter" data-level="3.3" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-interactions"><i class="fa fa-check"></i><b>3.3</b> Gene interactions</a></li>
<li class="chapter" data-level="3.4" data-path="prequel-analysis.html"><a href="prequel-analysis.html#decline"><i class="fa fa-check"></i><b>3.4</b> Decline</a></li>
<li class="chapter" data-level="3.5" data-path="prequel-analysis.html"><a href="prequel-analysis.html#geo-prequel-analysis"><i class="fa fa-check"></i><b>3.5</b> Geography of prequel data analysis</a></li>
</ul></li>
<li class="part"><span><b>II Current era</b></span></li>
<li class="chapter" data-level="4" data-path="current.html"><a href="current.html"><i class="fa fa-check"></i><b>4</b> From the past to the present</a></li>
<li class="chapter" data-level="5" data-path="current-techs.html"><a href="current-techs.html"><i class="fa fa-check"></i><b>5</b> Current era technologies</a><ul>
<li class="chapter" data-level="5.1" data-path="current-techs.html"><a href="current-techs.html#microdissection"><i class="fa fa-check"></i><b>5.1</b> Microdissection</a><ul>
<li class="chapter" data-level="5.1.1" data-path="current-techs.html"><a href="current-techs.html#history-of-lcm"><i class="fa fa-check"></i><b>5.1.1</b> History of LCM</a></li>
<li class="chapter" data-level="5.1.2" data-path="current-techs.html"><a href="current-techs.html#lcm-usage"><i class="fa fa-check"></i><b>5.1.2</b> Usage of LCM</a></li>
<li class="chapter" data-level="5.1.3" data-path="current-techs.html"><a href="current-techs.html#physical-microdissection"><i class="fa fa-check"></i><b>5.1.3</b> Physical microdissection</a></li>
<li class="chapter" data-level="5.1.4" data-path="current-techs.html"><a href="current-techs.html#de-facto-microdissection"><i class="fa fa-check"></i><b>5.1.4</b> <em>De facto</em> microdissection</a></li>
<li class="chapter" data-level="5.1.5" data-path="current-techs.html"><a href="current-techs.html#summary"><i class="fa fa-check"></i><b>5.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="current-techs.html"><a href="current-techs.html#smfish"><i class="fa fa-check"></i><b>5.2</b> Single molecular FISH</a><ul>
<li class="chapter" data-level="5.2.1" data-path="current-techs.html"><a href="current-techs.html#barcoding"><i class="fa fa-check"></i><b>5.2.1</b> Barcoding strategies</a></li>
<li class="chapter" data-level="5.2.2" data-path="current-techs.html"><a href="current-techs.html#amplification"><i class="fa fa-check"></i><b>5.2.2</b> Signal amplification</a></li>
<li class="chapter" data-level="5.2.3" data-path="current-techs.html"><a href="current-techs.html#crowding"><i class="fa fa-check"></i><b>5.2.3</b> Optical crowding</a></li>
<li class="chapter" data-level="5.2.4" data-path="current-techs.html"><a href="current-techs.html#smfish-usage"><i class="fa fa-check"></i><b>5.2.4</b> Usage of smFISH based techniques</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="current-techs.html"><a href="current-techs.html#iss"><i class="fa fa-check"></i><b>5.3</b> In situ sequencing</a><ul>
<li class="chapter" data-level="5.3.1" data-path="current-techs.html"><a href="current-techs.html#solid"><i class="fa fa-check"></i><b>5.3.1</b> SOLiD and FISSEQ</a></li>
<li class="chapter" data-level="5.3.2" data-path="current-techs.html"><a href="current-techs.html#cpal"><i class="fa fa-check"></i><b>5.3.2</b> cPAL and ISS</a></li>
<li class="chapter" data-level="5.3.3" data-path="current-techs.html"><a href="current-techs.html#sedal"><i class="fa fa-check"></i><b>5.3.3</b> SEDAL and STARmap</a></li>
<li class="chapter" data-level="5.3.4" data-path="current-techs.html"><a href="current-techs.html#sbs"><i class="fa fa-check"></i><b>5.3.4</b> Sequencing by synthesis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="current-techs.html"><a href="current-techs.html#array"><i class="fa fa-check"></i><b>5.4</b> In situ array capture</a></li>
<li class="chapter" data-level="5.5" data-path="current-techs.html"><a href="current-techs.html#no-imaging"><i class="fa fa-check"></i><b>5.5</b> No imaging</a></li>
<li class="chapter" data-level="5.6" data-path="current-techs.html"><a href="current-techs.html#multiomics"><i class="fa fa-check"></i><b>5.6</b> Spatial multi-omics</a></li>
<li class="chapter" data-level="5.7" data-path="current-techs.html"><a href="current-techs.html#database-current"><i class="fa fa-check"></i><b>5.7</b> Databases of the current era</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="text-mining.html"><a href="text-mining.html"><i class="fa fa-check"></i><b>6</b> Text mining LCM transcriptomics abstracts</a><ul>
<li class="chapter" data-level="6.1" data-path="text-mining.html"><a href="text-mining.html#topic-model"><i class="fa fa-check"></i><b>6.1</b> Topic modeling</a></li>
<li class="chapter" data-level="6.2" data-path="text-mining.html"><a href="text-mining.html#word-time"><i class="fa fa-check"></i><b>6.2</b> Changes of word usage through time</a></li>
<li class="chapter" data-level="6.3" data-path="text-mining.html"><a href="text-mining.html#topic-time"><i class="fa fa-check"></i><b>6.3</b> Changes of topic prevalence through time</a></li>
<li class="chapter" data-level="6.4" data-path="text-mining.html"><a href="text-mining.html#association-of-topics-with-city"><i class="fa fa-check"></i><b>6.4</b> Association of topics with city</a></li>
<li class="chapter" data-level="6.5" data-path="text-mining.html"><a href="text-mining.html#glove"><i class="fa fa-check"></i><b>6.5</b> GloVe word embedding</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="current-analysis.html"><a href="current-analysis.html"><i class="fa fa-check"></i><b>7</b> Data analysis in the current era</a><ul>
<li class="chapter" data-level="7.1" data-path="current-analysis.html"><a href="current-analysis.html#preprocessing"><i class="fa fa-check"></i><b>7.1</b> Preprocessing</a><ul>
<li class="chapter" data-level="7.1.1" data-path="current-analysis.html"><a href="current-analysis.html#registration"><i class="fa fa-check"></i><b>7.1.1</b> Image registration</a></li>
<li class="chapter" data-level="7.1.2" data-path="current-analysis.html"><a href="current-analysis.html#calling"><i class="fa fa-check"></i><b>7.1.2</b> Spot and barcode calling</a></li>
<li class="chapter" data-level="7.1.3" data-path="current-analysis.html"><a href="current-analysis.html#segmentation"><i class="fa fa-check"></i><b>7.1.3</b> Cell segmentation</a></li>
<li class="chapter" data-level="7.1.4" data-path="current-analysis.html"><a href="current-analysis.html#seg-alternatives"><i class="fa fa-check"></i><b>7.1.4</b> Alternatives to cell segmentation</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="current-analysis.html"><a href="current-analysis.html#eda"><i class="fa fa-check"></i><b>7.2</b> Exploratory data analysis</a></li>
<li class="chapter" data-level="7.3" data-path="current-analysis.html"><a href="current-analysis.html#reconstruction"><i class="fa fa-check"></i><b>7.3</b> Spatial reconstruction of scRNA-seq data</a><ul>
<li class="chapter" data-level="7.3.1" data-path="current-analysis.html"><a href="current-analysis.html#direct-dr"><i class="fa fa-check"></i><b>7.3.1</b> Direct dimension reduction</a></li>
<li class="chapter" data-level="7.3.2" data-path="current-analysis.html"><a href="current-analysis.html#ad-hoc"><i class="fa fa-check"></i><b>7.3.2</b> <em>Ad hoc</em> scoring</a></li>
<li class="chapter" data-level="7.3.3" data-path="current-analysis.html"><a href="current-analysis.html#generative"><i class="fa fa-check"></i><b>7.3.3</b> Generative models</a></li>
<li class="chapter" data-level="7.3.4" data-path="current-analysis.html"><a href="current-analysis.html#latent"><i class="fa fa-check"></i><b>7.3.4</b> Shared latent space</a></li>
<li class="chapter" data-level="7.3.5" data-path="current-analysis.html"><a href="current-analysis.html#reconst-other"><i class="fa fa-check"></i><b>7.3.5</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="current-analysis.html"><a href="current-analysis.html#deconvolution"><i class="fa fa-check"></i><b>7.4</b> Cell type deconvolution</a></li>
<li class="chapter" data-level="7.5" data-path="current-analysis.html"><a href="current-analysis.html#variable"><i class="fa fa-check"></i><b>7.5</b> Spatially variable genes</a><ul>
<li class="chapter" data-level="7.5.1" data-path="current-analysis.html"><a href="current-analysis.html#gpr"><i class="fa fa-check"></i><b>7.5.1</b> Gaussian process regression</a></li>
<li class="chapter" data-level="7.5.2" data-path="current-analysis.html"><a href="current-analysis.html#laplacian"><i class="fa fa-check"></i><b>7.5.2</b> Laplacian score</a></li>
<li class="chapter" data-level="7.5.3" data-path="current-analysis.html"><a href="current-analysis.html#variable-other"><i class="fa fa-check"></i><b>7.5.3</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="current-analysis.html"><a href="current-analysis.html#pattern"><i class="fa fa-check"></i><b>7.6</b> Gene patterns</a></li>
<li class="chapter" data-level="7.7" data-path="current-analysis.html"><a href="current-analysis.html#region"><i class="fa fa-check"></i><b>7.7</b> Spatial regions</a></li>
<li class="chapter" data-level="7.8" data-path="current-analysis.html"><a href="current-analysis.html#cell-interaction"><i class="fa fa-check"></i><b>7.8</b> Cell-cell interaction</a><ul>
<li class="chapter" data-level="7.8.1" data-path="current-analysis.html"><a href="current-analysis.html#l-r"><i class="fa fa-check"></i><b>7.8.1</b> Ligand-receptor pairs</a></li>
<li class="chapter" data-level="7.8.2" data-path="current-analysis.html"><a href="current-analysis.html#gene-cell-interact"><i class="fa fa-check"></i><b>7.8.2</b> Genes associated with cell-cell interaction</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="current-analysis.html"><a href="current-analysis.html#gene-interaction"><i class="fa fa-check"></i><b>7.9</b> Gene-gene interaction</a></li>
<li class="chapter" data-level="7.10" data-path="current-analysis.html"><a href="current-analysis.html#subcellular"><i class="fa fa-check"></i><b>7.10</b> Subcellular transcript localization</a></li>
<li class="chapter" data-level="7.11" data-path="current-analysis.html"><a href="current-analysis.html#HnE"><i class="fa fa-check"></i><b>7.11</b> Gene expression imputation from H&amp;E</a></li>
</ul></li>
<li class="part"><span><b>III Future perspectives</b></span></li>
<li class="chapter" data-level="8" data-path="future.html"><a href="future.html"><i class="fa fa-check"></i><b>8</b> From the past to the present to the future</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://bookdown.org" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Museum of Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="current-techs" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Current era technologies</h1>
<div id="microdissection" class="section level2">
<h2><span class="header-section-number">5.1</span> Microdissection</h2>
<p>A simple way to preserve spatial information is to isolate the samples from known locations in the tissue. The samples can be isolated physically or by molecular techniques. The known locations can be targeted, for cells with certain histological characteristics, or untargeted, on a grid over the tissue.</p>
<div id="history-of-lcm" class="section level3">
<h3><span class="header-section-number">5.1.1</span> History of LCM</h3>
<div id="microdissection-1" class="section level4">
<h4><span class="header-section-number">5.1.1.1</span> Microdissection</h4>
<p>LCM, also known as laser microdissection (LMD), is by far the most commonly used method of microdissection. Before LCM, manual microdissection could isolate small pieces of tissue, but the process was laborious <span class="citation">(Bidarimath, Edwards, and Tayade <a href="#ref-Bidarimath2015" role="doc-biblioref">2015</a>)</span>. Laser microdissection predates ISH, though it was not used for spatial transcriptomics until it was possible to profile the transcriptome from small quantity of tissue.</p>
<p>A precursor to laser microdissection is the 1912 “Strahlenstich”, which focused a conventional light source to a spot a few micrometers in size to cut tissues <span class="citation">(Greulich <a href="#ref-Greulich1999" role="doc-biblioref">1999</a>)</span>. Soon after the invention of the laser in 1960, ruby laser was used to manipulate mitochondria, and a ruby laser microdissection system was commercialized by Zeiss in 1965 <span class="citation">(Greulich <a href="#ref-Greulich1999" role="doc-biblioref">1999</a>)</span>. UV laser was used to create chromosomal lesions in 1969 <span class="citation">(Berns, Olson, and Rounds <a href="#ref-BERNS1969" role="doc-biblioref">1969</a>)</span>. The first use of UV laser to cut tissue was in 1976 <span class="citation">(Meier-Ruge et al. <a href="#ref-Meier-Ruge1976" role="doc-biblioref">1976</a>)</span> (Figure <a href="current.html#fig:tl3">4.3</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:lcm"></span>
<img src="LCM.png" alt="A) IR LCM schematic. B) UV LCM and LPC schematic, like in Zeiss PALM Microbeam. C) UV LCM, letting microdissected region fall by gravity, like in Leica LMD. All schematics in this book, i.e. anything not made with ggplot2, were created with BioRender.com" width="100%" />
<p class="caption">
Figure 5.1: A) IR LCM schematic. B) UV LCM and LPC schematic, like in Zeiss PALM Microbeam. C) UV LCM, letting microdissected region fall by gravity, like in Leica LMD. All schematics in this book, i.e. anything not made with <code>ggplot2</code>, were created with BioRender.com
</p>
</div>
<p>At present, there are two main types of LCM: IR and UV. IR LCM was introduced in 1996 <span class="citation">(Emmert-Buck et al. <a href="#ref-Emmert-Buck1996" role="doc-biblioref">1996</a>)</span>. It utilizes a cap with thermoplastic film which is placed over area of interest, and an IR laser to briefly heat select areas of tissues to 90 °C so the film melts in the area and fuses to the area of tissue of interest <span class="citation">(Emmert-Buck et al. <a href="#ref-Emmert-Buck1996" role="doc-biblioref">1996</a>)</span> (Figure <a href="current-techs.html#fig:lcm">5.1</a> A). This was commercialized as the Arcturus PixCell II LCM System in 1997, which was used in several early LCM studies including the first one in 1999 <span class="citation">(Luo et al. <a href="#ref-Luo1999" role="doc-biblioref">1999</a>; Ohyama et al. <a href="#ref-Ohyama2000" role="doc-biblioref">2000</a>; Sgroi et al. <a href="#ref-Sgroi1999" role="doc-biblioref">1999</a>; Kitahara et al. <a href="#ref-Kitahara2001" role="doc-biblioref">2001</a>)</span> (Figure <a href="current.html#fig:tl3">4.3</a>).</p>
<p>UV LCM is also known as laser microbeam microdissection (LMM) due to the microbeam of UV laser used. A popular commercial UV LCM system is the Robot-Microbeam (P.A.L.M. Wolfratshausen, Germany), now Zeiss PALM Microbeam. In this method, a narrow UV laser beam ablates a narrow strip of tissue surrounding the area of interest, isolating the area of interest from the rest of the section, so the area of interest is minimally heated. Then, the area of interest is removed from the slide into the collection vial with laser pressure catapult (LPC), avoiding physical contact so as to prevent cross contamination (Figure <a href="current-techs.html#fig:lcm">5.1</a> B). An early version of this system was first used in 1996 to isolate single cells from gastric tumors, followed by PCR to analyze E-cadherin mutations, but the cells were removed with a needle rather than LPC <span class="citation">(Becker et al. <a href="#ref-Becker1996" role="doc-biblioref">1996</a>)</span>. Another popular commercial UV LCM system is the Leica LMD; unlike the PALM system, the Leica system lets the isolated tissue fall into collection vials by gravity, still avoiding physical contact (Figure <a href="current-techs.html#fig:lcm">5.1</a> C). UV LCM was used in some early LCM spatial transcriptomics studies as well <span class="citation">(Nakamura et al. <a href="#ref-Nakamura2004" role="doc-biblioref">2004</a>)</span>, and remains popular in recent years while IR LCM seems to have fallen out of favor <span class="citation">(Moor et al. <a href="#ref-Moor2017a" role="doc-biblioref">2017</a>; Zechel et al. <a href="#ref-Zechel2014" role="doc-biblioref">2014</a>; Baccin et al. <a href="#ref-Baccin2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>Recent versions of the Arcturus LCM system have both IR and UV, which can be used in conjunction. UV can be used to cut the region of interest (ROI) and IR can then be used to fuse the region to the film at a few points for removal <span class="citation">(“Arcturus XT Laser Capture Microdissection (LCM) Instrument - US,” <a href="#ref-ArcturusUS" role="doc-biblioref">n.d.</a>)</span>.</p>
</div>
<div id="lcm-amplification" class="section level4">
<h4><span class="header-section-number">5.1.1.2</span> Amplification</h4>
<p>The minuscule amount of transcripts from microdissected tissues, which can be single cells, needs to be amplified to be detected by microarray or RNA-seq. Indeed, RNA amplification is a part of one of the most prevalent topics in LCM related search results (Figures <a href="text-mining.html#fig:topics">6.3</a>, <a href="text-mining.html#fig:topic-words">6.4</a>). To this day, there are two main strategies of amplification of minuscule amount of mRNA or cDNA – <em>in vitro</em> transcription (IVT) of cDNA (linear amplification) and PCR (exponential amplification), or a combination of both <span class="citation">(Tang, Lao, and Surani <a href="#ref-Tang2011" role="doc-biblioref">2011</a>)</span>. These two strategies have coexisted since their beginnings in 1989 and 1990 (Figure <a href="current.html#fig:tl3">4.3</a>).</p>
<p>Heterogeneous cDNAs can be amplified with PCR by appending known sequences to one or both ends of the cDNA so primers with known sequences can be used to amplify the heterogeneous cDNAs. Early approaches meant for single cells or small number of cells include tailing the cDNA with poly-dA <span class="citation">(Belyavsky, Vinogradova, and Rajewsky <a href="#ref-Belyavsky1989" role="doc-biblioref">1989</a>)</span> or poly-dG <span class="citation">(Brady, Barbara, and Iscove <a href="#ref-Brady1990" role="doc-biblioref">1990</a>)</span> after reverse transcription, and use as PCR primers sequences containing poly-dT (both poly-dA tail and reverse transcription (RT) primer of poly-A mRNAs) or poly-dC (poly-dG tail) and poly-dT (RT primer). Alternatively, lone linkers (“lone” because they are designed to prevent linker polymerization) could be ligated to both ends of the DNA fragments of interest to anneal to PCR primers <span class="citation">(Ko et al. <a href="#ref-Ko1990" role="doc-biblioref">1990</a>)</span>. Some of the early single cell (or small number of cells) transcriptomic studies used PCR amplification, prior to quantification or differential expression analayses with Southern blot with radiolabeled cDNA probes hybridizing to cDNA clones of interest screened from plaque lift hybridization of a phage cDNA library <span class="citation">(Dulac and Axel <a href="#ref-Dulac1995" role="doc-biblioref">1995</a>)</span>, or with cDNA microarray <span class="citation">(Klein et al. <a href="#ref-Klein2002" role="doc-biblioref">2002</a>; Tietjen et al. <a href="#ref-Tietjen2003" role="doc-biblioref">2003</a>)</span>. LCM was used to isolate the single cells in <span class="citation">(Tietjen et al. <a href="#ref-Tietjen2003" role="doc-biblioref">2003</a>)</span>. Before the advent of CEL-seq, early scRNA-seq methods also used PCR amplification <span class="citation">(Tang et al. <a href="#ref-Tang2009" role="doc-biblioref">2009</a>; Islam et al. <a href="#ref-Islam2011" role="doc-biblioref">2011</a>)</span>. An influential method is switching mechanism at the 5′ end of the RNA transcript (SMART) <span class="citation">(Zhu et al. <a href="#ref-Zhu2001" role="doc-biblioref">2001</a>)</span>, for construction of cDNA (clone) libraries covering the full length of mRNAs, though not originally for single cells. The full length scRNA-seq method Smart-seq(2) <span class="citation">(Ramsköld et al. <a href="#ref-Ramskold2012" role="doc-biblioref">2012</a>)</span> is based on SMART but adapted to the minuscule amount of transcripts from single cells, with PCR amplification of the cDNA. Smart-seq(2) is one of the most commonly used library preparation methods for LCM since the 2010s, and was used for RNA-seq of LCM isolated single cells <span class="citation">(Nichterwitz et al. <a href="#ref-Nichterwitz2016" role="doc-biblioref">2016</a>)</span>.</p>
<p>Alternatively, transcripts can be amplified by IVT, with a T7 RNA polymerase promoter attached to the 5’ end of the poly-dT primer, so the RNA polymerase transcribes the cDNAs into many copies of antisense RNAs (aRNA) <span class="citation">(Van Gelder et al. <a href="#ref-VanGelder1990" role="doc-biblioref">1990</a>; Eberwine et al. <a href="#ref-Eberwine1992" role="doc-biblioref">1992</a>)</span>. Some of the early single cell (or small number of cells) transcriptomic studies used IVT amplification. Quantification and differential expression analyses of the aRNA can be performed with differential display <span class="citation">(Liang and Pardee <a href="#ref-Liang1992" role="doc-biblioref">1992</a>; Kacharmina, Crino, and Eberwine <a href="#ref-Kacharmina1999" role="doc-biblioref">1999</a>)</span>, cDNA microarray <span class="citation">(Hemby et al. <a href="#ref-Hemby2002" role="doc-biblioref">2002</a>; Kamme et al. <a href="#ref-Kamme2003" role="doc-biblioref">2003</a>)</span>, or with “expression profiling” <span class="citation">(Eberwine et al. <a href="#ref-Eberwine1992" role="doc-biblioref">1992</a>; Kacharmina, Crino, and Eberwine <a href="#ref-Kacharmina1999" role="doc-biblioref">1999</a>)</span>, i.e. reverse northern blot with radiolabeled aRNAs hybridizing to cDNA clones of interest, where the cDNA clones can be blotted onto a Southern blot membrane in a macroarray, which may have inspired the development of the cDNA microarray printed on glass <span class="citation">(Kacharmina, Crino, and Eberwine <a href="#ref-Kacharmina1999" role="doc-biblioref">1999</a>)</span>. LCM was used to isolate the single cells in <span class="citation">(Kamme et al. <a href="#ref-Kamme2003" role="doc-biblioref">2003</a>)</span>. Since the 2010s, Cell Expression by Linear amplification and Sequencing (CEL-seq) <span class="citation">(Hashimshony et al. <a href="#ref-Hashimshony2012" role="doc-biblioref">2012</a>)</span> and derivatives (e.g. CEL-seq2, MARS-seq, and SORT-seq), which use IVT amplification, have been commonly used for library preparation for microdissected or <em>de facto</em> microdissected samples such as from LCM <span class="citation">(Tzur et al. <a href="#ref-Tzur2018" role="doc-biblioref">2018</a>)</span>, Tomo-seq <span class="citation">(Junker et al. <a href="#ref-Junker2014" role="doc-biblioref">2014</a>)</span>, and Niche-seq <span class="citation">(Medaglia et al. <a href="#ref-Medaglia2017" role="doc-biblioref">2017</a>)</span>.</p>
</div>
</div>
<div id="lcm-usage" class="section level3">
<h3><span class="header-section-number">5.1.2</span> Usage of LCM</h3>
<p>Usage trends of LCM as reflected in PubMed and bioRxiv search results are analyzed in Chapter <a href="text-mining.html#text-mining">6</a>. LCM can be used to isolate targeted ROIs based on histology, or to create a grid for untargeted search of gene expression patterns in space, and examples of both are highlighted here. Moreover, the three themes of screening, atlas curation, and new technique development, are all represented in LCM literature. In the “screening” theme, LCM is used to isolate cell populations of interest based on histology (targeted) to discover genes associated with pathological conditions such as cancer metastasis <span class="citation">(Nakamura et al. <a href="#ref-Nakamura2004" role="doc-biblioref">2004</a>)</span> and cell types <span class="citation">(Aguila et al. <a href="#ref-Aguila2018" role="doc-biblioref">2018</a>)</span>, or to discover cell type localization in healthy tissue difficult to other spatial transcriptomics techniques such as the bone marrow <span class="citation">(Baccin et al. <a href="#ref-Baccin2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>LCM can also be used to dissect the tissue in a grid, not targeting very specific histological regions (untargeted), to identify genes associated with locations on the grid <span class="citation">(Moor et al. <a href="#ref-Moor2018" role="doc-biblioref">2018</a>; Peng et al. <a href="#ref-Peng2016" role="doc-biblioref">2016</a>)</span> or transcriptomically defined regions <span class="citation">(Zechel et al. <a href="#ref-Zechel2014" role="doc-biblioref">2014</a>; Peng et al. <a href="#ref-Peng2016" role="doc-biblioref">2016</a>)</span>, or to map cells from scRNA-seq to spatial locations <span class="citation">(Zechel et al. <a href="#ref-Zechel2014" role="doc-biblioref">2014</a>; Peng et al. <a href="#ref-Peng2016" role="doc-biblioref">2016</a>)</span>. The untargeted studies can also touch upon the “atlas” theme, providing an online interface to query and explore the spatial transcriptomes <span class="citation">(Peng et al. <a href="#ref-Peng2016" role="doc-biblioref">2016</a>)</span>.</p>
<p>However, targeted approaches can also be used for the “atlas” theme, such as in the human <span class="citation">(Hawrylycz et al. <a href="#ref-Hawrylycz2012" role="doc-biblioref">2012</a>; Miller et al. <a href="#ref-Miller2014" role="doc-biblioref">2014</a>)</span> and macaque <span class="citation">(Bakken et al. <a href="#ref-Bakken2016" role="doc-biblioref">2016</a>)</span> atlases of the ABA, isolating histologically annotated regions for microarray profiling to build systematic resources for exploration. This addresses the limitation of bright field ISH that only one gene can be stained per section thus requiring large number of brains, which is too costly for primates; in LCM, while often not single cell resolution, the same brain can be used to profile the whole transcriptome. The “technique development” theme is evident in the text mining results (Figure <a href="text-mining.html#fig:topics">6.3</a>), and contributes to some of the advantages of LCM as discussed below.</p>
<p>As shown in Chapter <a href="text-mining.html#text-mining">6</a>, LCM transcriptomics has spread far and wide, and has been used on many research topics rarely featured in (WM)ISH atlases. These include cancer and botany (Figure <a href="text-mining.html#fig:lcm-map">6.2</a>, Figure <a href="text-mining.html#fig:topics">6.3</a>). The following advantages of LCM might have contributed to its popularization: first, as already mentioned, both IR and UV LCM systems have been commercialized prior to their use for transcriptomics, making setup convenient. Second, while LCM equipment can be expensive and require specialized training to use, many institutions have core facilities that can perform LCM <span class="citation">(“Translational Pathology Core Laboratory (TPCL)” <a href="#ref-TPCL" role="doc-biblioref">n.d.</a>; “Veritas Laser Capture Microdissection (LCM) and Laser Cutting System from Applied Biosystems” <a href="#ref-Veritas" role="doc-biblioref">n.d.</a>; “Dana-Farber Core Facilities” <a href="#ref-Dana-Farber" role="doc-biblioref">n.d.</a>; “Johns Hopkins Cell Imaging Core Facility” <a href="#ref-JHU" role="doc-biblioref">n.d.</a>)</span>, reducing cost and personnel training time in individual laboratories.</p>
<p>Third, in some cases, especially in the clinical setting, only archival formalin fixed, paraffin embedded (FFPE) tissues are available. While in 2020, newer current era technologies such as Visium <span class="citation">(Villacampa et al. <a href="#ref-Villacampa2020" role="doc-biblioref">2020</a>)</span> and GeoMX DSP <span class="citation">(Hwang et al. <a href="#ref-Hwang2020" role="doc-biblioref">2020</a>)</span> have been demonstrated on FFPE tissues, LCM followed by microarray was already demonstrated on FFPE tissues in 2007 <span class="citation">(Coudry et al. <a href="#ref-Coudry2007" role="doc-biblioref">2007</a>)</span> and with RNA-seq by 2014 <span class="citation">(Morton et al. <a href="#ref-Morton2014" role="doc-biblioref">2014</a>)</span>. As a result, for several years, LCM may have been the only option to perform spatial transcriptomics on FFPE samples. In addition, LCM might still be the only way to profile transcriptomes of single cells in FFPE samples. With scRNA-seq library preparation methods such as Smart-seq2 <span class="citation">(Nichterwitz et al. <a href="#ref-Nichterwitz2016" role="doc-biblioref">2016</a>)</span>, and CEL-seq <span class="citation">(Tzur et al. <a href="#ref-Tzur2018" role="doc-biblioref">2018</a>)</span> it is possible to profile the transcriptome in minuscule amount of LCM isolated tissue, and even single cells <span class="citation">(Nichterwitz et al. <a href="#ref-Nichterwitz2016" role="doc-biblioref">2016</a>)</span>. More recently, with Smart-3SEQ, LCM single cell transcriptomics has been made possible for FFPE tissues as well, even for samples that are several years old <span class="citation">(Foley et al. <a href="#ref-Foley2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>Finally, despite its long history, LCM cannot yet be replaced by newer spatial transcriptomics technologies. Unlike smFISH or ISS based techniques, LCM followed by RNA-seq is not restricted to known genes and allows for transcriptome wide profiling and other omics. Unlike ST and Visium, LCM can have single cell resolution, and unlike array based techniques with resolution of the size of a cell or higher, such as Slide-seq(2) and HDST, LCM can more unequivocally isolate individual cells or nuclei based on histology.</p>
<p>LCM has a number of disadvantages, some of which are addressed by other current era spatial transcriptomics technologies. First, compared to droplet based scRNA-seq and highly multiplexed barcoding, using LCM to isolate single cells is still too laborious, limiting its throughput. Second, LCM requires tissue sections, while preparation of many slides to cover a 3D volume can be laborious and it can be challenging to reconstruct 3D structures from tissue sections. To reiterate, sections of blastoderm stage embryos are hard to interpret, which motivated WMISH. Third, because it can be challenging to segment cells based on hematoxylin and eosin (H&amp;E) or immunohistochemistry (IHC) staining and parts of different cells can be stacked within the thickness of the section even in thin sections, single cells isolated by LCM can have contents of other cells. Fourth, LCM can reduce RNA quality in cells <span class="citation">(Kerman et al. <a href="#ref-Kerman2006" role="doc-biblioref">2006</a>)</span>, perhaps due to heat and UV.</p>
</div>
<div id="physical-microdissection" class="section level3">
<h3><span class="header-section-number">5.1.3</span> Physical microdissection</h3>
<div id="voxelation" class="section level4">
<h4><span class="header-section-number">5.1.3.1</span> Voxelation</h4>

<div class="figure" style="text-align: center"><span id="fig:voxelation"></span>
<img src="voxelation.png" alt="Voxelation of human brain, as in (Vanessa M. Brown 2002)." width="20%" />
<p class="caption">
Figure 5.2: Voxelation of human brain, as in <span class="citation">(Vanessa M. Brown <a href="#ref-Brown2002a" role="doc-biblioref">2002</a>)</span>.
</p>
</div>
<p>LCM did not completely replace microdissection with a physical blade. Voxelation was one of the alternatives to LCM developed to profile spatial transcriptomes in 3D and address the limitation of throughput of ISH. In voxelation, a grid of steel blades is used to cut tissue into cubes for microarray profiling, but the resolution is low. Human brains were first cut into 8 mm thick slabs and then a grid of 1 cm per side <span class="citation">(Vanessa M. Brown <a href="#ref-Brown2002a" role="doc-biblioref">2002</a>; Singh et al. <a href="#ref-Singh2003" role="doc-biblioref">2003</a>)</span>, and mouse brains were first cut into 1 mm thick slabs and then a grid of 1 mm per side <span class="citation">(V. M. Brown et al. <a href="#ref-Brown2002" role="doc-biblioref">2002</a>; Singh et al. <a href="#ref-Singh2003" role="doc-biblioref">2003</a>; Chin et al. <a href="#ref-Chin2007" role="doc-biblioref">2007</a>)</span> (Figure <a href="current-techs.html#fig:voxelation">5.2</a>). With low resolution, it’s easier to use voxelation to profile large 3D tissues of multiple slabs that would be much more laborious with LCM’s thinner sections and higher resolution <span class="citation">(V. M. Brown et al. <a href="#ref-Brown2002" role="doc-biblioref">2002</a>)</span>. As the human voxels were quite large (almost 1 ml) and corresponding voxels of 20 to 30 mice were pooled <span class="citation">(V. M. Brown et al. <a href="#ref-Brown2002" role="doc-biblioref">2002</a>; Chin et al. <a href="#ref-Chin2007" role="doc-biblioref">2007</a>)</span> to get enough transcripts, the voxelation studies did not mention T7-based PCR amplification of transcripts, unlike for LCM samples <span class="citation">(Nakamura et al. <a href="#ref-Nakamura2004" role="doc-biblioref">2004</a>)</span>. To the best of our knowledge, voxelation never spread beyond its institution of origin, UCLA School of Medicine, and has not been used in a publication to generate new data since 2007 <span class="citation">(Chin et al. <a href="#ref-Chin2007" role="doc-biblioref">2007</a>)</span> and for data analysis since 2009 <span class="citation">(An et al. <a href="#ref-An2009" role="doc-biblioref">2009</a>)</span>.</p>
</div>
<div id="tomo-seq" class="section level4">
<h4><span class="header-section-number">5.1.3.2</span> Tomo-seq</h4>
<div class="figure" style="text-align: center"><span id="fig:tomo"></span>
<img src="tomo.png" alt="Tomo-seq, here showing C. elegans." width="30%" />
<p class="caption">
Figure 5.3: Tomo-seq, here showing C. elegans.
</p>
</div>
<p>Another alternative to LCM is Tomo-seq/array, which has continued to be utilized in recent years. In this approach, the tissue is sectioned with a cryotome like tomography (hence the “Tomo”), and the transcripts in each section are extracted for microarray (Tomo-array) or RNA-seq (Tomo-seq) profiling; the resolution is limited by section thickness, which has gone down to 8 <span class="math inline">\(\mu\)</span>m <span class="citation">(Brink et al. <a href="#ref-VandenBrink2020" role="doc-biblioref">2020</a>)</span>. Three-D expression maps can be reconstructed from sections along the anterior-posterior (AP), dorsal-ventral (DV), and left-right (LR) axes. All three themes, namely screening, atlas cutation, and new technique development, are present in Tomo-seq/array literature.</p>
<p>Tomo-array was first used in 2012 to build a 3D mouse brain transcriptome atlas, attempting to address difficulties in image registration in ISH atlases, low resolution of voxelation, and limitation of LCM to specific regions <span class="citation">(Okamura-Oho et al. <a href="#ref-Okamura-Oho2012" role="doc-biblioref">2012</a>)</span> (Figure <a href="current.html#fig:tl3">4.3</a>). Mouse brains were sectioned along all three axes and 200 adjacent 5 <span class="math inline">\(\mu\)</span>m sections were pooled as “fractions” for microarray; again, PCR amplification was not mentioned. Fractions from the three axes were then used to reconstruct a 3D atlas.</p>
<p>Tomo-seq was first demonstrated in 2013, on <em>Drosophila melanogaster</em> embryos, with 60 and 25 <span class="math inline">\(\mu\)</span>m sections, again in response to the difficulty to scale ISH atlases to the whole transcriptome <span class="citation">(Combs and Eisen <a href="#ref-Combs2013" role="doc-biblioref">2013</a>)</span>. Genes patterned along the AP axis were identified, and the data is stored in an <a href="http://eisenlab.org/sliceseq/">online database</a>. However, Tomo-seq is more commonly credited to a 2014 method first demonstrated on zebrafish embryos, with 18 <span class="math inline">\(\mu\)</span>m sections <span class="citation">(Junker et al. <a href="#ref-Junker2014" role="doc-biblioref">2014</a>)</span>. Gene expression patterns along the AP axis of straightened embryos were identified, and sections along all three axes were used for 3D reconstruction of embryos that were not straightened. The data and the 3D reconstruction are also stored in an <a href="http://zebrafish.genomes.nl/tomoseq/">online database</a>, though the 3D reconstruction algorithm produced many artefacts.</p>
<p>Since then, Tomo-seq has been used in several different biological systems, typically when one axis is of primary interest. Tomo-seq has been used in <em>C. elegans</em> <span class="citation">(Ebbing et al. <a href="#ref-Ebbing2018" role="doc-biblioref">2018</a>)</span>, developing zebrafish hearts <span class="citation">(Burkhard and Bakkers <a href="#ref-Burkhard2018" role="doc-biblioref">2018</a>)</span>, <em>Drosophila</em> embryos <span class="citation">(Combs and Fraser <a href="#ref-Combs2018" role="doc-biblioref">2018</a>)</span>, ischemic mouse hearts <span class="citation">(Lacraz et al. <a href="#ref-Lacraz2017" role="doc-biblioref">2017</a>)</span>, and <a href="http://wormbook.org/chapters/www_genomesPristionchus/genomesPristionchus.html"><em>Pristionchus pacificus</em></a> <span class="citation">(Rödelsperger et al. <a href="#ref-Rodelsperger2020" role="doc-biblioref">2020</a>)</span> to identify genes associated with that axis of interest. Tomo-seq was also used on mouse <span class="citation">(Brink et al. <a href="#ref-VandenBrink2020" role="doc-biblioref">2020</a>)</span> and human <span class="citation">(Moris et al. <a href="#ref-Moris2020" role="doc-biblioref">2020</a>)</span> gastruloids to demonstrate the viability of this in vitro and potentially high-throughput model for developmental biology. Again, due to the minuscule amount of tissue in each section, library preparation methods designed for scRNA-seq, such as CEL-seq(2) <span class="citation">(Junker et al. <a href="#ref-Junker2014" role="doc-biblioref">2014</a>; Rödelsperger et al. <a href="#ref-Rodelsperger2020" role="doc-biblioref">2020</a>; Ebbing et al. <a href="#ref-Ebbing2018" role="doc-biblioref">2018</a>)</span> have been adapted to Tomo-seq.</p>
</div>
<div id="other-methods-of-physical-microdissection" class="section level4">
<h4><span class="header-section-number">5.1.3.3</span> Other methods of physical microdissection</h4>
<p>More recently, Spatial Transcriptomics by Reoriented Projections and sequencing (STRP-seq) was developed in response to the limited number of genes of smFISH and ISS based techniques, degradation of RNA and technical complexity of LCM, and number of specimens required by and inadequacy of the 2014 Tomo-seq 3D reconstruction <span class="citation">(Schede et al. <a href="#ref-Schede2020" role="doc-biblioref">2020</a>)</span>. In STRP-seq, adjacent sections of the tissue are sectioned in different orientations, and are then used for 3D construction with an algorithm inspired by reconstruction of ray-based computerized tomography. This has been shown to perform better than the 2014 Tomo-seq 3D reconstruction method, and was demonstrated on the brain of a non-model organism, the lizard <a href="https://en.wikipedia.org/wiki/Pogona_vitticeps"><em>Pogona vitticeps</em></a>.</p>
<p>Because of the specialized equipment and technical complexity of LCM and degradation of RNA, other methods of physical microdissection have been developed. Examples of such techniques are Cell and Tissue Acquisition System (CTAS), which uses a disposable capillary unit connect to the vacuum to aspirate tissue <span class="citation">(Kudo et al. <a href="#ref-Kudo2012" role="doc-biblioref">2012</a>)</span>, and an automated micropuncch system that collects samples of tissue with diameter of 110 <span class="math inline">\(\mu\)</span>m at 300 <span class="math inline">\(\mu\)</span>m intervals <span class="citation">(Yoda et al. <a href="#ref-Yoda2017" role="doc-biblioref">2017</a>)</span>. In addition, for similar reasons, manual microdissection is still used (Figure <a href="current.html#fig:n-insts">4.7</a>), such as to dissect leaves on a grid of distances from a lesion to characterize response to infection <span class="citation">(Giolai et al. <a href="#ref-Giolai2019" role="doc-biblioref">2019</a>; Lukan et al. <a href="#ref-Lukan2020" role="doc-biblioref">2020</a>)</span>. Manual microdissection of pre-defined anatomical regions was also used to create low resolution gene expression atlases of <em>Xenopus laevis</em> <span class="citation">(Plouhinec et al. <a href="#ref-Plouhinec2017" role="doc-biblioref">2017</a>)</span> and <em>Xenopus tropicalis</em> <span class="citation">(Blitz et al. <a href="#ref-Blitz2017" role="doc-biblioref">2017</a>)</span> embryos, to avoid sectioning as required for LCM and artefacts in Tomo-seq 3D reconstruction.</p>
</div>
</div>
<div id="de-facto-microdissection" class="section level3">
<h3><span class="header-section-number">5.1.4</span> <em>De facto</em> microdissection</h3>
<p>Some methods have been developed that do not directly cut tissues. Instead, cells, or ROIs judged from histology, are optically and molecularly marked so that only transcripts or cells from the marked regions are captured. Because these methods involve selection of pre-defined ROIs within the section, we call them <em>de facto</em> microdissection.</p>
<p>Transcriptome <em>in vivo</em> analysis (TIVA) from 2014 can be viewed as the first of these methods <span class="citation">(Lovatt et al. <a href="#ref-Lovatt2014" role="doc-biblioref">2014</a>)</span>. Live cell culture is incubated with the photoactivable cage with a poly-U sequence that captures poly-A transcripts. Select cells are photoactivated by 405 nm laser and the captured transcripts are sequenced. TIVA is widely cited, perhaps because it is one of the earliest single cell resolution and transcriptome wide methods, predating RNA-seq from LCM isolated single cells. However, because TIVA has only been demonstrated on fewer than a dozen cells per sample, to the best of our knowledge it has not been used in any other publication to collect new data.</p>
<div class="figure" style="text-align: center"><span id="fig:niche"></span>
<img src="niche.png" alt="Niche-seq schematics. Green: cells with photoactivated PA-GFP." width="35%" />
<p class="caption">
Figure 5.4: Niche-seq schematics. Green: cells with photoactivated PA-GFP.
</p>
</div>
<p>Two <em>de facto</em> microdissection methods have spread beyond their institutions of origin. One of them is Niche-seq, which was developed as LCM is still usually used to isolate groups of cells rather than single cells and involves tissue fixation <span class="citation">(Medaglia et al. <a href="#ref-Medaglia2017" role="doc-biblioref">2017</a>)</span>. Select regions of <em>ex vivo</em> tissues from transgenic mice expressing photoactivable GFP (PA-GFP), here lymph node and spleen B cell and T cell niches, are photoactivated at 820 nm with two photon irradiation. Then the tissue is dissociated and cells with photoactivated PA-GFP are collected from flow cytometry–based fluorescence-activated cell sorting (FACS) for scRNA-seq with MARS-seq (Figure <a href="current-techs.html#fig:niche">5.4</a>). This approach was originally used in 2010 to isolate B cells from light and dark zones of the lymph node followed by transcriptome profiling with microarray in bulk <span class="citation">(<span class="citeproc-not-found" data-reference-id="Victoria2010"><strong>???</strong></span>)</span>; the difference in Niche-seq is that the scRNA-seq of the sorted cells. After its inception, Niche-seq has been used once more in lymph node niches <span class="citation">(De Giovanni et al. <a href="#ref-DeGiovanni2020" role="doc-biblioref">2020</a>)</span>. However, as Niche-seq requires transgenic mice expressing PA-GFP and living tissue, it cannot be applied to human tissues, to fixed tissues, or when a PA-GFP line is unavailable. This might limit further growth of Niche-seq. Moreover, the spatial context of cells within the photoactivated region is lost, limiting spatial resolution.</p>

<div class="figure" style="text-align: center"><span id="fig:dsp"></span>
<img src="dsp.png" alt="GeoMX DSP schematics, inspired by figures in (Merritt et al. 2019). Black: transcripts in tissue. Gray: probes. Green: indexing oligo." width="100%" />
<p class="caption">
Figure 5.5: GeoMX DSP schematics, inspired by figures in <span class="citation">(Merritt et al. <a href="#ref-Merritt2019" role="doc-biblioref">2019</a>)</span>. Black: transcripts in tissue. Gray: probes. Green: indexing oligo.
</p>
</div>
<p>Another method that spread beyond its institution of origin is the commercial GeoMX DSP from NanoString <span class="citation">(Merritt et al. <a href="#ref-Merritt2019" role="doc-biblioref">2019</a>)</span>, which can be used for both high throughput immunofluorescence and transcript quantification in FFPE tissue sections. For transcript quantification, probes are attached to indexing oligos with a UV cleavable linker (Figure <a href="current-techs.html#fig:dsp">5.5</a>). The selected ROI is illuminated by UV to remove the index oligos from the probes. Then the released index oligos are aspirated and quantified with either NGS or NanoString nCounter. This can be repeated for multiple ROIs, which can be a grid for unbiased profiling <span class="citation">(Merritt et al. <a href="#ref-Merritt2019" role="doc-biblioref">2019</a>)</span>. The probes tile the transcripts, and each probe has a distinct index oligo, so in NGS, each tile is counted separately, enabling isoform quantification <span class="citation">(Merritt et al. <a href="#ref-Merritt2019" role="doc-biblioref">2019</a>)</span>. GeoMx DSP is not transcriptome wide; up to 1860 target transcripts have been quantified <span class="citation">(Margaroli et al. <a href="#ref-Margaroli2021" role="doc-biblioref">2021</a>)</span>. Also, as pre-defined probes are required, unlike in RNA-seq, novel transcripts cannot be quantified. Ready made probe sets for oncology, immunology, and neuroscience are sold by NanoString <span class="citation">(“Gene Expression Panels | NanoString Technologies” <a href="#ref-DSPPanels" role="doc-biblioref">n.d.</a>)</span>. Although GeoMx DSP was published in 2019, it has spread to several different institutionss, and has been used on pancreatic ductal adenocarcinoma (PDAC) <span class="citation">(Hwang et al. <a href="#ref-Hwang2020" role="doc-biblioref">2020</a>)</span>, hepatocellular carcinoma (HCC) <span class="citation">(Sharma et al. <a href="#ref-Sharma2020" role="doc-biblioref">2020</a>)</span>, reactive lymph nodes <span class="citation">(Tripodo et al. <a href="#ref-Tripodo2020" role="doc-biblioref">2020</a>)</span>, and COVID infected lungs from autopsy <span class="citation">(Park et al. <a href="#ref-Park2021" role="doc-biblioref">2021</a>; Butler et al. <a href="#ref-Butler2021" role="doc-biblioref">2021</a>; Delorey et al. <a href="#ref-Delorey2021" role="doc-biblioref">2021</a>; Margaroli et al. <a href="#ref-Margaroli2021" role="doc-biblioref">2021</a>)</span>. A variant of GeoMX DSP, called GeoMX Whole Transcriptome Atlas (WTA), has been used to profile transcripts of 18190 genes, nearly covering the whole transcriptome <span class="citation">(Roberts et al. <a href="#ref-Roberts2021" role="doc-biblioref">2021</a>)</span>. In GeoMX WTA, the UV cleaved index oligo must be sequenced with NGS to identify the gene each transcript is from.</p>
</div>
<div id="summary" class="section level3">
<h3><span class="header-section-number">5.1.5</span> Summary</h3>
<div class="figure" style="text-align: center"><span id="fig:categories"></span>
<img src="05-current-techs_files/figure-html/categories-1.png" alt="Number of publications per category of techniques in the current era. Non-curated LCM literature is excluded." width="576" />
<p class="caption">
Figure 5.6: Number of publications per category of techniques in the current era. Non-curated LCM literature is excluded.
</p>
</div>
<p>Overall, in the current era, microdissection is the most widely used type of technique (Figure <a href="current-techs.html#fig:categories">5.6</a>). Excluding LCM, Tomo-seq is the most popular technique after ST and Visium (Figure <a href="current.html#fig:n-insts">4.7</a>). Microdissection has not been replaced by other seemingly more sophisticated techniques such as ST and MERFISH, and is still popular in 2020 and 2021 (Figure <a href="current.html#fig:current-hist">4.1</a>, Figure <a href="text-mining.html#fig:lcm-year">6.1</a>). Microdissection techniques generally do not have single cell resolution, but combined with scRNA-seq or snRNA-seq data, cell type compositions of ROIs can be computationally deconvoluted <span class="citation">(Baccin et al. <a href="#ref-Baccin2020" role="doc-biblioref">2020</a>; Hwang et al. <a href="#ref-Hwang2020" role="doc-biblioref">2020</a>)</span>. The popularity may be due to availability of commercial platforms (LCM and GeoMX DSP), core facilities (LCM), Nanostring’s commercial data collection and analysis service for GeoMX DSP <span class="citation">(“DSP Technology Access Program (TAP)” <a href="#ref-nanostring_tap" role="doc-biblioref">n.d.</a>)</span>, not requiring specialized equipment (Tomo-seq, manual microdissection), or disadvantages of other techniques discussed later in this chapter.</p>
</div>
</div>
<div id="smfish" class="section level2">
<h2><span class="header-section-number">5.2</span> Single molecular FISH</h2>
<p>One quantitative approach to transcripts abundance estimation is to display individual transcripts as distinct puncta with FISH and count them. Prior to smFISH, transmission electron microscopy was used to visualize individual mRNA molecules in fibroblasts by labeling the poly-A tail with a single large colloidal gold particle and the <em>in situ</em> reverse transcribed cDNA with small gold particles <span class="citation">(Bassell et al. <a href="#ref-Bassell1994" role="doc-biblioref">1994</a>)</span>. That FISH can be used to visualize single mRNA molecules was first demonstrated in 1998 <span class="citation">(Femino et al. <a href="#ref-Femino1998" role="doc-biblioref">1998</a>)</span> (Figure <a href="current.html#fig:tl3">4.3</a>). Five or more probes targeting adjacent parts of the transcript, each about 50 nt long and labeled with 5 fluorophores were hybridized to the transcripts. The puncta seen were shown to be likely individual mRNA molecules, as the fluorescence intensity of each punctum was consistent with the number of fluorophores, and the number of puncta for <span class="math inline">\(\beta\)</span>-actin was consistent with the number of <span class="math inline">\(\beta\)</span>-actin transcripts measured by other means, and the colors of puncta seen from probes with different colored fluorophores targeting different parts of the transcript were consistent with organization of the fluorophores on the transcript (Figure <a href="current-techs.html#fig:smfish">5.7</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:smfish"></span>
<img src="smfish1998.png" alt="A) Schematic of smFISH from (Femino et al. 1998). The long thick line stands for the mRNA, and short think line stands for DNA oligo probe. B) smFISH with singly labeled probes from (Raj et al. 2008)." width="50%" />
<p class="caption">
Figure 5.7: A) Schematic of smFISH from <span class="citation">(Femino et al. <a href="#ref-Femino1998" role="doc-biblioref">1998</a>)</span>. The long thick line stands for the mRNA, and short think line stands for DNA oligo probe. B) smFISH with singly labeled probes from <span class="citation">(Raj et al. <a href="#ref-Raj2008" role="doc-biblioref">2008</a>)</span>.
</p>
</div>
<p>The 1998 approach had a number of disadvantages, leading to development of an alternative approach in 2008 <span class="citation">(Raj et al. <a href="#ref-Raj2008" role="doc-biblioref">2008</a>)</span>. First, probes labeled with multiple fluorophore moieties are difficult to synthesize and purify. Second, the multiple fluorophores on the same probe can interact with each other and self-quench. Third, out of the 5 probes per transcript, only 1 or 2 may have actually hybridize to the transcript in most cases, making it difficult to distinguish between true signal and non-specific binding. In the 2008 method, each 17-22 nt probe is labeled with one fluorophore at the 3’ end, and a larger number of probes (48 or more) targeting tandem sequences of the transcript were used to improve signal to noise ratio (Figure <a href="current-techs.html#fig:smfish">5.7</a>). The probes were computationally designed and ordered from Biosearch Technologies. This method influenced later highly multiplexed smFISH techniques; computational probe design and commercial synthesis would remain crucial.</p>
<div id="barcoding" class="section level3">
<h3><span class="header-section-number">5.2.1</span> Barcoding strategies</h3>
<p>To use smFISH to quantify transcripts transcriptome wide, there is an obvious challenge – how to distinguish among over 20,000 genes with only about 5 easily distinguishable colors? Various strategies using multiple colors and/or rounds of hybridization or imaging have been devised to drastically expand the palette. The first attempt to do so was in 1989, using 3 colors to visualize 4 chromosomes in immunological DNA FISH <span class="citation">(Nederlof et al. <a href="#ref-Nederlof1990" role="doc-biblioref">1990</a>)</span> (Figure <a href="current.html#fig:tl3">4.3</a>). Each probe can be labeled with one or two of the 3 haptens: biotin, 2-acetyl aminofluorene (AAF), and Chemiprobe. Red fluorophore was attached to avidin to target biotin label, and blue and green to different secondary antibodies targeting, respectively, mouse anti-Chemiprobe and rabbit anti-AAF primary antibodies (Figure <a href="current-techs.html#fig:hb">5.8</a>). Then with one doubly labeled and 3 singly labeled probes, imaged with different excitation wavelengths or channels, 3 colors can distinguish 4 chromosomes. However, with this method, the palette size is limited by the number of haptens available and the number of their combinations.</p>

<div class="figure" style="text-align: center"><span id="fig:hb"></span>
<img src="historical_barcoding.png" alt="A) Combinatorial barcoding in immunological DNA FISH, as described in (Nederlof et al. 1990). The line stands for the probe and the circle, triangle, and square stand for haptens. Not to scale, and only one hapten of each kind is shown on one probe. B) Combinatorial barcoding in (Levsky 2002). Short colored lines stand for probes with fluorophores of the color. C) Schematic of SRM seqFISH as described in (Lubeck and Cai 2012)." width="90%" />
<p class="caption">
Figure 5.8: A) Combinatorial barcoding in immunological DNA FISH, as described in <span class="citation">(Nederlof et al. <a href="#ref-Nederlof1990" role="doc-biblioref">1990</a>)</span>. The line stands for the probe and the circle, triangle, and square stand for haptens. Not to scale, and only one hapten of each kind is shown on one probe. B) Combinatorial barcoding in <span class="citation">(Levsky <a href="#ref-Levsky2002" role="doc-biblioref">2002</a>)</span>. Short colored lines stand for probes with fluorophores of the color. C) Schematic of SRM seqFISH as described in <span class="citation">(Lubeck and Cai <a href="#ref-Lubeck2012" role="doc-biblioref">2012</a>)</span>.
</p>
</div>
<p>For transcript detection, the first attempt was in 2002 <span class="citation">(Levsky <a href="#ref-Levsky2002" role="doc-biblioref">2002</a>)</span>; fluorophore labeled probes were synthesized as in the 1998 smFISH method, and either probes of one color or a mixture of probes of 2 colors were hybridized to the transcript, and imaged with different channels, to visualize transcription foci in the nucleus (Figure <a href="current-techs.html#fig:hb">5.8</a>). This way, combinations of 2 of the 4 available colors plus blank were used to encode 10 different transcripts.</p>
<p>The above mentioned historical works in smFISH and combinatorial barcoding laid foundation to smFISH based spatial transcriptomics. The first attempt to quantify transcripts with combinatorial barcoding at single molecular resolution was in 2012 by Long Cai’s group, which later developed seqFISH and its variants <span class="citation">(Lubeck and Cai <a href="#ref-Lubeck2012" role="doc-biblioref">2012</a>)</span>. Like in the 2008 smFISH study, singly labeled probes purchased from Biosearch were used, but forming blocks of different colors as in the 1998 smFISH <span class="math inline">\(\beta\)</span>-actin experiment. Then the transcripts were imaged with super-resolution microscopy (SRM), in particular stochastic optical reconstruction microscopy (STORM). In the spatial barcoding strategy, the ordering of the colors in space would distinguish between transcripts, but would require linearization of the transcripts and high resolution (20 nm) (Figure <a href="current-techs.html#fig:hb">5.8</a>). To improve signal to noise ratio, cyanine dye–based photoswitchable dye pairs <span class="citation">(Bates et al. <a href="#ref-Bates2012" role="doc-biblioref">2012</a>)</span> was used so both the activator and the emitter fluorophores must be present and adjacent for the fluorophores to be reactivated. In the spectral barcoding approach, the pairs of fluorophores are spread across the transcript, so the transcripts are recognized by the pairs of fluorophores detected (Figure <a href="current-techs.html#fig:hb">5.8</a>). The spectral approach requires lower resolution (100 nm) and does not require linearization, but because the ordering of the colors is not used, the number of possible barcode from the same number of colors is smaller than in the spatial approach. With spectral barcoding, transcripts of 32 genes were quantified in yeast, with 3 color barcodes chosen from 7 available colors. To the best of our knowledge, after its inception, this SRM method has not been used to generate new data, perhaps because it requires specialized equipment for SRM. None of the later methods in our curated database used SRM.</p>
<p>Thus far, probes with fluorophores of different colors were hybridized to mRNAs at the same time, without multiple rounds of hybridization. To obtain single molecular resolution but without SRM, there is a challenge of needing to use multiple probes of the same color to strengthen signal, which requires transcripts that are long enough to accommodate probes of different colors. The more colors that are used to encode more genes, the longer the transcripts must be.</p>
<p>This changed in 2014, with the advent of seqFISH <span class="citation">(Lubeck et al. <a href="#ref-Lubeck2014" role="doc-biblioref">2014</a>)</span>. Twenty four singly labeled probes were designed for each gene, and 12 genes were encoded with 4 colors and 2 rounds of hybridization (Figure <a href="current-techs.html#fig:seqfish">5.9</a>). After imaging the first round of hybridization and DAPI staining for DNA, the probes are removed with DNase I, and then probes for the second round are hybridized. Let <span class="math inline">\(F\)</span> denote the number of fluorophores or colors, and <span class="math inline">\(N\)</span> denote the number of rounds of hybridization, then the number of genes that can be barcoded is <span class="math inline">\(F^N\)</span>. However, with longer barcodes to encode more genes, error can build up.</p>

<div class="figure" style="text-align: center"><span id="fig:seqfish"></span>
<img src="seqfish.png" alt="Probe structures of 2014 seqFISH (Lubeck et al. 2014) and seqFISH error correction." width="100%" />
<p class="caption">
Figure 5.9: Probe structures of 2014 seqFISH <span class="citation">(Lubeck et al. <a href="#ref-Lubeck2014" role="doc-biblioref">2014</a>)</span> and seqFISH error correction.
</p>
</div>
<p>The most common error in multi-round smFISH is missing signal, most likely in one round <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>; K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>)</span>. If all <span class="math inline">\(F^N\)</span> barcodes are used and one round is missing for a mRNA molecule, then the existing signal of this molecule is consistent to <span class="math inline">\(F\)</span> genes, so it cannot be uniquely identified. If a small proportion of barcodes are intentionally left out to control for false positives, as done in this first version of seqFISH (4 out of 16), then error correction is still not guaranteed. A further defense against errors in 2014 seqFISH was to repeat the 2 rounds of hybridization 3 times, so 6 rounds were performed. This filtered out false positives where repeated rounds didn’t match, and barring false positives, this can recover the original 2 barcoding rounds if up to 2 of the 6 total rounds have missing signal.</p>
<p>Another error correction scheme was introduced in 2016, with hybridization chain reaction (HCR) seqFISH <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>)</span>, and was used in seqFISH+ <span class="citation">(Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span> as well. One more round of hybridization than necessary to encode the number of genes of interest was used, and the barcodes are designed so that if one of the rounds is missing, the remaining rounds still uniquely identify the gene (Figure <a href="current-techs.html#fig:seqfish">5.9</a>). For example, with 5 colors, 3 rounds are enough to encode 100 genes, as 125 barcodes are possible. However, a fourth round is used, so missing one round can still result in 3 remaining rounds that uniquely identify the gene.</p>

<div class="figure" style="text-align: center"><span id="fig:merfish"></span>
<img src="merfish.png" alt="Schematic of MERFISH (K. H. Chen et al. 2015; Moffitt et al. 2016) and MERFISH error correction." width="100%" />
<p class="caption">
Figure 5.10: Schematic of MERFISH <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>; Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>)</span> and MERFISH error correction.
</p>
</div>
<p>An alternative to seqFISH was developed with error correction in mind – multiplexed error-robust FISH (MERFISH) <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>)</span>. In MERFISH each encoding probe has a 30 nt long region that targets the transcript, and 2 or 3 20 nt <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>)</span> readout sequences to bind to readout probes (Figure <a href="current-techs.html#fig:merfish">5.10</a>). First, the encoding probes are hybridized to the transcripts. For each round of hybridization, readout probes, singly labeled, are hybridized to the readout sequences on the encoding probes and imaged. Then the fluorescence of the previous round is either photobleached (version 1) <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>)</span> or when the fluorophore is bound to the readout probe with a disulfide bond, cleaved off with a reducing agent such as Tris(2-carboxyethyl)phosphine (TCEP) (version 2) <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>)</span>. The readout probes are not stripped, and in the next round, new readout probes are hybridized to new readout sequences and imaged.</p>
<p>The MERFISH barcodes are binary, with “1” for a round with fluorescence, and “0” without, and must differ from other barcodes at at least 4 places, i.e. with <a href="https://en.wikipedia.org/wiki/Hamming_distance">Hamming distance</a> of at least 4 (HD4). As missing signal is the most common error, each barcode has 4 1’s, or Hamming weight 4. This way, when one round is missing, the gene can still be uniquely identified, but when 2 rounds are missing, the remaining barcode is equally distant to 2 genes, so the error cannot be corrected (Figure <a href="current-techs.html#fig:merfish">5.10</a>). Sixteen rounds of imaging, or 16 bits, would result in 140 barcodes. In this case, there are 16 different readout sequences, and each gene is assigned 4 of them, for the 4 1’s in the barcode. If the code is expanded to 69 bits, then about 10,000 genes can be encoded, and imaging time can be cut to a third by using 3 colors to image 3 bits per round <span class="citation">(C. Xia, Fan, et al. <a href="#ref-Xia2019a" role="doc-biblioref">2019</a>)</span>. An HD2 code, i.e. barcodes are at least hamming distance 2 away from each other, can also be used, but errors can only be recognized but not corrected. All variants of MERFISH use this type of binary barcoding.</p>
<div class="figure" style="text-align: center"><span id="fig:sf-plus"></span>
<img src="seqfish-plus.png" alt="Schematic of seqFISH with pseudocolors." width="100%" />
<p class="caption">
Figure 5.11: Schematic of seqFISH with pseudocolors.
</p>
</div>
<p>More recently, a new variant of seqFISH was devised to scale up to 10,000 genes <span class="citation">(Shah et al. <a href="#ref-Shah2018" role="doc-biblioref">2018</a>)</span>. The barcoding and hybridization scheme enabling such scale was first introduced in vitro in 2017 as RNA SPOTs <span class="citation">(Eng et al. <a href="#ref-Eng2017" role="doc-biblioref">2017</a>)</span>, and was then adapted to cultured cells in 2018, targeting introns of nascent transcripts of over 10,000 genes <span class="citation">(Shah et al. <a href="#ref-Shah2018" role="doc-biblioref">2018</a>)</span>. In 2019, this scheme was used to profile mature transcripts of 10,000 genes in both cell culture and the mouse brain, and with super-resolution <span class="citation">(Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span>. Super-resolution beyond the diffraction limit can be achieved by computationally super-resolving the transcript spots with a radial center algorithm <span class="citation">(Parthasarathy <a href="#ref-Parthasarathy2012" role="doc-biblioref">2012</a>)</span> when spot density is very high to help with decoding barcodes; the super-resolution version is known as seqFISH+. While this new version of seqFISH can reduce optical crowding and greatly expand the palette, the super-resolution algorithm that can further reduce crowding does not have to be used to locate the transcript spots when density is low. This version of seqFISH was again used to visualize genomic loci (super-resolution) <span class="citation">(Takei et al. <a href="#ref-Takei2020" role="doc-biblioref">2020</a>)</span> and mature transcripts of a smaller number of genes (not super-resolution) <span class="citation">(Lohoff et al. <a href="#ref-Lohoff2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>This method is quite different from previous seqFISH variants, and is in some ways reminiscent of MERFISH. Like previous versions of seqFISH, each barcode is a series of colors, but a large number of “pseudocolors”, specifically 20 per channel in the seqFISH+ study, are used rather than the 5 fluorophores, so 3 rounds of hybridization can encode <span class="math inline">\(20^3\)</span> or 8000 genes per channel. Any number of pseudocolors and rounds can be used depending on the number of genes profiled. Each primary probe has a 28 nt region targeting the transcript and 4 readout sites of 15 nt. Each readout site has as many different sequences as there are pseudocolors, and the 4 sites correspond to the series of 4 pseudocolors in the barcode. First, 24 primary probes are hybridized to the transcripts. Then for each place of the barcode, 20 (or whatever number of pseudocolors) rounds of hybridization with readout probes are performed, stripping with formamide between rounds. In these 20 rounds, each gene should light up only once, and its place in the 20 rounds is its pseudocolor (Figure <a href="current-techs.html#fig:sf-plus">5.11</a>). This way, in each image, only 1 out of 20 molecules of interest imaged in the channel fluoresce, reducing optical crowding. For the entire barcode of length 4, there would be 80 rounds of hybridization. In contrast, in MERFISH, with the 16 bit barcode, this would be 1 out of 4. Like in MERFISH, a larger number of real colors, or channels, can be used to increase throughput, to image multiple pseudocolors simultaneously. So with 3 channels, 24,000 genes can be encoded. The same error correction method as in HCR seqFISH was used, so while a barcode of length 3 is sufficient, length 4 was used.</p>
<div class="figure" style="text-align: center"><span id="fig:split-fish"></span>
<img src="split-fish.png" alt="Schematic of split-FISH." width="100%" />
<p class="caption">
Figure 5.12: Schematic of split-FISH.
</p>
</div>
<p>Another new method, called split-FISH <span class="citation">(Goh et al. <a href="#ref-Goh2020" role="doc-biblioref">2020</a>)</span> was devised to reduce off target hybridization, and thus background noise and some barcoding errors. For each encoding probe or bridge probe like in MERFISH, a pair of split probes hybridize to the transcript itself (Figure <a href="current-techs.html#fig:split-fish">5.12</a>). Half of the split probes would bind to the transcript, and the other half bind to the bridge probe. Then as in MERFISH, the bridge probe has 2 readout sequences and singly labeled readout probes bind to the bridge probe for imaging. This method reduces off target hybridization because the bridge probe can only indirectly bind to the transcript if both of the split probes hybridize to the transcript. To encode 317 genes, 2 places out of 26 in binary barcodes are chosen to be “1”, resulting into 325 possible barcodes; 8 of them are left blank to control for false positives. Error correction is not mentioned.</p>
<p>Despite the availability of the above barcoding schemes, when the number of genes stained for is not too large, each gene can still be encoded by only one round of hybridization and one color. When the number of genes is larger than the number of colors, each round of hybridization stains for as many genes as there as colors, and the probes are stripped so the next round stains for a different set of genes. This has been done in osmFISH <span class="citation">(Codeluppi et al. <a href="#ref-Codeluppi2018" role="doc-biblioref">2018</a>)</span> staining for 33 genes, in a non-barcoded adaptation of HCR-seqFISH called Spatial Genomic Analysis (SGA) <span class="citation">(Lignell et al. <a href="#ref-Lignell2017" role="doc-biblioref">2017</a>)</span> staining for 35 genes, and in Expansion-Assisted Iterative Fluorescence <em>In Situ</em> Hybridization (EASI-FISH) 26 genes <span class="citation">(Wang et al. <a href="#ref-Wang2021" role="doc-biblioref">2021</a>)</span>.</p>
</div>
<div id="amplification" class="section level3">
<h3><span class="header-section-number">5.2.2</span> Signal amplification</h3>
<p>As already mentioned, in smFISH, a large number of singly labeled probes can be used to boost signal, but not all transcripts are long enough to accommodate this number of probes. Furthermore, isoform specific exons are often not long enough to accommodate these probes for isoform specific staining. Without increasing the number of probes, background reduction such as by tissue clearing, split probes (e.g. in split-FISH), and using fluorophores with colors very different from the color of autofluorescence <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>)</span> can increase signal to noise ratio. There are also ways to boost signals without increasing the number of probes, the most common of which are branched DNA (bDNA), rolling circle amplification (RCA), and HCR. All of these methods non-covalently attach numerous fluorophores to the probe to amplify signal. Background reduction and signal amplification can be used in conjunction.</p>
<div id="bdna" class="section level4">
<h4><span class="header-section-number">5.2.2.1</span> Branched DNA</h4>
<p>Dating back at least as far back as to 1993 <span class="citation">(Urdea <a href="#ref-Urdea1993" role="doc-biblioref">1993</a>)</span>, early use of bDNA in ISH was to detect low copy number of viral genomes, eventually down to single copies <span class="citation">(Player et al. <a href="#ref-Player2001" role="doc-biblioref">2001</a>)</span>. bDNA signal amplification involves several steps of hybridization (Figure <a href="current-techs.html#fig:bdna">5.13</a>). First, usually some sort of bridge probe binds to the transcript itself. Then the primary amplifier binds to the bridge probe, leaving a long overhang. Then multiple secondary amplifiers bind to the primary amplifier on the overhang of the primary amplifier, and each secondary amplifier also leaves an overhang. Finally, multiple labeled readout probes bind to each secondary amplifier. This way, space available for hybridization of the readout probes is drastically expanded, allowing for more fluorophores per unit transcript length.</p>
<div class="figure" style="text-align: center"><span id="fig:bdna"></span>
<img src="bdna.png" alt="Schematic of bDNA. The Z probes are specific to RNAscope, but the other parts are generic to bDNA." width="100%" />
<p class="caption">
Figure 5.13: Schematic of bDNA. The Z probes are specific to RNAscope, but the other parts are generic to bDNA.
</p>
</div>
<p>For FISH, a particularly influential bDNA method is RNAscope, introduced in 2012 for FFPE tissues, and is now commercially available from ACD <span class="citation">(Wang et al. <a href="#ref-Wang2012" role="doc-biblioref">2012</a>)</span>. In addition to bDNA amplification, RNAscope reduces background noise from non-specific hybridization by using 2 bridge Z probes in between the transcript and the primary amplifier, so the primary amplifier will only bind when both Z probes are present. An smFISH RNAscope method has been used to profile around 1000 genes in cell culture <span class="citation">(Battich, Stoeger, and Pelkmans <a href="#ref-Battich2013" role="doc-biblioref">2013</a>)</span> and 49 genes in the mouse somatosensory cortex <span class="citation">(Bayraktar et al. <a href="#ref-Bayraktar2020" role="doc-biblioref">2020</a>)</span>, although these experiments were not highly multiplexed and only one or a handful of genes distinguishable by fluorophore color were stained for in the same cells or sections; numerous cells and sections were stained to cover all genes in the gene panels. However, bDNA has also made its way into highly multiplexed smFISH, as a variant of MERFISH <span class="citation">(C. Xia, Babcock, et al. <a href="#ref-Xia2019c" role="doc-biblioref">2019</a>)</span>. Here, the primary amplifier binds to the readout regions of the MERFISH encoding probe. Like in regular MERFISH (v2), the fluorophores are attached to the readout probes by a disulfide bond and removed by TCEP after each round of hybridization; the bDNA moiety is not removed. With bDNA amplification, only 16 probes per gene can detect about as many transcripts as with 92 unamplified probes <span class="citation">(C. Xia, Babcock, et al. <a href="#ref-Xia2019c" role="doc-biblioref">2019</a>)</span>.</p>
</div>
<div id="RCA" class="section level4">
<h4><span class="header-section-number">5.2.2.2</span> Rolling circle amplification</h4>
<p>Chronologically, the next of the popular signal amplification method is padlock probe RCA. Padlock probe was introduced in 1994 by Mats Nilsson as a way to reduce background in ISH and to detect single nucleotide variants (SNVs) <span class="citation">(Nilsson et al. <a href="#ref-Nilsson1994" role="doc-biblioref">1994</a>)</span>. Both ends of of the padlock probe must hybridize to the target without terminal mismatches for the ligase to connect the ends of the probe to form a circle (Figure <a href="current-techs.html#fig:rca">5.14</a>); thus padlock probe and RCA can detect SNPs and point mutations <span class="citation">(Larsson et al. <a href="#ref-Larsson2010" role="doc-biblioref">2010</a>; Lizardi et al. <a href="#ref-Lizardi1998" role="doc-biblioref">1998</a>)</span>. The circle encloses the target like a padlock on a string, hence the name “padlock probe”. Then probes that are not circularized are digested by an exonuclease. RCA was introduced in 1995 as a way to create tandem repeats and potentially point to origins of tandem repeats in genomes, not seeming to have signal amplification in mind <span class="citation">(Fire and Xu <a href="#ref-Fire1995" role="doc-biblioref">1995</a>)</span>. A primer anneals to circularized DNA and is then elongated by <span class="math inline">\(\Phi29\)</span> DNA polymerase, and as the polymerase goes around the circle many times, many copies of the complimentary sequences of the circle are made (Figure <a href="current-techs.html#fig:rca">5.14</a>). In 1998, padlock probes and RCA were united to create a method of signal amplification <span class="citation">(Baner et al. <a href="#ref-Baner1998" role="doc-biblioref">1998</a>; Lizardi et al. <a href="#ref-Lizardi1998" role="doc-biblioref">1998</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:rca"></span>
<img src="rca.png" alt="Schematic of RCA, here shown with target priming though a separate primer can also be used. Red segment is the gene barcode." width="100%" />
<p class="caption">
Figure 5.14: Schematic of RCA, here shown with target priming though a separate primer can also be used. Red segment is the gene barcode.
</p>
</div>
<p>In spatial transcriptomics, padlock probe and RCA were initially used for <em>in situ</em> sequencing (ISS) <span class="citation">(Ke et al. <a href="#ref-Ke2013" role="doc-biblioref">2013</a>)</span>, but more recently adapted to smFISH. The padlock probe with the gene barcode is hybridized to <em>in situ</em> reverse transcribed cDNA as in ISS and hybridization-based ISS (HybISS) <span class="citation">(Gyllborg et al. <a href="#ref-Gyllborg2020" role="doc-biblioref">2020</a>)</span>, or the mRNA itself as in SCRINSHOT <span class="citation">(Sountoulidis et al. <a href="#ref-Sountoulidis2020" role="doc-biblioref">2020</a>)</span> and hybridization-based RNA ISS (HybRISS) <span class="citation">(Lee et al. <a href="#ref-Lee2020" role="doc-biblioref">2020</a>)</span>. RCA can be initiated with the target cDNA itself as a primer or with a separate primer when the target is mRNA. Then readout probes are hybridized to the RCA amplified gene barcode, with <span class="citation">(Gyllborg et al. <a href="#ref-Gyllborg2020" role="doc-biblioref">2020</a>)</span> or without <span class="citation">(Sountoulidis et al. <a href="#ref-Sountoulidis2020" role="doc-biblioref">2020</a>)</span> a bridge probe. In Hyb(R)ISS and SCRINSHOT, multiple rounds of readout hybridization encode each gene with a sequence of colors as in seqFISH; although error correction is not discussed, the seqFISH error correction scheme can be easily adapted. Perhaps because of larger number of copies of the gene barcode sequence produced by RCA, Hyb(R)ISS and SCRINSHOT use 5 probes per gene, each with a 30 nt (HybISS, target sequences are proprietary information of CARTANA for HybRISS) or 40 nt (SCRINSHOT) region to target the transcript. While we are unaware of isoform specific studies conducted with Hyb(R)ISS or SCRINSHOT, isoform specific exons may more realistically accommodate the 5 probes. While smFISH based techniques were typically designed for frozen sections, SCRINSHOT was designed for FFPE sections.</p>
</div>
<div id="hcr" class="section level4">
<h4><span class="header-section-number">5.2.2.3</span> Hybridization chain reaction</h4>
<div class="figure" style="text-align: center"><span id="fig:hcr"></span>
<img src="hcr.png" alt="Schematic of HCR, showing 3 cycles, but this can continue indefinitely until H1 and H2 are exhausted. Arrow shows 5' to 3' direction." width="100%" />
<p class="caption">
Figure 5.15: Schematic of HCR, showing 3 cycles, but this can continue indefinitely until H1 and H2 are exhausted. Arrow shows 5’ to 3’ direction.
</p>
</div>
<p>A third signal amplification method is HCR, introduced in 2004 <span class="citation">(Dirks and Pierce <a href="#ref-Dirks2004" role="doc-biblioref">2004</a>)</span>, which has been adapted to seqFISH, giving rise to HCR-seqFISH. EASI-FISH also uses HCR for signal amplification. In singly labeled hairpins, the long stem is protected by the short stem, but can also hybridize with short stems of other hairpins (Figure <a href="current-techs.html#fig:hcr">5.15</a>). The long stem of H1 can hybridize to the short stem of H2, and vice versa (Figure <a href="current-techs.html#fig:hcr">5.15</a>). First, an initiator probe is hybridized to the transcript (24 per gene in the 2016 HCR-seqFISH study). Then the long stem of H1 hybridizes to the part of initiator not hybridized to the transcribe, now leaving the short stem vacant. Then the long stem of H2 hybridizes to the vacant short stem of H1, and now the short stem of H2 is vacant for another H1. This cycle can continue indefinitely until H1 and H2 are depleted. This way, many fluorophores are tethered to the target transcript without increasing the number of probes bound to the transcript, thus amplifying signal.</p>
<p>Similarly, RCA can continue indefinitely until DNA polymerase is inhibited or removed or when deoxynucleotides are depleted. In contrast, the bDNA moiety has a controlled size and does not grow indefinitely until stopped. In both bDNA and HCR, the amplified moiety is still anchored on the target transcript. In contrast, since when the padlock probe encloses the target, the DNA polymerase is inhibited <span class="citation">(Baner et al. <a href="#ref-Baner1998" role="doc-biblioref">1998</a>)</span>, the padlock must be dissociated from the target before RCA, or in the case of target priming, the target cDNA itself grows into the RCA hairball. As the hairball is not anchored to the original target, it can drift away and obscure the original location of the target.</p>
</div>
</div>
<div id="crowding" class="section level3">
<h3><span class="header-section-number">5.2.3</span> Optical crowding</h3>
<p>As we have seen, smFISH based spatial transcriptomics has been scaled to around 10,000 genes and can potentially be scaled to the whole transcriptome. With increasing number of mRNA molecules visualized, it’s also increasingly likely for different target molecules to be so close to each other that their fluorescent spots overlap or are even within the diffraction limit of the optical microscope and appear as one point. This is the problem of optical crowding, and some existing ways to mitigate this problem are summarized below.</p>
<p>As already mentioned, SRM is not susceptible to this problem <span class="citation">(Lubeck and Cai <a href="#ref-Lubeck2012" role="doc-biblioref">2012</a>)</span>, though access to SRM is not as common as access to regular confocal or epifluorescent microscopes. Another simple strategy is to select the most highly expressed genes from RNA-seq. These genes are imaged separately with smFISH, with one color and one round of hybridization per gene instead of combinatorial barcoding, as was done in the first MERFISH study <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>)</span>. However, with increasing number of highly expressed genes, this method becomes increasingly laborious. Also as already mentioned, in seqFISH+, only 1 in 60 mRNA molecules of interest light up in each channel and round of hybridization (20 pseudocolors per channel and 3 channels), and the transcript spots can be computationally super-resolved, thus reducing optical crowding <span class="citation">(Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span>.</p>
<p>Another strategy is to allow transcript spots to overlap but computationally resolve them, as in corrFISH <span class="citation">(Coskun and Cai <a href="#ref-Coskun2016a" role="doc-biblioref">2016</a>)</span>, BarDensr <span class="citation">(S. Chen et al. <a href="#ref-Chen2021" role="doc-biblioref">2021</a>)</span>, ISTDECO <span class="citation">(Andersson et al. <a href="#ref-Andersson2021" role="doc-biblioref">2021</a>)</span>, and Composite In Situ Imaging (CISI) <span class="citation">(Cleary et al. <a href="#ref-Cleary2021" role="doc-biblioref">2021</a>)</span>. In corrFISH, Transcripts of highly expressed genes encoding ribosomal proteins were visualized with sequential hybridization and 2 colors but not every gene lights up in each round of hybridization; each gene is encoded by one color and a sequence of 0’s (absence of fluorescence) and 1’s (presence) of that color. Then images from different rounds of hybridization in the same FOV are correlated to identify transcripts that are 1’s in both rounds amidst transcripts that are not 1’s in both rounds. To the best of our knowledge, after its conception, corrFISH has not been applied to generate any new high throughput dataset.</p>
<p>A more recent method, BarDensr, models the observed brightness of potentially mixed spots in terms of the point spread function (PSF), codebook, unknown spot density, probe washing, background, and per round per channel gain. Then the unknown spot density and deconvolution of barcodes at mixed spots are inferred by maximizing sparsity of the spots in space (most voxels don’t have spots) while keeping reconstruction loss of the observed brightness sufficiently low. BarDensr is very recently published, and, as of writing, we are unaware of studies that used the method. ISTDECO is similar but only uses a Gaussian PSF, codebook, and background.</p>
<p>CISI uses seqFISH-like barcoding, but does not even require spot detection. Gene abundance is computationally inferred with <a href="https://en.wikipedia.org/wiki/Compressed_sensing">compressed sensing</a>. First, an autoencoder is trained on composite images with different channels. Then in the latent space inferred by the autoencoder, the channels are decompressed with compressed sensing principles and decoded into genes with the decoder branch of the trained autoencoder. The barcodes and genes must be carefully chosen from an existing dataset. The genes must be described by a small number of coexpression modules so module activity is sparse. Inferring the sparse module activity before inferring individual gene levels at the decompression step is more tractable than directly inferring individual gene abundances.</p>
<div class="figure" style="text-align: center"><span id="fig:exm"></span>
<img src="exm.png" alt="Schematic of expansion microscopy." width="100%" />
<p class="caption">
Figure 5.16: Schematic of expansion microscopy.
</p>
</div>
<p>A strategy that has been reused is expansion microscopy (ExM). When a polyelectrolyte gel is dialyzed in water, it expands as its polymer network changes into extended conformations <span class="citation">(Chen, Tillberg, and Boyden <a href="#ref-Chen2015b" role="doc-biblioref">2015</a>)</span>. First, the tissue is infused with monomers of the gel. Then with small molecule linkers, molecules of interest such as fluorophores and RNAs can be covalently incorporated to the polymer network over the course of free radical polymerization. After the gel forms, proteins in the tissue are digested to homogenize mechanical properties of the gel and to clear the tissue to reduce autofluorescent background. Then the gel is soaked in water to expand, linearly expanding 3 to 4.5 times on each side <span class="citation">(Chen, Tillberg, and Boyden <a href="#ref-Chen2015b" role="doc-biblioref">2015</a>; Chen et al. <a href="#ref-Chen2016" role="doc-biblioref">2016</a>)</span> (Figure <a href="current-techs.html#fig:exm">5.16</a>). This way, transcripts attached to the gel are physically separated, avoiding optical crowding. ExM has thus been adapted to MERFISH for this purpose <span class="citation">(Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>, as well as EASI-FISH. In addition, EASI-FISH was used to quantify transcripts in 300 <span class="math inline">\(\mu\)</span>m thick brain slices and imaging was accelerated with light sheet microscopy. However, a disadvantage of ExM is that each FOV now covers less of the original tissue, thus increasing imaging time. Furthermore, the expanded gel would continue to expand during the rounds of hybridization. As the expansion is non-linear and non-isotropic, barcode decoding is challenging as it’s difficult to match transcript spots across rounds of hybridizations.</p>
</div>
<div id="smfish-usage" class="section level3">
<h3><span class="header-section-number">5.2.4</span> Usage of smFISH based techniques</h3>
<div class="figure" style="text-align: center"><span id="fig:records-gene"></span>
<img src="05-current-techs_files/figure-html/records-gene-1.png" alt="Record number of genes per dataset quantified by smFISH based techniques over time." width="576" />
<p class="caption">
Figure 5.17: Record number of genes per dataset quantified by smFISH based techniques over time.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:records-cell"></span>
<img src="05-current-techs_files/figure-html/records-cell-1.png" alt="Record total number of cells per study profiled by smFISH based techniques over time." width="576" />
<p class="caption">
Figure 5.18: Record total number of cells per study profiled by smFISH based techniques over time.
</p>
</div>
<p>As already noted, the number of genes whose transcripts can be possibly quantified simultaneously in the same piece of tissue with highly multiplexed smFISH based technology has increased over time (Figure <a href="current-techs.html#fig:records-gene">5.17</a>). The number of cells that can be imaged in one study has also increased (Figure <a href="current-techs.html#fig:records-cell">5.18</a>). However, in practice, the actual number of genes and cells profiled has not significantly increased (Figure <a href="current-techs.html#fig:smfish-lm-gene">5.19</a>, Figure <a href="current-techs.html#fig:smfish-lm-cell">5.21</a>). These plots only show papers that reported the number of cells and genes in the main text; if we download and process all publicly available datasets associated with such papers, the trends might change, although figures of papers that do not report the number of cells (number of genes is usually reported in smFISH and ISS studies) don’t seem to indicate that the trend would change significantly. Moreover, as discussed in Section <a href="current-techs.html#multiomics">5.6</a>, some of the studies used smFISH based methods to visualize DNA loci and 3D chromatin structure alongside transcripts. The number of genes here is for the transcripts, including when only introns are targeted.</p>
<p>An earlier version of the plot of number of genes over time plotted the mean number of genes for each study, due to difficulty in defining what constitutes a dataset. However, since that version caused confusion as sometimes one study profiled very different number of genes in different experiments, we decided to give some definition of “dataset” and not to plot the mean. Here a “dataset” means either a different tissue, cell type, experimental or clinical condition, or a separate experiment profiling a different number or set of genes in the same study. One dataset can involve multiple sections and individuals.</p>

<div class="figure" style="text-align: center"><span id="fig:smfish-lm-gene"></span>
<img src="05-current-techs_files/figure-html/smfish-lm-gene-1.png" alt="Number of genes per datasets in each study, over time. Gray ribbon is 95% confidence interval (CI). The points are translucent; more opaque points are multiple datasets from the same study." width="864" />
<p class="caption">
Figure 5.19: Number of genes per datasets in each study, over time. Gray ribbon is 95% confidence interval (CI). The points are translucent; more opaque points are multiple datasets from the same study.
</p>
</div>
<p>The trend line looks pretty flat. Although studies quantifying a very large number of genes tend to be recent, many other studies profiling fewer genes pulled the line down. The slope (with all data, outliers and all) is not significantly different from 0 (t-test). The number of genes was not log transformed precisely to show that even the outliers can’t give a significantly rising trend and to visually emphasize that the number of genes profiled by the vast majority of smFISH based studies is well below 1000.</p>
<pre><code>## 
## Call:
## lm(formula = n_genes ~ date_published, data = smfish)
## 
## Residuals:
##    Min     1Q Median     3Q    Max 
## -710.0 -608.4 -516.1 -410.1 9809.5 
## 
## Coefficients:
##                  Estimate Std. Error t value Pr(&gt;|t|)
## (Intercept)    -1324.3427  4899.7555  -0.270    0.788
## date_published     0.1092     0.2739   0.399    0.691
## 
## Residual standard error: 2098 on 90 degrees of freedom
##   (5 observations deleted due to missingness)
## Multiple R-squared:  0.001764,   Adjusted R-squared:  -0.009327 
## F-statistic: 0.159 on 1 and 90 DF,  p-value: 0.691</code></pre>
<p>To better show the trend of the vast majority of datasets that profiled far fewer than 10,000 genes, a broken y-axis is introduced to zoom into the majority of datasets (<a href="current-techs.html#fig:smfish-broken-y">5.20</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:smfish-broken-y"></span>
<img src="05-current-techs_files/figure-html/smfish-broken-y-1.png" alt="Like the previous figure, but more zoomed in to datasets with fewer than 10,000 genes." width="864" />
<p class="caption">
Figure 5.20: Like the previous figure, but more zoomed in to datasets with fewer than 10,000 genes.
</p>
</div>
<p>How total number of cells profiled in each study that reported the number of cells in the main text is shown here. The total number across datasets is used because sometimes number of cells per dataset is not reported.</p>
<div class="figure" style="text-align: center"><span id="fig:smfish-lm-cell"></span>
<img src="05-current-techs_files/figure-html/smfish-lm-cell-1.png" alt="Total number of cells per study profiled by smFISH based techniques over time." width="864" />
<p class="caption">
Figure 5.21: Total number of cells per study profiled by smFISH based techniques over time.
</p>
</div>
<p>Again, although studies that profiled large numbers of cells tend to be more recent, as there are many recent studies with smaller numbers of cells, the slope (with outliers and all) is not significantly different from 0 (t-test).</p>
<pre><code>## 
## Call:
## lm(formula = n_cells ~ date_published, data = sum_cells)
## 
## Residuals:
##     Min      1Q  Median      3Q     Max 
## -161866 -117427  -93644   -6971  916882 
## 
## Coefficients:
##                  Estimate Std. Error t value Pr(&gt;|t|)
## (Intercept)    -1.090e+06  9.211e+05  -1.184    0.247
## date_published  6.728e+01  5.177e+01   1.300    0.205
## 
## Residual standard error: 234600 on 26 degrees of freedom
## Multiple R-squared:  0.061,  Adjusted R-squared:  0.02489 
## F-statistic: 1.689 on 1 and 26 DF,  p-value: 0.2051</code></pre>
<p>MERFISH is the technique used in the most studies (Figure <a href="current-techs.html#fig:smfish-trend">5.22</a>), although most of the smFISH based techniques barely spread beyond their institutions of origin, if at all (Figure <a href="current-techs.html#fig:smfish-n-insts">5.23</a>). The following advantages and disadvantages of smFISH based techniques may explain these trends in usage. Advantages and disadvantages of individual smFISH based techniques reviewed so far are summarized in Table <a href="current-techs.html#tab:table2">5.1</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:smfish-trend"></span>
<img src="05-current-techs_files/figure-html/smfish-trend-1.png" alt="Number of publications over time, broken down by technique type. Preprints are included, and the gray histogram in the background is the overall trend of all smFISH based techniques." width="576" />
<p class="caption">
Figure 5.22: Number of publications over time, broken down by technique type. Preprints are included, and the gray histogram in the background is the overall trend of all smFISH based techniques.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:smfish-n-insts"></span>
<img src="05-current-techs_files/figure-html/smfish-n-insts-1.png" alt="Number of techniques that have been used by each number of institutions; most techniques have only been used by 1 institution, i.e. the institution of origin." width="576" />
<p class="caption">
Figure 5.23: Number of techniques that have been used by each number of institutions; most techniques have only been used by 1 institution, i.e. the institution of origin.
</p>
</div>
<p>MERFISH and seqFISH and their variants are more used and have spread, though their use still concentrates in their institutions of origin (Harvard for MERFISH and Caltech for seqFISH) (Figure <a href="current-techs.html#fig:smfish-map">5.24</a>). The one use of MERFISH at Caltech and the use of seqFISH at Dana-Farber and Cambridge are due to collaboration between the two institutions; the first authors analyzed the data while the data itself was most likely still collected at the techniques’ institutions of origin. Nevertheless, this shows interest in MERFISH and seqFISH beyond their institutions of origin.</p>

<div class="figure" style="text-align: center"><span id="fig:smfish-map"></span>
<img src="05-current-techs_files/figure-html/smfish-map-1.png" alt="Geographical locations of institutions that used certain techniques. Point area is proportional to number of publication from the city of interest. Gray points in the background is all publications using smFISH based techniques. The cities and institutions labeled are those of the first author. Note that for seqFISH, the hidden Markov random field (HMRF) study at Dana Faber (Zhu et al. 2018) and the mouse embryo study (Lohoff et al. 2020) had collaboration with Long Cai’s group at Caltech, so the dataset was most likely still collected at Caltech." width="960" />
<p class="caption">
Figure 5.24: Geographical locations of institutions that used certain techniques. Point area is proportional to number of publication from the city of interest. Gray points in the background is all publications using smFISH based techniques. The cities and institutions labeled are those of the first author. Note that for seqFISH, the hidden Markov random field (HMRF) study at Dana Faber <span class="citation">(Zhu et al. <a href="#ref-Zhu2018" role="doc-biblioref">2018</a>)</span> and the mouse embryo study <span class="citation">(Lohoff et al. <a href="#ref-Lohoff2020" role="doc-biblioref">2020</a>)</span> had collaboration with Long Cai’s group at Caltech, so the dataset was most likely still collected at Caltech.
</p>
</div>
<p>SmFISH based techniques have the following advantages. First, smFISH, especially with larger number of probes, have nearly 100% detection efficiency of transcripts <span class="citation">(Lubeck and Cai <a href="#ref-Lubeck2012" role="doc-biblioref">2012</a>)</span>. With combinatorial barcoding, however, the efficiency is decreased. Compared to smFISH, MERFISH version 2 with HD4 code has about 95% detection efficiency on 130 genes and 92 probes per gene, although the efficiency dropped to ~25% with the HD2 code that can encode nearly 1000 genes but can only identify but not correct errors <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>; Foreman and Wollman <a href="#ref-Foreman2019" role="doc-biblioref">2019</a>)</span>. When scaled to 10,050 genes, MERFISH has around 79% detection efficiency <span class="citation">(C. Xia, Babcock, et al. <a href="#ref-Xia2019c" role="doc-biblioref">2019</a>)</span>. As for HCR-seqFISH, the efficiency is around 84% <span class="citation">(Shah et al. <a href="#ref-Shah2016" role="doc-biblioref">2016</a>)</span>, and for seqFISH+, around 49% <span class="citation">(Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span>. Nevertheless, this is much better than the efficiency of ST, which is around 6.9% compared to smFISH <span class="citation">(Ståhl et al. <a href="#ref-Stahl2016a" role="doc-biblioref">2016</a>)</span>. To put the 6.9% in context, from ERCC spike ins and in some cases comparison to smFISH, scRNA-seq methods such as Drop-seq, 10X, inDrop, CEL-seq, and CEL-seq2 have capture efficiency of between 3% and 25% <span class="citation">(Macosko et al. <a href="#ref-Macosko2015" role="doc-biblioref">2015</a>; Zheng et al. <a href="#ref-Zheng2017" role="doc-biblioref">2017</a>; Klein et al. <a href="#ref-Klein2015" role="doc-biblioref">2015</a>; Hashimshony et al. <a href="#ref-Hashimshony2016" role="doc-biblioref">2016</a>; Grün, Kester, and Oudenaarden <a href="#ref-Grun2014" role="doc-biblioref">2014</a>)</span>. Thus smFISH based spatial transcriptomics methods can be much more efficient than scRNA-seq, though efficiency of RCA based smFISH compared to regular smFISH has not been reported.</p>
<p>Second, since individual transcripts are imaged and counted, smFISH based methods are highly quantitative and records subcellular localization of transcripts. While most smFISH based spatial transcriptomics studies analyze data at the cellular gene count level, not using subcellular transcript localization, cells have been shown to show great variation in subcellular localization of transcripts of the same set of genes and a number of “archetypal” patterns have been described <span class="citation">(Samacoits et al. <a href="#ref-Samacoits2018a" role="doc-biblioref">2018</a>; Stoeger et al. <a href="#ref-Stoeger2015" role="doc-biblioref">2015</a>; Cabili et al. <a href="#ref-Cabili2015" role="doc-biblioref">2015</a>)</span>.</p>
<p>The following disadvantages may explain why smFISH based spatial transcriptomics has not been widely used on large number of cells and genes (Figure <a href="current-techs.html#fig:smfish-lm-gene">5.19</a>, Figure <a href="current-techs.html#fig:smfish-lm-cell">5.21</a>), and why MERFISH is the most used technique (Figure <a href="current-techs.html#fig:smfish-trend">5.22</a>). First, multiple rounds of hybridization and high magnification mean that data collection is time consuming. MERFISH version 2 greatly sped up imaging, as version 1 requires higher magnification and needs to photobleach fields of view (FOV) one at a time; one FOV in version 1 is 40 <span class="math inline">\(\mu\)</span>m <span class="math inline">\(\times\)</span> 40 <span class="math inline">\(\mu\)</span>m, while one FOV in version 2 is 223 <span class="math inline">\(\mu\)</span>m <span class="math inline">\(\times\)</span> 223 <span class="math inline">\(\mu\)</span>m. Version 2 also cut imaging time in half by using 2 colors, targeting 2 bits per round. This way, for 130 genes and 40,000 cells, MERFISH took about 18 hours <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>)</span>, while HCR-seqFISH would take days because of overnight hybridization after probes are stripped for each round of hybridization although the seqFISH barcode is much shorter. When scaled to 10,000 genes, MERFISH takes 23 rounds of hybridization <span class="citation">(C. Xia, Fan, et al. <a href="#ref-Xia2019a" role="doc-biblioref">2019</a>)</span>, while seqFISH+ takes 80 rounds <span class="citation">(Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>)</span>, although because ExM was used for MERFISH in this case to reduce optical crowding, expanding the area to be images ~4 folds, the actual imaging time of ExM-MERFISH and seqFISH+ here may have been comparable. Perhaps MERFISH has been scaled to larger number of cells and used in more studies beyond the institution of origin (Figure <a href="current-techs.html#fig:smfish-map">5.24</a>) because of the higher detection efficiency and shorter imaging time.</p>
<div class="figure" style="text-align: center"><span id="fig:smfish-langs"></span>
<img src="05-current-techs_files/figure-html/smfish-langs-1.png" alt="Number of publications using smFISH based techniques that used each of the 5 most common programming languages. Each icon stands for 2 publications." width="576" />
<p class="caption">
Figure 5.25: Number of publications using smFISH based techniques that used each of the 5 most common programming languages. Each icon stands for 2 publications.
</p>
</div>
<p>Second, with increasing area of tissue and number of genes covered, smFISH based spatial transcriptomics generates terabytes of images – for each FOV, there is an image for each channel, z-plane, and round of hybridization. Images from the MERFISH dataset of 40,000 cells and 130 genes took 2 to 3 days to process on a multi-core server, although the number of cores was not stated <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>)</span>. In contrast, it takes hours, or even just minutes, to process the fastq files of a scRNA-seq dataset to get the gene count matrix <span class="citation">(Melsted et al. <a href="#ref-Melsted2021" role="doc-biblioref">2021</a>)</span>, nor do the fastq files take up so much disk space. Until 2019, software to process such images and to decode the combinatorial barcodes was typically written in the proprietary programming language MATLAB (Figure <a href="current-techs.html#fig:smfish-langs">5.25</a>), and poorly documented, so it was difficult for people outside the lab of origin to use.</p>
<p>More recently, Python is replacing MATLAB as the programming language of choice to write such image processing software. The Chan Zuckerberg Initiative developed <a href="https://spacetx-starfish.readthedocs.io/en/stable/">starfish</a> in Python as a unified framework to process smFISH based spatial transcriptomics data <span class="citation">(Axelrod et al., <a href="#ref-starfish" role="doc-biblioref">n.d.</a>)</span>. However, <code>starfish</code> is still immature and not sufficiently efficient, nor is the functionality to integrate multiple FOVs developed. Perhaps because of this, <code>starfish</code> has not been widely used and image processing pipelines specific to each technology have been developed instead, such as MERlin for MERFISH <span class="citation">(C. Xia, Fan, et al. <a href="#ref-Xia2019a" role="doc-biblioref">2019</a>)</span> and IRIS for ISS <span class="citation">(Zhou et al. <a href="#ref-Zhou2020" role="doc-biblioref">2020</a>)</span>. In contrast, for scRNA-seq, there are popular data processing tools that apply across technologies, such as STAR (wrapped by Cell Ranger) <span class="citation">(Dobin et al. <a href="#ref-Dobin2012" role="doc-biblioref">2012</a>)</span>, alevin <span class="citation">(Srivastava et al. <a href="#ref-Srivastava2019" role="doc-biblioref">2019</a>)</span>, and kallisto <span class="citation">(Melsted et al. <a href="#ref-Melsted2021" role="doc-biblioref">2021</a>)</span>. Furthermore, even with an open source and interoperable image processing pipeline, cell segmentation, which is essential to obtaining the gene count matrix commonly used in data analysis, is challenging.</p>
<p>Third, custom flow cells have been used for the numerous rounds of hybridization <span class="citation">(Eng et al. <a href="#ref-Eng2019a" role="doc-biblioref">2019</a>; Moffitt et al. <a href="#ref-Moffitt2016a" role="doc-biblioref">2016</a>; Codeluppi et al. <a href="#ref-Codeluppi2018" role="doc-biblioref">2018</a>)</span>. These custom flow cell and pump systems are not commercially available and need to be built by any lab that wishes to adopt the smFISH based technologies. To the best of our knowledge, there are no core facilities that perform smFISH based spatial transcriptomics. Thus for the user, adopting an smFISH based spatial transcriptomics technique means not only learning a new syntax to process images, made difficult in some cases by the cost of MATLAB and lack of documentation, but also setting up a complex custom flow cell system, which may not be feasible with microscopy cores. Finally, smFISH based techniques require a pre-defined list of genes and probes, so unlike in RNA-seq, novel transcripts would be missed.</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:table2">Table 5.1: </span>Pros and cons of smFISH based techniques.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Technique
</th>
<th style="text-align:left;">
Pro
</th>
<th style="text-align:left;">
Con
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6em; ">
HCR-seqFISH
</td>
<td style="text-align:left;width: 6em; ">
Relatively high efficiency (84%), fewer rounds of hybridization, error correction
</td>
<td style="text-align:left;width: 6em; ">
Lower efficiency than MERFISH, time consuming to re-hybridize probes to target after stripping
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
seqFISH+
</td>
<td style="text-align:left;width: 6em; ">
Avoids optical crowding, scalable
</td>
<td style="text-align:left;width: 6em; ">
Lower efficiency (49%), numerous rounds of hybridization
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
MERFISH
</td>
<td style="text-align:left;width: 6em; ">
High efficiency (95%) with HD4 code, error correction, version 2 relatively fast, scalable
</td>
<td style="text-align:left;width: 6em; ">
Numerous rounds of hybridization, numerous probes requiring long transcripts though this is resolved by bDNA signal amplification
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
ExM-MERFISH
</td>
<td style="text-align:left;width: 6em; ">
Avoids optical crowding, clears tissue
</td>
<td style="text-align:left;width: 6em; ">
Each FOV contains less of the original tissue
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
HybISS
</td>
<td style="text-align:left;width: 6em; ">
Only 5 probes per gene, applicable to isoform specific exons, padlock probe reduces background, lower magnification when imaging (20x and 40x, while MERFISH uses 60x), can discern SNPs
</td>
<td style="text-align:left;width: 6em; ">
Error correction not reported, amplicon takes up space and might drift away
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
SCRINSHOT
</td>
<td style="text-align:left;width: 6em; ">
Similar to HybISS, but designed for FFPE tissue
</td>
<td style="text-align:left;width: 6em; ">
Same as HybISS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
HybRISS
</td>
<td style="text-align:left;width: 6em; ">
Avoids inefficiency of reverse transcription, better signal to noise ratio and more transcripts detected then HybISS.
</td>
<td style="text-align:left;width: 6em; ">
Padlock probe sequences are proprietary to CARTANA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6em; ">
bDNA-smFISH
</td>
<td style="text-align:left;width: 6em; ">
Commercial RNAscope kit, reduces background and amplifies signal, amplified moiety does not grow indefinitely
</td>
<td style="text-align:left;width: 6em; ">
Except for bDNA-MERFISH, it has not been used in a highly multiplexed setting
</td>
</tr>
</tbody>
</table>
<p>So far we have reviewed studies that showcase new techniques and technical improvements such as signal amplification and resolving optical crowding. Some smFISH based techniques have been used in studies that focus on biological problems rather than new techniques. HCR-seqFISH has been used twice in biological studies, in chicken neural tube (35 genes) <span class="citation">(Lignell et al. <a href="#ref-Lignell2017" role="doc-biblioref">2017</a>)</span> and mouse T cell precursors (65 genes) <span class="citation">(Zhou et al. <a href="#ref-Zhou2019" role="doc-biblioref">2019</a>)</span> though both were conducted within Caltech, the institution of origin. Moreover, spatial location of cells is not necessarily a reason to use HCR-seqFISH; Zhou et al. used HCR-seqFISH because of the high detection efficiency compared to scRNA-seq in dissociated FACS sorted T cell progenitors, so when spatial information is already lost. More recently, pseudocolor seqFISH was used in a mouse embryo atlas at University of Cambridge (though Long Cai is still a coauthor), finally moving beyond the stage of testing into new biological research <span class="citation">(Lohoff et al. <a href="#ref-Lohoff2020" role="doc-biblioref">2020</a>)</span>. Combinatorial barcoding has also been used to profile bacterial species in the microbiome by targeting rRNAs, though this does not profile the transcriptome, nor is it single molecular <span class="citation">(Shi et al. <a href="#ref-Shi2020" role="doc-biblioref">2020</a>; Cao et al. <a href="#ref-Cao2021" role="doc-biblioref">2021</a>)</span>. For spatial transcriptome in bacteria, a new version of seqFISH, par-seqFISH, was developed to profile 105 genes in the biofilm bacterium <em>Pseudomonas aeruginosa</em> <span class="citation">(Dar et al. <a href="#ref-Dar2021" role="doc-biblioref">2021</a>)</span>. This may open the way to spatial transcriptomics in not only biofilms, but in the microbiome in general.</p>
<p>MERFISH has been used more broadly in biological studies. Within Harvard, the institution of origin, MERFISH has been used to create atlases of the hypothalamic preoptic region (155 genes) <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2018" role="doc-biblioref">2018</a>)</span> and the primary motor cortex (MOp) (258 genes) <span class="citation">(Zhang et al. <a href="#ref-Zhang2020" role="doc-biblioref">2020</a>)</span> in mice, and adapted to stain for chromatin conformation and transcription foci (introns) <span class="citation">(Su and Song <a href="#ref-Su2020" role="doc-biblioref">2020</a>)</span>. Outside Harvard (Figure <a href="current-techs.html#fig:smfish-map">5.24</a>), MERFISH has been used to study how gene expression variability relates to cell state in cell culture <span class="citation">(Foreman and Wollman <a href="#ref-Foreman2019" role="doc-biblioref">2019</a>)</span> and used in conjunction with smFISH based chromatin tracing to study the relationships between chromatin compartmentalization and gene expression <span class="citation">(M. Liu et al. <a href="#ref-Liu2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>After its inception, HybISS became part of a single cell atlas of the developing mouse nervous system <span class="citation">(Manno et al. <a href="#ref-Manno2020" role="doc-biblioref">2020</a>)</span>. This atlas is mostly scRNA-seq data, but 119 genes were stained with HybISS to validate secondary organizers discovered via scRNA-seq.</p>
</div>
</div>
<div id="iss" class="section level2">
<h2><span class="header-section-number">5.3</span> In situ sequencing</h2>
<p>In contrast to smFISH based techniques, techniques reviewed in this section determine the sequences of the target transcript or the gene specific barcode by <em>in situ</em> sequencing by ligation (SBL) or sequencing by synthesis (SBS) to distinguish between transcripts of different genes. This section reviews 3 <em>in situ</em> SBL strategies, SOLiD, cPAL, and SEDAL, and the spatial transcriptomics techniques using them.</p>
<p>SBL relies on the specificity of the DNA ligase, so ligation only occurs when both sequences to ligate match the template in the vicinity of the site of ligation. Prior to SBL, this specificity was used to detect SNVs that would otherwise be missed as ISH probes can tolerate some mismatches. A technique using ligation of two oligonucleotides to detect SNVs was introduced in 1988 by Ulf Landegren <span class="citation">(Landegren et al. <a href="#ref-Landegren1988" role="doc-biblioref">1988</a>)</span>, laying the foundation of SBL (Figure <a href="current.html#fig:tl3">4.3</a>). The padlock probe came in the same tradition of SNV detection, and Mats Nilsson worked with Landergren when creating the padlock probe <span class="citation">(Nilsson et al. <a href="#ref-Nilsson1994" role="doc-biblioref">1994</a>)</span>.</p>
<p>Almost all spatial transcriptomics techniques based on SBL require <em>in situ</em> reverse transcription of the mRNAs as ligation with RNA as template is inefficient. As already mentioned in Section <a href="current-techs.html#lcm-amplification">5.1.1.2</a>, IVT amplification of transcripts from single cells for expression profiling originated in the Eberwine group <span class="citation">(Van Gelder et al. <a href="#ref-VanGelder1990" role="doc-biblioref">1990</a>; Eberwine et al. <a href="#ref-Eberwine1992" role="doc-biblioref">1992</a>)</span>, where rather than LCM, the cDNAs from the single cells were reverse transcribed <em>in situ</em> during electrophysiological recording before the cellular content was aspirated for IVT amplification. This was built upon the <em>in situ</em> reverse transcription technique from the Eberwine group in 1988 <span class="citation">(Tecott, Barchas, and Eberwine <a href="#ref-Tecott1988" role="doc-biblioref">1988</a>)</span>, where the cDNA of proopiomelanocortin (POMC) was radiolabeled so the spatial distribution of the mRNA was visualized on an autoradiograph. This made most <em>in situ</em> SBL techniques possible, which instead of radioactivity, use gene barcodes to locate the transcripts in a multiplexed and safer way.</p>
<div id="solid" class="section level3">
<h3><span class="header-section-number">5.3.1</span> SOLiD and FISSEQ</h3>
<p>The earliest proposal of SBL we are able to locate is a patent filed in 1995 describing a method similar to sequencing by oligo ligation detection (SOLiD). An initiator oligonucleotide hybridizes to the template to be sequenced, and is extended by ligation to a 9-mer probe with a label such as a fluorophore that indicates one or two nucleotides of the probe <span class="citation">(Macevicz <a href="#ref-Macevicz1995" role="doc-biblioref">1995</a>)</span>. The probe has a blocking moiety so only one probe is ligated in each cycle. Then the blocking moiety is removed so the initiator can be further extended by ligation in the next cycle. As mismatches in the probe inhibit ligation, the nucleotide of interest in the probe can be read off from the label after probes that are not ligated are removed. This can determine every 9th nucleotide in the template, and with 9 different initiators, each out of phase by one nucleotide, the sequence of the entire template can be determined. However, this method existed only on paper, while since 2006, Applied Biosystems (Applera) seemed to have developed SOLiD independently from that patent after acquiring Agencourt, which developed a sequencing by ligation method that would be the foundation of SOLiD <span class="citation">(Alsup <a href="#ref-Alsup2009" role="doc-biblioref">2009</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:fisseq"></span>
<img src="fisseq.png" alt="Schematic of RCA in FISSEQ." width="100%" />
<p class="caption">
Figure 5.26: Schematic of RCA in FISSEQ.
</p>
</div>
<p>In 2014, single cell resolution and transcriptome wide spatial transcriptomics was far out of reach (Figure <a href="current-techs.html#fig:records-gene">5.17</a>). An attempt to reach this goal was fluorescent in situ sequencing (FISSEQ) <span class="citation">(Lee et al. <a href="#ref-Lee2014" role="doc-biblioref">2014</a>)</span>. A universal adapter and random hexamer reverse transcription (RT) primer was hybridized to the mRNAs to reverse transcribe them into cDNA (Figure <a href="current-techs.html#fig:fisseq">5.26</a>). Then the cDNA, now with the adaptor on the 5’ end, is circularized, and amplified with RCA with a primer complementary to the adaptor. Then again, with sequencing primers receding into the adaptor, SOLiD is used to sequence the cDNA amplicons in situ.</p>
<div class="figure" style="text-align: center"><span id="fig:solid"></span>
<img src="solid.png" alt="Schematic of SOLiD sequencing, determining the sequence GATTACA. The rows are arranged in the order of 5' to 3' positions of the first fluorescent probe, but the actual hybridization and ligation can take a different order. As part of the constant region, the 'A' highlighted in red is known." width="100%" />
<p class="caption">
Figure 5.27: Schematic of SOLiD sequencing, determining the sequence GATTACA. The rows are arranged in the order of 5’ to 3’ positions of the first fluorescent probe, but the actual hybridization and ligation can take a different order. As part of the constant region, the ‘A’ highlighted in red is known.
</p>
</div>
<p>In SOLiD, color of the fluorophore encodes the two 3’-most bases of 8-mer probes with other bases degenerate (Figure <a href="current-techs.html#fig:solid">5.27</a>). Once a probe perfectly matching the target right after the primer, the probe is ligated to the primer and the fluorescent signal is recorded. Then the fluorophore and the nearest 3 bases of the probe are cleaved off. In the next cycle, a new matching probe is ligated to the now extended primer. This is continued until the end of the target, for 7 cycles per primer in the case of FISSEQ <span class="citation">(Lee et al. <a href="#ref-Lee2015" role="doc-biblioref">2015</a>)</span>. For the first 7 cycles, the primer matches the adaptor (N). Then the primer N, extended for 7 cycles is stripped, and a new primer receding one nucleotide to the 5’ end of the adaptor (N-1) is added in cycle 8. Again 7 cycles of ligation are performed and the extended primer N-1 is stripped after cycle 14 to make room for N-2. For N-2, N-3, and N-4, a bridge oligo is used so the target with unknown sequence, rather than the adaptor with known sequence, is interrogated by the probes. With N through N-4, the entire target is covered. With the fluorescent signals recorded from the rounds of ligation, and the knowledge of the last nucleotide of the adaptor interrogated by the first ligation to primer N-1, the sequence of the target can be determined. Figure <a href="current-techs.html#fig:solid">5.27</a> shows how SOLiD determines the sequence “GATTACA”. As already mentioned in the smFISH section, with increasing number of genes profiled, optical crowding is increasingly a problem. To mitigate optical crowding, the primer N can have one or more degenerate bases at the 5’ end reaching into the target; with one degenerate base, only 1/4 of the amplicons are sequenced. With two bases, this would be 1/16. This is repeated to cover all transcripts, but increases imaging time.</p>
<p>While FISSEQ may seem a promising approach to reach the goal of single cell resolution and transcriptome wide spatial transcriptomics that unlike smFISH based techniques, is not limited by pre-defined gene panels, it has been largely dormant since its inception due to the following disadvantages. First, SOLiD has fallen out of favor because of limited read length when used in situ (5-30 nt), propagation of errors from previous cycles <span class="citation">(Alon et al. <a href="#ref-Alon2020" role="doc-biblioref">2020</a>)</span>, and difficulty in sequencing panlindromic sequences <span class="citation">(Giani et al. <a href="#ref-Giani2020" role="doc-biblioref">2020</a>)</span>. SOLiD was chosen for FISSEQ because it works well at room temperature; though SBS supports longer read lengths, it requires a heated stage <span class="citation">(Lee et al. <a href="#ref-Lee2015" role="doc-biblioref">2015</a>)</span>. Second, FISSEQ is extremely inefficient, over 20 times less sensitive than scRNA-seq and two orders of magnitude less sensitive than 2013 Nilsson ISS (discussed later in this section) <span class="citation">(Lee et al. <a href="#ref-Lee2015" role="doc-biblioref">2015</a>)</span>, in part because of inefficiency of random RT priming <span class="citation">(Lee et al. <a href="#ref-Lee2014" role="doc-biblioref">2014</a>)</span> and tight packing of amplicons <span class="citation">(Alon et al. <a href="#ref-Alon2020" role="doc-biblioref">2020</a>)</span>. Furthermore, as ribosomal RNA (rRNA) is not depleted, ~40-80% of FISSEQ reads are rRNA <span class="citation">(Lee et al. <a href="#ref-Lee2014" role="doc-biblioref">2014</a>, <a href="#ref-Lee2015" role="doc-biblioref">2015</a>)</span>. Third, highly abundant genes involved in translation and splicing is depleted in FISSEQ compared to bulk RNA-seq <span class="citation">(Lee et al. <a href="#ref-Lee2014" role="doc-biblioref">2014</a>)</span>. Finally, FISSEQ imaging is time consuming, taking 2 to 3 weeks if performed manually <span class="citation">(Lee et al. <a href="#ref-Lee2015" role="doc-biblioref">2015</a>)</span>.</p>
<p>With expansion microscopy, the idea of FISSEQ was revived in ExSeq <span class="citation">(Alon et al. <a href="#ref-Alon2020" role="doc-biblioref">2020</a>)</span>. Just like in ExM-MERFISH, transcripts are incorporated into a polyelectrolyte gel, which is expanded, so the amplicons are no longer so tightly packed. This eliminated the depletion of highly abundant genes compared to bulk RNA-seq, and the detection efficiency and proportion of rRNA reads of ExSeq seem on par with randomly primed bulk RNA-seq of adjacent sections. In addition to SOLiD sequencing as in FISSEQ, the amplicons are also sequenced ex situ with Illumina SBS. The in situ sequences are matched to ex situ sequences and only unique matches are kept, to more effectively align amplicons to the genome and to localize mRNA sequence variations such as alternative splicing that are more difficult to detect with SOLiD’s short read length. There is also a targeted version of ExSeq, in which padlock probes with gene specific barcodes are RCA amplified and the barcodes are sequenced in situ by either SOLiD or Illumina SBS, profiling up to 297 genes; the detection efficiency is 62% compared to smFISH (i.e. for select genes in the same cell types, the number of transcripts detected is about 62% compared to smFISH), which is high compared to ~5% for 2013 Nilsson ISS but lower than that of MERFISH (HD4) and HCR-seqFISH <span class="citation">(Alon et al. <a href="#ref-Alon2020" role="doc-biblioref">2020</a>; Lein, Borm, and Linnarsson <a href="#ref-Lein2017" role="doc-biblioref">2017</a>)</span>. Eight probes were designed for each gene, and the transcripts must be at least 960 nt long, shorter than required by MERFISH (without bDNA) and seqFISH variants. To our best knowledge, ExSeq has yet been used to collect new datasets after its inception.</p>
</div>
<div id="cpal" class="section level3">
<h3><span class="header-section-number">5.3.2</span> cPAL and ISS</h3>
<p>An alternative SBL scheme is combinatorial probe anchor ligation (cPAL), which to our best knowledge, was first demonstrated in 2005 <span class="citation">(Shendure <a href="#ref-Shendure2005" role="doc-biblioref">2005</a>)</span>. In cPAL, an anchor primer is hybridized to a constant region immediately adjacent to the target. T4 DNA ligase requires matching base pairing up to 6 bases from the ligation junction when ligating from 5’ to 3’ and 7 bases when ligating from 3’ to 5’. The first base of the target 5’ to the constant region is interrogated by a 9-mer probe whose 5’ most base is represented by the color of a fluorophore and ligated to the primer if a perfect match is present (Figure <a href="current-techs.html#fig:cpal">5.28</a>). Then the ligated construct is stripped and a new primer is hybridized to the constant region. The second base is interrogated by a 9-mer probe whose second 5’ most base is represented by the fluorophore. This can carry on until the 6th base on the 5’ direction. When the constant region is 5’ to the target, bases 3’ to the constant region are interrogated in a similar fashion. With constant regions flanking a target so primers bind in both direction, a 13 nt target can be sequenced this way, and the read length can be somewhat increased by adding degenerate bases to the anchor primer extending into the target <span class="citation">(Drmanac et al. <a href="#ref-Drmanac2010" role="doc-biblioref">2010</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:cpal"></span>
<img src="cpal.png" alt="Schematic of cPAL as used in ISS." width="100%" />
<p class="caption">
Figure 5.28: Schematic of cPAL as used in ISS.
</p>
</div>
<p>The only in situ sequencing method that was reused after its inception was originally demonstrated in 2013 by Mats Nilsson’s group <span class="citation">(Ke et al. <a href="#ref-Ke2013" role="doc-biblioref">2013</a>)</span>, which we call ISS here (Figure <a href="current.html#fig:n-insts">4.7</a>). First, padlock probes are hybridized to in situ reverse transcribed cDNAs and RCA amplified (Figure <a href="current-techs.html#fig:rca">5.14</a>). The padlock probe can carry a gene specific 4 nt barcode (barcode version), or leave a 4 nt gap between the ends of the probe after it’s hybridized to the cDNA to be filled when the probe is circularized (gap filling version). Then the barcode or the filled gap is sequenced in situ, with an anchor primer binding 3’ to the target, with cPAL. Because of limited read length of cPAL, short sequences uniquely identify each gene and isoform for the gap filling approach becomes difficult to find with increasing number of genes and isoforms. In contrast, a barcode with length <span class="math inline">\(n\)</span> can encode <span class="math inline">\(4^n\)</span> genes and isoforms. As a result, the barcode approach was repeatedly used after the inception of ISS and was commercialized by CARTANA, which was recently acquired by 10X Genomics.</p>
<p>The barcode approach was initially used to profile 39 genes <span class="citation">(Ke et al. <a href="#ref-Ke2013" role="doc-biblioref">2013</a>)</span>, but has been used to profile up to 222 genes in human brains affected by Alzheimer’s disease <span class="citation">(Chen et al. <a href="#ref-Chen2020" role="doc-biblioref">2020</a>)</span>. Although, as already mentioned, ISS has much lower detection efficiency than smFISH based methods, because of RCA and this low detection efficiency, the density of imaged amplicons is lower, allowing for imaging at lower resolution (20x; MERFISH uses 60x) and thus facilitating profiling large areas of tissues such as whole mouse brain coronal sections <span class="citation">(Qian et al. <a href="#ref-Qian2020" role="doc-biblioref">2020</a>; Partel et al. <a href="#ref-Partel2019" role="doc-biblioref">2019</a>)</span>. ISS has also been used in conjunction with spatial transcriptomics techniques that are transcriptome wide but lack single cell resolution, such as ST. Panels of usually fewer than 100 genes of interest are selected from ST and scRNA-seq data, to be profiled with ISS for more in depth characterization of these genes <span class="citation">(Chen et al. <a href="#ref-Chen2020" role="doc-biblioref">2020</a>; Asp et al. <a href="#ref-Asp2019" role="doc-biblioref">2019</a>)</span>. In addition, because of the specificity conferred by the padlock probe and the small number of probes required per gene (usually 5 per gene but can be fewer), ISS has been used to quantify isoforms from isoform specific exons and exon-exon junctions <span class="citation">(Lebrigand et al. <a href="#ref-Lebrigand2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>The number of genes that can be profiled by ISS is limited by the barcode length. Just like in seqFISH, only a small subset of all possible barcodes given a barcode length is used for error correction. As a result, to profile the entire transcriptome of over 20,000 genes, the barcode should be at least 8 nt long (65,536 barcodes), while in one direction, cPAL can only sequence 6 or 7 nt and degenerate bases. It is possible in theory to lengthen the barcode to up to 13 nt by sequencing from both ends of the barcode as in the original 2005 method <span class="citation">(Shendure <a href="#ref-Shendure2005" role="doc-biblioref">2005</a>)</span>. However, with increasing number of transcripts comes the problem of optical crowding, which is exacerbated by the physical size of the RCA amplicon. Perhaps ExM can be used here to mitigate optical crowding just like in ExSeq. To address the limitation in barcode length, HybISS, i.e. hybridization-based in situ sequencing, was devised <span class="citation">(Gyllborg et al. <a href="#ref-Gyllborg2020" role="doc-biblioref">2020</a>)</span> so the now seqFISH-like barcode can be arbitrarily lengthened by increasing the number of rounds of hybridization. HybISS has already been reviewed in Section<a href="current-techs.html#RCA">5.2.2.2</a>; despite the “ISS” in its name, HybISS is classified as smFISH based because it does not involve SBL or SBS. HybISS also has up to 5 fold higher signal to noise ratio than ISS, and has somewhat higher detection efficiency than ISS though the improvement is less than 2 fold <span class="citation">(Gyllborg et al. <a href="#ref-Gyllborg2020" role="doc-biblioref">2020</a>)</span>. Comparison between HybISS and smFISH has not been reported. Nevertheless, HybISS has not yet been scaled to more than 120 genes and ExM may still be needed for transcriptome wide profiling.</p>
</div>
<div id="sedal" class="section level3">
<h3><span class="header-section-number">5.3.3</span> SEDAL and STARmap</h3>
<p>Both SOLiD and cPAL have some drawbacks. As the gene specific barcode does not have to be long to encode all genes in the genome, when the barcode is used, limits in read length is not a major limitation. Because one color encodes two bases, SOLiD is very accurate <span class="citation">(Liu et al. <a href="#ref-Liu2012" role="doc-biblioref">2012</a>)</span>, but error in one cycle propagates to later cycles. At least in the mouse brain, SOLiD also has high background <span class="citation">(Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>. In contrast, cPAL does not have an inbuilt error rejection mechanism; the barcode must be elongated to allow for error correction, much like in the error correction scheme of seqFISH. Furthermore, in ISS, the mRNA is first reverse transcribed into cDNA because ligation of the padlock probe is inefficient when the template is RNA <span class="citation">(Larsson et al. <a href="#ref-Larsson2010" role="doc-biblioref">2010</a>)</span>. However, the efficiency of RT depends on the gene of interest and the variability of RT efficiency depends on RNA concentration <span class="citation">(Schwaber, Andersen, and Nielsen <a href="#ref-Schwaber2019" role="doc-biblioref">2019</a>; Bustin et al. <a href="#ref-Bustin2015" role="doc-biblioref">2015</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:sedal"></span>
<img src="sedal.png" alt="Schematic of RCA of SNAIL probe and SEDAL. Also showing error propagation and identification of 2 base encoding. As part of the constant region, the 'G' highlighted in red is known." width="100%" />
<p class="caption">
Figure 5.29: Schematic of RCA of SNAIL probe and SEDAL. Also showing error propagation and identification of 2 base encoding. As part of the constant region, the ‘G’ highlighted in red is known.
</p>
</div>
<p>A new method of in situ sequencing, namely sequencing with error-reduction by dynamic annealing and ligation (SEDAL) in spatially-resolved transcript amplicon readout mapping (STARmap), was devised to address these shortcomings <span class="citation">(Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span>. In STARmap, the specific amplification of nucleic acids via intramolecular ligation (SNAIL) probe is a derivative of the original padlock probe that avoids RT altogether. A primer partially hybridizes to the mRNA, and partially to the padlock probe (Figure <a href="current-techs.html#fig:sedal">5.29</a>). The padlock probe carrying a 5 nt gene specific barcode hybridizes to the mRNA adjacent to the primer, but both ends of the padlock probe hybridize to the primer instead, so when the ends are ligated together, the template is DNA rather than RNA, thus avoiding both RT and inefficiency of ligation with RNA template, and then the primer is used to initiate RCA. As both the primer and the padlock probe must match the mRNA template for RCA to occur, SNAIL probes are specific and background of non-specific binding is eliminated. To reduce background autofluorescence and prevent the RCA amplicons from moving, the amplicons are crosslinked into a hydrogel and the tissue is cleared of proteins and lipids.</p>
<p>Then SEDAL is used to sequence the gene specific barcodes. The sequences flanking the gene barcode are known. In the first round an anchor or reading probe binds to the constant region 5’ to the barcode, one base away from the barcode (Figure <a href="current-techs.html#fig:sedal">5.29</a>). The decoding probes are 8-mers labeled with a fluorophore at the 5’ end whose color represents the 2 nucleotides at the 3’ end that interrogates the barcode; the other bases are degenerate. If the decoding probe matches the barcode, then it is ligated to the reading probe and the fluorescent signal is recorded. In the first round, the decoding probe interrogates the last base of the constant region and the first base of the barcode, as the last base of the constant region is necessary to decode the sequence of colors. Then the reading and decoding probes are stripped. In the second round, the reading probe stops right where the barcode starts. In the third round, the reading probe has a degenerate base extending into the barcode. Reading probes of the following rounds extend further into the barcode with degenerate bases. In the last round, the decoding probe interrogates the last base of the barcode and the first base of the following constant region. Like in SOLiD, with 2 base encoding, an error in a previous round propagates into later rounds; with propagation, when there is an error when decoding, then the first base of the constant region after the barcode would be incorrectly decoded, so the error is identified and rejected. Comparison of detection efficiency of STARmap with that of smFISH has not been reported; the efficiency is reported to be somewhat better, at least not worse, than that of scRNA-seq, suggesting that STARmap is perhaps more efficient than ISS, but most likely much less efficient than MERFISH (HD4) and seqFISH.</p>
</div>
<div id="sbs" class="section level3">
<h3><span class="header-section-number">5.3.4</span> Sequencing by synthesis</h3>
<p>While most <em>in situ</em> sequencing techniques use SBL, some use SBS, indeed with a heated stage to perform SBS in situ. Because Illumina SBS is much more well-known and widely used than SBL for NGS, we will not recap it here. SBS has been tried to sequence DNA barcodes of antibodies in highly multiplexed immunofluorescence <span class="citation">(Kohman and Church <a href="#ref-Kohman2020" role="doc-biblioref">2020</a>)</span>. BARseq <span class="citation">(Chen et al. <a href="#ref-Chen2019" role="doc-biblioref">2019</a>)</span>, a method to trace neuron projections is also based on SBS. In BARseq, the gap filling version of ISS (Section @ref{cpal}) is used and the filled gap that is the projection tracing barcode is sequenced with Illumina SBS chemistry. BARseq has also been adapted to profile endogenous transcripts (up to 79 genes as of writing) and image projection barcodes in the same neurons (BARseq2) <span class="citation">(Sun et al. <a href="#ref-Sun2020a" role="doc-biblioref">2020</a>; S. Chen et al. <a href="#ref-Chen2021" role="doc-biblioref">2021</a>)</span>; gene expression and projection can be correlated in some though not all cells. For endogenous transcripts, the mRNA is first reverse transcribed, and the barcode version of ISS (Section @ref{cpal}) is used to amplify the barcodes (in the padlock probe but not the cDNA) with RCA, which are then sequenced <em>in situ</em> with SBS. For transcripts, BARseq2 detects slightly more copies of mRNAs than 10X v3 scRNA-seq for the same gene in the same tissue.</p>
</div>
</div>
<div id="array" class="section level2">
<h2><span class="header-section-number">5.4</span> In situ array capture</h2>
<p>This section reviews techniques that capture transcripts from a permeabilized tissue section on a spatially organized array for RNA-seq. These techniques are similar to 3’ based scRNA-seq, with amplification and sequencing handle, barcode, UMI, and poly-T to capture polyadenylated transcripts, except that each spot in the array has its own barcode, rather than each droplet. These techniques can be transcriptome wide, but do not have single cell resolution; the resolution is the size and shape of the spots. In ST and Visium, the array is constructed by printing the capture sequences onto commercial microarray slides, so the 5’ end of the sequences are attached to the slide; where each spatial barcode is placed is known. Alternatively, the capture sequences can be attached to beads like in droplet scRNA-seq, as in Slide-seq and HDST. The beads are randomly placed on a slide in a single layer, and the location of barcodes are determined before library preparation when the capture sequences and transcripts are released from the slide.</p>
<div class="figure" style="text-align: center"><span id="fig:array-map"></span>
<img src="05-current-techs_files/figure-html/array-map-1.png" alt="Cities and institutions using ST and Visium. Preprints are included." width="960" />
<p class="caption">
Figure 5.30: Cities and institutions using ST and Visium. Preprints are included.
</p>
</div>
<p>ST <span class="citation">(Ståhl et al. <a href="#ref-Stahl2016a" role="doc-biblioref">2016</a>)</span> and Visium are the most widely used current era technique after LCM (Figure <a href="current.html#fig:n-insts">4.7</a>, Figure <a href="current-techs.html#fig:array-map">5.30</a>). In ST, the printed spots have diameter of 100 <span class="math inline">\(\mu\)</span>m and are 200 <span class="math inline">\(\mu\)</span>m apart from center to center (Figure <a href="current-techs.html#fig:array-dims">5.31</a>). Multiple sections can be mounted to the same slide, separated by a rubber mask. For each section, there are 1007 spots covering an area of 6200 <span class="math inline">\(\times\)</span> 6600 <span class="math inline">\(\mu\)</span>m. The 5’ end of the capture sequence is a linker to be cleaved to release the transcripts, followed by amplification and sequencing handle, an 18 nt spatial barcode, a 9 nt UMI, and poly-T (Figure <a href="current-techs.html#fig:array-umi">5.32</a>). For the genes quantified with smFISH, ST’s detection efficiency is around 6.9% compared to smFISH, within the range of the efficiency of scRNA-seq techniques. Despite the low resolution, ST is popular probably due to transcriptome wide profiling, ease to apply to larger area of tissue, not requiring specialized equipment such as SRM and custom flow cells, commercial kits, possible automation of library preparation <span class="citation">(Jemt et al. <a href="#ref-Jemt2016" role="doc-biblioref">2016</a>)</span>, availability of a documented and open source data preprocessing pipeline called ST Pipeline <span class="citation">(Navarro et al. <a href="#ref-Navarro2017" role="doc-biblioref">2017</a>)</span>, and the extra information from H&amp;E staining before library preparation.</p>
<div class="figure" style="text-align: center"><span id="fig:array-dims"></span>
<img src="fig13a.png" alt="Schematic of spot construction and size of array based techniques." width="50%" />
<p class="caption">
Figure 5.31: Schematic of spot construction and size of array based techniques.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:array-umi"></span>
<img src="fig13b.png" alt="Barcode and UMI structure and lengths of array based techniques." width="50%" />
<p class="caption">
Figure 5.32: Barcode and UMI structure and lengths of array based techniques.
</p>
</div>
<p>After its inception, ST has been used in a wide range of clinical pathological tissues, such as heart after heart failure <span class="citation">(Asp et al. <a href="#ref-Asp2017" role="doc-biblioref">2017</a>)</span>, peritonitis-affected gingival tissue <span class="citation">(Lundmark et al. <a href="#ref-Lundmark2018" role="doc-biblioref">2018</a>)</span>, prostate cancer <span class="citation">(Berglund et al. <a href="#ref-Berglund2018" role="doc-biblioref">2018</a>)</span>, breast cancer <span class="citation">(He et al. <a href="#ref-He2020" role="doc-biblioref">2020</a>)</span>, arthritic joint biopsies <span class="citation">(Carlberg et al. <a href="#ref-Carlberg2019" role="doc-biblioref">2019</a>)</span>, lymph nodes affected by melanoma metastasis <span class="citation">(Thrane et al. <a href="#ref-Thrane2018" role="doc-biblioref">2018</a>)</span>, spinal cords <span class="citation">(Maniatis et al. <a href="#ref-Maniatis2019" role="doc-biblioref">2019</a>)</span> and cerebellums <span class="citation">(Gregory et al. <a href="#ref-Gregory2020" role="doc-biblioref">2020</a>)</span> affected by amyotrophic lateral sclerosis (ALS), and squamous cell carcinoma <span class="citation">(Ji et al. <a href="#ref-Ji2020" role="doc-biblioref">2020</a>)</span>. ST has also been used to construct gene expression atlases of healthy tissues such as the developing human heart <span class="citation">(Asp et al. <a href="#ref-Asp2019" role="doc-biblioref">2019</a>)</span> and the mouse brain <span class="citation">(Ortiz et al. <a href="#ref-Ortiz2020" role="doc-biblioref">2020</a>)</span>. In addition, ST is the only current era technique other than LCM and manual microdissection that has been adapted to plants <span class="citation">(Giacomello et al. <a href="#ref-Giacomello2017" role="doc-biblioref">2017</a>)</span>. Common downstream data analyses include identifying differentially expressed (DE) genes between diseased and healthy regions, gene set enrichment analysis (GSEA) among DE genes, and cell type deconvolution of the spots by integrating ST and scRNA-seq data. Data analysis methods designed specifically for ST or Visium will be reviewed in more detail in Chapter <a href="current-analysis.html#current-analysis">7</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:array-trend"></span>
<img src="05-current-techs_files/figure-html/array-trend-1.png" alt="Number of publications over time, broken down by technique. The facets are ordered by recent usage of the technique. Bin width is 90 days." width="576" />
<p class="caption">
Figure 5.33: Number of publications over time, broken down by technique. The facets are ordered by recent usage of the technique. Bin width is 90 days.
</p>
</div>
<p>Although introduced fairly recently, after 10X Genomics acquired ST in December 2018, the 10X Visium has quickly gained popularity and spread to multiple institutions, and is used by many studies that utilize an array method in late 2020 and 2021 (Figure <a href="current-techs.html#fig:array-map">5.30</a>, Figure <a href="current-techs.html#fig:array-trend">5.33</a>). While usage of ST seems concentrated in Sweden, where ST comes from, usage of Visium is more decentralized (Figure <a href="current-techs.html#fig:array-map">5.30</a>). Visium is similar to ST and shares the advantages of ST, but with higher spatial resolution. The spots are 100 <span class="math inline">\(\mu\)</span>m apart center to center, each with a diameter of 55 <span class="math inline">\(\mu\)</span>m, arranged in a hexagonal configuration (Figure <a href="current-techs.html#fig:array-dims">5.31</a>). After adjusting for spot area, Visium seems to capture somewhat more transcripts and genes compared to ST <span class="citation">(Y. Liu et al. <a href="#ref-Liu2020a" role="doc-biblioref">2020</a>)</span>. In addition, Visium’s growth in popularity may also be due to core facilities at multiple institutions providing Visium services <span class="citation">(“10X VISIUM SPATIAL TRANSCRIPTOMICS” <a href="#ref-illinois" role="doc-biblioref">n.d.</a>; “ADVANCED GENOMICS CORE PRICING” <a href="#ref-umich" role="doc-biblioref">n.d.</a>; “SpaRTAN” <a href="#ref-ncl" role="doc-biblioref">n.d.</a>)</span>. As a new version of ST, Visium was originally designed for fresh frozen OCT embedded tissue and 3’ Illumina sequencing. However, Visium has more recently been adapted to FFPE tissue <span class="citation">(Villacampa et al. <a href="#ref-Villacampa2020" role="doc-biblioref">2020</a>)</span>, as well as to Nanopore long read sequencing to quantify isoforms <span class="citation">(Lebrigand et al. <a href="#ref-Lebrigand2020" role="doc-biblioref">2020</a>; Joglekar et al. <a href="#ref-Joglekar2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>In response to the low resolution of ST, Slide-seq was developed to increase the resolution of array based spatial transcriptomics <span class="citation">(Rodriques et al. <a href="#ref-Rodriques2019" role="doc-biblioref">2019</a>)</span>. Beads like those used in Drop-seq <span class="citation">(Macosko et al. <a href="#ref-Macosko2015" role="doc-biblioref">2015</a>)</span> with diameter 10 <span class="math inline">\(\mu\)</span>m are spread on a slide in a single layer, not necessarily in a regular grid, and bead barcodes are generated with 16 rounds of split pool, each round adding one nucleotide, broken into 2 blocks of 8 nt (2 blocks of 8 and 7 nt in version 2) (Figure <a href="current-techs.html#fig:array-dims">5.31</a>, Figure <a href="current-techs.html#fig:array-umi">5.32</a>). As the location of each barcode is not pre-determined, the slide is imaged and the barcodes are sequenced in situ with SOLiD. Then the OCT frozen tissue section is mounted on the layer of beads on the slide and the beads are removed for library preparation. The first version of Slide-seq is very inefficient; for the genes compared, the Slide-seq only detects 2 to 3 orders of magnitude fewer transcripts per cell than smFISH and about 2.7% compared to Drop-seq <span class="citation">(Rodriques et al. <a href="#ref-Rodriques2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>In the second version of Slide-seq (Slide-seq2) <span class="citation">(Stickels et al. <a href="#ref-Stickels2020" role="doc-biblioref">2020</a>)</span>, the barcodes are sequenced by SEDAL (like in Figure <a href="current-techs.html#fig:sedal">5.29</a>, but with one color per base) rather than SOLiD, which increased the efficiency of spatial mapping of Illumina reads, probably because of error propagation in the 2 base encoding of SOLiD. Moreover, bead synthesis is further optimized and a second strand synthesis step is added to the library preparation to increase the number of cDNAs for PCR amplification. Efficiency is improved in Slide-seq2, which is ~9.3x higher than version 1, about on par with Drop-seq, 1 order of magnitude lower than that of smFISH, and somewhat better than Visium in the dataset chosen. The official software to process the in situ sequencing images is written in MATLAB, which is proprietary. Although the size of the bead is close to the size of a single cell, Slide-seq does not have single cell resolution as one bead can capture transcripts from more than one cells nearby, so cell type deconvolution of beads is still needed. After its inception, Slide-seq2 has been used on mouse and human testes, at the institution of origin <span class="citation">(Chen et al. <a href="#ref-Chen2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>Spatial resolution of array based techniques has been further increased with HDST, with a resolution of 2 <span class="math inline">\(\mu\)</span>m <span class="citation">(Vickovic et al. <a href="#ref-Vickovic2019" role="doc-biblioref">2019</a>)</span>, which is smaller than a single cell. Like in Slide-seq, beads like those used in droplet scRNA-seq are used. The diameter of each bead is 2 <span class="math inline">\(\mu\)</span>m, and hexagonal wells with diameter 2.05 <span class="math inline">\(\mu\)</span>m are carved into a slides so each well contains one bead (Figure <a href="current-techs.html#fig:array-dims">5.31</a>). The spatial barcodes are generated by 3 rounds of split-pool, each round adding 15 nt from the barcode pool (Figure <a href="current-techs.html#fig:array-umi">5.32</a>). The UMI is only 5 nt but such a small area does not contain that many transcripts. As the beads are randomly placed in the wells, the locations of barcodes need to be determined. Four rounds of FISH, with combinations of red, green, and no color, encode each of the 3 barcodes on each bead. Again, HDST was originally designed for fresh frozen OCT embedded tissue rather than FFPE. HDST is very inefficient; for the genes compared, the detection efficiency is only ~1.3% compared to smFISH per bead. To our best knowledge, HDST has not been used for new datasets after its inception.</p>
<p>In response to the low efficiency and complicated procedure to localize barcodes of Slide-seq and HDST, Deterministic Barcoding in Tissue for spatial omics sequencing (DBiT-seq) was developed, with resolution up to 10 <span class="math inline">\(\mu\)</span>m <span class="citation">(Y. Liu et al. <a href="#ref-Liu2020a" role="doc-biblioref">2020</a>)</span>. Let <span class="math inline">\(i, j\)</span> denote the index of channel in each direction. Barcode <span class="math inline">\(A_i\)</span>, attached to poly-T, is flown across the slide in microfluidic channels and RT is performed (Figure <a href="current-techs.html#fig:array-dims">5.31</a>). Then barcode <span class="math inline">\(B_j\)</span>, attached to the UMI, PCR handle, and biotin, is flown across the slide in microfluidic channels perpendicular to those that delivered barcode <span class="math inline">\(A_i\)</span>, and barcode <span class="math inline">\(B_j\)</span> is ligated to barcode <span class="math inline">\(A_i\)</span> and the cDNA (Figure <a href="current-techs.html#fig:array-dims">5.31</a>, Figure <a href="current-techs.html#fig:array-umi">5.32</a>). Then the ligated barcodes and cDNA can be purified by streptavidin-coated magnetic beads. Each microfluidic channel carries a different barcode, so where the channels for barcodes <span class="math inline">\(A_i\)</span> and <span class="math inline">\(B_j\)</span> intersect, an array is created and the location of each spot is encoded by <span class="math inline">\(i, j\)</span>. The resolution is limited by the width of the channels and the spacing between them; widths of 50, 25, and 10 <span class="math inline">\(\mu\)</span>m have been tested. Per unit spot area, DBiT-seq seems to detect at least 3 times more genes and UMIs than ST and Visium and the improvement is even starker at the 10 <span class="math inline">\(\mu\)</span>m resolution. For the genes compared, DBiT-seq’s detection efficiency is ~15.5% of that of smFISH, making it relative more sensitive among the array based methods reviewed here. To our best knowledge, DBiT-seq has not been used for new datasets after its inception.</p>
<div class="figure" style="text-align: center"><span id="fig:spot-diam"></span>
<img src="05-current-techs_files/figure-html/spot-diam-1.png" alt="Record spot diameter of array based methods over time." width="576" />
<p class="caption">
Figure 5.34: Record spot diameter of array based methods over time.
</p>
</div>
<p>The record resolution of array based techniques is ever increasing (Figure <a href="current-techs.html#fig:spot-diam">5.34</a>); sub-micron techniques are appearing in 2021. The record is broken by Stereo-seq in January 2021, reporting a spot diameter of 220 nm although the distance between spots is 500 or 715 nm <span class="citation">(A. Chen et al. <a href="#ref-Chen2021a" role="doc-biblioref">2021</a>)</span>. In Stereo-seq, circularized DNA containing a random 25 nt barcode is RCA amplified and deposited into an etched grid. The barcode is sequenced and then oligos with polyT and molecular ID are hybridized to the barcode to capture polyA transcripts from the mounted tissue. The reported capture efficiency is around 170 transcripts per 100 <span class="math inline">\(\mu m^2\)</span> in mouse brain, on par with that of the Visium mouse brain dataset from the 10X website reanalyzed in the same study.</p>
<p>Another sub-micron array capture method is Seq-Scope <span class="citation">(Cho et al. <a href="#ref-Cho2021" role="doc-biblioref">2021</a>)</span>, which creates clusters of polyT capture sequences each with its own spatial barcode (20-32 nt) from Illumina bridge amplification on a repurposed Illumina flow cell. The spatial barcode is sequenced with SBS. Then the flow cell is dismantled so the tissue can be mounted for transcript capture. The captured transcripts are then sequenced with NGS. The clusters can have a diameter down to 0.5 <span class="math inline">\(\mu\)</span>m, and the clusters are randomly seeded, not distributed in a grid. The reported capture efficiency is around 1000 and up to 2000 transcripts per 100 <span class="math inline">\(\mu m^2\)</span> in mouse colon, much higher than that of Stereo-seq, although we are not sure whether colon data is comparable to brain data here.</p>
<p>A more recent nearly sub-micron technique is PIXEL-seq <span class="citation">(Fu et al. <a href="#ref-Fu2021" role="doc-biblioref">2021</a>)</span>. Again, as in the Illumina flow cell, PIXEL-seq amplifies each randomly seeded spatial barcode (24 nt) and polyT capture sequence into polonies. However, here a crosslinked polyacrylamide gel (rather than a linear one in Illumina) is used, to form continuous polonies without much space between their “territories” rather than discrete clusters. The spatial barcodes are also first sequenced with SBS before the tissue is mounted for transcript capture. On average, the polony is around 1.17 <span class="math inline">\(\mu m^2\)</span> in area, so assuming it is circular, then the diameter is 1.22 <span class="math inline">\(\mu\)</span>m. The reported capture efficiency is around 1000 transcripts per <span class="math inline">\(\mu m^2\)</span> in mouse brain, which might be comparable to that of Seq-Scope.</p>
<p>While such sub-micron techniques have subcellular resolution, in practice, the data is binned into much larger grids for standard scRNA-seq analysis, such as <span class="math inline">\(36\mu m\times36\mu m\)</span> in Stereo-seq and <span class="math inline">\(10\mu m\times10\mu m\)</span> or <span class="math inline">\(7\mu m\times7\mu m\)</span> or <span class="math inline">\(5\mu m\times5\mu m\)</span> in Seq-Scope. The subcelluar information was not directly use in the analyses, although even with binning, the resolution is still higher than that of ST and Visium.</p>
<p>In summary, a putative ranking, from high to low, of capture efficiencies of current era techniques is:</p>
<p>smFISH (~100%) &gt; MERFISH (HD4, ~95%) &gt; HCR-seqFISH (~86%) &gt; ExSeq (targeted, 62%) &gt; seqFISH+ (~49%) &gt; (maybe) Seq-Scope ~ PIXEL-seq &gt; (maybe) DBiT-seq (~15%) ~ Visium ~ Stereo-seq &gt; (maybe) HybRISS &gt; HybISS ~ (maybe) STARmap ~ (maybe) scRNA-seq ~ slide-seq2 ~ ST (~6.9%) ~ ISS (~5%) &gt; HDST &gt; slide-seq1 &gt; FISSEQ</p>
<p>This is putative because this is based on reports in the main text. There are conflicting reports of capture efficiency of Visium and DBiT-seq. Furthermore, comparison of different tissues and different genes from those studies may be problematic. For some of the technologies, the capture efficiency is compared to that of smFISH with only a few genes. Multiple datasets from each technology for as similar a tissue as possible for the same set of genes should be compared to get a better idea about the capture efficiency of each technique. Moreover, other factors such as tissue handling, sequencing depth, and data processing software may influence the results.</p>
<p>All these array based techniques reviewed so far capture polyadenylated transcripts. While miRNAs form a major topic in LCM literature (Figure <a href="text-mining.html#fig:topics">6.3</a>) and are profiled in some prequel era ISH atlases, current era techniques mostly preclude miRNA quantification. For smFISH based techniques, miRNAs are way too short to accommodate the large number of probes, even with signal amplification. While RCA has been adapted to miRNA, to our best knowledge, it has not been demonstrated to show individual miRNAs as discrete puncta like in smFISH and ISS, nor has it been demonstrated in a highly multiplexed fashion <span class="citation">(Neubacher and Arenz <a href="#ref-Neubacher2009" role="doc-biblioref">2009</a>; Kim, Kim, and Kim <a href="#ref-Kim2020" role="doc-biblioref">2020</a>; Zhou et al. <a href="#ref-Zhou2020" role="doc-biblioref">2020</a>)</span>. Without a poly-A tail, miRNAs are precluded by the other array based techniques as well. To quantify miRNAs in space, an array based technique was developed as an alternative to LCM and designed for FFPE tissues <span class="citation">(Nagarajan et al. <a href="#ref-Nagarajan2020" role="doc-biblioref">2020</a>)</span>. The tissue is pixelated, and each pixel is 300 <span class="math inline">\(\mu\)</span>m <span class="math inline">\(\times\)</span> 300 <span class="math inline">\(\mu\)</span>m. Within each pixel is a smaller 3 <span class="math inline">\(\times\)</span> 3 array, each spot of which has probes for one miRNA; The locations of the spots within each pixel can be easily discerned with a fluorescent microscope. This way, up to 9 miRNAs can be profiled in the same tissue section at the same time, although the 9 miRNAs are from somewhat nearby cells but not the same cells.</p>
</div>
<div id="no-imaging" class="section level2">
<h2><span class="header-section-number">5.5</span> No imaging</h2>
<p>The techniques reviewed above, involve either imaging (e.g. LCM, smFISH, ISS, Slide-seq, and HDST) or prior knowledge of locations (e.g. Tomo-seq, ST, Visium, and DBiT-seq). Some spatial transcriptomics techniques have been developed that require neither imaging nor prior knowledge of locations, and we review these in this section.</p>
<p>It is possible to reconstruct relative locations of cells or transcripts from colocalization without imaging, albeit imperfectly. These techniques are reviewed in more details in <span class="citation">(Boulgakov, Ellington, and Marcotte <a href="#ref-Boulgakov2020" role="doc-biblioref">2020</a>)</span>; we will only briefly summarize techniques that do not require DNA bound to a surface so they can be applied in cells and tissues. An early method to do so is Puzzle Imaging, published in 2015 <span class="citation">(Glaser et al. <a href="#ref-Glaser2015" role="doc-biblioref">2015</a>)</span>. Here “colocalization” can mean whether two neurons have axons in the same voxel or whether two neurons are synaptically connected. The spatial reconstruction is framed as a dimension reduction problem; each voxel is represented as a vector with <span class="math inline">\(n\)</span> dimensions, where <span class="math inline">\(n\)</span> stands for the number of neurons, and these vectors are to be projected into 2 or 3 dimensions, representing spatial dimensions, for reconstruction. Puzzle Imaging was only demonstrated in synthetic datasets, but not real biological datasets. Such reconstruction was made possible for transcripts with DNA microscopy <span class="citation">(Weinstein, Regev, and Zhang <a href="#ref-Weinstein2019" role="doc-biblioref">2019</a>)</span> Transcripts are reverse transcribed in situ, and the cDNA, with an UMI added, is PCR amplified in situ. The amplified products diffuse and encounter amplified products from other transcripts. The nearby cDNAs are concatenated with overlap extension PCR, with additional random sequences in the overlapping primers to encode each concatenation event, called unique event identifier (UEI). When the concatenated cDNAs are sequenced, the two UMIs and the UEI are recorded. Because amplified products from two nearby transcripts are more likely to be concatenated than those from two transcripts that are far apart, the number of UEIs between two UMIs can be used to reconstruct relative distance between transcripts.</p>
<p>Techniques have also been developed to quantify transcripts from subcellular compartments, such as APEX-RIP <span class="citation">(Kaewsapsak et al. <a href="#ref-Kaewsapsak2017" role="doc-biblioref">2017</a>)</span> and APEX-seq <span class="citation">(Fazal et al. <a href="#ref-Fazal2019a" role="doc-biblioref">2019</a>)</span>. Although these techniques do not record or reconstruct spatial coordinates, they are included in this review because the publications describing them described them with terms such as “spatial”, “localization”, and “spatial transcriptome”. APEX is an engineered ascorbate peroxidase, which can be targeted to specific cellular compartments by expressing a fusion of APEX and a protein targeted to the compartment of interest. With substrates H<sub>2</sub>O<sub>2</sub> and biotin-phenol (BP), APEX catalyzes formation of biotin-phenoxyl radicals that can biotinylate nearby proteins, which can be isolated with streptavidin. In APEX-RIP, mRNAs are cross linked to nearby proteins and thus isolated after isolating biotinylated proteins. In contrast, in APEX-seq, the mRNAs are directly biotinylated. Compared to APEX-RIP, APEX-seq better discerns transcript localization in compartments not bound by membrane. However, both APEX-RIP and APEX-seq were originally designed for bulk rather than single cell samples and was tested only on cell culture. Also, because a fusion protein is required, they cannot be performed in human tissue sections.</p>
<p>Rare cell types are difficult to characterize with most spatial transcriptomics techniques. ST and Visium lack single cell resolution and signal from rare cell types may be diluted by signal from common cell types in the same spot. LCM is still typically not used on single cells and rare cell types may or may not be easily discernible with H&amp;E. SmFISH based techniques and targeted ISS require a pre-defined panel of genes, often selected from scRNA-seq and well-known markers, but such selection is more challenging for rare cell types, which may not be well-studied enough to begin with due to challenges in other transcriptomics techniques. However, spatial pattern of genes expressed in rare cell types can be characterized by deliberately creating doublets or multiplets involving both common and rare cell types, as in paired cell sequencing <span class="citation">(Halpern et al. <a href="#ref-Halpern2018" role="doc-biblioref">2018</a>)</span> and ClumpSeq <span class="citation">(Manco et al. <a href="#ref-Manco2020" role="doc-biblioref">2020</a>)</span>. Spatial patterns of genes expressed in common cell types such as hepatocytes and small intestine enterocytes are already known from smFISH or LCM and spatial reconstruction of scRNA-seq data <span class="citation">(Halpern et al. <a href="#ref-Halpern2017" role="doc-biblioref">2017</a>; Moor et al. <a href="#ref-Moor2018" role="doc-biblioref">2018</a>)</span>. Genes expressed in the rare cell types are identified from genes much more highly expressed in the multiplet than in individual cells from common cell types in scRNA-seq, or markers of rare cell types from scRNA-seq if such data exists. Then the multiplets are mapped to spatial locations with patterned genes expressed by common cell types and existing smFISH or LCM data as reference. Then rare cell types and their characteristic gene programs are mapped to spatial locations as well and their patterns can be characterized without directly imaging these cells.</p>
</div>
<div id="multiomics" class="section level2">
<h2><span class="header-section-number">5.6</span> Spatial multi-omics</h2>
<p>Some spatial transcriptomics techniques have been adapted to collect data of other modalities, such as proteomics, neuron projection, and 3D chromatin conformation. These modalities can give a fuller picture of cell state than transcriptomics alone. In both MERFISH <span class="citation">(Wang, Moffitt, and Zhuang <a href="#ref-Wang2018" role="doc-biblioref">2018</a>)</span> and GeoMx DSP <span class="citation">(Merritt et al. <a href="#ref-Merritt2019" role="doc-biblioref">2019</a>)</span>, a panel of proteins can be quantified with oligonucleotide tagged secondary antibodies, and the oligo tag is detected and counted as spots just like mRNA. More recently, oligo tagged antibodies are also incorporated into ST as SM-Omics <span class="citation">(Vickovic et al. <a href="#ref-Vickovic2020" role="doc-biblioref">2020</a>)</span>. We have already mentioned adaptation of MERFISH targeting introns and genomic DNA to determine 3D chromatin conformation <span class="citation">(M. Liu et al. <a href="#ref-Liu2020" role="doc-biblioref">2020</a>; Su and Song <a href="#ref-Su2020" role="doc-biblioref">2020</a>)</span>, and pseudocolor seqFISH and seqFISH+ have been used for this purpose in cell culture as well <span class="citation">(Shah et al. <a href="#ref-Shah2018" role="doc-biblioref">2018</a>; Takei et al. <a href="#ref-Takei2020" role="doc-biblioref">2020</a>)</span>.</p>
<p>In MERFISH, traditional Nissl or poly-A based staining miss cellular processes, but neuron projection tracing can be performed prior to MERFISH. In the mouse motor cortex MERFISH atlas <span class="citation">(Zhang et al. <a href="#ref-Zhang2020" role="doc-biblioref">2020</a>)</span>, axons are first visualized by injecting cholera toxin subunit b (CTb) conjugated to 3 different dyes into 3 cortical areas as a retrograde tracer, tracing from terminals of the axons to the cell bodies. After imaging the axons, transcripts are imaged and quantified with MERFISH so neuronal projection can be related to the transcriptome. Viruses can be used for anterograde tracing, i.e. from cell bodies to axon terminals <span class="citation">(Xu et al. <a href="#ref-Xu2020" role="doc-biblioref">2020</a>)</span>, and can in theory be performed prior to MERFISH imaging. Axonal projections are traced in BARseq(2). Electrophysiological recordings from cultured cardiomyocytes in space has been coupled to STARmap with electro-seq; the recording is performed before the cells are fixed and cleared for STARmap <span class="citation">(Li et al. <a href="#ref-Li2021" role="doc-biblioref">2021</a>)</span>.</p>
</div>
<div id="database-current" class="section level2">
<h2><span class="header-section-number">5.7</span> Databases of the current era</h2>
<p>The database holding various spatial gene expression data was proposed early in the prequel era (1990s), when enhancer and gene trap data was proliferating and major WMISH atlas projects were in progress. In contrast, in the current era, databases only emerged after datasets from various techniques have already proliferated. One of such databases is SpatialDB <span class="citation">(Fan, Chen, and Chen <a href="#ref-Fan2020" role="doc-biblioref">2020</a>)</span>, published in late 2019, which holds gene count matrices from ST, LCM, Tomo-seq, and etc. and spatially variable genes identified with SpatialDE <span class="citation">(Svensson, Teichmann, and Stegle <a href="#ref-Svensson2018" role="doc-biblioref">2018</a>)</span> and trendsceek <span class="citation">(Edsgärd, Johnsson, and Sandberg <a href="#ref-Edsgard2018" role="doc-biblioref">2018</a>)</span>. In addition, the SpatialDB website provides interactive visualization of gene expression in space. Data can be queried by gene symbols, species, and data collection techniques.</p>
<p>Another database by the Brain Research through Advancing Innovative Neurotechnologies (BRAIN) Initiative - Cell Census Network (BICCN) is under construction as of late 2020 <span class="citation">(Adkins et al. <a href="#ref-Adkins2020" role="doc-biblioref">2020</a>)</span>. This is an international collaboration providing and generating multi-modal data for the mouse, human, and non-human primate brain, collected with scRNA-seq, ATAC-seq, neuron projection tracing, MRI, IHC, MERFISH <span class="citation">(Zhang et al. <a href="#ref-Zhang2020" role="doc-biblioref">2020</a>)</span>, osmFISH, seqFISH, and etc. The database website is hosted by the Allen Institute, and thus may be considered a continuation of the ABA. Data can be queried by species, technique, modality, and the lab that generated the data, but not by gene symbols.</p>
<p>While current era mouse brain atlases still reference the prequel ABA ontologies <span class="citation">(Ortiz et al. <a href="#ref-Ortiz2020" role="doc-biblioref">2020</a>; Chen et al. <a href="#ref-Chen2020" role="doc-biblioref">2020</a>; Vickovic et al. <a href="#ref-Vickovic2020" role="doc-biblioref">2020</a>; Lohoff et al. <a href="#ref-Lohoff2020" role="doc-biblioref">2020</a>)</span>, data cannot be queried by ontology in the current era databases, nor by a reference gene expression pattern as in the prequel database FlyExpress <span class="citation">(Kumar et al. <a href="#ref-Kumar2017" role="doc-biblioref">2017</a>)</span>. With more quantitative and comprehensive data, the traditional ontology may need to be revised. Unlike prequel databases such as ABA, EMAGE, and FlyExpress, to the best of our knowledge, current era spatial data has not been systematically registered to a 3D model for integrative analysis across datasets and for visualization.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-illinois">
<p>“10X VISIUM SPATIAL TRANSCRIPTOMICS.” n.d. Accessed October 7, 2020. <a href="https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics">https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics</a>.</p>
</div>
<div id="ref-Adkins2020">
<p>Adkins, Ricky S, Andrew I Aldridge, Shona Allen, Seth A Ament, Xu An, Ethan Armand, Giorgio A Ascoli, et al. 2020. “A multimodal cell census and atlas of the mammalian primary motor cortex.” <em>bioRxiv</em>, January, 2020.10.19.343129. <a href="https://doi.org/10.1101/2020.10.19.343129">https://doi.org/10.1101/2020.10.19.343129</a>.</p>
</div>
<div id="ref-umich">
<p>“ADVANCED GENOMICS CORE PRICING.” n.d. Accessed October 7, 2020. <a href="https://brcf.medicine.umich.edu/cores/advanced-genomics/price-list/">https://brcf.medicine.umich.edu/cores/advanced-genomics/price-list/</a>.</p>
</div>
<div id="ref-Aguila2018">
<p>Aguila, Julio, Shangli Cheng, Nigel Kee, Ming Cao, Qiaolin Deng, and Eva Hedlund. 2018. “Spatial transcriptomics and in silico random pooling identify novel markers of vulnerable and resistant midbrain dopamine neurons.” <em>bioRxiv</em>, October, 334417. <a href="https://doi.org/10.1101/334417">https://doi.org/10.1101/334417</a>.</p>
</div>
<div id="ref-Alon2020">
<p>Alon, Shahar, Daniel R Goodwin, Anubhav Sinha, Asmamaw T Wassie, Fei Chen, Evan R Daugharthy, Yosuke Bando, et al. 2020. “Expansion Sequencing: Spatially Precise &lt;em&gt;In Situ&lt;/em&gt; Transcriptomics in Intact Biological Systems.” <em>bioRxiv</em>, January, 2020.05.13.094268. <a href="https://doi.org/10.1101/2020.05.13.094268">https://doi.org/10.1101/2020.05.13.094268</a>.</p>
</div>
<div id="ref-Alsup2009">
<p>Alsup, William H. 2009. “Applera v. Illumina.” <a href="https://www.leagle.com/decision/infco20100325236">https://www.leagle.com/decision/infco20100325236</a>.</p>
</div>
<div id="ref-An2009">
<p>An, Li, Hongbo Xie, Mark H Chin, Zoran Obradovic, Desmond J Smith, and Vasileios Megalooikonomou. 2009. “Analysis of multiplex gene expression maps obtained by voxelation.” <em>BMC Bioinformatics</em> 10 (S4): S10. <a href="https://doi.org/10.1186/1471-2105-10-S4-S10">https://doi.org/10.1186/1471-2105-10-S4-S10</a>.</p>
</div>
<div id="ref-Andersson2021">
<p>Andersson, Axel, Ferran Diego, Fred A Hamprecht, and Carolina Wählby. 2021. “ISTDECO: In Situ Transcriptomics Decoding by Deconvolution.” <em>bioRxiv</em>, January, 2021.03.01.433040. <a href="https://doi.org/10.1101/2021.03.01.433040">https://doi.org/10.1101/2021.03.01.433040</a>.</p>
</div>
<div id="ref-ArcturusUS">
<p>“Arcturus XT Laser Capture Microdissection (LCM) Instrument - US.” n.d. <a href="//www.thermofisher.com/us/en/home/life-science/gene-expression-analysis-genotyping/laser-capture-microdissection/arcturus-laser-capture-microdissection-lcm-instrument.html">//www.thermofisher.com/us/en/home/life-science/gene-expression-analysis-genotyping/laser-capture-microdissection/arcturus-laser-capture-microdissection-lcm-instrument.html</a>.</p>
</div>
<div id="ref-Asp2019">
<p>Asp, Michaela, Stefania Giacomello, Ludvig Larsson, Chenglin Wu, Daniel Fürth, Xiaoyan Qian, Eva Wärdell, et al. 2019. “A Spatiotemporal Organ-Wide Gene Expression and Cell Atlas of the Developing Human Heart.” <em>Cell</em> 179 (7): 1647–1660.e19. <a href="https://doi.org/10.1016/j.cell.2019.11.025">https://doi.org/10.1016/j.cell.2019.11.025</a>.</p>
</div>
<div id="ref-Asp2017">
<p>Asp, Michaela, Fredrik Salmén, Patrik L Ståhl, Sanja Vickovic, Ulrika Felldin, Marie Löfling, José Fernandez Navarro, et al. 2017. “Spatial detection of fetal marker genes expressed at low level in adult human heart tissue.” <em>Scientific Reports</em> 7 (1): 12941. <a href="https://doi.org/10.1038/s41598-017-13462-5">https://doi.org/10.1038/s41598-017-13462-5</a>.</p>
</div>
<div id="ref-starfish">
<p>Axelrod, Shannon, Ambrose J Carr, Jeremy Freeman, Deep Ganguli, Brian Long, Tony Tung, and Others. n.d. “{Starfish}: Open Source Image Based Transcriptomics and Proteomics Tools.” <a href="http://github.com/spacetx/starfish">http://github.com/spacetx/starfish</a>.</p>
</div>
<div id="ref-Baccin2020">
<p>Baccin, Chiara, Jude Al-Sabah, Lars Velten, Patrick M. Helbling, Florian Grünschläger, Pablo Hernández-Malmierca, César Nombela-Arrieta, Lars M. Steinmetz, Andreas Trumpp, and Simon Haas. 2020. “Combined single-cell and spatial transcriptomics reveal the molecular, cellular and spatial bone marrow niche organization.” <em>Nature Cell Biology</em> 22 (1): 38–48. <a href="https://doi.org/10.1038/s41556-019-0439-6">https://doi.org/10.1038/s41556-019-0439-6</a>.</p>
</div>
<div id="ref-Bakken2016">
<p>Bakken, Trygve E., Jeremy A. Miller, Song Lin Ding, Susan M. Sunkin, Kimberly A. Smith, Lydia Ng, Aaron Szafer, et al. 2016. “A comprehensive transcriptional map of primate brain development.” <em>Nature</em> 535 (7612): 367–75. <a href="https://doi.org/10.1038/nature18637">https://doi.org/10.1038/nature18637</a>.</p>
</div>
<div id="ref-Baner1998">
<p>Baner, J., Mats Nilsson, Maritha Mendel-Hartvig, and Uif Landegren. 1998. “Signal amplification of padlock probes by rolling circle replication.” <em>Nucleic Acids Research</em> 26 (22): 5073–8. <a href="https://doi.org/10.1093/nar/26.22.5073">https://doi.org/10.1093/nar/26.22.5073</a>.</p>
</div>
<div id="ref-Bassell1994">
<p>Bassell, G J, C M Powers, K L Taneja, and R H Singer. 1994. “Single mRNAs visualized by ultrastructural in situ hybridization are principally localized at actin filament intersections in fibroblasts.” <em>The Journal of Cell Biology</em> 126 (4): 863–76. <a href="https://doi.org/10.1083/jcb.126.4.863">https://doi.org/10.1083/jcb.126.4.863</a>.</p>
</div>
<div id="ref-Bates2012">
<p>Bates, Mark, Graham T. Dempsey, Kok Hao Chen, and Xiaowei Zhuang. 2012. “Multicolor Super-Resolution Fluorescence Imaging via Multi-Parameter Fluorophore Detection.” <em>ChemPhysChem</em> 13 (1): 99–107. <a href="https://doi.org/10.1002/cphc.201100735">https://doi.org/10.1002/cphc.201100735</a>.</p>
</div>
<div id="ref-Battich2013">
<p>Battich, Nico, Thomas Stoeger, and Lucas Pelkmans. 2013. “Image-based transcriptomics in thousands of single human cells at single-molecule resolution.” <em>Nature Methods</em> 10 (11): 1127–36. <a href="https://doi.org/10.1038/nmeth.2657">https://doi.org/10.1038/nmeth.2657</a>.</p>
</div>
<div id="ref-Bayraktar2020">
<p>Bayraktar, Omer Ali, Theresa Bartels, Staffan Holmqvist, Vitalii Kleshchevnikov, Araks Martirosyan, Damon Polioudakis, Lucile Ben Haim, et al. 2020. “Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map.” <em>Nature Neuroscience</em> 23 (4): 500–509. <a href="https://doi.org/10.1038/s41593-020-0602-1">https://doi.org/10.1038/s41593-020-0602-1</a>.</p>
</div>
<div id="ref-Becker1996">
<p>Becker, Ingrid, Karl Friedrich Becker, Michael H. Röhrl, Günter Minkus, Karin Schütze, and Heinz Höfler. 1996. “Single-cell mutation analysis of tumors from stained histologic slides.” <em>Laboratory Investigation</em> 75 (6): 801–7. <a href="https://pubmed.ncbi.nlm.nih.gov/8973475/">https://pubmed.ncbi.nlm.nih.gov/8973475/</a>.</p>
</div>
<div id="ref-Belyavsky1989">
<p>Belyavsky, A, T Vinogradova, and K Rajewsky. 1989. “PCR-based cDNA library construction: general cDNA libraries at the level of a few cells.” <em>Nucleic Acids Research</em> 17 (8): 2919–32. <a href="https://doi.org/10.1093/nar/17.8.2919">https://doi.org/10.1093/nar/17.8.2919</a>.</p>
</div>
<div id="ref-Berglund2018">
<p>Berglund, Emelie, Jonas Maaskola, Niklas Schultz, Stefanie Friedrich, Maja Marklund, Joseph Bergenstråhle, Firas Tarish, et al. 2018. “Spatial maps of prostate cancer transcriptomes reveal an unexplored landscape of heterogeneity.” <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-018-04724-5">https://doi.org/10.1038/s41467-018-04724-5</a>.</p>
</div>
<div id="ref-BERNS1969">
<p>Berns, Michael W., Roberts S. Olson, and Donald E. Rounds. 1969. “In vitro Production of Chromosomal Lesions with an Argon Laser Microbeam.” <em>Nature</em> 221 (5175): 74–75. <a href="https://doi.org/10.1038/221074a0">https://doi.org/10.1038/221074a0</a>.</p>
</div>
<div id="ref-Bidarimath2015">
<p>Bidarimath, Mallikarjun, Andrew K. Edwards, and Chandrakant Tayade. 2015. “Laser Capture Microdissection for Gene Expression Analysis.” In <em>Methods in Molecular Biology</em>, 1219:115–37. Humana Press Inc. <a href="https://doi.org/10.1007/978-1-4939-1661-0_10">https://doi.org/10.1007/978-1-4939-1661-0_10</a>.</p>
</div>
<div id="ref-Blitz2017">
<p>Blitz, Ira L., Kitt D. Paraiso, Ilya Patrushev, William T. Y. Chiu, Ken W. Y. Cho, and Michael J. Gilchrist. 2017. “A catalog of Xenopus tropicalis transcription factors and their regional expression in the early gastrula stage embryo.” <em>Developmental Biology</em> 426 (2): 409–17. <a href="https://doi.org/10.1016/j.ydbio.2016.07.002">https://doi.org/10.1016/j.ydbio.2016.07.002</a>.</p>
</div>
<div id="ref-Boulgakov2020">
<p>Boulgakov, Alexander A, Andrew D Ellington, and Edward M Marcotte. 2020. “Bringing Microscopy-By-Sequencing into View.” <em>Trends in Biotechnology</em> 38 (2): 154–62. <a href="https://doi.org/https://doi.org/10.1016/j.tibtech.2019.06.001">https://doi.org/https://doi.org/10.1016/j.tibtech.2019.06.001</a>.</p>
</div>
<div id="ref-Brady1990">
<p>Brady, Gerard, Mary Barbara, and Norman N. Iscove. 1990. “Representative in Vitro cDNA Amplification From Individual Hemopoietic Cells and Colonies.” <em>Methods in Molecular and Cellular Biology</em> 2: 17–25.</p>
</div>
<div id="ref-VandenBrink2020">
<p>Brink, Susanne C. van den, Anna Alemany, Vincent van Batenburg, Naomi Moris, Marloes Blotenburg, Judith Vivié, Peter Baillie-Johnson, et al. 2020. “Single-cell and spatial transcriptomics reveal somitogenesis in gastruloids.” <em>Nature</em> 582 (7812): 405. <a href="https://doi.org/10.1038/s41586-020-2024-3">https://doi.org/10.1038/s41586-020-2024-3</a>.</p>
</div>
<div id="ref-Brown2002a">
<p>Brown, Vanessa M. 2002. “High-Throughput Imaging of Brain Gene Expression.” <em>Genome Research</em> 12 (2): 244–54. <a href="https://doi.org/10.1101/gr.204102">https://doi.org/10.1101/gr.204102</a>.</p>
</div>
<div id="ref-Brown2002">
<p>Brown, V. M., A. Ossadtchi, A. H. Khan, S. Yee, G. Lacan, W. P. Melega, S. R. Cherry, R. M. Leahy, and D. J. Smith. 2002. “Multiplex Three-Dimensional Brain Gene Expression Mapping in a Mouse Model of Parkinson’s Disease.” <em>Genome Research</em> 12 (6): 868–84. <a href="https://doi.org/10.1101/gr.229002">https://doi.org/10.1101/gr.229002</a>.</p>
</div>
<div id="ref-Burkhard2018">
<p>Burkhard, Silja Barbara, and Jeroen Bakkers. 2018. “Spatially resolved RNA-sequencing of the embryonic heart identifies a role for Wnt/<span class="math inline">\(\beta\)</span>-catenin signaling in autonomic control of heart rate.” <em>eLife</em> 7 (February). <a href="https://doi.org/10.7554/eLife.31515">https://doi.org/10.7554/eLife.31515</a>.</p>
</div>
<div id="ref-Bustin2015">
<p>Bustin, Stephen, Harvinder S Dhillon, Sara Kirvell, Christina Greenwood, Michael Parker, Gregory L Shipley, and Tania Nolan. 2015. “Variability of the Reverse Transcription Step: Practical Implications.” <em>Clinical Chemistry</em> 61 (1): 202–12. <a href="https://doi.org/10.1373/clinchem.2014.230615">https://doi.org/10.1373/clinchem.2014.230615</a>.</p>
</div>
<div id="ref-Butler2021">
<p>Butler, Daniel, Christopher Mozsary, Cem Meydan, Jonathan Foox, Joel Rosiene, Alon Shaiber, David Danko, et al. 2021. “Shotgun transcriptome, spatial omics, and isothermal profiling of SARS-CoV-2 infection reveals unique host responses, viral diversification, and drug interactions.” <em>Nature Communications</em> 12 (1): 1660. <a href="https://doi.org/10.1038/s41467-021-21361-7">https://doi.org/10.1038/s41467-021-21361-7</a>.</p>
</div>
<div id="ref-Cabili2015">
<p>Cabili, Moran N., Margaret C. Dunagin, Patrick D. McClanahan, Andrew Biaesch, Olivia Padovan-Merhar, Aviv Regev, John L. Rinn, and Arjun Raj. 2015. “Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution.” <em>Genome Biology</em> 16 (1): 20. <a href="https://doi.org/10.1186/s13059-015-0586-4">https://doi.org/10.1186/s13059-015-0586-4</a>.</p>
</div>
<div id="ref-Cao2021">
<p>Cao, Zhaohui, Wenlong Zuo, Lanxiang Wang, Junyu Chen, Zepeng Qu, Fan Jin, and Lei Dai. 2021. “Spatial profiling of microbial communities by sequential FISH with error-robust encoding.” <em>bioRxiv</em>, January, 2021.05.27.445923. <a href="https://doi.org/10.1101/2021.05.27.445923">https://doi.org/10.1101/2021.05.27.445923</a>.</p>
</div>
<div id="ref-Carlberg2019">
<p>Carlberg, Konstantin, Marina Korotkova, Ludvig Larsson, Anca I Catrina, Patrik L Ståhl, and Vivianne Malmström. 2019. “Exploring inflammatory signatures in arthritic joint biopsies with Spatial Transcriptomics.” <em>Scientific Reports</em> 9 (1): 18975. <a href="https://doi.org/10.1038/s41598-019-55441-y">https://doi.org/10.1038/s41598-019-55441-y</a>.</p>
</div>
<div id="ref-Chen2021a">
<p>Chen, Ao, Sha Liao, Mengnan Cheng, Kailong Ma, Liang Wu, Yiwei Lai, Jin Yang, et al. 2021. “Large field of view-spatially resolved transcriptomics at nanoscale resolution.” <em>bioRxiv</em>, January, 2021.01.17.427004. <a href="https://doi.org/10.1101/2021.01.17.427004">https://doi.org/10.1101/2021.01.17.427004</a>.</p>
</div>
<div id="ref-Chen2015b">
<p>Chen, Fei, Paul W. Tillberg, and Edward S. Boyden. 2015. “Expansion microscopy.” <em>Science</em> 347 (6221): 543–48. <a href="https://doi.org/10.1126/science.1260088">https://doi.org/10.1126/science.1260088</a>.</p>
</div>
<div id="ref-Chen2016">
<p>Chen, Fei, Asmamaw T. Wassie, Allison J. Cote, Anubhav Sinha, Shahar Alon, Shoh Asano, Evan R. Daugharthy, et al. 2016. “Nanoscale imaging of RNA with expansion microscopy.” <em>Nature Methods</em> 13 (8): 679–84. <a href="https://doi.org/10.1038/nmeth.3899">https://doi.org/10.1038/nmeth.3899</a>.</p>
</div>
<div id="ref-Chen2015">
<p>Chen, Kok Hao, Alistair N. Boettiger, Jeffrey R. Moffitt, Siyuan Wang, and Xiaowei Zhuang. 2015. “Spatially resolved, highly multiplexed RNA profiling in single cells.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaa6090">https://doi.org/10.1126/science.aaa6090</a>.</p>
</div>
<div id="ref-Chen2021">
<p>Chen, Shuonan, Jackson Loper, Xiaoyin Chen, Alex Vaughan, Anthony M Zador, and Liam Paninski. 2021. “BARcode DEmixing through Non-negative Spatial Regression (BarDensr).” <em>PLOS Computational Biology</em> 17 (3): e1008256. <a href="https://doi.org/10.1371/journal.pcbi.1008256">https://doi.org/10.1371/journal.pcbi.1008256</a>.</p>
</div>
<div id="ref-Chen2020">
<p>Chen, Wei-Ting, Ashley Lu, Katleen Craessaerts, Benjamin Pavie, Carlo Sala Frigerio, Nikky Corthout, Xiaoyan Qian, et al. 2020. “Spatial Transcriptomics and In Situ Sequencing to Study Alzheimer’s Disease.” <em>Cell</em> 0 (0). <a href="https://doi.org/10.1016/j.cell.2020.06.038">https://doi.org/10.1016/j.cell.2020.06.038</a>.</p>
</div>
<div id="ref-Chen2019">
<p>Chen, Xiaoyin, Yu-Chi Sun, Huiqing Zhan, Justus M Kebschull, Stephan Fischer, Katherine Matho, Z Josh Huang, Jesse Gillis, and Anthony M Zador. 2019. “High-Throughput Mapping of Long-Range Neuronal Projection Using &lt;em&gt;In Situ&lt;/em&gt; Sequencing.” <em>Cell</em> 179 (3): 772–786.e19. <a href="https://doi.org/10.1016/j.cell.2019.09.023">https://doi.org/10.1016/j.cell.2019.09.023</a>.</p>
</div>
<div id="ref-Chin2007">
<p>Chin, Mark H., Alex B. Geng, Arshad H. Khan, Wei-Jun Qian, Vladislav A. Petyuk, Jyl Boline, Shawn Levy, et al. 2007. “A genome-scale map of expression for a mouse brain section obtained using voxelation.” <em>Physiological Genomics</em> 30 (3): 313–21. <a href="https://doi.org/10.1152/physiolgenomics.00287.2006">https://doi.org/10.1152/physiolgenomics.00287.2006</a>.</p>
</div>
<div id="ref-Cho2021">
<p>Cho, Chun-Seok, Jingyue Xi, Sung-Rye Park, Jer-En Hsu, Myungjin Kim, Goo Jun, Hyun-Min Kang, and Jun Hee Lee. 2021. “Seq-Scope: Submicrometer-resolution spatial transcriptomics for single cell and subcellular studies.” <em>bioRxiv</em>, January, 2021.01.25.427807. <a href="https://doi.org/10.1101/2021.01.25.427807">https://doi.org/10.1101/2021.01.25.427807</a>.</p>
</div>
<div id="ref-Cleary2021">
<p>Cleary, Brian, Brooke Simonton, Jon Bezney, Evan Murray, Shahul Alam, Anubhav Sinha, Ehsan Habibi, et al. 2021. “Compressed sensing for highly efficient imaging transcriptomics.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/s41587-021-00883-x">https://doi.org/10.1038/s41587-021-00883-x</a>.</p>
</div>
<div id="ref-Codeluppi2018">
<p>Codeluppi, Simone, Lars E. Borm, Amit Zeisel, Gioele La Manno, Josina A. van Lunteren, Camilla I. Svensson, and Sten Linnarsson. 2018. “Spatial organization of the somatosensory cortex revealed by osmFISH.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/s41592-018-0175-z">https://doi.org/10.1038/s41592-018-0175-z</a>.</p>
</div>
<div id="ref-Combs2013">
<p>Combs, Peter A., and Michael B. Eisen. 2013. “Sequencing mRNA from Cryo-Sliced Drosophila Embryos to Determine Genome-Wide Spatial Patterns of Gene Expression.” Edited by Barbara Jennings. <em>PLoS ONE</em> 8 (8): e71820. <a href="https://doi.org/10.1371/journal.pone.0071820">https://doi.org/10.1371/journal.pone.0071820</a>.</p>
</div>
<div id="ref-Combs2018">
<p>Combs, Peter A., and Hunter B. Fraser. 2018. “Spatially varying cis-regulatory divergence in Drosophila embryos elucidates cis-regulatory logic.” Edited by Claude Desplan. <em>PLOS Genetics</em> 14 (11): e1007631. <a href="https://doi.org/10.1371/journal.pgen.1007631">https://doi.org/10.1371/journal.pgen.1007631</a>.</p>
</div>
<div id="ref-Coskun2016a">
<p>Coskun, Ahmet F., and Long Cai. 2016. “Dense transcript profiling in single cells by image correlation decoding.” <em>Nature Methods</em> 13 (8): 657–60. <a href="https://doi.org/10.1038/nmeth.3895">https://doi.org/10.1038/nmeth.3895</a>.</p>
</div>
<div id="ref-Coudry2007">
<p>Coudry, Renata A., Sibele I. Meireles, Radka Stoyanova, Harry S. Cooper, Alan Carpino, Xianqun Wang, Paul F. Engstrom, and Margie L. Clapper. 2007. “Successful Application of Microarray Technology to Microdissected Formalin-Fixed, Paraffin-Embedded Tissue.” <em>The Journal of Molecular Diagnostics</em> 9 (1): 70–79. <a href="https://doi.org/10.2353/jmoldx.2007.060004">https://doi.org/10.2353/jmoldx.2007.060004</a>.</p>
</div>
<div id="ref-Dana-Farber">
<p>“Dana-Farber Core Facilities.” n.d. Accessed October 14, 2020. <a href="https://www.dana-farber.org/research/core-facilities/">https://www.dana-farber.org/research/core-facilities/</a>.</p>
</div>
<div id="ref-Dar2021">
<p>Dar, Daniel, Nina Dar, Long Cai, and Dianne K Newman. 2021. “In situ single-cell activities of microbial populations revealed by spatial transcriptomics.” <em>bioRxiv</em>, January, 2021.02.24.432792. <a href="https://doi.org/10.1101/2021.02.24.432792">https://doi.org/10.1101/2021.02.24.432792</a>.</p>
</div>
<div id="ref-DeGiovanni2020">
<p>De Giovanni, Marco, Valeria Cutillo, Amir Giladi, Eleonora Sala, Carmela G Maganuco, Chiara Medaglia, Pietro Di Lucia, et al. 2020. “Spatiotemporal regulation of type I interferon expression determines the antiviral polarization of CD4+ T cells.” <em>Nature Immunology</em> 21 (3): 321–30. <a href="https://doi.org/10.1038/s41590-020-0596-6">https://doi.org/10.1038/s41590-020-0596-6</a>.</p>
</div>
<div id="ref-Delorey2021">
<p>Delorey, Toni M, Carly G K Ziegler, Graham Heimberg, Rachelly Normand, Yiming Yang, Asa Segerstolpe, Domenic Abbondanza, et al. 2021. “A single-cell and spatial atlas of autopsy tissues reveals pathology and cellular targets of SARS-CoV-2.” <em>bioRxiv</em>, January, 2021.02.25.430130. <a href="https://doi.org/10.1101/2021.02.25.430130">https://doi.org/10.1101/2021.02.25.430130</a>.</p>
</div>
<div id="ref-Dirks2004">
<p>Dirks, Robert M., and Niles A. Pierce. 2004. “From The Cover: Triggered amplification by hybridization chain reaction.” <em>Proceedings of the National Academy of Sciences</em> 101 (43): 15275–8. <a href="https://doi.org/10.1073/pnas.0407024101">https://doi.org/10.1073/pnas.0407024101</a>.</p>
</div>
<div id="ref-Dobin2012">
<p>Dobin, Alexander, Carrie A Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, and Thomas R Gingeras. 2012. “STAR: ultrafast universal RNA-seq aligner.” <em>Bioinformatics</em> 29 (1): 15–21. <a href="https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635</a>.</p>
</div>
<div id="ref-Drmanac2010">
<p>Drmanac, R., Andrew B. Sparks, Matthew J. Callow, Aaron L. Halpern, Norman L. Burns, Bahram G. Kermani, Paolo Carnevali, et al. 2010. “Human Genome Sequencing Using Unchained Base Reads on Self-Assembling DNA Nanoarrays.” <em>Science</em> 327 (5961): 78–81. <a href="https://doi.org/10.1126/science.1181498">https://doi.org/10.1126/science.1181498</a>.</p>
</div>
<div id="ref-nanostring_tap">
<p>“DSP Technology Access Program (TAP).” n.d. Accessed March 30, 2021.</p>
</div>
<div id="ref-Dulac1995">
<p>Dulac, Catherine, and Richard Axel. 1995. “A novel family of genes encoding putative pheromone receptors in mammals.” <em>Cell</em> 83 (2): 195–206. <a href="https://doi.org/https://doi.org/10.1016/0092-8674(95)90161-2">https://doi.org/https://doi.org/10.1016/0092-8674(95)90161-2</a>.</p>
</div>
<div id="ref-Ebbing2018">
<p>Ebbing, Annabel, Abel Vertesy, Marco Betist, Bastiaan Spanjaard, Jan Philipp Junker, Eugene Berezikov, Alexander van Oudenaarden, and Hendrik Korswagen. 2018. “Spatial transcriptomics of C. elegans males and hermaphrodites identifies novel fertility genes.” <em>bioRxiv</em>, June, 348201. <a href="https://doi.org/10.1101/348201">https://doi.org/10.1101/348201</a>.</p>
</div>
<div id="ref-Eberwine1992">
<p>Eberwine, J, H Yeh, K Miyashiro, Y Cao, S Nair, R Finnell, M Zettel, and P Coleman. 1992. “Analysis of gene expression in single live neurons.” <em>Proceedings of the National Academy of Sciences</em> 89 (7): 3010 LP–3014. <a href="https://doi.org/10.1073/pnas.89.7.3010">https://doi.org/10.1073/pnas.89.7.3010</a>.</p>
</div>
<div id="ref-Edsgard2018">
<p>Edsgärd, Daniel, Per Johnsson, and Rickard Sandberg. 2018. “Identification of spatial expression trends in single-cell gene expression data.” <em>Nature Methods</em> 15 (5): 339–42. <a href="https://doi.org/10.1038/nmeth.4634">https://doi.org/10.1038/nmeth.4634</a>.</p>
</div>
<div id="ref-Emmert-Buck1996">
<p>Emmert-Buck, Michael R., Robert F. Bonner, Paul D. Smith, Rodrigo F. Chuaqui, Zhengping Zhuang, Seth R. Goldstein, Rhonda A. Weiss, and Lance A. Liotta. 1996. “Laser Capture Microdissection.” <em>Science</em> 274 (5289): 998–1001. <a href="https://doi.org/10.1126/science.274.5289.998">https://doi.org/10.1126/science.274.5289.998</a>.</p>
</div>
<div id="ref-Eng2019a">
<p>Eng, Chee Huat Linus, Michael Lawson, Qian Zhu, Ruben Dries, Noushin Koulena, Yodai Takei, Jina Yun, et al. 2019. “Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+.” <em>Nature</em> 568 (7751): 235–39. <a href="https://doi.org/10.1038/s41586-019-1049-y">https://doi.org/10.1038/s41586-019-1049-y</a>.</p>
</div>
<div id="ref-Eng2017">
<p>Eng, Chee-Huat Linus, Sheel Shah, Julian Thomassie, and Long Cai. 2017. “Profiling the transcriptome with RNA SPOTs.” <em>Nature Methods</em> 14 (12): 1153–5. <a href="https://doi.org/10.1038/nmeth.4500">https://doi.org/10.1038/nmeth.4500</a>.</p>
</div>
<div id="ref-Fan2020">
<p>Fan, Zhen, Runsheng Chen, and Xiaowei Chen. 2020. “SpatialDB: a database for spatially resolved transcriptomes.” <em>Nucleic Acids Research</em> 48 (D1): D233–D237. <a href="https://doi.org/10.1093/nar/gkz934">https://doi.org/10.1093/nar/gkz934</a>.</p>
</div>
<div id="ref-Fazal2019a">
<p>Fazal, Furqan M., Shuo Han, Kevin R. Parker, Pornchai Kaewsapsak, Jin Xu, Alistair N. Boettiger, Howard Y. Chang, and Alice Y. Ting. 2019. “Atlas of Subcellular RNA Localization Revealed by APEX-Seq.” <em>Cell</em> 178 (2): 473–490.e26. <a href="https://doi.org/10.1016/j.cell.2019.05.027">https://doi.org/10.1016/j.cell.2019.05.027</a>.</p>
</div>
<div id="ref-Femino1998">
<p>Femino, Andrea M, Fredric S Fay, Kevin Fogarty, and Robert H Singer. 1998. “Visualization of Single RNA Transcripts in Situ.” <em>Science</em> 280 (5363): 585 LP–590. <a href="https://doi.org/10.1126/science.280.5363.585">https://doi.org/10.1126/science.280.5363.585</a>.</p>
</div>
<div id="ref-Fire1995">
<p>Fire, Andrew, and Si Qun Xu. 1995. “Rolling replication of short DNA circles.” <em>Proceedings of the National Academy of Sciences</em> 92 (10): 4641–5. <a href="https://doi.org/10.1073/pnas.92.10.4641">https://doi.org/10.1073/pnas.92.10.4641</a>.</p>
</div>
<div id="ref-Foley2019">
<p>Foley, Joseph W., Chunfang Zhu, Philippe Jolivet, Shirley X. Zhu, Peipei Lu, Michael J. Meaney, and Robert B. West. 2019. “Gene expression profiling of single cells from archival tissue with laser-capture microdissection and Smart-3SEQ.” <em>Genome Research</em> 29 (11): 1816–25. <a href="https://doi.org/10.1101/gr.234807.118">https://doi.org/10.1101/gr.234807.118</a>.</p>
</div>
<div id="ref-Foreman2019">
<p>Foreman, Robert, and Roy Wollman. 2019. “Mammalian gene expression variability is explained by underlying cell state.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/626424">https://doi.org/10.1101/626424</a>.</p>
</div>
<div id="ref-Fu2021">
<p>Fu, Xiaonan, Li Sun, Jane Y Chen, Runze Dong, Yiing Lin, Richard D Palmiter, Shin Lin, and Liangcai Gu. 2021. “Continuous Polony Gels for Tissue Mapping with High Resolution and RNA Capture Efficiency.” <em>bioRxiv</em>, January, 2021.03.17.435795. <a href="https://doi.org/10.1101/2021.03.17.435795">https://doi.org/10.1101/2021.03.17.435795</a>.</p>
</div>
<div id="ref-DSPPanels">
<p>“Gene Expression Panels | NanoString Technologies.” n.d. Accessed September 15, 2020. <a href="https://www.nanostring.com/products/gene-expression-panels/gene-expression-panels-overview">https://www.nanostring.com/products/gene-expression-panels/gene-expression-panels-overview</a>.</p>
</div>
<div id="ref-Giacomello2017">
<p>Giacomello, Stefania, Fredrik Salmén, Barbara K. Terebieniec, Sanja Vickovic, José Fernandez Navarro, Andrey Alexeyenko, Johan Reimegård, et al. 2017. “Spatially resolved transcriptome profiling in model plant species.” <em>Nature Plants</em>. <a href="https://doi.org/10.1038/nplants.2017.61">https://doi.org/10.1038/nplants.2017.61</a>.</p>
</div>
<div id="ref-Giani2020">
<p>Giani, Alice Maria, Guido Roberto Gallo, Luca Gianfranceschi, and Giulio Formenti. 2020. “Long walk to genomics: History and current approaches to genome sequencing and assembly.” <em>Computational and Structural Biotechnology Journal</em> 18 (January): 9–19. <a href="https://doi.org/10.1016/j.csbj.2019.11.002">https://doi.org/10.1016/j.csbj.2019.11.002</a>.</p>
</div>
<div id="ref-Giolai2019">
<p>Giolai, Michael, Walter Verweij, Ashleigh Lister, Darren Heavens, Iain Macaulay, and Matthew D. Clark. 2019. “Spatially resolved transcriptomics reveals plant host responses to pathogens.” <em>Plant Methods</em> 15 (1): 114. <a href="https://doi.org/10.1186/s13007-019-0498-5">https://doi.org/10.1186/s13007-019-0498-5</a>.</p>
</div>
<div id="ref-Glaser2015">
<p>Glaser, Joshua I., Bradley M. Zamft, George M. Church, and Konrad P. Kording. 2015. “Puzzle Imaging: Using Large-Scale Dimensionality Reduction Algorithms for Localization.” Edited by Joseph Najbauer. <em>PLOS ONE</em> 10 (7): e0131593. <a href="https://doi.org/10.1371/journal.pone.0131593">https://doi.org/10.1371/journal.pone.0131593</a>.</p>
</div>
<div id="ref-Goh2020">
<p>Goh, Jolene Jie Lin, Nigel Chou, Wan Yi Seow, Norbert Ha, Chung Pui Paul Cheng, Yun Ching Chang, Ziqing Winston Zhao, and Kok Hao Chen. 2020. “Highly specific multiplexed RNA imaging in tissues with split-FISH.” <em>Nature Methods</em> 17 (7): 689–93. <a href="https://doi.org/10.1038/s41592-020-0858-0">https://doi.org/10.1038/s41592-020-0858-0</a>.</p>
</div>
<div id="ref-Gregory2020">
<p>Gregory, J M, K McDade, M R Livesey, I Croy, S Marion de Proce, T Aitman, S Chandran, and C Smith. 2020. “Spatial transcriptomics identifies spatially dysregulated expression of GRM3 and USP47 in amyotrophic lateral sclerosis.” <em>Neuropathology and Applied Neurobiology</em> 46 (5): 441–57. <a href="https://doi.org/10.1111/nan.12597">https://doi.org/10.1111/nan.12597</a>.</p>
</div>
<div id="ref-Greulich1999">
<p>Greulich, Karl Otto. 1999. “Introduction: The history of using light as a working tool.” In <em>Micromanipulation by Light in Biology and Medicine</em>, 1–6. Boston, MA: Birkhäuser Boston. <a href="https://doi.org/10.1007/978-1-4612-4110-2_1">https://doi.org/10.1007/978-1-4612-4110-2_1</a>.</p>
</div>
<div id="ref-Grun2014">
<p>Grün, Dominic, Lennart Kester, and Alexander van Oudenaarden. 2014. “Validation of noise models for single-cell transcriptomics.” <em>Nature Methods</em> 11 (6): 637–40. <a href="https://doi.org/10.1038/nmeth.2930">https://doi.org/10.1038/nmeth.2930</a>.</p>
</div>
<div id="ref-Gyllborg2020">
<p>Gyllborg, Daniel, Christoffer Mattsson Langseth, Xiaoyan Qian, Sergio Marco Salas, Markus Hilscher, Ed Lein, and Mats Nilsson. 2020. “Hybridization-based In Situ Sequencing (HybISS): spatial transcriptomic detection in human and mouse brain tissue.” <em>bioRxiv</em>, February, 2020.02.03.931618. <a href="https://doi.org/10.1101/2020.02.03.931618">https://doi.org/10.1101/2020.02.03.931618</a>.</p>
</div>
<div id="ref-Halpern2018">
<p>Halpern, Keren Bahar, Rom Shenhav, Hassan Massalha, Beata Toth, Adi Egozi, Efi E. Massasa, Chiara Medgalia, et al. 2018. “Paired-cell sequencing enables spatial gene expression mapping of liver endothelial cells.” <em>Nature Biotechnology</em> 36 (10): 962. <a href="https://doi.org/10.1038/nbt.4231">https://doi.org/10.1038/nbt.4231</a>.</p>
</div>
<div id="ref-Halpern2017">
<p>Halpern, Keren Bahar, Rom Shenhav, Orit Matcovitch-Natan, Beáta Tóth, Doron Lemze, Matan Golan, Efi E. Massasa, et al. 2017. “Single-cell spatial reconstruction reveals global division of labour in the mammalian liver.” <em>Nature</em> 542 (7641): 1–5. <a href="https://doi.org/10.1038/nature21065">https://doi.org/10.1038/nature21065</a>.</p>
</div>
<div id="ref-Hashimshony2016">
<p>Hashimshony, Tamar, Naftalie Senderovich, Gal Avital, Agnes Klochendler, Yaron de Leeuw, Leon Anavy, Dave Gennert, et al. 2016. “CEL-Seq2: sensitive highly-multiplexed single-cell RNA-Seq.” <em>Genome Biology</em> 17 (1): 77. <a href="https://doi.org/10.1186/s13059-016-0938-8">https://doi.org/10.1186/s13059-016-0938-8</a>.</p>
</div>
<div id="ref-Hashimshony2012">
<p>Hashimshony, Tamar, Florian Wagner, Noa Sher, and Itai Yanai. 2012. “CEL-Seq: Single-Cell RNA-Seq by Multiplexed Linear Amplification.” <em>Cell Reports</em> 2 (3): 666–73. <a href="https://doi.org/https://doi.org/10.1016/j.celrep.2012.08.003">https://doi.org/https://doi.org/10.1016/j.celrep.2012.08.003</a>.</p>
</div>
<div id="ref-Hawrylycz2012">
<p>Hawrylycz, Michael J., Ed S. Lein, Angela L. Guillozet-Bongaarts, Elaine H. Shen, Lydia Ng, Jeremy A. Miller, Louie N. van de Lagemaat, et al. 2012. “An anatomically comprehensive atlas of the adult human brain transcriptome.” <em>Nature</em> 489 (7416): 391–99. <a href="https://doi.org/10.1038/nature11405">https://doi.org/10.1038/nature11405</a>.</p>
</div>
<div id="ref-He2020">
<p>He, Bryan, Ludvig Bergenstråhle, Linnea Stenbeck, Abubakar Abid, Alma Andersson, Åke Borg, Jonas Maaskola, Joakim Lundeberg, and James Zou. 2020. “Integrating spatial gene expression and breast tumour morphology via deep learning.” <em>Nature Biomedical Engineering</em>, June, 1–8. <a href="https://doi.org/10.1038/s41551-020-0578-x">https://doi.org/10.1038/s41551-020-0578-x</a>.</p>
</div>
<div id="ref-Hemby2002">
<p>Hemby, Scott E, Stephen D Ginsberg, Brian Brunk, Steven E Arnold, John Q Trojanowski, and James H Eberwine. 2002. “Gene Expression Profile for Schizophrenia: Discrete Neuron Transcription Patterns in the Entorhinal Cortex.” <em>Archives of General Psychiatry</em> 59 (7): 631–40. <a href="https://doi.org/10.1001/archpsyc.59.7.631">https://doi.org/10.1001/archpsyc.59.7.631</a>.</p>
</div>
<div id="ref-Hwang2020">
<p>Hwang, William L, Karthik A Jagadeesh, Jimmy A Guo, Hannah I Hoffman, Payman Yadollahpour, Rahul Mohan, Eugene Drokhlyansky, et al. 2020. “Single-nucleus and spatial transcriptomics of archival pancreatic cancer reveals multi-compartment reprogramming after neoadjuvant treatment.” <em>bioRxiv</em>, August, 2020.08.25.267336. <a href="https://doi.org/10.1101/2020.08.25.267336">https://doi.org/10.1101/2020.08.25.267336</a>.</p>
</div>
<div id="ref-Islam2011">
<p>Islam, Saiful, Una Kjällquist, Annalena Moliner, Pawel Zajac, Jian-Bing Fan, Peter Lönnerberg, and Sten Linnarsson. 2011. “Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq.” <em>Genome Research</em> 21 (7): 1160–7. <a href="https://doi.org/10.1101/gr.110882.110">https://doi.org/10.1101/gr.110882.110</a>.</p>
</div>
<div id="ref-Jemt2016">
<p>Jemt, Anders, Fredrik Salmén, Anna Lundmark, Annelie Mollbrink, José Fernández Navarro, Patrik L. Ståhl, Tülay Yucel-Lindberg, and Joakim Lundeberg. 2016. “An automated approach to prepare tissue-derived spatially barcoded RNA-sequencing libraries.” <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/srep37137">https://doi.org/10.1038/srep37137</a>.</p>
</div>
<div id="ref-Ji2020">
<p>Ji, Andrew L, Adam J Rubin, Kim Thrane, Sizun Jiang, David L Reynolds, Robin M Meyers, Margaret G Guo, et al. 2020. “Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma.” <em>Cell</em> 182 (2): 497–514.e22. <a href="https://doi.org/https://doi.org/10.1016/j.cell.2020.05.039">https://doi.org/https://doi.org/10.1016/j.cell.2020.05.039</a>.</p>
</div>
<div id="ref-Joglekar2020">
<p>Joglekar, Anoushka, Andrey Prjibelski, Ahmed Mahfouz, Paul Collier, Susan Lin, Anna Katharina Schlusche, Jordan Marrocco, et al. 2020. “Cell-type, single-cell, and spatial signatures of brain-region specific splicing in postnatal development.” <em>bioRxiv</em>, January, 2020.08.27.268730. <a href="https://doi.org/10.1101/2020.08.27.268730">https://doi.org/10.1101/2020.08.27.268730</a>.</p>
</div>
<div id="ref-JHU">
<p>“Johns Hopkins Cell Imaging Core Facility.” n.d. Accessed October 14, 2020. <a href="https://www.hopkinsmedicine.org/kimmel%7B\_%7Dcancer%7B\_%7Dcenter/research/shared%7B\_%7Dresources/cell%7B\_%7Dimaging.html">https://www.hopkinsmedicine.org/kimmel{\_}cancer{\_}center/research/shared{\_}resources/cell{\_}imaging.html</a>.</p>
</div>
<div id="ref-Junker2014">
<p>Junker, Jan Philipp, Emily S. Noël, Victor Guryev, Kevin A. Peterson, Gopi Shah, Jan Huisken, Andrew P. McMahon, Eugene Berezikov, Jeroen Bakkers, and Alexander Van Oudenaarden. 2014. “Genome-wide RNA Tomography in the Zebrafish Embryo.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2014.09.038">https://doi.org/10.1016/j.cell.2014.09.038</a>.</p>
</div>
<div id="ref-Kacharmina1999">
<p>Kacharmina, Janet Estee, Peter B Crino, and James B T - Methods in Enzymology Eberwine. 1999. “[1] Preparation of cDNA from single cells and subcellular regions.” In <em>CDNA Preparation and Characterization</em>, 303:3–18. Academic Press. <a href="https://doi.org/https://doi.org/10.1016/S0076-6879(99)03003-7">https://doi.org/https://doi.org/10.1016/S0076-6879(99)03003-7</a>.</p>
</div>
<div id="ref-Kaewsapsak2017">
<p>Kaewsapsak, Pornchai, David Michael Shechner, William Mallard, John L. Rinn, and Alice Y. Ting. 2017. “Live-cell mapping of organelle-associated RNAs via proximity biotinylation combined with protein-RNA crosslinking.” <em>eLife</em> 6 (December). <a href="https://doi.org/10.7554/eLife.29224">https://doi.org/10.7554/eLife.29224</a>.</p>
</div>
<div id="ref-Kamme2003">
<p>Kamme, Fredrik, Ranelle Salunga, Jingxue Yu, Da-Thao Tran, Jessica Zhu, Lin Luo, Anton Bittner, et al. 2003. “Single-Cell Microarray Analysis in Hippocampus CA1: Demonstration and Validation of Cellular Heterogeneity.” <em>The Journal of Neuroscience</em> 23 (9): 3607 LP–3615. <a href="https://doi.org/10.1523/JNEUROSCI.23-09-03607.2003">https://doi.org/10.1523/JNEUROSCI.23-09-03607.2003</a>.</p>
</div>
<div id="ref-Ke2013">
<p>Ke, Rongqin, Marco Mignardi, Alexandra Pacureanu, Jessica Svedlund, Johan Botling, Carolina Wählby, and Mats Nilsson. 2013. “In situ sequencing for RNA analysis in preserved tissue and cells.” <em>Nature Methods</em> 10 (9): 857–60. <a href="https://doi.org/10.1038/nmeth.2563">https://doi.org/10.1038/nmeth.2563</a>.</p>
</div>
<div id="ref-Kerman2006">
<p>Kerman, Ilan A., Bradley J. Buck, Simon J. Evans, Huda Akil, and Stanley J. Watson. 2006. “Combining laser capture microdissection with quantitative real-time PCR: Effects of tissue manipulation on RNA quality and gene expression.” <em>Journal of Neuroscience Methods</em> 153 (1): 71–85. <a href="https://doi.org/10.1016/j.jneumeth.2005.10.010">https://doi.org/10.1016/j.jneumeth.2005.10.010</a>.</p>
</div>
<div id="ref-Kim2020">
<p>Kim, Minhee, Dong-Min Kim, and Dong-Eun Kim. 2020. “Label-free fluorometric detection of microRNA using isothermal rolling circle amplification generating tandem G-quadruplex.” <em>Analyst</em> 145 (18): 6130–7. <a href="https://doi.org/10.1039/D0AN01329C">https://doi.org/10.1039/D0AN01329C</a>.</p>
</div>
<div id="ref-Kitahara2001">
<p>Kitahara, Osamu, Yoichi Furukawa, Toshihiro Tanaka, Chikashi Kihara, Kenji Ono, Renpei Yanagawa, Marcelo E Nita, Toshihisa Takagi, Yusuke Nakamura, and Tatsuhiko Tsunoda. 2001. “Alterations of Gene Expression during Colorectal Carcinogenesis Revealed by cDNA Microarrays after Laser-Capture Microdissection of Tumor Tissues and Normal Epithelia.” <em>Cancer Research</em> 61 (9): 3544 LP–3549. <a href="http://cancerres.aacrjournals.org/content/61/9/3544.abstract">http://cancerres.aacrjournals.org/content/61/9/3544.abstract</a>.</p>
</div>
<div id="ref-Klein2015">
<p>Klein, Allon M., Linas Mazutis, Ilke Akartuna, Naren Tallapragada, Adrian Veres, Victor Li, Leonid Peshkin, David A. Weitz, and Marc W. Kirschner. 2015. “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells.” <em>Cell</em> 161 (5): 1187–1201. <a href="https://doi.org/10.1016/j.cell.2015.04.044">https://doi.org/10.1016/j.cell.2015.04.044</a>.</p>
</div>
<div id="ref-Klein2002">
<p>Klein, Christoph A, Stefan Seidl, Karina Petat-Dutter, Sonja Offner, Jochen B Geigl, Oleg Schmidt-Kittler, Nicole Wendler, et al. 2002. “Combined transcriptome and genome analysis of single micrometastatic cells.” <em>Nature Biotechnology</em> 20 (4): 387–92. <a href="https://doi.org/10.1038/nbt0402-387">https://doi.org/10.1038/nbt0402-387</a>.</p>
</div>
<div id="ref-Ko1990">
<p>Ko, M S H, S B H Ko, N Takahashi, K Nishiguchi, and K Abe. 1990. “Unbiased amplification of a highly complex mixture of DNA fragments by ‘lone linker’-tagged PCR.” <em>Nucleic Acids Research</em> 18 (14): 4293. <a href="https://doi.org/10.1093/nar/18.14.4293">https://doi.org/10.1093/nar/18.14.4293</a>.</p>
</div>
<div id="ref-Kohman2020">
<p>Kohman, Richie E, and George M Church. 2020. “Fluorescent in situ sequencing of DNA barcoded antibodies.” <em>bioRxiv</em> 20 (April): 2020.04.27.060624. <a href="https://doi.org/10.1101/2020.04.27.060624">https://doi.org/10.1101/2020.04.27.060624</a>.</p>
</div>
<div id="ref-Kudo2012">
<p>Kudo, Lili C., Nancy Vi, Zhongcai Ma, Tony Fields, Nuraly K. Avliyakulov, Michael J. Haykinson, Anatol Bragin, and Stanislav L. Karsten. 2012. “Novel Cell and Tissue Acquisition System (CTAS): Microdissection of Live and Frozen Brain Tissues.” Edited by Rafael Aldabe. <em>PLoS ONE</em> 7 (7): e41564. <a href="https://doi.org/10.1371/journal.pone.0041564">https://doi.org/10.1371/journal.pone.0041564</a>.</p>
</div>
<div id="ref-Kumar2017">
<p>Kumar, Sudhir, Charlotte Konikoff, Maxwell Sanderford, Li Liu, Stuart Newfeld, Jieping Ye, and Rob J. Kulathinal. 2017. “FlyExpress 7: An Integrated Discovery Platform To Study Coexpressed Genes Using in situ Hybridization Images in Drosophila.” <em>G3; Genes|Genomes|Genetics</em> 7 (8): 2791–7. <a href="https://doi.org/10.1534/g3.117.040345">https://doi.org/10.1534/g3.117.040345</a>.</p>
</div>
<div id="ref-Lacraz2017">
<p>Lacraz, Grégory P. A., Jan Philipp Junker, Monika M. Gladka, Bas Molenaar, Koen T. Scholman, Marta Vigil-Garcia, Danielle Versteeg, et al. 2017. “Tomo-Seq Identifies SOX9 as a Key Regulator of Cardiac Fibrosis During Ischemic Injury.” <em>Circulation</em> 136 (15): 1396–1409. <a href="https://doi.org/10.1161/CIRCULATIONAHA.117.027832">https://doi.org/10.1161/CIRCULATIONAHA.117.027832</a>.</p>
</div>
<div id="ref-Landegren1988">
<p>Landegren, U, R Kaiser, J Sanders, and L Hood. 1988. “A ligase-mediated gene detection technique.” <em>Science</em> 241 (4869): 1077 LP–1080. <a href="https://doi.org/10.1126/science.3413476">https://doi.org/10.1126/science.3413476</a>.</p>
</div>
<div id="ref-Larsson2010">
<p>Larsson, Chatarina, Ida Grundberg, Ola Söderberg, and Mats Nilsson. 2010. “In situ detection and genotyping of individual mRNA molecules.” <em>Nature Methods</em> 7 (5): 395–97. <a href="https://doi.org/10.1038/nmeth.1448">https://doi.org/10.1038/nmeth.1448</a>.</p>
</div>
<div id="ref-Lebrigand2020">
<p>Lebrigand, Kevin, Joseph Bergenstråhle, Kim Thrane, Annelie Mollbrink, Pascal Barbry, Rainer Waldmann, and Joakim Lundeberg. 2020. “The spatial landscape of gene expression isoforms in tissue sections.” <em>bioRxiv</em>, August, 2020.08.24.252296. <a href="https://doi.org/10.1101/2020.08.24.252296">https://doi.org/10.1101/2020.08.24.252296</a>.</p>
</div>
<div id="ref-Lee2020">
<p>Lee, Hower, Sergio Marco Salas, Daniel Gyllborg, and Mats Nilsson. 2020. “Direct RNA targeted transcriptomic profiling in tissue using Hybridization-based RNA In Situ Sequencing (HybRISS).” <em>bioRxiv</em>, January, 2020.12.02.408781. <a href="https://doi.org/10.1101/2020.12.02.408781">https://doi.org/10.1101/2020.12.02.408781</a>.</p>
</div>
<div id="ref-Lee2015">
<p>Lee, Je Hyuk, Evan R. Daugharthy, Jonathan Scheiman, Reza Kalhor, Thomas C. Ferrante, Richard Terry, Brian M. Turczyk, et al. 2015. “Fluorescent in situ sequencing (FISSEQ) of RNA for gene expression profiling in intact cells and tissues.” <em>Nature Protocols</em> 10 (3): 442–58. <a href="https://doi.org/10.1038/nprot.2014.191">https://doi.org/10.1038/nprot.2014.191</a>.</p>
</div>
<div id="ref-Lee2014">
<p>Lee, Je Hyuk, Evan R Daugharthy, Jonathan Scheiman, Reza Kalhor, Joyce L Yang, Thomas C Ferrante, Richard Terry, et al. 2014. “Highly Multiplexed Subcellular RNA Sequencing in Situ.” <em>Science</em> 343 (6177): 1360 LP–1363. <a href="https://doi.org/10.1126/science.1250212">https://doi.org/10.1126/science.1250212</a>.</p>
</div>
<div id="ref-Lein2017">
<p>Lein, Ed, Lars E. Borm, and Sten Linnarsson. 2017. “The promise of spatial transcriptomics for neuroscience in the era of molecular cell typing.” <a href="https://doi.org/10.1126/science.aan6827">https://doi.org/10.1126/science.aan6827</a>.</p>
</div>
<div id="ref-Levsky2002">
<p>Levsky, Jeffrey M. 2002. “Single-Cell Gene Expression Profiling.” <em>Science</em> 297 (5582): 836–40. <a href="https://doi.org/10.1126/science.1072241">https://doi.org/10.1126/science.1072241</a>.</p>
</div>
<div id="ref-Li2021">
<p>Li, Qiang, Zuwan Lin, Ren Liu, Xin Tang, Jiahao Huang, Yichun He, Haowen Zhou, et al. 2021. “&lt;em&gt;In situ&lt;/em&gt; electro-sequencing in three-dimensional tissues.” <em>bioRxiv</em>, January, 2021.04.22.440941. <a href="https://doi.org/10.1101/2021.04.22.440941">https://doi.org/10.1101/2021.04.22.440941</a>.</p>
</div>
<div id="ref-Liang1992">
<p>Liang, P, and A B Pardee. 1992. “Differential display of eukaryotic messenger RNA by means of the polymerase chain reaction.” <em>Science</em> 257 (5072): 967 LP–971. <a href="https://doi.org/10.1126/science.1354393">https://doi.org/10.1126/science.1354393</a>.</p>
</div>
<div id="ref-Lignell2017">
<p>Lignell, Antti, Laura Kerosuo, Sebastian J. Streichan, Long Cai, and Marianne E. Bronner. 2017. “Identification of a neural crest stem cell niche by Spatial Genomic Analysis.” <em>Nature Communications</em> 8 (1): 1830. <a href="https://doi.org/10.1038/s41467-017-01561-w">https://doi.org/10.1038/s41467-017-01561-w</a>.</p>
</div>
<div id="ref-Liu2012">
<p>Liu, Lin, Yinhu Li, Siliang Li, Ni Hu, Yimin He, Ray Pong, Danni Lin, Lihua Lu, and Maggie Law. 2012. “Comparison of Next-Generation Sequencing Systems.” Edited by P J Oefner. <em>Journal of Biomedicine and Biotechnology</em> 2012: 251364. <a href="https://doi.org/10.1155/2012/251364">https://doi.org/10.1155/2012/251364</a>.</p>
</div>
<div id="ref-Liu2020">
<p>Liu, Miao, Yanfang Lu, Bing Yang, Yanbo Chen, Jonathan S. D. Radda, Mengwei Hu, Samuel G. Katz, and Siyuan Wang. 2020. “Multiplexed imaging of nucleome architectures in single cells of mammalian tissue.” <em>Nature Communications</em> 11 (1): 1–14. <a href="https://doi.org/10.1038/s41467-020-16732-5">https://doi.org/10.1038/s41467-020-16732-5</a>.</p>
</div>
<div id="ref-Liu2020a">
<p>Liu, Yang, Mingyu Yang, Yanxiang Deng, Graham Su, Archibald Enninful, Cindy C Guo, Toma Tebaldi, et al. 2020. “High-Spatial-Resolution Multi-Omics Sequencing via Deterministic Barcoding in Tissue.” <em>Cell</em> 183 (6): 1665–1681.e18. <a href="https://doi.org/https://doi.org/10.1016/j.cell.2020.10.026">https://doi.org/https://doi.org/10.1016/j.cell.2020.10.026</a>.</p>
</div>
<div id="ref-Lizardi1998">
<p>Lizardi, Paul M., Xiaohua Huang, Zhengrong Zhu, Patricia Bray-Ward, David C. Thomas, and David C. Ward. 1998. “Mutation detection and single-molecule counting using isothermal rolling-circle amplification.” <em>Nature Genetics</em> 19 (3): 225–32. <a href="https://doi.org/10.1038/898">https://doi.org/10.1038/898</a>.</p>
</div>
<div id="ref-Lohoff2020">
<p>Lohoff, T, S Ghazanfar, A Missarova, N Koulena, N Pierson, J A Griffiths, E S Bardot, et al. 2020. “Highly multiplexed spatially resolved gene expression profiling of mouse organogenesis.” <em>bioRxiv</em>, January, 2020.11.20.391896. <a href="https://doi.org/10.1101/2020.11.20.391896">https://doi.org/10.1101/2020.11.20.391896</a>.</p>
</div>
<div id="ref-Lovatt2014">
<p>Lovatt, Ditte, Brittani K. Ruble, Jaehee Lee, Hannah Dueck, Tae Kyung Kim, Stephen Fisher, Chantal Francis, et al. 2014. “Transcriptome in vivo analysis (TIVA) of spatially defined single cells in live tissue.” <em>Nature Methods</em> 11 (2): 190–96. <a href="https://doi.org/10.1038/nmeth.2804">https://doi.org/10.1038/nmeth.2804</a>.</p>
</div>
<div id="ref-Lubeck2012">
<p>Lubeck, Eric, and Long Cai. 2012. “Single-cell systems biology by super-resolution imaging and combinatorial labeling.” <em>Nature Methods 2012 9:7</em> 9 (7): 743–48. <a href="https://doi.org/10.1038/nmeth.2069">https://doi.org/10.1038/nmeth.2069</a>.</p>
</div>
<div id="ref-Lubeck2014">
<p>Lubeck, Eric, Ahmet F. Coskun, Timur Zhiyentayev, Mubhij Ahmad, and Long Cai. 2014. “Single-cell in situ RNA profiling by sequential hybridization.” Nature Publishing Group. <a href="https://doi.org/10.1038/nmeth.2892">https://doi.org/10.1038/nmeth.2892</a>.</p>
</div>
<div id="ref-Lukan2020">
<p>Lukan, Tjaša, Maruša Pompe‐Novak, Špela Baebler, Magda Tušek‐Žnidarič, Aleš Kladnik, Maja Križnik, Andrej Blejec, et al. 2020. “Precision transcriptomics of viral foci reveals the spatial regulation of immune‐signaling genes and identifies RBOHD as an important player in the incompatible interaction between potato virus Y and potato.” <em>The Plant Journal</em> 104 (3): 645–61. <a href="https://doi.org/10.1111/tpj.14953">https://doi.org/10.1111/tpj.14953</a>.</p>
</div>
<div id="ref-Lundmark2018">
<p>Lundmark, Anna, Natalija Gerasimcik, Tove Båge, Anders Jemt, Annelie Mollbrink, Fredrik Salmén, Joakim Lundeberg, and Tülay Yucel-Lindberg. 2018. “Gene expression profiling of periodontitis-affected gingival tissue by spatial transcriptomics.” <em>Scientific Reports</em> 8 (1): 9370. <a href="https://doi.org/10.1038/s41598-018-27627-3">https://doi.org/10.1038/s41598-018-27627-3</a>.</p>
</div>
<div id="ref-Luo1999">
<p>Luo, Lin, Ranelle C. Salunga, Hongqing Guo, Anton Bittner, K. C. Joy, Jose E. Galindo, Huinian Xiao, et al. 1999. “Gene expression profiles of laser-captured adjacent neuronal subtypes.” <em>Nature Medicine</em> 5 (1): 117–22. <a href="https://doi.org/10.1038/4806">https://doi.org/10.1038/4806</a>.</p>
</div>
<div id="ref-Macevicz1995">
<p>Macevicz, Stephen. 1995. “DNA sequencing by parallel olgonucleotide extensions.” <a href="https://patents.google.com/patent/US5969119A/en">https://patents.google.com/patent/US5969119A/en</a>.</p>
</div>
<div id="ref-Macosko2015">
<p>Macosko, Evan Z., Anindita Basu, Rahul Satija, James Nemesh, Karthik Shekhar, Melissa Goldman, Itay Tirosh, et al. 2015. “Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets.” <em>Cell</em> 161 (5): 1202–14. <a href="https://doi.org/10.1016/j.cell.2015.05.002">https://doi.org/10.1016/j.cell.2015.05.002</a>.</p>
</div>
<div id="ref-Manco2020">
<p>Manco, Rita, Inna Averbukh, Ziv Porat, Keren Bahar Halpern, Ido Amit, and Shalev Itzkovitz. 2020. “Clump sequencing exposes the spatial expression programs of intestinal secretory cells.” <em>bioRxiv</em>, August, 2020.08.05.237917. <a href="https://doi.org/10.1101/2020.08.05.237917">https://doi.org/10.1101/2020.08.05.237917</a>.</p>
</div>
<div id="ref-Maniatis2019">
<p>Maniatis, Silas, Tarmo Äijö, Sanja Vickovic, Catherine Braine, Kristy Kang, Annelie Mollbrink, Delphine Fagegaltier, et al. 2019. “Spatiotemporal dynamics of molecular pathology in amyotrophic lateral sclerosis.” <em>Science</em> 364 (6435): 89 LP–93. <a href="https://doi.org/10.1126/science.aav9776">https://doi.org/10.1126/science.aav9776</a>.</p>
</div>
<div id="ref-Manno2020">
<p>Manno, Gioele La, Kimberly Siletti, Alessandro Furlan, Daniel Gyllborg, Elin Vinsland, Christoffer Mattsson Langseth, Irina Khven, et al. 2020. “Molecular architecture of the developing mouse brain.” <em>bioRxiv</em>, July, 2020.07.02.184051. <a href="https://doi.org/10.1101/2020.07.02.184051">https://doi.org/10.1101/2020.07.02.184051</a>.</p>
</div>
<div id="ref-Margaroli2021">
<p>Margaroli, C, P Benson, N S Sharma, M C Madison, S W Robison, N Arora, K Ton, et al. 2021. “Spatial mapping of SARS-CoV-2 and H1N1 Lung Injury Identifies Differential Transcriptional Signatures.” <em>Cell Reports. Medicine</em>, March, 100242. <a href="https://doi.org/10.1016/j.xcrm.2021.100242">https://doi.org/10.1016/j.xcrm.2021.100242</a>.</p>
</div>
<div id="ref-Medaglia2017">
<p>Medaglia, Chiara, Amir Giladi, Liat Stoler-Barak, Marco De Giovanni, Tomer Meir Salame, Adi Biram, Eyal David, et al. 2017. “Spatial reconstruction of immune niches by combining photoactivatable reporters and scRNA-seq.” <em>Science</em> 358 (6370): 1622–6. <a href="https://doi.org/10.1126/science.aao4277">https://doi.org/10.1126/science.aao4277</a>.</p>
</div>
<div id="ref-Meier-Ruge1976">
<p>Meier-Ruge, W., W. Bielser, E. Remy, F. Hillenkamp, R. Nitsche, and R. Uns�ld. 1976. “The laser in the Lowry technique for microdissection of freeze-dried tissue slices.” <em>The Histochemical Journal</em> 8 (4): 387–401. <a href="https://doi.org/10.1007/BF01003828">https://doi.org/10.1007/BF01003828</a>.</p>
</div>
<div id="ref-Melsted2021">
<p>Melsted, Páll, A Sina Booeshaghi, Lauren Liu, Fan Gao, Lambda Lu, Kyung Hoi (Joseph) Min, Eduardo da Veiga Beltrame, Kristján Eldjárn Hjörleifsson, Jase Gehring, and Lior Pachter. 2021. “Modular, efficient and constant-memory single-cell RNA-seq preprocessing.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/s41587-021-00870-2">https://doi.org/10.1038/s41587-021-00870-2</a>.</p>
</div>
<div id="ref-Merritt2019">
<p>Merritt, Christopher R., Giang T. Ong, Sarah Church, Kristi Barker, Gary Geiss, Margaret Hoang, Jaemyeong Jung, et al. 2019. “High multiplex, digital spatial profiling of proteins and RNA in fixed tissue using genomic detection methods.” <em>bioRxiv</em> 38 (May): 559021. <a href="https://doi.org/10.1101/559021">https://doi.org/10.1101/559021</a>.</p>
</div>
<div id="ref-Miller2014">
<p>Miller, Jeremy A., Song-Lin Ding, Susan M. Sunkin, Kimberly A. Smith, Lydia Ng, Aaron Szafer, Amanda Ebbert, et al. 2014. “Transcriptional landscape of the prenatal human brain.” <em>Nature</em> 508 (7495): 199–206. <a href="https://doi.org/10.1038/nature13185">https://doi.org/10.1038/nature13185</a>.</p>
</div>
<div id="ref-Moffitt2018">
<p>Moffitt, Jeffrey R., Dhananjay Bambah-Mukku, Stephen W. Eichhorn, Eric Vaughn, Karthik Shekhar, Julio D. Perez, Nimrod D. Rubinstein, et al. 2018. “Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aau5324">https://doi.org/10.1126/science.aau5324</a>.</p>
</div>
<div id="ref-Moffitt2016a">
<p>Moffitt, Jeffrey R., Junjie Hao, Guiping Wang, Kok Hao Chen, Hazen P. Babcock, and Xiaowei Zhuang. 2016. “High-throughput single-cell gene-expression profiling with multiplexed error-robust fluorescence in situ hybridization.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1612826113">https://doi.org/10.1073/pnas.1612826113</a>.</p>
</div>
<div id="ref-Moor2017a">
<p>Moor, Andreas E., Matan Golan, Efi E. Massasa, Doron Lemze, Tomer Weizman, Rom Shenhav, Shaked Baydatch, et al. 2017. “Global mRNA polarization regulates translation efficiency in the intestinal epithelium.” <em>Science</em> 357 (6357): 1299–1303. <a href="https://doi.org/10.1126/science.aan2399">https://doi.org/10.1126/science.aan2399</a>.</p>
</div>
<div id="ref-Moor2018">
<p>Moor, Andreas E., Yotam Harnik, Shani Ben-Moshe, Efi E. Massasa, Milena Rozenberg, Raya Eilam, Keren Bahar Halpern, and Shalev Itzkovitz. 2018. “Spatial Reconstruction of Single Enterocytes Uncovers Broad Zonation along the Intestinal Villus Axis.” <em>Cell</em> 175 (4): 1156–1167.e15. <a href="https://doi.org/10.1016/j.cell.2018.08.063">https://doi.org/10.1016/j.cell.2018.08.063</a>.</p>
</div>
<div id="ref-Moris2020">
<p>Moris, Naomi, Kerim Anlas, Susanne C. van den Brink, Anna Alemany, Julia Schröder, Sabitri Ghimire, Tina Balayo, Alexander van Oudenaarden, and Alfonso Martinez Arias. 2020. “An in vitro model of early anteroposterior organization during human development.” <em>Nature</em> 582 (7812): 410–15. <a href="https://doi.org/10.1038/s41586-020-2383-9">https://doi.org/10.1038/s41586-020-2383-9</a>.</p>
</div>
<div id="ref-Morton2014">
<p>Morton, Matthew L., Xiaodong Bai, Callie R. Merry, Philip A. Linden, Ahmad M. Khalil, Rom S. Leidner, and Cheryl L. Thompson. 2014. “Identification of mRNAs and lincRNAs associated with lung cancer progression using next-generation RNA sequencing from laser micro-dissected archival FFPE tissue specimens.” <em>Lung Cancer</em> 85 (1): 31–39. <a href="https://doi.org/10.1016/j.lungcan.2014.03.020">https://doi.org/10.1016/j.lungcan.2014.03.020</a>.</p>
</div>
<div id="ref-Nagarajan2020">
<p>Nagarajan, Maxwell B, Augusto M Tentori, Wen Cai Zhang, Frank J Slack, and Patrick S Doyle. 2020. “Spatially resolved and multiplexed MicroRNA quantification from tissue using nanoliter well arrays.” <em>Microsystems &amp; Nanoengineering</em> 6 (1): 51. <a href="https://doi.org/10.1038/s41378-020-0169-8">https://doi.org/10.1038/s41378-020-0169-8</a>.</p>
</div>
<div id="ref-Nakamura2004">
<p>Nakamura, Toru, Yoichi Furukawa, Hidewaki Nakagawa, Tatsuhiko Tsunoda, Hiroaki Ohigashi, Kohei Murata, Osamu Ishikawa, et al. 2004. “Genome-wide cDNA microarray analysis of gene expression profiles in pancreatic cancers using populations of tumor cells and normal ductal epithelial cells selected for purity by laser microdissection.” <em>Oncogene</em> 23 (13): 2385–2400. <a href="https://doi.org/10.1038/sj.onc.1207392">https://doi.org/10.1038/sj.onc.1207392</a>.</p>
</div>
<div id="ref-Navarro2017">
<p>Navarro, José Fernández, Joel Sjöstrand, Fredrik Salmén, Joakim Lundeberg, and Patrik L. Ståhl. 2017. “ST Pipeline: an automated pipeline for spatial mapping of unique transcripts.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btx211">https://doi.org/10.1093/bioinformatics/btx211</a>.</p>
</div>
<div id="ref-Nederlof1990">
<p>Nederlof, P. M., S. van der Flier, J. Wiegant, A. K. Raap, H. J. Tanke, J. S. Ploem, and M. van der Ploeg. 1990. “Multiple fluorescence in situ hybridization.” <em>Cytometry</em> 11 (1): 126–31. <a href="https://doi.org/10.1002/cyto.990110115">https://doi.org/10.1002/cyto.990110115</a>.</p>
</div>
<div id="ref-Neubacher2009">
<p>Neubacher, Saskia, and Christoph Arenz. 2009. “Rolling-Circle Amplification: Unshared Advantages in miRNA Detection.” <em>ChemBioChem</em> 10 (8): 1289–91. <a href="https://doi.org/10.1002/cbic.200900116">https://doi.org/10.1002/cbic.200900116</a>.</p>
</div>
<div id="ref-Nichterwitz2016">
<p>Nichterwitz, Susanne, Geng Chen, Julio Aguila Benitez, Marlene Yilmaz, Helena Storvall, Ming Cao, Rickard Sandberg, Qiaolin Deng, and Eva Hedlund. 2016. “Laser capture microscopy coupled with Smart-seq2 for precise spatial transcriptomic profiling.” <em>Nature Communications</em> 7 (1): 12139. <a href="https://doi.org/10.1038/ncomms12139">https://doi.org/10.1038/ncomms12139</a>.</p>
</div>
<div id="ref-Nilsson1994">
<p>Nilsson, Mats, Helena Malmgren, Martina Samiotaki, Marek Kwiatkowski, B. Chowdhary, and Ulf Landegren. 1994. “Padlock probes: circularizing oligonucleotides for localized DNA detection.” <em>Science</em> 265 (5181): 2085–8. <a href="https://doi.org/10.1126/science.7522346">https://doi.org/10.1126/science.7522346</a>.</p>
</div>
<div id="ref-Ohyama2000">
<p>Ohyama, H., X. Zhang, Y. Kohno, I. Alevizos, M. Posner, D. T. Wong, and R. Todd. 2000. “Laser Capture Microdissection-Generated Target Sample for High-Density Oligonucleotide Array Hybridization.” <em>BioTechniques</em> 29 (3): 530–36. <a href="https://doi.org/10.2144/00293st05">https://doi.org/10.2144/00293st05</a>.</p>
</div>
<div id="ref-Okamura-Oho2012">
<p>Okamura-Oho, Yuko, Kazuro Shimokawa, Satoko Takemoto, Asami Hirakiyama, Sakiko Nakamura, Yuki Tsujimura, Masaomi Nishimura, et al. 2012. “Transcriptome Tomography for Brain Analysis in the Web-Accessible Anatomical Space.” Edited by Satoru Hayasaka. <em>PLoS ONE</em> 7 (9): e45373. <a href="https://doi.org/10.1371/journal.pone.0045373">https://doi.org/10.1371/journal.pone.0045373</a>.</p>
</div>
<div id="ref-Ortiz2020">
<p>Ortiz, Cantin, Jose Fernandez Navarro, Aleksandra Jurek, Antje Märtin, Joakim Lundeberg, and Konstantinos Meletis. 2020. “Molecular atlas of the adult mouse brain.” <em>Science Advances</em> 6 (26): eabb3446. <a href="https://doi.org/10.1126/sciadv.abb3446">https://doi.org/10.1126/sciadv.abb3446</a>.</p>
</div>
<div id="ref-Park2021">
<p>Park, Jiwoon, Jonathan Foox, Tyler Hether, David Danko, Sarah Warren, Youngmi Kim, Jason Reeves, et al. 2021. “Systemic Tissue and Cellular Disruption from SARS-CoV-2 Infection revealed in COVID-19 Autopsies and Spatial Omics Tissue Maps.” <em>bioRxiv</em>, January, 2021.03.08.434433. <a href="https://doi.org/10.1101/2021.03.08.434433">https://doi.org/10.1101/2021.03.08.434433</a>.</p>
</div>
<div id="ref-Partel2019">
<p>Partel, Gabriele, Markus Hilscher, Giorgia Milli, Leslie Solorzano, Anna Klemm, Mats Nilsson, and Carolina Wählby. 2019. “Identification of spatial compartments in tissue from in situ sequencing data.” <em>bioRxiv</em>, September, 765842. <a href="https://doi.org/10.1101/765842">https://doi.org/10.1101/765842</a>.</p>
</div>
<div id="ref-Parthasarathy2012">
<p>Parthasarathy, Raghuveer. 2012. “Rapid, accurate particle tracking by calculation of radial symmetry centers.” <em>Nature Methods</em> 9 (7): 724–26. <a href="https://doi.org/10.1038/nmeth.2071">https://doi.org/10.1038/nmeth.2071</a>.</p>
</div>
<div id="ref-Peng2016">
<p>Peng, Guangdun, Shengbao Suo, Jun Chen, Weiyang Chen, Chang Liu, Fang Yu, Ran Wang, et al. 2016. “Spatial Transcriptome for the Molecular Annotation of Lineage Fates and Cell Identity in Mid-gastrula Mouse Embryo.” <em>Developmental Cell</em> 36 (6): 681–97. <a href="https://doi.org/10.1016/j.devcel.2016.02.020">https://doi.org/10.1016/j.devcel.2016.02.020</a>.</p>
</div>
<div id="ref-Player2001">
<p>Player, Audrey N., Lu Ping Shen, Daryn Kenny, Vincent P. Antao, and Janice A. Kolberg. 2001. “Single-copy gene detection using branched DNA (bDNA) in situ hybridization.” <em>Journal of Histochemistry and Cytochemistry</em> 49 (5): 603–11. <a href="https://doi.org/10.1177/002215540104900507">https://doi.org/10.1177/002215540104900507</a>.</p>
</div>
<div id="ref-Plouhinec2017">
<p>Plouhinec, Jean-Louis, Sofía Medina-Ruiz, Caroline Borday, Elsa Bernard, Jean-Philippe Vert, Michael B. Eisen, Richard M. Harland, and Anne H. Monsoro-Burq. 2017. “A molecular atlas of the developing ectoderm defines neural, neural crest, placode, and nonneural progenitor identity in vertebrates.” Edited by James Briscoe. <em>PLOS Biology</em> 15 (10): e2004045. <a href="https://doi.org/10.1371/journal.pbio.2004045">https://doi.org/10.1371/journal.pbio.2004045</a>.</p>
</div>
<div id="ref-Qian2020">
<p>Qian, Xiaoyan, Kenneth D. Harris, Thomas Hauling, Dimitris Nicoloutsopoulos, Ana B. Muñoz-Manchado, Nathan Skene, Jens Hjerling-Leffler, and Mats Nilsson. 2020. “Probabilistic cell typing enables fine mapping of closely related cell types in situ.” <em>Nature Methods</em> 17 (1): 101–6. <a href="https://doi.org/10.1038/s41592-019-0631-4">https://doi.org/10.1038/s41592-019-0631-4</a>.</p>
</div>
<div id="ref-Raj2008">
<p>Raj, Arjun, Patrick van den Bogaard, Scott A. Rifkin, Alexander van Oudenaarden, and Sanjay Tyagi. 2008. “Imaging individual mRNA molecules using multiple singly labeled probes.” <em>Nature Methods</em> 5 (10): 877–79. <a href="https://doi.org/10.1038/nmeth.1253">https://doi.org/10.1038/nmeth.1253</a>.</p>
</div>
<div id="ref-Ramskold2012">
<p>Ramsköld, Daniel, Shujun Luo, Yu-Chieh Wang, Robin Li, Qiaolin Deng, Omid R Faridani, Gregory A Daniels, et al. 2012. “Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells.” <em>Nature Biotechnology</em> 30 (8): 777–82. <a href="https://doi.org/10.1038/nbt.2282">https://doi.org/10.1038/nbt.2282</a>.</p>
</div>
<div id="ref-Roberts2021">
<p>Roberts, Kenny, Alexander Aivazidis, Vitalii Kleshchevnikov, Tong Li, Robin Fropf, Michael Rhodes, Joseph M Beechem, Martin Hemberg, and Omer Ali Bayraktar. 2021. “Transcriptome-wide spatial RNA profiling maps the cellular architecture of the developing human neocortex.” <em>bioRxiv</em>, January, 2021.03.20.436265. <a href="https://doi.org/10.1101/2021.03.20.436265">https://doi.org/10.1101/2021.03.20.436265</a>.</p>
</div>
<div id="ref-Rodriques2019">
<p>Rodriques, Samuel G., Robert R. Stickels, Aleksandrina Goeva, Carly A. Martin, Evan Murray, Charles R. Vanderburg, Joshua Welch, Linlin M. Chen, Fei Chen, and Evan Z. Macosko. 2019. “Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaw1219">https://doi.org/10.1126/science.aaw1219</a>.</p>
</div>
<div id="ref-Rodelsperger2020">
<p>Rödelsperger, Christian, Annabel Ebbing, Devansh Raj Sharma, Misako Okumura, Ralf J Sommer, and Hendrik C Korswagen. 2020. “Spatial transcriptomics of nematodes identifies sperm cells as a source of genomic novelty and rapid evolution.” <em>Molecular Biology and Evolution</em>, August. <a href="https://doi.org/10.1093/molbev/msaa207">https://doi.org/10.1093/molbev/msaa207</a>.</p>
</div>
<div id="ref-Samacoits2018a">
<p>Samacoits, Aubin, Racha Chouaib, Adham Safieddine, Abdel-Meneem Traboulsi, Wei Ouyang, Christophe Zimmer, Marion Peter, Edouard Bertrand, Thomas Walter, and Florian Mueller. 2018. “A computational framework to study sub-cellular RNA localization.” <em>Nature Communications</em> 9 (1): 4584. <a href="https://doi.org/10.1038/s41467-018-06868-w">https://doi.org/10.1038/s41467-018-06868-w</a>.</p>
</div>
<div id="ref-Schede2020">
<p>Schede, Halima Hannah, Christian Gabriel Schneider, Johanna Stergiadou, Lars E Borm, Anurag Ranjak, Tracy M Yamawaki, Fabrice P A David, et al. 2020. “Spatial tissue profiling by imaging-free molecular tomography.” <em>bioRxiv</em>, August, 2020.08.04.235655. <a href="https://doi.org/10.1101/2020.08.04.235655">https://doi.org/10.1101/2020.08.04.235655</a>.</p>
</div>
<div id="ref-Schwaber2019">
<p>Schwaber, Jessica, Stacey Andersen, and Lars Nielsen. 2019. “Shedding light: The importance of reverse transcription efficiency standards in data interpretation.” <em>Biomolecular Detection and Quantification</em> 17 (March): 100077. <a href="https://doi.org/10.1016/j.bdq.2018.12.002">https://doi.org/10.1016/j.bdq.2018.12.002</a>.</p>
</div>
<div id="ref-Sgroi1999">
<p>Sgroi, Dennis C, Sarena Teng, Greg Robinson, Rebbecca LeVangie, James R Hudson, and Abdel G Elkahloun. 1999. “&lt;em&gt;In Vivo&lt;/em&gt; Gene Expression Profile Analysis of Human Breast Cancer Progression.” <em>Cancer Research</em> 59 (22): 5656 LP–5661. <a href="http://cancerres.aacrjournals.org/content/59/22/5656.abstract">http://cancerres.aacrjournals.org/content/59/22/5656.abstract</a>.</p>
</div>
<div id="ref-Shah2016">
<p>Shah, Sheel, Eric Lubeck, Wen Zhou, and Long Cai. 2016. “In Situ Transcription Profiling of Single Cells Reveals Spatial Organization of Cells in the Mouse Hippocampus.” <em>Neuron</em>. <a href="https://doi.org/10.1016/j.neuron.2016.10.001">https://doi.org/10.1016/j.neuron.2016.10.001</a>.</p>
</div>
<div id="ref-Shah2018">
<p>Shah, Sheel, Yodai Takei, Wen Zhou, Eric Lubeck, Jina Yun, Chee Huat Linus Eng, Noushin Koulena, et al. 2018. “Dynamics and Spatial Genomics of the Nascent Transcriptome by Intron seqFISH.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2018.05.035">https://doi.org/10.1016/j.cell.2018.05.035</a>.</p>
</div>
<div id="ref-Sharma2020">
<p>Sharma, Ankur, Justine Jia Wen Seow, Charles-Antoine Dutertre, Rhea Pai, Camille Blériot, Archita Mishra, Regina Men Men Wong, et al. 2020. “Onco-fetal Reprogramming of Endothelial Cells Drives Immunosuppressive Macrophages in Hepatocellular Carcinoma.” <em>Cell</em> 183 (2): 377–394.e21. <a href="https://doi.org/10.1016/j.cell.2020.08.040">https://doi.org/10.1016/j.cell.2020.08.040</a>.</p>
</div>
<div id="ref-Shendure2005">
<p>Shendure, Jay. 2005. “Accurate Multiplex Polony Sequencing of an Evolved Bacterial Genome.” <em>Science</em> 309 (5741): 1728–32. <a href="https://doi.org/10.1126/science.1117389">https://doi.org/10.1126/science.1117389</a>.</p>
</div>
<div id="ref-Shi2020">
<p>Shi, Hao, Qiaojuan Shi, Benjamin Grodner, Joan Sesing Lenz, Warren R Zipfel, Ilana Lauren Brito, and Iwijn De Vlaminck. 2020. “Highly multiplexed spatial mapping of microbial communities.” <em>Nature</em> 588 (7839): 676–81. <a href="https://doi.org/10.1038/s41586-020-2983-4">https://doi.org/10.1038/s41586-020-2983-4</a>.</p>
</div>
<div id="ref-Singh2003">
<p>Singh, Ram P., Vanessa M. Brown, Abhijit Chaudhari, Arshad H. Khan, Alex Ossadtchi, Daniel M. Sforza, A.Ken Meadors, Simon R. Cherry, Richard M. Leahy, and Desmond J. Smith. 2003. “High-resolution voxelation mapping of human and rodent brain gene expression.” <em>Journal of Neuroscience Methods</em> 125 (1-2): 93–101. <a href="https://doi.org/10.1016/S0165-0270(03)00045-1">https://doi.org/10.1016/S0165-0270(03)00045-1</a>.</p>
</div>
<div id="ref-Sountoulidis2020">
<p>Sountoulidis, Alexandros, Andreas Liontos, Hong Phuong Nguyen, Alexandra B Firsova, Athanasios Fysikopoulos, Xiaoyan Qian, Werner Seeger, Erik Sundström, Mats Nilsson, and Christos Samakovlis. 2020. “SCRINSHOT, a spatial method for single-cell resolution mapping of cell states in tissue sections.” <em>bioRxiv</em>, February, 2020.02.07.938571. <a href="https://doi.org/10.1101/2020.02.07.938571">https://doi.org/10.1101/2020.02.07.938571</a>.</p>
</div>
<div id="ref-ncl">
<p>“SpaRTAN.” n.d. Accessed October 7, 2020. <a href="https://www.ncl.ac.uk/gcf/spatiallyresolvedtranscriptomics/">https://www.ncl.ac.uk/gcf/spatiallyresolvedtranscriptomics/</a>.</p>
</div>
<div id="ref-Srivastava2019">
<p>Srivastava, Avi, Laraib Malik, Tom Smith, Ian Sudbery, and Rob Patro. 2019. “Alevin efficiently estimates accurate gene abundances from dscRNA-seq data.” <em>Genome Biology</em> 20 (1): 65. <a href="https://doi.org/10.1186/s13059-019-1670-y">https://doi.org/10.1186/s13059-019-1670-y</a>.</p>
</div>
<div id="ref-Stahl2016a">
<p>Ståhl, Patrik L., Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. “Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.” <em>Science</em> 353 (6294): 78–82. <a href="https://doi.org/10.1126/science.aaf2403">https://doi.org/10.1126/science.aaf2403</a>.</p>
</div>
<div id="ref-Stickels2020">
<p>Stickels, Robert, Evan Murray, Pawan Kumar, Jilong Li, Jamie Marshall, Daniela Di Bella, Paola Arlotta, Evan Macosko, and Fei Chen. 2020. “Sensitive spatial genome wide expression profiling at cellular resolution.” <em>bioRxiv</em>, March, 2020.03.12.989806. <a href="https://doi.org/10.1101/2020.03.12.989806">https://doi.org/10.1101/2020.03.12.989806</a>.</p>
</div>
<div id="ref-Stoeger2015">
<p>Stoeger, Thomas, Nico Battich, Markus D. Herrmann, Yauhen Yakimovich, and Lucas Pelkmans. 2015. “Computer vision for image-based transcriptomics.” <em>Methods</em> 85 (September): 44–53. <a href="https://doi.org/10.1016/j.ymeth.2015.05.016">https://doi.org/10.1016/j.ymeth.2015.05.016</a>.</p>
</div>
<div id="ref-Su2020">
<p>Su, Jing, and Qianqian Song. 2020. “DSTG: Deconvoluting Spatial Transcriptomics Data through Graph-based Artificial Intelligence.” <em>bioRxiv</em>, January, 2020.10.20.347195. <a href="https://doi.org/10.1101/2020.10.20.347195">https://doi.org/10.1101/2020.10.20.347195</a>.</p>
</div>
<div id="ref-Sun2020a">
<p>Sun, Yu-Chi, Xiaoyin Chen, Stephan Fischer, Shaina Lu, Jesse Gillis, and Anthony M Zador. 2020. “Integrating barcoded neuroanatomy with spatial transcriptional profiling reveals cadherin correlates of projections shared across the cortex.” <em>bioRxiv</em>, January, 2020.08.25.266460. <a href="https://doi.org/10.1101/2020.08.25.266460">https://doi.org/10.1101/2020.08.25.266460</a>.</p>
</div>
<div id="ref-Svensson2018">
<p>Svensson, Valentine, Sarah A. Teichmann, and Oliver Stegle. 2018. “SpatialDE: Identification of spatially variable genes.” <em>Nature Methods</em> 15 (5): 343–46. <a href="https://doi.org/10.1038/nmeth.4636">https://doi.org/10.1038/nmeth.4636</a>.</p>
</div>
<div id="ref-Takei2020">
<p>Takei, Yodai, Jina Yun, Noah Ollikainen, Shiwei Zheng, Nico Pierson, Jonathan White, Sheel Shah, et al. 2020. “Global architecture of the nucleus in single cells by DNA seqFISH+ and multiplexed immunofluorescence.” <em>bioRxiv</em>, January, 2020.11.29.403055. <a href="https://doi.org/10.1101/2020.11.29.403055">https://doi.org/10.1101/2020.11.29.403055</a>.</p>
</div>
<div id="ref-Tang2009">
<p>Tang, Fuchou, Catalin Barbacioru, Yangzhou Wang, Ellen Nordman, Clarence Lee, Nanlan Xu, Xiaohui Wang, et al. 2009. “mRNA-Seq whole-transcriptome analysis of a single cell.” <em>Nature Methods</em> 6 (5): 377–82. <a href="https://doi.org/10.1038/nmeth.1315">https://doi.org/10.1038/nmeth.1315</a>.</p>
</div>
<div id="ref-Tang2011">
<p>Tang, Fuchou, Kaiqin Lao, and M Azim Surani. 2011. “Development and applications of single-cell transcriptome analysis.” <em>Nature Methods</em> 8 (4 Suppl): S6–S11. <a href="https://doi.org/10.1038/nmeth.1557">https://doi.org/10.1038/nmeth.1557</a>.</p>
</div>
<div id="ref-Tecott1988">
<p>Tecott, L H, J D Barchas, and J H Eberwine. 1988. “In situ transcription: specific synthesis of complementary DNA in fixed tissue sections.” <em>Science</em> 240 (4859): 1661 LP–1664. <a href="https://doi.org/10.1126/science.2454508">https://doi.org/10.1126/science.2454508</a>.</p>
</div>
<div id="ref-Thrane2018">
<p>Thrane, Kim, Hanna Eriksson, Jonas Maaskola, Johan Hansson, and Joakim Lundeberg. 2018. “Spatially resolved transcriptomics enables dissection of genetic heterogeneity in stage III cutaneous malignant melanoma.” <em>Cancer Research</em>. <a href="https://doi.org/10.1158/0008-5472.CAN-18-0747">https://doi.org/10.1158/0008-5472.CAN-18-0747</a>.</p>
</div>
<div id="ref-Tietjen2003">
<p>Tietjen, Ian, Jason M Rihel, Yanxiang Cao, Georgy Koentges, Lisa Zakhary, and Catherine Dulac. 2003. “Single-Cell Transcriptional Analysis of Neuronal Progenitors.” <em>Neuron</em> 38 (2): 161–75. <a href="https://doi.org/https://doi.org/10.1016/S0896-6273(03)00229-0">https://doi.org/https://doi.org/10.1016/S0896-6273(03)00229-0</a>.</p>
</div>
<div id="ref-TPCL">
<p>“Translational Pathology Core Laboratory (TPCL).” n.d. Accessed October 14, 2020. <a href="https://www.uclahealth.org/pathology/tpcl-services">https://www.uclahealth.org/pathology/tpcl-services</a>.</p>
</div>
<div id="ref-Tripodo2020">
<p>Tripodo, Claudio, Federica Zanardi, Fabio Iannelli, Saveria Mazzara, Mariella Vegliante, Gaia Morello, Arianna Di Napoli, et al. 2020. “A Spatially Resolved Dark- versus Light-Zone Microenvironment Signature Subdivides Germinal Center-Related Aggressive B Cell Lymphomas.” <em>iScience</em> 23 (10). <a href="https://doi.org/10.1016/j.isci.2020.101562">https://doi.org/10.1016/j.isci.2020.101562</a>.</p>
</div>
<div id="ref-Tzur2018">
<p>Tzur, Yonatan B., Eitan Winter, Jinmin Gao, Tamar Hashimshony, Itai Yanai, and Monica P. Colaiácovo. 2018. “Spatiotemporal Gene Expression Analysis of the Caenorhabditis elegans Germline Uncovers a Syncytial Expression Switch.” <em>Genetics</em> 210 (2): 587–605. <a href="https://doi.org/10.1534/genetics.118.301315">https://doi.org/10.1534/genetics.118.301315</a>.</p>
</div>
<div id="ref-Urdea1993">
<p>Urdea, M. S. 1993. “Synthesis and Characterization of Branched DNA (bDNA) for the Direct and Quantitative Detection of CMV, HBV, HCV, and HIV.” <em>Clinical Chemistry</em> 39 (4): 725–26. <a href="https://doi.org/10.1093/clinchem/39.4.725">https://doi.org/10.1093/clinchem/39.4.725</a>.</p>
</div>
<div id="ref-VanGelder1990">
<p>Van Gelder, R N, M E von Zastrow, A Yool, W C Dement, J D Barchas, and J H Eberwine. 1990. “Amplified RNA synthesized from limited quantities of heterogeneous cDNA.” <em>Proceedings of the National Academy of Sciences</em> 87 (5): 1663 LP–1667. <a href="http://www.pnas.org/content/87/5/1663.abstract">http://www.pnas.org/content/87/5/1663.abstract</a>.</p>
</div>
<div id="ref-Veritas">
<p>“Veritas Laser Capture Microdissection (LCM) and Laser Cutting System from Applied Biosystems.” n.d. Accessed October 14, 2020. <a href="https://www.unthsc.edu/research/flow-cytometry-and-laser-capture-microdissection-core-facility/beckman-coulter-cytomics-fc500-flow-cytometry-analyzer/veritas-laser-capture">https://www.unthsc.edu/research/flow-cytometry-and-laser-capture-microdissection-core-facility/beckman-coulter-cytomics-fc500-flow-cytometry-analyzer/veritas-laser-capture</a>.</p>
</div>
<div id="ref-Vickovic2019">
<p>Vickovic, Sanja, Gökcen Eraslan, Fredrik Salmén, Johanna Klughammer, Linnea Stenbeck, Denis Schapiro, Tarmo Äijö, et al. 2019. “High-definition spatial transcriptomics for in situ tissue profiling.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/s41592-019-0548-y">https://doi.org/10.1038/s41592-019-0548-y</a>.</p>
</div>
<div id="ref-Vickovic2020">
<p>Vickovic, Sanja, Britta Lötstedt, Johanna Klughammer, Åsa Segerstolpe, Orit Rozenblatt-Rosen, and Aviv Regev. 2020. “SM-Omics: An automated platform for high-throughput spatial multi-omics.” <em>bioRxiv</em>, January, 2020.10.14.338418. <a href="https://doi.org/10.1101/2020.10.14.338418">https://doi.org/10.1101/2020.10.14.338418</a>.</p>
</div>
<div id="ref-Villacampa2020">
<p>Villacampa, Eva Gracia, Ludvig Larsson, Linda Kvastad, Alma Andersson, Joseph Carlson, and Joakim Lundeberg. 2020. “Genome-wide Spatial Expression Profiling in FFPE Tissues.” <em>bioRxiv</em>, July, 2020.07.24.219758. <a href="https://doi.org/10.1101/2020.07.24.219758">https://doi.org/10.1101/2020.07.24.219758</a>.</p>
</div>
<div id="ref-Wang2012">
<p>Wang, Fay, John Flanagan, Nan Su, Li-Chong Wang, Son Bui, Allissa Nielson, Xingyong Wu, Hong-Thuy Vo, Xiao-Jun Ma, and Yuling Luo. 2012. “RNAscope.” <em>The Journal of Molecular Diagnostics</em> 14 (1): 22–29. <a href="https://doi.org/10.1016/j.jmoldx.2011.08.002">https://doi.org/10.1016/j.jmoldx.2011.08.002</a>.</p>
</div>
<div id="ref-Wang2018">
<p>Wang, Guiping, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2018. “Multiplexed imaging of high-density libraries of RNAs with MERFISH and expansion microscopy.” <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/s41598-018-22297-7">https://doi.org/10.1038/s41598-018-22297-7</a>.</p>
</div>
<div id="ref-Wang2021">
<p>Wang, Yuhan, Mark Eddison, Greg Fleishman, Martin Weigert, Shengjin Xu, Fredrick E Henry, Tim Wang, et al. 2021. “Expansion-Assisted Iterative-FISH defines lateral hypothalamus spatio-molecular organization.” <em>bioRxiv</em>, January, 2021.03.08.434304. <a href="https://doi.org/10.1101/2021.03.08.434304">https://doi.org/10.1101/2021.03.08.434304</a>.</p>
</div>
<div id="ref-Weinstein2019">
<p>Weinstein, Joshua A., Aviv Regev, and Feng Zhang. 2019. “DNA Microscopy: Optics-free Spatio-genetic Imaging by a Stand-Alone Chemical Reaction.” <em>Cell</em> 178 (1): 229–241.e16. <a href="https://doi.org/10.1016/j.cell.2019.05.019">https://doi.org/10.1016/j.cell.2019.05.019</a>.</p>
</div>
<div id="ref-Xia2019c">
<p>Xia, Chenglong, Hazen P. Babcock, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2019. “Multiplexed detection of RNA using MERFISH and branched DNA amplification.” <em>Scientific Reports</em> 9 (1): 1–13. <a href="https://doi.org/10.1038/s41598-019-43943-8">https://doi.org/10.1038/s41598-019-43943-8</a>.</p>
</div>
<div id="ref-Xia2019a">
<p>Xia, Chenglong, Jean Fan, George Emanuel, Junjie Hao, and Xiaowei Zhuang. 2019. “Spatial transcriptome profiling by MERFISH reveals subcellular RNA compartmentalization and cell cycle-dependent gene expression.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1912459116">https://doi.org/10.1073/pnas.1912459116</a>.</p>
</div>
<div id="ref-Xu2020">
<p>Xu, Xiangmin, Todd C Holmes, Min-Hua Luo, Kevin T Beier, Gregory D Horwitz, Fei Zhao, Wenbo Zeng, May Hui, Bert L Semler, and Rozanne M Sandri-Goldin. 2020. “Viral Vectors for Neural Circuit Mapping and Recent Advances in Trans-synaptic Anterograde Tracers.” <em>Neuron</em> 107 (6): 1029–47. <a href="https://doi.org/https://doi.org/10.1016/j.neuron.2020.07.010">https://doi.org/https://doi.org/10.1016/j.neuron.2020.07.010</a>.</p>
</div>
<div id="ref-Yoda2017">
<p>Yoda, Takuya, Masahito Hosokawa, Kiyofumi Takahashi, Chikako Sakanashi, Haruko Takeyama, and Hideki Kambara. 2017. “Site-specific gene expression analysis using an automated tissue micro-dissection punching system.” <em>Scientific Reports</em> 7 (1): 4325. <a href="https://doi.org/10.1038/s41598-017-04616-6">https://doi.org/10.1038/s41598-017-04616-6</a>.</p>
</div>
<div id="ref-Zechel2014">
<p>Zechel, Sabrina, Pawel Zajac, Peter Lönnerberg, Carlos F. Ibáñez, and Sten Linnarsson. 2014. “Topographical transcriptome mapping of the mouse medial ganglionic eminence by spatially resolved RNA-seq.” <em>Genome Biology</em> 15 (10): 486. <a href="https://doi.org/10.1186/s13059-014-0486-z">https://doi.org/10.1186/s13059-014-0486-z</a>.</p>
</div>
<div id="ref-Zhang2020">
<p>Zhang, Meng, Stephen W Eichhorn, Brian Zingg, Zizhen Yao, Hongkui Zeng, Hongwei Dong, and Xiaowei Zhuang. 2020. “Molecular, spatial and projection diversity of neurons in primary motor cortex revealed by in situ single-cell transcriptomics.” <em>bioRxiv</em>, January, 2020.06.04.105700. <a href="https://doi.org/10.1101/2020.06.04.105700">https://doi.org/10.1101/2020.06.04.105700</a>.</p>
</div>
<div id="ref-Zheng2017">
<p>Zheng, Grace X. Y., Jessica M. Terry, Phillip Belgrader, Paul Ryvkin, Zachary W. Bent, Ryan Wilson, Solongo B. Ziraldo, et al. 2017. “Massively parallel digital transcriptional profiling of single cells.” <em>Nature Communications</em> 8 (1): 14049. <a href="https://doi.org/10.1038/ncomms14049">https://doi.org/10.1038/ncomms14049</a>.</p>
</div>
<div id="ref-Zhou2020">
<p>Zhou, Chuan, Ru Huang, Xiaoming Zhou, and Da Xing. 2020. “Sensitive and specific microRNA detection by RNA dependent DNA ligation and rolling circle optical signal amplification.” <em>Talanta</em> 216: 120954. <a href="https://doi.org/https://doi.org/10.1016/j.talanta.2020.120954">https://doi.org/https://doi.org/10.1016/j.talanta.2020.120954</a>.</p>
</div>
<div id="ref-Zhou2019">
<p>Zhou, Wen, Mary A. Yui, Brian A. Williams, Jina Yun, Barbara J. Wold, Long Cai, and Ellen V. Rothenberg. 2019. “Single-Cell Analysis Reveals Regulatory Gene Expression Dynamics Leading to Lineage Commitment in Early T Cell Development.” <em>Cell Systems</em> 9 (4): 321–337.e9. <a href="https://doi.org/10.1016/j.cels.2019.09.008">https://doi.org/10.1016/j.cels.2019.09.008</a>.</p>
</div>
<div id="ref-Zhu2018">
<p>Zhu, Qian, Sheel Shah, Ruben Dries, Long Cai, and Guo Cheng Yuan. 2018. “Identification of spatially associated subpopulations by combining scRNAseq and sequential fluorescence in situ hybridization data.” <em>Nature Biotechnology</em> 36 (12): 1183–90. <a href="https://doi.org/10.1038/nbt.4260">https://doi.org/10.1038/nbt.4260</a>.</p>
</div>
<div id="ref-Zhu2001">
<p>Zhu, Y Y, E M Machleder, A Chenchik, R Li, and P D Siebert. 2001. “Reverse Transcriptase Template Switching: A SMART™ Approach for Full-Length cDNA Library Construction.” <em>BioTechniques</em> 30 (4): 892–97. <a href="https://doi.org/10.2144/01304pf02">https://doi.org/10.2144/01304pf02</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="current.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="text-mining.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["museumst.pdf"],
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
