<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Museum of Spatial Transcriptomics</title>
  <meta name="description" content="Chapter 1 Introduction | Museum of Spatial Transcriptomics" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Museum of Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Chapter 1 Introduction | Museum of Spatial Transcriptomics" />
  <meta name="github-repo" content="pachterlab/LP_2021" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Museum of Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="Chapter 1 Introduction | Museum of Spatial Transcriptomics" />
  

<meta name="author" content="Lambda Moses, dlu2@caltech.edu" />
<meta name="author" content="Lior Pachter, lpachter@caltech.edu" />


<meta name="date" content="2021-07-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="prequel.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Museum of Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="part"><span><b>I Prequel era</b></span></li>
<li class="chapter" data-level="2" data-path="prequel.html"><a href="prequel.html"><i class="fa fa-check"></i><b>2</b> Prequel era</a><ul>
<li class="chapter" data-level="2.1" data-path="prequel.html"><a href="prequel.html#traps"><i class="fa fa-check"></i><b>2.1</b> Enhancer and gene traps</a></li>
<li class="chapter" data-level="2.2" data-path="prequel.html"><a href="prequel.html#in-situ-reporter"><i class="fa fa-check"></i><b>2.2</b> In situ reporter</a></li>
<li class="chapter" data-level="2.3" data-path="prequel.html"><a href="prequel.html#wmish"><i class="fa fa-check"></i><b>2.3</b> ISH and WMISH atlases</a></li>
<li class="chapter" data-level="2.4" data-path="prequel.html"><a href="prequel.html#database_prequel"><i class="fa fa-check"></i><b>2.4</b> Databases of the prequel era</a></li>
<li class="chapter" data-level="2.5" data-path="prequel.html"><a href="prequel.html#geo_prequel"><i class="fa fa-check"></i><b>2.5</b> Geography of the prequel era</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prequel-analysis.html"><a href="prequel-analysis.html"><i class="fa fa-check"></i><b>3</b> Data analysis in the prequel era</a><ul>
<li class="chapter" data-level="3.1" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-patterns"><i class="fa fa-check"></i><b>3.1</b> Gene patterns</a></li>
<li class="chapter" data-level="3.2" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-spatial-regions"><i class="fa fa-check"></i><b>3.2</b> Spatial regions</a></li>
<li class="chapter" data-level="3.3" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-interactions"><i class="fa fa-check"></i><b>3.3</b> Gene interactions</a></li>
<li class="chapter" data-level="3.4" data-path="prequel-analysis.html"><a href="prequel-analysis.html#decline"><i class="fa fa-check"></i><b>3.4</b> Decline</a></li>
<li class="chapter" data-level="3.5" data-path="prequel-analysis.html"><a href="prequel-analysis.html#geo-prequel-analysis"><i class="fa fa-check"></i><b>3.5</b> Geography of prequel data analysis</a></li>
</ul></li>
<li class="part"><span><b>II Current era</b></span></li>
<li class="chapter" data-level="4" data-path="current.html"><a href="current.html"><i class="fa fa-check"></i><b>4</b> From the past to the present</a></li>
<li class="chapter" data-level="5" data-path="current-techs.html"><a href="current-techs.html"><i class="fa fa-check"></i><b>5</b> Current era technologies</a><ul>
<li class="chapter" data-level="5.1" data-path="current-techs.html"><a href="current-techs.html#microdissection"><i class="fa fa-check"></i><b>5.1</b> Microdissection</a><ul>
<li class="chapter" data-level="5.1.1" data-path="current-techs.html"><a href="current-techs.html#history-of-lcm"><i class="fa fa-check"></i><b>5.1.1</b> History of LCM</a></li>
<li class="chapter" data-level="5.1.2" data-path="current-techs.html"><a href="current-techs.html#lcm-usage"><i class="fa fa-check"></i><b>5.1.2</b> Usage of LCM</a></li>
<li class="chapter" data-level="5.1.3" data-path="current-techs.html"><a href="current-techs.html#physical-microdissection"><i class="fa fa-check"></i><b>5.1.3</b> Physical microdissection</a></li>
<li class="chapter" data-level="5.1.4" data-path="current-techs.html"><a href="current-techs.html#de-facto-microdissection"><i class="fa fa-check"></i><b>5.1.4</b> <em>De facto</em> microdissection</a></li>
<li class="chapter" data-level="5.1.5" data-path="current-techs.html"><a href="current-techs.html#summary"><i class="fa fa-check"></i><b>5.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="current-techs.html"><a href="current-techs.html#smfish"><i class="fa fa-check"></i><b>5.2</b> Single molecular FISH</a><ul>
<li class="chapter" data-level="5.2.1" data-path="current-techs.html"><a href="current-techs.html#barcoding"><i class="fa fa-check"></i><b>5.2.1</b> Barcoding strategies</a></li>
<li class="chapter" data-level="5.2.2" data-path="current-techs.html"><a href="current-techs.html#amplification"><i class="fa fa-check"></i><b>5.2.2</b> Signal amplification</a></li>
<li class="chapter" data-level="5.2.3" data-path="current-techs.html"><a href="current-techs.html#crowding"><i class="fa fa-check"></i><b>5.2.3</b> Optical crowding</a></li>
<li class="chapter" data-level="5.2.4" data-path="current-techs.html"><a href="current-techs.html#smfish-usage"><i class="fa fa-check"></i><b>5.2.4</b> Usage of smFISH based techniques</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="current-techs.html"><a href="current-techs.html#iss"><i class="fa fa-check"></i><b>5.3</b> In situ sequencing</a><ul>
<li class="chapter" data-level="5.3.1" data-path="current-techs.html"><a href="current-techs.html#solid"><i class="fa fa-check"></i><b>5.3.1</b> SOLiD and FISSEQ</a></li>
<li class="chapter" data-level="5.3.2" data-path="current-techs.html"><a href="current-techs.html#cpal"><i class="fa fa-check"></i><b>5.3.2</b> cPAL and ISS</a></li>
<li class="chapter" data-level="5.3.3" data-path="current-techs.html"><a href="current-techs.html#sedal"><i class="fa fa-check"></i><b>5.3.3</b> SEDAL and STARmap</a></li>
<li class="chapter" data-level="5.3.4" data-path="current-techs.html"><a href="current-techs.html#sbs"><i class="fa fa-check"></i><b>5.3.4</b> Sequencing by synthesis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="current-techs.html"><a href="current-techs.html#array"><i class="fa fa-check"></i><b>5.4</b> In situ array capture</a></li>
<li class="chapter" data-level="5.5" data-path="current-techs.html"><a href="current-techs.html#no-imaging"><i class="fa fa-check"></i><b>5.5</b> No imaging</a></li>
<li class="chapter" data-level="5.6" data-path="current-techs.html"><a href="current-techs.html#multiomics"><i class="fa fa-check"></i><b>5.6</b> Spatial multi-omics</a></li>
<li class="chapter" data-level="5.7" data-path="current-techs.html"><a href="current-techs.html#database-current"><i class="fa fa-check"></i><b>5.7</b> Databases of the current era</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="text-mining.html"><a href="text-mining.html"><i class="fa fa-check"></i><b>6</b> Text mining LCM transcriptomics abstracts</a><ul>
<li class="chapter" data-level="6.1" data-path="text-mining.html"><a href="text-mining.html#topic-model"><i class="fa fa-check"></i><b>6.1</b> Topic modeling</a></li>
<li class="chapter" data-level="6.2" data-path="text-mining.html"><a href="text-mining.html#word-time"><i class="fa fa-check"></i><b>6.2</b> Changes of word usage through time</a></li>
<li class="chapter" data-level="6.3" data-path="text-mining.html"><a href="text-mining.html#topic-time"><i class="fa fa-check"></i><b>6.3</b> Changes of topic prevalence through time</a></li>
<li class="chapter" data-level="6.4" data-path="text-mining.html"><a href="text-mining.html#association-of-topics-with-city"><i class="fa fa-check"></i><b>6.4</b> Association of topics with city</a></li>
<li class="chapter" data-level="6.5" data-path="text-mining.html"><a href="text-mining.html#glove"><i class="fa fa-check"></i><b>6.5</b> GloVe word embedding</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="current-analysis.html"><a href="current-analysis.html"><i class="fa fa-check"></i><b>7</b> Data analysis in the current era</a><ul>
<li class="chapter" data-level="7.1" data-path="current-analysis.html"><a href="current-analysis.html#preprocessing"><i class="fa fa-check"></i><b>7.1</b> Preprocessing</a><ul>
<li class="chapter" data-level="7.1.1" data-path="current-analysis.html"><a href="current-analysis.html#registration"><i class="fa fa-check"></i><b>7.1.1</b> Image registration</a></li>
<li class="chapter" data-level="7.1.2" data-path="current-analysis.html"><a href="current-analysis.html#calling"><i class="fa fa-check"></i><b>7.1.2</b> Spot and barcode calling</a></li>
<li class="chapter" data-level="7.1.3" data-path="current-analysis.html"><a href="current-analysis.html#segmentation"><i class="fa fa-check"></i><b>7.1.3</b> Cell segmentation</a></li>
<li class="chapter" data-level="7.1.4" data-path="current-analysis.html"><a href="current-analysis.html#seg-alternatives"><i class="fa fa-check"></i><b>7.1.4</b> Alternatives to cell segmentation</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="current-analysis.html"><a href="current-analysis.html#eda"><i class="fa fa-check"></i><b>7.2</b> Exploratory data analysis</a></li>
<li class="chapter" data-level="7.3" data-path="current-analysis.html"><a href="current-analysis.html#reconstruction"><i class="fa fa-check"></i><b>7.3</b> Spatial reconstruction of scRNA-seq data</a><ul>
<li class="chapter" data-level="7.3.1" data-path="current-analysis.html"><a href="current-analysis.html#direct-dr"><i class="fa fa-check"></i><b>7.3.1</b> Direct dimension reduction</a></li>
<li class="chapter" data-level="7.3.2" data-path="current-analysis.html"><a href="current-analysis.html#ad-hoc"><i class="fa fa-check"></i><b>7.3.2</b> <em>Ad hoc</em> scoring</a></li>
<li class="chapter" data-level="7.3.3" data-path="current-analysis.html"><a href="current-analysis.html#generative"><i class="fa fa-check"></i><b>7.3.3</b> Generative models</a></li>
<li class="chapter" data-level="7.3.4" data-path="current-analysis.html"><a href="current-analysis.html#latent"><i class="fa fa-check"></i><b>7.3.4</b> Shared latent space</a></li>
<li class="chapter" data-level="7.3.5" data-path="current-analysis.html"><a href="current-analysis.html#reconst-other"><i class="fa fa-check"></i><b>7.3.5</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="current-analysis.html"><a href="current-analysis.html#deconvolution"><i class="fa fa-check"></i><b>7.4</b> Cell type deconvolution</a></li>
<li class="chapter" data-level="7.5" data-path="current-analysis.html"><a href="current-analysis.html#variable"><i class="fa fa-check"></i><b>7.5</b> Spatially variable genes</a><ul>
<li class="chapter" data-level="7.5.1" data-path="current-analysis.html"><a href="current-analysis.html#gpr"><i class="fa fa-check"></i><b>7.5.1</b> Gaussian process regression</a></li>
<li class="chapter" data-level="7.5.2" data-path="current-analysis.html"><a href="current-analysis.html#laplacian"><i class="fa fa-check"></i><b>7.5.2</b> Laplacian score</a></li>
<li class="chapter" data-level="7.5.3" data-path="current-analysis.html"><a href="current-analysis.html#variable-other"><i class="fa fa-check"></i><b>7.5.3</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="current-analysis.html"><a href="current-analysis.html#pattern"><i class="fa fa-check"></i><b>7.6</b> Gene patterns</a></li>
<li class="chapter" data-level="7.7" data-path="current-analysis.html"><a href="current-analysis.html#region"><i class="fa fa-check"></i><b>7.7</b> Spatial regions</a></li>
<li class="chapter" data-level="7.8" data-path="current-analysis.html"><a href="current-analysis.html#cell-interaction"><i class="fa fa-check"></i><b>7.8</b> Cell-cell interaction</a><ul>
<li class="chapter" data-level="7.8.1" data-path="current-analysis.html"><a href="current-analysis.html#l-r"><i class="fa fa-check"></i><b>7.8.1</b> Ligand-receptor pairs</a></li>
<li class="chapter" data-level="7.8.2" data-path="current-analysis.html"><a href="current-analysis.html#gene-cell-interact"><i class="fa fa-check"></i><b>7.8.2</b> Genes associated with cell-cell interaction</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="current-analysis.html"><a href="current-analysis.html#gene-interaction"><i class="fa fa-check"></i><b>7.9</b> Gene-gene interaction</a></li>
<li class="chapter" data-level="7.10" data-path="current-analysis.html"><a href="current-analysis.html#subcellular"><i class="fa fa-check"></i><b>7.10</b> Subcellular transcript localization</a></li>
<li class="chapter" data-level="7.11" data-path="current-analysis.html"><a href="current-analysis.html#HnE"><i class="fa fa-check"></i><b>7.11</b> Gene expression imputation from H&amp;E</a></li>
</ul></li>
<li class="part"><span><b>III Future perspectives</b></span></li>
<li class="chapter" data-level="8" data-path="future.html"><a href="future.html"><i class="fa fa-check"></i><b>8</b> From the past to the present to the future</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://bookdown.org" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Museum of Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="intro" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>The spatial organization of the components of biological systems is crucial for their proper function. For instance, morphogen gradients in embryos are tightly regulated to ensure that the right cell types differentiate at the right place. In adults, spatial organization of cells in tissues is important to proper functions of organs. For instance, the liver lobule is divided in labor according to distance from the portal triad as such distance affects suitability of different tasks. Both oxygen level and morphogen gradient regulate zonation of metabolism <span class="citation">(Gebhardt <a href="#ref-Gebhardt2014" role="doc-biblioref">2014</a>)</span>; there is more oxidative phosphorylation and gluconeogenesis in the more oxygenated periportal region and more glycolysis in the more deoxygenated pericentral region. How cell types and cellular functions vary in space is measured by quantifying gene expression in space. Conversely, the expression of an unknown gene in space can give clues to its function. Gene expression is usually quantified by quantifying proteins or transcripts encoded by the gene, and high throughput spatial methods exist for both protein and transcripts. In other words, cellular function exemplifies the maxim that “the whole is greater than the sum of its parts”, and in large part this follows from “location, location, location”.</p>
<p>Here we focus on spatial transcriptomics (the field of spatial proteomics is covered elsewhere <span class="citation">(Lundberg and Borner <a href="#ref-Lundberg2019" role="doc-biblioref">2019</a>; Baharlou et al. <a href="#ref-Baharlou2019" role="doc-biblioref">2019</a>; Buchberger et al. <a href="#ref-Buchberger2018" role="doc-biblioref">2018</a>)</span>). Even spatial transcriptomics is a vast field, and it is useful to begin by considering the scope of what it contains. Naïvely, one may say, spatial transcriptomics means quantifying the complete set of RNAs encoded by the genome in space. Usually the “in space” is at some microscopic resolution rather than geospatial as often assumed in the term “spatial statistics”; the resolution is usually cellular, though sometimes subcellular. The “spatial” is in contrast to other transcriptomics methods that by virtue of the nature of their assays, lose information of tissue structure in space. That is the case with microarray technology for bulk tissue analysis, for bulk RNA-seq, and single cell RNA-seq (scRNA-seq) that is based on dissociation of tissue – the “spatial” usually means tissue structure in space. More broadly, the “spatial” can mean knowing spatial context of samples although the spatial context is only a label and the coordinates are not collected or not used, such as in some laser capture microdissection (LCM) literature <span class="citation">(Baccin et al. <a href="#ref-Baccin2020" role="doc-biblioref">2020</a>; Nichterwitz et al. <a href="#ref-Nichterwitz2016" role="doc-biblioref">2016</a>)</span>, Niche-seq <span class="citation">(Medaglia et al. <a href="#ref-Medaglia2017" role="doc-biblioref">2017</a>)</span>, and APEX-seq <span class="citation">(Fazal et al. <a href="#ref-Fazal2019a" role="doc-biblioref">2019</a>)</span>. The “spatial” can also mean preserving spatial coordinates of samples within tissue, though the coordinates may or may not be explicitly used in data analysis, such as in the various single molecular fluorescent in situ hybridization (smFISH) based technologies such as seqFISH <span class="citation">(Lubeck et al. <a href="#ref-Lubeck2014" role="doc-biblioref">2014</a>)</span> and MERFISH <span class="citation">(K. H. Chen et al. <a href="#ref-Chen2015" role="doc-biblioref">2015</a>)</span> and array based technologies such as Spatial Transciptomics (ST) <span class="citation">(Ståhl et al. <a href="#ref-Stahl2016a" role="doc-biblioref">2016</a>)</span>.</p>
<p>There is more complexity in defining “transcriptomics”. While some technologies usually called “spatial transcriptomics” are indeed transcriptome-wide, such as ST, Visium, and LCM followed by RNA-seq, many technologies that only profile a panel of usually a few hundred genes are nevertheless considered part of “spatial transcriptomics”. Here “transcriptomics” actually means high-throughput quantification of gene expression, preferably highly multiplexed, quantifying numerous genes within the same piece of tissue at the same time. However, what counts as “high-throughput”? Is there a minimum number of genes required? Should 50 genes be enough? Or a hundred genes? The threshold number of genes required to be considered “high-throughput” is difficult to define; here, by “high-throughput”, we mean the intent to quantify expression of more genes than normally done with fluorescent in situ hybridization (FISH) or immunofluorescence when only color distinguishes between genes, which can mean more than about 5 genes. There is also some complication regarding whether “highly multiplexed” should be required. Some fairly recent studies that intended to perform high-throughput gene expression profiling in space did not profile most genes at the same time (e.g. multiple rounds of smFISH hybridization, each round for a different set of genes) <span class="citation">(Lignell et al. <a href="#ref-Lignell2017" role="doc-biblioref">2017</a>)</span>, or even profiled different genes in different tissue sections <span class="citation">(Bayraktar et al. <a href="#ref-Bayraktar2020" role="doc-biblioref">2020</a>; Battich, Stoeger, and Pelkmans <a href="#ref-Battich2013" role="doc-biblioref">2013</a>)</span>; these papers nevertheless claimed to be spatial transcriptomic or something similar.</p>
<p>When terms are to be defined by how they are used, then we rely on a generic and inclusive definition of “spatial transcriptomics”, which can be summarized as: Quantifying transcripts while keeping spatial context of samples within tissue or cell, with intent to quantify transcripts of more genes than normally done with one round of FISH or immunofluorescence when color is the only way to distinguish between genes. This is the criterion we used in considering what methods to include in our review.</p>
<p>The field of spatial transcriptomics has grown drastically in the past 5 years, during which several reviews have already been written. These survey existing technologies <span class="citation">(Crosetto, Bienko, and Van Oudenaarden <a href="#ref-Crosetto2015" role="doc-biblioref">2015</a>; Moor and Itzkovitz <a href="#ref-Moor2017" role="doc-biblioref">2017</a>; Strell et al. <a href="#ref-Strell2019" role="doc-biblioref">2019</a>; Liao et al. <a href="#ref-Liao2020" role="doc-biblioref">2020</a>; Waylen et al. <a href="#ref-Waylen2020" role="doc-biblioref">2020</a>)</span> or discuss how the technologies apply to specific biological systems such as tumors <span class="citation">(Smith et al. <a href="#ref-Smith2019a" role="doc-biblioref">2019</a>)</span>, brain <span class="citation">(Lein, Borm, and Linnarsson <a href="#ref-Lein2017" role="doc-biblioref">2017</a>)</span>, and liver <span class="citation">(Saviano, Henderson, and Baumert <a href="#ref-Saviano2020" role="doc-biblioref">2020</a>)</span>. Unlike the review papers, we aim to be more systematic and detailed in our review of spatial transcriptomics technology. In addition, we review existing data analysis methods in this field, a crucial aspect of spatial transcriptomics which has not yet been reviewed in depth. Moreover, we present a curated database of spatial transcriptomics literature and analyses of the literature metadata to show trends in different aspects of spatial transcriptomics. This database is publicly available <a href="https://docs.google.com/spreadsheets/d/1sJDb9B7AtYmfKv4-m8XR7uc3XXw_k4kGSout8cqZ8bY/edit#gid=1693202466">here</a>. A similar database has been curated for scRNA-seq literature, which has been analyzed to show trends in the field <span class="citation">(Svensson, da Veiga Beltrame, and Pachter <a href="#ref-Svensson2020" role="doc-biblioref">2020</a>)</span> although the metadata in our database and the analyses are much more extensive.</p>
<p>There are some caveats to our review and database. First, while we narrate a history of evolution of techniques and in some cases explain how one technique influenced another, we do not present aspects of the history that are not apparent from the publications. Studying those aspects of the history of the field may require interviewing the people who developed the techniques, as well as exploration of additional unpublished material. Second, our database was originally only meant for papers, so relevant materials that are not in presented in that format are underrepresented. Examples of such materials include databases and software not presented as papers (e.g. the XDB3 database <span class="citation">(“XDB3” <a href="#ref-XDB3" role="doc-biblioref">2004</a>)</span>). This means that the metadata analyses in this book might not be representative of all material that exists in spatial transcriptomics. Third, as the curation was done manually, the database might not include some relevant literature unknown to us.</p>
<p>Our database starts with articles published in the 1980s. This provides historical context of what is now commonly known as spatial transcriptomics; this literature is summarized in Chapter <a href="prequel.html#prequel">2</a>, and historical methods of data analysis are reviewed in Chapter <a href="prequel-analysis.html#prequel-analysis">3</a>. The literature is broken down into the following categories, to be defined and elaborated on in the subsequent chapters. Technologies to collect data (Chapter <a href="current.html#current">4</a>): Microdissection (Section <a href="current-techs.html#microdissection">5.1</a>), array based techniques (Section <a href="current-techs.html#array">5.4</a>), single molecular FISH (smFISH) (Section <a href="current-techs.html#smfish">5.2</a>), in situ sequencing (ISS) (Section <a href="current-techs.html#iss">5.3</a>), and no imaging (Section <a href="current-techs.html#no-imaging">5.5</a>). Data analysis methods (Chapter <a href="current-analysis.html#current-analysis">7</a>): Preprocessing (Section <a href="current-analysis.html#preprocessing">7.1</a>), exploratory data analysis (EDA) (Section <a href="current-analysis.html#eda">7.2</a>), spatial reconstruction of single cell RNA-seq (scRNA-seq) data (Section <a href="current-analysis.html#reconstruction">7.3</a>), spatially variable genes (Section <a href="current-analysis.html#variable">7.5</a>), archetypal gene expression patterns (Section <a href="current-analysis.html#pattern">7.6</a>), using transcriptome to identify spatially coherent regions in tissue (Section <a href="current-analysis.html#region">7.7</a>), cell type deconvolution of non-single-cell resolution spatial data (Section <a href="current-analysis.html#deconvolution">7.4</a>), cell-cell interaction (Section <a href="current-analysis.html#cell-interaction">7.8</a>), and other types of analyses. Also, the literature metadata is analyzed to show relevant sociological trends such as who is using each technology, usage trends of technologies, and the programming languages used. The metadata analyses can be run interactively in <a href="https://rstudio.cloud/project/1981124">RStudio Cloud</a>.</p>

</div>



<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Baccin2020">
<p>Baccin, Chiara, Jude Al-Sabah, Lars Velten, Patrick M. Helbling, Florian Grünschläger, Pablo Hernández-Malmierca, César Nombela-Arrieta, Lars M. Steinmetz, Andreas Trumpp, and Simon Haas. 2020. “Combined single-cell and spatial transcriptomics reveal the molecular, cellular and spatial bone marrow niche organization.” <em>Nature Cell Biology</em> 22 (1): 38–48. <a href="https://doi.org/10.1038/s41556-019-0439-6">https://doi.org/10.1038/s41556-019-0439-6</a>.</p>
</div>
<div id="ref-Baharlou2019">
<p>Baharlou, Heeva, Nicolas P. Canete, Anthony L. Cunningham, Andrew N. Harman, and Ellis Patrick. 2019. “Mass Cytometry Imaging for the Study of Human Diseases—Applications and Data Analysis Strategies.” <em>Frontiers in Immunology</em> 10 (November): 2657. <a href="https://doi.org/10.3389/fimmu.2019.02657">https://doi.org/10.3389/fimmu.2019.02657</a>.</p>
</div>
<div id="ref-Battich2013">
<p>Battich, Nico, Thomas Stoeger, and Lucas Pelkmans. 2013. “Image-based transcriptomics in thousands of single human cells at single-molecule resolution.” <em>Nature Methods</em> 10 (11): 1127–36. <a href="https://doi.org/10.1038/nmeth.2657">https://doi.org/10.1038/nmeth.2657</a>.</p>
</div>
<div id="ref-Bayraktar2020">
<p>Bayraktar, Omer Ali, Theresa Bartels, Staffan Holmqvist, Vitalii Kleshchevnikov, Araks Martirosyan, Damon Polioudakis, Lucile Ben Haim, et al. 2020. “Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map.” <em>Nature Neuroscience</em> 23 (4): 500–509. <a href="https://doi.org/10.1038/s41593-020-0602-1">https://doi.org/10.1038/s41593-020-0602-1</a>.</p>
</div>
<div id="ref-Buchberger2018">
<p>Buchberger, Amanda Rae, Kellen DeLaney, Jillian Johnson, and Lingjun Li. 2018. “Mass Spectrometry Imaging: A Review of Emerging Advancements and Future Insights.” <em>Analytical Chemistry</em> 90 (1): 240–65. <a href="https://doi.org/10.1021/acs.analchem.7b04733">https://doi.org/10.1021/acs.analchem.7b04733</a>.</p>
</div>
<div id="ref-Chen2015">
<p>Chen, Kok Hao, Alistair N. Boettiger, Jeffrey R. Moffitt, Siyuan Wang, and Xiaowei Zhuang. 2015. “Spatially resolved, highly multiplexed RNA profiling in single cells.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaa6090">https://doi.org/10.1126/science.aaa6090</a>.</p>
</div>
<div id="ref-Crosetto2015">
<p>Crosetto, Nicola, Magda Bienko, and Alexander Van Oudenaarden. 2015. “Spatially resolved transcriptomics and beyond.” <a href="https://doi.org/10.1038/nrg3832">https://doi.org/10.1038/nrg3832</a>.</p>
</div>
<div id="ref-Fazal2019a">
<p>Fazal, Furqan M., Shuo Han, Kevin R. Parker, Pornchai Kaewsapsak, Jin Xu, Alistair N. Boettiger, Howard Y. Chang, and Alice Y. Ting. 2019. “Atlas of Subcellular RNA Localization Revealed by APEX-Seq.” <em>Cell</em> 178 (2): 473–490.e26. <a href="https://doi.org/10.1016/j.cell.2019.05.027">https://doi.org/10.1016/j.cell.2019.05.027</a>.</p>
</div>
<div id="ref-Gebhardt2014">
<p>Gebhardt, Rolf. 2014. “Liver zonation: Novel aspects of its regulation and its impact on homeostasis.” <em>World Journal of Gastroenterology</em> 20 (26): 8491. <a href="https://doi.org/10.3748/wjg.v20.i26.8491">https://doi.org/10.3748/wjg.v20.i26.8491</a>.</p>
</div>
<div id="ref-Lein2017">
<p>Lein, Ed, Lars E. Borm, and Sten Linnarsson. 2017. “The promise of spatial transcriptomics for neuroscience in the era of molecular cell typing.” <a href="https://doi.org/10.1126/science.aan6827">https://doi.org/10.1126/science.aan6827</a>.</p>
</div>
<div id="ref-Liao2020">
<p>Liao, Jie, Xiaoyan Lu, Xin Shao, Ling Zhu, and Xiaohui Fan. 2020. “Uncovering an Organ’s Molecular Architecture at Single-Cell Resolution by Spatially Resolved Transcriptomics.” <em>Trends in Biotechnology</em>, June. <a href="https://doi.org/10.1016/j.tibtech.2020.05.006">https://doi.org/10.1016/j.tibtech.2020.05.006</a>.</p>
</div>
<div id="ref-Lignell2017">
<p>Lignell, Antti, Laura Kerosuo, Sebastian J. Streichan, Long Cai, and Marianne E. Bronner. 2017. “Identification of a neural crest stem cell niche by Spatial Genomic Analysis.” <em>Nature Communications</em> 8 (1): 1830. <a href="https://doi.org/10.1038/s41467-017-01561-w">https://doi.org/10.1038/s41467-017-01561-w</a>.</p>
</div>
<div id="ref-Lubeck2014">
<p>Lubeck, Eric, Ahmet F. Coskun, Timur Zhiyentayev, Mubhij Ahmad, and Long Cai. 2014. “Single-cell in situ RNA profiling by sequential hybridization.” Nature Publishing Group. <a href="https://doi.org/10.1038/nmeth.2892">https://doi.org/10.1038/nmeth.2892</a>.</p>
</div>
<div id="ref-Lundberg2019">
<p>Lundberg, Emma, and Georg H. H. Borner. 2019. “Spatial proteomics: a powerful discovery tool for cell biology.” <em>Nature Reviews Molecular Cell Biology</em> 20 (5): 285–302. <a href="https://doi.org/10.1038/s41580-018-0094-y">https://doi.org/10.1038/s41580-018-0094-y</a>.</p>
</div>
<div id="ref-Medaglia2017">
<p>Medaglia, Chiara, Amir Giladi, Liat Stoler-Barak, Marco De Giovanni, Tomer Meir Salame, Adi Biram, Eyal David, et al. 2017. “Spatial reconstruction of immune niches by combining photoactivatable reporters and scRNA-seq.” <em>Science</em> 358 (6370): 1622–6. <a href="https://doi.org/10.1126/science.aao4277">https://doi.org/10.1126/science.aao4277</a>.</p>
</div>
<div id="ref-Moor2017">
<p>Moor, Andreas E., and Shalev Itzkovitz. 2017. “Spatial transcriptomics: paving the way for tissue-level systems biology.” <em>Current Opinion in Biotechnology</em> 46 (August): 126–33. <a href="https://doi.org/10.1016/j.copbio.2017.02.004">https://doi.org/10.1016/j.copbio.2017.02.004</a>.</p>
</div>
<div id="ref-Nichterwitz2016">
<p>Nichterwitz, Susanne, Geng Chen, Julio Aguila Benitez, Marlene Yilmaz, Helena Storvall, Ming Cao, Rickard Sandberg, Qiaolin Deng, and Eva Hedlund. 2016. “Laser capture microscopy coupled with Smart-seq2 for precise spatial transcriptomic profiling.” <em>Nature Communications</em> 7 (1): 12139. <a href="https://doi.org/10.1038/ncomms12139">https://doi.org/10.1038/ncomms12139</a>.</p>
</div>
<div id="ref-Saviano2020">
<p>Saviano, Antonio, Neil C. Henderson, and Thomas F. Baumert. 2020. “Single-cell genomics and spatial transcriptomics: Discovery of novel cell states and cellular interactions in liver physiology and disease biology.” <em>Journal of Hepatology</em> 73 (5): 1219–30. <a href="https://doi.org/10.1016/j.jhep.2020.06.004">https://doi.org/10.1016/j.jhep.2020.06.004</a>.</p>
</div>
<div id="ref-Smith2019a">
<p>Smith, Constance M, Terry F. Hayamizu, Jacqueline H. Finger, Susan M. Bello, Ingeborg J. McCright, Jingxia Xu, Richard M. Baldarelli, et al. 2019. “The mouse Gene Expression Database (GXD): 2019 update.” <em>Nucleic Acids Research</em> 47 (D1): D774–D779. <a href="https://doi.org/10.1093/nar/gky922">https://doi.org/10.1093/nar/gky922</a>.</p>
</div>
<div id="ref-Stahl2016a">
<p>Ståhl, Patrik L., Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. “Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.” <em>Science</em> 353 (6294): 78–82. <a href="https://doi.org/10.1126/science.aaf2403">https://doi.org/10.1126/science.aaf2403</a>.</p>
</div>
<div id="ref-Strell2019">
<p>Strell, Carina, Markus M. Hilscher, Navya Laxman, Jessica Svedlund, Chenglin Wu, Chika Yokota, and Mats Nilsson. 2019. “Placing RNA in context and space – methods for spatially resolved transcriptomics.” <a href="https://doi.org/10.1111/febs.14435">https://doi.org/10.1111/febs.14435</a>.</p>
</div>
<div id="ref-Svensson2020">
<p>Svensson, Valentine, Eduardo da Veiga Beltrame, and Lior Pachter. 2020. “A curated database reveals trends in single-cell transcriptomics.” <em>Database</em> 2020 (January). <a href="https://doi.org/10.1093/database/baaa073">https://doi.org/10.1093/database/baaa073</a>.</p>
</div>
<div id="ref-Waylen2020">
<p>Waylen, Lisa N, Hieu T Nim, Luciano G Martelotto, and Mirana Ramialison. 2020. “From whole-mount to single-cell spatial assessment of gene expression in 3D.” <em>Communications Biology</em> 3 (1): 602. <a href="https://doi.org/10.1038/s42003-020-01341-1">https://doi.org/10.1038/s42003-020-01341-1</a>.</p>
</div>
<div id="ref-XDB3">
<p>“XDB3.” 2004. <a href="http://xenopus.nibb.ac.jp/">http://xenopus.nibb.ac.jp/</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="prequel.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["museumst.pdf"],
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
