<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Prequel era | Museum of Spatial Transcriptomics</title>
  <meta name="description" content="Chapter 2 Prequel era | Museum of Spatial Transcriptomics" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Prequel era | Museum of Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Chapter 2 Prequel era | Museum of Spatial Transcriptomics" />
  <meta name="github-repo" content="pachterlab/LP_2021" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Prequel era | Museum of Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="Chapter 2 Prequel era | Museum of Spatial Transcriptomics" />
  

<meta name="author" content="Lambda Moses, dlu2@caltech.edu" />
<meta name="author" content="Lior Pachter, lpachter@caltech.edu" />


<meta name="date" content="2021-07-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="intro.html"/>
<link rel="next" href="prequel-analysis.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Museum of Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="part"><span><b>I Prequel era</b></span></li>
<li class="chapter" data-level="2" data-path="prequel.html"><a href="prequel.html"><i class="fa fa-check"></i><b>2</b> Prequel era</a><ul>
<li class="chapter" data-level="2.1" data-path="prequel.html"><a href="prequel.html#traps"><i class="fa fa-check"></i><b>2.1</b> Enhancer and gene traps</a></li>
<li class="chapter" data-level="2.2" data-path="prequel.html"><a href="prequel.html#in-situ-reporter"><i class="fa fa-check"></i><b>2.2</b> In situ reporter</a></li>
<li class="chapter" data-level="2.3" data-path="prequel.html"><a href="prequel.html#wmish"><i class="fa fa-check"></i><b>2.3</b> ISH and WMISH atlases</a></li>
<li class="chapter" data-level="2.4" data-path="prequel.html"><a href="prequel.html#database_prequel"><i class="fa fa-check"></i><b>2.4</b> Databases of the prequel era</a></li>
<li class="chapter" data-level="2.5" data-path="prequel.html"><a href="prequel.html#geo_prequel"><i class="fa fa-check"></i><b>2.5</b> Geography of the prequel era</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prequel-analysis.html"><a href="prequel-analysis.html"><i class="fa fa-check"></i><b>3</b> Data analysis in the prequel era</a><ul>
<li class="chapter" data-level="3.1" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-patterns"><i class="fa fa-check"></i><b>3.1</b> Gene patterns</a></li>
<li class="chapter" data-level="3.2" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-spatial-regions"><i class="fa fa-check"></i><b>3.2</b> Spatial regions</a></li>
<li class="chapter" data-level="3.3" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-interactions"><i class="fa fa-check"></i><b>3.3</b> Gene interactions</a></li>
<li class="chapter" data-level="3.4" data-path="prequel-analysis.html"><a href="prequel-analysis.html#decline"><i class="fa fa-check"></i><b>3.4</b> Decline</a></li>
<li class="chapter" data-level="3.5" data-path="prequel-analysis.html"><a href="prequel-analysis.html#geo-prequel-analysis"><i class="fa fa-check"></i><b>3.5</b> Geography of prequel data analysis</a></li>
</ul></li>
<li class="part"><span><b>II Current era</b></span></li>
<li class="chapter" data-level="4" data-path="current.html"><a href="current.html"><i class="fa fa-check"></i><b>4</b> From the past to the present</a></li>
<li class="chapter" data-level="5" data-path="current-techs.html"><a href="current-techs.html"><i class="fa fa-check"></i><b>5</b> Current era technologies</a><ul>
<li class="chapter" data-level="5.1" data-path="current-techs.html"><a href="current-techs.html#microdissection"><i class="fa fa-check"></i><b>5.1</b> Microdissection</a><ul>
<li class="chapter" data-level="5.1.1" data-path="current-techs.html"><a href="current-techs.html#history-of-lcm"><i class="fa fa-check"></i><b>5.1.1</b> History of LCM</a></li>
<li class="chapter" data-level="5.1.2" data-path="current-techs.html"><a href="current-techs.html#lcm-usage"><i class="fa fa-check"></i><b>5.1.2</b> Usage of LCM</a></li>
<li class="chapter" data-level="5.1.3" data-path="current-techs.html"><a href="current-techs.html#physical-microdissection"><i class="fa fa-check"></i><b>5.1.3</b> Physical microdissection</a></li>
<li class="chapter" data-level="5.1.4" data-path="current-techs.html"><a href="current-techs.html#de-facto-microdissection"><i class="fa fa-check"></i><b>5.1.4</b> <em>De facto</em> microdissection</a></li>
<li class="chapter" data-level="5.1.5" data-path="current-techs.html"><a href="current-techs.html#summary"><i class="fa fa-check"></i><b>5.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="current-techs.html"><a href="current-techs.html#smfish"><i class="fa fa-check"></i><b>5.2</b> Single molecular FISH</a><ul>
<li class="chapter" data-level="5.2.1" data-path="current-techs.html"><a href="current-techs.html#barcoding"><i class="fa fa-check"></i><b>5.2.1</b> Barcoding strategies</a></li>
<li class="chapter" data-level="5.2.2" data-path="current-techs.html"><a href="current-techs.html#amplification"><i class="fa fa-check"></i><b>5.2.2</b> Signal amplification</a></li>
<li class="chapter" data-level="5.2.3" data-path="current-techs.html"><a href="current-techs.html#crowding"><i class="fa fa-check"></i><b>5.2.3</b> Optical crowding</a></li>
<li class="chapter" data-level="5.2.4" data-path="current-techs.html"><a href="current-techs.html#smfish-usage"><i class="fa fa-check"></i><b>5.2.4</b> Usage of smFISH based techniques</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="current-techs.html"><a href="current-techs.html#iss"><i class="fa fa-check"></i><b>5.3</b> In situ sequencing</a><ul>
<li class="chapter" data-level="5.3.1" data-path="current-techs.html"><a href="current-techs.html#solid"><i class="fa fa-check"></i><b>5.3.1</b> SOLiD and FISSEQ</a></li>
<li class="chapter" data-level="5.3.2" data-path="current-techs.html"><a href="current-techs.html#cpal"><i class="fa fa-check"></i><b>5.3.2</b> cPAL and ISS</a></li>
<li class="chapter" data-level="5.3.3" data-path="current-techs.html"><a href="current-techs.html#sedal"><i class="fa fa-check"></i><b>5.3.3</b> SEDAL and STARmap</a></li>
<li class="chapter" data-level="5.3.4" data-path="current-techs.html"><a href="current-techs.html#sbs"><i class="fa fa-check"></i><b>5.3.4</b> Sequencing by synthesis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="current-techs.html"><a href="current-techs.html#array"><i class="fa fa-check"></i><b>5.4</b> In situ array capture</a></li>
<li class="chapter" data-level="5.5" data-path="current-techs.html"><a href="current-techs.html#no-imaging"><i class="fa fa-check"></i><b>5.5</b> No imaging</a></li>
<li class="chapter" data-level="5.6" data-path="current-techs.html"><a href="current-techs.html#multiomics"><i class="fa fa-check"></i><b>5.6</b> Spatial multi-omics</a></li>
<li class="chapter" data-level="5.7" data-path="current-techs.html"><a href="current-techs.html#database-current"><i class="fa fa-check"></i><b>5.7</b> Databases of the current era</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="text-mining.html"><a href="text-mining.html"><i class="fa fa-check"></i><b>6</b> Text mining LCM transcriptomics abstracts</a><ul>
<li class="chapter" data-level="6.1" data-path="text-mining.html"><a href="text-mining.html#topic-model"><i class="fa fa-check"></i><b>6.1</b> Topic modeling</a></li>
<li class="chapter" data-level="6.2" data-path="text-mining.html"><a href="text-mining.html#word-time"><i class="fa fa-check"></i><b>6.2</b> Changes of word usage through time</a></li>
<li class="chapter" data-level="6.3" data-path="text-mining.html"><a href="text-mining.html#topic-time"><i class="fa fa-check"></i><b>6.3</b> Changes of topic prevalence through time</a></li>
<li class="chapter" data-level="6.4" data-path="text-mining.html"><a href="text-mining.html#association-of-topics-with-city"><i class="fa fa-check"></i><b>6.4</b> Association of topics with city</a></li>
<li class="chapter" data-level="6.5" data-path="text-mining.html"><a href="text-mining.html#glove"><i class="fa fa-check"></i><b>6.5</b> GloVe word embedding</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="current-analysis.html"><a href="current-analysis.html"><i class="fa fa-check"></i><b>7</b> Data analysis in the current era</a><ul>
<li class="chapter" data-level="7.1" data-path="current-analysis.html"><a href="current-analysis.html#preprocessing"><i class="fa fa-check"></i><b>7.1</b> Preprocessing</a><ul>
<li class="chapter" data-level="7.1.1" data-path="current-analysis.html"><a href="current-analysis.html#registration"><i class="fa fa-check"></i><b>7.1.1</b> Image registration</a></li>
<li class="chapter" data-level="7.1.2" data-path="current-analysis.html"><a href="current-analysis.html#calling"><i class="fa fa-check"></i><b>7.1.2</b> Spot and barcode calling</a></li>
<li class="chapter" data-level="7.1.3" data-path="current-analysis.html"><a href="current-analysis.html#segmentation"><i class="fa fa-check"></i><b>7.1.3</b> Cell segmentation</a></li>
<li class="chapter" data-level="7.1.4" data-path="current-analysis.html"><a href="current-analysis.html#seg-alternatives"><i class="fa fa-check"></i><b>7.1.4</b> Alternatives to cell segmentation</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="current-analysis.html"><a href="current-analysis.html#eda"><i class="fa fa-check"></i><b>7.2</b> Exploratory data analysis</a></li>
<li class="chapter" data-level="7.3" data-path="current-analysis.html"><a href="current-analysis.html#reconstruction"><i class="fa fa-check"></i><b>7.3</b> Spatial reconstruction of scRNA-seq data</a><ul>
<li class="chapter" data-level="7.3.1" data-path="current-analysis.html"><a href="current-analysis.html#direct-dr"><i class="fa fa-check"></i><b>7.3.1</b> Direct dimension reduction</a></li>
<li class="chapter" data-level="7.3.2" data-path="current-analysis.html"><a href="current-analysis.html#ad-hoc"><i class="fa fa-check"></i><b>7.3.2</b> <em>Ad hoc</em> scoring</a></li>
<li class="chapter" data-level="7.3.3" data-path="current-analysis.html"><a href="current-analysis.html#generative"><i class="fa fa-check"></i><b>7.3.3</b> Generative models</a></li>
<li class="chapter" data-level="7.3.4" data-path="current-analysis.html"><a href="current-analysis.html#latent"><i class="fa fa-check"></i><b>7.3.4</b> Shared latent space</a></li>
<li class="chapter" data-level="7.3.5" data-path="current-analysis.html"><a href="current-analysis.html#reconst-other"><i class="fa fa-check"></i><b>7.3.5</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="current-analysis.html"><a href="current-analysis.html#deconvolution"><i class="fa fa-check"></i><b>7.4</b> Cell type deconvolution</a></li>
<li class="chapter" data-level="7.5" data-path="current-analysis.html"><a href="current-analysis.html#variable"><i class="fa fa-check"></i><b>7.5</b> Spatially variable genes</a><ul>
<li class="chapter" data-level="7.5.1" data-path="current-analysis.html"><a href="current-analysis.html#gpr"><i class="fa fa-check"></i><b>7.5.1</b> Gaussian process regression</a></li>
<li class="chapter" data-level="7.5.2" data-path="current-analysis.html"><a href="current-analysis.html#laplacian"><i class="fa fa-check"></i><b>7.5.2</b> Laplacian score</a></li>
<li class="chapter" data-level="7.5.3" data-path="current-analysis.html"><a href="current-analysis.html#variable-other"><i class="fa fa-check"></i><b>7.5.3</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="current-analysis.html"><a href="current-analysis.html#pattern"><i class="fa fa-check"></i><b>7.6</b> Gene patterns</a></li>
<li class="chapter" data-level="7.7" data-path="current-analysis.html"><a href="current-analysis.html#region"><i class="fa fa-check"></i><b>7.7</b> Spatial regions</a></li>
<li class="chapter" data-level="7.8" data-path="current-analysis.html"><a href="current-analysis.html#cell-interaction"><i class="fa fa-check"></i><b>7.8</b> Cell-cell interaction</a><ul>
<li class="chapter" data-level="7.8.1" data-path="current-analysis.html"><a href="current-analysis.html#l-r"><i class="fa fa-check"></i><b>7.8.1</b> Ligand-receptor pairs</a></li>
<li class="chapter" data-level="7.8.2" data-path="current-analysis.html"><a href="current-analysis.html#gene-cell-interact"><i class="fa fa-check"></i><b>7.8.2</b> Genes associated with cell-cell interaction</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="current-analysis.html"><a href="current-analysis.html#gene-interaction"><i class="fa fa-check"></i><b>7.9</b> Gene-gene interaction</a></li>
<li class="chapter" data-level="7.10" data-path="current-analysis.html"><a href="current-analysis.html#subcellular"><i class="fa fa-check"></i><b>7.10</b> Subcellular transcript localization</a></li>
<li class="chapter" data-level="7.11" data-path="current-analysis.html"><a href="current-analysis.html#HnE"><i class="fa fa-check"></i><b>7.11</b> Gene expression imputation from H&amp;E</a></li>
</ul></li>
<li class="part"><span><b>III Future perspectives</b></span></li>
<li class="chapter" data-level="8" data-path="future.html"><a href="future.html"><i class="fa fa-check"></i><b>8</b> From the past to the present to the future</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://bookdown.org" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Museum of Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="prequel" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Prequel era</h1>
<p>Some previous reviews on spatial transcriptomics start the history of spatial transcriptomics with laser capture microdissection (LCM) followed by microarray or RNA-seq and single molecular fluorescent <em>in situ</em> hybridization (smFISH) in the late 1990s <span class="citation">(Lein, Borm, and Linnarsson <a href="#ref-Lein2017" role="doc-biblioref">2017</a>; Liao et al. <a href="#ref-Liao2020" role="doc-biblioref">2020</a>; Crosetto, Bienko, and Van Oudenaarden <a href="#ref-Crosetto2015" role="doc-biblioref">2015</a>)</span>. We will discuss these later, but note that by 1999 and the early 2000s, when the earliest LCM microarray studies were published <span class="citation">(Luo et al. <a href="#ref-Luo1999" role="doc-biblioref">1999</a>; Sgroi et al. <a href="#ref-Sgroi1999" role="doc-biblioref">1999</a>; Ohyama et al. <a href="#ref-Ohyama2000" role="doc-biblioref">2000</a>; Kitahara et al. <a href="#ref-Kitahara2001" role="doc-biblioref">2001</a>)</span>, the quest to profile the transcriptome in space had already begun, with enhancer and gene trap screens, <em>in situ</em> reporter screens, and (whole mount) <em>in situ</em> hybridization ((WM)ISH) atlases. Although this early literature, dating from the late 1980s, generally does not refer to itself as “spatial transcriptomics”, it fits into the definition of spatial transcriptomics as stated in Chapter <a href="intro.html#intro">1</a>.</p>
<p>We call this body of literature “prequel”, because first, its origin predates LCM microarray. Second, unlike most technologies covered by existing spatial transcriptomics reviews, the techniques used were not multiplexed and were less quantitative, and as a result, they have fallen out of favor. In contrast, what comes after “prequel” will be called “current”, although the prequel and current eras chronologically overlap. Given what current era spatial transcriptomics is commonly perceived to be, here “prequel” is broadly defined as methods that fulfill the more relaxed definition of “spatial transcriptomics” in this book, but do not involve cDNA microarray, next generation sequencing (NGS), or single molecular imaging.</p>
<p>There are 204 prequel papers in our database. Prequel literature is included in the database and covered here for the following reasons. First, the legacy of the prequel era has influenced more recent spatial transcriptomic research; the present and future are shaped by the past. For example, spatial reconstruction of scRNA-seq data in Seurat v1 <span class="citation">(Satija et al. <a href="#ref-Satija2015" role="doc-biblioref">2015</a>)</span>, the Achim et al. <em>Platynereis</em> study <span class="citation">(Achim et al. <a href="#ref-Achim2015" role="doc-biblioref">2015</a>)</span>, <code>DistMap</code> <span class="citation">(Karaiskos et al. <a href="#ref-Karaiskos2017" role="doc-biblioref">2017</a>)</span>, and the Zeisel et al. Mouse Brain Atlas <span class="citation">(Zeisel et al. <a href="#ref-Zeisel2018" role="doc-biblioref">2018</a>)</span> used (WM)ISH atlases as spatial references. Recent Spatial Transcriptomics<sup>TM</sup> (ST) mouse brain data are still compared to the ISH atlas of Allen Brain Atlas (ABA) <span class="citation">(Ortiz et al. <a href="#ref-Ortiz2020" role="doc-biblioref">2020</a>; Chen et al. <a href="#ref-Chen2020" role="doc-biblioref">2020</a>)</span>. A study on spatial reconstruction of scATAC-seq data compared the <em>in silico</em> reconstruction to the FlyLight <em>Drosophila</em> enhancer atlas <span class="citation">(Jenett et al. <a href="#ref-Jenett2012" role="doc-biblioref">2012</a>; Bravo González-Blas et al. <a href="#ref-BravoGonzalez-Blas2020" role="doc-biblioref">2020</a>)</span>. Hence prequel resources can still be useful in the current era. Second, some features of the prequel era may benefit future spatial transcriptomics studies; this will be discussed after more recent technologies are reviewed. Third, the various quests in the current era have already begun in the prequel era, and this history can show where we are in these quests.</p>
<p>Fourth, as shown later in this book, existing current era spatial transcriptomics data are by and large from humans and mice, and especially the brain (Figure <a href="current.html#fig:species-pie">4.4</a>, Figure <a href="prequel.html#fig:anat1">2.7</a>). For other model and non-model organisms (e.g. <em>Xenopus laevis</em> <span class="citation">(Bowes et al. <a href="#ref-Bowes2009" role="doc-biblioref">2009</a>; “XDB3” <a href="#ref-XDB3" role="doc-biblioref">2004</a>)</span>, <em>Ciona intestinalis</em> <span class="citation">(Satou et al. <a href="#ref-Satou2001" role="doc-biblioref">2001</a>)</span>, <em>Danio rerio</em> <span class="citation">(Sprague <a href="#ref-Sprague2003" role="doc-biblioref">2003</a>; Belmamoune and Verbeek <a href="#ref-Belmamoune2008" role="doc-biblioref">2008</a>)</span>, <em>Oryzias latipes</em> <span class="citation">(Henrich <a href="#ref-Henrich2003" role="doc-biblioref">2003</a>)</span>, <em>Gallus gallus</em> <span class="citation">(Bell, Yatskievych, and Antin <a href="#ref-Bell2004" role="doc-biblioref">2004</a>)</span>, <em>Taeniopygia guttata</em> <span class="citation">(Lovell et al. <a href="#ref-Lovell2020" role="doc-biblioref">2020</a>)</span>, and to some extent, even <em>Drosophila melanogaster</em> <span class="citation">(Tomancak et al. <a href="#ref-Tomancak2002" role="doc-biblioref">2002</a>; Luengo Hendriks et al. <a href="#ref-LuengoHendriks2006a" role="doc-biblioref">2006</a>)</span>), some tissues other than the brain (e.g. lung <span class="citation">(Ardini-Poleske et al. <a href="#ref-Ardini-Poleske2017" role="doc-biblioref">2017</a>)</span>, retina <span class="citation">(Blackshaw et al. <a href="#ref-Blackshaw2004" role="doc-biblioref">2004</a>)</span>, genitourinary tract <span class="citation">(Harding et al. <a href="#ref-Harding2011" role="doc-biblioref">2011</a>)</span>), and miRNAs <span class="citation">(Ahmed et al. <a href="#ref-Ahmed2015" role="doc-biblioref">2015</a>; Karali et al. <a href="#ref-Karali2010" role="doc-biblioref">2010</a>; Diez-Roux et al. <a href="#ref-Diez-Roux2011" role="doc-biblioref">2011</a>; Aboobaker et al. <a href="#ref-Aboobaker2005" role="doc-biblioref">2005</a>; Wienholds <a href="#ref-Wienholds2005" role="doc-biblioref">2005</a>; Darnell et al. <a href="#ref-Darnell2006" role="doc-biblioref">2006</a>)</span>, the most comprehensive spatial transcriptomic resources, if any are are available at all, are still (WM)ISH atlases. For plants, the most comprehensive resources can still be enhancer and gene trap screens <span class="citation">(Johnson et al. <a href="#ref-Johnson2005" role="doc-biblioref">2005</a>; Nakayama et al. <a href="#ref-Nakayama2005" role="doc-biblioref">2005</a>)</span>. Hence, while current era technologies may produce more quantitative and highly multiplexed data, they have not completely superseded (WM)ISH atlases. Finally, the historical literature is curated for the same reason why museums and libraries keep historical maps and scientific works that have been superseded by more recent work; it is part of our heritage.</p>
<p>An overall timeline for prequel techniques is shown in Figure <a href="prequel.html#fig:tl1">2.1</a>, which will be discussed in more details in the rest of this chapter.</p>
<div class="figure" style="text-align: center"><span id="fig:tl1"></span>
<img src="02-prequel_files/figure-html/tl1-1.png" alt="Timeline of prequel techniques." width="768" />
<p class="caption">
Figure 2.1: Timeline of prequel techniques.
</p>
</div>
<div id="traps" class="section level2">
<h2><span class="header-section-number">2.1</span> Enhancer and gene traps</h2>
<p>Long before the advent of reference genomes for common model organisms, the quest to characterize genes based on expression pattern in space had already begun. The earliest high-throughput efforts to identify and characterize such genes were enhancer traps. To the best of our knowledge, the first use of a reporter to visualize gene expression in space was reported in 1983. It used lacZ fused to sequences upstream to the hsp70 gene encoding a heat shock protein in <em>Drosophila melanogaster</em> and inserted into the genome with P element to characterize the puffs formed in polytene chromosomes and the tissue distribution of hsp70 in response to heat shock <span class="citation">(Lis, Simon, and Sutton <a href="#ref-Lis1983" role="doc-biblioref">1983</a>)</span>.</p>
<p>The first enhancer trap screen in <em>Drosophila melanogaster</em> was published in 1987 <span class="citation">(O’Kane and Gehring <a href="#ref-OKane1987" role="doc-biblioref">1987</a>)</span>. The P element is a transposable element found in <em>Drosophila</em>. In an enhancer trap vector, a reporter gene, such as lacZ, here with the polyadenylation site of the hsp70 gene, and a marker gene with its own promoter that can be used to identify individuals and their offspring with the vector integrated into the germline, such as rosy which can be used in <em>Drosophila</em> to identify the individuals with eye color, are flanked by the 5’ and 3’ ends of the P element necessary for transposition (Figure <a href="prequel.html#fig:trap1">2.2</a>). The vector is injected into <em>Drosophila</em> embryos before the formation of pole cells <span class="citation">(Spradling and Rubin <a href="#ref-Spradling1982" role="doc-biblioref">1982</a>)</span>. As a transposon, the construct is randomly inserted into the genome, and since the P element promoter is so weak that an enhancer is required for the promoter to drive transcription of the reporter gene, the location of the reporter gene expression marks where the enhancer is active. As the transposon is inserted into different locations of the genome in different individuals, each individual that has the vector integrated into the germline forms a transformant line. In <em>Drosophila</em>, in many cases, expression patterns of <span class="math inline">\(\beta\)</span>-galactosidase do reflect expression pattern of a nearby gene <span class="citation">(Bellen et al. <a href="#ref-Bellen1989" role="doc-biblioref">1989</a>; Wilson et al. <a href="#ref-Wilson1989" role="doc-biblioref">1989</a>)</span>.</p>

<div class="figure" style="text-align: center"><span id="fig:trap1"></span>
<img src="fig1A.png" alt="Illustrations of enhancer trap as described in (O’Kane and Gehring 1987) and gene trap as described in (Gossler et al. 1989) (Created with BioRender.com)." width="100%" />
<p class="caption">
Figure 2.2: Illustrations of enhancer trap as described in <span class="citation">(O’Kane and Gehring <a href="#ref-OKane1987" role="doc-biblioref">1987</a>)</span> and gene trap as described in <span class="citation">(Gossler et al. <a href="#ref-Gossler1989" role="doc-biblioref">1989</a>)</span> (Created with BioRender.com).
</p>
</div>
<p>Since then, different vectors have been developed for better efficiency and flexibility <span class="citation">(Stanford, Cohn, and Cordes <a href="#ref-Stanford2001" role="doc-biblioref">2001</a>)</span>, and enhancer traps have been applied at increasing scale. The 1987 study recovered 39 lines <span class="citation">(O’Kane and Gehring <a href="#ref-OKane1987" role="doc-biblioref">1987</a>)</span>, possibly characterizing 39 genes, but already in 1989, over 3000 lines were possible in one study <span class="citation">(Bier et al. <a href="#ref-Bier1989" role="doc-biblioref">1989</a>)</span>. Enhancer trapping was also adapted to other species, such as mouse <span class="citation">(Gossler et al. <a href="#ref-Gossler1989" role="doc-biblioref">1989</a>; Allen et al. <a href="#ref-Allen1988" role="doc-biblioref">1988</a>)</span> and <em>Arabidopsis thaliana</em> <span class="citation">(Sundaresan et al. <a href="#ref-Sundaresan1995" role="doc-biblioref">1995</a>)</span>.</p>
<p>Enhancer traps were not intended to be mutagenic <span class="citation">(O’Kane and Gehring <a href="#ref-OKane1987" role="doc-biblioref">1987</a>)</span>, nor is it highly mutagenic <span class="citation">(Stanford, Cohn, and Cordes <a href="#ref-Stanford2001" role="doc-biblioref">2001</a>)</span>. Gene trap and promoter traps were introduced to not only screen for genes with restricted expression patterns, but also to enable functional analysis of the gene from homozygote mutant phenotypes <span class="citation">(Friedrich and Soriano <a href="#ref-Friedrich1991" role="doc-biblioref">1991</a>)</span>. Like the typical enhancer trap vector, gene trap and promoter trap vectors contain a reporter gene, such as lacZ (<span class="math inline">\(\beta\)</span>-gal), to visualize gene expression, and sometimes also a marker to screen for integration, such as the neomycin resistance gene (neo). Though often, lacZ itself, or in a fusion with neo (<span class="math inline">\(\beta\)</span>-geo), was also used as the marker when screening mouse embryonic stem (ES) cells (Figure <a href="prequel.html#fig:trap1">2.2</a>).</p>
<p>Unlike the enhancer trap vector, gene trap and promoter trap vectors do not have a promoter for the reporter, though the marker, if present, can have its own promoter. In a promoter trap, the construct needs to be inserted in frame and in the correct orientation into an exon of a gene to be expressed, making it very inefficient <span class="citation">(Friedrich and Soriano <a href="#ref-Friedrich1991" role="doc-biblioref">1991</a>; Stanford, Cohn, and Cordes <a href="#ref-Stanford2001" role="doc-biblioref">2001</a>)</span>.</p>
<p>In contrast, in gene traps, a splice acceptance site is added to the 5’ end of the reporter, so the construct can be expressed when inserted into an intron at the right orientation; this is over 50 times more efficient than a promoter trap because introns tend to be much longer than exons and the construct does not have to be in frame to an exon <span class="citation">(Friedrich and Soriano <a href="#ref-Friedrich1991" role="doc-biblioref">1991</a>; Stanford, Cohn, and Cordes <a href="#ref-Stanford2001" role="doc-biblioref">2001</a>)</span>. Gene traps and promoter traps are mutagenic as the reporter has a stop codon, thus truncating the endogenous protein.</p>
<p>While enhancer traps are more commonly used in <em>Drosophila</em>, gene traps are more commonly used in mice. In mice, in 1988, the enhancer trap vector was initially introduced by injection into the male pronucleus in the fertilized egg <span class="citation">(Allen et al. <a href="#ref-Allen1988" role="doc-biblioref">1988</a>)</span>. The throughput of the screen is increased by inserting the construct into genomes of ES cells by electroporation or retroviral infection <span class="citation">(Stanford, Cohn, and Cordes <a href="#ref-Stanford2001" role="doc-biblioref">2001</a>)</span>, screening for ES cells expressing lacZ or the marker, injecting these ES cells into blastocysts to generate chimeric mice to characterize gene expression patterns; chimera are especially useful for characterizing dominant and lethal mutations <span class="citation">(Friedrich and Soriano <a href="#ref-Friedrich1991" role="doc-biblioref">1991</a>; Gossler et al. <a href="#ref-Gossler1989" role="doc-biblioref">1989</a>)</span>.</p>
<p>The first gene trap screen in mouse ES cells was reported in 1989 <span class="citation">(Gossler et al. <a href="#ref-Gossler1989" role="doc-biblioref">1989</a>)</span>, recovering 14 lines. Again, variants of the vector emerged and gene trap screens increased in scale. In 1995, nearly 300 mouse gene trap lines were recovered from one study <span class="citation">(Wurst et al. <a href="#ref-Wurst1995a" role="doc-biblioref">1995</a>)</span>. Later, smaller gene trap studies specific to particular types of genes made possible by additional steps to screen ES cell colonies were performed, such as genes encoding membrane and secreted proteins <span class="citation">(Skarnes et al. <a href="#ref-Skarnes1995" role="doc-biblioref">1995</a>)</span>, genes responding to retinoic acid <span class="citation">(Forrester et al. <a href="#ref-Forrester1996" role="doc-biblioref">1996</a>)</span>, and genes expressed in hematopoitic and endothelial lineages <span class="citation">(Stanford et al. <a href="#ref-Stanford1998" role="doc-biblioref">1998</a>)</span>. In 2001, gene trapping was used to examine not only expression pattern of genes in cell bodies of neurons in the mouse brain, but also axon guidance <span class="citation">(Leighton et al. <a href="#ref-Leighton2001" role="doc-biblioref">2001</a>)</span>. By 2001, a number of gene trap consortia have been established as resources of gene trap vectors and transformant mouse ES cell lines, hoping to create at least one line for each gene in the mouse genome <span class="citation">(Stanford, Cohn, and Cordes <a href="#ref-Stanford2001" role="doc-biblioref">2001</a>)</span>.</p>
<p>In the 1980s and 1990s, with increasing throughput of Sanger sequencing and the advent of shotgun sequencing, the amount of sequencing data in GenBank exploded <span class="citation">(Giani et al. <a href="#ref-Giani2020" role="doc-biblioref">2020</a>)</span>. With 5’ or 3’ rapid amplification of cDNA ends (RACE) PCR, the fusion transcript of the reporter and an endogenous gene could be cloned <span class="citation">(Frohman, Dush, and Martin <a href="#ref-Frohman1988" role="doc-biblioref">1988</a>)</span>, sequenced, and potentially aligned to the existing sequences to identify the gene of interest <span class="citation">(Stanford et al. <a href="#ref-Stanford1998" role="doc-biblioref">1998</a>)</span>. However, the golden age of gene trapping was soon to pass, with the rise of ISH atlases in the late 1990s and the advent of reference genomes of <em>Drosophila melanogaster</em> <span class="citation">(Myers <a href="#ref-Myers2000" role="doc-biblioref">2000</a>)</span>, mouse <span class="citation">(Waterston et al. <a href="#ref-Waterston2002" role="doc-biblioref">2002</a>)</span>, and human <span class="citation">(Lander et al. <a href="#ref-Lander2001" role="doc-biblioref">2001</a>; Venter et al. <a href="#ref-Venter2001" role="doc-biblioref">2001</a>)</span> in the early 2000s that would make it easier to design ISH probes from the reference genome to target annotated genes, as is done today. Nevertheless, enhancer and gene traps were not rendered obsolete by these developments. They have been used in plants and zebrafish through the 2000s and 2010s, as resources of gene expression patterns <span class="citation">(Johnson et al. <a href="#ref-Johnson2005" role="doc-biblioref">2005</a>; Nakayama et al. <a href="#ref-Nakayama2005" role="doc-biblioref">2005</a>; Pérez-Martín et al. <a href="#ref-Perez-Martin2017" role="doc-biblioref">2017</a>; Hiwatashi et al. <a href="#ref-Hiwatashi2001" role="doc-biblioref">2001</a>; Kawakami et al. <a href="#ref-Kawakami2010" role="doc-biblioref">2010</a>; Marquart et al. <a href="#ref-Marquart2015" role="doc-biblioref">2015</a>)</span> (Figure <a href="prequel.html#fig:hist1">2.3</a>).</p>

<div class="figure" style="text-align: center"><span id="fig:hist1"></span>
<img src="02-prequel_files/figure-html/hist1-1.png" alt="Number of publications over time in the prequel era, broken down by technique and colored by species. The gray histogram in the background is the histogram for all prequel publications over time. The bin width of this histogram is 365 days. Here WMISH and ISH exclude fluorescent ISH (FISH)." width="768" />
<p class="caption">
Figure 2.3: Number of publications over time in the prequel era, broken down by technique and colored by species. The gray histogram in the background is the histogram for all prequel publications over time. The bin width of this histogram is 365 days. Here WMISH and ISH exclude fluorescent ISH (FISH).
</p>
</div>
</div>
<div id="in-situ-reporter" class="section level2">
<h2><span class="header-section-number">2.2</span> In situ reporter</h2>
<p>In enhancer, gene, or promoter trap screens, the reporter is randomly inserted into the genome, not targeting predetermined genes. In contrast, in what we call <em>in situ</em> reporter screens, the reporter is fused to predefined regulatory sequences of a gene of interest, with the hope that expression pattern of the reporter would recapitulate that of the gene of interest. Chronologically, this is the second type of high throughput method to profile gene expression patterns (Figure <a href="prequel.html#fig:hist1">2.3</a>).</p>
<p>A precursor to this type of method was used in 1991, where random genomic fragments were fused to a lacZ reporter lacking a transcription start signal and injected as plasmids, screening for fragments driving lacZ expression and characterizing the expression patterns in <em>C. elegans</em> <span class="citation">(Hope <a href="#ref-Hope1991" role="doc-biblioref">1991</a>)</span>. To the best of our knowledge, the first time <em>in situ</em> reporter with predefined regulatory sequences was used to screen for gene expression patterns in a multicellular organism, was in 1995, in <em>C. elegans</em> <span class="citation">(Lynch, Briggs, and Hope <a href="#ref-Lynch1995" role="doc-biblioref">1995</a>)</span>. At that time, the <em>C. elegans</em> genome sequencing project was already in progress <span class="citation">(Lynch, Briggs, and Hope <a href="#ref-Lynch1995" role="doc-biblioref">1995</a>; Sulston et al. <a href="#ref-Sulston1992" role="doc-biblioref">1992</a>)</span>, and the genome sequence was declared “essentially complete” in 1998 <span class="citation">(The C. elegans Sequencing Consortium <a href="#ref-Consortium1998" role="doc-biblioref">1998</a>)</span>. Computationally predicted upstream regulatory sequences of 35 putative genes were fused to a promoterless lacZ as a reporter, cloned into plasmid vectors, and microinjected into <em>C. elegans</em> gonads to create transformed lines then stained with X-gal <span class="citation">(Lynch, Briggs, and Hope <a href="#ref-Lynch1995" role="doc-biblioref">1995</a>)</span>.</p>
<p>A reliable <em>in situ</em> reporter was first reported in mice in 1997. It used a recombinant bacterial artificial chromosome (BAC) with part of the full RU49 gene in the BAC replaced by a lacZ construct and showed that the construct is heritable <span class="citation">(Yang, Model, and Heintz <a href="#ref-Yang1997" role="doc-biblioref">1997</a>)</span>. In 2003, a similar strategy, replacing coding sequences of genes in BACs with EGFP reporter gene, was used to create a mouse brain gene expression atlas <a href="http://www.gensat.org/index.html">GENSAT</a> with BAC transgenic mouse lines <span class="citation">(Gong et al. <a href="#ref-Gong2003" role="doc-biblioref">2003</a>)</span>. The GENSAT lines were used again in 2009 to create a gene expression atlas for retina <span class="citation">(Siegert et al. <a href="#ref-Siegert2009" role="doc-biblioref">2009</a>)</span>. Again, GENSAT benefited from the reference genome, which greatly helped with identifying BACs that include sequences flanking a gene that may contain regulatory elements that make the reporter better recapitulate expression pattern of the endogenous gene <span class="citation">(Gong et al. <a href="#ref-Gong2003" role="doc-biblioref">2003</a>)</span>.</p>
<p>Through the 2000s and 2010s, <em>in situ</em> reporters have been used as a targeted alternative to enhancer and gene trap screens informed by the reference genomes. To address limitations of gene traps, such as inability to precisely define the allele and favoring genes expressed in ES cells when screening for transformant colonies, high-throughput mouse knock out resources with knock out alleles computationally designed according to a reference genome and annotations have been established <span class="citation">(Skarnes et al. <a href="#ref-Skarnes2011" role="doc-biblioref">2011</a>; “A Mouse for All Reasons” <a href="#ref-AMouseForAllReasons" role="doc-biblioref">2007</a>)</span>. As these alleles contain a lacZ reporter, these resources have been used to characterize gene expression in over 40 tissues in mutant mice with lacZ staining <span class="citation">(White et al. <a href="#ref-White2013" role="doc-biblioref">2013</a>; West et al. <a href="#ref-West2015" role="doc-biblioref">2015</a>; Tuck et al. <a href="#ref-Tuck2015" role="doc-biblioref">2015</a>)</span>. However, for some tissues, only low resolution whole mount staining was performed. Similarly, in both mouse <span class="citation">(Visel et al. <a href="#ref-Visel2013" role="doc-biblioref">2013</a>)</span> and <em>Drosophila</em> <span class="citation">(Jenett et al. <a href="#ref-Jenett2012" role="doc-biblioref">2012</a>; Kvon et al. <a href="#ref-Kvon2014" role="doc-biblioref">2014</a>)</span>, transgenic lines with genomic fragments containing putative enhancers driving expression of reporter genes were established as alternatives to enhancer traps. The enhancer candidates can be selected from sequence homology and ChIP-seq predictions <span class="citation">(Visel et al. <a href="#ref-Visel2013" role="doc-biblioref">2013</a>)</span>, or from tiles of sequences flanking genes thought to have restricted expression patterns or within introns of such genes <span class="citation">(Jenett et al. <a href="#ref-Jenett2012" role="doc-biblioref">2012</a>)</span>.</p>
<p><em>In situ</em> reporter atlases exceeded the scale of enhancer and gene trap screens. The largest such atlas in <em>C. elegans</em>, WormAtlas, profiled 1886 genes <span class="citation">(Hunt-Newbury et al. <a href="#ref-Hunt-Newbury2007" role="doc-biblioref">2007</a>)</span>; we are unaware of enhancer and gene trap screens in <em>C. elegans</em> because <em>C. elegans</em> genome sequencing was already underway by 1992 <span class="citation">(Sulston et al. <a href="#ref-Sulston1992" role="doc-biblioref">1992</a>)</span>, making <em>in situ</em> reporter screening feasible before it was so in mice and <em>Drosophila</em>. The largest such study in <em>Drosophila</em> profiled 7705 enhancer candidates <span class="citation">(Kvon et al. <a href="#ref-Kvon2014" role="doc-biblioref">2014</a>)</span>, which far exceeded the 3768 enhancer trap lines in 1989 <span class="citation">(Bier et al. <a href="#ref-Bier1989" role="doc-biblioref">1989</a>)</span>. <em>In situ</em> reporters were used in mice to profile up to 536 genes<span class="citation">(Siegert et al. <a href="#ref-Siegert2009" role="doc-biblioref">2009</a>)</span> and 329 enhancer candidates <span class="citation">(Visel et al. <a href="#ref-Visel2013" role="doc-biblioref">2013</a>)</span>, while the large scale gene trap screen in 1995 only reached 279 lines <span class="citation">(Wurst et al. <a href="#ref-Wurst1995a" role="doc-biblioref">1995</a>)</span> and later mouse gene trap screens did not typically exceed 100 lines. However, where comparable, <em>in situ</em> reporter atlases never reached the scale of (WM)ISH atlases, perhaps because of the large number of transgenic lines required. Allen Brain Atlas (<a href="https://portal.brain-map.org">ABA</a>) profiled over 20,000 genes in the mouse brain, and as of April 2021, the Berkeley Drosophila Genome Project (<a href="https://insitu.fruitfly.org/cgi-bin/ex/insitu.pl">BDGP</a>) WMISH atlas already has 8533 genes. However, <em>in situ</em> reporters might still be a good way to profile enhancer usage in space.</p>
</div>
<div id="wmish" class="section level2">
<h2><span class="header-section-number">2.3</span> ISH and WMISH atlases</h2>
<p><em>In situ</em> hybridization was first used in 1969 to visualize ribosomal RNA (rRNA) <span class="citation">(Gall et al. <a href="#ref-Gall1969" role="doc-biblioref">1969</a>)</span> and ribosomal DNA (rDNA) <span class="citation">(John, Birnstiel, and Jones <a href="#ref-John1969" role="doc-biblioref">1969</a>)</span> in <em>Xenopus laevis</em> oocytes with probes labeled with radioisotope <sup>3</sup>H (Figure <a href="prequel.html#fig:tl1">2.1</a>). To the best of our knowledge, the earliest use of ISH to visualize what was thought to be a specific transcript was done in 1973, to visualize globin mRNAs in various cultured erythroid and non-erythoid cell types by hybridization of radiolabeled cDNA to the mRNA <span class="citation">(Harrison et al. <a href="#ref-Harrison1973" role="doc-biblioref">1973</a>)</span>. As radioactive ISH requires long exposure time (several weeks), has low spatial resolution and high background, and requires handling hazardous radioactive material, alternatives emerged in the mid 1970s and early 1980s. Among the alternatives were variants of FISH and labeled probes detected by primary and enzyme or fluorophore labeled secondary antibodies <span class="citation">(Huber, Voith von Voithenberg, and Kaigala <a href="#ref-Huber2018" role="doc-biblioref">2018</a>; Langer-Safer, Levine, and Ward <a href="#ref-Langer-Safer1982" role="doc-biblioref">1982</a>)</span>; the latter, immunological method is commonly used in ISH and WMISH atlases. To the best of our knowledge, the first report of using immunological fluorescent and peroxidase ISH to visualize expression of a specific gene was published in 1982, the same year such technique was published <span class="citation">(Langer-Safer, Levine, and Ward <a href="#ref-Langer-Safer1982" role="doc-biblioref">1982</a>)</span>, visualizing actin transcripts in chicken muscle tissue culture; the authors reported puncta of cytoplasmic fluorescence which might be clumps of mRNAs or artefact, but could possibly be individual transcripts <span class="citation">(Singer and Ward <a href="#ref-Singer1982" role="doc-biblioref">1982</a>)</span>.</p>
<p>Non-radioactive ISH not only has shorter exposure time and higher resolution than radioactive ISH, but also made WMISH possible. WMISH was first reported in Drosophila embryos in 1989 <span class="citation">(Tautz and Pfeifle <a href="#ref-Tautz1989" role="doc-biblioref">1989</a>)</span>, and was adapted to other model organisms such as mice, <em>Xenopus laevis</em>, and <em>Paracentrotus lividus</em> (purple sea urchin) in the early 1990s <span class="citation">(Rosen and Beddington <a href="#ref-Rosen1993" role="doc-biblioref">1993</a>)</span>. Advantages of WMISH compared to section ISH is preservation of 3D structure of the tissue, ease of interpretation in blastoderm stage embryos, and ease of performing ISH on larger number of embryos <span class="citation">(Rosen and Beddington <a href="#ref-Rosen1993" role="doc-biblioref">1993</a>; Tautz and Pfeifle <a href="#ref-Tautz1989" role="doc-biblioref">1989</a>)</span>.</p>
<p>Just like genome sequencing in multi-cellular organisms and <em>in situ</em> reporter screens, WMISH atlases got a head start in <em>C. elegans</em>. The first WMISH screen with higher throughput than typically used on select marker genes was reported in 1994, of 21 genes in <em>C. elegans</em> <span class="citation">(Seydoux and Fire <a href="#ref-Seydoux1994" role="doc-biblioref">1994</a>)</span>. Early (WM)ISH atlases in the late 1990s typically made probes from cDNA clones from poly-A selected RNAs in tissue or developmental stage of interest without pre-selecting genes to stain for <span class="citation">(Tomancak et al. <a href="#ref-Tomancak2002" role="doc-biblioref">2002</a>; Stapleton <a href="#ref-Stapleton2002" role="doc-biblioref">2002</a>; Gawantka et al. <a href="#ref-Gawantka1998" role="doc-biblioref">1998</a>; Bettenhausen and Gossler <a href="#ref-Bettenhausen1995" role="doc-biblioref">1995</a>)</span>. Some early atlases were intended to be improvements to enhancer and gene trapping and <em>in situ</em> reporter screens, as a simpler and more direct alternative <span class="citation">(Bettenhausen and Gossler <a href="#ref-Bettenhausen1995" role="doc-biblioref">1995</a>)</span> or as a way that can better capture endogenous and dynamic spatial distribution of transcripts <span class="citation">(Gawantka et al. <a href="#ref-Gawantka1998" role="doc-biblioref">1998</a>)</span>. Since 1998, (WM)ISH has been automated, enabling staining for thousands of probes <span class="citation">(Gawantka et al. <a href="#ref-Gawantka1998" role="doc-biblioref">1998</a>; Carson, Thaller, and Eichele <a href="#ref-Carson2002" role="doc-biblioref">2002</a>)</span>.</p>
<p>The genes from which the clones come from were often unknown, so early (WM)ISH atlases referred to the entities stained for as “clones” (Figure <a href="prequel.html#fig:items">2.4</a>), though the genes, homology, and putative functions of the genes can be identified by aligning sequences of the cDNA clones to existing sequences in databases <span class="citation">(Bettenhausen and Gossler <a href="#ref-Bettenhausen1995" role="doc-biblioref">1995</a>; Gawantka et al. <a href="#ref-Gawantka1998" role="doc-biblioref">1998</a>; Kopczynski et al. <a href="#ref-Kopczynski1998" role="doc-biblioref">1998</a>)</span>. However, again, the first WMISH screen with probes made from cloning PCR amplified pre-defined genomic sequences was performed in <em>C. elegans</em> in 1995 <span class="citation">(Birchall, Fishpool, and Albertson <a href="#ref-Birchall1995" role="doc-biblioref">1995</a>)</span>. By the turn of the century, the entities stained for were sometimes referred to as “clusters”, especially in the GHOST atlas for <em>Ciona intestinalis</em> <span class="citation">(Satou et al. <a href="#ref-Satou2001" role="doc-biblioref">2001</a>)</span> (Figure <a href="prequel.html#fig:items">2.4</a>); the sequences of the probes were clustered by alignment and these probes might have come from the same gene.</p>

<div class="figure" style="text-align: center"><span id="fig:items"></span>
<img src="02-prequel_files/figure-html/items-1.png" alt="Number of prequel publications over time, broken down by what the entities stained for were called and colored by species. Bin width is 365 days. Vertical line marks the date when the draft mouse reference genome was published (Waterston et al. 2002), as context of transition from “clone” and “line” to “gene”." width="768" />
<p class="caption">
Figure 2.4: Number of prequel publications over time, broken down by what the entities stained for were called and colored by species. Bin width is 365 days. Vertical line marks the date when the draft mouse reference genome was published <span class="citation">(Waterston et al. <a href="#ref-Waterston2002" role="doc-biblioref">2002</a>)</span>, as context of transition from “clone” and “line” to “gene”.
</p>
</div>
<p>The rise of (WM)ISH atlases started before the completion of genome projects in humans and common model organisms, although their later growth was transformed by the reference genome. In the 2000s, with the availability of sequenced cDNA collections covering increasing proportion of predicted genes and the consequent rise of transcriptome-wide microarray <span class="citation">(Stapleton <a href="#ref-Stapleton2002" role="doc-biblioref">2002</a>; Carter <a href="#ref-Carter2003" role="doc-biblioref">2003</a>)</span>, genes to be stained for in (WM)ISH atlases could be pre-screened based on microarray data of the tissue of interest, with probes made from cDNA clones readily available from such collections <span class="citation">(Yoshikawa et al. <a href="#ref-Yoshikawa2006" role="doc-biblioref">2006</a>; Lein <a href="#ref-Lein2004" role="doc-biblioref">2004</a>)</span>. In addition, probes could be computationally designed based on reference genome sequences <span class="citation">(Lein et al. <a href="#ref-Lein2007" role="doc-biblioref">2007</a>)</span>. Perhaps because of these developments, since the turn of the century, entities stained for have been predominantly referred to as “genes” (Figure <a href="prequel.html#fig:items">2.4</a>). Notably, while radioactive ISH has been mostly replaced by non-radioactive ISH by the 2000s, there is a mouse hippocampus ISH atlas published in 2004 that used radioactive ISH to profile all of its 104 genes <span class="citation">(Lein <a href="#ref-Lein2004" role="doc-biblioref">2004</a>)</span>.</p>
<p>Also with the rise of cDNA microarray in the late 1990s and early 2000s, some (WM)ISH atlases were made as an improvement to microarray with bulk tissue to profile the transcriptome, not only at cellular resolution, but also preserving spatial and sometimes temporal context <span class="citation">(Lein et al. <a href="#ref-Lein2007" role="doc-biblioref">2007</a>; Bell, Yatskievych, and Antin <a href="#ref-Bell2004" role="doc-biblioref">2004</a>)</span>, analogous to how scRNA-seq and various later forms of spatial transcriptomics were developed in response to bulk RNA-seq.</p>
<p>Since the 2000s, (WM)ISH atlases have been made for specific types of genes and a number of mouse tissues. In 2004, locked nucleic acid (LNA) modified oligonucleotide probes were introduced, greatly improving sensitivity of miRNA northern blot <span class="citation">(Valoczi <a href="#ref-Valoczi2004" role="doc-biblioref">2004</a>)</span> and made (WM)ISH atlases for miRNAs possible. The first miRNA WMISH atlas was published in 2005, which profiled 115 miRNAs in zebrafish embryos <span class="citation">(Wienholds <a href="#ref-Wienholds2005" role="doc-biblioref">2005</a>)</span>. Since then, miRNA atlases were created for mice <span class="citation">(Karali et al. <a href="#ref-Karali2010" role="doc-biblioref">2010</a>; Diez-Roux et al. <a href="#ref-Diez-Roux2011" role="doc-biblioref">2011</a>; Kloosterman et al. <a href="#ref-Kloosterman2006" role="doc-biblioref">2006</a>)</span>, Drosophila <span class="citation">(Aboobaker et al. <a href="#ref-Aboobaker2005" role="doc-biblioref">2005</a>)</span>, chicken <span class="citation">(Darnell et al. <a href="#ref-Darnell2006" role="doc-biblioref">2006</a>)</span>, and Xenopus laevis <span class="citation">(Ahmed et al. <a href="#ref-Ahmed2015" role="doc-biblioref">2015</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:species"></span>
<img src="02-prequel_files/figure-html/species-1.png" alt="Number of (WM)ISH publications per species." width="576" />
<p class="caption">
Figure 2.5: Number of (WM)ISH publications per species.
</p>
</div>
<p>While (WM)ISH atlases are available for several species, the mouse is by far the favored model organism (Figure <a href="prequel.html#fig:species">2.5</a>). A timeline of the first (WM)ISH atlas for each of the species and some notable atlases are shown in Figure <a href="prequel.html#fig:tl2">2.6</a>. Especially for mice, atlases for other specific types of genes were published in the late 2000s and the 2010s, such as genes coding for RNA binding proteins <span class="citation">(McKee et al. <a href="#ref-McKee2005" role="doc-biblioref">2005</a>)</span>, fibroblast growth factors and their receptors <span class="citation">(Yaylaoglu et al. <a href="#ref-Yaylaoglu2005" role="doc-biblioref">2005</a>)</span>, proteins with catalytic activities <span class="citation">(Cankaya et al. <a href="#ref-Cankaya2007" role="doc-biblioref">2007</a>)</span>, transcription factors and cofactors <span class="citation">(Yokoyama et al. <a href="#ref-Yokoyama2009" role="doc-biblioref">2009</a>)</span>, metabolic enzymes and soluble carriers <span class="citation">(Geffers et al. <a href="#ref-Geffers2013" role="doc-biblioref">2012</a>)</span>, cholesterol biosynthetic enzymes <span class="citation">(Şişecioğlu et al. <a href="#ref-Sisecioglu2015" role="doc-biblioref">2015</a>)</span>, and ion channels (in rats) <span class="citation">(Shcherbatyy et al. <a href="#ref-Shcherbatyy2014" role="doc-biblioref">2015</a>)</span>. Among the mouse atlases, while the brain gets disproportionately strong interests, with the influential <a href="https://portal.brain-map.org">ABA</a> <span class="citation">(Lein et al. <a href="#ref-Lein2007" role="doc-biblioref">2007</a>)</span> and <a href="https://gp3.mpg.de">GenePaint</a> <span class="citation">(Carson, Thaller, and Eichele <a href="#ref-Carson2002" role="doc-biblioref">2002</a>)</span>, ISH atlases exist for the eye <span class="citation">(Thut et al. <a href="#ref-Thut2001" role="doc-biblioref">2001</a>; Blackshaw et al. <a href="#ref-Blackshaw2004" role="doc-biblioref">2004</a>)</span>, genitourinary tract (<a href="https://www.gudmap.org">GenitoUrinary Development Molecular Anatomy Project (GUDMAP)</a>) <span class="citation">(Harding et al. <a href="#ref-Harding2011" role="doc-biblioref">2011</a>)</span>, and lung (<a href="https://lungmap.net">LungMAP</a>) <span class="citation">(Ardini-Poleske et al. <a href="#ref-Ardini-Poleske2017" role="doc-biblioref">2017</a>)</span> (Figure <a href="prequel.html#fig:tl2">2.6</a>, Figure <a href="prequel.html#fig:anat1">2.7</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:tl2"></span>
<img src="02-prequel_files/figure-html/tl2-1.png" alt="Timeline of the first (WM)ISH databases for each species for which such databases are available, as well as some notable databases." width="960" />
<p class="caption">
Figure 2.6: Timeline of the first (WM)ISH databases for each species for which such databases are available, as well as some notable databases.
</p>
</div>
<p>While the vast majority of (WM)ISH atlases used bright field imaging, a few used FISH (Figure <a href="prequel.html#fig:hist1">2.3</a>), for advantages conferred by FISH discussed below. A notable FISH atlas is the Berkeley <em>Drosophila</em> Transcription Network Project (<a href="http://www.cb.uu.se/~cris/BDTNP_Imaging.html">BDTNP</a>) from 2006 to 2008, which profiled expression patterns of 95 genes in the <em>Drosophila</em> embryo across 6 developmental stages up to the beginning of gastrulation <span class="citation">(Fowlkes et al. <a href="#ref-Fowlkes2008" role="doc-biblioref">2008</a>; Luengo Hendriks et al. <a href="#ref-LuengoHendriks2006a" role="doc-biblioref">2006</a>)</span>. Two genes are imaged in each embryo, and the images of 1822 embryos were registered across both space and time to construct 3D virtual embryos on which patterns of different genes can be quantitatively compared <span class="citation">(Fowlkes et al. <a href="#ref-Fowlkes2008" role="doc-biblioref">2008</a>)</span>; the 3D imaging and penetration into the opaque yolk is made possible by two photon microscopy, in which only the fluorophores in the region of focus are excited <span class="citation">(Luengo Hendriks et al. <a href="#ref-LuengoHendriks2006a" role="doc-biblioref">2006</a>)</span>. Another notable FISH atlas is <a href="http://fly-fish.ccbr.utoronto.ca">Fly-FISH</a> from 2007, which profiled subcellular localization of transcripts of 3370 genes in <em>Drosophila</em> embryos <span class="citation">(Lécuyer et al. <a href="#ref-Lecuyer2007" role="doc-biblioref">2007</a>)</span>. While subcellular localization of transcripts can sometimes be discerned in bright field WMISH <span class="citation">(Tomancak et al. <a href="#ref-Tomancak2002" role="doc-biblioref">2002</a>)</span>, Fly-FISH shows higher subcellular resolution thanks to a FISH protocol using tyramide signal amplification. To our best knowledge, this is the first transcriptomic atlas of a multi-cellular organism to profile subcellular transcript localization. While more recent smFISH based methods record subcellular information, such information is typically not used in downstream analyses.</p>
<div class="figure" style="text-align: center"><span id="fig:anat1"></span>
<img src="02-prequel_files/figure-html/anat1-1.png" alt="A) Number of mouse publications per organ for (WM)ISH atlases (including FISH). B) Maximum number of genes in atlases for each organ, as of publication of the paper about the atlases. The color is in log scale to improve dynamic range." width="576" />
<p class="caption">
Figure 2.7: A) Number of mouse publications per organ for (WM)ISH atlases (including FISH). B) Maximum number of genes in atlases for each organ, as of publication of the paper about the atlases. The color is in log scale to improve dynamic range.
</p>
</div>
<p>WMISH was the most commonly used technique in the prequel era, followed by ISH (Figure <a href="prequel.html#fig:tech1">2.8</a>). In summary, advances of non-radioactive ISH and WMISH from radioactive ISH, limitations of enhancer and gene trap and <em>in situ</em> reporter screens, cDNA collections that cover most of predicted genes, limitations of bulk microarray, reference genomes that allow for computational probe design, and ISH robots may have been responsible for the rise of (WM)ISH atlases. Another important factor may be the rise of digital photography and the internet in the 1990s, as developing thousands of analogue photos is an arduous task. Moreover, online digital atlases have been much more accessible to the wider community. Assuming that the number of publications in a field reflects interest in that field during a period of time, and if our collection is representative of the actual body of literature, then the golden age of the prequel era was the 2000s and WMISH was responsible for that peak, while section ISH and “collection”, i.e. databases of gene expression patterns curated from publications and some (WM)ISH atlases, account for much of the interest after 2010 (Figure <a href="prequel.html#fig:hist1">2.3</a>). The websites of many of the older (WM)ISH atlases are no longer accessible. However, some of the atlases from that period of time still live on in extant curated databases, which will be discussed in the next section.</p>
<div class="figure" style="text-align: center"><span id="fig:tech1"></span>
<img src="02-prequel_files/figure-html/tech1-1.png" alt="Number of prequel publications per technique." width="576" />
<p class="caption">
Figure 2.8: Number of prequel publications per technique.
</p>
</div>
<p>The golden age declined before the rise of current era spatial transcriptomics, which started around 2014 <a href="current.html#fig:current-vs-prequel">4.2</a>. What contributed to the decline of the golden age? Perhaps with proliferation of such atlases, curated databases exceeding 10,000 genes, and especially with over 20,000 genes in ABA mapped to a high quality 3D mouse brain model, there are already enough gene expression pattern resources for the most commonly studied genes, tissues (especially the brain), and developmental stages in the most common model organisms, thus making new atlases in those systems unnecessary. Moreover, in the last decade, the under-utilization of gene expression atlases <span class="citation">(Boer et al. <a href="#ref-DeBoer2009" role="doc-biblioref">2009</a>)</span> may have reduced motivation to build new atlases. Or perhaps, more importantly, inherent limitations of non-multiplexed (WM)ISH contributed to the decline in interest in such methods. In these atlases, typically only one gene is stained for in each individual embryo or tissue section. Gene expression patterns of different genes can only be meaningfully compared and classified in tissues with a stereotypical structure, such as wild type embryos and the brain, but not tumors and pathological tissues, even though there is intense interest in spatial transcriptomics in tumors as evidenced by the LCM and ST literature <a href="text-mining.html#fig:topics">6.3</a>. A large number of embryos or sections are required for such atlases, thus increasing cost and making human atlases extremely difficult and costly, if ethical at all. Furthermore, since the chromogenic reaction in bright field ISH can be prolonged to increase staining intensity, the patterns are not quantitative and consequently, analyses of such patterns typically involve binarization and quantitative expression levels of genes cannot be compared. Even with a stereotypical structure, image registration can be challenging because of biological differences between individuals <span class="citation">(Fowlkes et al. <a href="#ref-Fowlkes2008" role="doc-biblioref">2008</a>)</span>.</p>
</div>
<div id="database_prequel" class="section level2">
<h2><span class="header-section-number">2.4</span> Databases of the prequel era</h2>
<p>Many of the (WM)ISH atlases discussed above, such as BDGP <span class="citation">(Tomancak et al. <a href="#ref-Tomancak2002" role="doc-biblioref">2002</a>)</span>, <a href="http://geisha.arizona.edu/geisha/">Gallus <em>In Situ</em> Hybridization Atlas (GEISHA)</a> <span class="citation">(Bell, Yatskievych, and Antin <a href="#ref-Bell2004" role="doc-biblioref">2004</a>)</span>, ABA <span class="citation">(Lein et al. <a href="#ref-Lein2007" role="doc-biblioref">2007</a>)</span>, BDTNP <span class="citation">(Fowlkes et al. <a href="#ref-Fowlkes2008" role="doc-biblioref">2008</a>)</span>, GUDMAP <span class="citation">(Harding et al. <a href="#ref-Harding2011" role="doc-biblioref">2011</a>)</span>, and LungMAP <span class="citation">(Ardini-Poleske et al. <a href="#ref-Ardini-Poleske2017" role="doc-biblioref">2017</a>)</span> are stored in databases that can be queried online, typically by gene symbol or by controlled anatomical or developmental vocabulary (i.e. ontology, reviewed in depth in <span class="citation">(Clarkson <a href="#ref-Clarkson2016" role="doc-biblioref">2016</a>)</span>). There are additional gene expression databases for images curated from publications, some containing non-spatial data as well and some specifically for spatial data.</p>
<p>The rise of the curated databases started in the 1990s. Already in 1992, the challenges of managing the increasing amount of gene expression data in developmental biology emerged and a spatiotemporal database of mouse gene expression that would later become the Edinburgh Mouse Atlas of Gene Expression (<a href="http://www.emouseatlas.org/emage/home.php">EMAGE</a>) was discussed <span class="citation">(Baldock et al. <a href="#ref-Baldock1992" role="doc-biblioref">1992</a>)</span>. In 1994, Jackson Laboratory proposed the Gene Expression Database (<a href="http://www.informatics.jax.org/menus/expression_menu.shtml">GXD</a>) <span class="citation">(Ringwald et al. <a href="#ref-Ringwald1994" role="doc-biblioref">1994</a>)</span>, in collaboration with EMAGE to build the most comprehensive mouse gene expression database. In 1997, work was already in progress to produce (WM)ISH atlases and construct the database infrastructure for mouse <span class="citation">(Ringwald et al. <a href="#ref-Ringwald1997" role="doc-biblioref">1997</a>)</span> (GXD and EMAGE), Drosophila melanogaster <span class="citation">(Janning <a href="#ref-Janning1997" role="doc-biblioref">1997</a>)</span>, <em>C. elegans</em> <span class="citation">(Martinelli, Brown, and Durbin <a href="#ref-Martinelli1997" role="doc-biblioref">1997</a>)</span>, and zebrafish <span class="citation">(Westerfield et al. <a href="#ref-Westerfield1997" role="doc-biblioref">1997</a>)</span>. Curated databases of mice (GXD and EMAGE), zebrafish (<a href="http://zfin.org">Zebrafish Information Network (ZFIN)</a> <span class="citation">(Howe et al. <a href="#ref-Howe2017" role="doc-biblioref">2017</a>)</span>), and <em>Xenopus laevis</em> (<a href="http://www.xenbase.org/entry/">Xenbase</a> <span class="citation">(Bowes et al. <a href="#ref-Bowes2009" role="doc-biblioref">2009</a>)</span>) were released in the 2000s, within a tide of (WM)ISH atlases for new species (Figure <a href="prequel.html#fig:tl2">2.6</a>). Some of these databases are regularly updated and the updates are responsible for many of the “collection” publications after 2010 (Figure <a href="prequel.html#fig:hist1">2.3</a>, Figure <a href="prequel.html#fig:tech1">2.8</a>); our historical literature collection has not only the original publications for the databases, but also publications for later updates that involve new spatial gene expression images. Examples of other extant curated databases: for <em>Drosophila melanogaster</em> <a href="http://www.flyexpress.net">FlyExpress</a> <span class="citation">(Kumar et al. <a href="#ref-Kumar2017" role="doc-biblioref">2017</a>)</span>, for <em>Xenopus laevis</em> <a href="http://genomics.crick.ac.uk/cgi-bin/search.exe">XenMARK</a> <span class="citation">(Gilchrist et al. <a href="#ref-Gilchrist2009" role="doc-biblioref">2009</a>)</span>, and for ascidians <a href="https://www.aniseed.cnrs.fr">Ascidian Network for <em>In Situ</em> Expression and Embryological Data (ANISEED)</a> <span class="citation">(Tassy et al. <a href="#ref-Tassy2010" role="doc-biblioref">2010</a>)</span>. Databases, curated or not, are available for several species; mice, <em>Drosophila</em>, and zebrafish have the most extant databases (Figure <a href="prequel.html#fig:dbs">2.9</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:dbs"></span>
<img src="02-prequel_files/figure-html/dbs-1.png" alt="Number of extant spatial gene expression databases per species." width="576" />
<p class="caption">
Figure 2.9: Number of extant spatial gene expression databases per species.
</p>
</div>
<p>Data can be exchanged between databases. For example, among mouse databases GenePaint <span class="citation">(Carson, Thaller, and Eichele <a href="#ref-Carson2002" role="doc-biblioref">2002</a>)</span> and EMAGE now contain data from <a href="http://www.eurexpress.org/ee/">Eurexpress</a> <span class="citation">(Diez-Roux et al. <a href="#ref-Diez-Roux2011" role="doc-biblioref">2011</a>; Boer et al. <a href="#ref-DeBoer2009" role="doc-biblioref">2009</a>)</span>, and EMAGE uses data from GXD for the 3D gene expression models <span class="citation">(Ringwald et al. <a href="#ref-Ringwald1999" role="doc-biblioref">1999</a>)</span>. ANISEED contains data from WMISH atlases <a href="http://ghost.zool.kyoto-u.ac.jp">GHOST</a> for Ciona intestinalis <span class="citation">(Satou et al. <a href="#ref-Satou2001" role="doc-biblioref">2001</a>)</span> and MAboya Gene Expression patterns and Sequence Tags (MAGEST) for <em>Halocynthia roretzi</em> <span class="citation">(Kawashima <a href="#ref-Kawashima2000" role="doc-biblioref">2000</a>)</span>. FlyExpress contains data from <em>Drosophila</em> atlases such as BDGP and Fly-FISH. Data in databases that ceased to operate may still be available in extant databases. For instance, AXelDb WMISH atlas and database for <em>Xenopus laevis</em> <span class="citation">(Gawantka et al. <a href="#ref-Gawantka1998" role="doc-biblioref">1998</a>)</span> has been subsumed in Xenbase while AXelDb’s own website has long been defunct. Likewise, as of April 2021, the MAGEST website is defunct but the data lives on in ANISEED.</p>
<p>Some of the databases go beyond collecting data from other databases. Databases such as EMAGE, ANISEED, and ABA registered multiple 2D section images to map gene expression patterns onto 3D anatomical models for better comparison between different genes. FlyExpress also standardized the images from the atlases and enables search for coexpressed genes by expression pattern <span class="citation">(Kumar et al. <a href="#ref-Kumar2017" role="doc-biblioref">2017</a>)</span>. There have also been efforts to integrate databases from multiple model organisms. In 2007, COMPARE <span class="citation">(Salgado et al. <a href="#ref-Salgado2008" role="doc-biblioref">2008</a>)</span> and 4DXpress <span class="citation">(Haudry et al. <a href="#ref-Haudry2007" role="doc-biblioref">2007</a>)</span> were developed to make gene expression patterns and developmental stages in zebrafish, mouse, and <em>Drosophila</em> (also medaka in 4DXpress) comparable. While COMPARE and 4DXpress are no longer available, interest in integrating the databases continues, so in 2016, the Alliance of Genome Resource was founded, producing a unified user interface to genome and gene expression databases for <em>Saccharomyces cerevisiae</em>, <em>C. elegans</em>, <em>Drosophila melanogaster</em>, mouse, rat, and zebrafish <span class="citation">(Agapite et al. <a href="#ref-Agapite2020" role="doc-biblioref">2020</a>)</span>, although spatial patterns are not its focus.</p>
</div>
<div id="geo_prequel" class="section level2">
<h2><span class="header-section-number">2.5</span> Geography of the prequel era</h2>
<p>Where were prequel era research conducted? Our database includes affiliation of the first author as of publication for all papers, and the affiliations have been geocoded to plot on maps. Around the world, most of prequel studies were performed in coastal US and Western Europe, but a some studies were performed in Asia and Oceania, but especially Japan (Figure <a href="prequel.html#fig:prequel-world">2.10</a>). Not all of the top contributing institutions are readily recognizable “elite” institutions. Institutions include BDGP from UC Berkeley, ZFIN from University of Oregon (UO), ABA from Allen Brain Institute (Allen), GEISHA from University of Arizona (UofA), GXD from Jackson Laboratory (JAX), EMAGE from Western General Hospital (WGH), <a href="https://www.embl-heidelberg.de/mepd/">MEPD</a> (for <em>Oryzias latipes</em>) from European Molecular Biology Laboratory (EMBL), and GHOST from Kyoto University (Kyodai), and mouse gene trap lines from Mount Sinai.</p>
<div class="figure" style="text-align: center"><span id="fig:prequel-world"></span>
<img src="02-prequel_files/figure-html/prequel-world-1.png" alt="Number of prequel publications per city around the world, with top contributing institutions labeled." width="960" />
<p class="caption">
Figure 2.10: Number of prequel publications per city around the world, with top contributing institutions labeled.
</p>
</div>
<p>This can be better visualized by breaking the map down by species. Here we see locations of some model organism consortia, and that GHOST is a result of collaboration of multiple Japanese institutions (Figure <a href="prequel.html#fig:map-species">2.11</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:map-species"></span>
<img src="02-prequel_files/figure-html/map-species-1.png" alt="Number of prequel publications per city broken down by species. Gray points are the overall number as a reference of contributions from each city and region." width="960" />
<p class="caption">
Figure 2.11: Number of prequel publications per city broken down by species. Gray points are the overall number as a reference of contributions from each city and region.
</p>
</div>
<p>That some institutions have disproportional contribution of one technique can also be shown. Here it’s clear that prequel techniques are used by many different institutions (Figure <a href="prequel.html#fig:map-tech">2.12</a>). In contrast, as will be shown in Chapter <a href="current.html#current">4</a>, most current era techniques never spread beyond their institutions of origin. The LCM study comes from Allen Brain Institute’s atlases for Allen’s mouse sleep deprivation atlas <span class="citation">(Thompson et al. <a href="#ref-Thompson2010" role="doc-biblioref">2010</a>)</span> and human glioblastoma atlas <span class="citation">(Puchalski et al. <a href="#ref-Puchalski2018" role="doc-biblioref">2018</a>)</span>; although LCM is a current era technique, those two studies are in the prequel sheet because they also have ISH atlases.</p>
<div class="figure" style="text-align: center"><span id="fig:map-tech"></span>
<img src="02-prequel_files/figure-html/map-tech-1.png" alt="Number of prequel publications per city broken down by technique. Gray points are the overall number as a reference of contributions from each city and region." width="960" />
<p class="caption">
Figure 2.12: Number of prequel publications per city broken down by technique. Gray points are the overall number as a reference of contributions from each city and region.
</p>
</div>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Aboobaker2005">
<p>Aboobaker, A. Aziz, Pavel Tomancak, Nipam Patel, Gerald M. Rubin, and Eric C. Lai. 2005. “Drosophila microRNAs exhibit diverse spatial expression patterns during embryonic development.” <em>Proceedings of the National Academy of Sciences</em> 102 (50): 18017–22. <a href="https://doi.org/10.1073/pnas.0508823102">https://doi.org/10.1073/pnas.0508823102</a>.</p>
</div>
<div id="ref-Achim2015">
<p>Achim, Kaia, Jean Baptiste Pettit, Luis R. Saraiva, Daria Gavriouchkina, Tomas Larsson, Detlev Arendt, and John C. Marioni. 2015. “High-throughput spatial mapping of single-cell RNA-seq data to tissue of origin.” <em>Nature Biotechnology</em> 33 (5): 503–9. <a href="https://doi.org/10.1038/nbt.3209">https://doi.org/10.1038/nbt.3209</a>.</p>
</div>
<div id="ref-Agapite2020">
<p>Agapite, Julie, Laurent-Philippe Albou, Suzi Aleksander, Joanna Argasinska, Valerio Arnaboldi, Helen Attrill, Susan M. Bello, et al. 2020. “Alliance of Genome Resources Portal: unified model organism research platform.” <em>Nucleic Acids Research</em> 48 (D1): D650–D658. <a href="https://doi.org/10.1093/nar/gkz813">https://doi.org/10.1093/nar/gkz813</a>.</p>
</div>
<div id="ref-Ahmed2015">
<p>Ahmed, Ayisha, Nicole J Ward, Simon Moxon, Sara Lopez-Gomollon, Camille Viaut, Matthew L Tomlinson, Ilya Patrushev, et al. 2015. “A Database of microRNA Expression Patterns in Xenopus laevis.” <em>PLOS ONE</em> 10 (10): e0138313. <a href="https://doi.org/10.1371/journal.pone.0138313">https://doi.org/10.1371/journal.pone.0138313</a>.</p>
</div>
<div id="ref-Allen1988">
<p>Allen, Nicholas D., David G. Cran, Sheila C. Barton, Simon Hettle, Wolf Reik, and M. Azim Surani. 1988. “Transgenes as probes for active chromosomal domains in mouse development.” <em>Nature</em> 333 (6176): 852–55. <a href="https://doi.org/10.1038/333852a0">https://doi.org/10.1038/333852a0</a>.</p>
</div>
<div id="ref-AMouseForAllReasons">
<p>“A Mouse for All Reasons.” 2007. <em>Cell</em> 128 (1): 9–13. <a href="https://doi.org/10.1016/j.cell.2006.12.018">https://doi.org/10.1016/j.cell.2006.12.018</a>.</p>
</div>
<div id="ref-Ardini-Poleske2017">
<p>Ardini-Poleske, Maryanne E., Robert F. Clark, Charles Ansong, James P. Carson, Richard A. Corley, Gail H. Deutsch, James S. Hagood, et al. 2017. “LungMAP: The Molecular Atlas of Lung Development Program.” <em>American Journal of Physiology-Lung Cellular and Molecular Physiology</em> 313 (5): L733–L740. <a href="https://doi.org/10.1152/ajplung.00139.2017">https://doi.org/10.1152/ajplung.00139.2017</a>.</p>
</div>
<div id="ref-Baldock1992">
<p>Baldock, Richard, Jonathan Bard, Matt Kaufman, and Duncan Davidson. 1992. “What’s New? A real mouse for your computer.” <em>BioEssays</em> 14 (7): 501–2. <a href="https://doi.org/10.1002/bies.950140713">https://doi.org/10.1002/bies.950140713</a>.</p>
</div>
<div id="ref-Bell2004">
<p>Bell, George W., Tatiana A. Yatskievych, and Parker B. Antin. 2004. “GEISHA, a whole-mount in situ hybridization gene expression screen in chicken embryos.” <em>Developmental Dynamics</em> 229 (3): 677–87. <a href="https://doi.org/10.1002/dvdy.10503">https://doi.org/10.1002/dvdy.10503</a>.</p>
</div>
<div id="ref-Bellen1989">
<p>Bellen, H. J., C. J. O’Kane, C. Wilson, U. Grossniklaus, R. K. Pearson, and W. J. Gehring. 1989. “P-element-mediated enhancer detection: a versatile method to study development in Drosophila.” <em>Genes &amp; Development</em> 3 (9): 1288–1300. <a href="https://doi.org/10.1101/gad.3.9.1288">https://doi.org/10.1101/gad.3.9.1288</a>.</p>
</div>
<div id="ref-Belmamoune2008">
<p>Belmamoune, Mounia, and Fons J. Verbeek. 2008. “Data Integration for Spatio-Temporal Patterns of Gene Expression of Zebrafish development: the GEMS database.” <em>Journal of Integrative Bioinformatics</em> 5 (2): 35–48. <a href="https://doi.org/10.1515/jib-2008-92">https://doi.org/10.1515/jib-2008-92</a>.</p>
</div>
<div id="ref-Bettenhausen1995">
<p>Bettenhausen, Berthold, and Achim Gossler. 1995. “Efficient Isolation of Novel Mouse Genes Differentially Expressed in Early Postimplantation Embryos.” <em>Genomics</em> 28 (3): 436–41. <a href="https://doi.org/10.1006/geno.1995.1172">https://doi.org/10.1006/geno.1995.1172</a>.</p>
</div>
<div id="ref-Bier1989">
<p>Bier, E., H. Vaessin, S. Shepherd, K. Lee, K. McCall, S. Barbel, L. Ackerman, R. Carretto, T. Uemura, and E. Grell. 1989. “Searching for pattern and mutation in the Drosophila genome with a P-lacZ vector.” <em>Genes &amp; Development</em> 3 (9): 1273–87. <a href="https://doi.org/10.1101/gad.3.9.1273">https://doi.org/10.1101/gad.3.9.1273</a>.</p>
</div>
<div id="ref-Birchall1995">
<p>Birchall, Philip S, Rita M Fishpool, and Donna G Albertson. 1995. “Expression patterns of predicted genes from the C. elegans genome sequence visualized by FISH in whole organisms,” 314–20. <a href="https://doi.org/10.1038/ng1195-314">https://doi.org/10.1038/ng1195-314</a>.</p>
</div>
<div id="ref-Blackshaw2004">
<p>Blackshaw, Seth, Sanjiv Harpavat, Jeff Trimarchi, Li Cai, Haiyan Huang, Winston P Kuo, Griffin Weber, et al. 2004. “Genomic Analysis of Mouse Retinal Development.” Edited by William Harris. <em>PLoS Biology</em> 2 (9): e247. <a href="https://doi.org/10.1371/journal.pbio.0020247">https://doi.org/10.1371/journal.pbio.0020247</a>.</p>
</div>
<div id="ref-DeBoer2009">
<p>Boer, B. A. de, Jan M. Ruijter, F. P. J. M. Voorbraak, and A. F. M. Moorman. 2009. “More than a decade of developmental gene expression atlases: where are we now?” <em>Nucleic Acids Research</em> 37 (22): 7349–59. <a href="https://doi.org/10.1093/nar/gkp819">https://doi.org/10.1093/nar/gkp819</a>.</p>
</div>
<div id="ref-Bowes2009">
<p>Bowes, Jeff B., Kevin A. Snyder, Erik Segerdell, Chris J. Jarabek, Kenan Azam, Aaron M. Zorn, and Peter D. Vize. 2009. “Xenbase: Gene expression and improved integration.” <em>Nucleic Acids Research</em> 38 (SUPPL.1): D607–D612. <a href="https://doi.org/10.1093/nar/gkp953">https://doi.org/10.1093/nar/gkp953</a>.</p>
</div>
<div id="ref-BravoGonzalez-Blas2020">
<p>Bravo González-Blas, Carmen, Xiao-Jiang Quan, Ramon Duran-Romaña, Ibrahim Ihsan Taskiran, Duygu Koldere, Kristofer Davie, Valerie Christiaens, et al. 2020. “Identification of genomic enhancers through spatial integration of single-cell transcriptomics and epigenomics.” <em>Molecular Systems Biology</em> 16 (5): e9438. <a href="https://doi.org/10.15252/msb.20209438">https://doi.org/10.15252/msb.20209438</a>.</p>
</div>
<div id="ref-Cankaya2007">
<p>Cankaya, Murat, Ana Hernandez, Mehmet Ciftci, Sukru Beydemir, Hasan Ozdemir, Harun Budak, Ilhami Gulcin, et al. 2007. “An analysis of expression patterns of genes encoding proteins with catalytic activities.” <em>BMC Genomics</em> 8 (1): 232. <a href="https://doi.org/10.1186/1471-2164-8-232">https://doi.org/10.1186/1471-2164-8-232</a>.</p>
</div>
<div id="ref-Carson2002">
<p>Carson, James P., Christina Thaller, and Gregor Eichele. 2002. “A transcriptome atlas of the mouse brain at cellular resolution.” <em>Current Opinion in Neurobiology</em> 12 (5): 562–65. <a href="https://doi.org/10.1016/S0959-4388(02)00356-2">https://doi.org/10.1016/S0959-4388(02)00356-2</a>.</p>
</div>
<div id="ref-Carter2003">
<p>Carter, Mark G. 2003. “In Situ-Synthesized Novel Microarray Optimized for Mouse Stem Cell and Early Developmental Expression Profiling.” <em>Genome Research</em> 13 (5): 1011–21. <a href="https://doi.org/10.1101/gr.878903">https://doi.org/10.1101/gr.878903</a>.</p>
</div>
<div id="ref-Chen2020">
<p>Chen, Wei-Ting, Ashley Lu, Katleen Craessaerts, Benjamin Pavie, Carlo Sala Frigerio, Nikky Corthout, Xiaoyan Qian, et al. 2020. “Spatial Transcriptomics and In Situ Sequencing to Study Alzheimer’s Disease.” <em>Cell</em> 0 (0). <a href="https://doi.org/10.1016/j.cell.2020.06.038">https://doi.org/10.1016/j.cell.2020.06.038</a>.</p>
</div>
<div id="ref-Clarkson2016">
<p>Clarkson, Melissa D. 2016. “Representation of anatomy in online atlases and databases: a survey and collection of patterns for interface design.” <em>BMC Developmental Biology</em> 16 (1): 18. <a href="https://doi.org/10.1186/s12861-016-0116-y">https://doi.org/10.1186/s12861-016-0116-y</a>.</p>
</div>
<div id="ref-Crosetto2015">
<p>Crosetto, Nicola, Magda Bienko, and Alexander Van Oudenaarden. 2015. “Spatially resolved transcriptomics and beyond.” <a href="https://doi.org/10.1038/nrg3832">https://doi.org/10.1038/nrg3832</a>.</p>
</div>
<div id="ref-Darnell2006">
<p>Darnell, Diana K., Simran Kaur, Stacey Stanislaw, Jay K. Konieczka, Tatiana A. Yatskievych, and Parker B. Antin. 2006. “MicroRNA expression during chick embryo development.” <em>Developmental Dynamics</em> 235 (11): 3156–65. <a href="https://doi.org/10.1002/dvdy.20956">https://doi.org/10.1002/dvdy.20956</a>.</p>
</div>
<div id="ref-Diez-Roux2011">
<p>Diez-Roux, Graciana, Sandro Banfi, Marc Sultan, Lars Geffers, Santosh Anand, David Rozado, Alon Magen, et al. 2011. “A High-Resolution Anatomical Atlas of the Transcriptome in the Mouse Embryo.” Edited by Gregory S. Barsh. <em>PLoS Biology</em> 9 (1): e1000582. <a href="https://doi.org/10.1371/journal.pbio.1000582">https://doi.org/10.1371/journal.pbio.1000582</a>.</p>
</div>
<div id="ref-Forrester1996">
<p>Forrester, Lesley M., Andras Nagy, Mehran Sam, Alistair Watt, Lois Stevenson, Alan Bernstein, Alexandra L. Joyner, and Wolfgang Wurst. 1996. “An induction gene trap screen in embryonic stem cells: Identification of genes that respond to retinoic acid in vitro.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 93 (4): 1677–82. <a href="https://doi.org/10.1073/pnas.93.4.1677">https://doi.org/10.1073/pnas.93.4.1677</a>.</p>
</div>
<div id="ref-Fowlkes2008">
<p>Fowlkes, Charless C., Cris L. Luengo Hendriks, Soile V. E. Keränen, Gunther H. Weber, Oliver Rübel, Min-Yu Huang, Sohail Chatoor, et al. 2008. “A Quantitative Spatiotemporal Atlas of Gene Expression in the Drosophila Blastoderm.” <em>Cell</em> 133 (2): 364–74. <a href="https://doi.org/10.1016/j.cell.2008.01.053">https://doi.org/10.1016/j.cell.2008.01.053</a>.</p>
</div>
<div id="ref-Friedrich1991">
<p>Friedrich, G., and P. Soriano. 1991. “Promoter traps in embryonic stem cells: a genetic screen to identify and mutate developmental genes in mice.” <em>Genes &amp; Development</em> 5 (9): 1513–23. <a href="https://doi.org/10.1101/gad.5.9.1513">https://doi.org/10.1101/gad.5.9.1513</a>.</p>
</div>
<div id="ref-Frohman1988">
<p>Frohman, M. A., M. K. Dush, and G. R. Martin. 1988. “Rapid production of full-length cDNAs from rare transcripts: amplification using a single gene-specific oligonucleotide primer.” <em>Proceedings of the National Academy of Sciences</em> 85 (23): 8998–9002. <a href="https://doi.org/10.1073/pnas.85.23.8998">https://doi.org/10.1073/pnas.85.23.8998</a>.</p>
</div>
<div id="ref-Gall1969">
<p>Gall, Joseph G, Mary Lou, Pardue Kline, and Norman H Giles. 1969. “Formation and Detecction of RNA-DNA Hybrid Molecules in Cytological Preparations.” <em>PNAS</em> 63 (2): 378–83. <a href="https://doi.org/10.1073/pnas.63.2.378">https://doi.org/10.1073/pnas.63.2.378</a>.</p>
</div>
<div id="ref-Gawantka1998">
<p>Gawantka, Volker, Nicolas Pollet, Hajo Delius, Martin Vingron, Ralf Pfister, Rebecca Nitsch, Claudia Blumenstock, and Christof Niehrs. 1998. “Gene expression screening in Xenopus identifies molecular pathways, predicts gene function and provides a global view of embryonic patterning.” <em>Mechanisms of Development</em> 77 (2): 95–141. <a href="https://doi.org/10.1016/S0925-4773(98)00115-4">https://doi.org/10.1016/S0925-4773(98)00115-4</a>.</p>
</div>
<div id="ref-Geffers2013">
<p>Geffers, Lars, Benjamin Tetzlaff, Xiao Cui, Jun Yan, and Gregor Eichele. 2012. “METscout: a pathfinder exploring the landscape of metabolites, enzymes and transporters.” <em>Nucleic Acids Research</em> 41 (D1): D1047–D1054. <a href="https://doi.org/10.1093/nar/gks886">https://doi.org/10.1093/nar/gks886</a>.</p>
</div>
<div id="ref-Giani2020">
<p>Giani, Alice Maria, Guido Roberto Gallo, Luca Gianfranceschi, and Giulio Formenti. 2020. “Long walk to genomics: History and current approaches to genome sequencing and assembly.” <em>Computational and Structural Biotechnology Journal</em> 18 (January): 9–19. <a href="https://doi.org/10.1016/j.csbj.2019.11.002">https://doi.org/10.1016/j.csbj.2019.11.002</a>.</p>
</div>
<div id="ref-Gilchrist2009">
<p>Gilchrist, Michael J., Mikkel B. Christensen, Odile Bronchain, Frédéric Brunet, Albert Chesneau, Ursula Fenger, Timothy J. Geach, et al. 2009. “Database of queryable gene expression patterns for Xenopus.” <em>Developmental Dynamics</em> 238 (6): 1379–88. <a href="https://doi.org/10.1002/dvdy.21940">https://doi.org/10.1002/dvdy.21940</a>.</p>
</div>
<div id="ref-Gong2003">
<p>Gong, Shiaoching, Chen Zheng, Martin L. Doughty, Kasia Losos, Nicholas Didkovsky, Uta B. Schambra, Norma J. Nowak, et al. 2003. “A gene expression atlas of the central nervous system based on bacterial artificial chromosomes.” <em>Nature</em> 425 (6961): 917–25. <a href="https://doi.org/10.1038/nature02033">https://doi.org/10.1038/nature02033</a>.</p>
</div>
<div id="ref-Gossler1989">
<p>Gossler, Achim, A. Joyner, Janet Rossant, and W. Skarnes. 1989. “Mouse embryonic stem cells and reporter constructs to detect developmentally regulated genes.” <em>Science</em> 244 (4903): 463–65. <a href="https://doi.org/10.1126/science.2497519">https://doi.org/10.1126/science.2497519</a>.</p>
</div>
<div id="ref-Harding2011">
<p>Harding, Simon D., Chris Armit, Jane Armstrong, Jane Brennan, Ying Cheng, Bernard Haggarty, Derek Houghton, et al. 2011. “The GUDMAP database - an online resource for genitourinary research.” <em>Development</em> 138 (13): 2845–53. <a href="https://doi.org/10.1242/dev.063594">https://doi.org/10.1242/dev.063594</a>.</p>
</div>
<div id="ref-Harrison1973">
<p>Harrison, P. R., D. Conkie, J. Paul, and K. Jones. 1973. “Localisation of cellular globin messenger RNA by in situ hybridisation to complementary DNA.” <em>FEBS Letters</em> 32 (1): 109–12. <a href="https://doi.org/10.1016/0014-5793(73)80749-5">https://doi.org/10.1016/0014-5793(73)80749-5</a>.</p>
</div>
<div id="ref-Haudry2007">
<p>Haudry, Yannick, Hugo Berube, Ivica Letunic, P.-D. Weeber, Julien Gagneur, Charles Girardot, Misha Kapushesky, et al. 2007. “4DXpress: a database for cross-species expression pattern comparisons.” <em>Nucleic Acids Research</em> 36 (Database): D847–D853. <a href="https://doi.org/10.1093/nar/gkm797">https://doi.org/10.1093/nar/gkm797</a>.</p>
</div>
<div id="ref-Henrich2003">
<p>Henrich, Thorsten. 2003. “MEPD: a Medaka gene expression pattern database.” <em>Nucleic Acids Research</em> 31 (1): 72–74. <a href="https://doi.org/10.1093/nar/gkg017">https://doi.org/10.1093/nar/gkg017</a>.</p>
</div>
<div id="ref-Hiwatashi2001">
<p>Hiwatashi, Yuji, Tomoaki Nishiyama, Tomomichi Fujita, and Mitsuyasu Hasebe. 2001. “Establishment of gene-trap and enhancer-trap systems in the moss Physcomitrella patens.” <em>The Plant Journal</em> 28 (1): 105–16. <a href="https://doi.org/10.1046/j.1365-313X.2001.01121.x">https://doi.org/10.1046/j.1365-313X.2001.01121.x</a>.</p>
</div>
<div id="ref-Hope1991">
<p>Hope, I A. 1991. “’Promoter trapping’ in Caenorhabditis elegans.” Vol. 113. <a href="https://dev.biologists.org/content/develop/113/2/399.full.pdf">https://dev.biologists.org/content/develop/113/2/399.full.pdf</a>.</p>
</div>
<div id="ref-Howe2017">
<p>Howe, Douglas G., Yvonne M. Bradford, Anne Eagle, David Fashena, Ken Frazer, Patrick Kalita, Prita Mani, et al. 2017. “The Zebrafish Model Organism Database: New support for human disease models, mutation details, gene expression phenotypes and searching.” <em>Nucleic Acids Research</em> 45 (D1): D758–D768. <a href="https://doi.org/10.1093/nar/gkw1116">https://doi.org/10.1093/nar/gkw1116</a>.</p>
</div>
<div id="ref-Huber2018">
<p>Huber, D., L. Voith von Voithenberg, and G. V. Kaigala. 2018. “Fluorescence in situ hybridization (FISH): History, limitations and what to expect from micro-scale FISH?” <em>Micro and Nano Engineering</em> 1 (November): 15–24. <a href="https://doi.org/10.1016/j.mne.2018.10.006">https://doi.org/10.1016/j.mne.2018.10.006</a>.</p>
</div>
<div id="ref-Hunt-Newbury2007">
<p>Hunt-Newbury, Rebecca, Ryan Viveiros, Robert Johnsen, Allan Mah, Dina Anastas, Lily Fang, Erin Halfnight, et al. 2007. “High-Throughput In Vivo Analysis of Gene Expression in Caenorhabditis elegans.” Edited by John Sulston. <em>PLoS Biology</em> 5 (9): e237. <a href="https://doi.org/10.1371/journal.pbio.0050237">https://doi.org/10.1371/journal.pbio.0050237</a>.</p>
</div>
<div id="ref-Janning1997">
<p>Janning, Wilfried. 1997. “FlyView, aDrosophilaimage database, and otherDrosophiladatabases.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 469–75. <a href="https://doi.org/https://doi.org/10.1006/scdb.1997.0172">https://doi.org/https://doi.org/10.1006/scdb.1997.0172</a>.</p>
</div>
<div id="ref-Jenett2012">
<p>Jenett, Arnim, Gerald M. Rubin, Teri T. B. Ngo, David Shepherd, Christine Murphy, Heather Dionne, Barret D. Pfeiffer, et al. 2012. “A GAL4-Driver Line Resource for Drosophila Neurobiology.” <em>Cell Reports</em> 2 (4): 991–1001. <a href="https://doi.org/10.1016/j.celrep.2012.09.011">https://doi.org/10.1016/j.celrep.2012.09.011</a>.</p>
</div>
<div id="ref-John1969">
<p>John, H. A., M. L. Birnstiel, and K. W. Jones. 1969. “RNA-DNA hybrids at the cytological level.” <em>Nature</em> 223 (5206): 582–87. <a href="https://doi.org/10.1038/223582a0">https://doi.org/10.1038/223582a0</a>.</p>
</div>
<div id="ref-Johnson2005">
<p>Johnson, Alexander A. T., Julian M. Hibberd, Céline Gay, Pauline A. Essah, Jim Haseloff, Mark Tester, and Emmanuel Guiderdoni. 2005. “Spatial control of transgene expression in rice (Oryza sativa L.) using the GAL4 enhancer trapping system.” <em>The Plant Journal</em> 41 (5): 779–89. <a href="https://doi.org/10.1111/j.1365-313X.2005.02339.x">https://doi.org/10.1111/j.1365-313X.2005.02339.x</a>.</p>
</div>
<div id="ref-Karaiskos2017">
<p>Karaiskos, Nikos, Philipp Wahle, Jonathan Alles, Anastasiya Boltengagen, Salah Ayoub, Claudia Kipar, Christine Kocks, Nikolaus Rajewsky, and Robert P. Zinzen. 2017. “The Drosophila embryo at single-cell transcriptome resolution.” <em>Science</em> 358 (6360): 194–99. <a href="https://doi.org/10.1126/science.aan3235">https://doi.org/10.1126/science.aan3235</a>.</p>
</div>
<div id="ref-Karali2010">
<p>Karali, Marianthi, Ivana Peluso, Vincenzo A. Gennarino, Marchesa Bilio, Roberta Verde, Giampiero Lago, Pascal Dollé, and Sandro Banfi. 2010. “miRNeye: a microRNA expression atlas of the mouse eye.” <em>BMC Genomics</em> 11 (1): 715. <a href="https://doi.org/10.1186/1471-2164-11-715">https://doi.org/10.1186/1471-2164-11-715</a>.</p>
</div>
<div id="ref-Kawakami2010">
<p>Kawakami, Koichi, Gembu Abe, Tokuko Asada, Kazuhide Asakawa, Ryuichi Fukuda, Aki Ito, Pradeep Lal, et al. 2010. “ZTrap: Zebrafish gene trap and enhancer trap database.” <em>BMC Developmental Biology</em> 10 (1): 105. <a href="https://doi.org/10.1186/1471-213X-10-105">https://doi.org/10.1186/1471-213X-10-105</a>.</p>
</div>
<div id="ref-Kawashima2000">
<p>Kawashima, T. 2000. “MAGEST: MAboya Gene Expression patterns and Sequence Tags.” <em>Nucleic Acids Research</em> 28 (1): 133–35. <a href="https://doi.org/10.1093/nar/28.1.133">https://doi.org/10.1093/nar/28.1.133</a>.</p>
</div>
<div id="ref-Kitahara2001">
<p>Kitahara, Osamu, Yoichi Furukawa, Toshihiro Tanaka, Chikashi Kihara, Kenji Ono, Renpei Yanagawa, Marcelo E Nita, Toshihisa Takagi, Yusuke Nakamura, and Tatsuhiko Tsunoda. 2001. “Alterations of Gene Expression during Colorectal Carcinogenesis Revealed by cDNA Microarrays after Laser-Capture Microdissection of Tumor Tissues and Normal Epithelia.” <em>Cancer Research</em> 61 (9): 3544 LP–3549. <a href="http://cancerres.aacrjournals.org/content/61/9/3544.abstract">http://cancerres.aacrjournals.org/content/61/9/3544.abstract</a>.</p>
</div>
<div id="ref-Kloosterman2006">
<p>Kloosterman, Wigard P., Erno Wienholds, Ewart de Bruijn, Sakari Kauppinen, and Ronald H A Plasterk. 2006. “In situ detection of miRNAs in animal embryos using LNA-modified oligonucleotide probes.” <em>Nature Methods</em> 3 (1): 27–29. <a href="https://doi.org/10.1038/nmeth843">https://doi.org/10.1038/nmeth843</a>.</p>
</div>
<div id="ref-Kopczynski1998">
<p>Kopczynski, Casey C., Jasprina N. Noordermeer, Thomas L. Serano, W.-Y. Chen, John D. Pendleton, Suzanna Lewis, Corey S. Goodman, and Gerald M. Rubin. 1998. “A high throughput screen to identify secreted and transmembrane proteins involved in Drosophila embryogenesis.” <em>Proceedings of the National Academy of Sciences</em> 95 (17): 9973–8. <a href="https://doi.org/10.1073/pnas.95.17.9973">https://doi.org/10.1073/pnas.95.17.9973</a>.</p>
</div>
<div id="ref-Kumar2017">
<p>Kumar, Sudhir, Charlotte Konikoff, Maxwell Sanderford, Li Liu, Stuart Newfeld, Jieping Ye, and Rob J. Kulathinal. 2017. “FlyExpress 7: An Integrated Discovery Platform To Study Coexpressed Genes Using in situ Hybridization Images in Drosophila.” <em>G3; Genes|Genomes|Genetics</em> 7 (8): 2791–7. <a href="https://doi.org/10.1534/g3.117.040345">https://doi.org/10.1534/g3.117.040345</a>.</p>
</div>
<div id="ref-Kvon2014">
<p>Kvon, Evgeny Z., Tomas Kazmar, Gerald Stampfel, J. Omar Yáñez-Cuna, Michaela Pagani, Katharina Schernhuber, Barry J. Dickson, and Alexander Stark. 2014. “Genome-scale functional characterization of Drosophila developmental enhancers in vivo.” <em>Nature</em> 512 (7512): 91–95. <a href="https://doi.org/10.1038/nature13395">https://doi.org/10.1038/nature13395</a>.</p>
</div>
<div id="ref-Lander2001">
<p>Lander, Eric S., Lauren M. Linton, Bruce Birren, Chad Nusbaum, Michael C. Zody, Jennifer Baldwin, Keri Devon, et al. 2001. “Initial sequencing and analysis of the human genome.” <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.</p>
</div>
<div id="ref-Langer-Safer1982">
<p>Langer-Safer, P. R., M. Levine, and D. C. Ward. 1982. “Immunological method for mapping genes on Drosophila polytene chromosomes.” <em>Proceedings of the National Academy of Sciences</em> 79 (14): 4381–5. <a href="https://doi.org/10.1073/pnas.79.14.4381">https://doi.org/10.1073/pnas.79.14.4381</a>.</p>
</div>
<div id="ref-Leighton2001">
<p>Leighton, Philip A., Kevin J. Mitchell, Lisa V. Goodrich, Xiaowei Lu, Kathy Pinson, Paul Scherz, William C. Skarnes, and Marc Tessier-Lavigne. 2001. “Defining brain wiring patterns and mechanisms through gene trapping in mice.” <em>Nature</em> 410 (6825): 174–79. <a href="https://doi.org/10.1038/35065539">https://doi.org/10.1038/35065539</a>.</p>
</div>
<div id="ref-Lein2017">
<p>Lein, Ed, Lars E. Borm, and Sten Linnarsson. 2017. “The promise of spatial transcriptomics for neuroscience in the era of molecular cell typing.” <a href="https://doi.org/10.1126/science.aan6827">https://doi.org/10.1126/science.aan6827</a>.</p>
</div>
<div id="ref-Lein2004">
<p>Lein, Ed S. 2004. “Defining a Molecular Atlas of the Hippocampus Using DNA Microarrays and High-Throughput In Situ Hybridization.” <em>Journal of Neuroscience</em> 24 (15): 3879–89. <a href="https://doi.org/10.1523/JNEUROSCI.4710-03.2004">https://doi.org/10.1523/JNEUROSCI.4710-03.2004</a>.</p>
</div>
<div id="ref-Lein2007">
<p>Lein, Ed S., Michael J. Hawrylycz, Nancy Ao, Mikael Ayres, Amy Bensinger, Amy Bernard, Andrew F. Boe, et al. 2007. “Genome-wide atlas of gene expression in the adult mouse brain.” <em>Nature</em> 445 (7124): 168–76. <a href="https://doi.org/10.1038/nature05453">https://doi.org/10.1038/nature05453</a>.</p>
</div>
<div id="ref-Lecuyer2007">
<p>Lécuyer, Eric, Hideki Yoshida, Neela Parthasarathy, Christina Alm, Tomas Babak, Tanja Cerovina, Timothy R. Hughes, Pavel Tomancak, and Henry M. Krause. 2007. “Global Analysis of mRNA Localization Reveals a Prominent Role in Organizing Cellular Architecture and Function.” <em>Cell</em> 131 (1): 174–87. <a href="https://doi.org/10.1016/j.cell.2007.08.003">https://doi.org/10.1016/j.cell.2007.08.003</a>.</p>
</div>
<div id="ref-Liao2020">
<p>Liao, Jie, Xiaoyan Lu, Xin Shao, Ling Zhu, and Xiaohui Fan. 2020. “Uncovering an Organ’s Molecular Architecture at Single-Cell Resolution by Spatially Resolved Transcriptomics.” <em>Trends in Biotechnology</em>, June. <a href="https://doi.org/10.1016/j.tibtech.2020.05.006">https://doi.org/10.1016/j.tibtech.2020.05.006</a>.</p>
</div>
<div id="ref-Lis1983">
<p>Lis, John T., Jeffrey A. Simon, and Claudia A. Sutton. 1983. “New heat shock puffs and <span class="math inline">\(\beta\)</span>=galactosidase activity resulting from transformation of Drosophila with an hsp70-lacZ hybrid gene.” <em>Cell</em> 35 (2): 403–10. <a href="https://doi.org/10.1016/0092-8674(83)90173-3">https://doi.org/10.1016/0092-8674(83)90173-3</a>.</p>
</div>
<div id="ref-Lovell2020">
<p>Lovell, Peter V., Morgan Wirthlin, Taylor Kaser, Alexa A. Buckner, Julia B. Carleton, Brian R. Snider, Anne K. McHugh, Alexander Tolpygo, Partha P. Mitra, and Claudio V. Mello. 2020. “ZEBrA: Zebra finch Expression Brain Atlas—A resource for comparative molecular neuroanatomy and brain evolution studies.” <em>Journal of Comparative Neurology</em> 528 (12): 2099–2131. <a href="https://doi.org/10.1002/cne.24879">https://doi.org/10.1002/cne.24879</a>.</p>
</div>
<div id="ref-LuengoHendriks2006a">
<p>Luengo Hendriks, Cris L., Soile V. E. Keränen, Charless C. Fowlkes, Lisa Simirenko, Gunther H. Weber, Angela H. DePace, Clara Henriquez, et al. 2006. “Three-dimensional morphology and gene expression in the Drosophila blastoderm at cellular resolution I: Data acquisition pipeline.” <em>Genome Biology</em> 7 (12): R123. <a href="https://doi.org/10.1186/gb-2006-7-12-r123">https://doi.org/10.1186/gb-2006-7-12-r123</a>.</p>
</div>
<div id="ref-Luo1999">
<p>Luo, Lin, Ranelle C. Salunga, Hongqing Guo, Anton Bittner, K. C. Joy, Jose E. Galindo, Huinian Xiao, et al. 1999. “Gene expression profiles of laser-captured adjacent neuronal subtypes.” <em>Nature Medicine</em> 5 (1): 117–22. <a href="https://doi.org/10.1038/4806">https://doi.org/10.1038/4806</a>.</p>
</div>
<div id="ref-Lynch1995">
<p>Lynch, Andrew S., David Briggs, and Ian A. Hope. 1995. “Developmental expression pattern screen for genes predicted in the C. elegans genome sequencing project.” <em>Nature Genetics</em> 11 (3): 309–13. <a href="https://doi.org/10.1038/ng1195-309">https://doi.org/10.1038/ng1195-309</a>.</p>
</div>
<div id="ref-Marquart2015">
<p>Marquart, Gregory D., Kathryn M. Tabor, Mary Brown, Jennifer L. Strykowski, Gaurav K. Varshney, Matthew C. LaFave, Thomas Mueller, Shawn M. Burgess, Shin-ichi Higashijima, and Harold A. Burgess. 2015. “A 3D Searchable Database of Transgenic Zebrafish Gal4 and Cre Lines for Functional Neuroanatomy Studies.” <em>Frontiers in Neural Circuits</em> 9 (November): 1–17. <a href="https://doi.org/10.3389/fncir.2015.00078">https://doi.org/10.3389/fncir.2015.00078</a>.</p>
</div>
<div id="ref-Martinelli1997">
<p>Martinelli, Sylvia D., Clive G. Brown, and Richard Durbin. 1997. “Gene expression and development databases forC. elegans.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 459–67. <a href="https://doi.org/10.1006/scdb.1997.0171">https://doi.org/10.1006/scdb.1997.0171</a>.</p>
</div>
<div id="ref-McKee2005">
<p>McKee, Adrienne E., Emmanuel Minet, Charlene Stern, Shervin Riahi, Charles D. Stiles, and Pamela A. Silver. 2005. “A genome-wide in situ hybridization map of RNA-binding proteins reveals anatomically restricted expression in the developing mouse brain.” <em>BMC Developmental Biology</em> 5 (1): 14. <a href="https://doi.org/10.1186/1471-213X-5-14">https://doi.org/10.1186/1471-213X-5-14</a>.</p>
</div>
<div id="ref-Myers2000">
<p>Myers, Eugene W. 2000. “A Whole-Genome Assembly of Drosophila.” <em>Science</em> 287 (5461): 2196–2204. <a href="https://doi.org/10.1126/science.287.5461.2196">https://doi.org/10.1126/science.287.5461.2196</a>.</p>
</div>
<div id="ref-Nakayama2005">
<p>Nakayama, Naomi, Juana M. Arroyo, Joseph Simorowski, Bruce May, Robert Martienssen, and Vivian F. Irish. 2005. “Gene Trap Lines Define Domains of Gene Regulation in Arabidopsis Petals and Stamens.” <em>The Plant Cell</em> 17 (9): 2486–2506. <a href="https://doi.org/10.1105/tpc.105.033985">https://doi.org/10.1105/tpc.105.033985</a>.</p>
</div>
<div id="ref-Ohyama2000">
<p>Ohyama, H., X. Zhang, Y. Kohno, I. Alevizos, M. Posner, D. T. Wong, and R. Todd. 2000. “Laser Capture Microdissection-Generated Target Sample for High-Density Oligonucleotide Array Hybridization.” <em>BioTechniques</em> 29 (3): 530–36. <a href="https://doi.org/10.2144/00293st05">https://doi.org/10.2144/00293st05</a>.</p>
</div>
<div id="ref-OKane1987">
<p>O’Kane, C. J., and W. J. Gehring. 1987. “Detection in situ of genomic regulatory elements in Drosophila.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 84 (24): 9123–7. <a href="https://doi.org/10.1073/pnas.84.24.9123">https://doi.org/10.1073/pnas.84.24.9123</a>.</p>
</div>
<div id="ref-Ortiz2020">
<p>Ortiz, Cantin, Jose Fernandez Navarro, Aleksandra Jurek, Antje Märtin, Joakim Lundeberg, and Konstantinos Meletis. 2020. “Molecular atlas of the adult mouse brain.” <em>Science Advances</em> 6 (26): eabb3446. <a href="https://doi.org/10.1126/sciadv.abb3446">https://doi.org/10.1126/sciadv.abb3446</a>.</p>
</div>
<div id="ref-Perez-Martin2017">
<p>Pérez-Martín, Fernando, Fernando J. Yuste-Lisbona, Benito Pineda, María Pilar Angarita-Díaz, Begoña García-Sogo, Teresa Antón, Sibilla Sánchez, et al. 2017. “A collection of enhancer trap insertional mutants for functional genomics in tomato.” <em>Plant Biotechnology Journal</em> 15 (11): 1439–52. <a href="https://doi.org/10.1111/pbi.12728">https://doi.org/10.1111/pbi.12728</a>.</p>
</div>
<div id="ref-Puchalski2018">
<p>Puchalski, Ralph B, Nameeta Shah, Jeremy Miller, Rachel Dalley, Steve R Nomura, Jae-Guen Yoon, Kimberly A Smith, et al. 2018. “An anatomic transcriptional atlas of human glioblastoma.” <em>Science</em> 360 (6389): 660 LP–663. <a href="https://doi.org/10.1126/science.aaf2666">https://doi.org/10.1126/science.aaf2666</a>.</p>
</div>
<div id="ref-Ringwald1994">
<p>Ringwald, Martin, Richard Baldock, Jonathan Bard, Matthew Kaufman, Janan T. Eppig, Joel E. Richardson, Joseph H. Nadeau, and Duncan Davidson. 1994. “A database for mouse development.” <em>Science</em> 265 (5181): 2033–4. <a href="https://doi.org/10.1126/science.8091224">https://doi.org/10.1126/science.8091224</a>.</p>
</div>
<div id="ref-Ringwald1997">
<p>Ringwald, Martin, Geoffrey L Davis, Alex G Smith, Laura E Trepanier, Dale A Begley, Joel E Richardson, and Janan T Eppig. 1997. “The mouse gene expression database GXD.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 489–97. <a href="https://doi.org/https://doi.org/10.1006/scdb.1997.0177">https://doi.org/https://doi.org/10.1006/scdb.1997.0177</a>.</p>
</div>
<div id="ref-Ringwald1999">
<p>Ringwald, Martin, Mary E Mangan, Janan T Eppig, James A Kadin, and Joel E Richardson. 1999. “GXD: a Gene Expression Database for the laboratory mouse.” <em>Nucleic Acids Research</em> 27 (1): 106–12. <a href="https://doi.org/10.1093/nar/27.1.106">https://doi.org/10.1093/nar/27.1.106</a>.</p>
</div>
<div id="ref-Rosen1993">
<p>Rosen, Barry, and Rosa S. P. Beddington. 1993. “Whole-mount in situ hybridization in the mouse embryo: gene expression in three dimensions.” <em>Trends in Genetics</em> 9 (5): 162–67. <a href="https://doi.org/10.1016/0168-9525(93)90162-B">https://doi.org/10.1016/0168-9525(93)90162-B</a>.</p>
</div>
<div id="ref-Salgado2008">
<p>Salgado, David, Gregory Gimenez, François Coulier, and Christophe Marcelle. 2008. “COMPARE, a multi-organism system for cross-species data comparison and transfer of information.” <em>Bioinformatics</em> 24 (3): 447–49. <a href="https://doi.org/10.1093/bioinformatics/btm599">https://doi.org/10.1093/bioinformatics/btm599</a>.</p>
</div>
<div id="ref-Satija2015">
<p>Satija, Rahul, Jeffrey A. Farrell, David Gennert, Alexander F. Schier, and Aviv Regev. 2015. “Spatial reconstruction of single-cell gene expression data.” <em>Nature Biotechnology</em> 33 (5): 495–502. <a href="https://doi.org/10.1038/nbt.3192">https://doi.org/10.1038/nbt.3192</a>.</p>
</div>
<div id="ref-Satou2001">
<p>Satou, Yutaka, Naohito Takatori, Lixy Yamada, Yasuaki Mochizuki, Makoto Hamaguchi, Hisayoshi Ishikawa, Shota Chiba, et al. 2001. “Gene expression profiles in &lt;em&gt;Ciona intestinalis&lt;/em&gt; tailbud embryos.” <em>Development</em> 128 (15): 2893 LP–2904. <a href="http://dev.biologists.org/content/128/15/2893.abstract">http://dev.biologists.org/content/128/15/2893.abstract</a>.</p>
</div>
<div id="ref-Seydoux1994">
<p>Seydoux, G, and A Fire. 1994. “Soma-germline asymmetry in the distributions of embryonic RNAs in Caenorhabditis elegans.” <em>Development</em> 120 (10): 2823 LP–2834. <a href="http://dev.biologists.org/content/120/10/2823.abstract">http://dev.biologists.org/content/120/10/2823.abstract</a>.</p>
</div>
<div id="ref-Sgroi1999">
<p>Sgroi, Dennis C, Sarena Teng, Greg Robinson, Rebbecca LeVangie, James R Hudson, and Abdel G Elkahloun. 1999. “&lt;em&gt;In Vivo&lt;/em&gt; Gene Expression Profile Analysis of Human Breast Cancer Progression.” <em>Cancer Research</em> 59 (22): 5656 LP–5661. <a href="http://cancerres.aacrjournals.org/content/59/22/5656.abstract">http://cancerres.aacrjournals.org/content/59/22/5656.abstract</a>.</p>
</div>
<div id="ref-Shcherbatyy2014">
<p>Shcherbatyy, Volodymyr, James Carson, Murat Yaylaoglu, Katharina Jäckle, Frauke Grabbe, Maren Brockmeyer, Halenur Yavuz, and Gregor Eichele. 2015. “A Digital Atlas of Ion Channel Expression Patterns in the Two-Week-Old Rat Brain.” <em>Neuroinformatics</em> 13 (1): 111–25. <a href="https://doi.org/10.1007/s12021-014-9247-0">https://doi.org/10.1007/s12021-014-9247-0</a>.</p>
</div>
<div id="ref-Siegert2009">
<p>Siegert, Sandra, Brigitte Gross Scherf, Karina Del Punta, Nick Didkovsky, Nathaniel Heintz, and Botond Roska. 2009. “Genetic address book for retinal cell types.” <em>Nature Neuroscience</em> 12 (9): 1197–1204. <a href="https://doi.org/10.1038/nn.2370">https://doi.org/10.1038/nn.2370</a>.</p>
</div>
<div id="ref-Singer1982">
<p>Singer, R. H., and D. C. Ward. 1982. “Actin gene expression visualized in chicken muscle tissue culture by using in situ hybridization with a biotinated nucleotide analog.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 79 (23 I): 7331–5. <a href="https://doi.org/10.1073/pnas.79.23.7331">https://doi.org/10.1073/pnas.79.23.7331</a>.</p>
</div>
<div id="ref-Skarnes1995">
<p>Skarnes, William C., Julie E. Moss, Stella M. Hurtley, and Rosa S. P. Beddington. 1995. “Capturing genes encoding membrane and secreted proteins important for mouse development.” <em>Proceedings of the National Academy of Sciences</em> 92 (14): 6592–6. <a href="https://doi.org/10.1073/pnas.92.14.6592">https://doi.org/10.1073/pnas.92.14.6592</a>.</p>
</div>
<div id="ref-Skarnes2011">
<p>Skarnes, William C., Barry Rosen, Anthony P. West, Manousos Koutsourakis, Wendy Bushell, Vivek Iyer, Alejandro O. Mujica, et al. 2011. “A conditional knockout resource for the genome-wide study of mouse gene function.” <em>Nature</em> 474 (7351): 337–42. <a href="https://doi.org/10.1038/nature10163">https://doi.org/10.1038/nature10163</a>.</p>
</div>
<div id="ref-Spradling1982">
<p>Spradling, A., and G. Rubin. 1982. “Transposition of cloned P elements into Drosophila germ line chromosomes.” <em>Science</em> 218 (4570): 341–47. <a href="https://doi.org/10.1126/science.6289435">https://doi.org/10.1126/science.6289435</a>.</p>
</div>
<div id="ref-Sprague2003">
<p>Sprague, Judy. 2003. “The Zebrafish Information Network (ZFIN): the zebrafish model organism database.” <em>Nucleic Acids Research</em> 31 (1): 241–43. <a href="https://doi.org/10.1093/nar/gkg027">https://doi.org/10.1093/nar/gkg027</a>.</p>
</div>
<div id="ref-Stanford1998">
<p>Stanford, William L., Georgina Caruana, Katherine A. Vallis, Maneesha Inamdar, Michihiro Hidaka, Victoria L. Bautch, and Alan Bernstein. 1998. “Expression Trapping: Identification of Novel Genes Expressed in Hematopoietic and Endothelial Lineages by Gene Trapping in ES Cells.” <em>Blood</em> 92 (12): 4622–31. <a href="https://doi.org/10.1182/blood.V92.12.4622">https://doi.org/10.1182/blood.V92.12.4622</a>.</p>
</div>
<div id="ref-Stanford2001">
<p>Stanford, William L., Jason B. Cohn, and Sabine P. Cordes. 2001. “Gene-trap mutagenesis: past, present and beyond.” <em>Nature Reviews Genetics</em> 2 (10): 756–68. <a href="https://doi.org/10.1038/35093548">https://doi.org/10.1038/35093548</a>.</p>
</div>
<div id="ref-Stapleton2002">
<p>Stapleton, Mark. 2002. “The Drosophila Gene Collection: Identification of Putative Full-Length cDNAs for 70% of D. melanogaster Genes.” <em>Genome Research</em> 12 (8): 1294–1300. <a href="https://doi.org/10.1101/gr.269102">https://doi.org/10.1101/gr.269102</a>.</p>
</div>
<div id="ref-Sulston1992">
<p>Sulston, J., Z. Du, K. Thomas, R. Wilson, L. Hillier, R. Staden, N. Halloran, et al. 1992. “The C. elegans genome sequencing project: A beginning.” <em>Nature</em> 356 (6364): 37–41. <a href="https://doi.org/10.1038/356037a0">https://doi.org/10.1038/356037a0</a>.</p>
</div>
<div id="ref-Sundaresan1995">
<p>Sundaresan, Venkatesan, Patricia Springer, Thomas Volpe, Samuel Haward, Jonathan D. G. Jones, Caroline Dean, Hong Ma, and Robert Martienssen. 1995. “Patterns of gene action in plant development revealed by enhancer trap and gene trap transposable elements.” <em>Genes &amp; Development</em> 9 (14): 1797–1810. <a href="https://doi.org/10.1101/gad.9.14.1797">https://doi.org/10.1101/gad.9.14.1797</a>.</p>
</div>
<div id="ref-Sisecioglu2015">
<p>Şişecioğlu, Melda, Harun Budak, Lars Geffers, Murat Çankaya, Mehmet Çiftci, Christina Thaller, Gregor Eichele, Ömer İrfan Küfrevioğlu, and Hasan Özdemir. 2015. “A compendium of expression patterns of cholesterol biosynthetic enzymes in the mouse embryo.” <em>Journal of Lipid Research</em> 56 (8): 1551–9. <a href="https://doi.org/10.1194/jlr.M059634">https://doi.org/10.1194/jlr.M059634</a>.</p>
</div>
<div id="ref-Tassy2010">
<p>Tassy, Olivier, Delphine Dauga, Fabrice Daian, Daniel Sobral, François Robin, Pierre Khoueiry, David Salgado, et al. 2010. “The ANISEED database: Digital representation, formalization, and elucidation of a chordate developmental program.” <em>Genome Research</em> 20 (10): 1459–68. <a href="https://doi.org/10.1101/gr.108175.110">https://doi.org/10.1101/gr.108175.110</a>.</p>
</div>
<div id="ref-Tautz1989">
<p>Tautz, Diethard, and Christine Pfeifle. 1989. “A non-radioactive in situ hybridization method for the localization of specific RNAs in Drosophila embryos reveals translational control of the segmentation gene hunchback.” <em>Chromosoma</em> 98 (2): 81–85. <a href="https://doi.org/10.1007/BF00291041">https://doi.org/10.1007/BF00291041</a>.</p>
</div>
<div id="ref-Consortium1998">
<p>The C. elegans Sequencing Consortium. 1998. “Genome Sequence of the Nematode C. elegans: A Platform for Investigating Biology.” <em>Science</em> 282 (5396): 2012–8. <a href="https://doi.org/10.1126/science.282.5396.2012">https://doi.org/10.1126/science.282.5396.2012</a>.</p>
</div>
<div id="ref-Thompson2010">
<p>Thompson, Carol, Jonathan Wisor, Chang-Kyu Lee, Sayan Pathak, Dmitry Gerashchenko, Kimberly Smith, Shanna Fischer, et al. 2010. “Molecular and Anatomical Signatures of Sleep Deprivation in the Mouse Brain.” <a href="https://www.frontiersin.org/article/10.3389/fnins.2010.00165">https://www.frontiersin.org/article/10.3389/fnins.2010.00165</a>.</p>
</div>
<div id="ref-Thut2001">
<p>Thut, Catherine J., Ryan B. Rountree, Michael Hwa, and David M. Kingsley. 2001. “A Large-Scale in Situ Screen Provides Molecular Evidence for the Induction of Eye Anterior Segment Structures by the Developing Lens.” <em>Developmental Biology</em> 231 (1): 63–76. <a href="https://doi.org/10.1006/dbio.2000.0140">https://doi.org/10.1006/dbio.2000.0140</a>.</p>
</div>
<div id="ref-Tomancak2002">
<p>Tomancak, Pavel, Amy Beaton, Richard Weiszmann, Elaine Kwan, Sheng Qiang Shu, Suzanna E. Lewis, Stephen Richards, et al. 2002. “Systematic determination of patterns of gene expression during Drosophila embryogenesis.” <em>Genome Biology</em> 3 (12): research0088.1. <a href="https://doi.org/10.1186/gb-2002-3-12-research0088">https://doi.org/10.1186/gb-2002-3-12-research0088</a>.</p>
</div>
<div id="ref-Tuck2015">
<p>Tuck, Elizabeth, Jeanne Estabel, Anika Oellrich, A. K. Maguire, Hibret A. Adissu, Luke Souter, Emma Siragher, et al. 2015. “A gene expression resource generated by genome-wide lacZ profiling in the mouse.” <em>Disease Models &amp; Mechanisms</em> 8 (11): 1467–78. <a href="https://doi.org/10.1242/dmm.021238">https://doi.org/10.1242/dmm.021238</a>.</p>
</div>
<div id="ref-Valoczi2004">
<p>Valoczi, A. 2004. “Sensitive and specific detection of microRNAs by northern blot analysis using LNA-modified oligonucleotide probes.” <em>Nucleic Acids Research</em> 32 (22): e175–e175. <a href="https://doi.org/10.1093/nar/gnh171">https://doi.org/10.1093/nar/gnh171</a>.</p>
</div>
<div id="ref-Venter2001">
<p>Venter, J. Craig, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, et al. 2001. “The Sequence of the Human Genome.” <em>Science</em> 291 (5507): 1304–51. <a href="https://doi.org/10.1126/science.1058040">https://doi.org/10.1126/science.1058040</a>.</p>
</div>
<div id="ref-Visel2013">
<p>Visel, Axel, Leila Taher, Hani Girgis, Dalit May, Olga Golonzhka, Renee V. Hoch, Gabriel L. McKinsey, et al. 2013. “A high-resolution enhancer atlas of the developing telencephalon.” <em>Cell</em> 152 (4): 895–908. <a href="https://doi.org/10.1016/j.cell.2012.12.041">https://doi.org/10.1016/j.cell.2012.12.041</a>.</p>
</div>
<div id="ref-Waterston2002">
<p>Waterston, Robert H., Kerstin Lindblad-Toh, Ewan Birney, Jane Rogers, Josep F. Abril, Pankaj Agarwal, Richa Agarwala, et al. 2002. “Initial sequencing and comparative analysis of the mouse genome.” <em>Nature</em> 420 (6915): 520–62. <a href="https://doi.org/10.1038/nature01262">https://doi.org/10.1038/nature01262</a>.</p>
</div>
<div id="ref-West2015">
<p>West, David B., Ravi K. Pasumarthi, Brian Baridon, Esi Djan, Amanda Trainor, Stephen M. Griffey, Eric K. Engelhard, et al. 2015. “A lacZ reporter gene expression atlas for 313 adult KOMP mutant mouse lines.” <em>Genome Research</em> 25 (4): 598–607. <a href="https://doi.org/10.1101/gr.184184.114">https://doi.org/10.1101/gr.184184.114</a>.</p>
</div>
<div id="ref-Westerfield1997">
<p>Westerfield, Monte, Eckehard Doerry, Arthur E. Kirkpatrick, Wolfgang Driever, and Sarah A. Douglas. 1997. “An on-line database for zebrafish development and genetics research.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 477–88. <a href="https://doi.org/10.1006/scdb.1997.0173">https://doi.org/10.1006/scdb.1997.0173</a>.</p>
</div>
<div id="ref-White2013">
<p>White, Jacqueline K., Anna-Karin Gerdin, Natasha A. Karp, Ed Ryder, Marija Buljan, James N. Bussell, Jennifer Salisbury, et al. 2013. “Genome-wide Generation and Systematic Phenotyping of Knockout Mice Reveals New Roles for Many Genes.” <em>Cell</em> 154 (2): 452–64. <a href="https://doi.org/10.1016/j.cell.2013.06.022">https://doi.org/10.1016/j.cell.2013.06.022</a>.</p>
</div>
<div id="ref-Wienholds2005">
<p>Wienholds, Erno. 2005. “MicroRNA Expression in Zebrafish Embryonic Development.” <em>Science</em> 309 (5732): 310–11. <a href="https://doi.org/10.1126/science.1114519">https://doi.org/10.1126/science.1114519</a>.</p>
</div>
<div id="ref-Wilson1989">
<p>Wilson, Clive, Rebecca Kurth Pearson, Hugo J Bellen, C J O’Kane, Ueli Grossniklaus, and Walter J Gehring. 1989. “P-element-mediated enhancer detection: an efficient method for isolating and characterizing developmentally regulated genes in Drosophila.” <em>Genes &amp; Development</em> 3 (9): 1301–13. <a href="https://doi.org/10.1101/gad.3.9.1301">https://doi.org/10.1101/gad.3.9.1301</a>.</p>
</div>
<div id="ref-Wurst1995a">
<p>Wurst, W, J Rossant, V Prideaux, M Kownacka, A Joyner, D P Hill, F Guillemot, S Gasca, D Cado, and A Auerbach. 1995. “A large-scale gene-trap screen for insertional mutations in developmentally regulated genes in mice.” <em>Genetics</em> 139 (2). <a href="https://www.genetics.org/content/139/2/889">https://www.genetics.org/content/139/2/889</a>.</p>
</div>
<div id="ref-XDB3">
<p>“XDB3.” 2004. <a href="http://xenopus.nibb.ac.jp/">http://xenopus.nibb.ac.jp/</a>.</p>
</div>
<div id="ref-Yang1997">
<p>Yang, Xiangdong W., Peter Model, and Nathaniel Heintz. 1997. “Homologous recombination based modification in Esherichia coli and germline transmission in transgenic mice of a bacterial artificial chromsome.” <em>Nature Biotechnology</em> 15 (9): 859–65. <a href="https://doi.org/10.1038/nbt0997-859">https://doi.org/10.1038/nbt0997-859</a>.</p>
</div>
<div id="ref-Yaylaoglu2005">
<p>Yaylaoglu, Murat Burak, Andrew Titmus, Axel Visel, Gonzalo Alvarez-Bolado, Christina Thaller, and Gregor Eichele. 2005. “Comprehensive expression atlas of fibroblast growth factors and their receptors generated by a novel robotic in situ hybridization platform.” <em>Developmental Dynamics</em> 234 (2): 371–86. <a href="https://doi.org/10.1002/dvdy.20441">https://doi.org/10.1002/dvdy.20441</a>.</p>
</div>
<div id="ref-Yokoyama2009">
<p>Yokoyama, Shigetoshi, Yoshiaki Ito, Hiroe Ueno-Kudoh, Hirohito Shimizu, Kenta Uchibe, Sonia Albini, Kazuhiko Mitsuoka, et al. 2009. “A Systems Approach Reveals that the Myogenesis Genome Network Is Regulated by the Transcriptional Repressor RP58.” <em>Developmental Cell</em> 17 (6): 836–48. <a href="https://doi.org/10.1016/j.devcel.2009.10.011">https://doi.org/10.1016/j.devcel.2009.10.011</a>.</p>
</div>
<div id="ref-Yoshikawa2006">
<p>Yoshikawa, Toshiyuki, Yulan Piao, Jinhui Zhong, Ryo Matoba, Mark G. Carter, Yuxia Wang, Ilya Goldberg, and Minoru S. H. Ko. 2006. “High-throughput screen for genes predominantly expressed in the ICM of mouse blastocysts by whole mount in situ hybridization.” <em>Gene Expression Patterns</em> 6 (2): 213–24. <a href="https://doi.org/10.1016/j.modgep.2005.06.003">https://doi.org/10.1016/j.modgep.2005.06.003</a>.</p>
</div>
<div id="ref-Zeisel2018">
<p>Zeisel, Amit, Hannah Hochgerner, Peter Lönnerberg, Anna Johnsson, Fatima Memic, Job van der Zwan, Martin Häring, et al. 2018. “Molecular Architecture of the Mouse Nervous System.” <em>Cell</em> 174 (4): 999–1014.e22. <a href="https://doi.org/10.1016/j.cell.2018.06.021">https://doi.org/10.1016/j.cell.2018.06.021</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="prequel-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["museumst.pdf"],
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
