<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Museum of Spatial Transcriptomics</title>
  <meta name="description" content="References | Museum of Spatial Transcriptomics" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Museum of Spatial Transcriptomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="References | Museum of Spatial Transcriptomics" />
  <meta name="github-repo" content="pachterlab/LP_2021" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Museum of Spatial Transcriptomics" />
  
  <meta name="twitter:description" content="References | Museum of Spatial Transcriptomics" />
  

<meta name="author" content="Lambda Moses, dlu2@caltech.edu" />
<meta name="author" content="Lior Pachter, lpachter@caltech.edu" />


<meta name="date" content="2021-06-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="future.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Museum of Spatial Transcriptomics</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="part"><span><b>I Prequel era</b></span></li>
<li class="chapter" data-level="2" data-path="prequel.html"><a href="prequel.html"><i class="fa fa-check"></i><b>2</b> Prequel era</a><ul>
<li class="chapter" data-level="2.1" data-path="prequel.html"><a href="prequel.html#traps"><i class="fa fa-check"></i><b>2.1</b> Enhancer and gene traps</a></li>
<li class="chapter" data-level="2.2" data-path="prequel.html"><a href="prequel.html#in-situ-reporter"><i class="fa fa-check"></i><b>2.2</b> In situ reporter</a></li>
<li class="chapter" data-level="2.3" data-path="prequel.html"><a href="prequel.html#wmish"><i class="fa fa-check"></i><b>2.3</b> ISH and WMISH atlases</a></li>
<li class="chapter" data-level="2.4" data-path="prequel.html"><a href="prequel.html#database_prequel"><i class="fa fa-check"></i><b>2.4</b> Databases of the prequel era</a></li>
<li class="chapter" data-level="2.5" data-path="prequel.html"><a href="prequel.html#geo_prequel"><i class="fa fa-check"></i><b>2.5</b> Geography of the prequel era</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="prequel-analysis.html"><a href="prequel-analysis.html"><i class="fa fa-check"></i><b>3</b> Data analysis in the prequel era</a><ul>
<li class="chapter" data-level="3.1" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-patterns"><i class="fa fa-check"></i><b>3.1</b> Gene patterns</a></li>
<li class="chapter" data-level="3.2" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-spatial-regions"><i class="fa fa-check"></i><b>3.2</b> Spatial regions</a></li>
<li class="chapter" data-level="3.3" data-path="prequel-analysis.html"><a href="prequel-analysis.html#prequel-gene-interactions"><i class="fa fa-check"></i><b>3.3</b> Gene interactions</a></li>
<li class="chapter" data-level="3.4" data-path="prequel-analysis.html"><a href="prequel-analysis.html#decline"><i class="fa fa-check"></i><b>3.4</b> Decline</a></li>
<li class="chapter" data-level="3.5" data-path="prequel-analysis.html"><a href="prequel-analysis.html#geo-prequel-analysis"><i class="fa fa-check"></i><b>3.5</b> Geography of prequel data analysis</a></li>
</ul></li>
<li class="part"><span><b>II Current era</b></span></li>
<li class="chapter" data-level="4" data-path="current.html"><a href="current.html"><i class="fa fa-check"></i><b>4</b> From the past to the present</a></li>
<li class="chapter" data-level="5" data-path="current-techs.html"><a href="current-techs.html"><i class="fa fa-check"></i><b>5</b> Current era technologies</a><ul>
<li class="chapter" data-level="5.1" data-path="current-techs.html"><a href="current-techs.html#microdissection"><i class="fa fa-check"></i><b>5.1</b> Microdissection</a><ul>
<li class="chapter" data-level="5.1.1" data-path="current-techs.html"><a href="current-techs.html#history-of-lcm"><i class="fa fa-check"></i><b>5.1.1</b> History of LCM</a></li>
<li class="chapter" data-level="5.1.2" data-path="current-techs.html"><a href="current-techs.html#lcm-usage"><i class="fa fa-check"></i><b>5.1.2</b> Usage of LCM</a></li>
<li class="chapter" data-level="5.1.3" data-path="current-techs.html"><a href="current-techs.html#physical-microdissection"><i class="fa fa-check"></i><b>5.1.3</b> Physical microdissection</a></li>
<li class="chapter" data-level="5.1.4" data-path="current-techs.html"><a href="current-techs.html#de-facto-microdissection"><i class="fa fa-check"></i><b>5.1.4</b> <em>De facto</em> microdissection</a></li>
<li class="chapter" data-level="5.1.5" data-path="current-techs.html"><a href="current-techs.html#summary"><i class="fa fa-check"></i><b>5.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="current-techs.html"><a href="current-techs.html#smfish"><i class="fa fa-check"></i><b>5.2</b> Single molecular FISH</a><ul>
<li class="chapter" data-level="5.2.1" data-path="current-techs.html"><a href="current-techs.html#barcoding"><i class="fa fa-check"></i><b>5.2.1</b> Barcoding strategies</a></li>
<li class="chapter" data-level="5.2.2" data-path="current-techs.html"><a href="current-techs.html#amplification"><i class="fa fa-check"></i><b>5.2.2</b> Signal amplification</a></li>
<li class="chapter" data-level="5.2.3" data-path="current-techs.html"><a href="current-techs.html#crowding"><i class="fa fa-check"></i><b>5.2.3</b> Optical crowding</a></li>
<li class="chapter" data-level="5.2.4" data-path="current-techs.html"><a href="current-techs.html#smfish-usage"><i class="fa fa-check"></i><b>5.2.4</b> Usage of smFISH based techniques</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="current-techs.html"><a href="current-techs.html#iss"><i class="fa fa-check"></i><b>5.3</b> In situ sequencing</a><ul>
<li class="chapter" data-level="5.3.1" data-path="current-techs.html"><a href="current-techs.html#solid"><i class="fa fa-check"></i><b>5.3.1</b> SOLiD and FISSEQ</a></li>
<li class="chapter" data-level="5.3.2" data-path="current-techs.html"><a href="current-techs.html#cpal"><i class="fa fa-check"></i><b>5.3.2</b> cPAL and ISS</a></li>
<li class="chapter" data-level="5.3.3" data-path="current-techs.html"><a href="current-techs.html#sedal"><i class="fa fa-check"></i><b>5.3.3</b> SEDAL and STARmap</a></li>
<li class="chapter" data-level="5.3.4" data-path="current-techs.html"><a href="current-techs.html#sbs"><i class="fa fa-check"></i><b>5.3.4</b> Sequencing by synthesis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="current-techs.html"><a href="current-techs.html#array"><i class="fa fa-check"></i><b>5.4</b> In situ array capture</a></li>
<li class="chapter" data-level="5.5" data-path="current-techs.html"><a href="current-techs.html#no-imaging"><i class="fa fa-check"></i><b>5.5</b> No imaging</a></li>
<li class="chapter" data-level="5.6" data-path="current-techs.html"><a href="current-techs.html#multiomics"><i class="fa fa-check"></i><b>5.6</b> Spatial multi-omics</a></li>
<li class="chapter" data-level="5.7" data-path="current-techs.html"><a href="current-techs.html#database-current"><i class="fa fa-check"></i><b>5.7</b> Databases of the current era</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="text-mining.html"><a href="text-mining.html"><i class="fa fa-check"></i><b>6</b> Text mining LCM transcriptomics abstracts</a><ul>
<li class="chapter" data-level="6.1" data-path="text-mining.html"><a href="text-mining.html#topic-model"><i class="fa fa-check"></i><b>6.1</b> Topic modeling</a></li>
<li class="chapter" data-level="6.2" data-path="text-mining.html"><a href="text-mining.html#word-time"><i class="fa fa-check"></i><b>6.2</b> Changes of word usage through time</a></li>
<li class="chapter" data-level="6.3" data-path="text-mining.html"><a href="text-mining.html#topic-time"><i class="fa fa-check"></i><b>6.3</b> Changes of topic prevalence through time</a></li>
<li class="chapter" data-level="6.4" data-path="text-mining.html"><a href="text-mining.html#association-of-topics-with-city"><i class="fa fa-check"></i><b>6.4</b> Association of topics with city</a></li>
<li class="chapter" data-level="6.5" data-path="text-mining.html"><a href="text-mining.html#glove"><i class="fa fa-check"></i><b>6.5</b> GloVe word embedding</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="current-analysis.html"><a href="current-analysis.html"><i class="fa fa-check"></i><b>7</b> Data analysis in the current era</a><ul>
<li class="chapter" data-level="7.1" data-path="current-analysis.html"><a href="current-analysis.html#preprocessing"><i class="fa fa-check"></i><b>7.1</b> Preprocessing</a><ul>
<li class="chapter" data-level="7.1.1" data-path="current-analysis.html"><a href="current-analysis.html#registration"><i class="fa fa-check"></i><b>7.1.1</b> Image registration</a></li>
<li class="chapter" data-level="7.1.2" data-path="current-analysis.html"><a href="current-analysis.html#calling"><i class="fa fa-check"></i><b>7.1.2</b> Spot and barcode calling</a></li>
<li class="chapter" data-level="7.1.3" data-path="current-analysis.html"><a href="current-analysis.html#segmentation"><i class="fa fa-check"></i><b>7.1.3</b> Cell segmentation</a></li>
<li class="chapter" data-level="7.1.4" data-path="current-analysis.html"><a href="current-analysis.html#seg-alternatives"><i class="fa fa-check"></i><b>7.1.4</b> Alternatives to cell segmentation</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="current-analysis.html"><a href="current-analysis.html#eda"><i class="fa fa-check"></i><b>7.2</b> Exploratory data analysis</a></li>
<li class="chapter" data-level="7.3" data-path="current-analysis.html"><a href="current-analysis.html#reconstruction"><i class="fa fa-check"></i><b>7.3</b> Spatial reconstruction of scRNA-seq data</a><ul>
<li class="chapter" data-level="7.3.1" data-path="current-analysis.html"><a href="current-analysis.html#direct-dr"><i class="fa fa-check"></i><b>7.3.1</b> Direct dimension reduction</a></li>
<li class="chapter" data-level="7.3.2" data-path="current-analysis.html"><a href="current-analysis.html#ad-hoc"><i class="fa fa-check"></i><b>7.3.2</b> <em>Ad hoc</em> scoring</a></li>
<li class="chapter" data-level="7.3.3" data-path="current-analysis.html"><a href="current-analysis.html#generative"><i class="fa fa-check"></i><b>7.3.3</b> Generative models</a></li>
<li class="chapter" data-level="7.3.4" data-path="current-analysis.html"><a href="current-analysis.html#latent"><i class="fa fa-check"></i><b>7.3.4</b> Shared latent space</a></li>
<li class="chapter" data-level="7.3.5" data-path="current-analysis.html"><a href="current-analysis.html#reconst-other"><i class="fa fa-check"></i><b>7.3.5</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="current-analysis.html"><a href="current-analysis.html#deconvolution"><i class="fa fa-check"></i><b>7.4</b> Cell type deconvolution</a></li>
<li class="chapter" data-level="7.5" data-path="current-analysis.html"><a href="current-analysis.html#variable"><i class="fa fa-check"></i><b>7.5</b> Spatially variable genes</a><ul>
<li class="chapter" data-level="7.5.1" data-path="current-analysis.html"><a href="current-analysis.html#gpr"><i class="fa fa-check"></i><b>7.5.1</b> Gaussian process regression</a></li>
<li class="chapter" data-level="7.5.2" data-path="current-analysis.html"><a href="current-analysis.html#laplacian"><i class="fa fa-check"></i><b>7.5.2</b> Laplacian score</a></li>
<li class="chapter" data-level="7.5.3" data-path="current-analysis.html"><a href="current-analysis.html#variable-other"><i class="fa fa-check"></i><b>7.5.3</b> Other principles</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="current-analysis.html"><a href="current-analysis.html#pattern"><i class="fa fa-check"></i><b>7.6</b> Gene patterns</a></li>
<li class="chapter" data-level="7.7" data-path="current-analysis.html"><a href="current-analysis.html#region"><i class="fa fa-check"></i><b>7.7</b> Spatial regions</a></li>
<li class="chapter" data-level="7.8" data-path="current-analysis.html"><a href="current-analysis.html#cell-interaction"><i class="fa fa-check"></i><b>7.8</b> Cell-cell interaction</a><ul>
<li class="chapter" data-level="7.8.1" data-path="current-analysis.html"><a href="current-analysis.html#l-r"><i class="fa fa-check"></i><b>7.8.1</b> Ligand-receptor pairs</a></li>
<li class="chapter" data-level="7.8.2" data-path="current-analysis.html"><a href="current-analysis.html#gene-cell-interact"><i class="fa fa-check"></i><b>7.8.2</b> Genes associated with cell-cell interaction</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="current-analysis.html"><a href="current-analysis.html#gene-interaction"><i class="fa fa-check"></i><b>7.9</b> Gene-gene interaction</a></li>
<li class="chapter" data-level="7.10" data-path="current-analysis.html"><a href="current-analysis.html#subcellular"><i class="fa fa-check"></i><b>7.10</b> Subcellular transcript localization</a></li>
<li class="chapter" data-level="7.11" data-path="current-analysis.html"><a href="current-analysis.html#HnE"><i class="fa fa-check"></i><b>7.11</b> Gene expression imputation from H&amp;E</a></li>
</ul></li>
<li class="part"><span><b>III Future perspectives</b></span></li>
<li class="chapter" data-level="8" data-path="future.html"><a href="future.html"><i class="fa fa-check"></i><b>8</b> From the past to the present to the future</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://bookdown.org" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Museum of Spatial Transcriptomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references">
<div>
<p>“10X VISIUM SPATIAL TRANSCRIPTOMICS.” n.d. Accessed October 7, 2020. <a href="https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics">https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics</a>.</p>
</div>
<div>
<p>Abdelaal, Tamim, Soufiane Mourragui, Ahmed Mahfouz, and Marcel J T Reinders. 2020. “SpaGE: Spatial Gene Enhancement using scRNA-seq.” <em>Nucleic Acids Research</em> 48 (18): e107–e107. <a href="https://doi.org/10.1093/nar/gkaa740">https://doi.org/10.1093/nar/gkaa740</a>.</p>
</div>
<div>
<p>Abed-Esfahani, Pegah, Benjamin C Darwin, Derek Howard, Nick Wang, Ethan Kim, Jason Lerch, and Leon French. 2021. “Evaluation of deep convolutional neural networks for in situ hybridization gene expression image representation.” <em>bioRxiv</em>, January, 2021.01.22.427860. <a href="https://doi.org/10.1101/2021.01.22.427860">https://doi.org/10.1101/2021.01.22.427860</a>.</p>
</div>
<div>
<p>Aboobaker, A. Aziz, Pavel Tomancak, Nipam Patel, Gerald M. Rubin, and Eric C. Lai. 2005. “Drosophila microRNAs exhibit diverse spatial expression patterns during embryonic development.” <em>Proceedings of the National Academy of Sciences</em> 102 (50): 18017–22. <a href="https://doi.org/10.1073/pnas.0508823102">https://doi.org/10.1073/pnas.0508823102</a>.</p>
</div>
<div>
<p>Achim, Kaia, Jean Baptiste Pettit, Luis R. Saraiva, Daria Gavriouchkina, Tomas Larsson, Detlev Arendt, and John C. Marioni. 2015. “High-throughput spatial mapping of single-cell RNA-seq data to tissue of origin.” <em>Nature Biotechnology</em> 33 (5): 503–9. <a href="https://doi.org/10.1038/nbt.3209">https://doi.org/10.1038/nbt.3209</a>.</p>
</div>
<div>
<p>Adkins, Ricky S, Andrew I Aldridge, Shona Allen, Seth A Ament, Xu An, Ethan Armand, Giorgio A Ascoli, et al. 2020. “A multimodal cell census and atlas of the mammalian primary motor cortex.” <em>bioRxiv</em>, January, 2020.10.19.343129. <a href="https://doi.org/10.1101/2020.10.19.343129">https://doi.org/10.1101/2020.10.19.343129</a>.</p>
</div>
<div>
<p>“ADVANCED GENOMICS CORE PRICING.” n.d. Accessed October 7, 2020. <a href="https://brcf.medicine.umich.edu/cores/advanced-genomics/price-list/">https://brcf.medicine.umich.edu/cores/advanced-genomics/price-list/</a>.</p>
</div>
<div>
<p>Agapite, Julie, Laurent-Philippe Albou, Suzi Aleksander, Joanna Argasinska, Valerio Arnaboldi, Helen Attrill, Susan M. Bello, et al. 2020. “Alliance of Genome Resources Portal: unified model organism research platform.” <em>Nucleic Acids Research</em> 48 (D1): D650–D658. <a href="https://doi.org/10.1093/nar/gkz813">https://doi.org/10.1093/nar/gkz813</a>.</p>
</div>
<div>
<p>Aguila, Julio, Shangli Cheng, Nigel Kee, Ming Cao, Qiaolin Deng, and Eva Hedlund. 2018. “Spatial transcriptomics and in silico random pooling identify novel markers of vulnerable and resistant midbrain dopamine neurons.” <em>bioRxiv</em>, October, 334417. <a href="https://doi.org/10.1101/334417">https://doi.org/10.1101/334417</a>.</p>
</div>
<div>
<p>Ahmed, Ayisha, Nicole J Ward, Simon Moxon, Sara Lopez-Gomollon, Camille Viaut, Matthew L Tomlinson, Ilya Patrushev, et al. 2015. “A Database of microRNA Expression Patterns in Xenopus laevis.” <em>PLOS ONE</em> 10 (10): e0138313. <a href="https://doi.org/10.1371/journal.pone.0138313">https://doi.org/10.1371/journal.pone.0138313</a>.</p>
</div>
<div>
<p>Allen, Nicholas D., David G. Cran, Sheila C. Barton, Simon Hettle, Wolf Reik, and M. Azim Surani. 1988. “Transgenes as probes for active chromosomal domains in mouse development.” <em>Nature</em> 333 (6176): 852–55. <a href="https://doi.org/10.1038/333852a0">https://doi.org/10.1038/333852a0</a>.</p>
</div>
<div>
<p>Alon, Shahar, Daniel R Goodwin, Anubhav Sinha, Asmamaw T Wassie, Fei Chen, Evan R Daugharthy, Yosuke Bando, et al. 2020. “Expansion Sequencing: Spatially Precise &lt;em&gt;In Situ&lt;/em&gt; Transcriptomics in Intact Biological Systems.” <em>bioRxiv</em>, January, 2020.05.13.094268. <a href="https://doi.org/10.1101/2020.05.13.094268">https://doi.org/10.1101/2020.05.13.094268</a>.</p>
</div>
<div>
<p>Alonso, A M, A Carrea, and L Diambra. 2020. “Prediction of cell position using single-cell transcriptomic data: an iterative procedure [version 2; peer review: 2 approved].” <em>F1000Research</em> 8 (1775). <a href="https://doi.org/10.12688/f1000research.20715.2">https://doi.org/10.12688/f1000research.20715.2</a>.</p>
</div>
<div>
<p>Alsup, William H. 2009. “Applera v. Illumina.” <a href="https://www.leagle.com/decision/infco20100325236">https://www.leagle.com/decision/infco20100325236</a>.</p>
</div>
<div>
<p>“A Mouse for All Reasons.” 2007. <em>Cell</em> 128 (1): 9–13. <a href="https://doi.org/10.1016/j.cell.2006.12.018">https://doi.org/10.1016/j.cell.2006.12.018</a>.</p>
</div>
<div>
<p>An, Li, Hongbo Xie, Mark H Chin, Zoran Obradovic, Desmond J Smith, and Vasileios Megalooikonomou. 2009. “Analysis of multiplex gene expression maps obtained by voxelation.” <em>BMC Bioinformatics</em> 10 (S4): S10. <a href="https://doi.org/10.1186/1471-2105-10-S4-S10">https://doi.org/10.1186/1471-2105-10-S4-S10</a>.</p>
</div>
<div>
<p>Andersson, Alma, Joseph Bergenstråhle, Michaela Asp, Ludvig Bergenstråhle, Aleksandra Jurek, José Fernández Navarro, and Joakim Lundeberg. 2019. “Spatial mapping of cell types by integration of transcriptomics data.” <em>bioRxiv</em>, December, 2019.12.13.874495. <a href="https://doi.org/10.1101/2019.12.13.874495">https://doi.org/10.1101/2019.12.13.874495</a>.</p>
</div>
<div>
<p>Andersson, Axel, Ferran Diego, Fred A Hamprecht, and Carolina Wählby. 2021. “ISTDECO: In Situ Transcriptomics Decoding by Deconvolution.” <em>bioRxiv</em>, January, 2021.03.01.433040. <a href="https://doi.org/10.1101/2021.03.01.433040">https://doi.org/10.1101/2021.03.01.433040</a>.</p>
</div>
<div>
<p>Andonian, Alexander, Daniel Paseltiner, Travis J. Gould, and Jason B. Castro. 2019. “A deep learning based method for large-scale classification, registration, and clustering of in-situ hybridization experiments in the mouse olfactory bulb.” <em>Journal of Neuroscience Methods</em> 312 (January): 162–68. <a href="https://doi.org/10.1016/j.jneumeth.2018.12.003">https://doi.org/10.1016/j.jneumeth.2018.12.003</a>.</p>
</div>
<div>
<p>“Arcturus XT Laser Capture Microdissection (LCM) Instrument - US.” n.d. <a href="//www.thermofisher.com/us/en/home/life-science/gene-expression-analysis-genotyping/laser-capture-microdissection/arcturus-laser-capture-microdissection-lcm-instrument.html">//www.thermofisher.com/us/en/home/life-science/gene-expression-analysis-genotyping/laser-capture-microdissection/arcturus-laser-capture-microdissection-lcm-instrument.html</a>.</p>
</div>
<div>
<p>Ardini-Poleske, Maryanne E., Robert F. Clark, Charles Ansong, James P. Carson, Richard A. Corley, Gail H. Deutsch, James S. Hagood, et al. 2017. “LungMAP: The Molecular Atlas of Lung Development Program.” <em>American Journal of Physiology-Lung Cellular and Molecular Physiology</em> 313 (5): L733–L740. <a href="https://doi.org/10.1152/ajplung.00139.2017">https://doi.org/10.1152/ajplung.00139.2017</a>.</p>
</div>
<div>
<p>Arnol, Damien, Denis Schapiro, Bernd Bodenmiller, Julio Saez-Rodriguez, and Oliver Stegle. 2019. “Modeling Cell-Cell Interactions from Spatial Molecular Data with Spatial Variance Component Analysis.” <em>Cell Reports</em> 29 (1): 202–211.e6. <a href="https://doi.org/10.1016/j.celrep.2019.08.077">https://doi.org/10.1016/j.celrep.2019.08.077</a>.</p>
</div>
<div>
<p>Asp, Michaela, Stefania Giacomello, Ludvig Larsson, Chenglin Wu, Daniel Fürth, Xiaoyan Qian, Eva Wärdell, et al. 2019. “A Spatiotemporal Organ-Wide Gene Expression and Cell Atlas of the Developing Human Heart.” <em>Cell</em> 179 (7): 1647–1660.e19. <a href="https://doi.org/10.1016/j.cell.2019.11.025">https://doi.org/10.1016/j.cell.2019.11.025</a>.</p>
</div>
<div>
<p>Asp, Michaela, Fredrik Salmén, Patrik L Ståhl, Sanja Vickovic, Ulrika Felldin, Marie Löfling, José Fernandez Navarro, et al. 2017. “Spatial detection of fetal marker genes expressed at low level in adult human heart tissue.” <em>Scientific Reports</em> 7 (1): 12941. <a href="https://doi.org/10.1038/s41598-017-13462-5">https://doi.org/10.1038/s41598-017-13462-5</a>.</p>
</div>
<div>
<p>Axelrod, Shannon, Ambrose J Carr, Jeremy Freeman, Deep Ganguli, Brian Long, Tony Tung, and Others. n.d. “{Starfish}: Open Source Image Based Transcriptomics and Proteomics Tools.” <a href="http://github.com/spacetx/starfish">http://github.com/spacetx/starfish</a>.</p>
</div>
<div>
<p>Baccin, Chiara, Jude Al-Sabah, Lars Velten, Patrick M. Helbling, Florian Grünschläger, Pablo Hernández-Malmierca, César Nombela-Arrieta, Lars M. Steinmetz, Andreas Trumpp, and Simon Haas. 2020. “Combined single-cell and spatial transcriptomics reveal the molecular, cellular and spatial bone marrow niche organization.” <em>Nature Cell Biology</em> 22 (1): 38–48. <a href="https://doi.org/10.1038/s41556-019-0439-6">https://doi.org/10.1038/s41556-019-0439-6</a>.</p>
</div>
<div>
<p>Baharlou, Heeva, Nicolas P. Canete, Anthony L. Cunningham, Andrew N. Harman, and Ellis Patrick. 2019. “Mass Cytometry Imaging for the Study of Human Diseases—Applications and Data Analysis Strategies.” <em>Frontiers in Immunology</em> 10 (November): 2657. <a href="https://doi.org/10.3389/fimmu.2019.02657">https://doi.org/10.3389/fimmu.2019.02657</a>.</p>
</div>
<div>
<p>Bakken, Trygve E., Jeremy A. Miller, Song Lin Ding, Susan M. Sunkin, Kimberly A. Smith, Lydia Ng, Aaron Szafer, et al. 2016. “A comprehensive transcriptional map of primate brain development.” <em>Nature</em> 535 (7612): 367–75. <a href="https://doi.org/10.1038/nature18637">https://doi.org/10.1038/nature18637</a>.</p>
</div>
<div>
<p>Baldock, Richard, Jonathan Bard, Matt Kaufman, and Duncan Davidson. 1992. “What’s New? A real mouse for your computer.” <em>BioEssays</em> 14 (7): 501–2. <a href="https://doi.org/10.1002/bies.950140713">https://doi.org/10.1002/bies.950140713</a>.</p>
</div>
<div>
<p>Baner, J., Mats Nilsson, Maritha Mendel-Hartvig, and Uif Landegren. 1998. “Signal amplification of padlock probes by rolling circle replication.” <em>Nucleic Acids Research</em> 26 (22): 5073–8. <a href="https://doi.org/10.1093/nar/26.22.5073">https://doi.org/10.1093/nar/26.22.5073</a>.</p>
</div>
<div>
<p>Bassell, G J, C M Powers, K L Taneja, and R H Singer. 1994. “Single mRNAs visualized by ultrastructural in situ hybridization are principally localized at actin filament intersections in fibroblasts.” <em>The Journal of Cell Biology</em> 126 (4): 863–76. <a href="https://doi.org/10.1083/jcb.126.4.863">https://doi.org/10.1083/jcb.126.4.863</a>.</p>
</div>
<div>
<p>Bates, Mark, Graham T. Dempsey, Kok Hao Chen, and Xiaowei Zhuang. 2012. “Multicolor Super-Resolution Fluorescence Imaging via Multi-Parameter Fluorophore Detection.” <em>ChemPhysChem</em> 13 (1): 99–107. <a href="https://doi.org/10.1002/cphc.201100735">https://doi.org/10.1002/cphc.201100735</a>.</p>
</div>
<div>
<p>Battich, Nico, Thomas Stoeger, and Lucas Pelkmans. 2013. “Image-based transcriptomics in thousands of single human cells at single-molecule resolution.” <em>Nature Methods</em> 10 (11): 1127–36. <a href="https://doi.org/10.1038/nmeth.2657">https://doi.org/10.1038/nmeth.2657</a>.</p>
</div>
<div>
<p>Bayraktar, Omer Ali, Theresa Bartels, Staffan Holmqvist, Vitalii Kleshchevnikov, Araks Martirosyan, Damon Polioudakis, Lucile Ben Haim, et al. 2020. “Astrocyte layers in the mammalian cerebral cortex revealed by a single-cell in situ transcriptomic map.” <em>Nature Neuroscience</em> 23 (4): 500–509. <a href="https://doi.org/10.1038/s41593-020-0602-1">https://doi.org/10.1038/s41593-020-0602-1</a>.</p>
</div>
<div>
<p>Becker, Ingrid, Karl Friedrich Becker, Michael H. Röhrl, Günter Minkus, Karin Schütze, and Heinz Höfler. 1996. “Single-cell mutation analysis of tumors from stained histologic slides.” <em>Laboratory Investigation</em> 75 (6): 801–7. <a href="https://pubmed.ncbi.nlm.nih.gov/8973475/">https://pubmed.ncbi.nlm.nih.gov/8973475/</a>.</p>
</div>
<div>
<p>Bell, George W., Tatiana A. Yatskievych, and Parker B. Antin. 2004. “GEISHA, a whole-mount in situ hybridization gene expression screen in chicken embryos.” <em>Developmental Dynamics</em> 229 (3): 677–87. <a href="https://doi.org/10.1002/dvdy.10503">https://doi.org/10.1002/dvdy.10503</a>.</p>
</div>
<div>
<p>Bellen, H. J., C. J. O’Kane, C. Wilson, U. Grossniklaus, R. K. Pearson, and W. J. Gehring. 1989. “P-element-mediated enhancer detection: a versatile method to study development in Drosophila.” <em>Genes &amp; Development</em> 3 (9): 1288–1300. <a href="https://doi.org/10.1101/gad.3.9.1288">https://doi.org/10.1101/gad.3.9.1288</a>.</p>
</div>
<div>
<p>Belmamoune, Mounia, and Fons J. Verbeek. 2008. “Data Integration for Spatio-Temporal Patterns of Gene Expression of Zebrafish development: the GEMS database.” <em>Journal of Integrative Bioinformatics</em> 5 (2): 35–48. <a href="https://doi.org/10.1515/jib-2008-92">https://doi.org/10.1515/jib-2008-92</a>.</p>
</div>
<div>
<p>Belyavsky, A, T Vinogradova, and K Rajewsky. 1989. “PCR-based cDNA library construction: general cDNA libraries at the level of a few cells.” <em>Nucleic Acids Research</em> 17 (8): 2919–32. <a href="https://doi.org/10.1093/nar/17.8.2919">https://doi.org/10.1093/nar/17.8.2919</a>.</p>
</div>
<div>
<p>Bergenstråhle, Joseph, Ludvig Bergenstråhle, and Joakim Lundeberg. 2020. “SpatialCPie: an R/Bioconductor package for spatial transcriptomics cluster evaluation.” <em>BMC Bioinformatics</em> 21 (1): 161. <a href="https://doi.org/10.1186/s12859-020-3489-7">https://doi.org/10.1186/s12859-020-3489-7</a>.</p>
</div>
<div>
<p>Bergenstråhle, Ludvig, Bryan He, Joseph Bergenstråhle, Alma Andersson, Joakim Lundeberg, James Zou, and Jonas Maaskola. 2020. “Super-resolved spatial transcriptomics by deep data fusion.” <em>bioRxiv</em>, March, 2020.02.28.963413. <a href="https://doi.org/10.1101/2020.02.28.963413">https://doi.org/10.1101/2020.02.28.963413</a>.</p>
</div>
<div>
<p>Berglund, Emelie, Jonas Maaskola, Niklas Schultz, Stefanie Friedrich, Maja Marklund, Joseph Bergenstråhle, Firas Tarish, et al. 2018. “Spatial maps of prostate cancer transcriptomes reveal an unexplored landscape of heterogeneity.” <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-018-04724-5">https://doi.org/10.1038/s41467-018-04724-5</a>.</p>
</div>
<div>
<p>Berns, Michael W., Roberts S. Olson, and Donald E. Rounds. 1969. “In vitro Production of Chromosomal Lesions with an Argon Laser Microbeam.” <em>Nature</em> 221 (5175): 74–75. <a href="https://doi.org/10.1038/221074a0">https://doi.org/10.1038/221074a0</a>.</p>
</div>
<div>
<p>Bettenhausen, Berthold, and Achim Gossler. 1995. “Efficient Isolation of Novel Mouse Genes Differentially Expressed in Early Postimplantation Embryos.” <em>Genomics</em> 28 (3): 436–41. <a href="https://doi.org/10.1006/geno.1995.1172">https://doi.org/10.1006/geno.1995.1172</a>.</p>
</div>
<div>
<p>Biancalani, Tommaso, Gabriele Scalia, Lorenzo Buffoni, Raghav Avasthi, Ziqing Lu, Aman Sanger, Neriman Tokcan, et al. 2020. “Deep learning and alignment of spatially-resolved whole transcriptomes of single cells in the mouse brain with Tangram.” <em>bioRxiv</em>, January, 2020.08.29.272831. <a href="https://doi.org/10.1101/2020.08.29.272831">https://doi.org/10.1101/2020.08.29.272831</a>.</p>
</div>
<div>
<p>Bidarimath, Mallikarjun, Andrew K. Edwards, and Chandrakant Tayade. 2015. “Laser Capture Microdissection for Gene Expression Analysis.” In <em>Methods in Molecular Biology</em>, 1219:115–37. Humana Press Inc. <a href="https://doi.org/10.1007/978-1-4939-1661-0_10">https://doi.org/10.1007/978-1-4939-1661-0_10</a>.</p>
</div>
<div>
<p>Bier, E., H. Vaessin, S. Shepherd, K. Lee, K. McCall, S. Barbel, L. Ackerman, R. Carretto, T. Uemura, and E. Grell. 1989. “Searching for pattern and mutation in the Drosophila genome with a P-lacZ vector.” <em>Genes &amp; Development</em> 3 (9): 1273–87. <a href="https://doi.org/10.1101/gad.3.9.1273">https://doi.org/10.1101/gad.3.9.1273</a>.</p>
</div>
<div>
<p>BinTayyash, Nuha, Sokratia Georgaka, S T John, Sumon Ahmed, Alexis Boukouvalas, James Hensman, and Magnus Rattray. 2020. “Non-parametric modelling of temporal and spatial counts data from RNA-seq experiments.” <em>bioRxiv</em>, January, 2020.07.29.227207. <a href="https://doi.org/10.1101/2020.07.29.227207">https://doi.org/10.1101/2020.07.29.227207</a>.</p>
</div>
<div>
<p>Birchall, Philip S, Rita M Fishpool, and Donna G Albertson. 1995. “Expression patterns of predicted genes from the C. elegans genome sequence visualized by FISH in whole organisms,” 314–20. <a href="https://doi.org/10.1038/ng1195-314">https://doi.org/10.1038/ng1195-314</a>.</p>
</div>
<div>
<p>Blackshaw, Seth, Sanjiv Harpavat, Jeff Trimarchi, Li Cai, Haiyan Huang, Winston P Kuo, Griffin Weber, et al. 2004. “Genomic Analysis of Mouse Retinal Development.” Edited by William Harris. <em>PLoS Biology</em> 2 (9): e247. <a href="https://doi.org/10.1371/journal.pbio.0020247">https://doi.org/10.1371/journal.pbio.0020247</a>.</p>
</div>
<div>
<p>Blitz, Ira L., Kitt D. Paraiso, Ilya Patrushev, William T. Y. Chiu, Ken W. Y. Cho, and Michael J. Gilchrist. 2017. “A catalog of Xenopus tropicalis transcription factors and their regional expression in the early gastrula stage embryo.” <em>Developmental Biology</em> 426 (2): 409–17. <a href="https://doi.org/10.1016/j.ydbio.2016.07.002">https://doi.org/10.1016/j.ydbio.2016.07.002</a>.</p>
</div>
<div>
<p>Boer, B. A. de, Jan M. Ruijter, F. P. J. M. Voorbraak, and A. F. M. Moorman. 2009. “More than a decade of developmental gene expression atlases: where are we now?” <em>Nucleic Acids Research</em> 37 (22): 7349–59. <a href="https://doi.org/10.1093/nar/gkp819">https://doi.org/10.1093/nar/gkp819</a>.</p>
</div>
<div>
<p>Bohland, Jason W., Hemant Bokil, Sayan D. Pathak, Chang-Kyu Lee, Lydia Ng, Christopher Lau, Chihchau Kuan, Michael Hawrylycz, and Partha P. Mitra. 2010. “Clustering of spatial gene expression patterns in the mouse brain and comparison with classical neuroanatomy.” <em>Methods</em> 50 (2): 105–12. <a href="https://doi.org/10.1016/j.ymeth.2009.09.001">https://doi.org/10.1016/j.ymeth.2009.09.001</a>.</p>
</div>
<div>
<p>Boulgakov, Alexander A, Andrew D Ellington, and Edward M Marcotte. 2020. “Bringing Microscopy-By-Sequencing into View.” <em>Trends in Biotechnology</em> 38 (2): 154–62. <a href="https://doi.org/https://doi.org/10.1016/j.tibtech.2019.06.001">https://doi.org/https://doi.org/10.1016/j.tibtech.2019.06.001</a>.</p>
</div>
<div>
<p>Bowes, Jeff B., Kevin A. Snyder, Erik Segerdell, Chris J. Jarabek, Kenan Azam, Aaron M. Zorn, and Peter D. Vize. 2009. “Xenbase: Gene expression and improved integration.” <em>Nucleic Acids Research</em> 38 (SUPPL.1): D607–D612. <a href="https://doi.org/10.1093/nar/gkp953">https://doi.org/10.1093/nar/gkp953</a>.</p>
</div>
<div>
<p>Brady, Gerard, Mary Barbara, and Norman N. Iscove. 1990. “Representative in Vitro cDNA Amplification From Individual Hemopoietic Cells and Colonies.” <em>Methods in Molecular and Cellular Biology</em> 2: 17–25.</p>
</div>
<div>
<p>Brady, Lauren, Michelle Kriner, Ilsa Coleman, Colm Morrissey, Martine Roudier, Lawrence D True, Roman Gulati, et al. 2021. “Inter- and intra-tumor heterogeneity of metastatic prostate cancer determined by digital spatial gene expression profiling.” <em>Nature Communications</em> 12 (1): 1426. <a href="https://doi.org/10.1038/s41467-021-21615-4">https://doi.org/10.1038/s41467-021-21615-4</a>.</p>
</div>
<div>
<p>Bravo González-Blas, Carmen, Xiao-Jiang Quan, Ramon Duran-Romaña, Ibrahim Ihsan Taskiran, Duygu Koldere, Kristofer Davie, Valerie Christiaens, et al. 2020. “Identification of genomic enhancers through spatial integration of single-cell transcriptomics and epigenomics.” <em>Molecular Systems Biology</em> 16 (5): e9438. <a href="https://doi.org/10.15252/msb.20209438">https://doi.org/10.15252/msb.20209438</a>.</p>
</div>
<div>
<p>Brink, Susanne C. van den, Anna Alemany, Vincent van Batenburg, Naomi Moris, Marloes Blotenburg, Judith Vivié, Peter Baillie-Johnson, et al. 2020. “Single-cell and spatial transcriptomics reveal somitogenesis in gastruloids.” <em>Nature</em> 582 (7812): 405. <a href="https://doi.org/10.1038/s41586-020-2024-3">https://doi.org/10.1038/s41586-020-2024-3</a>.</p>
</div>
<div>
<p>Brown, Vanessa M. 2002. “High-Throughput Imaging of Brain Gene Expression.” <em>Genome Research</em> 12 (2): 244–54. <a href="https://doi.org/10.1101/gr.204102">https://doi.org/10.1101/gr.204102</a>.</p>
</div>
<div>
<p>Brown, V. M., A. Ossadtchi, A. H. Khan, S. Yee, G. Lacan, W. P. Melega, S. R. Cherry, R. M. Leahy, and D. J. Smith. 2002. “Multiplex Three-Dimensional Brain Gene Expression Mapping in a Mouse Model of Parkinson’s Disease.” <em>Genome Research</em> 12 (6): 868–84. <a href="https://doi.org/10.1101/gr.229002">https://doi.org/10.1101/gr.229002</a>.</p>
</div>
<div>
<p>Buchberger, Amanda Rae, Kellen DeLaney, Jillian Johnson, and Lingjun Li. 2018. “Mass Spectrometry Imaging: A Review of Emerging Advancements and Future Insights.” <em>Analytical Chemistry</em> 90 (1): 240–65. <a href="https://doi.org/10.1021/acs.analchem.7b04733">https://doi.org/10.1021/acs.analchem.7b04733</a>.</p>
</div>
<div>
<p>Burgess, Darren J. 2019. “Spatial transcriptomics coming of age.” <em>Nature Reviews Genetics</em> 20 (6): 317–17. <a href="https://doi.org/10.1038/s41576-019-0129-z">https://doi.org/10.1038/s41576-019-0129-z</a>.</p>
</div>
<div>
<p>Burkhard, Silja Barbara, and Jeroen Bakkers. 2018. “Spatially resolved RNA-sequencing of the embryonic heart identifies a role for Wnt/<span class="math inline">\(\beta\)</span>-catenin signaling in autonomic control of heart rate.” <em>eLife</em> 7 (February). <a href="https://doi.org/10.7554/eLife.31515">https://doi.org/10.7554/eLife.31515</a>.</p>
</div>
<div>
<p>Bustin, Stephen, Harvinder S Dhillon, Sara Kirvell, Christina Greenwood, Michael Parker, Gregory L Shipley, and Tania Nolan. 2015. “Variability of the Reverse Transcription Step: Practical Implications.” <em>Clinical Chemistry</em> 61 (1): 202–12. <a href="https://doi.org/10.1373/clinchem.2014.230615">https://doi.org/10.1373/clinchem.2014.230615</a>.</p>
</div>
<div>
<p>Butler, Daniel, Christopher Mozsary, Cem Meydan, Jonathan Foox, Joel Rosiene, Alon Shaiber, David Danko, et al. 2021. “Shotgun transcriptome, spatial omics, and isothermal profiling of SARS-CoV-2 infection reveals unique host responses, viral diversification, and drug interactions.” <em>Nature Communications</em> 12 (1): 1660. <a href="https://doi.org/10.1038/s41467-021-21361-7">https://doi.org/10.1038/s41467-021-21361-7</a>.</p>
</div>
<div>
<p>Cabili, Moran N., Margaret C. Dunagin, Patrick D. McClanahan, Andrew Biaesch, Olivia Padovan-Merhar, Aviv Regev, John L. Rinn, and Arjun Raj. 2015. “Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution.” <em>Genome Biology</em> 16 (1): 20. <a href="https://doi.org/10.1186/s13059-015-0586-4">https://doi.org/10.1186/s13059-015-0586-4</a>.</p>
</div>
<div>
<p>Cable, Dylan M., Evan Murray, Luli S. Zou, Aleksandrina Goeva, Evan Z. Macosko, Fei Chen, and Rafael A. Irizarry. 2020. “Robust decomposition of cell type mixtures in spatial transcriptomics.” <em>bioRxiv</em>, May, 2020.05.07.082750. <a href="https://doi.org/10.1101/2020.05.07.082750">https://doi.org/10.1101/2020.05.07.082750</a>.</p>
</div>
<div>
<p>Campiteli, Monica Guimarães, Cesar Henrique Comin, Luciano Da Fontoura Costa, M. Madan Babu, and Roberto Marcondes Cesar. 2013. “A methodology to infer gene networks from spatial patterns of expression – an application to fluorescence in situ hybridization images.” <em>Molecular BioSystems</em> 9 (7): 1926. <a href="https://doi.org/10.1039/c3mb25475e">https://doi.org/10.1039/c3mb25475e</a>.</p>
</div>
<div>
<p>Canete, Nicolas, and Ellis Patrick. 2020. <em>spicyR: Spatial analysis of in situ cytometry data</em>. <a href="https://doi.org/10.18129/B9.bioc.spicyR">https://doi.org/10.18129/B9.bioc.spicyR</a>.</p>
</div>
<div>
<p>Cang, Zixuan, and Qing Nie. 2020. “Inferring spatial and signaling relationships between cells from single cell transcriptomic data.” <em>Nature Communications</em> 11 (1): 2084. <a href="https://doi.org/10.1038/s41467-020-15968-5">https://doi.org/10.1038/s41467-020-15968-5</a>.</p>
</div>
<div>
<p>Cankaya, Murat, Ana Hernandez, Mehmet Ciftci, Sukru Beydemir, Hasan Ozdemir, Harun Budak, Ilhami Gulcin, et al. 2007. “An analysis of expression patterns of genes encoding proteins with catalytic activities.” <em>BMC Genomics</em> 8 (1): 232. <a href="https://doi.org/10.1186/1471-2164-8-232">https://doi.org/10.1186/1471-2164-8-232</a>.</p>
</div>
<div>
<p>Cao, Junyue, Malte Spielmann, Xiaojie Qiu, Xingfan Huang, Daniel M Ibrahim, Andrew J Hill, Fan Zhang, et al. 2019. “The single-cell transcriptional landscape of mammalian organogenesis.” <em>Nature</em> 566 (7745): 496–502. <a href="https://doi.org/10.1038/s41586-019-0969-x">https://doi.org/10.1038/s41586-019-0969-x</a>.</p>
</div>
<div>
<p>Cao, Zhaohui, Wenlong Zuo, Lanxiang Wang, Junyu Chen, Zepeng Qu, Fan Jin, and Lei Dai. 2021. “Spatial profiling of microbial communities by sequential FISH with error-robust encoding.” <em>bioRxiv</em>, January, 2021.05.27.445923. <a href="https://doi.org/10.1101/2021.05.27.445923">https://doi.org/10.1101/2021.05.27.445923</a>.</p>
</div>
<div>
<p>Carlberg, Konstantin, Marina Korotkova, Ludvig Larsson, Anca I Catrina, Patrik L Ståhl, and Vivianne Malmström. 2019. “Exploring inflammatory signatures in arthritic joint biopsies with Spatial Transcriptomics.” <em>Scientific Reports</em> 9 (1): 18975. <a href="https://doi.org/10.1038/s41598-019-55441-y">https://doi.org/10.1038/s41598-019-55441-y</a>.</p>
</div>
<div>
<p>Carson, James P., Christina Thaller, and Gregor Eichele. 2002. “A transcriptome atlas of the mouse brain at cellular resolution.” <em>Current Opinion in Neurobiology</em> 12 (5): 562–65. <a href="https://doi.org/10.1016/S0959-4388(02)00356-2">https://doi.org/10.1016/S0959-4388(02)00356-2</a>.</p>
</div>
<div>
<p>Carter, Mark G. 2003. “In Situ-Synthesized Novel Microarray Optimized for Mouse Stem Cell and Early Developmental Expression Profiling.” <em>Genome Research</em> 13 (5): 1011–21. <a href="https://doi.org/10.1101/gr.878903">https://doi.org/10.1101/gr.878903</a>.</p>
</div>
<div>
<p>Chen, Ao, Sha Liao, Mengnan Cheng, Kailong Ma, Liang Wu, Yiwei Lai, Jin Yang, et al. 2021. “Large field of view-spatially resolved transcriptomics at nanoscale resolution.” <em>bioRxiv</em>, January, 2021.01.17.427004. <a href="https://doi.org/10.1101/2021.01.17.427004">https://doi.org/10.1101/2021.01.17.427004</a>.</p>
</div>
<div>
<p>Chen, Fei, Paul W. Tillberg, and Edward S. Boyden. 2015. “Expansion microscopy.” <em>Science</em> 347 (6221): 543–48. <a href="https://doi.org/10.1126/science.1260088">https://doi.org/10.1126/science.1260088</a>.</p>
</div>
<div>
<p>Chen, Fei, Asmamaw T. Wassie, Allison J. Cote, Anubhav Sinha, Shahar Alon, Shoh Asano, Evan R. Daugharthy, et al. 2016. “Nanoscale imaging of RNA with expansion microscopy.” <em>Nature Methods</em> 13 (8): 679–84. <a href="https://doi.org/10.1038/nmeth.3899">https://doi.org/10.1038/nmeth.3899</a>.</p>
</div>
<div>
<p>Chen, Kok Hao, Alistair N. Boettiger, Jeffrey R. Moffitt, Siyuan Wang, and Xiaowei Zhuang. 2015. “Spatially resolved, highly multiplexed RNA profiling in single cells.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaa6090">https://doi.org/10.1126/science.aaa6090</a>.</p>
</div>
<div>
<p>Chen, Shuonan, Jackson Loper, Xiaoyin Chen, Alex Vaughan, Anthony M Zador, and Liam Paninski. 2021. “BARcode DEmixing through Non-negative Spatial Regression (BarDensr).” <em>PLOS Computational Biology</em> 17 (3): e1008256. <a href="https://doi.org/10.1371/journal.pcbi.1008256">https://doi.org/10.1371/journal.pcbi.1008256</a>.</p>
</div>
<div>
<p>Chen, Wei-Ting, Ashley Lu, Katleen Craessaerts, Benjamin Pavie, Carlo Sala Frigerio, Nikky Corthout, Xiaoyan Qian, et al. 2020. “Spatial Transcriptomics and In Situ Sequencing to Study Alzheimer’s Disease.” <em>Cell</em> 0 (0). <a href="https://doi.org/10.1016/j.cell.2020.06.038">https://doi.org/10.1016/j.cell.2020.06.038</a>.</p>
</div>
<div>
<p>Chen, Xiaoyin, Yu-Chi Sun, Huiqing Zhan, Justus M Kebschull, Stephan Fischer, Katherine Matho, Z Josh Huang, Jesse Gillis, and Anthony M Zador. 2019. “High-Throughput Mapping of Long-Range Neuronal Projection Using &lt;em&gt;In Situ&lt;/em&gt; Sequencing.” <em>Cell</em> 179 (3): 772–786.e19. <a href="https://doi.org/10.1016/j.cell.2019.09.023">https://doi.org/10.1016/j.cell.2019.09.023</a>.</p>
</div>
<div>
<p>Chin, Mark H., Alex B. Geng, Arshad H. Khan, Wei-Jun Qian, Vladislav A. Petyuk, Jyl Boline, Shawn Levy, et al. 2007. “A genome-scale map of expression for a mouse brain section obtained using voxelation.” <em>Physiological Genomics</em> 30 (3): 313–21. <a href="https://doi.org/10.1152/physiolgenomics.00287.2006">https://doi.org/10.1152/physiolgenomics.00287.2006</a>.</p>
</div>
<div>
<p>Cho, Chun-Seok, Jingyue Xi, Sung-Rye Park, Jer-En Hsu, Myungjin Kim, Goo Jun, Hyun-Min Kang, and Jun Hee Lee. 2021. “Seq-Scope: Submicrometer-resolution spatial transcriptomics for single cell and subcellular studies.” <em>bioRxiv</em>, January, 2021.01.25.427807. <a href="https://doi.org/10.1101/2021.01.25.427807">https://doi.org/10.1101/2021.01.25.427807</a>.</p>
</div>
<div>
<p>Ciolli Mattioli, Camilla, Aviv Rom, Vedran Franke, Koshi Imami, Gerard Arrey, Mandy Terne, Andrew Woehler, Altuna Akalin, Igor Ulitsky, and Marina Chekulaeva. 2019. “Alternative 3′ UTRs direct localization of functionally diverse protein isoforms in neuronal compartments.” <em>Nucleic Acids Research</em> 47 (5): 2560–73. <a href="https://doi.org/10.1093/nar/gky1270">https://doi.org/10.1093/nar/gky1270</a>.</p>
</div>
<div>
<p>Clarkson, Melissa D. 2016. “Representation of anatomy in online atlases and databases: a survey and collection of patterns for interface design.” <em>BMC Developmental Biology</em> 16 (1): 18. <a href="https://doi.org/10.1186/s12861-016-0116-y">https://doi.org/10.1186/s12861-016-0116-y</a>.</p>
</div>
<div>
<p>Cleary, Brian, Brooke Simonton, Jon Bezney, Evan Murray, Shahul Alam, Anubhav Sinha, Ehsan Habibi, et al. 2021. “Compressed sensing for highly efficient imaging transcriptomics.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/s41587-021-00883-x">https://doi.org/10.1038/s41587-021-00883-x</a>.</p>
</div>
<div>
<p>Codeluppi, Simone, Lars E. Borm, Amit Zeisel, Gioele La Manno, Josina A. van Lunteren, Camilla I. Svensson, and Sten Linnarsson. 2018. “Spatial organization of the somatosensory cortex revealed by osmFISH.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/s41592-018-0175-z">https://doi.org/10.1038/s41592-018-0175-z</a>.</p>
</div>
<div>
<p>Combs, Peter A., and Michael B. Eisen. 2013. “Sequencing mRNA from Cryo-Sliced Drosophila Embryos to Determine Genome-Wide Spatial Patterns of Gene Expression.” Edited by Barbara Jennings. <em>PLoS ONE</em> 8 (8): e71820. <a href="https://doi.org/10.1371/journal.pone.0071820">https://doi.org/10.1371/journal.pone.0071820</a>.</p>
</div>
<div>
<p>Combs, Peter A., and Hunter B. Fraser. 2018. “Spatially varying cis-regulatory divergence in Drosophila embryos elucidates cis-regulatory logic.” Edited by Claude Desplan. <em>PLOS Genetics</em> 14 (11): e1007631. <a href="https://doi.org/10.1371/journal.pgen.1007631">https://doi.org/10.1371/journal.pgen.1007631</a>.</p>
</div>
<div>
<p>Coskun, Ahmet F., and Long Cai. 2016. “Dense transcript profiling in single cells by image correlation decoding.” <em>Nature Methods</em> 13 (8): 657–60. <a href="https://doi.org/10.1038/nmeth.3895">https://doi.org/10.1038/nmeth.3895</a>.</p>
</div>
<div>
<p>Coudry, Renata A., Sibele I. Meireles, Radka Stoyanova, Harry S. Cooper, Alan Carpino, Xianqun Wang, Paul F. Engstrom, and Margie L. Clapper. 2007. “Successful Application of Microarray Technology to Microdissected Formalin-Fixed, Paraffin-Embedded Tissue.” <em>The Journal of Molecular Diagnostics</em> 9 (1): 70–79. <a href="https://doi.org/10.2353/jmoldx.2007.060004">https://doi.org/10.2353/jmoldx.2007.060004</a>.</p>
</div>
<div>
<p>Crosetto, Nicola, Magda Bienko, and Alexander Van Oudenaarden. 2015. “Spatially resolved transcriptomics and beyond.” <a href="https://doi.org/10.1038/nrg3832">https://doi.org/10.1038/nrg3832</a>.</p>
</div>
<div>
<p>Cunnea, Paula, Jill McMahon, Enda O’Connell, Kaveh Mashayekhi, Una Fitzgerald, and Stephen McQuaid. 2010. “Gene expression analysis of the microvascular compartment in multiple sclerosis using laser microdissected blood vessels.” <em>Acta Neuropathologica</em> 119 (5): 601–15. <a href="https://doi.org/10.1007/s00401-009-0618-9">https://doi.org/10.1007/s00401-009-0618-9</a>.</p>
</div>
<div>
<p>“Dana-Farber Core Facilities.” n.d. Accessed October 14, 2020. <a href="https://www.dana-farber.org/research/core-facilities/">https://www.dana-farber.org/research/core-facilities/</a>.</p>
</div>
<div>
<p>Dar, Daniel, Nina Dar, Long Cai, and Dianne K Newman. 2021. “In situ single-cell activities of microbial populations revealed by spatial transcriptomics.” <em>bioRxiv</em>, January, 2021.02.24.432792. <a href="https://doi.org/10.1101/2021.02.24.432792">https://doi.org/10.1101/2021.02.24.432792</a>.</p>
</div>
<div>
<p>Darnell, Diana K., Simran Kaur, Stacey Stanislaw, Jay K. Konieczka, Tatiana A. Yatskievych, and Parker B. Antin. 2006. “MicroRNA expression during chick embryo development.” <em>Developmental Dynamics</em> 235 (11): 3156–65. <a href="https://doi.org/10.1002/dvdy.20956">https://doi.org/10.1002/dvdy.20956</a>.</p>
</div>
<div>
<p>De Giovanni, Marco, Valeria Cutillo, Amir Giladi, Eleonora Sala, Carmela G Maganuco, Chiara Medaglia, Pietro Di Lucia, et al. 2020. “Spatiotemporal regulation of type I interferon expression determines the antiviral polarization of CD4+ T cells.” <em>Nature Immunology</em> 21 (3): 321–30. <a href="https://doi.org/10.1038/s41590-020-0596-6">https://doi.org/10.1038/s41590-020-0596-6</a>.</p>
</div>
<div>
<p>Delorey, Toni M, Carly G K Ziegler, Graham Heimberg, Rachelly Normand, Yiming Yang, Asa Segerstolpe, Domenic Abbondanza, et al. 2021. “A single-cell and spatial atlas of autopsy tissues reveals pathology and cellular targets of SARS-CoV-2.” <em>bioRxiv</em>, January, 2021.02.25.430130. <a href="https://doi.org/10.1101/2021.02.25.430130">https://doi.org/10.1101/2021.02.25.430130</a>.</p>
</div>
<div>
<p>Diez-Roux, Graciana, Sandro Banfi, Marc Sultan, Lars Geffers, Santosh Anand, David Rozado, Alon Magen, et al. 2011. “A High-Resolution Anatomical Atlas of the Transcriptome in the Mouse Embryo.” Edited by Gregory S. Barsh. <em>PLoS Biology</em> 9 (1): e1000582. <a href="https://doi.org/10.1371/journal.pbio.1000582">https://doi.org/10.1371/journal.pbio.1000582</a>.</p>
</div>
<div>
<p>Dirks, Robert M., and Niles A. Pierce. 2004. “From The Cover: Triggered amplification by hybridization chain reaction.” <em>Proceedings of the National Academy of Sciences</em> 101 (43): 15275–8. <a href="https://doi.org/10.1073/pnas.0407024101">https://doi.org/10.1073/pnas.0407024101</a>.</p>
</div>
<div>
<p>Dobin, Alexander, Carrie A Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, and Thomas R Gingeras. 2012. “STAR: ultrafast universal RNA-seq aligner.” <em>Bioinformatics</em> 29 (1): 15–21. <a href="https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635</a>.</p>
</div>
<div>
<p>Dries, Ruben, Qian Zhu, Chee-Huat Linus Eng, Arpan Sarkar, Feng Bao, Rani E George, Nico Pierson, Long Cai, and Guo-Cheng Yuan. 2019. “Giotto, a pipeline for integrative analysis and visualization of single-cell spatial transcriptomic data.” <em>bioRxiv</em>, May, 701680. <a href="https://doi.org/10.1101/701680">https://doi.org/10.1101/701680</a>.</p>
</div>
<div>
<p>Drmanac, R., Andrew B. Sparks, Matthew J. Callow, Aaron L. Halpern, Norman L. Burns, Bahram G. Kermani, Paolo Carnevali, et al. 2010. “Human Genome Sequencing Using Unchained Base Reads on Self-Assembling DNA Nanoarrays.” <em>Science</em> 327 (5961): 78–81. <a href="https://doi.org/10.1126/science.1181498">https://doi.org/10.1126/science.1181498</a>.</p>
</div>
<div>
<p>Droin, Colas, Jakob El Kholtei, Keren Bahar Halpern, Clémence Hurni, Milena Rozenberg, Sapir Muvkadi, Shalev Itzkovitz, and Felix Naef. 2020. “Space-time logic of liver gene expression at sublobular scale.” <em>bioRxiv</em>, January, 2020.03.05.976571. <a href="https://doi.org/10.1101/2020.03.05.976571">https://doi.org/10.1101/2020.03.05.976571</a>.</p>
</div>
<div>
<p>“DSP Technology Access Program (TAP).” n.d. Accessed March 30, 2021.</p>
</div>
<div>
<p>Dulac, Catherine, and Richard Axel. 1995. “A novel family of genes encoding putative pheromone receptors in mammals.” <em>Cell</em> 83 (2): 195–206. <a href="https://doi.org/https://doi.org/10.1016/0092-8674(95)90161-2">https://doi.org/https://doi.org/10.1016/0092-8674(95)90161-2</a>.</p>
</div>
<div>
<p>Durruthy-Durruthy, Jens, Mark Wossidlo, Sunil Pai, Yusuke Takahashi, Gugene Kang, Larsson Omberg, Bertha Chen, Hiromitsu Nakauchi, Renee Reijo Pera, and Vittorio Sebastiano. 2016. “Spatiotemporal Reconstruction of the Human Blastocyst by Single-Cell Gene-Expression Analysis Informs Induction of Naive Pluripotency.” <em>Developmental Cell</em> 38 (1): 100–115. <a href="https://doi.org/10.1016/j.devcel.2016.06.014">https://doi.org/10.1016/j.devcel.2016.06.014</a>.</p>
</div>
<div>
<p>Durruthy-Durruthy, Robert, Assaf Gottlieb, Byron H. Hartman, Jörg Waldhaus, Roman D. Laske, Russ Altman, and Stefan Heller. 2014. “Reconstruction of the Mouse Otocyst and Early Neuroblast Lineage at Single-Cell Resolution.” <em>Cell</em> 157 (4): 964–78. <a href="https://doi.org/10.1016/j.cell.2014.03.036">https://doi.org/10.1016/j.cell.2014.03.036</a>.</p>
</div>
<div>
<p>Ebbing, Annabel, Abel Vertesy, Marco Betist, Bastiaan Spanjaard, Jan Philipp Junker, Eugene Berezikov, Alexander van Oudenaarden, and Hendrik Korswagen. 2018. “Spatial transcriptomics of C. elegans males and hermaphrodites identifies novel fertility genes.” <em>bioRxiv</em>, June, 348201. <a href="https://doi.org/10.1101/348201">https://doi.org/10.1101/348201</a>.</p>
</div>
<div>
<p>Eberwine, J, H Yeh, K Miyashiro, Y Cao, S Nair, R Finnell, M Zettel, and P Coleman. 1992. “Analysis of gene expression in single live neurons.” <em>Proceedings of the National Academy of Sciences</em> 89 (7): 3010 LP–3014. <a href="https://doi.org/10.1073/pnas.89.7.3010">https://doi.org/10.1073/pnas.89.7.3010</a>.</p>
</div>
<div>
<p>Edsgärd, Daniel, Per Johnsson, and Rickard Sandberg. 2018. “Identification of spatial expression trends in single-cell gene expression data.” <em>Nature Methods</em> 15 (5): 339–42. <a href="https://doi.org/10.1038/nmeth.4634">https://doi.org/10.1038/nmeth.4634</a>.</p>
</div>
<div>
<p>Efremova, Mirjana, Miquel Vento-Tormo, Sarah A Teichmann, and Roser Vento-Tormo. 2020. “CellPhoneDB: inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes.” <em>Nature Protocols</em> 15 (4): 1484–1506. <a href="https://doi.org/10.1038/s41596-020-0292-x">https://doi.org/10.1038/s41596-020-0292-x</a>.</p>
</div>
<div>
<p>Eichenberger, Bastian Th., YinXiu Zhan, Markus Rempfler, Luca Giorgetti, and Jeffrey A Chao. 2020. “deepBlink: Threshold-independent detection and localization of diffraction-limited spots.” <em>bioRxiv</em>, January, 2020.12.14.422631. <a href="https://doi.org/10.1101/2020.12.14.422631">https://doi.org/10.1101/2020.12.14.422631</a>.</p>
</div>
<div>
<p>Elosua, Marc, Paula Nieto, Elisabetta Mereu, Ivo Gut, and Holger Heyn. 2020. “SPOTlight: Seeded NMF regression to Deconvolute Spatial Transcriptomics Spots with Single-Cell Transcriptomes.” <em>bioRxiv</em>, June, 2020.06.03.131334. <a href="https://doi.org/10.1101/2020.06.03.131334">https://doi.org/10.1101/2020.06.03.131334</a>.</p>
</div>
<div>
<p>Emmert-Buck, Michael R., Robert F. Bonner, Paul D. Smith, Rodrigo F. Chuaqui, Zhengping Zhuang, Seth R. Goldstein, Rhonda A. Weiss, and Lance A. Liotta. 1996. “Laser Capture Microdissection.” <em>Science</em> 274 (5289): 998–1001. <a href="https://doi.org/10.1126/science.274.5289.998">https://doi.org/10.1126/science.274.5289.998</a>.</p>
</div>
<div>
<p>Eng, Chee Huat Linus, Michael Lawson, Qian Zhu, Ruben Dries, Noushin Koulena, Yodai Takei, Jina Yun, et al. 2019. “Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+.” <em>Nature</em> 568 (7751): 235–39. <a href="https://doi.org/10.1038/s41586-019-1049-y">https://doi.org/10.1038/s41586-019-1049-y</a>.</p>
</div>
<div>
<p>Eng, Chee-Huat Linus, Sheel Shah, Julian Thomassie, and Long Cai. 2017. “Profiling the transcriptome with RNA SPOTs.” <em>Nature Methods</em> 14 (12): 1153–5. <a href="https://doi.org/10.1038/nmeth.4500">https://doi.org/10.1038/nmeth.4500</a>.</p>
</div>
<div>
<p>Fan, Zhen, Runsheng Chen, and Xiaowei Chen. 2020. “SpatialDB: a database for spatially resolved transcriptomes.” <em>Nucleic Acids Research</em> 48 (D1): D233–D237. <a href="https://doi.org/10.1093/nar/gkz934">https://doi.org/10.1093/nar/gkz934</a>.</p>
</div>
<div>
<p>Farris, Shannon, James M Ward, Kelly E Carstens, Mahsa Samadi, Yu Wang, and Serena M Dudek. 2019. “Hippocampal Subregions Express Distinct Dendritic Transcriptomes that Reveal Differences in Mitochondrial Function in CA2.” <em>Cell Reports</em> 29 (2): 522–539.e6. <a href="https://doi.org/10.1016/j.celrep.2019.08.093">https://doi.org/10.1016/j.celrep.2019.08.093</a>.</p>
</div>
<div>
<p>Fazal, Furqan M., Shuo Han, Kevin R. Parker, Pornchai Kaewsapsak, Jin Xu, Alistair N. Boettiger, Howard Y. Chang, and Alice Y. Ting. 2019. “Atlas of Subcellular RNA Localization Revealed by APEX-Seq.” <em>Cell</em> 178 (2): 473–490.e26. <a href="https://doi.org/10.1016/j.cell.2019.05.027">https://doi.org/10.1016/j.cell.2019.05.027</a>.</p>
</div>
<div>
<p>Femino, Andrea M, Fredric S Fay, Kevin Fogarty, and Robert H Singer. 1998. “Visualization of Single RNA Transcripts in Situ.” <em>Science</em> 280 (5363): 585 LP–590. <a href="https://doi.org/10.1126/science.280.5363.585">https://doi.org/10.1126/science.280.5363.585</a>.</p>
</div>
<div>
<p>Fire, Andrew, and Si Qun Xu. 1995. “Rolling replication of short DNA circles.” <em>Proceedings of the National Academy of Sciences</em> 92 (10): 4641–5. <a href="https://doi.org/10.1073/pnas.92.10.4641">https://doi.org/10.1073/pnas.92.10.4641</a>.</p>
</div>
<div>
<p>Foley, Joseph W., Chunfang Zhu, Philippe Jolivet, Shirley X. Zhu, Peipei Lu, Michael J. Meaney, and Robert B. West. 2019. “Gene expression profiling of single cells from archival tissue with laser-capture microdissection and Smart-3SEQ.” <em>Genome Research</em> 29 (11): 1816–25. <a href="https://doi.org/10.1101/gr.234807.118">https://doi.org/10.1101/gr.234807.118</a>.</p>
</div>
<div>
<p>Foreman, Robert, and Roy Wollman. 2019. “Mammalian gene expression variability is explained by underlying cell state.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/626424">https://doi.org/10.1101/626424</a>.</p>
</div>
<div>
<p>Forrester, Lesley M., Andras Nagy, Mehran Sam, Alistair Watt, Lois Stevenson, Alan Bernstein, Alexandra L. Joyner, and Wolfgang Wurst. 1996. “An induction gene trap screen in embryonic stem cells: Identification of genes that respond to retinoic acid in vitro.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 93 (4): 1677–82. <a href="https://doi.org/10.1073/pnas.93.4.1677">https://doi.org/10.1073/pnas.93.4.1677</a>.</p>
</div>
<div>
<p>Fowlkes, Charless C., Cris L. Luengo Hendriks, Soile V. E. Keränen, Gunther H. Weber, Oliver Rübel, Min-Yu Huang, Sohail Chatoor, et al. 2008. “A Quantitative Spatiotemporal Atlas of Gene Expression in the Drosophila Blastoderm.” <em>Cell</em> 133 (2): 364–74. <a href="https://doi.org/10.1016/j.cell.2008.01.053">https://doi.org/10.1016/j.cell.2008.01.053</a>.</p>
</div>
<div>
<p>Friedrich, G., and P. Soriano. 1991. “Promoter traps in embryonic stem cells: a genetic screen to identify and mutate developmental genes in mice.” <em>Genes &amp; Development</em> 5 (9): 1513–23. <a href="https://doi.org/10.1101/gad.5.9.1513">https://doi.org/10.1101/gad.5.9.1513</a>.</p>
</div>
<div>
<p>Frise, Erwin, Ann S Hammonds, and Susan E Celniker. 2010. “Systematic image‐driven analysis of the spatial Drosophila embryonic expression landscape.” <em>Molecular Systems Biology</em> 6 (1): 345. <a href="https://doi.org/10.1038/msb.2009.102">https://doi.org/10.1038/msb.2009.102</a>.</p>
</div>
<div>
<p>Frohman, M. A., M. K. Dush, and G. R. Martin. 1988. “Rapid production of full-length cDNAs from rare transcripts: amplification using a single gene-specific oligonucleotide primer.” <em>Proceedings of the National Academy of Sciences</em> 85 (23): 8998–9002. <a href="https://doi.org/10.1073/pnas.85.23.8998">https://doi.org/10.1073/pnas.85.23.8998</a>.</p>
</div>
<div>
<p>Fu, Xiaonan, Li Sun, Jane Y Chen, Runze Dong, Yiing Lin, Richard D Palmiter, Shin Lin, and Liangcai Gu. 2021. “Continuous Polony Gels for Tissue Mapping with High Resolution and RNA Capture Efficiency.” <em>bioRxiv</em>, January, 2021.03.17.435795. <a href="https://doi.org/10.1101/2021.03.17.435795">https://doi.org/10.1101/2021.03.17.435795</a>.</p>
</div>
<div>
<p>Gall, Joseph G, Mary Lou, Pardue Kline, and Norman H Giles. 1969. “Formation and Detecction of RNA-DNA Hybrid Molecules in Cytological Preparations.” <em>PNAS</em> 63 (2): 378–83. <a href="https://doi.org/10.1073/pnas.63.2.378">https://doi.org/10.1073/pnas.63.2.378</a>.</p>
</div>
<div>
<p>Gawantka, Volker, Nicolas Pollet, Hajo Delius, Martin Vingron, Ralf Pfister, Rebecca Nitsch, Claudia Blumenstock, and Christof Niehrs. 1998. “Gene expression screening in Xenopus identifies molecular pathways, predicts gene function and provides a global view of embryonic patterning.” <em>Mechanisms of Development</em> 77 (2): 95–141. <a href="https://doi.org/10.1016/S0925-4773(98)00115-4">https://doi.org/10.1016/S0925-4773(98)00115-4</a>.</p>
</div>
<div>
<p>Gebhardt, Rolf. 2014. “Liver zonation: Novel aspects of its regulation and its impact on homeostasis.” <em>World Journal of Gastroenterology</em> 20 (26): 8491. <a href="https://doi.org/10.3748/wjg.v20.i26.8491">https://doi.org/10.3748/wjg.v20.i26.8491</a>.</p>
</div>
<div>
<p>Geffers, Lars, Benjamin Tetzlaff, Xiao Cui, Jun Yan, and Gregor Eichele. 2012. “METscout: a pathfinder exploring the landscape of metabolites, enzymes and transporters.” <em>Nucleic Acids Research</em> 41 (D1): D1047–D1054. <a href="https://doi.org/10.1093/nar/gks886">https://doi.org/10.1093/nar/gks886</a>.</p>
</div>
<div>
<p>“Gene Expression Panels | NanoString Technologies.” n.d. Accessed September 15, 2020. <a href="https://www.nanostring.com/products/gene-expression-panels/gene-expression-panels-overview">https://www.nanostring.com/products/gene-expression-panels/gene-expression-panels-overview</a>.</p>
</div>
<div>
<p>Ghazanfar, Shila, Yingxin Lin, Xianbin Su, David Ming Lin, Ellis Patrick, Ze-Guang Han, John C Marioni, and Jean Yee Hwa Yang. 2020. “Investigating higher-order interactions in single-cell data with scHOT.” <em>Nature Methods</em> 17 (8): 799–806. <a href="https://doi.org/10.1038/s41592-020-0885-x">https://doi.org/10.1038/s41592-020-0885-x</a>.</p>
</div>
<div>
<p>Giacomello, Stefania, Fredrik Salmén, Barbara K. Terebieniec, Sanja Vickovic, José Fernandez Navarro, Andrey Alexeyenko, Johan Reimegård, et al. 2017. “Spatially resolved transcriptome profiling in model plant species.” <em>Nature Plants</em>. <a href="https://doi.org/10.1038/nplants.2017.61">https://doi.org/10.1038/nplants.2017.61</a>.</p>
</div>
<div>
<p>Giani, Alice Maria, Guido Roberto Gallo, Luca Gianfranceschi, and Giulio Formenti. 2020. “Long walk to genomics: History and current approaches to genome sequencing and assembly.” <em>Computational and Structural Biotechnology Journal</em> 18 (January): 9–19. <a href="https://doi.org/10.1016/j.csbj.2019.11.002">https://doi.org/10.1016/j.csbj.2019.11.002</a>.</p>
</div>
<div>
<p>Gilchrist, Michael J., Mikkel B. Christensen, Odile Bronchain, Frédéric Brunet, Albert Chesneau, Ursula Fenger, Timothy J. Geach, et al. 2009. “Database of queryable gene expression patterns for Xenopus.” <em>Developmental Dynamics</em> 238 (6): 1379–88. <a href="https://doi.org/10.1002/dvdy.21940">https://doi.org/10.1002/dvdy.21940</a>.</p>
</div>
<div>
<p>Giolai, Michael, Walter Verweij, Ashleigh Lister, Darren Heavens, Iain Macaulay, and Matthew D. Clark. 2019. “Spatially resolved transcriptomics reveals plant host responses to pathogens.” <em>Plant Methods</em> 15 (1): 114. <a href="https://doi.org/10.1186/s13007-019-0498-5">https://doi.org/10.1186/s13007-019-0498-5</a>.</p>
</div>
<div>
<p>Glaser, Joshua I., Bradley M. Zamft, George M. Church, and Konrad P. Kording. 2015. “Puzzle Imaging: Using Large-Scale Dimensionality Reduction Algorithms for Localization.” Edited by Joseph Najbauer. <em>PLOS ONE</em> 10 (7): e0131593. <a href="https://doi.org/10.1371/journal.pone.0131593">https://doi.org/10.1371/journal.pone.0131593</a>.</p>
</div>
<div>
<p>Goh, Jolene Jie Lin, Nigel Chou, Wan Yi Seow, Norbert Ha, Chung Pui Paul Cheng, Yun Ching Chang, Ziqing Winston Zhao, and Kok Hao Chen. 2020. “Highly specific multiplexed RNA imaging in tissues with split-FISH.” <em>Nature Methods</em> 17 (7): 689–93. <a href="https://doi.org/10.1038/s41592-020-0858-0">https://doi.org/10.1038/s41592-020-0858-0</a>.</p>
</div>
<div>
<p>Gong, Shiaoching, Chen Zheng, Martin L. Doughty, Kasia Losos, Nicholas Didkovsky, Uta B. Schambra, Norma J. Nowak, et al. 2003. “A gene expression atlas of the central nervous system based on bacterial artificial chromosomes.” <em>Nature</em> 425 (6961): 917–25. <a href="https://doi.org/10.1038/nature02033">https://doi.org/10.1038/nature02033</a>.</p>
</div>
<div>
<p>Gossler, Achim, A. Joyner, Janet Rossant, and W. Skarnes. 1989. “Mouse embryonic stem cells and reporter constructs to detect developmentally regulated genes.” <em>Science</em> 244 (4903): 463–65. <a href="https://doi.org/10.1126/science.2497519">https://doi.org/10.1126/science.2497519</a>.</p>
</div>
<div>
<p>Govek, Kiya W, Venkata S Yamajala, and Pablo G Camara. 2019. “Clustering-independent analysis of genomic data using spectral simplicial theory.” <em>PLOS Computational Biology</em> 15 (11): e1007509. <a href="https://doi.org/10.1371/journal.pcbi.1007509">https://doi.org/10.1371/journal.pcbi.1007509</a>.</p>
</div>
<div>
<p>Grange, Pascal, Jason W. Bohland, Benjamin W. Okaty, Ken Sugino, Hemant Bokil, Sacha B. Nelson, Lydia Ng, Michael Hawrylycz, and Partha P. Mitra. 2014. “Cell-type-based model explaining coexpression patterns of genes in the brain.” <em>Proceedings of the National Academy of Sciences</em> 111 (14): 5397–5402. <a href="https://doi.org/10.1073/pnas.1312098111">https://doi.org/10.1073/pnas.1312098111</a>.</p>
</div>
<div>
<p>Gregory, J M, K McDade, M R Livesey, I Croy, S Marion de Proce, T Aitman, S Chandran, and C Smith. 2020. “Spatial transcriptomics identifies spatially dysregulated expression of GRM3 and USP47 in amyotrophic lateral sclerosis.” <em>Neuropathology and Applied Neurobiology</em> 46 (5): 441–57. <a href="https://doi.org/10.1111/nan.12597">https://doi.org/10.1111/nan.12597</a>.</p>
</div>
<div>
<p>Greulich, Karl Otto. 1999. “Introduction: The history of using light as a working tool.” In <em>Micromanipulation by Light in Biology and Medicine</em>, 1–6. Boston, MA: Birkhäuser Boston. <a href="https://doi.org/10.1007/978-1-4612-4110-2_1">https://doi.org/10.1007/978-1-4612-4110-2_1</a>.</p>
</div>
<div>
<p>Grün, Dominic, Lennart Kester, and Alexander van Oudenaarden. 2014. “Validation of noise models for single-cell transcriptomics.” <em>Nature Methods</em> 11 (6): 637–40. <a href="https://doi.org/10.1038/nmeth.2930">https://doi.org/10.1038/nmeth.2930</a>.</p>
</div>
<div>
<p>Gurunathan, Rajalakshmi, Bernard Van Emden, Sethuraman Panchanathan, and Sudhir Kumar. 2004. “Identifying spatially similar gene expression patterns in early stage fruit fly embryo images: Binary feature versus invariant moment digital representations.” <em>BMC Bioinformatics</em> 5 (1): 202. <a href="https://doi.org/10.1186/1471-2105-5-202">https://doi.org/10.1186/1471-2105-5-202</a>.</p>
</div>
<div>
<p>Gyllborg, Daniel, Christoffer Mattsson Langseth, Xiaoyan Qian, Sergio Marco Salas, Markus Hilscher, Ed Lein, and Mats Nilsson. 2020. “Hybridization-based In Situ Sequencing (HybISS): spatial transcriptomic detection in human and mouse brain tissue.” <em>bioRxiv</em>, February, 2020.02.03.931618. <a href="https://doi.org/10.1101/2020.02.03.931618">https://doi.org/10.1101/2020.02.03.931618</a>.</p>
</div>
<div>
<p>Halpern, Keren Bahar, Rom Shenhav, Hassan Massalha, Beata Toth, Adi Egozi, Efi E. Massasa, Chiara Medgalia, et al. 2018. “Paired-cell sequencing enables spatial gene expression mapping of liver endothelial cells.” <em>Nature Biotechnology</em> 36 (10): 962. <a href="https://doi.org/10.1038/nbt.4231">https://doi.org/10.1038/nbt.4231</a>.</p>
</div>
<div>
<p>Halpern, Keren Bahar, Rom Shenhav, Orit Matcovitch-Natan, Beáta Tóth, Doron Lemze, Matan Golan, Efi E. Massasa, et al. 2017. “Single-cell spatial reconstruction reveals global division of labour in the mammalian liver.” <em>Nature</em> 542 (7641): 1–5. <a href="https://doi.org/10.1038/nature21065">https://doi.org/10.1038/nature21065</a>.</p>
</div>
<div>
<p>Hammonds, Ann S., Christopher A. Bristow, William W. Fisher, Richard Weiszmann, Siqi Wu, Volker Hartenstein, Manolis Kellis, Bin Yu, Erwin Frise, and Susan E. Celniker. 2013. “Spatial expression of transcription factors in Drosophila embryonic organ development.” <em>Genome Biology</em> 14 (12): R140. <a href="https://doi.org/10.1186/gb-2013-14-12-r140">https://doi.org/10.1186/gb-2013-14-12-r140</a>.</p>
</div>
<div>
<p>Hanchuan Peng, Fuhui Long, M. B. Eisen, and E. W. Myers. 2006. “Clustering Gene Expression Patterns of Fly Embryos.” In <em>3rd Ieee International Symposium on Biomedical Imaging: Macro to Nano, 2006.</em>, 2006:1144–7. IEEE. <a href="https://doi.org/10.1109/ISBI.2006.1625125">https://doi.org/10.1109/ISBI.2006.1625125</a>.</p>
</div>
<div>
<p>Hao, Minsheng, Kui Hua, and Xuegong Zhang. 2021. “SOMDE: A scalable method for identifying spatially variable genes with self-organizing map.” <em>bioRxiv</em>, January, 2020.12.10.419549. <a href="https://doi.org/10.1101/2020.12.10.419549">https://doi.org/10.1101/2020.12.10.419549</a>.</p>
</div>
<div>
<p>Harding, Simon D., Chris Armit, Jane Armstrong, Jane Brennan, Ying Cheng, Bernard Haggarty, Derek Houghton, et al. 2011. “The GUDMAP database - an online resource for genitourinary research.” <em>Development</em> 138 (13): 2845–53. <a href="https://doi.org/10.1242/dev.063594">https://doi.org/10.1242/dev.063594</a>.</p>
</div>
<div>
<p>Harrison, P. R., D. Conkie, J. Paul, and K. Jones. 1973. “Localisation of cellular globin messenger RNA by in situ hybridisation to complementary DNA.” <em>FEBS Letters</em> 32 (1): 109–12. <a href="https://doi.org/10.1016/0014-5793(73)80749-5">https://doi.org/10.1016/0014-5793(73)80749-5</a>.</p>
</div>
<div>
<p>Hashimshony, Tamar, Naftalie Senderovich, Gal Avital, Agnes Klochendler, Yaron de Leeuw, Leon Anavy, Dave Gennert, et al. 2016. “CEL-Seq2: sensitive highly-multiplexed single-cell RNA-Seq.” <em>Genome Biology</em> 17 (1): 77. <a href="https://doi.org/10.1186/s13059-016-0938-8">https://doi.org/10.1186/s13059-016-0938-8</a>.</p>
</div>
<div>
<p>Hashimshony, Tamar, Florian Wagner, Noa Sher, and Itai Yanai. 2012. “CEL-Seq: Single-Cell RNA-Seq by Multiplexed Linear Amplification.” <em>Cell Reports</em> 2 (3): 666–73. <a href="https://doi.org/https://doi.org/10.1016/j.celrep.2012.08.003">https://doi.org/https://doi.org/10.1016/j.celrep.2012.08.003</a>.</p>
</div>
<div>
<p>Haudry, Yannick, Hugo Berube, Ivica Letunic, P.-D. Weeber, Julien Gagneur, Charles Girardot, Misha Kapushesky, et al. 2007. “4DXpress: a database for cross-species expression pattern comparisons.” <em>Nucleic Acids Research</em> 36 (Database): D847–D853. <a href="https://doi.org/10.1093/nar/gkm797">https://doi.org/10.1093/nar/gkm797</a>.</p>
</div>
<div>
<p>Hawrylycz, Michael J., Ed S. Lein, Angela L. Guillozet-Bongaarts, Elaine H. Shen, Lydia Ng, Jeremy A. Miller, Louie N. van de Lagemaat, et al. 2012. “An anatomically comprehensive atlas of the adult human brain transcriptome.” <em>Nature</em> 489 (7416): 391–99. <a href="https://doi.org/10.1038/nature11405">https://doi.org/10.1038/nature11405</a>.</p>
</div>
<div>
<p>He, Bryan, Ludvig Bergenstråhle, Linnea Stenbeck, Abubakar Abid, Alma Andersson, Åke Borg, Jonas Maaskola, Joakim Lundeberg, and James Zou. 2020. “Integrating spatial gene expression and breast tumour morphology via deep learning.” <em>Nature Biomedical Engineering</em>, June, 1–8. <a href="https://doi.org/10.1038/s41551-020-0578-x">https://doi.org/10.1038/s41551-020-0578-x</a>.</p>
</div>
<div>
<p>Heffel, Andreas, Peter F. Stadler, Sonja J. Prohaska, Gerhard Kauer, and Jens-Peer Kuska. 2008. “Process flow for classification and clustering of fruit fly gene expression patterns.” In <em>2008 15th Ieee International Conference on Image Processing</em>, 721–24. IEEE. <a href="https://doi.org/10.1109/ICIP.2008.4711856">https://doi.org/10.1109/ICIP.2008.4711856</a>.</p>
</div>
<div>
<p>Hemby, Scott E, Stephen D Ginsberg, Brian Brunk, Steven E Arnold, John Q Trojanowski, and James H Eberwine. 2002. “Gene Expression Profile for Schizophrenia: Discrete Neuron Transcription Patterns in the Entorhinal Cortex.” <em>Archives of General Psychiatry</em> 59 (7): 631–40. <a href="https://doi.org/10.1001/archpsyc.59.7.631">https://doi.org/10.1001/archpsyc.59.7.631</a>.</p>
</div>
<div>
<p>Henrich, Thorsten. 2003. “MEPD: a Medaka gene expression pattern database.” <em>Nucleic Acids Research</em> 31 (1): 72–74. <a href="https://doi.org/10.1093/nar/gkg017">https://doi.org/10.1093/nar/gkg017</a>.</p>
</div>
<div>
<p>Hiwatashi, Yuji, Tomoaki Nishiyama, Tomomichi Fujita, and Mitsuyasu Hasebe. 2001. “Establishment of gene-trap and enhancer-trap systems in the moss Physcomitrella patens.” <em>The Plant Journal</em> 28 (1): 105–16. <a href="https://doi.org/10.1046/j.1365-313X.2001.01121.x">https://doi.org/10.1046/j.1365-313X.2001.01121.x</a>.</p>
</div>
<div>
<p>Hope, I A. 1991. “’Promoter trapping’ in Caenorhabditis elegans.” Vol. 113. <a href="https://dev.biologists.org/content/develop/113/2/399.full.pdf">https://dev.biologists.org/content/develop/113/2/399.full.pdf</a>.</p>
</div>
<div>
<p>Howe, Douglas G., Yvonne M. Bradford, Anne Eagle, David Fashena, Ken Frazer, Patrick Kalita, Prita Mani, et al. 2017. “The Zebrafish Model Organism Database: New support for human disease models, mutation details, gene expression phenotypes and searching.” <em>Nucleic Acids Research</em> 45 (D1): D758–D768. <a href="https://doi.org/10.1093/nar/gkw1116">https://doi.org/10.1093/nar/gkw1116</a>.</p>
</div>
<div>
<p>Huber, D., L. Voith von Voithenberg, and G. V. Kaigala. 2018. “Fluorescence in situ hybridization (FISH): History, limitations and what to expect from micro-scale FISH?” <em>Micro and Nano Engineering</em> 1 (November): 15–24. <a href="https://doi.org/10.1016/j.mne.2018.10.006">https://doi.org/10.1016/j.mne.2018.10.006</a>.</p>
</div>
<div>
<p>Hunt-Newbury, Rebecca, Ryan Viveiros, Robert Johnsen, Allan Mah, Dina Anastas, Lily Fang, Erin Halfnight, et al. 2007. “High-Throughput In Vivo Analysis of Gene Expression in Caenorhabditis elegans.” Edited by John Sulston. <em>PLoS Biology</em> 5 (9): e237. <a href="https://doi.org/10.1371/journal.pbio.0050237">https://doi.org/10.1371/journal.pbio.0050237</a>.</p>
</div>
<div>
<p>Hwang, William L, Karthik A Jagadeesh, Jimmy A Guo, Hannah I Hoffman, Payman Yadollahpour, Rahul Mohan, Eugene Drokhlyansky, et al. 2020. “Single-nucleus and spatial transcriptomics of archival pancreatic cancer reveals multi-compartment reprogramming after neoadjuvant treatment.” <em>bioRxiv</em>, August, 2020.08.25.267336. <a href="https://doi.org/10.1101/2020.08.25.267336">https://doi.org/10.1101/2020.08.25.267336</a>.</p>
</div>
<div>
<p>Islam, Saiful, Una Kjällquist, Annalena Moliner, Pawel Zajac, Jian-Bing Fan, Peter Lönnerberg, and Sten Linnarsson. 2011. “Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq.” <em>Genome Research</em> 21 (7): 1160–7. <a href="https://doi.org/10.1101/gr.110882.110">https://doi.org/10.1101/gr.110882.110</a>.</p>
</div>
<div>
<p>Jagalur, Manjunatha, Chris Pal, Erik Learned-Miller, R Thomas Zoeller, and David Kulp. 2007. “Analyzing in situ gene expression in the mouse brain with image registration, feature extraction and block clustering.” <em>BMC Bioinformatics</em> 8 (Suppl 10): S5. <a href="https://doi.org/10.1186/1471-2105-8-S10-S5">https://doi.org/10.1186/1471-2105-8-S10-S5</a>.</p>
</div>
<div>
<p>Janning, Wilfried. 1997. “FlyView, aDrosophilaimage database, and otherDrosophiladatabases.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 469–75. <a href="https://doi.org/https://doi.org/10.1006/scdb.1997.0172">https://doi.org/https://doi.org/10.1006/scdb.1997.0172</a>.</p>
</div>
<div>
<p>Jayaraman, Karthik, Sethuraman Panchanathan, and Sudhir Kumar. 2001. “Classification and Indexing of Gene Expression Images.”</p>
</div>
<div>
<p>Jemt, Anders, Fredrik Salmén, Anna Lundmark, Annelie Mollbrink, José Fernández Navarro, Patrik L. Ståhl, Tülay Yucel-Lindberg, and Joakim Lundeberg. 2016. “An automated approach to prepare tissue-derived spatially barcoded RNA-sequencing libraries.” <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/srep37137">https://doi.org/10.1038/srep37137</a>.</p>
</div>
<div>
<p>Jenett, Arnim, Gerald M. Rubin, Teri T. B. Ngo, David Shepherd, Christine Murphy, Heather Dionne, Barret D. Pfeiffer, et al. 2012. “A GAL4-Driver Line Resource for Drosophila Neurobiology.” <em>Cell Reports</em> 2 (4): 991–1001. <a href="https://doi.org/10.1016/j.celrep.2012.09.011">https://doi.org/10.1016/j.celrep.2012.09.011</a>.</p>
</div>
<div>
<p>Jerby-Arnon, Livnat, and Aviv Regev. 2020. “Mapping multicellular programs from single-cell profiles.” <em>bioRxiv</em>, January, 2020.08.11.245472. <a href="https://doi.org/10.1101/2020.08.11.245472">https://doi.org/10.1101/2020.08.11.245472</a>.</p>
</div>
<div>
<p>Ji, Andrew L, Adam J Rubin, Kim Thrane, Sizun Jiang, David L Reynolds, Robin M Meyers, Margaret G Guo, et al. 2020. “Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma.” <em>Cell</em> 182 (2): 497–514.e22. <a href="https://doi.org/https://doi.org/10.1016/j.cell.2020.05.039">https://doi.org/https://doi.org/10.1016/j.cell.2020.05.039</a>.</p>
</div>
<div>
<p>Ji, Shuiwang, Ying Xin Li, Zhi Hua Zhou, Sudhir Kumar, and Jieping Ye. 2009. “A bag-of-words approach for Drosophila gene expression pattern annotation.” <em>BMC Bioinformatics</em> 10 (1): 119. <a href="https://doi.org/10.1186/1471-2105-10-119">https://doi.org/10.1186/1471-2105-10-119</a>.</p>
</div>
<div>
<p>Ji, Shuiwang, Liang Sun, Rong Jin, Sudhir Kumar, and Jieping Ye. 2008. “Automated annotation of Drosophila gene expression patterns using a controlled vocabulary.” <em>Bioinformatics</em> 24 (17): 1881–8. <a href="https://doi.org/10.1093/bioinformatics/btn347">https://doi.org/10.1093/bioinformatics/btn347</a>.</p>
</div>
<div>
<p>Ji, Shuiwang, Lei Yuan, Ying-Xin Li, Zhi-Hua Zhou, Sudhir Kumar, and Jieping Ye. 2009. “Drosophila gene expression pattern annotation using sparse features and term-term interactions.” In <em>Proceedings of the 15th Acm Sigkdd International Conference on Knowledge Discovery and Data Mining - Kdd ’09</em>, 407. New York, New York, USA: ACM Press. <a href="https://doi.org/10.1145/1557019.1557068">https://doi.org/10.1145/1557019.1557068</a>.</p>
</div>
<div>
<p>Joglekar, Anoushka, Andrey Prjibelski, Ahmed Mahfouz, Paul Collier, Susan Lin, Anna Katharina Schlusche, Jordan Marrocco, et al. 2020. “Cell-type, single-cell, and spatial signatures of brain-region specific splicing in postnatal development.” <em>bioRxiv</em>, January, 2020.08.27.268730. <a href="https://doi.org/10.1101/2020.08.27.268730">https://doi.org/10.1101/2020.08.27.268730</a>.</p>
</div>
<div>
<p>John, H. A., M. L. Birnstiel, and K. W. Jones. 1969. “RNA-DNA hybrids at the cytological level.” <em>Nature</em> 223 (5206): 582–87. <a href="https://doi.org/10.1038/223582a0">https://doi.org/10.1038/223582a0</a>.</p>
</div>
<div>
<p>“Johns Hopkins Cell Imaging Core Facility.” n.d. Accessed October 14, 2020. <a href="https://www.hopkinsmedicine.org/kimmel%7B\_%7Dcancer%7B\_%7Dcenter/research/shared%7B\_%7Dresources/cell%7B\_%7Dimaging.html">https://www.hopkinsmedicine.org/kimmel{\_}cancer{\_}center/research/shared{\_}resources/cell{\_}imaging.html</a>.</p>
</div>
<div>
<p>Johnson, Alexander A. T., Julian M. Hibberd, Céline Gay, Pauline A. Essah, Jim Haseloff, Mark Tester, and Emmanuel Guiderdoni. 2005. “Spatial control of transgene expression in rice (Oryza sativa L.) using the GAL4 enhancer trapping system.” <em>The Plant Journal</em> 41 (5): 779–89. <a href="https://doi.org/10.1111/j.1365-313X.2005.02339.x">https://doi.org/10.1111/j.1365-313X.2005.02339.x</a>.</p>
</div>
<div>
<p>Junker, Jan Philipp, Emily S. Noël, Victor Guryev, Kevin A. Peterson, Gopi Shah, Jan Huisken, Andrew P. McMahon, Eugene Berezikov, Jeroen Bakkers, and Alexander Van Oudenaarden. 2014. “Genome-wide RNA Tomography in the Zebrafish Embryo.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2014.09.038">https://doi.org/10.1016/j.cell.2014.09.038</a>.</p>
</div>
<div>
<p>Kacharmina, Janet Estee, Peter B Crino, and James B T - Methods in Enzymology Eberwine. 1999. “[1] Preparation of cDNA from single cells and subcellular regions.” In <em>CDNA Preparation and Characterization</em>, 303:3–18. Academic Press. <a href="https://doi.org/https://doi.org/10.1016/S0076-6879(99)03003-7">https://doi.org/https://doi.org/10.1016/S0076-6879(99)03003-7</a>.</p>
</div>
<div>
<p>Kaewsapsak, Pornchai, David Michael Shechner, William Mallard, John L. Rinn, and Alice Y. Ting. 2017. “Live-cell mapping of organelle-associated RNAs via proximity biotinylation combined with protein-RNA crosslinking.” <em>eLife</em> 6 (December). <a href="https://doi.org/10.7554/eLife.29224">https://doi.org/10.7554/eLife.29224</a>.</p>
</div>
<div>
<p>Kamme, Fredrik, Ranelle Salunga, Jingxue Yu, Da-Thao Tran, Jessica Zhu, Lin Luo, Anton Bittner, et al. 2003. “Single-Cell Microarray Analysis in Hippocampus CA1: Demonstration and Validation of Cellular Heterogeneity.” <em>The Journal of Neuroscience</em> 23 (9): 3607 LP–3615. <a href="https://doi.org/10.1523/JNEUROSCI.23-09-03607.2003">https://doi.org/10.1523/JNEUROSCI.23-09-03607.2003</a>.</p>
</div>
<div>
<p>Karaiskos, Nikos, Philipp Wahle, Jonathan Alles, Anastasiya Boltengagen, Salah Ayoub, Claudia Kipar, Christine Kocks, Nikolaus Rajewsky, and Robert P. Zinzen. 2017. “The Drosophila embryo at single-cell transcriptome resolution.” <em>Science</em> 358 (6360): 194–99. <a href="https://doi.org/10.1126/science.aan3235">https://doi.org/10.1126/science.aan3235</a>.</p>
</div>
<div>
<p>Karali, Marianthi, Ivana Peluso, Vincenzo A. Gennarino, Marchesa Bilio, Roberta Verde, Giampiero Lago, Pascal Dollé, and Sandro Banfi. 2010. “miRNeye: a microRNA expression atlas of the mouse eye.” <em>BMC Genomics</em> 11 (1): 715. <a href="https://doi.org/10.1186/1471-2164-11-715">https://doi.org/10.1186/1471-2164-11-715</a>.</p>
</div>
<div>
<p>Kawakami, Koichi, Gembu Abe, Tokuko Asada, Kazuhide Asakawa, Ryuichi Fukuda, Aki Ito, Pradeep Lal, et al. 2010. “ZTrap: Zebrafish gene trap and enhancer trap database.” <em>BMC Developmental Biology</em> 10 (1): 105. <a href="https://doi.org/10.1186/1471-213X-10-105">https://doi.org/10.1186/1471-213X-10-105</a>.</p>
</div>
<div>
<p>Kawashima, T. 2000. “MAGEST: MAboya Gene Expression patterns and Sequence Tags.” <em>Nucleic Acids Research</em> 28 (1): 133–35. <a href="https://doi.org/10.1093/nar/28.1.133">https://doi.org/10.1093/nar/28.1.133</a>.</p>
</div>
<div>
<p>Ke, Rongqin, Marco Mignardi, Alexandra Pacureanu, Jessica Svedlund, Johan Botling, Carolina Wählby, and Mats Nilsson. 2013. “In situ sequencing for RNA analysis in preserved tissue and cells.” <em>Nature Methods</em> 10 (9): 857–60. <a href="https://doi.org/10.1038/nmeth.2563">https://doi.org/10.1038/nmeth.2563</a>.</p>
</div>
<div>
<p>Kerman, Ilan A., Bradley J. Buck, Simon J. Evans, Huda Akil, and Stanley J. Watson. 2006. “Combining laser capture microdissection with quantitative real-time PCR: Effects of tissue manipulation on RNA quality and gene expression.” <em>Journal of Neuroscience Methods</em> 153 (1): 71–85. <a href="https://doi.org/10.1016/j.jneumeth.2005.10.010">https://doi.org/10.1016/j.jneumeth.2005.10.010</a>.</p>
</div>
<div>
<p>Kim, Minhee, Dong-Min Kim, and Dong-Eun Kim. 2020. “Label-free fluorometric detection of microRNA using isothermal rolling circle amplification generating tandem G-quadruplex.” <em>Analyst</em> 145 (18): 6130–7. <a href="https://doi.org/10.1039/D0AN01329C">https://doi.org/10.1039/D0AN01329C</a>.</p>
</div>
<div>
<p>Kitahara, Osamu, Yoichi Furukawa, Toshihiro Tanaka, Chikashi Kihara, Kenji Ono, Renpei Yanagawa, Marcelo E Nita, Toshihisa Takagi, Yusuke Nakamura, and Tatsuhiko Tsunoda. 2001. “Alterations of Gene Expression during Colorectal Carcinogenesis Revealed by cDNA Microarrays after Laser-Capture Microdissection of Tumor Tissues and Normal Epithelia.” <em>Cancer Research</em> 61 (9): 3544 LP–3549. <a href="http://cancerres.aacrjournals.org/content/61/9/3544.abstract">http://cancerres.aacrjournals.org/content/61/9/3544.abstract</a>.</p>
</div>
<div>
<p>Kitamura, S, T Tanahashi, E Aoyagi, T Nakagawa, K Okamoto, T Kimura, H Miyamoto, et al. 2017. “Response Predictors of S-1, Cisplatin, and Docetaxel Combination Chemotherapy for Metastatic Gastric Cancer: Microarray Analysis of Whole Human Genes.” <em>Oncology</em> 93 (2): 127–35. <a href="https://doi.org/10.1159/000464329">https://doi.org/10.1159/000464329</a>.</p>
</div>
<div>
<p>Klein, Allon M., Linas Mazutis, Ilke Akartuna, Naren Tallapragada, Adrian Veres, Victor Li, Leonid Peshkin, David A. Weitz, and Marc W. Kirschner. 2015. “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells.” <em>Cell</em> 161 (5): 1187–1201. <a href="https://doi.org/10.1016/j.cell.2015.04.044">https://doi.org/10.1016/j.cell.2015.04.044</a>.</p>
</div>
<div>
<p>Klein, Christoph A, Stefan Seidl, Karina Petat-Dutter, Sonja Offner, Jochen B Geigl, Oleg Schmidt-Kittler, Nicole Wendler, et al. 2002. “Combined transcriptome and genome analysis of single micrometastatic cells.” <em>Nature Biotechnology</em> 20 (4): 387–92. <a href="https://doi.org/10.1038/nbt0402-387">https://doi.org/10.1038/nbt0402-387</a>.</p>
</div>
<div>
<p>Kloosterman, Wigard P., Erno Wienholds, Ewart de Bruijn, Sakari Kauppinen, and Ronald H A Plasterk. 2006. “In situ detection of miRNAs in animal embryos using LNA-modified oligonucleotide probes.” <em>Nature Methods</em> 3 (1): 27–29. <a href="https://doi.org/10.1038/nmeth843">https://doi.org/10.1038/nmeth843</a>.</p>
</div>
<div>
<p>Ko, M S H, S B H Ko, N Takahashi, K Nishiguchi, and K Abe. 1990. “Unbiased amplification of a highly complex mixture of DNA fragments by ‘lone linker’-tagged PCR.” <em>Nucleic Acids Research</em> 18 (14): 4293. <a href="https://doi.org/10.1093/nar/18.14.4293">https://doi.org/10.1093/nar/18.14.4293</a>.</p>
</div>
<div>
<p>Ko, Younhee, Seth A. Ament, James A. Eddy, Juan Caballero, John C. Earls, Leroy Hood, and Nathan D. Price. 2013. “Cell type-specific genes show striking and distinct patterns of spatial expression in the mouse brain.” <em>Proceedings of the National Academy of Sciences</em> 110 (8): 3095–3100. <a href="https://doi.org/10.1073/pnas.1222897110">https://doi.org/10.1073/pnas.1222897110</a>.</p>
</div>
<div>
<p>Kohman, Richie E, and George M Church. 2020. “Fluorescent in situ sequencing of DNA barcoded antibodies.” <em>bioRxiv</em> 20 (April): 2020.04.27.060624. <a href="https://doi.org/10.1101/2020.04.27.060624">https://doi.org/10.1101/2020.04.27.060624</a>.</p>
</div>
<div>
<p>Kopczynski, Casey C., Jasprina N. Noordermeer, Thomas L. Serano, W.-Y. Chen, John D. Pendleton, Suzanna Lewis, Corey S. Goodman, and Gerald M. Rubin. 1998. “A high throughput screen to identify secreted and transmembrane proteins involved in Drosophila embryogenesis.” <em>Proceedings of the National Academy of Sciences</em> 95 (17): 9973–8. <a href="https://doi.org/10.1073/pnas.95.17.9973">https://doi.org/10.1073/pnas.95.17.9973</a>.</p>
</div>
<div>
<p>Korsunsky, Ilya, Nghia Millard, Jean Fan, Kamil Slowikowski, Fan Zhang, Kevin Wei, Yuriy Baglaenko, Michael Brenner, Po-ru Loh, and Soumya Raychaudhuri. 2019. “Fast, sensitive and accurate integration of single-cell data with Harmony.” <em>Nature Methods</em> 16 (12): 1289–96. <a href="https://doi.org/10.1038/s41592-019-0619-0">https://doi.org/10.1038/s41592-019-0619-0</a>.</p>
</div>
<div>
<p>Köster, Johannes, Myles Brown, and X. Shirley Liu. 2019. “A Bayesian model for single cell transcript expression analysis on MERFISH data.” <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/bty718">https://doi.org/10.1093/bioinformatics/bty718</a>.</p>
</div>
<div>
<p>Kudo, Lili C., Nancy Vi, Zhongcai Ma, Tony Fields, Nuraly K. Avliyakulov, Michael J. Haykinson, Anatol Bragin, and Stanislav L. Karsten. 2012. “Novel Cell and Tissue Acquisition System (CTAS): Microdissection of Live and Frozen Brain Tissues.” Edited by Rafael Aldabe. <em>PLoS ONE</em> 7 (7): e41564. <a href="https://doi.org/10.1371/journal.pone.0041564">https://doi.org/10.1371/journal.pone.0041564</a>.</p>
</div>
<div>
<p>Kueckelhaus, Jan, Jasmin von Ehr, Vidhya M Ravi, Paulina Will, Kevin Joseph, Juergen Beck, Ulrich G Hofmann, Daniel Delev, Oliver Schnell, and Dieter Henrik Heiland. 2020. “Inferring spatially transient gene expression pattern from spatial transcriptomic studies.” <em>bioRxiv</em>, January, 2020.10.20.346544. <a href="https://doi.org/10.1101/2020.10.20.346544">https://doi.org/10.1101/2020.10.20.346544</a>.</p>
</div>
<div>
<p>Kumar, Sudhir, Karthik Jayaraman, Sethuraman Panchanathan, Rajalakshmi Gurunathan, Ana Marti-Subirana, and Stuart J Newfeld. 2002. “BEST: A Novel Computational Approach for Comparing Gene Expression Patterns From Early Stages of &lt;em&gt;Drosophila melanogaster&lt;/em&gt; Development.” <em>Genetics</em> 162 (4): 2037 LP–2047. <a href="http://www.genetics.org/content/162/4/2037.abstract">http://www.genetics.org/content/162/4/2037.abstract</a>.</p>
</div>
<div>
<p>Kumar, Sudhir, Charlotte Konikoff, Maxwell Sanderford, Li Liu, Stuart Newfeld, Jieping Ye, and Rob J. Kulathinal. 2017. “FlyExpress 7: An Integrated Discovery Platform To Study Coexpressed Genes Using in situ Hybridization Images in Drosophila.” <em>G3; Genes|Genomes|Genetics</em> 7 (8): 2791–7. <a href="https://doi.org/10.1534/g3.117.040345">https://doi.org/10.1534/g3.117.040345</a>.</p>
</div>
<div>
<p>Kvon, Evgeny Z., Tomas Kazmar, Gerald Stampfel, J. Omar Yáñez-Cuna, Michaela Pagani, Katharina Schernhuber, Barry J. Dickson, and Alexander Stark. 2014. “Genome-scale functional characterization of Drosophila developmental enhancers in vivo.” <em>Nature</em> 512 (7512): 91–95. <a href="https://doi.org/10.1038/nature13395">https://doi.org/10.1038/nature13395</a>.</p>
</div>
<div>
<p>Lacraz, Grégory P. A., Jan Philipp Junker, Monika M. Gladka, Bas Molenaar, Koen T. Scholman, Marta Vigil-Garcia, Danielle Versteeg, et al. 2017. “Tomo-Seq Identifies SOX9 as a Key Regulator of Cardiac Fibrosis During Ischemic Injury.” <em>Circulation</em> 136 (15): 1396–1409. <a href="https://doi.org/10.1161/CIRCULATIONAHA.117.027832">https://doi.org/10.1161/CIRCULATIONAHA.117.027832</a>.</p>
</div>
<div>
<p>La Manno, Gioele, Ruslan Soldatov, Amit Zeisel, Emelie Braun, Hannah Hochgerner, Viktor Petukhov, Katja Lidschreiber, et al. 2018. “RNA velocity of single cells.” <em>Nature</em> 560 (7719): 494–98. <a href="https://doi.org/10.1038/s41586-018-0414-6">https://doi.org/10.1038/s41586-018-0414-6</a>.</p>
</div>
<div>
<p>Landegren, U, R Kaiser, J Sanders, and L Hood. 1988. “A ligase-mediated gene detection technique.” <em>Science</em> 241 (4869): 1077 LP–1080. <a href="https://doi.org/10.1126/science.3413476">https://doi.org/10.1126/science.3413476</a>.</p>
</div>
<div>
<p>Lander, Eric S., Lauren M. Linton, Bruce Birren, Chad Nusbaum, Michael C. Zody, Jennifer Baldwin, Keri Devon, et al. 2001. “Initial sequencing and analysis of the human genome.” <em>Nature</em> 409 (6822): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.</p>
</div>
<div>
<p>Langer-Safer, P. R., M. Levine, and D. C. Ward. 1982. “Immunological method for mapping genes on Drosophila polytene chromosomes.” <em>Proceedings of the National Academy of Sciences</em> 79 (14): 4381–5. <a href="https://doi.org/10.1073/pnas.79.14.4381">https://doi.org/10.1073/pnas.79.14.4381</a>.</p>
</div>
<div>
<p>Larsson, Chatarina, Ida Grundberg, Ola Söderberg, and Mats Nilsson. 2010. “In situ detection and genotyping of individual mRNA molecules.” <em>Nature Methods</em> 7 (5): 395–97. <a href="https://doi.org/10.1038/nmeth.1448">https://doi.org/10.1038/nmeth.1448</a>.</p>
</div>
<div>
<p>Lebrigand, Kevin, Joseph Bergenstråhle, Kim Thrane, Annelie Mollbrink, Pascal Barbry, Rainer Waldmann, and Joakim Lundeberg. 2020. “The spatial landscape of gene expression isoforms in tissue sections.” <em>bioRxiv</em>, August, 2020.08.24.252296. <a href="https://doi.org/10.1101/2020.08.24.252296">https://doi.org/10.1101/2020.08.24.252296</a>.</p>
</div>
<div>
<p>Lee, Hower, Sergio Marco Salas, Daniel Gyllborg, and Mats Nilsson. 2020. “Direct RNA targeted transcriptomic profiling in tissue using Hybridization-based RNA In Situ Sequencing (HybRISS).” <em>bioRxiv</em>, January, 2020.12.02.408781. <a href="https://doi.org/10.1101/2020.12.02.408781">https://doi.org/10.1101/2020.12.02.408781</a>.</p>
</div>
<div>
<p>Lee, Je Hyuk, Evan R. Daugharthy, Jonathan Scheiman, Reza Kalhor, Thomas C. Ferrante, Richard Terry, Brian M. Turczyk, et al. 2015. “Fluorescent in situ sequencing (FISSEQ) of RNA for gene expression profiling in intact cells and tissues.” <em>Nature Protocols</em> 10 (3): 442–58. <a href="https://doi.org/10.1038/nprot.2014.191">https://doi.org/10.1038/nprot.2014.191</a>.</p>
</div>
<div>
<p>Lee, Je Hyuk, Evan R Daugharthy, Jonathan Scheiman, Reza Kalhor, Joyce L Yang, Thomas C Ferrante, Richard Terry, et al. 2014. “Highly Multiplexed Subcellular RNA Sequencing in Situ.” <em>Science</em> 343 (6177): 1360 LP–1363. <a href="https://doi.org/10.1126/science.1250212">https://doi.org/10.1126/science.1250212</a>.</p>
</div>
<div>
<p>Leighton, Philip A., Kevin J. Mitchell, Lisa V. Goodrich, Xiaowei Lu, Kathy Pinson, Paul Scherz, William C. Skarnes, and Marc Tessier-Lavigne. 2001. “Defining brain wiring patterns and mechanisms through gene trapping in mice.” <em>Nature</em> 410 (6825): 174–79. <a href="https://doi.org/10.1038/35065539">https://doi.org/10.1038/35065539</a>.</p>
</div>
<div>
<p>Lein, Ed, Lars E. Borm, and Sten Linnarsson. 2017. “The promise of spatial transcriptomics for neuroscience in the era of molecular cell typing.” <a href="https://doi.org/10.1126/science.aan6827">https://doi.org/10.1126/science.aan6827</a>.</p>
</div>
<div>
<p>Lein, Ed S. 2004. “Defining a Molecular Atlas of the Hippocampus Using DNA Microarrays and High-Throughput In Situ Hybridization.” <em>Journal of Neuroscience</em> 24 (15): 3879–89. <a href="https://doi.org/10.1523/JNEUROSCI.4710-03.2004">https://doi.org/10.1523/JNEUROSCI.4710-03.2004</a>.</p>
</div>
<div>
<p>Lein, Ed S., Michael J. Hawrylycz, Nancy Ao, Mikael Ayres, Amy Bensinger, Amy Bernard, Andrew F. Boe, et al. 2007. “Genome-wide atlas of gene expression in the adult mouse brain.” <em>Nature</em> 445 (7124): 168–76. <a href="https://doi.org/10.1038/nature05453">https://doi.org/10.1038/nature05453</a>.</p>
</div>
<div>
<p>Levsky, Jeffrey M. 2002. “Single-Cell Gene Expression Profiling.” <em>Science</em> 297 (5582): 836–40. <a href="https://doi.org/10.1126/science.1072241">https://doi.org/10.1126/science.1072241</a>.</p>
</div>
<div>
<p>Levy-Jurgenson, Alona, Xavier Tekpli, Vessela N Kristensen, and Zohar Yakhini. 2020. “Spatial transcriptomics inferred from pathology whole-slide images links tumor heterogeneity to survival in breast and lung cancer.” <em>Scientific Reports</em> 10 (1): 18802. <a href="https://doi.org/10.1038/s41598-020-75708-z">https://doi.org/10.1038/s41598-020-75708-z</a>.</p>
</div>
<div>
<p>Lécuyer, Eric, Hideki Yoshida, Neela Parthasarathy, Christina Alm, Tomas Babak, Tanja Cerovina, Timothy R. Hughes, Pavel Tomancak, and Henry M. Krause. 2007. “Global Analysis of mRNA Localization Reveals a Prominent Role in Organizing Cellular Architecture and Function.” <em>Cell</em> 131 (1): 174–87. <a href="https://doi.org/10.1016/j.cell.2007.08.003">https://doi.org/10.1016/j.cell.2007.08.003</a>.</p>
</div>
<div>
<p>Li, Junxiang, Haofei Luo, Rui Wang, Jidong Lang, Siyu Zhu, Zhenming Zhang, Jianhuo Fang, et al. 2016. “Systematic Reconstruction of Molecular Cascades Regulating GP Development Using Single-Cell RNA-Seq.” <em>Cell Reports</em> 15 (7): 1467–80. <a href="https://doi.org/10.1016/j.celrep.2016.04.043">https://doi.org/10.1016/j.celrep.2016.04.043</a>.</p>
</div>
<div>
<p>Li, Qiang, Zuwan Lin, Ren Liu, Xin Tang, Jiahao Huang, Yichun He, Haowen Zhou, et al. 2021. “&lt;em&gt;In situ&lt;/em&gt; electro-sequencing in three-dimensional tissues.” <em>bioRxiv</em>, January, 2021.04.22.440941. <a href="https://doi.org/10.1101/2021.04.22.440941">https://doi.org/10.1101/2021.04.22.440941</a>.</p>
</div>
<div>
<p>Li, Ying-Xin, Shuiwang Ji, Sudhir Kumar, Jieping Ye, and Zhi-Hua Zhou. 2009. “Drosophila Gene Expression Pattern Annotation through Multi-Instance Multi-Label Learning.” <em>IJCAI : Proceedings of the Conference</em> 2009 (January): 1445–50. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20824158%20http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC2932460">http://www.ncbi.nlm.nih.gov/pubmed/20824158 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC2932460</a>.</p>
</div>
<div>
<p>Li, Yujie, Hanbo Chen, Xi Jiang, Xiang Li, Jinglei Lv, Hanchuan Peng, Joe Z. Tsien, and Tianming Liu. 2017. “Discover mouse gene coexpression landscapes using dictionary learning and sparse coding.” <em>Brain Structure and Function</em> 222 (9): 4253–70. <a href="https://doi.org/10.1007/s00429-017-1460-9">https://doi.org/10.1007/s00429-017-1460-9</a>.</p>
</div>
<div>
<p>Li, Zhuliu, Tianci Song, Jeongsik Yong, and Rui Kuang. 2020. “Imputation of Spatially-resolved Transcriptomes by Graph-regularized Tensor Completion.” <em>bioRxiv</em>, January, 2020.08.05.237560. <a href="https://doi.org/10.1101/2020.08.05.237560">https://doi.org/10.1101/2020.08.05.237560</a>.</p>
</div>
<div>
<p>Liang, P, and A B Pardee. 1992. “Differential display of eukaryotic messenger RNA by means of the polymerase chain reaction.” <em>Science</em> 257 (5072): 967 LP–971. <a href="https://doi.org/10.1126/science.1354393">https://doi.org/10.1126/science.1354393</a>.</p>
</div>
<div>
<p>Liao, Jie, Xiaoyan Lu, Xin Shao, Ling Zhu, and Xiaohui Fan. 2020. “Uncovering an Organ’s Molecular Architecture at Single-Cell Resolution by Spatially Resolved Transcriptomics.” <em>Trends in Biotechnology</em>, June. <a href="https://doi.org/10.1016/j.tibtech.2020.05.006">https://doi.org/10.1016/j.tibtech.2020.05.006</a>.</p>
</div>
<div>
<p>Lignell, Antti, Laura Kerosuo, Sebastian J. Streichan, Long Cai, and Marianne E. Bronner. 2017. “Identification of a neural crest stem cell niche by Spatial Genomic Analysis.” <em>Nature Communications</em> 8 (1): 1830. <a href="https://doi.org/10.1038/s41467-017-01561-w">https://doi.org/10.1038/s41467-017-01561-w</a>.</p>
</div>
<div>
<p>Lis, John T., Jeffrey A. Simon, and Claudia A. Sutton. 1983. “New heat shock puffs and <span class="math inline">\(\beta\)</span>=galactosidase activity resulting from transformation of Drosophila with an hsp70-lacZ hybrid gene.” <em>Cell</em> 35 (2): 403–10. <a href="https://doi.org/10.1016/0092-8674(83)90173-3">https://doi.org/10.1016/0092-8674(83)90173-3</a>.</p>
</div>
<div>
<p>Liscovitch, Noa, Uri Shalit, and Gal Chechik. 2013. “FuncISH: learning a functional representation of neural ISH images.” <em>Bioinformatics</em> 29 (13): i36–i43. <a href="https://doi.org/10.1093/bioinformatics/btt207">https://doi.org/10.1093/bioinformatics/btt207</a>.</p>
</div>
<div>
<p>Littman, Russell, Zachary Hemminger, Robert Foreman, Douglas Arneson, Guanglin Zhang, Fernando Gómez-Pinilla, Xia Yang, and Roy Wollman. 2020. “JSTA: joint cell segmentation and cell type annotation for spatial transcriptomics.” <em>bioRxiv</em>, January, 2020.09.18.304147. <a href="https://doi.org/10.1101/2020.09.18.304147">https://doi.org/10.1101/2020.09.18.304147</a>.</p>
</div>
<div>
<p>Liu, Lin, Yinhu Li, Siliang Li, Ni Hu, Yimin He, Ray Pong, Danni Lin, Lihua Lu, and Maggie Law. 2012. “Comparison of Next-Generation Sequencing Systems.” Edited by P J Oefner. <em>Journal of Biomedicine and Biotechnology</em> 2012: 251364. <a href="https://doi.org/10.1155/2012/251364">https://doi.org/10.1155/2012/251364</a>.</p>
</div>
<div>
<p>Liu, Miao, Yanfang Lu, Bing Yang, Yanbo Chen, Jonathan S. D. Radda, Mengwei Hu, Samuel G. Katz, and Siyuan Wang. 2020. “Multiplexed imaging of nucleome architectures in single cells of mammalian tissue.” <em>Nature Communications</em> 11 (1): 1–14. <a href="https://doi.org/10.1038/s41467-020-16732-5">https://doi.org/10.1038/s41467-020-16732-5</a>.</p>
</div>
<div>
<p>Liu, Yang, Mingyu Yang, Yanxiang Deng, Graham Su, Archibald Enninful, Cindy C Guo, Toma Tebaldi, et al. 2020. “High-Spatial-Resolution Multi-Omics Sequencing via Deterministic Barcoding in Tissue.” <em>Cell</em> 183 (6): 1665–1681.e18. <a href="https://doi.org/https://doi.org/10.1016/j.cell.2020.10.026">https://doi.org/https://doi.org/10.1016/j.cell.2020.10.026</a>.</p>
</div>
<div>
<p>Liu, Zheng, S. Frank Yan, John R. Walker, Theresa A. Zwingman, Tao Jiang, Jing Li, and Yingyao Zhou. 2007. “Study of gene function based on spatial co-expression in a high-resolution mouse brain atlas.” <em>BMC Systems Biology</em> 1 (1): 19. <a href="https://doi.org/10.1186/1752-0509-1-19">https://doi.org/10.1186/1752-0509-1-19</a>.</p>
</div>
<div>
<p>Lizardi, Paul M., Xiaohua Huang, Zhengrong Zhu, Patricia Bray-Ward, David C. Thomas, and David C. Ward. 1998. “Mutation detection and single-molecule counting using isothermal rolling-circle amplification.” <em>Nature Genetics</em> 19 (3): 225–32. <a href="https://doi.org/10.1038/898">https://doi.org/10.1038/898</a>.</p>
</div>
<div>
<p>Lohoff, T, S Ghazanfar, A Missarova, N Koulena, N Pierson, J A Griffiths, E S Bardot, et al. 2020. “Highly multiplexed spatially resolved gene expression profiling of mouse organogenesis.” <em>bioRxiv</em>, January, 2020.11.20.391896. <a href="https://doi.org/10.1101/2020.11.20.391896">https://doi.org/10.1101/2020.11.20.391896</a>.</p>
</div>
<div>
<p>Long, W, T Li, Y Yang, and H -B. Shen. 2021. “FlyIT: Drosophila Embryogenesis Image Annotation based on Image Tiling and Convolutional Neural Networks.” <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em> 18 (1): 194–204. <a href="https://doi.org/10.1109/TCBB.2019.2935723">https://doi.org/10.1109/TCBB.2019.2935723</a>.</p>
</div>
<div>
<p>Long, Xi, Jennifer Colonell, Allan M Wong, Robert H Singer, and Timothée Lionnet. 2017. “Quantitative mRNA imaging throughout the entire Drosophila brain.” <em>Nature Methods</em> 14 (7): 703–6. <a href="https://doi.org/10.1038/nmeth.4309">https://doi.org/10.1038/nmeth.4309</a>.</p>
</div>
<div>
<p>Lopez, Romain, Achille Nazaret, Maxime Langevin, Jules Samaran, Jeffrey Regier, Michael I. Jordan, and Nir Yosef. 2019. “A joint model of unpaired data from scRNA-seq and spatial transcriptomics for imputing missing gene expression measurements,” May. <a href="http://arxiv.org/abs/1905.02269">http://arxiv.org/abs/1905.02269</a>.</p>
</div>
<div>
<p>Lovatt, Ditte, Brittani K. Ruble, Jaehee Lee, Hannah Dueck, Tae Kyung Kim, Stephen Fisher, Chantal Francis, et al. 2014. “Transcriptome in vivo analysis (TIVA) of spatially defined single cells in live tissue.” <em>Nature Methods</em> 11 (2): 190–96. <a href="https://doi.org/10.1038/nmeth.2804">https://doi.org/10.1038/nmeth.2804</a>.</p>
</div>
<div>
<p>Lovell, Peter V., Morgan Wirthlin, Taylor Kaser, Alexa A. Buckner, Julia B. Carleton, Brian R. Snider, Anne K. McHugh, Alexander Tolpygo, Partha P. Mitra, and Claudio V. Mello. 2020. “ZEBrA: Zebra finch Expression Brain Atlas—A resource for comparative molecular neuroanatomy and brain evolution studies.” <em>Journal of Comparative Neurology</em> 528 (12): 2099–2131. <a href="https://doi.org/10.1002/cne.24879">https://doi.org/10.1002/cne.24879</a>.</p>
</div>
<div>
<p>Lowe, David G. 2004. “Distinctive Image Features from Scale-Invariant Keypoints.” <em>International Journal of Computer Vision</em> 60 (2): 91–110. <a href="https://doi.org/10.1023/B:VISI.0000029664.99615.94">https://doi.org/10.1023/B:VISI.0000029664.99615.94</a>.</p>
</div>
<div>
<p>Lubeck, Eric, and Long Cai. 2012. “Single-cell systems biology by super-resolution imaging and combinatorial labeling.” <em>Nature Methods 2012 9:7</em> 9 (7): 743–48. <a href="https://doi.org/10.1038/nmeth.2069">https://doi.org/10.1038/nmeth.2069</a>.</p>
</div>
<div>
<p>Lubeck, Eric, Ahmet F. Coskun, Timur Zhiyentayev, Mubhij Ahmad, and Long Cai. 2014. “Single-cell in situ RNA profiling by sequential hybridization.” Nature Publishing Group. <a href="https://doi.org/10.1038/nmeth.2892">https://doi.org/10.1038/nmeth.2892</a>.</p>
</div>
<div>
<p>Luengo Hendriks, Cris L., Soile V. E. Keränen, Charless C. Fowlkes, Lisa Simirenko, Gunther H. Weber, Angela H. DePace, Clara Henriquez, et al. 2006. “Three-dimensional morphology and gene expression in the Drosophila blastoderm at cellular resolution I: Data acquisition pipeline.” <em>Genome Biology</em> 7 (12): R123. <a href="https://doi.org/10.1186/gb-2006-7-12-r123">https://doi.org/10.1186/gb-2006-7-12-r123</a>.</p>
</div>
<div>
<p>Lukan, Tjaša, Maruša Pompe‐Novak, Špela Baebler, Magda Tušek‐Žnidarič, Aleš Kladnik, Maja Križnik, Andrej Blejec, et al. 2020. “Precision transcriptomics of viral foci reveals the spatial regulation of immune‐signaling genes and identifies RBOHD as an important player in the incompatible interaction between potato virus Y and potato.” <em>The Plant Journal</em> 104 (3): 645–61. <a href="https://doi.org/10.1111/tpj.14953">https://doi.org/10.1111/tpj.14953</a>.</p>
</div>
<div>
<p>Lundberg, Emma, and Georg H. H. Borner. 2019. “Spatial proteomics: a powerful discovery tool for cell biology.” <em>Nature Reviews Molecular Cell Biology</em> 20 (5): 285–302. <a href="https://doi.org/10.1038/s41580-018-0094-y">https://doi.org/10.1038/s41580-018-0094-y</a>.</p>
</div>
<div>
<p>Lundmark, Anna, Natalija Gerasimcik, Tove Båge, Anders Jemt, Annelie Mollbrink, Fredrik Salmén, Joakim Lundeberg, and Tülay Yucel-Lindberg. 2018. “Gene expression profiling of periodontitis-affected gingival tissue by spatial transcriptomics.” <em>Scientific Reports</em> 8 (1): 9370. <a href="https://doi.org/10.1038/s41598-018-27627-3">https://doi.org/10.1038/s41598-018-27627-3</a>.</p>
</div>
<div>
<p>Luo, Lin, Ranelle C. Salunga, Hongqing Guo, Anton Bittner, K. C. Joy, Jose E. Galindo, Huinian Xiao, et al. 1999. “Gene expression profiles of laser-captured adjacent neuronal subtypes.” <em>Nature Medicine</em> 5 (1): 117–22. <a href="https://doi.org/10.1038/4806">https://doi.org/10.1038/4806</a>.</p>
</div>
<div>
<p>Lynch, Andrew S., David Briggs, and Ian A. Hope. 1995. “Developmental expression pattern screen for genes predicted in the C. elegans genome sequencing project.” <em>Nature Genetics</em> 11 (3): 309–13. <a href="https://doi.org/10.1038/ng1195-309">https://doi.org/10.1038/ng1195-309</a>.</p>
</div>
<div>
<p>Maaskola, Jonas, Ludvig Bergenstråhle, Aleksandra Jurek, José Fernández Navarro, Jens Lagergren, and Joakim Lundeberg. 2018. “Charting Tissue Expression Anatomy by Spatial Transcriptome Decomposition.” <em>bioRxiv</em>, December, 362624. <a href="https://doi.org/10.1101/362624">https://doi.org/10.1101/362624</a>.</p>
</div>
<div>
<p>Macevicz, Stephen. 1995. “DNA sequencing by parallel olgonucleotide extensions.” <a href="https://patents.google.com/patent/US5969119A/en">https://patents.google.com/patent/US5969119A/en</a>.</p>
</div>
<div>
<p>Macosko, Evan Z., Anindita Basu, Rahul Satija, James Nemesh, Karthik Shekhar, Melissa Goldman, Itay Tirosh, et al. 2015. “Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets.” <em>Cell</em> 161 (5): 1202–14. <a href="https://doi.org/10.1016/j.cell.2015.05.002">https://doi.org/10.1016/j.cell.2015.05.002</a>.</p>
</div>
<div>
<p>Manco, Rita, Inna Averbukh, Ziv Porat, Keren Bahar Halpern, Ido Amit, and Shalev Itzkovitz. 2020. “Clump sequencing exposes the spatial expression programs of intestinal secretory cells.” <em>bioRxiv</em>, August, 2020.08.05.237917. <a href="https://doi.org/10.1101/2020.08.05.237917">https://doi.org/10.1101/2020.08.05.237917</a>.</p>
</div>
<div>
<p>Maniatis, Silas, Tarmo Äijö, Sanja Vickovic, Catherine Braine, Kristy Kang, Annelie Mollbrink, Delphine Fagegaltier, et al. 2019. “Spatiotemporal dynamics of molecular pathology in amyotrophic lateral sclerosis.” <em>Science</em> 364 (6435): 89 LP–93. <a href="https://doi.org/10.1126/science.aav9776">https://doi.org/10.1126/science.aav9776</a>.</p>
</div>
<div>
<p>Manno, Gioele La, Kimberly Siletti, Alessandro Furlan, Daniel Gyllborg, Elin Vinsland, Christoffer Mattsson Langseth, Irina Khven, et al. 2020. “Molecular architecture of the developing mouse brain.” <em>bioRxiv</em>, July, 2020.07.02.184051. <a href="https://doi.org/10.1101/2020.07.02.184051">https://doi.org/10.1101/2020.07.02.184051</a>.</p>
</div>
<div>
<p>Mantri, Madhav, Gaetano J Scuderi, Roozbeh Abedini Nassab, Michael F Z Wang, David McKellar, Jonathan T Butcher, and Iwijn De Vlaminck. 2020. “Spatiotemporal single-cell RNA sequencing of developing hearts reveals interplay between cellular differentiation and morphogenesis.” <em>bioRxiv</em>, January, 2020.05.03.065102. <a href="https://doi.org/10.1101/2020.05.03.065102">https://doi.org/10.1101/2020.05.03.065102</a>.</p>
</div>
<div>
<p>Margaroli, C, P Benson, N S Sharma, M C Madison, S W Robison, N Arora, K Ton, et al. 2021. “Spatial mapping of SARS-CoV-2 and H1N1 Lung Injury Identifies Differential Transcriptional Signatures.” <em>Cell Reports. Medicine</em>, March, 100242. <a href="https://doi.org/10.1016/j.xcrm.2021.100242">https://doi.org/10.1016/j.xcrm.2021.100242</a>.</p>
</div>
<div>
<p>Marquart, Gregory D., Kathryn M. Tabor, Mary Brown, Jennifer L. Strykowski, Gaurav K. Varshney, Matthew C. LaFave, Thomas Mueller, Shawn M. Burgess, Shin-ichi Higashijima, and Harold A. Burgess. 2015. “A 3D Searchable Database of Transgenic Zebrafish Gal4 and Cre Lines for Functional Neuroanatomy Studies.” <em>Frontiers in Neural Circuits</em> 9 (November): 1–17. <a href="https://doi.org/10.3389/fncir.2015.00078">https://doi.org/10.3389/fncir.2015.00078</a>.</p>
</div>
<div>
<p>Martinelli, Sylvia D., Clive G. Brown, and Richard Durbin. 1997. “Gene expression and development databases forC. elegans.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 459–67. <a href="https://doi.org/10.1006/scdb.1997.0171">https://doi.org/10.1006/scdb.1997.0171</a>.</p>
</div>
<div>
<p>Maynard, Kristen R, Madhavi Tippani, Yoichiro Takahashi, BaDoi N Phan, Thomas M Hyde, Andrew E Jaffe, and Keri Martinowich. 2020. “dotdotdot: an automated approach to quantify multiplex single molecule fluorescent in situ hybridization (smFISH) images in complex tissues.” <em>Nucleic Acids Research</em> 48 (11): e66–e66. <a href="https://doi.org/10.1093/nar/gkaa312">https://doi.org/10.1093/nar/gkaa312</a>.</p>
</div>
<div>
<p>McKee, Adrienne E., Emmanuel Minet, Charlene Stern, Shervin Riahi, Charles D. Stiles, and Pamela A. Silver. 2005. “A genome-wide in situ hybridization map of RNA-binding proteins reveals anatomically restricted expression in the developing mouse brain.” <em>BMC Developmental Biology</em> 5 (1): 14. <a href="https://doi.org/10.1186/1471-213X-5-14">https://doi.org/10.1186/1471-213X-5-14</a>.</p>
</div>
<div>
<p>Medaglia, Chiara, Amir Giladi, Liat Stoler-Barak, Marco De Giovanni, Tomer Meir Salame, Adi Biram, Eyal David, et al. 2017. “Spatial reconstruction of immune niches by combining photoactivatable reporters and scRNA-seq.” <em>Science</em> 358 (6370): 1622–6. <a href="https://doi.org/10.1126/science.aao4277">https://doi.org/10.1126/science.aao4277</a>.</p>
</div>
<div>
<p>Meier-Ruge, W., W. Bielser, E. Remy, F. Hillenkamp, R. Nitsche, and R. Uns�ld. 1976. “The laser in the Lowry technique for microdissection of freeze-dried tissue slices.” <em>The Histochemical Journal</em> 8 (4): 387–401. <a href="https://doi.org/10.1007/BF01003828">https://doi.org/10.1007/BF01003828</a>.</p>
</div>
<div>
<p>Melsted, Páll, A Sina Booeshaghi, Lauren Liu, Fan Gao, Lambda Lu, Kyung Hoi (Joseph) Min, Eduardo da Veiga Beltrame, Kristján Eldjárn Hjörleifsson, Jase Gehring, and Lior Pachter. 2021. “Modular, efficient and constant-memory single-cell RNA-seq preprocessing.” <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/s41587-021-00870-2">https://doi.org/10.1038/s41587-021-00870-2</a>.</p>
</div>
<div>
<p>Merritt, Christopher R., Giang T. Ong, Sarah Church, Kristi Barker, Gary Geiss, Margaret Hoang, Jaemyeong Jung, et al. 2019. “High multiplex, digital spatial profiling of proteins and RNA in fixed tissue using genomic detection methods.” <em>bioRxiv</em> 38 (May): 559021. <a href="https://doi.org/10.1101/559021">https://doi.org/10.1101/559021</a>.</p>
</div>
<div>
<p>Middleton, Sarah A, James Eberwine, and Junhyong Kim. 2019. “Comprehensive catalog of dendritically localized mRNA isoforms from sub-cellular sequencing of single mouse neurons.” <em>BMC Biology</em> 17 (1): 5. <a href="https://doi.org/10.1186/s12915-019-0630-z">https://doi.org/10.1186/s12915-019-0630-z</a>.</p>
</div>
<div>
<p>Miller, Jeremy A., Song-Lin Ding, Susan M. Sunkin, Kimberly A. Smith, Lydia Ng, Aaron Szafer, Amanda Ebbert, et al. 2014. “Transcriptional landscape of the prenatal human brain.” <em>Nature</em> 508 (7495): 199–206. <a href="https://doi.org/10.1038/nature13185">https://doi.org/10.1038/nature13185</a>.</p>
</div>
<div>
<p>Moffitt, Jeffrey R., Dhananjay Bambah-Mukku, Stephen W. Eichhorn, Eric Vaughn, Karthik Shekhar, Julio D. Perez, Nimrod D. Rubinstein, et al. 2018. “Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aau5324">https://doi.org/10.1126/science.aau5324</a>.</p>
</div>
<div>
<p>Moffitt, Jeffrey R., Junjie Hao, Guiping Wang, Kok Hao Chen, Hazen P. Babcock, and Xiaowei Zhuang. 2016. “High-throughput single-cell gene-expression profiling with multiplexed error-robust fluorescence in situ hybridization.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1612826113">https://doi.org/10.1073/pnas.1612826113</a>.</p>
</div>
<div>
<p>Moncada, Reuben, Dalia Barkley, Florian Wagner, Marta Chiodin, Joseph C Devlin, Maayan Baron, Cristina H Hajdu, Diane M Simeone, and Itai Yanai. 2020. “Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas.” <em>Nature Biotechnology</em> 38 (3): 333–42. <a href="https://doi.org/10.1038/s41587-019-0392-8">https://doi.org/10.1038/s41587-019-0392-8</a>.</p>
</div>
<div>
<p>Moor, Andreas E., Matan Golan, Efi E. Massasa, Doron Lemze, Tomer Weizman, Rom Shenhav, Shaked Baydatch, et al. 2017. “Global mRNA polarization regulates translation efficiency in the intestinal epithelium.” <em>Science</em> 357 (6357): 1299–1303. <a href="https://doi.org/10.1126/science.aan2399">https://doi.org/10.1126/science.aan2399</a>.</p>
</div>
<div>
<p>Moor, Andreas E., Yotam Harnik, Shani Ben-Moshe, Efi E. Massasa, Milena Rozenberg, Raya Eilam, Keren Bahar Halpern, and Shalev Itzkovitz. 2018. “Spatial Reconstruction of Single Enterocytes Uncovers Broad Zonation along the Intestinal Villus Axis.” <em>Cell</em> 175 (4): 1156–1167.e15. <a href="https://doi.org/10.1016/j.cell.2018.08.063">https://doi.org/10.1016/j.cell.2018.08.063</a>.</p>
</div>
<div>
<p>Moor, Andreas E., and Shalev Itzkovitz. 2017. “Spatial transcriptomics: paving the way for tissue-level systems biology.” <em>Current Opinion in Biotechnology</em> 46 (August): 126–33. <a href="https://doi.org/10.1016/j.copbio.2017.02.004">https://doi.org/10.1016/j.copbio.2017.02.004</a>.</p>
</div>
<div>
<p>Mori, Tomoya, Haruka Takaoka, Junko Yamane, Cantas Alev, and Wataru Fujibuchi. 2019. “Novel computational model of gastrula morphogenesis to identify spatial discriminator genes by self-organizing map (SOM) clustering.” <em>Scientific Reports</em> 9 (1): 1–10. <a href="https://doi.org/10.1038/s41598-019-49031-1">https://doi.org/10.1038/s41598-019-49031-1</a>.</p>
</div>
<div>
<p>Mori, Tomoya, Junko Yamane, Kenta Kobayashi, Nobuko Taniyama, Takanori Tano, and Wataru Fujibuchi. 2017. “Development of 3D Tissue Reconstruction Method from Single-cell RNA-seq Data.” <em>Genomics and Computational Biology; Vol 3 No 1 (2017)</em>. <a href="https://doi.org/10.18547/gcb.2017.vol3.iss1.e53">https://doi.org/10.18547/gcb.2017.vol3.iss1.e53</a>.</p>
</div>
<div>
<p>Moris, Naomi, Kerim Anlas, Susanne C. van den Brink, Anna Alemany, Julia Schröder, Sabitri Ghimire, Tina Balayo, Alexander van Oudenaarden, and Alfonso Martinez Arias. 2020. “An in vitro model of early anteroposterior organization during human development.” <em>Nature</em> 582 (7812): 410–15. <a href="https://doi.org/10.1038/s41586-020-2383-9">https://doi.org/10.1038/s41586-020-2383-9</a>.</p>
</div>
<div>
<p>Morton, Matthew L., Xiaodong Bai, Callie R. Merry, Philip A. Linden, Ahmad M. Khalil, Rom S. Leidner, and Cheryl L. Thompson. 2014. “Identification of mRNAs and lincRNAs associated with lung cancer progression using next-generation RNA sequencing from laser micro-dissected archival FFPE tissue specimens.” <em>Lung Cancer</em> 85 (1): 31–39. <a href="https://doi.org/10.1016/j.lungcan.2014.03.020">https://doi.org/10.1016/j.lungcan.2014.03.020</a>.</p>
</div>
<div>
<p>Myers, Eugene W. 2000. “A Whole-Genome Assembly of Drosophila.” <em>Science</em> 287 (5461): 2196–2204. <a href="https://doi.org/10.1126/science.287.5461.2196">https://doi.org/10.1126/science.287.5461.2196</a>.</p>
</div>
<div>
<p>Nagalakshmi, Ugrappa, Zhong Wang, Karl Waern, Chong Shou, Debasish Raha, Mark Gerstein, and Michael Snyder. 2008. “The Transcriptional Landscape of the Yeast Genome Defined by RNA Sequencing.” <em>Science</em> 320 (5881): 1344–9. <a href="https://doi.org/10.1126/science.1158441">https://doi.org/10.1126/science.1158441</a>.</p>
</div>
<div>
<p>Nagarajan, Maxwell B, Augusto M Tentori, Wen Cai Zhang, Frank J Slack, and Patrick S Doyle. 2020. “Spatially resolved and multiplexed MicroRNA quantification from tissue using nanoliter well arrays.” <em>Microsystems &amp; Nanoengineering</em> 6 (1): 51. <a href="https://doi.org/10.1038/s41378-020-0169-8">https://doi.org/10.1038/s41378-020-0169-8</a>.</p>
</div>
<div>
<p>Nakamura, Toru, Yoichi Furukawa, Hidewaki Nakagawa, Tatsuhiko Tsunoda, Hiroaki Ohigashi, Kohei Murata, Osamu Ishikawa, et al. 2004. “Genome-wide cDNA microarray analysis of gene expression profiles in pancreatic cancers using populations of tumor cells and normal ductal epithelial cells selected for purity by laser microdissection.” <em>Oncogene</em> 23 (13): 2385–2400. <a href="https://doi.org/10.1038/sj.onc.1207392">https://doi.org/10.1038/sj.onc.1207392</a>.</p>
</div>
<div>
<p>Nakayama, Naomi, Juana M. Arroyo, Joseph Simorowski, Bruce May, Robert Martienssen, and Vivian F. Irish. 2005. “Gene Trap Lines Define Domains of Gene Regulation in Arabidopsis Petals and Stamens.” <em>The Plant Cell</em> 17 (9): 2486–2506. <a href="https://doi.org/10.1105/tpc.105.033985">https://doi.org/10.1105/tpc.105.033985</a>.</p>
</div>
<div>
<p>Navarro, José Fernández, Joel Sjöstrand, Fredrik Salmén, Joakim Lundeberg, and Patrik L. Ståhl. 2017. “ST Pipeline: an automated pipeline for spatial mapping of unique transcripts.” <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btx211">https://doi.org/10.1093/bioinformatics/btx211</a>.</p>
</div>
<div>
<p>Nederlof, P. M., S. van der Flier, J. Wiegant, A. K. Raap, H. J. Tanke, J. S. Ploem, and M. van der Ploeg. 1990. “Multiple fluorescence in situ hybridization.” <em>Cytometry</em> 11 (1): 126–31. <a href="https://doi.org/10.1002/cyto.990110115">https://doi.org/10.1002/cyto.990110115</a>.</p>
</div>
<div>
<p>Neubacher, Saskia, and Christoph Arenz. 2009. “Rolling-Circle Amplification: Unshared Advantages in miRNA Detection.” <em>ChemBioChem</em> 10 (8): 1289–91. <a href="https://doi.org/10.1002/cbic.200900116">https://doi.org/10.1002/cbic.200900116</a>.</p>
</div>
<div>
<p>Nichterwitz, Susanne, Geng Chen, Julio Aguila Benitez, Marlene Yilmaz, Helena Storvall, Ming Cao, Rickard Sandberg, Qiaolin Deng, and Eva Hedlund. 2016. “Laser capture microscopy coupled with Smart-seq2 for precise spatial transcriptomic profiling.” <em>Nature Communications</em> 7 (1): 12139. <a href="https://doi.org/10.1038/ncomms12139">https://doi.org/10.1038/ncomms12139</a>.</p>
</div>
<div>
<p>Nilsson, Mats, Helena Malmgren, Martina Samiotaki, Marek Kwiatkowski, B. Chowdhary, and Ulf Landegren. 1994. “Padlock probes: circularizing oligonucleotides for localized DNA detection.” <em>Science</em> 265 (5181): 2085–8. <a href="https://doi.org/10.1126/science.7522346">https://doi.org/10.1126/science.7522346</a>.</p>
</div>
<div>
<p>Nitzan, Mor, Nikos Karaiskos, Nir Friedman, and Nikolaus Rajewsky. 2019. “Gene expression cartography.” <em>Nature</em> 576 (7785): 132–37. <a href="https://doi.org/10.1038/s41586-019-1773-3">https://doi.org/10.1038/s41586-019-1773-3</a>.</p>
</div>
<div>
<p>Noto, Torben, Derrick Barnagian, and Jason B. Castro. 2017. “Genome-scale investigation of olfactory system spatial heterogeneity.” Edited by Hiroaki Matsunami. <em>PLOS ONE</em> 12 (5): e0178087. <a href="https://doi.org/10.1371/journal.pone.0178087">https://doi.org/10.1371/journal.pone.0178087</a>.</p>
</div>
<div>
<p>Ohyama, H., X. Zhang, Y. Kohno, I. Alevizos, M. Posner, D. T. Wong, and R. Todd. 2000. “Laser Capture Microdissection-Generated Target Sample for High-Density Oligonucleotide Array Hybridization.” <em>BioTechniques</em> 29 (3): 530–36. <a href="https://doi.org/10.2144/00293st05">https://doi.org/10.2144/00293st05</a>.</p>
</div>
<div>
<p>Oka, Yuma, and Thomas N Sato. 2015. “Whole-mount single molecule FISH method for zebrafish embryo.” <em>Scientific Reports</em> 5 (February): 8571. <a href="https://doi.org/10.1038/srep08571">https://doi.org/10.1038/srep08571</a>.</p>
</div>
<div>
<p>Okamura-Oho, Yuko, Kazuro Shimokawa, Satoko Takemoto, Asami Hirakiyama, Sakiko Nakamura, Yuki Tsujimura, Masaomi Nishimura, et al. 2012. “Transcriptome Tomography for Brain Analysis in the Web-Accessible Anatomical Space.” Edited by Satoru Hayasaka. <em>PLoS ONE</em> 7 (9): e45373. <a href="https://doi.org/10.1371/journal.pone.0045373">https://doi.org/10.1371/journal.pone.0045373</a>.</p>
</div>
<div>
<p>O’Kane, C. J., and W. J. Gehring. 1987. “Detection in situ of genomic regulatory elements in Drosophila.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 84 (24): 9123–7. <a href="https://doi.org/10.1073/pnas.84.24.9123">https://doi.org/10.1073/pnas.84.24.9123</a>.</p>
</div>
<div>
<p>Ortiz, Cantin, Jose Fernandez Navarro, Aleksandra Jurek, Antje Märtin, Joakim Lundeberg, and Konstantinos Meletis. 2020. “Molecular atlas of the adult mouse brain.” <em>Science Advances</em> 6 (26): eabb3446. <a href="https://doi.org/10.1126/sciadv.abb3446">https://doi.org/10.1126/sciadv.abb3446</a>.</p>
</div>
<div>
<p>Palla, Giovanni, Hannah Spitzer, Michal Klein, David Fischer, Anna Christina Schaar, Louis Benedikt Kuemmerle, Sergei Rybakov, et al. 2021. “Squidpy: a scalable framework for spatial single cell analysis.” <em>bioRxiv</em>, January, 2021.02.19.431994. <a href="https://doi.org/10.1101/2021.02.19.431994">https://doi.org/10.1101/2021.02.19.431994</a>.</p>
</div>
<div>
<p>Pan, Jia-Yu, André Guilherme, Ribeiro Balan, Eric P. Xing, Agma Juci Machado Traina, and Christos Faloutsos. 2006. “Automatic mining of fruit fly embryo images.” In <em>Proceedings of the 12th Acm Sigkdd International Conference on Knowledge Discovery and Data Mining - Kdd ’06</em>, 2006:693. New York, New York, USA: ACM Press. <a href="https://doi.org/10.1145/1150402.1150489">https://doi.org/10.1145/1150402.1150489</a>.</p>
</div>
<div>
<p>Park, Jeongbin, Wonyl Choi, Sebastian Tiesmeyer, Brian Long, Lars E. Borm, Emma Garren, Thuc Nghi Nguyen, et al. 2019. “Segmentation-free inference of cell types from in situ transcriptomics data.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/800748">https://doi.org/10.1101/800748</a>.</p>
</div>
<div>
<p>Park, Jiwoon, Jonathan Foox, Tyler Hether, David Danko, Sarah Warren, Youngmi Kim, Jason Reeves, et al. 2021. “Systemic Tissue and Cellular Disruption from SARS-CoV-2 Infection revealed in COVID-19 Autopsies and Spatial Omics Tissue Maps.” <em>bioRxiv</em>, January, 2021.03.08.434433. <a href="https://doi.org/10.1101/2021.03.08.434433">https://doi.org/10.1101/2021.03.08.434433</a>.</p>
</div>
<div>
<p>Partel, Gabriele, Markus Hilscher, Giorgia Milli, Leslie Solorzano, Anna Klemm, Mats Nilsson, and Carolina Wählby. 2019. “Identification of spatial compartments in tissue from in situ sequencing data.” <em>bioRxiv</em>, September, 765842. <a href="https://doi.org/10.1101/765842">https://doi.org/10.1101/765842</a>.</p>
</div>
<div>
<p>Partel, Gabriele, and Carolina Wählby. 2020. “Spage2vec: Unsupervised detection of spatial gene expression constellations.” <em>bioRxiv</em>, February, 2020.02.12.945345. <a href="https://doi.org/10.1101/2020.02.12.945345">https://doi.org/10.1101/2020.02.12.945345</a>.</p>
</div>
<div>
<p>Parthasarathy, Raghuveer. 2012. “Rapid, accurate particle tracking by calculation of radial symmetry centers.” <em>Nature Methods</em> 9 (7): 724–26. <a href="https://doi.org/10.1038/nmeth.2071">https://doi.org/10.1038/nmeth.2071</a>.</p>
</div>
<div>
<p>Patrushev, Ilya, Christina James-Zorn, Aldo Ciau-Uitz, Roger Patient, and Michael J. Gilchrist. 2018. “New methods for computational decomposition of whole-mount in situ images enable effective curation of a large, highly redundant collection of Xenopus images.” <em>PLoS Computational Biology</em> 14 (8): e1006077. <a href="https://doi.org/10.1371/journal.pcbi.1006077">https://doi.org/10.1371/journal.pcbi.1006077</a>.</p>
</div>
<div>
<p>Peng, Guangdun, Shengbao Suo, Jun Chen, Weiyang Chen, Chang Liu, Fang Yu, Ran Wang, et al. 2016. “Spatial Transcriptome for the Molecular Annotation of Lineage Fates and Cell Identity in Mid-gastrula Mouse Embryo.” <em>Developmental Cell</em> 36 (6): 681–97. <a href="https://doi.org/10.1016/j.devcel.2016.02.020">https://doi.org/10.1016/j.devcel.2016.02.020</a>.</p>
</div>
<div>
<p>Pennington, Jeffrey, Richard Socher, and Christopher D Manning. n.d. “GloVe: Global Vectors for Word Representation.”</p>
</div>
<div>
<p>Pettit, Jean-Baptiste, Raju Tomer, Kaia Achim, Sylvia Richardson, Lamiae Azizi, and John Marioni. 2014. “Identifying Cell Types from Spatially Referenced Single-Cell Expression Datasets.” Edited by Quaid Morris. <em>PLoS Computational Biology</em> 10 (9): e1003824. <a href="https://doi.org/10.1371/journal.pcbi.1003824">https://doi.org/10.1371/journal.pcbi.1003824</a>.</p>
</div>
<div>
<p>Petukhov, Viktor, Ruslan A Soldatov, Konstantin Khodosevich, and Peter V Kharchenko. 2020. “Bayesian segmentation of spatially resolved transcriptomics data.” <em>bioRxiv</em>, January, 2020.10.05.326777. <a href="https://doi.org/10.1101/2020.10.05.326777">https://doi.org/10.1101/2020.10.05.326777</a>.</p>
</div>
<div>
<p>Pérez-Martín, Fernando, Fernando J. Yuste-Lisbona, Benito Pineda, María Pilar Angarita-Díaz, Begoña García-Sogo, Teresa Antón, Sibilla Sánchez, et al. 2017. “A collection of enhancer trap insertional mutants for functional genomics in tomato.” <em>Plant Biotechnology Journal</em> 15 (11): 1439–52. <a href="https://doi.org/10.1111/pbi.12728">https://doi.org/10.1111/pbi.12728</a>.</p>
</div>
<div>
<p>Pham, Duy, Xiao Tan, Jun Xu, Laura F Grice, Pui Yeng Lam, Arti Raghubar, Jana Vukovic, Marc J Ruitenberg, and Quan Nguyen. 2020. “stLearn: integrating spatial location, tissue morphology and gene expression to find cell types, cell-cell interactions and spatial trajectories within undissociated tissues.” <em>bioRxiv</em>, January, 2020.05.31.125658. <a href="https://doi.org/10.1101/2020.05.31.125658">https://doi.org/10.1101/2020.05.31.125658</a>.</p>
</div>
<div>
<p>Player, Audrey N., Lu Ping Shen, Daryn Kenny, Vincent P. Antao, and Janice A. Kolberg. 2001. “Single-copy gene detection using branched DNA (bDNA) in situ hybridization.” <em>Journal of Histochemistry and Cytochemistry</em> 49 (5): 603–11. <a href="https://doi.org/10.1177/002215540104900507">https://doi.org/10.1177/002215540104900507</a>.</p>
</div>
<div>
<p>Plouhinec, Jean-Louis, Sofía Medina-Ruiz, Caroline Borday, Elsa Bernard, Jean-Philippe Vert, Michael B. Eisen, Richard M. Harland, and Anne H. Monsoro-Burq. 2017. “A molecular atlas of the developing ectoderm defines neural, neural crest, placode, and nonneural progenitor identity in vertebrates.” Edited by James Briscoe. <em>PLOS Biology</em> 15 (10): e2004045. <a href="https://doi.org/10.1371/journal.pbio.2004045">https://doi.org/10.1371/journal.pbio.2004045</a>.</p>
</div>
<div>
<p>Pruteanu-Malinici, Iulian, Daniel L. Mace, and Uwe Ohler. 2011. “Automatic Annotation of Spatial Expression Patterns via Sparse Bayesian Factor Models.” Edited by Joel S. Bader. <em>PLoS Computational Biology</em> 7 (7): e1002098. <a href="https://doi.org/10.1371/journal.pcbi.1002098">https://doi.org/10.1371/journal.pcbi.1002098</a>.</p>
</div>
<div>
<p>Puchalski, Ralph B, Nameeta Shah, Jeremy Miller, Rachel Dalley, Steve R Nomura, Jae-Guen Yoon, Kimberly A Smith, et al. 2018. “An anatomic transcriptional atlas of human glioblastoma.” <em>Science</em> 360 (6389): 660 LP–663. <a href="https://doi.org/10.1126/science.aaf2666">https://doi.org/10.1126/science.aaf2666</a>.</p>
</div>
<div>
<p>Puniyani, Kriti, and Eric P. Xing. 2013. “GINI: From ISH Images to Gene Interaction Networks.” Edited by Gal Chechik. <em>PLoS Computational Biology</em> 9 (10): e1003227. <a href="https://doi.org/10.1371/journal.pcbi.1003227">https://doi.org/10.1371/journal.pcbi.1003227</a>.</p>
</div>
<div>
<p>Qian, Xiaoyan, Kenneth D. Harris, Thomas Hauling, Dimitris Nicoloutsopoulos, Ana B. Muñoz-Manchado, Nathan Skene, Jens Hjerling-Leffler, and Mats Nilsson. 2020. “Probabilistic cell typing enables fine mapping of closely related cell types in situ.” <em>Nature Methods</em> 17 (1): 101–6. <a href="https://doi.org/10.1038/s41592-019-0631-4">https://doi.org/10.1038/s41592-019-0631-4</a>.</p>
</div>
<div>
<p>Raj, Arjun, Patrick van den Bogaard, Scott A. Rifkin, Alexander van Oudenaarden, and Sanjay Tyagi. 2008. “Imaging individual mRNA molecules using multiple singly labeled probes.” <em>Nature Methods</em> 5 (10): 877–79. <a href="https://doi.org/10.1038/nmeth.1253">https://doi.org/10.1038/nmeth.1253</a>.</p>
</div>
<div>
<p>Ramsköld, Daniel, Shujun Luo, Yu-Chieh Wang, Robin Li, Qiaolin Deng, Omid R Faridani, Gregory A Daniels, et al. 2012. “Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells.” <em>Nature Biotechnology</em> 30 (8): 777–82. <a href="https://doi.org/10.1038/nbt.2282">https://doi.org/10.1038/nbt.2282</a>.</p>
</div>
<div>
<p>Ren, Xianwen, Guojie Zhong, Qiming Zhang, Lei Zhang, Yujie Sun, and Zemin Zhang. 2020. “Reconstruction of cell spatial organization based on ligand-receptor mediated self-assembly.” <em>bioRxiv</em>, January, 2020.02.13.948521. <a href="https://doi.org/10.1101/2020.02.13.948521">https://doi.org/10.1101/2020.02.13.948521</a>.</p>
</div>
<div>
<p>Righelli, Dario, Lukas M Weber, Helena L Crowell, Brenda Pardo, Leonardo Collado-Torres, Shila Ghazanfar, Aaron T L Lun, Stephanie C Hicks, and Davide Risso. 2021. “SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor.” <em>bioRxiv</em>, January, 2021.01.27.428431. <a href="https://doi.org/10.1101/2021.01.27.428431">https://doi.org/10.1101/2021.01.27.428431</a>.</p>
</div>
<div>
<p>Ringwald, Martin, Richard Baldock, Jonathan Bard, Matthew Kaufman, Janan T. Eppig, Joel E. Richardson, Joseph H. Nadeau, and Duncan Davidson. 1994. “A database for mouse development.” <em>Science</em> 265 (5181): 2033–4. <a href="https://doi.org/10.1126/science.8091224">https://doi.org/10.1126/science.8091224</a>.</p>
</div>
<div>
<p>Ringwald, Martin, Geoffrey L Davis, Alex G Smith, Laura E Trepanier, Dale A Begley, Joel E Richardson, and Janan T Eppig. 1997. “The mouse gene expression database GXD.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 489–97. <a href="https://doi.org/https://doi.org/10.1006/scdb.1997.0177">https://doi.org/https://doi.org/10.1006/scdb.1997.0177</a>.</p>
</div>
<div>
<p>Ringwald, Martin, Mary E Mangan, Janan T Eppig, James A Kadin, and Joel E Richardson. 1999. “GXD: a Gene Expression Database for the laboratory mouse.” <em>Nucleic Acids Research</em> 27 (1): 106–12. <a href="https://doi.org/10.1093/nar/27.1.106">https://doi.org/10.1093/nar/27.1.106</a>.</p>
</div>
<div>
<p>Rizvi, Abbas H, Pablo G Camara, Elena K Kandror, Thomas J Roberts, Ira Schieren, Tom Maniatis, and Raul Rabadan. 2017. “Single-cell topological RNA-seq analysis reveals insights into cellular differentiation and development.” <em>Nature Biotechnology</em> 35 (6): 551–60. <a href="https://doi.org/10.1038/nbt.3854">https://doi.org/10.1038/nbt.3854</a>.</p>
</div>
<div>
<p>Roberts, Kenny, Alexander Aivazidis, Vitalii Kleshchevnikov, Tong Li, Robin Fropf, Michael Rhodes, Joseph M Beechem, Martin Hemberg, and Omer Ali Bayraktar. 2021. “Transcriptome-wide spatial RNA profiling maps the cellular architecture of the developing human neocortex.” <em>bioRxiv</em>, January, 2021.03.20.436265. <a href="https://doi.org/10.1101/2021.03.20.436265">https://doi.org/10.1101/2021.03.20.436265</a>.</p>
</div>
<div>
<p>Roberts, Margaret E., Brandon M. Stewart, and Dustin Tingley. 2019. “Stm: An R package for structural topic models.” <em>Journal of Statistical Software</em> 91. <a href="https://doi.org/10.18637/jss.v091.i02">https://doi.org/10.18637/jss.v091.i02</a>.</p>
</div>
<div>
<p>Rodriques, Samuel G., Robert R. Stickels, Aleksandrina Goeva, Carly A. Martin, Evan Murray, Charles R. Vanderburg, Joshua Welch, Linlin M. Chen, Fei Chen, and Evan Z. Macosko. 2019. “Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution.” <em>Science</em>. <a href="https://doi.org/10.1126/science.aaw1219">https://doi.org/10.1126/science.aaw1219</a>.</p>
</div>
<div>
<p>Ronneberger, Olaf, Philipp Fischer, and Thomas Brox. 2015. “U-Net: Convolutional Networks for Biomedical Image Segmentation BT - Medical Image Computing and Computer-Assisted Intervention – MICCAI 2015.” In, edited by Nassir Navab, Joachim Hornegger, William M Wells, and Alejandro F Frangi, 234–41. Cham: Springer International Publishing.</p>
</div>
<div>
<p>Rosen, Barry, and Rosa S. P. Beddington. 1993. “Whole-mount in situ hybridization in the mouse embryo: gene expression in three dimensions.” <em>Trends in Genetics</em> 9 (5): 162–67. <a href="https://doi.org/10.1016/0168-9525(93)90162-B">https://doi.org/10.1016/0168-9525(93)90162-B</a>.</p>
</div>
<div>
<p>Rödelsperger, Christian, Annabel Ebbing, Devansh Raj Sharma, Misako Okumura, Ralf J Sommer, and Hendrik C Korswagen. 2020. “Spatial transcriptomics of nematodes identifies sperm cells as a source of genomic novelty and rapid evolution.” <em>Molecular Biology and Evolution</em>, August. <a href="https://doi.org/10.1093/molbev/msaa207">https://doi.org/10.1093/molbev/msaa207</a>.</p>
</div>
<div>
<p>Saelens, Wouter, Robrecht Cannoodt, Helena Todorov, and Yvan Saeys. 2019. “A comparison of single-cell trajectory inference methods.” <em>Nature Biotechnology</em> 37 (5): 547–54. <a href="https://doi.org/10.1038/s41587-019-0071-9">https://doi.org/10.1038/s41587-019-0071-9</a>.</p>
</div>
<div>
<p>Salgado, David, Gregory Gimenez, François Coulier, and Christophe Marcelle. 2008. “COMPARE, a multi-organism system for cross-species data comparison and transfer of information.” <em>Bioinformatics</em> 24 (3): 447–49. <a href="https://doi.org/10.1093/bioinformatics/btm599">https://doi.org/10.1093/bioinformatics/btm599</a>.</p>
</div>
<div>
<p>Samacoits, Aubin, Racha Chouaib, Adham Safieddine, Abdel-Meneem Traboulsi, Wei Ouyang, Christophe Zimmer, Marion Peter, Edouard Bertrand, Thomas Walter, and Florian Mueller. 2018. “A computational framework to study sub-cellular RNA localization.” <em>Nature Communications</em> 9 (1): 4584. <a href="https://doi.org/10.1038/s41467-018-06868-w">https://doi.org/10.1038/s41467-018-06868-w</a>.</p>
</div>
<div>
<p>Satija, Rahul, Jeffrey A. Farrell, David Gennert, Alexander F. Schier, and Aviv Regev. 2015. “Spatial reconstruction of single-cell gene expression data.” <em>Nature Biotechnology</em> 33 (5): 495–502. <a href="https://doi.org/10.1038/nbt.3192">https://doi.org/10.1038/nbt.3192</a>.</p>
</div>
<div>
<p>Satou, Yutaka, Naohito Takatori, Lixy Yamada, Yasuaki Mochizuki, Makoto Hamaguchi, Hisayoshi Ishikawa, Shota Chiba, et al. 2001. “Gene expression profiles in &lt;em&gt;Ciona intestinalis&lt;/em&gt; tailbud embryos.” <em>Development</em> 128 (15): 2893 LP–2904. <a href="http://dev.biologists.org/content/128/15/2893.abstract">http://dev.biologists.org/content/128/15/2893.abstract</a>.</p>
</div>
<div>
<p>Saviano, Antonio, Neil C. Henderson, and Thomas F. Baumert. 2020. “Single-cell genomics and spatial transcriptomics: Discovery of novel cell states and cellular interactions in liver physiology and disease biology.” <em>Journal of Hepatology</em> 73 (5): 1219–30. <a href="https://doi.org/10.1016/j.jhep.2020.06.004">https://doi.org/10.1016/j.jhep.2020.06.004</a>.</p>
</div>
<div>
<p>Schede, Halima Hannah, Christian Gabriel Schneider, Johanna Stergiadou, Lars E Borm, Anurag Ranjak, Tracy M Yamawaki, Fabrice P A David, et al. 2020. “Spatial tissue profiling by imaging-free molecular tomography.” <em>bioRxiv</em>, August, 2020.08.04.235655. <a href="https://doi.org/10.1101/2020.08.04.235655">https://doi.org/10.1101/2020.08.04.235655</a>.</p>
</div>
<div>
<p>Schwaber, Jessica, Stacey Andersen, and Lars Nielsen. 2019. “Shedding light: The importance of reverse transcription efficiency standards in data interpretation.” <em>Biomolecular Detection and Quantification</em> 17 (March): 100077. <a href="https://doi.org/10.1016/j.bdq.2018.12.002">https://doi.org/10.1016/j.bdq.2018.12.002</a>.</p>
</div>
<div>
<p>Seydoux, G, and A Fire. 1994. “Soma-germline asymmetry in the distributions of embryonic RNAs in Caenorhabditis elegans.” <em>Development</em> 120 (10): 2823 LP–2834. <a href="http://dev.biologists.org/content/120/10/2823.abstract">http://dev.biologists.org/content/120/10/2823.abstract</a>.</p>
</div>
<div>
<p>Sgroi, Dennis C, Sarena Teng, Greg Robinson, Rebbecca LeVangie, James R Hudson, and Abdel G Elkahloun. 1999. “&lt;em&gt;In Vivo&lt;/em&gt; Gene Expression Profile Analysis of Human Breast Cancer Progression.” <em>Cancer Research</em> 59 (22): 5656 LP–5661. <a href="http://cancerres.aacrjournals.org/content/59/22/5656.abstract">http://cancerres.aacrjournals.org/content/59/22/5656.abstract</a>.</p>
</div>
<div>
<p>Shah, Sheel, Eric Lubeck, Wen Zhou, and Long Cai. 2016. “In Situ Transcription Profiling of Single Cells Reveals Spatial Organization of Cells in the Mouse Hippocampus.” <em>Neuron</em>. <a href="https://doi.org/10.1016/j.neuron.2016.10.001">https://doi.org/10.1016/j.neuron.2016.10.001</a>.</p>
</div>
<div>
<p>Shah, Sheel, Yodai Takei, Wen Zhou, Eric Lubeck, Jina Yun, Chee Huat Linus Eng, Noushin Koulena, et al. 2018. “Dynamics and Spatial Genomics of the Nascent Transcriptome by Intron seqFISH.” <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2018.05.035">https://doi.org/10.1016/j.cell.2018.05.035</a>.</p>
</div>
<div>
<p>Sharma, Ankur, Justine Jia Wen Seow, Charles-Antoine Dutertre, Rhea Pai, Camille Blériot, Archita Mishra, Regina Men Men Wong, et al. 2020. “Onco-fetal Reprogramming of Endothelial Cells Drives Immunosuppressive Macrophages in Hepatocellular Carcinoma.” <em>Cell</em> 183 (2): 377–394.e21. <a href="https://doi.org/10.1016/j.cell.2020.08.040">https://doi.org/10.1016/j.cell.2020.08.040</a>.</p>
</div>
<div>
<p>Shcherbatyy, Volodymyr, James Carson, Murat Yaylaoglu, Katharina Jäckle, Frauke Grabbe, Maren Brockmeyer, Halenur Yavuz, and Gregor Eichele. 2015. “A Digital Atlas of Ion Channel Expression Patterns in the Two-Week-Old Rat Brain.” <em>Neuroinformatics</em> 13 (1): 111–25. <a href="https://doi.org/10.1007/s12021-014-9247-0">https://doi.org/10.1007/s12021-014-9247-0</a>.</p>
</div>
<div>
<p>Shendure, Jay. 2005. “Accurate Multiplex Polony Sequencing of an Evolved Bacterial Genome.” <em>Science</em> 309 (5741): 1728–32. <a href="https://doi.org/10.1126/science.1117389">https://doi.org/10.1126/science.1117389</a>.</p>
</div>
<div>
<p>Shi, Hao, Qiaojuan Shi, Benjamin Grodner, Joan Sesing Lenz, Warren R Zipfel, Ilana Lauren Brito, and Iwijn De Vlaminck. 2020. “Highly multiplexed spatial mapping of microbial communities.” <em>Nature</em> 588 (7839): 676–81. <a href="https://doi.org/10.1038/s41586-020-2983-4">https://doi.org/10.1038/s41586-020-2983-4</a>.</p>
</div>
<div>
<p>Siegert, Sandra, Brigitte Gross Scherf, Karina Del Punta, Nick Didkovsky, Nathaniel Heintz, and Botond Roska. 2009. “Genetic address book for retinal cell types.” <em>Nature Neuroscience</em> 12 (9): 1197–1204. <a href="https://doi.org/10.1038/nn.2370">https://doi.org/10.1038/nn.2370</a>.</p>
</div>
<div>
<p>Singer, R. H., and D. C. Ward. 1982. “Actin gene expression visualized in chicken muscle tissue culture by using in situ hybridization with a biotinated nucleotide analog.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 79 (23 I): 7331–5. <a href="https://doi.org/10.1073/pnas.79.23.7331">https://doi.org/10.1073/pnas.79.23.7331</a>.</p>
</div>
<div>
<p>Singh, Ram P., Vanessa M. Brown, Abhijit Chaudhari, Arshad H. Khan, Alex Ossadtchi, Daniel M. Sforza, A.Ken Meadors, Simon R. Cherry, Richard M. Leahy, and Desmond J. Smith. 2003. “High-resolution voxelation mapping of human and rodent brain gene expression.” <em>Journal of Neuroscience Methods</em> 125 (1-2): 93–101. <a href="https://doi.org/10.1016/S0165-0270(03)00045-1">https://doi.org/10.1016/S0165-0270(03)00045-1</a>.</p>
</div>
<div>
<p>Skarnes, William C., Julie E. Moss, Stella M. Hurtley, and Rosa S. P. Beddington. 1995. “Capturing genes encoding membrane and secreted proteins important for mouse development.” <em>Proceedings of the National Academy of Sciences</em> 92 (14): 6592–6. <a href="https://doi.org/10.1073/pnas.92.14.6592">https://doi.org/10.1073/pnas.92.14.6592</a>.</p>
</div>
<div>
<p>Skarnes, William C., Barry Rosen, Anthony P. West, Manousos Koutsourakis, Wendy Bushell, Vivek Iyer, Alejandro O. Mujica, et al. 2011. “A conditional knockout resource for the genome-wide study of mouse gene function.” <em>Nature</em> 474 (7351): 337–42. <a href="https://doi.org/10.1038/nature10163">https://doi.org/10.1038/nature10163</a>.</p>
</div>
<div>
<p>Smith, Constance M, Terry F. Hayamizu, Jacqueline H. Finger, Susan M. Bello, Ingeborg J. McCright, Jingxia Xu, Richard M. Baldarelli, et al. 2019. “The mouse Gene Expression Database (GXD): 2019 update.” <em>Nucleic Acids Research</em> 47 (D1): D774–D779. <a href="https://doi.org/10.1093/nar/gky922">https://doi.org/10.1093/nar/gky922</a>.</p>
</div>
<div>
<p>Sountoulidis, Alexandros, Andreas Liontos, Hong Phuong Nguyen, Alexandra B Firsova, Athanasios Fysikopoulos, Xiaoyan Qian, Werner Seeger, Erik Sundström, Mats Nilsson, and Christos Samakovlis. 2020. “SCRINSHOT, a spatial method for single-cell resolution mapping of cell states in tissue sections.” <em>bioRxiv</em>, February, 2020.02.07.938571. <a href="https://doi.org/10.1101/2020.02.07.938571">https://doi.org/10.1101/2020.02.07.938571</a>.</p>
</div>
<div>
<p>“SpaRTAN.” n.d. Accessed October 7, 2020. <a href="https://www.ncl.ac.uk/gcf/spatiallyresolvedtranscriptomics/">https://www.ncl.ac.uk/gcf/spatiallyresolvedtranscriptomics/</a>.</p>
</div>
<div>
<p>Spradling, A., and G. Rubin. 1982. “Transposition of cloned P elements into Drosophila germ line chromosomes.” <em>Science</em> 218 (4570): 341–47. <a href="https://doi.org/10.1126/science.6289435">https://doi.org/10.1126/science.6289435</a>.</p>
</div>
<div>
<p>Sprague, Judy. 2003. “The Zebrafish Information Network (ZFIN): the zebrafish model organism database.” <em>Nucleic Acids Research</em> 31 (1): 241–43. <a href="https://doi.org/10.1093/nar/gkg027">https://doi.org/10.1093/nar/gkg027</a>.</p>
</div>
<div>
<p>Srivastava, Avi, Laraib Malik, Tom Smith, Ian Sudbery, and Rob Patro. 2019. “Alevin efficiently estimates accurate gene abundances from dscRNA-seq data.” <em>Genome Biology</em> 20 (1): 65. <a href="https://doi.org/10.1186/s13059-019-1670-y">https://doi.org/10.1186/s13059-019-1670-y</a>.</p>
</div>
<div>
<p>Stanford, William L., Georgina Caruana, Katherine A. Vallis, Maneesha Inamdar, Michihiro Hidaka, Victoria L. Bautch, and Alan Bernstein. 1998. “Expression Trapping: Identification of Novel Genes Expressed in Hematopoietic and Endothelial Lineages by Gene Trapping in ES Cells.” <em>Blood</em> 92 (12): 4622–31. <a href="https://doi.org/10.1182/blood.V92.12.4622">https://doi.org/10.1182/blood.V92.12.4622</a>.</p>
</div>
<div>
<p>Stanford, William L., Jason B. Cohn, and Sabine P. Cordes. 2001. “Gene-trap mutagenesis: past, present and beyond.” <em>Nature Reviews Genetics</em> 2 (10): 756–68. <a href="https://doi.org/10.1038/35093548">https://doi.org/10.1038/35093548</a>.</p>
</div>
<div>
<p>Stapleton, Mark. 2002. “The Drosophila Gene Collection: Identification of Putative Full-Length cDNAs for 70% of D. melanogaster Genes.” <em>Genome Research</em> 12 (8): 1294–1300. <a href="https://doi.org/10.1101/gr.269102">https://doi.org/10.1101/gr.269102</a>.</p>
</div>
<div>
<p>Ståhl, Patrik L., Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. “Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.” <em>Science</em> 353 (6294): 78–82. <a href="https://doi.org/10.1126/science.aaf2403">https://doi.org/10.1126/science.aaf2403</a>.</p>
</div>
<div>
<p>Stickels, Robert, Evan Murray, Pawan Kumar, Jilong Li, Jamie Marshall, Daniela Di Bella, Paola Arlotta, Evan Macosko, and Fei Chen. 2020. “Sensitive spatial genome wide expression profiling at cellular resolution.” <em>bioRxiv</em>, March, 2020.03.12.989806. <a href="https://doi.org/10.1101/2020.03.12.989806">https://doi.org/10.1101/2020.03.12.989806</a>.</p>
</div>
<div>
<p>Stoeger, Thomas, Nico Battich, Markus D. Herrmann, Yauhen Yakimovich, and Lucas Pelkmans. 2015. “Computer vision for image-based transcriptomics.” <em>Methods</em> 85 (September): 44–53. <a href="https://doi.org/10.1016/j.ymeth.2015.05.016">https://doi.org/10.1016/j.ymeth.2015.05.016</a>.</p>
</div>
<div>
<p>Strell, Carina, Markus M. Hilscher, Navya Laxman, Jessica Svedlund, Chenglin Wu, Chika Yokota, and Mats Nilsson. 2019. “Placing RNA in context and space – methods for spatially resolved transcriptomics.” <a href="https://doi.org/10.1111/febs.14435">https://doi.org/10.1111/febs.14435</a>.</p>
</div>
<div>
<p>Stuart, Tim, Andrew Butler, Paul Hoffman, Christoph Hafemeister, Efthymia Papalexi, William M. Mauck, Yuhan Hao, Marlon Stoeckius, Peter Smibert, and Rahul Satija. 2019. “Comprehensive Integration of Single-Cell Data.” <em>Cell</em> 177 (7): 1888–1902.e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>.</p>
</div>
<div>
<p>Su, Jing, and Qianqian Song. 2020. “DSTG: Deconvoluting Spatial Transcriptomics Data through Graph-based Artificial Intelligence.” <em>bioRxiv</em>, January, 2020.10.20.347195. <a href="https://doi.org/10.1101/2020.10.20.347195">https://doi.org/10.1101/2020.10.20.347195</a>.</p>
</div>
<div>
<p>Sulston, J., Z. Du, K. Thomas, R. Wilson, L. Hillier, R. Staden, N. Halloran, et al. 1992. “The C. elegans genome sequencing project: A beginning.” <em>Nature</em> 356 (6364): 37–41. <a href="https://doi.org/10.1038/356037a0">https://doi.org/10.1038/356037a0</a>.</p>
</div>
<div>
<p>Sun, Qian, Sherin Muckatira, Lei Yuan, Shuiwang Ji, Stuart Newfeld, Sudhir Kumar, and Jieping Ye. 2013. “Image-level and group-level models for Drosophila gene expression pattern annotation.” <em>BMC Bioinformatics</em> 14 (1): 350. <a href="https://doi.org/10.1186/1471-2105-14-350">https://doi.org/10.1186/1471-2105-14-350</a>.</p>
</div>
<div>
<p>Sun, Shiquan, Jiaqiang Zhu, and Xiang Zhou. 2020. “Statistical analysis of spatial expression patterns for spatially resolved transcriptomic studies.” <em>Nature Methods</em> 17 (2): 193–200. <a href="https://doi.org/10.1038/s41592-019-0701-7">https://doi.org/10.1038/s41592-019-0701-7</a>.</p>
</div>
<div>
<p>Sun, Yu-Chi, Xiaoyin Chen, Stephan Fischer, Shaina Lu, Jesse Gillis, and Anthony M Zador. 2020. “Integrating barcoded neuroanatomy with spatial transcriptional profiling reveals cadherin correlates of projections shared across the cortex.” <em>bioRxiv</em>, January, 2020.08.25.266460. <a href="https://doi.org/10.1101/2020.08.25.266460">https://doi.org/10.1101/2020.08.25.266460</a>.</p>
</div>
<div>
<p>Sundaresan, Venkatesan, Patricia Springer, Thomas Volpe, Samuel Haward, Jonathan D. G. Jones, Caroline Dean, Hong Ma, and Robert Martienssen. 1995. “Patterns of gene action in plant development revealed by enhancer trap and gene trap transposable elements.” <em>Genes &amp; Development</em> 9 (14): 1797–1810. <a href="https://doi.org/10.1101/gad.9.14.1797">https://doi.org/10.1101/gad.9.14.1797</a>.</p>
</div>
<div>
<p>Svensson, Valentine, Eduardo da Veiga Beltrame, and Lior Pachter. 2020. “A curated database reveals trends in single-cell transcriptomics.” <em>Database</em> 2020 (January). <a href="https://doi.org/10.1093/database/baaa073">https://doi.org/10.1093/database/baaa073</a>.</p>
</div>
<div>
<p>Svensson, Valentine, Sarah A. Teichmann, and Oliver Stegle. 2018. “SpatialDE: Identification of spatially variable genes.” <em>Nature Methods</em> 15 (5): 343–46. <a href="https://doi.org/10.1038/nmeth.4636">https://doi.org/10.1038/nmeth.4636</a>.</p>
</div>
<div>
<p>Şişecioğlu, Melda, Harun Budak, Lars Geffers, Murat Çankaya, Mehmet Çiftci, Christina Thaller, Gregor Eichele, Ömer İrfan Küfrevioğlu, and Hasan Özdemir. 2015. “A compendium of expression patterns of cholesterol biosynthetic enzymes in the mouse embryo.” <em>Journal of Lipid Research</em> 56 (8): 1551–9. <a href="https://doi.org/10.1194/jlr.M059634">https://doi.org/10.1194/jlr.M059634</a>.</p>
</div>
<div>
<p>Takei, Yodai, Jina Yun, Noah Ollikainen, Shiwei Zheng, Nico Pierson, Jonathan White, Sheel Shah, et al. 2020. “Global architecture of the nucleus in single cells by DNA seqFISH+ and multiplexed immunofluorescence.” <em>bioRxiv</em>, January, 2020.11.29.403055. <a href="https://doi.org/10.1101/2020.11.29.403055">https://doi.org/10.1101/2020.11.29.403055</a>.</p>
</div>
<div>
<p>Tanevski, Jovan, Thin Nguyen, Buu Truong, Nikos Karaiskos, Mehmet Eren Ahsen, Xinyu Zhang, Chang Shu, et al. 2020. “Gene selection for optimal prediction of cell position in tissues from single-cell transcriptomics data.” <em>Life Science Alliance</em> 3 (11): e202000867. <a href="https://doi.org/10.26508/lsa.202000867">https://doi.org/10.26508/lsa.202000867</a>.</p>
</div>
<div>
<p>Tang, Fuchou, Catalin Barbacioru, Yangzhou Wang, Ellen Nordman, Clarence Lee, Nanlan Xu, Xiaohui Wang, et al. 2009. “mRNA-Seq whole-transcriptome analysis of a single cell.” <em>Nature Methods</em> 6 (5): 377–82. <a href="https://doi.org/10.1038/nmeth.1315">https://doi.org/10.1038/nmeth.1315</a>.</p>
</div>
<div>
<p>Tang, Fuchou, Kaiqin Lao, and M Azim Surani. 2011. “Development and applications of single-cell transcriptome analysis.” <em>Nature Methods</em> 8 (4 Suppl): S6–S11. <a href="https://doi.org/10.1038/nmeth.1557">https://doi.org/10.1038/nmeth.1557</a>.</p>
</div>
<div>
<p>Tassy, Olivier, Delphine Dauga, Fabrice Daian, Daniel Sobral, François Robin, Pierre Khoueiry, David Salgado, et al. 2010. “The ANISEED database: Digital representation, formalization, and elucidation of a chordate developmental program.” <em>Genome Research</em> 20 (10): 1459–68. <a href="https://doi.org/10.1101/gr.108175.110">https://doi.org/10.1101/gr.108175.110</a>.</p>
</div>
<div>
<p>Tautz, Diethard, and Christine Pfeifle. 1989. “A non-radioactive in situ hybridization method for the localization of specific RNAs in Drosophila embryos reveals translational control of the segmentation gene hunchback.” <em>Chromosoma</em> 98 (2): 81–85. <a href="https://doi.org/10.1007/BF00291041">https://doi.org/10.1007/BF00291041</a>.</p>
</div>
<div>
<p>Tecott, L H, J D Barchas, and J H Eberwine. 1988. “In situ transcription: specific synthesis of complementary DNA in fixed tissue sections.” <em>Science</em> 240 (4859): 1661 LP–1664. <a href="https://doi.org/10.1126/science.2454508">https://doi.org/10.1126/science.2454508</a>.</p>
</div>
<div>
<p>The C. elegans Sequencing Consortium. 1998. “Genome Sequence of the Nematode C. elegans: A Platform for Investigating Biology.” <em>Science</em> 282 (5396): 2012–8. <a href="https://doi.org/10.1126/science.282.5396.2012">https://doi.org/10.1126/science.282.5396.2012</a>.</p>
</div>
<div>
<p>Thompson, Carol, Jonathan Wisor, Chang-Kyu Lee, Sayan Pathak, Dmitry Gerashchenko, Kimberly Smith, Shanna Fischer, et al. 2010. “Molecular and Anatomical Signatures of Sleep Deprivation in the Mouse Brain.” <a href="https://www.frontiersin.org/article/10.3389/fnins.2010.00165">https://www.frontiersin.org/article/10.3389/fnins.2010.00165</a>.</p>
</div>
<div>
<p>Thrane, Kim, Hanna Eriksson, Jonas Maaskola, Johan Hansson, and Joakim Lundeberg. 2018. “Spatially resolved transcriptomics enables dissection of genetic heterogeneity in stage III cutaneous malignant melanoma.” <em>Cancer Research</em>. <a href="https://doi.org/10.1158/0008-5472.CAN-18-0747">https://doi.org/10.1158/0008-5472.CAN-18-0747</a>.</p>
</div>
<div>
<p>Thut, Catherine J., Ryan B. Rountree, Michael Hwa, and David M. Kingsley. 2001. “A Large-Scale in Situ Screen Provides Molecular Evidence for the Induction of Eye Anterior Segment Structures by the Developing Lens.” <em>Developmental Biology</em> 231 (1): 63–76. <a href="https://doi.org/10.1006/dbio.2000.0140">https://doi.org/10.1006/dbio.2000.0140</a>.</p>
</div>
<div>
<p>Tietjen, Ian, Jason M Rihel, Yanxiang Cao, Georgy Koentges, Lisa Zakhary, and Catherine Dulac. 2003. “Single-Cell Transcriptional Analysis of Neuronal Progenitors.” <em>Neuron</em> 38 (2): 161–75. <a href="https://doi.org/https://doi.org/10.1016/S0896-6273(03)00229-0">https://doi.org/https://doi.org/10.1016/S0896-6273(03)00229-0</a>.</p>
</div>
<div>
<p>Tomancak, Pavel, Amy Beaton, Richard Weiszmann, Elaine Kwan, Sheng Qiang Shu, Suzanna E. Lewis, Stephen Richards, et al. 2002. “Systematic determination of patterns of gene expression during Drosophila embryogenesis.” <em>Genome Biology</em> 3 (12): research0088.1. <a href="https://doi.org/10.1186/gb-2002-3-12-research0088">https://doi.org/10.1186/gb-2002-3-12-research0088</a>.</p>
</div>
<div>
<p>Tomancak, Pavel, Benjamin P. Berman, Amy Beaton, Richard Weiszmann, Elaine Kwan, Volker Hartenstein, Susan E. Celniker, and Gerald M. Rubin. 2007. “Global analysis of patterns of gene expression during Drosophila embryogenesis.” <em>Genome Biology</em> 8 (7): R145. <a href="https://doi.org/10.1186/gb-2007-8-7-r145">https://doi.org/10.1186/gb-2007-8-7-r145</a>.</p>
</div>
<div>
<p>“Translational Pathology Core Laboratory (TPCL).” n.d. Accessed October 14, 2020. <a href="https://www.uclahealth.org/pathology/tpcl-services">https://www.uclahealth.org/pathology/tpcl-services</a>.</p>
</div>
<div>
<p>Tripodo, Claudio, Federica Zanardi, Fabio Iannelli, Saveria Mazzara, Mariella Vegliante, Gaia Morello, Arianna Di Napoli, et al. 2020. “A Spatially Resolved Dark- versus Light-Zone Microenvironment Signature Subdivides Germinal Center-Related Aggressive B Cell Lymphomas.” <em>iScience</em> 23 (10). <a href="https://doi.org/10.1016/j.isci.2020.101562">https://doi.org/10.1016/j.isci.2020.101562</a>.</p>
</div>
<div>
<p>Tuck, Elizabeth, Jeanne Estabel, Anika Oellrich, A. K. Maguire, Hibret A. Adissu, Luke Souter, Emma Siragher, et al. 2015. “A gene expression resource generated by genome-wide lacZ profiling in the mouse.” <em>Disease Models &amp; Mechanisms</em> 8 (11): 1467–78. <a href="https://doi.org/10.1242/dmm.021238">https://doi.org/10.1242/dmm.021238</a>.</p>
</div>
<div>
<p>Tzur, Yonatan B., Eitan Winter, Jinmin Gao, Tamar Hashimshony, Itai Yanai, and Monica P. Colaiácovo. 2018. “Spatiotemporal Gene Expression Analysis of the Caenorhabditis elegans Germline Uncovers a Syncytial Expression Switch.” <em>Genetics</em> 210 (2): 587–605. <a href="https://doi.org/10.1534/genetics.118.301315">https://doi.org/10.1534/genetics.118.301315</a>.</p>
</div>
<div>
<p>Urdea, M. S. 1993. “Synthesis and Characterization of Branched DNA (bDNA) for the Direct and Quantitative Detection of CMV, HBV, HCV, and HIV.” <em>Clinical Chemistry</em> 39 (4): 725–26. <a href="https://doi.org/10.1093/clinchem/39.4.725">https://doi.org/10.1093/clinchem/39.4.725</a>.</p>
</div>
<div>
<p>Valoczi, A. 2004. “Sensitive and specific detection of microRNAs by northern blot analysis using LNA-modified oligonucleotide probes.” <em>Nucleic Acids Research</em> 32 (22): e175–e175. <a href="https://doi.org/10.1093/nar/gnh171">https://doi.org/10.1093/nar/gnh171</a>.</p>
</div>
<div>
<p>Vandenbon, Alexis, and Diego Diez. 2020. “A clustering-independent method for finding differentially expressed genes in single-cell transcriptome data.” <em>Nature Communications</em> 11 (1): 4318. <a href="https://doi.org/10.1038/s41467-020-17900-3">https://doi.org/10.1038/s41467-020-17900-3</a>.</p>
</div>
<div>
<p>Van Gelder, R N, M E von Zastrow, A Yool, W C Dement, J D Barchas, and J H Eberwine. 1990. “Amplified RNA synthesized from limited quantities of heterogeneous cDNA.” <em>Proceedings of the National Academy of Sciences</em> 87 (5): 1663 LP–1667. <a href="http://www.pnas.org/content/87/5/1663.abstract">http://www.pnas.org/content/87/5/1663.abstract</a>.</p>
</div>
<div>
<p>Van Valen, David A, Takamasa Kudo, Keara M Lane, Derek N Macklin, Nicolas T Quach, Mialy M DeFelice, Inbal Maayan, Yu Tanouchi, Euan A Ashley, and Markus W Covert. 2016. “Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments.” <em>PLOS Computational Biology</em> 12 (11): e1005177. <a href="https://doi.org/10.1371/journal.pcbi.1005177">https://doi.org/10.1371/journal.pcbi.1005177</a>.</p>
</div>
<div>
<p>Venter, J. Craig, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, et al. 2001. “The Sequence of the Human Genome.” <em>Science</em> 291 (5507): 1304–51. <a href="https://doi.org/10.1126/science.1058040">https://doi.org/10.1126/science.1058040</a>.</p>
</div>
<div>
<p>“Veritas Laser Capture Microdissection (LCM) and Laser Cutting System from Applied Biosystems.” n.d. Accessed October 14, 2020. <a href="https://www.unthsc.edu/research/flow-cytometry-and-laser-capture-microdissection-core-facility/beckman-coulter-cytomics-fc500-flow-cytometry-analyzer/veritas-laser-capture">https://www.unthsc.edu/research/flow-cytometry-and-laser-capture-microdissection-core-facility/beckman-coulter-cytomics-fc500-flow-cytometry-analyzer/veritas-laser-capture</a>.</p>
</div>
<div>
<p>Verma, Archit, and Barbara Engelhardt. 2020. “A Bayesian nonparametric semi-supervised model for integration of multiple single-cell experiments.” <em>bioRxiv</em>, January, 2020.01.14.906313. <a href="https://doi.org/10.1101/2020.01.14.906313">https://doi.org/10.1101/2020.01.14.906313</a>.</p>
</div>
<div>
<p>Vickovic, Sanja, Gökcen Eraslan, Fredrik Salmén, Johanna Klughammer, Linnea Stenbeck, Denis Schapiro, Tarmo Äijö, et al. 2019. “High-definition spatial transcriptomics for in situ tissue profiling.” <em>Nature Methods</em>. <a href="https://doi.org/10.1038/s41592-019-0548-y">https://doi.org/10.1038/s41592-019-0548-y</a>.</p>
</div>
<div>
<p>Vickovic, Sanja, Britta Lötstedt, Johanna Klughammer, Åsa Segerstolpe, Orit Rozenblatt-Rosen, and Aviv Regev. 2020. “SM-Omics: An automated platform for high-throughput spatial multi-omics.” <em>bioRxiv</em>, January, 2020.10.14.338418. <a href="https://doi.org/10.1101/2020.10.14.338418">https://doi.org/10.1101/2020.10.14.338418</a>.</p>
</div>
<div>
<p>Villacampa, Eva Gracia, Ludvig Larsson, Linda Kvastad, Alma Andersson, Joseph Carlson, and Joakim Lundeberg. 2020. “Genome-wide Spatial Expression Profiling in FFPE Tissues.” <em>bioRxiv</em>, July, 2020.07.24.219758. <a href="https://doi.org/10.1101/2020.07.24.219758">https://doi.org/10.1101/2020.07.24.219758</a>.</p>
</div>
<div>
<p>Visel, Axel, Leila Taher, Hani Girgis, Dalit May, Olga Golonzhka, Renee V. Hoch, Gabriel L. McKinsey, et al. 2013. “A high-resolution enhancer atlas of the developing telencephalon.” <em>Cell</em> 152 (4): 895–908. <a href="https://doi.org/10.1016/j.cell.2012.12.041">https://doi.org/10.1016/j.cell.2012.12.041</a>.</p>
</div>
<div>
<p>Waldhaus, Jörg, Robert Durruthy-Durruthy, and Stefan Heller. 2015. “Quantitative High-Resolution Cellular Map of the Organ of Corti.” <em>Cell Reports</em> 11 (9): 1385–99. <a href="https://doi.org/10.1016/j.celrep.2015.04.062">https://doi.org/10.1016/j.celrep.2015.04.062</a>.</p>
</div>
<div>
<p>Wang, Chong, Tian Lu, George Emanuel, Hazen P. Babcock, and Xiaowei Zhuang. 2019. “Imaging-based pooled CRISPR screening reveals regulators of lncRNA localization.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1903808116">https://doi.org/10.1073/pnas.1903808116</a>.</p>
</div>
<div>
<p>Wang, Fay, John Flanagan, Nan Su, Li-Chong Wang, Son Bui, Allissa Nielson, Xingyong Wu, Hong-Thuy Vo, Xiao-Jun Ma, and Yuling Luo. 2012. “RNAscope.” <em>The Journal of Molecular Diagnostics</em> 14 (1): 22–29. <a href="https://doi.org/10.1016/j.jmoldx.2011.08.002">https://doi.org/10.1016/j.jmoldx.2011.08.002</a>.</p>
</div>
<div>
<p>Wang, Guiping, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2018. “Multiplexed imaging of high-density libraries of RNAs with MERFISH and expansion microscopy.” <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/s41598-018-22297-7">https://doi.org/10.1038/s41598-018-22297-7</a>.</p>
</div>
<div>
<p>Wang, Xiao, William E. Allen, Matthew A. Wright, Emily L. Sylwestrak, Nikolay Samusik, Sam Vesuna, Kathryn Evans, et al. 2018. “Three-dimensional intact-tissue sequencing of single-cell transcriptional states.” <em>Science</em> 361 (6400): eaat5691. <a href="https://doi.org/10.1126/science.aat5691">https://doi.org/10.1126/science.aat5691</a>.</p>
</div>
<div>
<p>Wang, Yuhan, Mark Eddison, Greg Fleishman, Martin Weigert, Shengjin Xu, Fredrick E Henry, Tim Wang, et al. 2021. “Expansion-Assisted Iterative-FISH defines lateral hypothalamus spatio-molecular organization.” <em>bioRxiv</em>, January, 2021.03.08.434304. <a href="https://doi.org/10.1101/2021.03.08.434304">https://doi.org/10.1101/2021.03.08.434304</a>.</p>
</div>
<div>
<p>Waterston, Robert H., Kerstin Lindblad-Toh, Ewan Birney, Jane Rogers, Josep F. Abril, Pankaj Agarwal, Richa Agarwala, et al. 2002. “Initial sequencing and comparative analysis of the mouse genome.” <em>Nature</em> 420 (6915): 520–62. <a href="https://doi.org/10.1038/nature01262">https://doi.org/10.1038/nature01262</a>.</p>
</div>
<div>
<p>Waylen, Lisa N, Hieu T Nim, Luciano G Martelotto, and Mirana Ramialison. 2020. “From whole-mount to single-cell spatial assessment of gene expression in 3D.” <em>Communications Biology</em> 3 (1): 602. <a href="https://doi.org/10.1038/s42003-020-01341-1">https://doi.org/10.1038/s42003-020-01341-1</a>.</p>
</div>
<div>
<p>Weinstein, Joshua A., Aviv Regev, and Feng Zhang. 2019. “DNA Microscopy: Optics-free Spatio-genetic Imaging by a Stand-Alone Chemical Reaction.” <em>Cell</em> 178 (1): 229–241.e16. <a href="https://doi.org/10.1016/j.cell.2019.05.019">https://doi.org/10.1016/j.cell.2019.05.019</a>.</p>
</div>
<div>
<p>Welch, Joshua D., Velina Kozareva, Ashley Ferreira, Charles Vanderburg, Carly Martin, and Evan Z. Macosko. 2019. “Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity.” <em>Cell</em> 177 (7): 1873–1887.e17. <a href="https://doi.org/10.1016/j.cell.2019.05.006">https://doi.org/10.1016/j.cell.2019.05.006</a>.</p>
</div>
<div>
<p>West, David B., Ravi K. Pasumarthi, Brian Baridon, Esi Djan, Amanda Trainor, Stephen M. Griffey, Eric K. Engelhard, et al. 2015. “A lacZ reporter gene expression atlas for 313 adult KOMP mutant mouse lines.” <em>Genome Research</em> 25 (4): 598–607. <a href="https://doi.org/10.1101/gr.184184.114">https://doi.org/10.1101/gr.184184.114</a>.</p>
</div>
<div>
<p>Westerfield, Monte, Eckehard Doerry, Arthur E. Kirkpatrick, Wolfgang Driever, and Sarah A. Douglas. 1997. “An on-line database for zebrafish development and genetics research.” <em>Seminars in Cell &amp; Developmental Biology</em> 8 (5): 477–88. <a href="https://doi.org/10.1006/scdb.1997.0173">https://doi.org/10.1006/scdb.1997.0173</a>.</p>
</div>
<div>
<p>White, Jacqueline K., Anna-Karin Gerdin, Natasha A. Karp, Ed Ryder, Marija Buljan, James N. Bussell, Jennifer Salisbury, et al. 2013. “Genome-wide Generation and Systematic Phenotyping of Knockout Mice Reveals New Roles for Many Genes.” <em>Cell</em> 154 (2): 452–64. <a href="https://doi.org/10.1016/j.cell.2013.06.022">https://doi.org/10.1016/j.cell.2013.06.022</a>.</p>
</div>
<div>
<p>Wienholds, Erno. 2005. “MicroRNA Expression in Zebrafish Embryonic Development.” <em>Science</em> 309 (5732): 310–11. <a href="https://doi.org/10.1126/science.1114519">https://doi.org/10.1126/science.1114519</a>.</p>
</div>
<div>
<p>Wilson, Clive, Rebecca Kurth Pearson, Hugo J Bellen, C J O’Kane, Ueli Grossniklaus, and Walter J Gehring. 1989. “P-element-mediated enhancer detection: an efficient method for isolating and characterizing developmentally regulated genes in Drosophila.” <em>Genes &amp; Development</em> 3 (9): 1301–13. <a href="https://doi.org/10.1101/gad.3.9.1301">https://doi.org/10.1101/gad.3.9.1301</a>.</p>
</div>
<div>
<p>Wu, Siqi, Antony Joseph, Ann S. Hammonds, Susan E. Celniker, Bin Yu, and Erwin Frise. 2016. “Stability-driven nonnegative matrix factorization to interpret spatial gene expression and build local gene networks.” <em>Proceedings of the National Academy of Sciences</em> 113 (16): 4290–5. <a href="https://doi.org/10.1073/pnas.1521171113">https://doi.org/10.1073/pnas.1521171113</a>.</p>
</div>
<div>
<p>Wurst, W, J Rossant, V Prideaux, M Kownacka, A Joyner, D P Hill, F Guillemot, S Gasca, D Cado, and A Auerbach. 1995. “A large-scale gene-trap screen for insertional mutations in developmentally regulated genes in mice.” <em>Genetics</em> 139 (2). <a href="https://www.genetics.org/content/139/2/889">https://www.genetics.org/content/139/2/889</a>.</p>
</div>
<div>
<p>“XDB3.” 2004. <a href="http://xenopus.nibb.ac.jp/">http://xenopus.nibb.ac.jp/</a>.</p>
</div>
<div>
<p>Xia, Chenglong, Hazen P. Babcock, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2019. “Multiplexed detection of RNA using MERFISH and branched DNA amplification.” <em>Scientific Reports</em> 9 (1): 1–13. <a href="https://doi.org/10.1038/s41598-019-43943-8">https://doi.org/10.1038/s41598-019-43943-8</a>.</p>
</div>
<div>
<p>Xia, Chenglong, Jean Fan, George Emanuel, Junjie Hao, and Xiaowei Zhuang. 2019. “Spatial transcriptome profiling by MERFISH reveals subcellular RNA compartmentalization and cell cycle-dependent gene expression.” <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1912459116">https://doi.org/10.1073/pnas.1912459116</a>.</p>
</div>
<div>
<p>Xu, Xiangmin, Todd C Holmes, Min-Hua Luo, Kevin T Beier, Gregory D Horwitz, Fei Zhao, Wenbo Zeng, May Hui, Bert L Semler, and Rozanne M Sandri-Goldin. 2020. “Viral Vectors for Neural Circuit Mapping and Recent Advances in Trans-synaptic Anterograde Tracers.” <em>Neuron</em> 107 (6): 1029–47. <a href="https://doi.org/https://doi.org/10.1016/j.neuron.2020.07.010">https://doi.org/https://doi.org/10.1016/j.neuron.2020.07.010</a>.</p>
</div>
<div>
<p>Yang, Xiangdong W., Peter Model, and Nathaniel Heintz. 1997. “Homologous recombination based modification in Esherichia coli and germline transmission in transgenic mice of a bacterial artificial chromsome.” <em>Nature Biotechnology</em> 15 (9): 859–65. <a href="https://doi.org/10.1038/nbt0997-859">https://doi.org/10.1038/nbt0997-859</a>.</p>
</div>
<div>
<p>Yang, Xiaoyu, Seth Bergenholtz, Lenka Maliskova, Mark-Phillip Pebworth, Arnold R Kriegstein, Yun Li, and Yin Shen. 2020. “SMART-Q: An Integrative Pipeline Quantifying Cell Type-Specific RNA Transcription.” <em>PLOS ONE</em> 15 (4): e0228760. <a href="https://doi.org/10.1371/journal.pone.0228760">https://doi.org/10.1371/journal.pone.0228760</a>.</p>
</div>
<div>
<p>Yang, Yang, Qingwei Fang, and Hong-Bin Shen. 2019. “Predicting gene regulatory interactions based on spatial gene expression data and deep learning.” Edited by Smita Krishnaswamy. <em>PLOS Computational Biology</em> 15 (9): e1007324. <a href="https://doi.org/10.1371/journal.pcbi.1007324">https://doi.org/10.1371/journal.pcbi.1007324</a>.</p>
</div>
<div>
<p>Yaylaoglu, Murat Burak, Andrew Titmus, Axel Visel, Gonzalo Alvarez-Bolado, Christina Thaller, and Gregor Eichele. 2005. “Comprehensive expression atlas of fibroblast growth factors and their receptors generated by a novel robotic in situ hybridization platform.” <em>Developmental Dynamics</em> 234 (2): 371–86. <a href="https://doi.org/10.1002/dvdy.20441">https://doi.org/10.1002/dvdy.20441</a>.</p>
</div>
<div>
<p>Yoda, Takuya, Masahito Hosokawa, Kiyofumi Takahashi, Chikako Sakanashi, Haruko Takeyama, and Hideki Kambara. 2017. “Site-specific gene expression analysis using an automated tissue micro-dissection punching system.” <em>Scientific Reports</em> 7 (1): 4325. <a href="https://doi.org/10.1038/s41598-017-04616-6">https://doi.org/10.1038/s41598-017-04616-6</a>.</p>
</div>
<div>
<p>Yokoyama, Shigetoshi, Yoshiaki Ito, Hiroe Ueno-Kudoh, Hirohito Shimizu, Kenta Uchibe, Sonia Albini, Kazuhiko Mitsuoka, et al. 2009. “A Systems Approach Reveals that the Myogenesis Genome Network Is Regulated by the Transcriptional Repressor RP58.” <em>Developmental Cell</em> 17 (6): 836–48. <a href="https://doi.org/10.1016/j.devcel.2009.10.011">https://doi.org/10.1016/j.devcel.2009.10.011</a>.</p>
</div>
<div>
<p>Yoshikawa, Toshiyuki, Yulan Piao, Jinhui Zhong, Ryo Matoba, Mark G. Carter, Yuxia Wang, Ilya Goldberg, and Minoru S. H. Ko. 2006. “High-throughput screen for genes predominantly expressed in the ICM of mouse blastocysts by whole mount in situ hybridization.” <em>Gene Expression Patterns</em> 6 (2): 213–24. <a href="https://doi.org/10.1016/j.modgep.2005.06.003">https://doi.org/10.1016/j.modgep.2005.06.003</a>.</p>
</div>
<div>
<p>Yuan, Lei, Alexander Woodard, Shuiwang Ji, Yuan Jiang, Zhi-Hua Zhou, Sudhir Kumar, and Jieping Ye. 2012. “Learning Sparse Representations for Fruit-Fly Gene Expression Pattern Image Annotation and Retrieval.” <em>BMC Bioinformatics</em> 13 (1): 107. <a href="https://doi.org/10.1186/1471-2105-13-107">https://doi.org/10.1186/1471-2105-13-107</a>.</p>
</div>
<div>
<p>Yuan, Ye, and Ziv Bar-Joseph. 2019. “GCNG: Graph convolutional networks for inferring cell-cell interactions.” <em>bioRxiv</em>, January, 2019.12.23.887133. <a href="https://doi.org/10.1101/2019.12.23.887133">https://doi.org/10.1101/2019.12.23.887133</a>.</p>
</div>
<div>
<p>Zechel, Sabrina, Pawel Zajac, Peter Lönnerberg, Carlos F. Ibáñez, and Sten Linnarsson. 2014. “Topographical transcriptome mapping of the mouse medial ganglionic eminence by spatially resolved RNA-seq.” <em>Genome Biology</em> 15 (10): 486. <a href="https://doi.org/10.1186/s13059-014-0486-z">https://doi.org/10.1186/s13059-014-0486-z</a>.</p>
</div>
<div>
<p>Zeisel, Amit, Hannah Hochgerner, Peter Lönnerberg, Anna Johnsson, Fatima Memic, Job van der Zwan, Martin Häring, et al. 2018. “Molecular Architecture of the Mouse Nervous System.” <em>Cell</em> 174 (4): 999–1014.e22. <a href="https://doi.org/10.1016/j.cell.2018.06.021">https://doi.org/10.1016/j.cell.2018.06.021</a>.</p>
</div>
<div>
<p>Zeng, Tao, Rongjian Li, Ravi Mukkamala, Jieping Ye, and Shuiwang Ji. 2015. “Deep convolutional neural networks for annotating gene expression patterns in the mouse brain.” <em>BMC Bioinformatics</em> 16 (1): 147. <a href="https://doi.org/10.1186/s12859-015-0553-9">https://doi.org/10.1186/s12859-015-0553-9</a>.</p>
</div>
<div>
<p>Zhang, Ke, Wanwan Feng, and Peng Wang. 2018. “Identification of spatially variable genes with graph cuts.” <em>bioRxiv</em>, January, 491472. <a href="https://doi.org/10.1101/491472">https://doi.org/10.1101/491472</a>.</p>
</div>
<div>
<p>Zhang, Meng, Stephen W Eichhorn, Brian Zingg, Zizhen Yao, Hongkui Zeng, Hongwei Dong, and Xiaowei Zhuang. 2020. “Molecular, spatial and projection diversity of neurons in primary motor cortex revealed by in situ single-cell transcriptomics.” <em>bioRxiv</em>, January, 2020.06.04.105700. <a href="https://doi.org/10.1101/2020.06.04.105700">https://doi.org/10.1101/2020.06.04.105700</a>.</p>
</div>
<div>
<p>Zhang, Wenlu, Daming Feng, Rongjian Li, Andrey Chernikov, Nikos Chrisochoides, Christopher Osgood, Charlotte Konikoff, Stuart Newfeld, Sudhir Kumar, and Shuiwang Ji. 2013. “A mesh generation and machine learning framework for Drosophila gene expression pattern image analysis.” <em>BMC Bioinformatics</em> 14 (1): 372. <a href="https://doi.org/10.1186/1471-2105-14-372">https://doi.org/10.1186/1471-2105-14-372</a>.</p>
</div>
<div>
<p>Zhao, Edward, Matthew R Stone, Xing Ren, Thomas Pulliam, Paul Nghiem, Jason H Bielas, and Raphael Gottardo. 2020. “BayesSpace enables the robust characterization of spatial gene expression architecture in tissue sections at increased resolution.” <em>bioRxiv</em>, January, 2020.09.04.283812. <a href="https://doi.org/10.1101/2020.09.04.283812">https://doi.org/10.1101/2020.09.04.283812</a>.</p>
</div>
<div>
<p>Zhao, Tuo, Han Liu, Kathryn Roeder, John Lafferty, and Larry Wasserman. 2012. “The Huge Package for High-Dimensional Undirected Graph Estimation in R.” <em>J. Mach. Learn. Res.</em> 13 (null): 1059–62.</p>
</div>
<div>
<p>Zheng, Grace X. Y., Jessica M. Terry, Phillip Belgrader, Paul Ryvkin, Zachary W. Bent, Ryan Wilson, Solongo B. Ziraldo, et al. 2017. “Massively parallel digital transcriptional profiling of single cells.” <em>Nature Communications</em> 8 (1): 14049. <a href="https://doi.org/10.1038/ncomms14049">https://doi.org/10.1038/ncomms14049</a>.</p>
</div>
<div>
<p>Zhou, Chuan, Ru Huang, Xiaoming Zhou, and Da Xing. 2020. “Sensitive and specific microRNA detection by RNA dependent DNA ligation and rolling circle optical signal amplification.” <em>Talanta</em> 216: 120954. <a href="https://doi.org/https://doi.org/10.1016/j.talanta.2020.120954">https://doi.org/https://doi.org/10.1016/j.talanta.2020.120954</a>.</p>
</div>
<div>
<p>Zhou, Jie, and Hanchuan Peng. 2007. “Automatic recognition and annotation of gene expression patterns of fly embryos.” <em>Bioinformatics</em> 23 (5): 589–96. <a href="https://doi.org/10.1093/bioinformatics/btl680">https://doi.org/10.1093/bioinformatics/btl680</a>.</p>
</div>
<div>
<p>Zhou, Wen, Mary A. Yui, Brian A. Williams, Jina Yun, Barbara J. Wold, Long Cai, and Ellen V. Rothenberg. 2019. “Single-Cell Analysis Reveals Regulatory Gene Expression Dynamics Leading to Lineage Commitment in Early T Cell Development.” <em>Cell Systems</em> 9 (4): 321–337.e9. <a href="https://doi.org/10.1016/j.cels.2019.09.008">https://doi.org/10.1016/j.cels.2019.09.008</a>.</p>
</div>
<div>
<p>Zhu, Junjie, and Chiara Sabatti. 2020. “Integrative Spatial Single-cell Analysis with Graph-based Feature Learning.” <em>bioRxiv</em>, January, 2020.08.12.248971. <a href="https://doi.org/10.1101/2020.08.12.248971">https://doi.org/10.1101/2020.08.12.248971</a>.</p>
</div>
<div>
<p>Zhu, Qian, Sheel Shah, Ruben Dries, Long Cai, and Guo Cheng Yuan. 2018. “Identification of spatially associated subpopulations by combining scRNAseq and sequential fluorescence in situ hybridization data.” <em>Nature Biotechnology</em> 36 (12): 1183–90. <a href="https://doi.org/10.1038/nbt.4260">https://doi.org/10.1038/nbt.4260</a>.</p>
</div>
<div>
<p>Zhu, Y Y, E M Machleder, A Chenchik, R Li, and P D Siebert. 2001. “Reverse Transcriptase Template Switching: A SMART™ Approach for Full-Length cDNA Library Construction.” <em>BioTechniques</em> 30 (4): 892–97. <a href="https://doi.org/10.2144/01304pf02">https://doi.org/10.2144/01304pf02</a>.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="future.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["museumst.pdf"],
"toc": {
"collapse": "subsection",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
