# Introduction {#intro}

The spatial organization of the components of biological systems is crucial for their proper function. For instance, morphogen gradients in embryos are tightly regulated to ensure that the right cell types differentiate at the right place. In adults, spatial organization of cells in tissues is important to proper functions of organs. For instance, the liver lobule is divided in labor according to distance from the portal triad as such distance affects suitability of different tasks. Both oxygen level and morphogen gradient regulate zonation of metabolism [@Gebhardt2014]; there is more oxidative phosphorylation and gluconeogenesis in the more oxygenated periportal region and more glycolysis in the more deoxygenated pericentral region. How cell types and cellular functions vary in space is measured by quantifying gene expression in space. Conversely, the expression of an unknown gene in space can give clues to its function. Gene expression is usually quantified by quantifying proteins or transcripts encoded by the gene, and high throughput spatial methods exist for both protein and transcripts. In other words, cellular function exemplifies the maxim that "the whole is greater than the sum of its parts", and in large part this follows from "location, location, location". 

Here we focus on spatial transcriptomics (the field of spatial proteomics is covered elsewhere [@Lundberg2019; @Baharlou2019; @Buchberger2018]). Even spatial transcriptomics is a vast field, and it is useful to begin by considering the scope of what it contains. Na√Øvely, one may say, spatial transcriptomics means quantifying the complete set of RNAs encoded by the genome in space. Usually the "in space" is at some microscopic resolution rather than geospatial as often assumed in the term "spatial statistics"; the resolution is usually cellular, though sometimes subcellular. The "spatial" is in contrast to other transcriptomics methods that by virtue of the nature of their assays, lose information of tissue structure in space. That is the case with microarray technology for bulk tissue analysis, for bulk RNA-seq, and single cell RNA-seq (scRNA-seq) that is based on dissociation of tissue -- the "spatial" usually means tissue structure in space. More broadly, the "spatial" can mean knowing spatial context of samples although the spatial context is only a label and the coordinates are not collected or not used, such as in some laser capture microdissection (LCM) literature [@Baccin2020; @Nichterwitz2016], Niche-seq [@Medaglia2017], and APEX-seq [@Fazal2019a]. The "spatial" can also mean preserving spatial coordinates of samples within tissue, though the coordinates may or may not be explicitly used in data analysis, such as in the various single molecular fluorescent in situ hybridization (smFISH) based technologies such as seqFISH [@Lubeck2014] and MERFISH [@Chen2015] and array based technologies such as Spatial Transciptomics (ST) [@Stahl2016a].

There is more complexity in defining "transcriptomics". While some technologies usually called "spatial transcriptomics" are indeed transcriptome-wide, such as ST, Visium, and LCM followed by RNA-seq, many technologies that only profile a panel of usually a few hundred genes are nevertheless considered part of "spatial transcriptomics". Here "transcriptomics" actually means high-throughput quantification of gene expression, preferably highly multiplexed, quantifying numerous genes within the same piece of tissue at the same time. However, what counts as "high-throughput"? Is there a minimum number of genes required? Should 50 genes be enough? Or a hundred genes? The threshold number of genes required to be considered "high-throughput" is difficult to define; here, by "high-throughput", we mean the intent to quantify expression of more genes than normally done with fluorescent in situ hybridization (FISH) or immunofluorescence when only color distinguishes between genes, which can mean more than about 5 genes. There is also some complication regarding whether "highly multiplexed" should be required. Some fairly recent studies that intended to perform high-throughput gene expression profiling in space did not profile most genes at the same time (e.g. multiple rounds of smFISH hybridization, each round for a different set of genes) [@Lignell2017], or even profiled different genes in different tissue sections [@Bayraktar2020; @Battich2013]; these papers nevertheless claimed to be spatial transcriptomic or something similar.

When terms are to be defined by how they are used, then we rely on a generic and inclusive definition of "spatial transcriptomics", which can be summarized as: Quantifying transcripts while keeping spatial context of samples within tissue or cell, with intent to quantify transcripts of more genes than normally done with one round of FISH or immunofluorescence when color is the only way to distinguish between genes. This is the criterion we used in considering what methods to include in our review.

The field of spatial transcriptomics has grown drastically in the past 5 years, during which several reviews have already been written. These survey existing technologies [@Crosetto2015; @Moor2017; @Strell2019; @Liao2020; @Waylen2020] or discuss how the technologies apply to specific biological systems such as tumors [@Smith2019a], brain [@Lein2017], and liver [@Saviano2020]. Unlike the review papers, we aim to be more systematic and detailed in our review of spatial transcriptomics technology. In addition, we review existing data analysis methods in this field, a crucial aspect of spatial transcriptomics which has not yet been reviewed in depth. Moreover, we present a curated database of spatial transcriptomics literature and analyses of the literature metadata to show trends in different aspects of spatial transcriptomics. This database is publicly available [here](https://docs.google.com/spreadsheets/d/1sJDb9B7AtYmfKv4-m8XR7uc3XXw_k4kGSout8cqZ8bY/edit#gid=1693202466). A similar database has been curated for scRNA-seq literature, which has been analyzed to show trends in the field [@Svensson2020] although the metadata in our database and the analyses are much more extensive.

There are some caveats to our review and database. First, while we narrate a history of evolution of techniques and in some cases explain how one technique influenced another, we do not present aspects of the history that are not apparent from the publications. Studying those aspects of the history of the field may require interviewing the people who developed the techniques, as well as exploration of additional unpublished material. Second, our database was originally only meant for papers, so relevant materials that are not in presented in that format are underrepresented. Examples of such materials include databases and software not presented as papers (e.g. the XDB3 database [@XDB3]). This means that the metadata analyses in this book might not be representative of all material that exists in spatial transcriptomics. Third, as the curation was done manually, the database might not include some relevant literature unknown to us.

Our database starts with articles published in the 1980s. This provides historical context of what is now commonly known as spatial transcriptomics; this literature is summarized in Chapter \@ref(prequel), and historical methods of data analysis are reviewed in Chapter \@ref(prequel-analysis). The literature is broken down into the following categories, to be defined and elaborated on in the subsequent chapters. Technologies to collect data (Chapter \@ref(current)): Microdissection (Section \@ref(microdissection)), array based techniques (Section \@ref(array)), single molecular FISH (smFISH) (Section \@ref(smfish)), in situ sequencing (ISS) (Section \@ref(iss)), and no imaging (Section \@ref(no-imaging)). Data analysis methods (Chapter \@ref(current-analysis)): Preprocessing (Section \@ref(preprocessing)), exploratory data analysis (EDA) (Section \@ref(eda)), spatial reconstruction of single cell RNA-seq (scRNA-seq) data (Section \@ref(reconstruction)), spatially variable genes (Section \@ref(variable)), archetypal gene expression patterns (Section \@ref(pattern)), using transcriptome to identify spatially coherent regions in tissue (Section \@ref(region)), cell type deconvolution of non-single-cell resolution spatial data (Section \@ref(deconvolution)), cell-cell interaction (Section \@ref(cell-interaction)), and other types of analyses. Also, the literature metadata is analyzed to show relevant sociological trends such as who is using each technology, usage trends of technologies, and the programming languages used. The metadata analyses can be run interactively in [RStudio Cloud](https://rstudio.cloud/project/1981124).
